<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/29/2020.03.28.013276.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.28.013276v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry" />
<meta name="DC.Identifier" content="10.1101/2020.03.28.013276" />
<meta name="DC.Date" content="2020-03-29" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="Â© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The global pandemic of severe acute pneumonia syndrome (COVID-19) caused by SARS-CoV-2 urgently calls for prevention and intervention strategies. The densely glycosylated spike (S) protein highly exposed on the viral surface is a determinant for virus binding and invasion into host cells as well as elicitation of a protective host immune response. Herein, we characterized the site-specific N-glycosylation of SARS-CoV-2 S protein using stepped collision energy (SCE) mass spectrometry (MS). Following digestion with two complementary proteases to cover all potential N-glycosylation sequons and integrated N-glycoproteomics analysis, we revealed the N-glycosylation profile of SARS-CoV-2 S proteins at the levels of intact N-glycopeptides and glycosites, along with the glycan composition and site-specific number of glycans. All 22 potential canonical N-glycosites were identified in S protein protomer. Of those, 18 N-glycosites were conserved between SARS-CoV and SARS-CoV-2 S proteins. Nearly all glycosites were preserved among the 753 SARS-CoV-2 genome sequences available in the public influenza database Global Initiative on Sharing All Influenza Data. By comparison, insect cell-expressed SARS-CoV-2 S protein contained 38 N-glycans, which were primarily assigned to the high-mannose type N-glycans, whereas the human cell-produced protein possessed up to 140 N-glycans largely belonging to the complex type. In particular, two N-glycosites located in the structurally exposed receptor-binding domain of S protein exhibited a relatively conserved N-glycan composition in human cells. This N-glycosylation profiling and determination of differences between distinct expression systems could shed light on the infection mechanism and promote development of vaccines and targeted drugs." />
<meta name="DC.Contributor" content="Yong Zhang" />
<meta name="DC.Contributor" content="Wanjun Zhao" />
<meta name="DC.Contributor" content="Yonghong Mao" />
<meta name="DC.Contributor" content="Shisheng Wang" />
<meta name="DC.Contributor" content="Yi Zhong" />
<meta name="DC.Contributor" content="Tao Su" />
<meta name="DC.Contributor" content="Meng Gong" />
<meta name="DC.Contributor" content="Xiaofeng Lu" />
<meta name="DC.Contributor" content="Jingqiu Cheng" />
<meta name="DC.Contributor" content="Hao Yang" />
<meta name="article:published_time" content="2020-03-29" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The global pandemic of severe acute pneumonia syndrome (COVID-19) caused by SARS-CoV-2 urgently calls for prevention and intervention strategies. The densely glycosylated spike (S) protein highly exposed on the viral surface is a determinant for virus binding and invasion into host cells as well as elicitation of a protective host immune response. Herein, we characterized the site-specific N-glycosylation of SARS-CoV-2 S protein using stepped collision energy (SCE) mass spectrometry (MS). Following digestion with two complementary proteases to cover all potential N-glycosylation sequons and integrated N-glycoproteomics analysis, we revealed the N-glycosylation profile of SARS-CoV-2 S proteins at the levels of intact N-glycopeptides and glycosites, along with the glycan composition and site-specific number of glycans. All 22 potential canonical N-glycosites were identified in S protein protomer. Of those, 18 N-glycosites were conserved between SARS-CoV and SARS-CoV-2 S proteins. Nearly all glycosites were preserved among the 753 SARS-CoV-2 genome sequences available in the public influenza database Global Initiative on Sharing All Influenza Data. By comparison, insect cell-expressed SARS-CoV-2 S protein contained 38 N-glycans, which were primarily assigned to the high-mannose type N-glycans, whereas the human cell-produced protein possessed up to 140 N-glycans largely belonging to the complex type. In particular, two N-glycosites located in the structurally exposed receptor-binding domain of S protein exhibited a relatively conserved N-glycan composition in human cells. This N-glycosylation profiling and determination of differences between distinct expression systems could shed light on the infection mechanism and promote development of vaccines and targeted drugs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.28.013276v1" />
<meta name="citation_id" content="2020.03.28.013276v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.28.013276" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.28.013276" />
<meta name="citation_author" content="Yong Zhang" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0513-8941" />
<meta name="citation_author" content="Wanjun Zhao" />
<meta name="citation_author_institution" content="Department of Thyroid Surgery, West China Hospital, Sichuan University" />
<meta name="citation_author" content="Yonghong Mao" />
<meta name="citation_author_institution" content="Thoracic Surgery Research Laboratory, West China Hospital, Sichuan University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9952-399X" />
<meta name="citation_author" content="Shisheng Wang" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author" content="Yi Zhong" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author" content="Tao Su" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author" content="Meng Gong" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author" content="Xiaofeng Lu" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author" content="Jingqiu Cheng" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author" content="Hao Yang" />
<meta name="citation_author_institution" content="Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital" />
<meta name="citation_author_email" content="yanghao@scu.edu.cn" />
<meta name="citation_reference" content="Lu, R.; Zhao, X.; Li, J.; Niu, P.; Yang, B.; Wu, H.; Wang, W.; Song, H.; Huang, B.; Zhu, N.; Bi, Y.; Ma, X.; Zhan, F.; Wang, L.; Hu, T.; Zhou, H.; Hu, Z.; Zhou, W.; Zhao, L.; Chen, J., et al. Lancet 2020, 395, 565â574." />
<meta name="citation_reference" content="Zhou, P.; Yang, X.-L.; Wang, X.-G.; Hu, B.; Zhang, L.; Zhang, W.; Si, H.-R.; Zhu, Y.; Li, B.; Huang, C.-L.; Chen, H.-D.; Chen, J.; Luo, Y.; Guo, H.; Jiang, R.-D.; Liu, M.-Q.; Chen, Y.; Shen, X.-R.; Wang, X.; Zheng, X.-S., et al. Nature 2020, 10.1038/s41586-41020-42012-41587." />
<meta name="citation_reference" content="Drosten, C.; GÃ¼nther, S.; Preiser, W.; van der Werf, S.; Brodt, H.-R.; Becker, S.; Rabenau, H.; Panning, M.; Kolesnikova, L.; Fouchier, R. A. M.; Berger, A.; BurguiÃ¨re, A.-M.; Cinatl, J.; Eickmann, M.; Escriou, N.; Grywna, K.; Kramme, S.; Manuguerra, J.-C.; MÃ¼ller, S.; Rickerts, V., et al. N Engl J Med 2003, 348, 1967â1976." />
<meta name="citation_reference" content="Wu, F.; Zhao, S.; Yu, B.; Chen, Y. M.; Wang, W.; Song, Z. G.; Hu, Y.; Tao, Z. W.; Tian, J. H.; Pei, Y. Y.; Yuan, M. L.; Zhang, Y. L.; Dai, F. H.; Liu, Y.; Wang, Q. M.; Zheng, J. J.; Xu, L.; Holmes, E. C.; Zhang, Y. Z. Nature 2020, 579, 265â269." />
<meta name="citation_reference" content="Wu, A.; Peng, Y.; Huang, B.; Ding, X.; Wang, X.; Niu, P.; Meng, J.; Zhu, Z.; Zhang, Z.; Wang, J.; Sheng, J.; Quan, L; Xia, Z.; Tan, W.; Cheng, G.; Jiang, T. Cell Host Microbe 2020, S1931-3128(1920)30072-X." />
<meta name="citation_reference" content="Li, W.; Moore, M. J.; Vasilieva, N.; Sui, J.; Wong, S. K.; Berne, M. A.; Somasundaran, M.; Sullivan, J. L.; Luzuriaga, K.; Greenough, T. C.; Choe, H.; Farzan, M. Nature 2003, 426, 450â454." />
<meta name="citation_reference" content="Wrapp, D.; Wang, N.; Corbett, K. S.; Goldsmith, J. A.; Hsieh, C. L.; Abiona, O.; Graham, B. S.; McLellan, J. S. Science 2020, 367, 1260â1263." />
<meta name="citation_reference" content="Du, L.; He, Y.; Zhou, Y.; Liu, S.; Zheng, B. J.; Jiang, S. Nat Rev Microbiol 2009, 7, 226â236." />
<meta name="citation_reference" content="Chen, W.-H.; Strych, U.; Hotez, P. J.; Bottazzi, M. E. Current Tropical Medicine Reports 2020." />
<meta name="citation_reference" content="Walls, A. C.; Park, Y. J.; Tortorici, M. A.; Wall, A.; McGuire, A. T.; Veesler, D. Cell 2020." />
<meta name="citation_reference" content="Walls, A. C.; Xiong, X.; Park, Y. J.; Tortorici, M. A.; Snijder, J.; Quispe, J.; Cameroni, E.; Gopal, R.; Dai, M.; Lanzavecchia, A.; Zambon, M.; Rey, F. A.; Corti, D.; Veesler, D. Cell 2019, 176, 1026â1039 e1015." />
<meta name="citation_reference" content="Li, F. Annual Review of Virology 2016, 3, 237â261." />
<meta name="citation_reference" content="Yan, R.; Zhang, Y.; Li, Y.; Xia, L.; Guo, Y.; Zhou, Q. Science 2020." />
<meta name="citation_reference" content="Walls, A. C.; Tortorici, M. A.; Frenz, B.; Snijder, J.; Li, W.; Rey, F. A.; DiMaio, F.; Bosch, B. J.; Veesler, D. Nat Struct Mol Biol 2016, 23, 899â905." />
<meta name="citation_reference" content="Clerc, F.; Reiding, K. R.; Jansen, B. C.; Kammeijer, G. S.; Bondt, A.; Wuhrer, M. Glycoconj J 2016, 33, 309â343." />
<meta name="citation_reference" content="Chang, D.; Zaia, J. Mol Cell Proteomics 2019, 18, 2348â2358." />
<meta name="citation_reference" content="Medzihradszky, K. F.; Maynard, J.; Kaasik, K.; Bern, M. Molecular &amp; Cellular Proteomics 2014, 13, S36âS36." />
<meta name="citation_reference" content="Kost, T. A.; Condreay, J. P.; Jarvis, D. L. Nat Biotechnol 2005, 23, 567â575." />
<meta name="citation_reference" content="Wu, F.; Zhao, S.; Yu, B.; Chen, Y.-M.; Wang, W.; Song, Z.-G.; Hu, Y.; Tao, Z.-W.; Tian, J.-H.; Pei, Y.-Y.; Yuan, M.-L.; Zhang, Y.-L.; Dai, F.-H.; Liu, Y.; Wang, Q.-M.; Zheng, J.-J.; Xu, L.; Holmes, E. C.; Zhang, Y.-Z. Nature 2020." />
<meta name="citation_reference" content="Gimenez, F.; Ramos-Hernan, R.; Benavente, E.; Barbosa, J.; Sanz-Nebot, V. Analytica chimica acta 2012, 709, 81â90." />
<meta name="citation_reference" content="Suttapitugsakul, S.; Sun, E.; Wu, R. Analytical Chemistry 2020, 92, 267â291." />
<meta name="citation_reference" content="Pohlentz, G.; Marx, K.; Mormann, M. Methods in molecular biology 2016, 1394, 163â179." />
<meta name="citation_reference" content="Zhang, Y.; Xie, X.; Zhao, X.; Tian, F.; Lv, J.; Ying, W.; Qian, X. J Proteomics 2018, 170, 14â27." />
<meta name="citation_reference" content="Sun, N.; Wu, H.; Chen, H.; Shen, X.; Deng, C. Chem Commun (Camb) 2019, 55, 10359â10375." />
<meta name="citation_reference" content="Ruhaak, L. R.; Xu, G.; Li, Q.; Goonatilleke, E.; Lebrilla, C. B. Chem Rev 2018, 118, 7886â7930." />
<meta name="citation_reference" content="Zhang, Y.; Mao, Y.; Zhao, W.; Su, T.; Zhong, Y.; Fu, L.; Zhu, J.; Cheng, J.; Yang, H. Journal of Proteome Research 2020, 19, 655â666." />
<meta name="citation_reference" content="Zhang, Y.; Zhao, W.; Zhao, Y.; Mao, Y.; Su, T.; Zhong, Y.; Wang, S.; Zhai, R.; Cheng, J.; Fang, X.; Zhu, J.; Yang, H. J Proteome Res 2019." />
<meta name="citation_reference" content="Yuan, Y.; Cao, D.; Zhang, Y.; Ma, J.; Qi, J.; Wang, Q.; Lu, G.; Wu, Y.; Yan, J.; Shi, Y.; Zhang, X.; Gao, G. F. Nat Commun 2017, 8, 15092." />
<meta name="citation_reference" content="Zhang, S.; Cao, X.; Liu, C.; Li, W.; Zeng, W.; Li, B.; Chi, H.; Liu, M.; Qin, X.; Tang, L.; Yan, G.; Ge, Z.; Liu, Y.; Gao, Q.; Lu, H. Molecular &amp; cellular proteomics: MCP 2019, 18, 2262â2272." />
<meta name="citation_reference" content="Wang, Y.; Xu, F.; Xiao, K.; Chen, Y.; Tian, Z. Chem Commun (Camb) 2019, 55, 7934â7937." />
<meta name="citation_reference" content="Riley, N. M.; Hebert, A. S.; Westphall, M. S.; Coon, J. J. Nat Commun 2019, 10, 1311." />
<meta name="citation_reference" content="Sun, S.; Hu, Y.; Jia, L.; Eshghi, S. T.; Liu, Y.; Shah, P.; Zhang, H. Anal Chem 2018, 90, 6292â6299." />
<meta name="citation_reference" content="Li, D.; von Schaewen, M.; Wang, X.; Tao, W.; Zhang, Y.; Li, L.; Heller, B.; Hrebikova, G.; Deng, Q.; Ploss, A.; Zhong, J.; Huang, Z. J Virol 2016, 90, 10486â10498." />
<meta name="citation_reference" content="Urbanowicz, R. A.; Wang, R.; Schiel, J. E.; Keck, Z. Y.; Kerzic, M. C.; Lau, P.; Rangarajan, S.; Garagusi, K. J.; Tan, L.; Guest, J. D.; Ball, J. K.; Pierce, B. G.; Mariuzza, R. A.; Foung, S. K. H.; Fuerst, T. R. J Virol 2019, 93." />
<meta name="citation_reference" content="Ozdilek, A.; Paschall, A. V.; Dookwah, M.; Tiemeyer, M.; Avci, F. Y. Proc Natl Acad Sci U S A 2020, 117, 1280â1282." />
<meta name="citation_reference" content="Patnaik, S. K.; Potvin, B.; Carlsson, S.; Sturm, D.; Leffler, H.; Stanley, P. Glycobiology 2006, 16, 305â317." />
<meta name="citation_reference" content="Robinson, B. S.; Arthur, C. M.; Evavold, B.; Roback, E.; Kamili, N. A.; Stowell, C. S.; Vallecillo-Zuniga, M. L.; Van Ry, P. M.; Dias-Baruffi, M.; Cummings, R. D.; Stowell, S. R. Front Immunol 2019, 10, 1762." />
<meta name="citation_reference" content="Wang, W. H.; Lin, C. Y.; Chang, M. R.; Urbina, A. N.; Assavalapsakul, W.; Thitithanyanont, A.; Chen, Y. H.; Liu, F.T.; Wang, S. F. J Microbiol Immunol Infect 2019." />
<meta name="twitter:title" content="Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The global pandemic of severe acute pneumonia syndrome (COVID-19) caused by SARS-CoV-2 urgently calls for prevention and intervention strategies. The densely glycosylated spike (S) protein highly exposed on the viral surface is a determinant for virus binding and invasion into host cells as well as elicitation of a protective host immune response. Herein, we characterized the site-specific N-glycosylation of SARS-CoV-2 S protein using stepped collision energy (SCE) mass spectrometry (MS). Following digestion with two complementary proteases to cover all potential N-glycosylation sequons and integrated N-glycoproteomics analysis, we revealed the N-glycosylation profile of SARS-CoV-2 S proteins at the levels of intact N-glycopeptides and glycosites, along with the glycan composition and site-specific number of glycans. All 22 potential canonical N-glycosites were identified in S protein protomer. Of those, 18 N-glycosites were conserved between SARS-CoV and SARS-CoV-2 S proteins. Nearly all glycosites were preserved among the 753 SARS-CoV-2 genome sequences available in the public influenza database Global Initiative on Sharing All Influenza Data. By comparison, insect cell-expressed SARS-CoV-2 S protein contained 38 N-glycans, which were primarily assigned to the high-mannose type N-glycans, whereas the human cell-produced protein possessed up to 140 N-glycans largely belonging to the complex type. In particular, two N-glycosites located in the structurally exposed receptor-binding domain of S protein exhibited a relatively conserved N-glycan composition in human cells. This N-glycosylation profiling and determination of differences between distinct expression systems could shed light on the infection mechanism and promote development of vaccines and targeted drugs." />
<meta name="og-title" property="og:title" content="Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The global pandemic of severe acute pneumonia syndrome (COVID-19) caused by SARS-CoV-2 urgently calls for prevention and intervention strategies. The densely glycosylated spike (S) protein highly exposed on the viral surface is a determinant for virus binding and invasion into host cells as well as elicitation of a protective host immune response. Herein, we characterized the site-specific N-glycosylation of SARS-CoV-2 S protein using stepped collision energy (SCE) mass spectrometry (MS). Following digestion with two complementary proteases to cover all potential N-glycosylation sequons and integrated N-glycoproteomics analysis, we revealed the N-glycosylation profile of SARS-CoV-2 S proteins at the levels of intact N-glycopeptides and glycosites, along with the glycan composition and site-specific number of glycans. All 22 potential canonical N-glycosites were identified in S protein protomer. Of those, 18 N-glycosites were conserved between SARS-CoV and SARS-CoV-2 S proteins. Nearly all glycosites were preserved among the 753 SARS-CoV-2 genome sequences available in the public influenza database Global Initiative on Sharing All Influenza Data. By comparison, insect cell-expressed SARS-CoV-2 S protein contained 38 N-glycans, which were primarily assigned to the high-mannose type N-glycans, whereas the human cell-produced protein possessed up to 140 N-glycans largely belonging to the complex type. In particular, two N-glycosites located in the structurally exposed receptor-binding domain of S protein exhibited a relatively conserved N-glycan composition in human cells. This N-glycosylation profiling and determination of differences between distinct expression systems could shed light on the infection mechanism and promote development of vaccines and targeted drugs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-29" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.28.013276v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.28.013276v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.28.013276v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1216242" />
    <title>Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ryHP0S8JEORlaw_q3CJS1CHc4Kt47ZJD7Mjo95fafWc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1216242","apath":"\/biorxiv\/early\/2020\/03\/29\/2020.03.28.013276.atom","pisa":"biorxiv;2020.03.28.013276v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.28.013276v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.28.013276v1","title":"Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry","identifier":"node\/1216242"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UJQk9eaIAWP0FrtFw1L9HtZ\\\\\\\/Wl0USP3YXElBxyLYYpPm8Pn8j40yblpe2096u33OLV\\\\\\\/YKwXVCI\\\\\\\/zJoSO7Jd8FPZBHEFwAZFg3d\\\\\\\/oldO\\\\\\\/vpDoRo4dyQq0XaG2i3uMRjYG6WeueIfX8RZ4eWmBSWClLS2FRHFSIPacbKfRJOia+TDOZHjnZk+3JHgYJr0xLvMoG058V8NRK4dgzdp119TLrpLQsnv0SYRf0PosMZ3VzD8RB3Z0Acz2tAVCBRY6JZy5p5jQxt6aopIysMxKSE22EKqutsG6SG4tPc40KrSJvUsC\\\\\\\/m8SsaoXNJwRin9\\\\\\\/pe+y7C5DCgAR7TeCFKGcbK+ZpCgcwQcO0AAPRU83h8rQRanRRAXdZcx2DYRju\\\\\\\/U35uIWwdfG8knuhTgvRjldX9Za4ge4PuIOf1u4qoXUODzIG\\\\\\\/uVogVWuYliZqgMJJkkxZwZz66lmjRvWgZbBqwGdZL56LLhabpI9clbg5dac0YsMP0z8EqS\\\\\\\/yA4ywhB+RHBLM978HIPpScSR8FXAf6n7OKzij6mBYSRhK7\\\\\\\/Jm1zBThxoRkgwbd\\\\\\\/3hZxRDaXFDDJIuv+Y\\\\\\\/4LndRkK+JPcwyOemtO6SlVdCwxLaXD+\\\\\\\/xEiLKDiT+8qRRkI7dx7aJA6IoGBlto0OhpdQhrxdmXd54atCHV1+spbhZTQ8dEC74mcbSSMuwrbbgBjrKrrymYiCiuhFiZjC98FmI6BEOz4hfIqWthZxdSMp9CuzQxCmA0f7d2Ylilk70eOmiT0JUXH+s9B0XJIPRT1w2DlnqNsFbAk7GSFtoW+bV62AWePik+YsSu1BpDfb4AGfr0PH\\\\\\\/g0ot7vBjLXc9N5oLAqrVmQv2aUdVOox4A+DS7Otz+1nVkSF6VaZKNrlFXSzfeE9\\\\\\\/V4TBG6B2s7WFEfWbM90XhfPylZwuBVI1VWLt3XZ5+rt3OKoRWRmxLfAOZ1th8+aaQH5Er9kfA16pYFNnLaFTMZv9BPMQSCTtgHRBtQ18ceQR8fqQhvKExPsYXUcSACwZHkkupAJS2WJRFeJac6MqkYCMISHqxKPWY3Ji1U0\\\\\\\/iIF2V0RrUDQhp6Dj\\\\\\\/BeBqnlKH4P4XsPR+TrV0Jb4ySwuAiRZFhXSx+qKeuvo6JZZUoRTxH6G4yuu32F5wuNo8ISJ\\\\\\\/Ld9qgX+KYtZ4xrQL6otmU9iTeCFVrTnDbUJjLGQlvyTSrUKRqGrTKj5dO1RsAi2bmSZQZ8d0lcmUsEGCqBitIZx4JS6xkVh2a6r8rjQuSfoVVWevPVU0sPHvKM0FX3ZwvJAcqCWau4JxVju9VOlbLlhAAC6FTfG4cw2MJQBM\\\\\\\/LyETF1me\\\\\\\/\\\\\\\/UYTJLX1ay0mPXYwnS84RO\\\u0022,\\\u0022iv\\\u0022:\\\u0022AeJQE0W19jK47XYXJ3gnJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022332cc2b76870aa7863d7e7e89d824122182dae05991a3e974dfbe2cc411aa418\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.28.013276v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1216242 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.28.013276v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CISVieJ0G_X6bNtSbnbMcdyNjUzBSCnq46F47uzqpHY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1216242" id="node1216242" data-pisa="biorxiv;2020.03.28.013276v1" data-pisa-master="biorxiv;2020.03.28.013276" data-apath="/biorxiv/early/2020/03/29/2020.03.28.013276.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0513-8941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Yong</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Wanjun</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9952-399X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Yonghong</span> <span class="nlm-surname">Mao</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Shisheng</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Yi</span> <span class="nlm-surname">Zhong</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Su</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Meng</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Xiaofeng</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jingqiu</span> <span class="nlm-surname">Cheng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Yang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.28.013276 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1216242" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0513-8941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong Zhang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wanjun Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Thyroid Surgery, West China Hospital, Sichuan University</span>, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wanjun%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWanjun%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yonghong Mao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Thoracic Surgery Research Laboratory, West China Hospital, Sichuan University</span>, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonghong%2BMao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYonghong%2BMao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9952-399X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yonghong Mao</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shisheng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shisheng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShisheng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yi Zhong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi%2BZhong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhong%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi%2BZhong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tao Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tao%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATao%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Meng Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meng%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMeng%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xiaofeng Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaofeng%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaofeng%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jingqiu Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingqiu%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingqiu%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Hao Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Key Laboratory of Transplant Engineering and Immunology, MOH, West China-Washington Mitochondria and Metabolism Research Center, West China Hospital</span>, Sichuan University, Chengdu 610041, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHao%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yanghao@scu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">yanghao@scu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.28.013276v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1216242" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1216242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013276v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1216242" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1216242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013276v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1216242" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1216242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013276v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1216242" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1216242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013276v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1216242" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1216242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.28.013276v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1216242" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1216242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-3">The global pandemic of severe acute pneumonia syndrome (COVID-19) caused by SARS-CoV-2 urgently calls for prevention and intervention strategies. The densely glycosylated spike (S) protein highly exposed on the viral surface is a determinant for virus binding and invasion into host cells as well as elicitation of a protective host immune response. Herein, we characterized the site-specific N-glycosylation of SARS-CoV-2 S protein using stepped collision energy (SCE) mass spectrometry (MS). Following digestion with two complementary proteases to cover all potential N-glycosylation sequons and integrated N-glycoproteomics analysis, we revealed the N-glycosylation profile of SARS-CoV-2 S proteins at the levels of intact N-glycopeptides and glycosites, along with the glycan composition and site-specific number of glycans. All 22 potential canonical N-glycosites were identified in S protein protomer. Of those, 18 N-glycosites were conserved between SARS-CoV and SARS-CoV-2 S proteins. Nearly all glycosites were preserved among the 753 SARS-CoV-2 genome sequences available in the public influenza database Global Initiative on Sharing All Influenza Data. By comparison, insect cell-expressed SARS-CoV-2 S protein contained 38 N-glycans, which were primarily assigned to the high-mannose type N-glycans, whereas the human cell-produced protein possessed up to 140 N-glycans largely belonging to the complex type. In particular, two N-glycosites located in the structurally exposed receptor-binding domain of S protein exhibited a relatively conserved N-glycan composition in human cells. This N-glycosylation profiling and determination of differences between distinct expression systems could shed light on the infection mechanism and promote development of vaccines and targeted drugs.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">In the last month of 2019, a novel coronavirus (SARS-CoV-2) emerged and rapidly spread, developing into an epidemic of severe acute pneumonia syndrome (COVID-19), which engulfed the city of Wuhan and Hubei Province of China. The virus quickly swept through the entirety of China and subsequently the outbreak came out in Asia and the entire world within a couple of months. Similar to SARS-CoV and MERS-CoV that emerged in 2002 and 2013, respectively, SARS-CoV-2 is highly transmissible from infected individuals, even without symptoms, to healthy humans and can cause lethal respiratory symptoms<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>â<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. The World Health Organization has declared the spread of SARS-CoV-2 a Public Health Emergency of International Concern as over 160 countries have reported confirmed cases. From SARS-CoV to SARS-CoV-2, the periodic outbreak of coronavirus infection in humans urgently calls for prevention and intervention measures. However, there are no approved vaccines or effective antiviral drugs for either SARS-CoV or SARS-CoV-2.</p><p id="p-5">Decoding the critical component and molecular characteristics of the virus is the key to developing a cure strategy. SARS-CoV-2 is a single-stranded RNA virus. RNA sequencing revealed that SARS-CoV-2 belongs to the beta-coronavirus genus and is most closely related to SARS-CoV, with a genome size of approximately 30 kb encoding 15 non-structural proteins, 4 structural proteins, and 8 auxiliary proteins<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. The structural proteins of mature SARS-CoV-2 include spike (S) protein, envelope (E) protein, membrane (M) protein, and nucleocapsid (N) protein<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. Theoretically, all structural proteins can serve as antigens for vaccine development or targets for anti-viral treatment. Of these proteins, the transmembrane S protein protruding from the virus surface is highly exposed and responsible for invasion into host cells, which has attracted the special attention of researchers. S protein is homotrimeric and is highly glycosylated on the virus surface, allowing for binding to the angiotensin converting enzyme II (ACE2) receptor on host cells to promote the fusion of viral and host cellular membranes<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Given its indispensable role in virus entry and infectivity, S protein is a promising target for vaccine design and drug discovery to block the interaction between the virus and host cells. S protein has been revealed as a crucial antigen for raising neutralizing antibodies and eliciting protective humoral as well as cellular immunity upon infection or vaccination with S protein-based vaccines<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>â<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>.</p><p id="p-6">Typically, S protein has an ectodomain linked to a single-pass transmembrane anchor and a short C-terminal intracellular tail<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. The ectodomain comprises a receptor-binding S1 subunit and a membrane-fusion S2 subunit. Following attachment to the host cell surface via S1, SARS-CoV-2 S protein is cleaved at the S1/S2 boundary sites together with the junction site and the S2â site by host cellular proteases for S protein priming, consequently mediating membrane fusion driven by S2 and making way for the viral genetic materials to enter the host cell<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. Based on cryoelectron microscopy (Cryo-EM) observations, recognition of S protein to the ACE2 receptor primarily involves extensive polar residue interactions between the SARS-CoV-2 receptor binding domain (RBD) and the peptidase domain of ACE2<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. The S protein RBD is located in the S1 subunit and undergoes a hinge-like dynamic movement to capture the receptor through the interaction of three group residue clusters between the RBD and ACE2. Compared to S protein of SARS-CoV, that of SARS-CoV-2 displays up to 10â20-fold higher affinity for the human ACE2 receptor, which partially explains the higher transmissibility of this new virus<sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></sup>.</p><p id="p-7">In addition to the structural information and core amino acid residues for receptor binding, SARS-CoV-2 S protein possesses 22 potential N-linked glycosylation motifs (N-X-S/T, Xâ P) in each monomer. The N-glycans on S protein play a pivotal role in proper protein folding and protein priming by host proteases. Importantly, glycosylation is an underlying mechanism for coronavirus to evade both the innate and adaptive immune responses, as the glycans might shield the amino acid residues from cell and antibody recognition<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. Cryo-EM observations provided evidence of the existence of glycans on 14â16 of 22 potential sites in SARS-CoV-2 S protein<sup><a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a></sup>. However, these glycosites and glycans need to be experimentally identified in detail. Glycosylation analysis via glycopeptides can provide insight into the N-glycan microheterogeneity of a specific site, as variation in site-specific glycosylation levels can be greater than that at the protein level<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Therefore, further identification of site-specific N-glycosylation information of SARS-CoV-2 S protein, including intact N-glycopeptides, glycosites, glycan composition, and the site-specific number of glycans, could be meaningful to obtain a deeper understanding of the mechanism of virus invasion, providing guidance for vaccine design and antiviral therapeutics development<sup><a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup></p><p id="p-8">Herein, we characterized the site-specific N-glycosylation of recombinant SARS-CoV-2 S proteins by combined analysis of intact and deglycosylated N-glycopeptides using tandem mass spectrometry (MS/MS). Based on this integrated method, we identified 22 potential canonical N-glycosites and their corresponding N-glycans from the recombinant ectodomain (residues 16â1213) expressed in insect cells. For comparison, glycosylation of the recombinant S1 subunit (residues 16â685) expressed in human cells was resolved in parallel. All of these glycosites were found to be highly conserved among 753 SARS-CoV-2 genome sequences from the Global Initiative on Sharing All Influenza Data (GISAID) database. These detailed glycosylation profiles decoded from MS/MS analysis are complementary to those observed from Cryo-EM and might help in the development of vaccines and therapeutic drugs. The raw MS data are publicly accessible at ProteomeXchange (ProteomeXchange.com) under accession number PXD018068.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Materials and chemicals</h3><p id="p-9">Dithiothreitol (DTT), iodoacetamide (IAA), formic acid (FA), trifluoroacetic acid (TFA), Tris base, and urea were purchased from Sigma (St. Louis, MO, USA). Acetonitrile (ACN) was purchased from Merck (Darmstadt, Germany). The zwitterionic hydrophilic interaction liquid chromatography (Zic-HILIC) materials were purchased from Fresh Bioscience (Shanghai, China). Commercially available recombinant SARS-CoV-2 S protein (S1+S2 ECD, His tag) expressed in insect cells via baculovirus and S protein (S1, His tag) expressed in human embryonic kidney (HEK293) cells were purchased from Sino Biological (Beijing, China). Sequencing-grade trypsin and Glu-C were obtained from Enzyme &amp; Spectrum (Beijing, China). The quantitative colorimetric peptide assay kit was purchased from Thermo Fisher Scientific (Waltham, MA, USA). Deionized water was prepared using a Milli-Q system (Millipore, Bedford, MA, USA). All other chemicals and reagents of the best available grade were purchased from Sigma-Aldrich or Thermo Fisher Scientific.</p></div><div id="sec-4" class="subsection"><h3>Protein digestion</h3><p id="p-10">The recombinant S proteins were proteolyzed using an in-solution protease digestion protocol. In brief, 50 Î¼g of protein in a tube was denatured for 10 min at 95 Â°C. After reduction by DTT (20 mM) for 45 min at 56 Â°C and alkylating with IAA (50 mM) for 1 h at 25 Â°C in the dark, 2 Î¼g of protease (trypsin or/and Glu-C) was added to the tube and incubated for 16 h at 37 Â°C. After desalting using a pipette tip packed with a C18 membrane, the peptide concentration was determined using a peptide assay kit based on the absorbance measured at 480 nm. The peptide mixtures (intact N-glycopeptides before enrichment) were freeze-dried for further analysis.</p></div><div id="sec-5" class="subsection"><h3>Selective enrichment of intact N-glycopeptides</h3><p id="p-11">Intact N-glycopeptides were enriched with Zic-HILIC materials (Fresh Bioscience, Shanghai, China). Specifically, 20 Î¼g of peptides was resuspended in 100 Î¼L of 80% ACN/0.2% TFA solution, and 2 mg of processed Zic-HILIC was added to the peptide solution and rotated for 2 h at 37 Â°C. Finally, the mixture was transferred to a 200-Î¼L pipette tip packed with the C8 membrane and washed twice with 80% ACN/0.2% TFA. After enrichment, intact N-glycopeptides were eluted three times with 70 Î¼L of 0.1% TFA and dried using SpeedVac for further analysis.</p></div><div id="sec-6" class="subsection"><h3>Deglycosylation</h3><p id="p-12">Enriched intact N-glycopeptides were digested using 1 U PNGase F dissolved in 50 Î¼L of 50 mM NH4HCO3 for 2 h at 37 Â°C. The reaction was terminated by the addition of 0.1% FA. The deglycosylated peptides were dried using SpeedVac for further analysis.</p></div><div id="sec-7" class="subsection"><h3>Liquid chromatography-MS/MS analysis</h3><p id="p-13">All samples were analyzed by SCE-higher-energy collisional dissociation (HCD)-MS/MS using an Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific). In brief, intact N-glycopeptides before or after enrichment and deglycosylated peptides were dissolved in 0.1% FA and separated on a column (ReproSil-Pur C18-AQ, 1.9 Î¼m, 75 Î¼m inner diameter, length 20 cm; Dr Maisch) over a 78-min gradient (buffer A, 0.1% FA in water; buffer B, 0.1% FA in 80% ACN) at a flow rate of 300 nL/min. MS1 was analyzed with a scan range (m/z) of 800â2000 (intact N-glycopeptides before or after enrichment) or 350â1550 (deglycosylated peptides) at an Orbitrap resolution of 120,000. The RF lens, AGC target, maximum injection time, and exclusion duration were 30%, 2.0 e<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup>, 100 ms, and 15 s, respectively. MS2 was analyzed with an isolation window (m/z) of 2 at an Orbitrap resolution of 15,000. The AGC target, maximum injection time, and HCD type were standard, 250 ms, and 30%, respectively. The stepped collision mode was turned on with an energy difference of Â±10%.</p></div><div id="sec-8" class="subsection"><h3>Data analysis</h3><p id="p-14">The raw data files were searched against the SARS-CoV-2 S protein sequence using Byonic software (version 3.6.0, Protein Metrics, Inc.)<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup> with the mass tolerance for precursors and fragment ions set at Â±10 ppm and Â±20 ppm, respectively. Two missed cleavage sites were allowed for trypsin or/and Glu-C digestion. The fixed modification was carbamidomethyl (C), and variable modifications included oxidation (M), acetyl (protein N-term), and deamidation (N). In addition, 38 insect N-glycans or 182 human N-glycans were specified as N-glycan modifications for intact N-glycopeptides before or after enrichment. We then checked the protein database options, including the decoy database. All other parameters were set at the default values, and protein groups were filtered to a 1% false discovery rate based on the number of hits obtained for searches against these databases. Stricter quality control methods for intact N-glycopeptides and peptide identification were implemented, requiring a score of no less than 200 and at least 7 amino acids to be identified. Furthermore, all of these peptide spectrum matches (PSMs) and glycopeptide-spectrum matches (GPSMs) were examined manually and filtered using the following standard criteria: PSMs were accepted if there were at least 3 b/y ions in the peptide backbone, and GPSMs were accepted if there were at least two glycan oxonium ions and at least 3 b/y ions in the peptide backbone. N-glycosite conservation analysis was performed using R software packages. Model building based on the Cryo-EM structure (PDB: 6VSB) of SARS-CoV-2 S protein was performed using PyMOL.</p></div></div><div class="section" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Strategy for site-specific N-glycosylation characterization</h3><p id="p-15">Previous studies have revealed that glycosylated coronavirus S protein plays a critical role in the induction of neutralizing antibodies and protective immunity. However, the glycans decorated on S protein might also shield the protein surface and lead to virus evasion from the immune system<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a></sup>. Herein, we aimed to decode the detailed site-specific N-glycosylation profile of SARS-CoV-2 S protein. A commercial S protein ectodomain expressed by the baculovirus expression vector in insect cells was first used to analyze the glycosylation patterns, since the baculovirus vector can express a large protein without resulting in splicing of the native S protein via host proteases<sup><a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. The recombinant SARS-CoV-2 S ectodomain contains 1209 amino acids (residues 16â1213) that were translated from the complete genome (GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947.3&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom">MN908947.3</a>)<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup> and 22 putative N-glycosites (motif N-X-S/T, Xâ P). Analysis of the theoretical enzymatic peptides showed that trypsin (hydrolyzing proteins at K and R) alone did not produce a sufficient amount of appropriate peptides to cover all potential N-glycosites. The missing potential N-glycosites could be found by introducing the endoproteinase Glu-C (hydrolyzing proteins at D in ammonium bicarbonate solution)<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>. Hence, we took advantage of this complementary trypsin and Glu-C digestion approach (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Since the N-glycan compositions of S protein expressed in insect cells would be different from those of the native S protein expressed in human host cells, despite insect cells mimicking the process of mammalian cell glycosylation<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup>, the recombinant SARS-CoV-2 S protein S1 subunit expressed in human cells was obtained for comparison. S2 subunit N-glycosylation sequons are conserved among SARS-CoV-2 and SARS-CoV and have been confirmed in previous studies<sup><a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a></sup>. The S1 subunit contains 681 amino acids (residues 16â685) and 13 potential N-glycosites. Analysis of the theoretical enzymatic peptides showed that trypsin digestion alone would produce a sufficient amount of appropriate peptides to cover all potential N-glycosites on the S1 subunit.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow for site-specific N-glycosylation characterization of recombinant SARS-CoV-2 S proteins using two complementary proteases for digestion and simultaneous N-glycoproteomics analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675390831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Workflow for site-specific N-glycosylation characterization of recombinant SARS-CoV-2 S proteins using two complementary proteases for digestion and simultaneous N-glycoproteomics analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F1.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F1.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-16" class="first-child">Workflow for site-specific N-glycosylation characterization of recombinant SARS-CoV-2 S proteins using two complementary proteases for digestion and simultaneous N-glycoproteomics analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">In general, the relative content of N-glycosylated peptides in a glycoprotein is low; hence, enrichment of intact N-glycopeptides is necessary<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. For this purpose, we used Zic-HILIC materials to enrich intact glycopeptides through hydrophilic interactions owing to their high selectivity and reproducibility<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. However, due to the microheterogeneity (different glycans attached to the same glycosite) and macroheterogeneity (glycosite occupancy) of glycosylation<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>, there are no materials available that can capture all glycopeptides without preference<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. For these reasons, site-specific glycosylation was determined based on a combined analysis of the intact N-glycopeptides before and after enrichment and the deglycosylated peptides following enrichment using SCE-HCD-MS/MS<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Analysis of intact N-glycopeptides before enrichment can retrieve the missing intact N-glycopeptides from Zic-HILIC materials, while detection of deglycosylated peptides can simultaneously confirm the N-glycosites. Therefore, integration of complementary digestion and N-glycoproteomics analysis from three levels is a promising approach to comprehensively and confidently profile the site-specific N-glycosylation of recombinant SARS-CoV-2 S proteins.</p></div><div id="sec-11" class="subsection"><h3>N-glycosite profiling of recombinant SARS-CoV-2 S proteins</h3><p id="p-18">S protein produced by the baculovirus insect cell expression system contains 22 potential N-glycosites. Using our integrated analysis method described above, 20 N-glycosites were assigned unambiguously with high-quality (score â¥ 200) spectral evidence (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, Table S1 and Table S2). Two N-glycosites (N17 and N1134) were ambiguously assigned with relatively lower spectral scores (score &lt; 200) (Figure S1). However, the N-glycosite N1134 has been reported in the Cryo-EM structure of SARS-CoV-2 S protein<sup><a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">10</a></sup>. In addition, three non-canonical motifs of N-glycosites (N164, N334, and N536) involving N-X-C sequons were not N-glycosylated. Before enrichment, 11 N-glycosites from trypsin-digested peptides and 9 N-glycosites from Glu-C-digested peptides were assigned unambiguously, whereas hydrophilic enrichment resulted in an increase of these glycosites to 14 and 11, respectively (Table S1).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="N-glycosites characterization of SARS-CoV-2 S proteins (A and B) N-glycosites of recombinant SARS-CoV-2 S proteins expressed in insect cells (A) and human cells (B). PSS: putative signal sequence; RBD: receptor-binding domain; S1/S2: S1/S2 protease cleavage site; Oval: potential N-glycosite; Yellow oval: ambiguously assigned N-glycosite; Red oval: unambiguously assigned N-glycosite; Blue arrow: unambiguously assigned N-glycosite using trypsin digestion; Green arrow: unambiguously assigned N-glycosite using Glu-C digestion; Black arrow: retrieved N-glycosite using the combination of trypsin and Glu-C digestion. (C) Surface demonstration of the SARS-CoV-2 S protein (PDB code: 6VSB) ectodomain trimers with RBDs (yellow) and N-glycosites (blue)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675390831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;N-glycosites characterization of SARS-CoV-2 S proteins (A and B) N-glycosites of recombinant SARS-CoV-2 S proteins expressed in insect cells (A) and human cells (B). PSS: putative signal sequence; RBD: receptor-binding domain; S1/S2: S1/S2 protease cleavage site; Oval: potential N-glycosite; Yellow oval: ambiguously assigned N-glycosite; Red oval: unambiguously assigned N-glycosite; Blue arrow: unambiguously assigned N-glycosite using trypsin digestion; Green arrow: unambiguously assigned N-glycosite using Glu-C digestion; Black arrow: retrieved N-glycosite using the combination of trypsin and Glu-C digestion. (C) Surface demonstration of the SARS-CoV-2 S protein (PDB code: 6VSB) ectodomain trimers with RBDs (yellow) and N-glycosites (blue)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F2.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F2.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-19" class="first-child">N-glycosites characterization of SARS-CoV-2 S proteins (A and B) N-glycosites of recombinant SARS-CoV-2 S proteins expressed in insect cells (A) and human cells (B). PSS: putative signal sequence; RBD: receptor-binding domain; S1/S2: S1/S2 protease cleavage site; Oval: potential N-glycosite; Yellow oval: ambiguously assigned N-glycosite; Red oval: unambiguously assigned N-glycosite; Blue arrow: unambiguously assigned N-glycosite using trypsin digestion; Green arrow: unambiguously assigned N-glycosite using Glu-C digestion; Black arrow: retrieved N-glycosite using the combination of trypsin and Glu-C digestion. (C) Surface demonstration of the SARS-CoV-2 S protein (PDB code: 6VSB) ectodomain trimers with RBDs (yellow) and N-glycosites (blue)</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">To further assess the necessity for enrichment, we compared the spectra of two intact N-glycopeptides (N61 and N74) before and after enrichment. Without interference from the non-glycosylated peptides, the intact N-glycopeptide had more fragmented ions assigned to N-glycosites after enrichment (Figure S2). Exceptionally, the intact N-glycopeptide containing an N-glycosite (N17) was missed after enrichment, presumably because of the selectivity of Zic-HILIC (Table S1).</p><p id="p-21">Complementary digestion with trypsin and Glu-C promoted the confident identification of two N-glycosites (N709 and N717) on an intact N-glycopeptide (Figure S3). The introduction of Glu-C digestion resulted in the production of a short intact N-glycopeptide containing 23 amino acids, which is more suitable for achieving good fragmentation than the long peptide of 48 amino acids obtained from trypsin digestion (Figure S3). Similarly, the N-glycosylation analysis strategy combining intact N-glycopeptides with deglycosylated peptides improved the identification of N-glycosite N234, which was ambiguously assigned in the spectrum of the intact N-glycopeptides alone (Figure S4).</p><p id="p-22">For the recombinant protein S1 subunit expressed in human cells, 12 out of 13 N-glycosites were assigned unambiguously with high-quality spectral evidence. One N-glycosite (N17) was assigned ambiguously with a relatively low spectral score (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>, Table S2 and Table S3). The relatively low spectral evidence of two N-glycosites (N17 and N1134) indicate the existence of low-frequency glycosylation on the these ambiguous glycosites since deglycosylation failed to improve the identification of all the two sites. Finally, using this strategy, we profiled all 22 potential N-glycosites of S protein. These sites were preferentially distributed in the S1 subunit of the N-terminus and in the S2 subunit of the C-terminus, including two sites in the RBD (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). To visualize N-glycosylation on the protein structure, all of the experimentally determined N-glycosites were hand-marked on the surface of trimeric S protein following refinement of the recently reported SARS-CoV-2 S protein Cryo-EM structure (PDB: 6VSB) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>)<sup><a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a></sup>.</p><p id="p-23">Based on these findings, we further analyzed the conservation of the glycosites among 753 SARS-CoV-2 genome sequences from the GISAID database. After removal of redundant sequences of S protein at the amino acid residue level, we found a very low frequency of alterations in 38 residue sites uniformly spanning over the full length of S protein among 145 protein variants, with the exception of the substitution G614D, which was found at relatively high frequency in 47 variants (Table S4). However, nearly all of the 22 N-glycosylated sequons were conserved in S protein, except for loss of the N717 glycosite due to the T719A substitution in only one S protein variant. Compared to SARS-CoV S protein, 18 of the 22 N-glycosites were found to be conserved in SARS-CoV-2 S protein, indicating the importance of glycosylation for the virus. Four newly arised N-glycosites (N17, N74, N149, and N657) are located in the SARS-CoV-2 S protein S1 subunit away from the RBD. Moreover, four confirmed N-glycosites (N29, N73, N109, and N357) in SARS-CoV S protein were missing in SARS-CoV-2 S, one of which (N357) lies in the RBD<sup><a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>.</p></div><div id="sec-12" class="subsection"><h3>Intact N-glycopeptides of recombinant SARS-CoV-2 S proteins</h3><p id="p-24">Precise characterization of intact N-glycopeptides is critical for understanding biological functions<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. Although intact N-gly copeptide analysis is more challenging than analysis of separate N-glycosites or N-glycans, it can provide more comprehensive information, including N-glycosites, N-glycan compositions, and the number of N-glycans<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>â<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. The potential N-glycopeptides in the S protein sequence are shown in Figure S5. Comparison of the intact N-glycopeptides spectra to the total spectra showed that the average enrichment efficiency of the Zic-HILIC materials reached up to 97%. Ultimately, 646 non-redundant intact N-glycopeptides were identified from SARS-CoV-2 S proteins (Table S1), and 410 non-redundant intact N-glycopeptides were identified from the recombinant S1 subunit (Table S3). Representative and high-quality spectra of intact N-glycopeptides are shown in Figure S6. The number of intact N-glycopeptides and N-glycans significantly increased after glycopeptide enrichment (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Site-specific N-glycosylation of recombinant SARS-CoV-2 S proteins. The number of intact N-glycopeptides and N-glycans in recombinant SARS-CoV-2 S proteins expressed in insect cells (A) or human cells (B). Top 10 N-glycans and their putative structures on the intact N-glycopeptides from recombinant SARS-CoV-2 S proteins expressed in insect cells (C) and human cells (D). Different types and numbers of N-glycans on each N-glycosite of recombinant SARS-CoV-2 S protein or subunit expressed in insect cells (E) or human cells (F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675390831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Site-specific N-glycosylation of recombinant SARS-CoV-2 S proteins. The number of intact N-glycopeptides and N-glycans in recombinant SARS-CoV-2 S proteins expressed in insect cells (A) or human cells (B). Top 10 N-glycans and their putative structures on the intact N-glycopeptides from recombinant SARS-CoV-2 S proteins expressed in insect cells (C) and human cells (D). Different types and numbers of N-glycans on each N-glycosite of recombinant SARS-CoV-2 S protein or subunit expressed in insect cells (E) or human cells (F).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F3.medium.gif" width="323" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F3.medium.gif" width="323" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-25" class="first-child">Site-specific N-glycosylation of recombinant SARS-CoV-2 S proteins. The number of intact N-glycopeptides and N-glycans in recombinant SARS-CoV-2 S proteins expressed in insect cells (A) or human cells (B). Top 10 N-glycans and their putative structures on the intact N-glycopeptides from recombinant SARS-CoV-2 S proteins expressed in insect cells (C) and human cells (D). Different types and numbers of N-glycans on each N-glycosite of recombinant SARS-CoV-2 S protein or subunit expressed in insect cells (E) or human cells (F).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Regarding the N-glycan composition, N-glycopeptides of S protein expressed in insect cells had smaller and fewer complex N-glycans compared with those of the S1 subunit produced in human cells. Both recombinant products contained the common N-acetylglucosamine as a canonical N-glycan characteristic (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">3D</a>). N-glycopeptides of S protein expressed in insect cells were decorated with 38 N-glycans, with the majority preferentially comprising paucimannose- and fucose-type oligosaccharides (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a> and Table S1). By contrast, N-glycopeptides of the S1 subunit expressed in human cells were attached with up to 140 N-glycans, mainly containing fucose-type and unique sialic acid-type oligosaccharides (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a> and Table S3). Returning to the glycosite level, most of the N-glycosites in S protein were modified with 17â35 types of N-glycans, with a high proportion of high-mannose N-glycans and a lower proportion of hybrid N-glycans (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3E</a>). For the S1 subunit, three N-glycosites (N122, N282, and N657) were surprisingly decorated with markedly heterogeneous N glycans of up to 113 types, including a high proportion of complex N-glycans and a small proportion of hybrid or high-mannose N-glycans (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3F</a>). These results showed that the two S proteins expressed in different cells displayed different N-glycosylation patterns with a distinctive N-glycan composition (microheterogeneity) and different numbers of N-glycans at the same site, along with distinct site occupancies in intact glycopeptides (macroheterogeneity) (Figure S7).</p><p id="p-27">The glycosylation patterns of proteins expressed in insect cells have been found to be more immunogenic than those produced in human cells<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>, and antigen production from mammalian cells does not always induce a strong humoral immune response<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. The complex and highly heterogeneous N-glycans modified on SARS-CoV-2 S protein expressed in human cells may be related to the differential immune response, which could possibly be caused by epitope masking by the glycan shield, although this hypothesis requires further testing and clarification. Therefore, the less complex N-glycans covering S protein expressed in insect cells might favor the development of vaccines to elicit neutralizing antibodies against SARS-CoV-2 virus.</p><p id="p-28">Two potential N-glycosites (N331 and N343) in RBD were confirmed in the N-glycopeptides from both insect cell-expressed S protein and human cell-produced S1 subunit (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3E</a> and <a id="xref-fig-3-10" class="xref-fig" href="#F3">3F</a>). Both sites closely located in the same glycopeptide (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Intriguingly, the composition of the N-glycans in human cells exhibited relatively high conservation compared to most of other N-glycosites (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3F</a>). However, the N-glycan composition were more variable in RBD expressed in insect cells (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3E</a>). These results imply that N-glycosylation modification might be associated with receptor binding since the recognition of RBD to ACE2 mainly depends on polar residue interaction<sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a></sup>. In addition, complex N-glycans are also ligands for galectins, which can engage different glycoproteins and regulate immune cell infiltration and activation upon virus infection<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>â<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. The RBDs of SARS-CoV-2 decorated with distinct N-glycans in different expression systems could be candidates for SARS-CoV-2 vaccine design as an alternative to full-length S protein which can lead to undesired immunopotentiation with respect to increased infectivity and eosinophilic infiltration<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. Based on these results, a large-scale intact N-glycopeptides database of recombinant SARS-CoV-2 S proteins was developed. Nevertheless, the implication of S protein site-specific N-glycosylation (including N-glycosites and N-glycans) on receptor binding, viral infectivity, and immunogenicity should be further investigated.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Site-specific N-glycosylation profile of RBD. (A) The putative N-glycans on N331 and N343 of RBD. (B and C) Representative spectra of the intact N-glycopeptides from recombinant SARS-CoV-2 S protein expressed in human cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675390831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Site-specific N-glycosylation profile of RBD. (A) The putative N-glycans on N331 and N343 of RBD. (B and C) Representative spectra of the intact N-glycopeptides from recombinant SARS-CoV-2 S protein expressed in human cells.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F4.medium.gif" width="320" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F4.medium.gif" width="320" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.28.013276/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-29" class="first-child">Site-specific N-glycosylation profile of RBD. (A) The putative N-glycans on N331 and N343 of RBD. (B and C) Representative spectra of the intact N-glycopeptides from recombinant SARS-CoV-2 S protein expressed in human cells.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-13"><h2 class="">Conclusions</h2><p id="p-30">A comprehensive analysis of site-specific N-glycosylation of SARS-CoV-2 S protein was performed at the levels of intact N-glycopeptides, glycosites, glycan composition, and site-specific numbers of N-glycans. By taking advantage of two complementary protease digestion systems and N-gly coproteomics analysis through enrichment and deglycosylation, we provided a global and site-specific profile of N-glycosylation on SARS-CoV-2 S proteins, revealing extensive heterogeneity in N-glycan composition and site occupancy. Almost all of these glycosites were conserved among the 753 published SARS-CoV-2 genome sequences. In particular, two N-glycosites in the S protein RBD produced in human cells showed relative conservation compared to the N-glycan composition of insect cells, suggesting the potential impact of N-glycosylation on receptor binding. Overall, our data indicate that N-glycosylation profiling and identifying differences among distinct expression systems might help to elucidate the infection mechanism toward development of an effective vaccine and targeted drugs.</p></div><div class="section" id="sec-15"><h2 class="">ASSOCIATED CONTENT</h2><div class="section" id="sec-16"><h3>Supporting Information</h3><p id="p-31">Figure S1, Spectra of intact N-glycopeptides with ambiguously assigned N-glycosites; Figure S2, Comparison of the spectra of intact N-glycopeptides before (A) and after (B) enrichment; Figure S3, Comparison of the spectra of intact N-glycopeptides from trypsin digestion (A) and Glu-C digestion (B); Figure S4, Comparison of the spectra of intact N-glycopeptides (A) and deglycopeptides (B); Figure S5, Amino acid sequence alignment of recombinant SARS-CoV-2 S proteins expressed in insect cells (A) and human cells (B). Yellow background: putative sequence containing a signal sequence; Red: potential N-glycosites; Red bold: identified N-glycosites; Green: theoretical cleavage sites of trypsin; Blue: theoretical cleavage sites of Glu-C; Figure S6, Representative and high-quality spectra of intact N-glycopeptides and deglycosylated peptides. Figure S7, Microheterogeneity and macroheterogeneity of the N-linked glycopeptides of S protein; Table S1, Site-specific N-glycosylation characterization of recombinant SARS-CoV-2 S protein expressed in insect cells; Table S2, Glycoproteomic identification results of recombinant SARS-CoV-2 spike protein using the combination of trypsin and Glu-C digestion; Table S3, Site-specific N-glycosylation characterization of recombinant SARS-CoV-2 S protein expressed in human cells; Table S4, Mutation frequency of SARS-CoV-2 spike protein.</p></div></div><div class="section" id="sec-17"><h2 class="">AUTHOR INFORMATION</h2><div class="section" id="sec-18"><h3>Author Contributions</h3><p id="p-32">#Y. Zhang and W. Zhao contributed equally to this work.</p></div><div class="section" id="sec-19"><h3>Notes</h3><p id="p-33">The authors declare no competing financial interests.</p></div></div><div class="section" id="sec-20"><h2 class="">Supplementary Figures</h2><p id="p-35">Supplementary Figure S1. Spectra of intact N-glycopeptides with ambiguously assigned N-glycosites</p><p id="p-36">Supplementary Figure S2. Comparison of the spectra of intact N-glycopeptides before (A) and after (B) enrichment</p><p id="p-37">Supplementary Figure S3. Comparison of the spectra of intact N-glycopeptides from trypsin digestion (A) and Glu-C digestion (B)</p><p id="p-38">Supplementary Figure S4. Comparison of the spectra of intact N-glycopeptides (A) and deglycopeptides (B)</p><p id="p-39">Supplementary Figure S5. Amino acid sequence alignment of recombinant SARS-CoV-2 spike proteins expressed in insect cells (A) and human cells (B). Yellow background: putative sequence containing signal sequence; Red: potential N-glycosites; Red bold: identified N-glycosites; Green: theoretical cleavage sites of trypsin; Blue: theoretical cleavage sites of Glu-C</p><p id="p-40">Supplementary Figure S6. Representative and high-quality spectra of intact N-glycopeptides and deglycosylated peptides</p><p id="p-41">Supplementary Figure S7. Microheterogeneity and macroheterogeneity of the N-linked glycopeptides of S protein</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENT</h2><p id="p-34">This work was funded by grants from the National Natural Science Foundation of China (grant number 31901038), the 1.3.5 Project for Disciplines of Excellence, West China Hospital, Sichuan University (ZYGD18014, CJQ), and the Chengdu Science and Technology Department Foundation (grant number 2020-YF05-00240-SN).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">(1).</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference (1) in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.1" data-doi="10.1016/S0140-6736(20)302518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bi</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhan</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">395</span>, <span class="cit-fpage">565</span>â<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%2529302518%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)302518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(2).</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference (2) in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.2" data-doi="10.1038/s41586-41020-42012-41587"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.-G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Si</span>,  <span class="cit-name-given-names">H.-R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.-D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">R.-D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.-Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.-R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">X.-S.</span></span>, <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2020</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41586-41020-42012-41587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-41020-42012-41587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-41020-42012-41587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(3).</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference (3) in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.3" data-doi="10.1056/NEJMoa030747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">GÃ¼nther</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Preiser</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">van der Werf</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brodt</span>,  <span class="cit-name-given-names">H.-R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rabenau</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Panning</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kolesnikova</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R. A. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">BurguiÃ¨re</span>,  <span class="cit-name-given-names">A.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cinatl</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eickmann</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Escriou</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grywna</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kramme</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Manuguerra</span>,  <span class="cit-name-given-names">J.-C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">MÃ¼ller</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rickerts</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">348</span>, <span class="cit-fpage">1967</span>â<span class="cit-lpage">1976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030747%26rft_id%253Dinfo%253Apmid%252F12690091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(4).</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference (4) in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z. G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">Z. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">J. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">Y. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">M. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">F. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">J. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y. Z.</span></span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">579</span>, <span class="cit-fpage">265</span>â<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D265%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(5).</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference (5) in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.5" data-doi="S1931-3128(1920)30072-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ding</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Meng</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sheng</span>,  <span class="cit-name-given-names">J.</span></span>; Quan, L; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">T.</span></span> <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-pub-date">2020</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>S1931-3128(1920)30072-X</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft_id%253Dinfo%253Adoi%252FS1931-3128%25281920%252930072-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=S1931-3128(1920)30072-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(6).</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference (6) in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.6" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">M. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vasilieva</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sui</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Berne</span>,  <span class="cit-name-given-names">M. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Somasundaran</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>,  <span class="cit-name-given-names">J. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Luzuriaga</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Greenough</span>,  <span class="cit-name-given-names">T. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Choe</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>â<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(7).</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference (7) in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.7" data-doi="10.1126/science.abb2507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">K. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>,  <span class="cit-name-given-names">J. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>,  <span class="cit-name-given-names">C. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Abiona</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">B. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J. S.</span></span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">367</span>, <span class="cit-fpage">1260</span>â<span class="cit-lpage">1263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjkvMjAyMC4wMy4yOC4wMTMyNzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(8).</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference (8) in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.8" data-doi="10.1038/nrmicro2090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">B. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">S.</span></span> <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">7</span>, <span class="cit-fpage">226</span>â<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDu%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D226%26rft.epage%253D236%26rft.atitle%253DThe%2Bspike%2Bprotein%2Bof%2BSARS-CoV--a%2Btarget%2Bfor%2Bvaccine%2Band%2Btherapeutic%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2090%26rft_id%253Dinfo%253Apmid%252F19198616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19198616&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(9).</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference (9) in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">W.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Strych</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hotez</span>,  <span class="cit-name-given-names">P. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bottazzi</span>,  <span class="cit-name-given-names">M. E.</span></span> <abbr class="cit-jnl-abbrev">Current Tropical Medicine Reports</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(10).</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference (10) in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wall</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McGuire</span>,  <span class="cit-name-given-names">A. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Veesler</span>,  <span class="cit-name-given-names">D.</span></span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(11).</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference (11) in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.11" data-doi="10.1016/j.cell.2018.12.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xiong</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Quispe</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cameroni</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gopal</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zambon</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rey</span>,  <span class="cit-name-given-names">F. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Corti</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Veesler</span>,  <span class="cit-name-given-names">D.</span></span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">176</span>, <span class="cit-fpage">1026</span>â<span class="cit-lpage">1039</span> <span class="cit-page-range">e1015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft_id%253Dinfo%253Apmid%252F30712865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30712865&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(12).</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference (12) in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> <abbr class="cit-jnl-abbrev">Annual Review of Virology</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">3</span>, <span class="cit-fpage">237</span>â<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BVirology%26rft.volume%253D3%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(13).</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference (13) in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Q.</span></span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(14).</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference (14) in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.14" data-doi="10.1038/nsmb.3293"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Frenz</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rey</span>,  <span class="cit-name-given-names">F. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">DiMaio</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bosch</span>,  <span class="cit-name-given-names">B. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Veesler</span>,  <span class="cit-name-given-names">D.</span></span> <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">23</span>, <span class="cit-fpage">899</span>â<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BStruct%2BMol%2BBiol%26rft.volume%253D23%26rft.spage%253D899%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3293%26rft_id%253Dinfo%253Apmid%252F27617430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27617430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(15).</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference (15) in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.15" data-doi="10.1007/s10719-015-9626-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clerc</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Reiding</span>,  <span class="cit-name-given-names">K. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">B. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kammeijer</span>,  <span class="cit-name-given-names">G. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bondt</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wuhrer</span>,  <span class="cit-name-given-names">M.</span></span> <abbr class="cit-jnl-abbrev">Glycoconj J</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">33</span>, <span class="cit-fpage">309</span>â<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycoconj%2BJ%26rft.volume%253D33%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10719-015-9626-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10719-015-9626-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(16).</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference (16) in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.16" data-doi="10.1074/mcp.R119.001491"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zaia</span>,  <span class="cit-name-given-names">J.</span></span> <abbr class="cit-jnl-abbrev">Mol Cell Proteomics</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">18</span>, <span class="cit-fpage">2348</span>â<span class="cit-lpage">2358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.R119.001491%26rft_id%253Dinfo%253Apmid%252F31604803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8yMzQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjkvMjAyMC4wMy4yOC4wMTMyNzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(17).</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference (17) in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Medzihradszky</span>,  <span class="cit-name-given-names">K. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Maynard</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kaasik</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bern</span>,  <span class="cit-name-given-names">M.</span></span> <abbr class="cit-jnl-abbrev">Molecular &amp; Cellular Proteomics</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">13</span>, <span class="cit-fpage">S36</span>â<span class="cit-lpage">S36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2BCellular%2BProteomics%26rft.volume%253D13%26rft.spage%253D36S%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(18).</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference (18) in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.18" data-doi="10.1038/nbt1095"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kost</span>,  <span class="cit-name-given-names">T. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Condreay</span>,  <span class="cit-name-given-names">J. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>,  <span class="cit-name-given-names">D. L.</span></span> <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">23</span>, <span class="cit-fpage">567</span>â<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKost%26rft.auinit1%253DT.%2BA.%26rft.volume%253D23%26rft.issue%253D5%26rft.spage%253D567%26rft.epage%253D575%26rft.atitle%253DBaculovirus%2Bas%2Bversatile%2Bvectors%2Bfor%2Bprotein%2Bexpression%2Bin%2Binsect%2Band%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1095%26rft_id%253Dinfo%253Apmid%252F15877075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15877075&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228943500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(19).</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference (19) in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z.-G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">Z.-W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">J.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">Y.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">M.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">F.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">J.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-Z.</span></span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(20).</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference (20) in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.20" data-doi="10.1016/j.aca.2011.10.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gimenez</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ramos-Hernan</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Benavente</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Barbosa</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sanz-Nebot</span>,  <span class="cit-name-given-names">V.</span></span> <abbr class="cit-jnl-abbrev">Analytica chimica acta</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">709</span>, <span class="cit-fpage">81</span>â<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytica%2BChimica%2BActa%26rft.stitle%253DAnalytica%2BChimica%2BActa%26rft.aulast%253DGimenez%26rft.auinit1%253DE.%26rft.volume%253D709%26rft.spage%253D81%26rft.epage%253D90%26rft.atitle%253DAnalysis%2Bof%2Brecombinant%2Bhuman%2Berythropoietin%2Bglycopeptides%2Bby%2Bcapillary%2Belectrophoresis%2Belectrospray-time%2Bof%2Bflight-mass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aca.2011.10.028%26rft_id%253Dinfo%253Apmid%252F22122935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aca.2011.10.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22122935&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(21).</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference (21) in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suttapitugsakul</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">R.</span></span> <abbr class="cit-jnl-abbrev">Analytical Chemistry</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">92</span>, <span class="cit-fpage">267</span>â<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%26rft.volume%253D92%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(22).</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference (22) in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pohlentz</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Marx</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mormann</span>,  <span class="cit-name-given-names">M.</span></span> <abbr class="cit-jnl-abbrev">Methods in molecular biology</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">1394</span>, <span class="cit-fpage">163</span>â<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%26rft.volume%253D1394%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(23).</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference (23) in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.23" data-doi="10.1016/j.jprot.2017.09.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lv</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ying</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qian</span>,  <span class="cit-name-given-names">X.</span></span> <abbr class="cit-jnl-abbrev">J Proteomics</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">170</span>, <span class="cit-fpage">14</span>â<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BProteomics%26rft.volume%253D170%26rft.spage%253D14%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jprot.2017.09.014%26rft_id%253Dinfo%253Apmid%252F28970103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jprot.2017.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28970103&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(24).</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference (24) in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">C.</span></span> <abbr class="cit-jnl-abbrev">Chem Commun (Camb)</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">55</span>, <span class="cit-fpage">10359</span>â<span class="cit-lpage">10375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem%2BCommun%2B%2528Camb%2529%26rft.volume%253D55%26rft.spage%253D10359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(25).</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference (25) in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruhaak</span>,  <span class="cit-name-given-names">L. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Goonatilleke</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lebrilla</span>,  <span class="cit-name-given-names">C. B.</span></span> <abbr class="cit-jnl-abbrev">Chem Rev</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">118</span>, <span class="cit-fpage">7886</span>â<span class="cit-lpage">7930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem%2BRev%26rft.volume%253D118%26rft.spage%253D7886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(26).</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference (26) in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">H.</span></span> <abbr class="cit-jnl-abbrev">Journal of Proteome Research</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">19</span>, <span class="cit-fpage">655</span>â<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BProteome%2BResearch%26rft.volume%253D19%26rft.spage%253D655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(27).</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference (27) in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhai</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">H.</span></span> <abbr class="cit-jnl-abbrev">J Proteome Res</abbr> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(28).</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference (28) in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.28" data-doi="10.1038/ncomms15092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G. F.</span></span> <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">8</span>, <span class="cit-fpage">15092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D15092%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15092%26rft_id%253Dinfo%253Apmid%252F28393837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28393837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(29).</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference (29) in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chi</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">H.</span></span> <abbr class="cit-jnl-abbrev">Molecular &amp; cellular proteomics: MCP</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">18</span>, <span class="cit-fpage">2262</span>â<span class="cit-lpage">2272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bcellular%2Bproteomics%253A%2BMCP%26rft.volume%253D18%26rft.spage%253D2262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(30).</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference (30) in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">Z.</span></span> <abbr class="cit-jnl-abbrev">Chem Commun (Camb)</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">55</span>, <span class="cit-fpage">7934</span>â<span class="cit-lpage">7937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem%2BCommun%2B%2528Camb%2529%26rft.volume%253D55%26rft.spage%253D7934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(31).</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013276v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Riley</span>,  <span class="cit-name-given-names">N. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hebert</span>,  <span class="cit-name-given-names">A. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Westphall</span>,  <span class="cit-name-given-names">M. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Coon</span>,  <span class="cit-name-given-names">J. J.</span></span> <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">10</span>, <span class="cit-fpage">1311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D1311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(32).</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference (32) in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eshghi</span>,  <span class="cit-name-given-names">S. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shah</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span> <abbr class="cit-jnl-abbrev">Anal Chem</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">90</span>, <span class="cit-fpage">6292</span>â<span class="cit-lpage">6299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal%2BChem%26rft.volume%253D90%26rft.spage%253D6292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(33).</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference (33) in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.33" data-doi="10.1128/JVI.01462-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">von Schaewen</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Heller</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hrebikova</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ploss</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Z.</span></span> <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">90</span>, <span class="cit-fpage">10486</span>â<span class="cit-lpage">10498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01462-16%26rft_id%253Dinfo%253Apmid%252F27630242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI5MC8yMy8xMDQ4NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDMuMjguMDEzMjc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(34).</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference (34) in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Urbanowicz</span>,  <span class="cit-name-given-names">R. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schiel</span>,  <span class="cit-name-given-names">J. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Keck</span>,  <span class="cit-name-given-names">Z. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kerzic</span>,  <span class="cit-name-given-names">M. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rangarajan</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Garagusi</span>,  <span class="cit-name-given-names">K. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guest</span>,  <span class="cit-name-given-names">J. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ball</span>,  <span class="cit-name-given-names">J. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pierce</span>,  <span class="cit-name-given-names">B. G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mariuzza</span>,  <span class="cit-name-given-names">R. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Foung</span>,  <span class="cit-name-given-names">S. K. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fuerst</span>,  <span class="cit-name-given-names">T. R.</span></span> <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">93</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(35).</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference (35) in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.35" data-doi="10.1073/pnas.1916131117"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ozdilek</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Paschall</span>,  <span class="cit-name-given-names">A. V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dookwah</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tiemeyer</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Avci</span>,  <span class="cit-name-given-names">F. Y.</span></span> <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">117</span>, <span class="cit-fpage">1280</span>â<span class="cit-lpage">1282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1916131117%26rft_id%253Dinfo%253Apmid%252F31907319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE3LzMvMTI4MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDMuMjguMDEzMjc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(36).</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference (36) in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.36" data-doi="10.1093/glycob/cwj063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patnaik</span>,  <span class="cit-name-given-names">S. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Potvin</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Carlsson</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sturm</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leffler</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stanley</span>,  <span class="cit-name-given-names">P.</span></span> <abbr class="cit-jnl-abbrev">Glycobiology</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">16</span>, <span class="cit-fpage">305</span>â<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycobiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fglycob%252Fcwj063%26rft_id%253Dinfo%253Apmid%252F16319083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/glycob/cwj063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319083&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.28.013276.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236106200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(37).</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013276v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">B. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Arthur</span>,  <span class="cit-name-given-names">C. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Evavold</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Roback</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kamili</span>,  <span class="cit-name-given-names">N. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stowell</span>,  <span class="cit-name-given-names">C. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vallecillo-Zuniga</span>,  <span class="cit-name-given-names">M. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Van Ry</span>,  <span class="cit-name-given-names">P. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dias-Baruffi</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cummings</span>,  <span class="cit-name-given-names">R. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stowell</span>,  <span class="cit-name-given-names">S. R.</span></span> <abbr class="cit-jnl-abbrev">Front Immunol</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">10</span>, <span class="cit-fpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BImmunol%26rft.volume%253D10%26rft.spage%253D1762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(38).</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference (38) in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013276v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">C. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">M. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Urbina</span>,  <span class="cit-name-given-names">A. N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Assavalapsakul</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thitithanyanont</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">F.T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S. F.</span></span> <abbr class="cit-jnl-abbrev">J Microbiol Immunol Infect</abbr> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.28.013276v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.27.011692v1" title="Bifidobacterium breve UCC2003 induces a distinct global transcriptomic programme in neonatal murine intestinal epithelial cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.27.012955v1" title="Simulating the Influence of Conjugative Plasmids Kinetic Values on the Multilevel Dynamics of Antimicrobial Resistance in a Membrane Computing Model" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 29, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.28.013276v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.28.013276v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.28.013276v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.28.013276v1" class="active" data-icon-position="" data-hide-link-title="0">Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1216242" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-A7bMzKEG27EfTME9z0AYeMufaVVYtOJ0enM88kxul9I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1216242" id="node1216242" data-pisa="biorxiv;2020.03.28.013276v1" data-pisa-master="biorxiv;2020.03.28.013276" data-seqnum="1216242" data-apath="/biorxiv/early/2020/03/29/2020.03.28.013276.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yong</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Wanjun</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yonghong</span> <span class="nlm-surname">Mao</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Shisheng</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Yi</span> <span class="nlm-surname">Zhong</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Su</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Meng</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Xiaofeng</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jingqiu</span> <span class="nlm-surname">Cheng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Yang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.28.013276; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.28.013276 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;title=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;title=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;text=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;title=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;t=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;title=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;title=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1216242" id="node1216242--2" data-pisa="biorxiv;2020.03.28.013276v1" data-pisa-master="biorxiv;2020.03.28.013276" data-seqnum="1216242" data-apath="/biorxiv/early/2020/03/29/2020.03.28.013276.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yong</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Wanjun</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yonghong</span> <span class="nlm-surname">Mao</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Shisheng</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Yi</span> <span class="nlm-surname">Zhong</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Su</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Meng</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Xiaofeng</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jingqiu</span> <span class="nlm-surname">Cheng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Yang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.28.013276; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.28.013276 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Zhang%2C+Yong&amp;rft.au=Zhao%2C+Wanjun&amp;rft.au=Mao%2C+Yonghong&amp;rft.au=Wang%2C+Shisheng&amp;rft.au=Zhong%2C+Yi&amp;rft.au=Su%2C+Tao&amp;rft.au=Gong%2C+Meng&amp;rft.au=Lu%2C+Xiaofeng&amp;rft.au=Cheng%2C+Jingqiu&amp;rft.au=Yang%2C+Hao"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1216242/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1216242/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1216242/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1216242/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1216242/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1216242/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1216242/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1216242/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1216242/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1216242/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1216242/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1216242/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;count=horizontal&amp;via=&amp;text=Site-specific%20N-glycosylation%20Characterization%20of%20Recombinant%20SARS-CoV-2%20Spike%20Proteins%20using%20High-Resolution%20Mass%20Spectrometry&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013276v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.28.013276v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biochemistry" class="highlight" data-icon-position="" data-hide-link-title="0">Biochemistry<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
