<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/08/06/385005.atom" />
<meta name="HW.pisa" content="biorxiv;385005v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis" />
<meta name="DC.Identifier" content="10.1101/385005" />
<meta name="DC.Date" content="2018-08-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Background Febrile illness is a major burden in African children, and non-malarial causes of fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics platform and service to identify microbes from metagenomic next-generation sequencing of tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, Uganda.

Results The most common pathogens identified were Plasmodium falciparum (51.1% of samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx of a child with rash and sepsis. We also identified two novel human rhinovirus C species.

Conclusions This exploratory pilot study demonstrates the utility of mNGS and the IDseq platform for defining the molecular landscape of febrile infectious diseases in resource limited areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases.

*   ### List of abbreviations
    
    AWS
    :   Amazon Web Services
    ASG
    :   Auto Scaling Groups
    BLAST
    :   Basic Local Alignment Search Tool
    EC2
    :   Elastic Compute Cloud
    ESBL
    :   Extended spectrum β-lactamase
    HHV
    :   human herpesvirus
    HIV-1
    :   human immunodeficiency 1 virus
    HRV
    :   Human Rhinovirus
    IQR
    :   Inter-quartile range
    L coding region
    :   Large segment coding region (RNA-dependent RNA polymerase)
    LZW
    :   Lempel-Ziv-Welch
    M coding region
    :   Medium segment coding region (glycoprotein)
    MEGA
    :   Molecular Evolutionary Genetics Analysis
    mNGS
    :   metagenomic next-generation sequencing
    NCBI
    :   National Center for Biotechnology Information
    NP swab
    :   Nasopharyngeal swab
    nr database
    :   non-redundant database
    nt database
    :   nucleotide database
    PRICE
    :   Paired-Read Iterative Contig Extension
    rpM
    :   reads per million
    RSV
    :   respiratory syncytial virus
    S coding region
    :   Small segment coding region (nucleocapsid)
    SDI
    :   Simpsons diversity Index
    SPAdes
    :   St Petersburg genome assembler
    SRST2
    :   Short Read Sequencing Typing
    STAR
    :   Spliced Transcripts Alignment to a Reference
    TTV
    :   torque teno virus
    VP1
    :   Capsid protein VP1
    WHO
    :   World Health Organization" />
<meta name="DC.Contributor" content="Akshaya Ramesh" />
<meta name="DC.Contributor" content="Sara Nakielny" />
<meta name="DC.Contributor" content="Jennifer Hsu" />
<meta name="DC.Contributor" content="Mary Kyohere" />
<meta name="DC.Contributor" content="Oswald Byaruhanga" />
<meta name="DC.Contributor" content="Charles de Bourcy" />
<meta name="DC.Contributor" content="Rebecca Egger" />
<meta name="DC.Contributor" content="Boris Dimitrov" />
<meta name="DC.Contributor" content="Yun-Fang Juan" />
<meta name="DC.Contributor" content="Jonathan Sheu" />
<meta name="DC.Contributor" content="James Wang" />
<meta name="DC.Contributor" content="Katrina Kalantar" />
<meta name="DC.Contributor" content="Charles Langelier" />
<meta name="DC.Contributor" content="Theodore Ruel" />
<meta name="DC.Contributor" content="Arthur Mpimbaza" />
<meta name="DC.Contributor" content="Michael R. Wilson" />
<meta name="DC.Contributor" content="Philip J. Rosenthal" />
<meta name="DC.Contributor" content="Joseph L. DeRisi" />
<meta name="article:published_time" content="2018-08-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;h3&gt;Background&lt;/h3&gt;
&lt;p&gt;Febrile illness is a major burden in African children, and non-malarial causes of fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics platform and service to identify microbes from metagenomic next-generation sequencing of tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, Uganda.&lt;/p&gt;&lt;h3&gt;Results&lt;/h3&gt;
&lt;p&gt;The most common pathogens identified were &lt;i&gt;Plasmodium falciparum&lt;/i&gt; (51.1% of samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx of a child with rash and sepsis. We also identified two novel human rhinovirus C species.&lt;/p&gt;&lt;h3&gt;Conclusions&lt;/h3&gt;
&lt;p&gt;This exploratory pilot study demonstrates the utility of mNGS and the IDseq platform for defining the molecular landscape of febrile infectious diseases in resource limited areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;385005v2" />
<meta name="citation_id" content="385005v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/385005v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/385005v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/385005v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005.full.pdf" />
<meta name="citation_doi" content="10.1101/385005" />
<meta name="citation_num_pages" content="44" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="385005" />
<meta name="citation_author" content="Akshaya Ramesh" />
<meta name="citation_author_institution" content="Weill Institute for Neurosciences, University of California" />
<meta name="citation_author_institution" content="Department of Neurology, University of California" />
<meta name="citation_author" content="Sara Nakielny" />
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of California" />
<meta name="citation_author" content="Jennifer Hsu" />
<meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, University of California" />
<meta name="citation_author" content="Mary Kyohere" />
<meta name="citation_author_institution" content="Infectious Diseases Research Collaboration" />
<meta name="citation_author" content="Oswald Byaruhanga" />
<meta name="citation_author_institution" content="Infectious Diseases Research Collaboration" />
<meta name="citation_author" content="Charles de Bourcy" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="Rebecca Egger" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="Boris Dimitrov" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="Yun-Fang Juan" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="Jonathan Sheu" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="James Wang" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="Katrina Kalantar" />
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of California" />
<meta name="citation_author" content="Charles Langelier" />
<meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, University of California" />
<meta name="citation_author" content="Theodore Ruel" />
<meta name="citation_author_institution" content="Division of Pediatric Infectious Diseases and Global Health, Department of Pediatrics, University of California" />
<meta name="citation_author" content="Arthur Mpimbaza" />
<meta name="citation_author_institution" content="Child Health and Development Centre, Makerere University" />
<meta name="citation_author" content="Michael R. Wilson" />
<meta name="citation_author_institution" content="Weill Institute for Neurosciences, University of California" />
<meta name="citation_author_institution" content="Department of Neurology, University of California" />
<meta name="citation_author" content="Philip J. Rosenthal" />
<meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, University of California" />
<meta name="citation_author" content="Joseph L. DeRisi" />
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of California" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_reference" content="Maze MJ, Bassat Q, Feasey NA, Mandomando I, Musicha P, Crump JA. The epidemiology of febrile illness in sub-Saharan Africa: implications for diagnosis and management. Clin Microbiol Infect. 2018; 24(8):808-814. doi:10.1016/j.cmi.2018.02.011." />
<meta name="citation_reference" content="Prasad N, Sharples KJ, Murdoch DR, Crump JA. Community prevalence of fever and relationship with malaria among infants and children in low-resource areas. Am J Trop Med Hyg. 2015;93(1):178-180. doi:10.4269/ajtmh.14-0646." />
<meta name="citation_reference" content="Guidelines for the treatment of malaria, 2nd edition. World Health Organization. 2010." />
<meta name="citation_reference" content="Bibby K. Metagenomic identification of viral pathogens. Trends Biotechnol. 2013;31(5):275-279. doi:10.1016/j.tibtech.2013.01.016." />
<meta name="citation_reference" content="Yozwiak NL, Skewes-Cox P, Stenglein MD, Balmaseda A, Harris E, DeRisi JL. Virus identification in unknown tropical febrile illness cases using deep sequencing. PLoS Negl Trop Dis. 2012;6(2):e1485. doi:10.1371/journal.pntd.0001485." />
<meta name="citation_reference" content="Wilson MR, Naccache SN, Samayoa E, et al. Actionable Diagnosis of Neuroleptospirosis by Next-Generation Sequencing. N Engl J Med. 2014;370(25):2408-2417. doi:10.1056/NEJMoa1401268." />
<meta name="citation_reference" content="Wilson MR, Shanbhag NM, Reid MJ, et al. Diagnosing Balamuthia mandrillaris Encephalitis with Metagenomic Deep Sequencing. Ann Neurol. 2015;78(5):722-730. doi:10.1002/ana.24499." />
<meta name="citation_reference" content="Doan T, Wilson MR, Crawford ED, et al. Illuminating uveitis: metagenomic deep sequencing identifies common and rare pathogens. Genome Med. 2016;8:1-2. doi:10.1186/s13073-016-0344-6." />
<meta name="citation_reference" content="Quan PL, Wagner TA, Briese T, et al. Astrovirus encephalitis in boy with X-linked agammaglobulinemia. Emerg Infect Dis. 2010;16(6):918-925. doi:10.3201/eid1606.091536." />
<meta name="citation_reference" content="Palacios G, Druce J, Du L, et al. A New Arenavirus in a Cluster of Fatal Transplant-Associated Diseases. N Engl J Med. 2008;358:991-998. doi: 10.1056/NEJMoa073785." />
<meta name="citation_reference" content="Muir P, Li S, Lou S, et al. The real cost of sequencing: Scaling computation to keep pace with data generation. Genome Biol. 2016;17(1):1-9. doi:10.1186/s13059-016-0917-0." />
<meta name="citation_reference" content="Naoumov N V. TT virus - highly prevalent, but still in search of a disease. J Hepatol. 2000;33(13):157-159. doi:10.1016/S0168-8278(00)80173-7." />
<meta name="citation_reference" content="Hafez MM, Shaarawy SM, Hassan AA, Salim RF, Abd El Salam FM, Ali AE. Prevalence of transfusion transmitted virus (TTV) genotypes among HCC patients in Qaluobia governorate. Virol J. 2007;4:1-6. doi:10.1186/1743-422X-4-135." />
<meta name="citation_reference" content="Abreu NA, Nagalingam NA, Song Y, et al. Sinus Microbiome Diversity Depletion and Corynebacterium tuberculostearicum Enrichment Mediates Rhinosinusitis. Sci Transl Med. 2012;4(151):151ra124-151ra124. doi:10.1126/scitranslmed.3003783." />
<meta name="citation_reference" content="Langelier C, Zinter M, Kalantar K, et al. Metagenomic Next-Generation Sequencing Detects Pulmonary Pathogens in Hematopoietic Cellular Transplant Patients with Acute Respiratory Illnesses. Am J Respir Crit Care Med. 2018 Feb 15;197(4):524-528. doi: 10.1164/rccm.201706-1097LE." />
<meta name="citation_reference" content="Park DE, Baggett HC, Howie SRC, et al. Colonization density of the upper respiratory tract as a predictor of pneumonia - Haemophilus influenzae, Moraxella catarrhalis, Staphylococcus aureus, and Pneumocystis jirovecii. Clin Infect Dis. 2017;64:S328-S336. doi:10.1093/cid/cix104." />
<meta name="citation_reference" content="Loens K, Van Heirstraeten L, Malhotra-Kumar S, Goossens H, Ieven M. Optimal sampling sites and methods for detection of pathogen possibly causing community-acquired lower respiratory tract infections. J Clin Microbiol. 2009;47(1):21-31. doi:10.1128/JCM.02037-08." />
<meta name="citation_reference" content="Zar HJ, Barnett W, Stadler A, Gardner-Lubbe S, Myer L, Nicol MP. Aetiology of childhood pneumonia in a well vaccinated South African birth cohort: A nested case-control study of the Drakenstein Child Health Study. Lancet Respir Med. 2016;4(6):463-472. doi:10.1016/S2213-2600(16)00096-5." />
<meta name="citation_reference" content="Lukashev AN, Vakulenko YA. Molecular evolution of types in non-polio enteroviruses. J Gen Virol. 2017;98(12):2968-2981. doi:10.1099/jgv.0.000966." />
<meta name="citation_reference" content="McIntyre CL, Knowles NJ, Simmonds P. Proposals for the classification of human rhinovirus species A, B and C into genotypically assigned types. J Gen Virol. 2013;94(PART8):1791-1806. doi:10.1099/vir.0.053686-0." />
<meta name="citation_reference" content="Inouye M, Dashnow H, Raven LA, et al. SRST2: Rapid genomic surveillance for public health and hospital microbiology labs. Genome Med. 2014;6(11):1-16. doi:10.1186/s13073-014-0090-6." />
<meta name="citation_reference" content="Chipwaza B, Mugasa JP, Selemani M, et al. Dengue and Chikungunya Fever among Viral Diseases in Outpatient Febrile Children in Kilosa District Hospital, Tanzania. PLoS Negl Trop Dis. 2014;8(11). doi:10.1371/journal.pntd.0003335." />
<meta name="citation_reference" content="Jacob ST, Pavlinac PB, Nakiyingi L, et al. Mycobacterium tuberculosis Bacteremia in a Cohort of HIV-Infected Patients Hospitalized with Severe Sepsis in Uganda-High Frequency, Low Clinical Sand Derivation of a Clinical Prediction Score. PLoS One. 2013;8(8). doi:10.1371/journal.pone.0070305." />
<meta name="citation_reference" content="Crump JA, Morrissey AB, Nicholson WL, et al. Etiology of Severe Non-malaria Febrile Illness in Northern Tanzania: A Prospective Cohort Study. PLoS Negl Trop Dis. 2013;7(7). doi:10.1371/journal.pntd.0002324." />
<meta name="citation_reference" content="Chipwaza B, Mhamphi GG, Ngatunga SD, et al. Prevalence of Bacterial Febrile Illnesses in Children in Kilosa District, Tanzania. PLoS Negl Trop Dis. 2015;9(5). doi:10.1371/journal.pntd.0003750." />
<meta name="citation_reference" content="D’Acremont V, Kilowoko M, Kyungu E, et al. Beyond Malaria — Causes of Fever in Outpatient Tanzanian Children. N Engl J Med. 2014;370(9):809-817. doi:10.1056/NEJMoa1214482." />
<meta name="citation_reference" content="O’Meara WP, Mott JA, Laktabai J, et al. Etiology of pediatric fever in Western Kenya: A case-control study of falciparum Malaria, Respiratory Viruses, and Streptococcal Pharyngitis. Am J Trop Med Hyg. 2015;92(5):1030-1037. doi:10.4269/ajtmh.14-0560." />
<meta name="citation_reference" content="Maina AN, Farris CM, Odhiambo A, et al. Q fever, scrub typhus, and rickettsial diseases in children, Kenya, 2011–2012. Emerg Infect Dis. 2016;22(5):883-886. doi:10.3201/eid2205.150953." />
<meta name="citation_reference" content="Kibuuka A, Byakika-Kibwika P, Achan J, et al. Bacteremia among febrile ugandan children treated with antimalarials despite a negative malaria test. Am J Trop Med Hyg. 2015;93(2):276-280. doi:10.4269/ajtmh.14-0494." />
<meta name="citation_reference" content="Prasad N, Murdoch DR, Reyburn H, Crump JA. Etiology of severe febrile illness in low- and middle-income countries: A systematic review. PLoS One. 2015;10(6):1-25. doi:10.1371/journal.pone.0127962." />
<meta name="citation_reference" content="Wilson MR, O’Donovan BD, Gelfand JM, et al. Chronic Meningitis Investigated via Metagenomic Next-Generation Sequencing. JAMA Neurol. 2018;94158. doi:10.1001/jamaneurol.2018.0463." />
<meta name="citation_reference" content="Naccache SN, Federman S, Veeraraghavan N, et al. A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples. Genome Res. 2014;24(7):1180-1192. doi:10.1101/gr.171934.113." />
<meta name="citation_reference" content="Oguttu DW, Matovu JKB, Okumu DC, et al. Rapid reduction of malaria following introduction of vector control interventions in Tororo District, Uganda: a descriptive study. Malar J. 2017;16(1):1-8. doi:10.1186/s12936-017-1871-3." />
<meta name="citation_reference" content="Mekonnen SK, Aseffa A, Medhin G, Berhe N, Velavan TP. Re-evaluation of microscopy confirmed Plasmodium falciparum and Plasmodium vivax malaria by nested PCR detection in southern Ethiopia. Malar J. 2014;13(48). doi:10.1186/1475-2875-13-48." />
<meta name="citation_reference" content="Agarwal R, Baid R, Datta R, Saha M, Sarkar N. Falciparum malaria and parvovirus B19 coinfection: A rare entity. Trop Parasitol. 2017;7(1):47-48. doi:10.4103/2229-682-5070.202299." />
<meta name="citation_reference" content="Duedu KO, Sagoe KWC, Ayeh-Kumi PF, Affrim RB, Adiku T. The effects of co-infection with human parvovirus B19 and Plasmodium falciparum on type and degree of anaemia in Ghanaian children. Asian Pac J Trop Biomed. 2013;3(2):129-139. doi:10.1016/S2221-1691(13)60037-4." />
<meta name="citation_reference" content="Toan NL, Sy BT, Song LH, et al. Co-infection of human parvovirus B19 with Plasmodium falciparum contributes to malaria disease severity in Gabonese patients. BMC Infect Dis. 2013;13(1). doi:10.1186/1471-2334-13-375." />
<meta name="citation_reference" content="Onyango CO, Welch SR, Munywoki PK, et al. Molecular epidemiology of human rhinovirus infections in Kilifi, coastal Kenya. J Med Virol. 2012;84(5):823-831. doi:10.1002/jmv.23251." />
<meta name="citation_reference" content="O’Callaghan-Gordo C, Bassat Q, Morais L, et al. Etiology and epidemiology of viral pneumonia among hospitalized children in rural mozambique: A malaria endemic area with high prevalence of human immunodeficiency virus. Pediatr Infect Dis J. 2011;30(1):39-44. doi:10.1097/INF.0b013e3181f232fe." />
<meta name="citation_reference" content="Mbayame Ndiaye Niang, Ousmane M. Diop, Fatoumata Diene Sarr, Deborah Goudiaby, Hubert Malou-Sompy, Kader Ndiaye, Astrid Vabret LB. Viral Etiology of Respiratory Infections in Children Under 5 Years Old Living in Tropical Rural Areas of Senegal: The EVIRA Project. J Med Virol. 2010;82:866-872. doi: 10.1002/jmv.21665." />
<meta name="citation_reference" content="Smuts HE, Workman LJ, Zar HJ. Human rhinovirus infection in young African children with acute wheezing. BMC Infect Dis. 2011;11(1):65. doi:10.1186/1471-2334-11-65." />
<meta name="citation_reference" content="Jain S, Self WH, Wunderink RG, et al. Community-Acquired Pneumonia Requiring Hospitalization among U.S. Adults. N Engl J Med. 2015;373(5):415-427. doi:10.1056/NEJMoa1500245." />
<meta name="citation_reference" content="Scully EJ, Basnet S, Wrangham RW, et al. Lethal Respiratory Disease Associated with Human Rhinovirus C in Wild Chimpanzees, Uganda, 2013. Emerg Infect Dis. 2018;24(2). doi:10.3201/eid2402.170778." />
<meta name="citation_reference" content="Diarrhoeal Disease. World Health Organization. 2017. http://www.who.int/news-room/fact-sheets/detail/diarrhoeal-disease. Accessed 20 June 2018." />
<meta name="citation_reference" content="Mwenda JM, Burke RM, Shaba K, et al. Implementation of Rotavirus Surveillance and Vaccine Introduction - World Health Organization African Region, 2007-2016. MMWR Morb Mortal Wkly Rep. 2017;66(43):1192-1196. doi:10.15585/mmwr.mm6643a7." />
<meta name="citation_reference" content="Calisher CH, Monath TP, Sabattini MS, et al. A newly recognized vesiculovirus, Calchaqui virus, and subtypes of Melao and Maguari viruses from Argentina, with serologic evidence for infections of humans and horses. Am J Trop Med Hyg. 1987;36(1):114-119." />
<meta name="citation_reference" content="Mohamed M, McLees A, Elliott RM. Viruses in the Anopheles A, Anopheles B, and Tete Serogroups in the Orthobunyavirus Genus (Family Bunyaviridae) Do Not Encode an NSs Protein. J Virol. 2009;83(15):7612-7618. doi:10.1128/JVI.02080-08." />
<meta name="citation_reference" content="Williams JE, Imlarp S, Top FH, Cavanaugh DC, Russell PK. Kaeng Khoi virus from naturally infected bedbugs (Cimicidae) and immature free tailed bats. Bull World Health Organ. 1976;53(4):365-369." />
<meta name="citation_reference" content="Patroca da Silva S, Chiang JO, Marciel de Souza W, et al. Characterization of the Gamboa Virus Serogroup (Orthobunyavirus Genus, Peribunyaviridae Family). Am J Trop Med Hyg. 2018;98(5):tpmd170810. doi:10.4269/ajtmh.17-0810." />
<meta name="citation_reference" content="Plyusnin A, Elliott RM. Bunyaviridae: Molecular and Cellular Biology.; 2011." />
<meta name="citation_reference" content="Lutwama JJ, Rwaguma EB, Nawanga PL, Mukuye A. Isolations of Bwamba virus from south central Uganda and north eastern Tanzania. Afr Health Sci. 2002;2(1):24-28." />
<meta name="citation_reference" content="WHO Model List of Essential Medicines. Edition 14, 2017" />
<meta name="citation_reference" content="UGANDA CLINICAL GUIDELINES. Ministry of Health. 2012." />
<meta name="citation_reference" content="Stenglein MD, Sanchez-Migallon Guzman D, Garcia VE, et al. Differential Disease Susceptibilities in Experimentally Reptarenavirus-Infected Boa Constrictors and Ball Pythons. J Virol. 2017;91(15):e00451-17. doi:10.1128/JVI.00451-17." />
<meta name="citation_reference" content="Dobin A, Davis CA, Schlesinger F, et al. STAR: Ultrafast universal RNA-seq aligner. Bioinformatics. 2013;29(1):15-21. doi:10.1093/bioinformatics/bts635." />
<meta name="citation_reference" content="Ruby JG, Bellare P, Derisi JL. PRICE: software for the targeted assembly of components of (Meta) genomic sequence data. G3 (Bethesda). 2013;3(5):865-880. doi:10.1534/g3.113.005967." />
<meta name="citation_reference" content="Li W, Godzik A. Cd-hit: A fast program for clustering and comparing large sets of protein or nucleotide sequences. Bioinformatics. 2006;22(13):1658-1659. doi:10.1093/bioinformatics/btl158." />
<meta name="citation_reference" content="Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357-359. doi:10.1038/nmeth.1923." />
<meta name="citation_reference" content="Wu TD, Nacu S. Fast and SNP-tolerant detection of complex variants and splicing in short reads. Bioinformatics. 2010;26(7):873-881. doi:10.1093/bioinformatics/btq057." />
<meta name="citation_reference" content="Ye Y, Choi J-H, Tang H. RAPSearch: a fast protein similarity search tool for short reads. BMC Bioinformatics. 2011;12(1):159. doi:10.1186/1471-2105-12-159." />
<meta name="citation_reference" content="Wilson MR, Fedewa G, Stenglein MD, et al. Multiplexed Metagenomic Deep Sequencing To Analyze the Composition of High-Priority Pathogen Reagents. mSystems. 2016;1(4):e00058-16. doi:10.1128/mSystems.00058-16." />
<meta name="citation_reference" content="Bogaert D, Keijser B, Huse S, et al. Variability and diversity of nasopharyngeal microbiota in children: A metagenomic analysis. PLoS One. 2011;6(2). doi:10.1371/journal.pone.0017035." />
<meta name="citation_reference" content="Pérez-Losada M, Alamri L, Crandall KA, Freishtat RJ. Nasopharyngeal microbiome diversity changes over time in children with asthma. PLoS One. 2017;12(1):1-13. doi:10.1371/journal.pone.0170543." />
<meta name="citation_reference" content="Hooper L V, Gordon JI. Commensal Host-Bacterial Relationships in the Gut. Science (80-). 2001;1115(10):1-7. doi:10.1126/science.1058709." />
<meta name="citation_reference" content="Sekirov I, Russell S, Antunes L. Gut microbiota in health and disease. Physiol Rev. 2010;90(3):859-904. doi:10.1152/physrev.00045.2009." />
<meta name="citation_reference" content="Bankevich A, Nurk S, Antipov D, et al. SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. J Comput Biol. 2012;19(5):455-477. doi:10.1089/cmb.2012.0021." />
<meta name="citation_reference" content="Mitchell A, Chang HY, Daugherty L, et al. The InterPro protein families database: The classification resource after 15 years. Nucleic Acids Res. 2015;43(D1):D213-D221. doi:10.1093/nar/gku1243." />
<meta name="citation_reference" content="Tsirigos KD, Peters C, Shu N, Käll L, Elofsson A. The TOPCONS web server for consensus prediction of membrane protein topology and signal peptides. Nucleic Acids Res. 2015;43(W1):W401-W407. doi:10.1093/nar/gkv485." />
<meta name="citation_reference" content="Giske CG, Cornaglia G. Supranational surveillance of antimicrobial resistance: The legacy of the last decade and proposals for the future. Drug Resist Updat. 2010;13(4-5):93-98. doi:10.1016/j.drup.2010.08.002." />
<meta name="twitter:title" content="Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Background Febrile illness is a major burden in African children, and non-malarial causes of fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics platform and service to identify microbes from metagenomic next-generation sequencing of tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, Uganda.

Results The most common pathogens identified were Plasmodium falciparum (51.1% of samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx of a child with rash and sepsis. We also identified two novel human rhinovirus C species.

Conclusions This exploratory pilot study demonstrates the utility of mNGS and the IDseq platform for defining the molecular landscape of febrile infectious diseases in resource limited areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases.

*   ### List of abbreviations
    
    AWS
    :   Amazon Web Services
    ASG
    :   Auto Scaling Groups
    BLAST
    :   Basic Local Alignment Search Tool
    EC2
    :   Elastic Compute Cloud
    ESBL
    :   Extended spectrum β-lactamase
    HHV
    :   human herpesvirus
    HIV-1
    :   human immunodeficiency 1 virus
    HRV
    :   Human Rhinovirus
    IQR
    :   Inter-quartile range
    L coding region
    :   Large segment coding region (RNA-dependent RNA polymerase)
    LZW
    :   Lempel-Ziv-Welch
    M coding region
    :   Medium segment coding region (glycoprotein)
    MEGA
    :   Molecular Evolutionary Genetics Analysis
    mNGS
    :   metagenomic next-generation sequencing
    NCBI
    :   National Center for Biotechnology Information
    NP swab
    :   Nasopharyngeal swab
    nr database
    :   non-redundant database
    nt database
    :   nucleotide database
    PRICE
    :   Paired-Read Iterative Contig Extension
    rpM
    :   reads per million
    RSV
    :   respiratory syncytial virus
    S coding region
    :   Small segment coding region (nucleocapsid)
    SDI
    :   Simpsons diversity Index
    SPAdes
    :   St Petersburg genome assembler
    SRST2
    :   Short Read Sequencing Typing
    STAR
    :   Spliced Transcripts Alignment to a Reference
    TTV
    :   torque teno virus
    VP1
    :   Capsid protein VP1
    WHO
    :   World Health Organization" />
<meta name="og-title" property="og:title" content="Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/385005v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Background Febrile illness is a major burden in African children, and non-malarial causes of fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics platform and service to identify microbes from metagenomic next-generation sequencing of tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, Uganda.

Results The most common pathogens identified were Plasmodium falciparum (51.1% of samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx of a child with rash and sepsis. We also identified two novel human rhinovirus C species.

Conclusions This exploratory pilot study demonstrates the utility of mNGS and the IDseq platform for defining the molecular landscape of febrile infectious diseases in resource limited areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases.

*   ### List of abbreviations
    
    AWS
    :   Amazon Web Services
    ASG
    :   Auto Scaling Groups
    BLAST
    :   Basic Local Alignment Search Tool
    EC2
    :   Elastic Compute Cloud
    ESBL
    :   Extended spectrum β-lactamase
    HHV
    :   human herpesvirus
    HIV-1
    :   human immunodeficiency 1 virus
    HRV
    :   Human Rhinovirus
    IQR
    :   Inter-quartile range
    L coding region
    :   Large segment coding region (RNA-dependent RNA polymerase)
    LZW
    :   Lempel-Ziv-Welch
    M coding region
    :   Medium segment coding region (glycoprotein)
    MEGA
    :   Molecular Evolutionary Genetics Analysis
    mNGS
    :   metagenomic next-generation sequencing
    NCBI
    :   National Center for Biotechnology Information
    NP swab
    :   Nasopharyngeal swab
    nr database
    :   non-redundant database
    nt database
    :   nucleotide database
    PRICE
    :   Paired-Read Iterative Contig Extension
    rpM
    :   reads per million
    RSV
    :   respiratory syncytial virus
    S coding region
    :   Small segment coding region (nucleocapsid)
    SDI
    :   Simpsons diversity Index
    SPAdes
    :   St Petersburg genome assembler
    SRST2
    :   Short Read Sequencing Typing
    STAR
    :   Spliced Transcripts Alignment to a Reference
    TTV
    :   torque teno virus
    VP1
    :   Capsid protein VP1
    WHO
    :   World Health Organization" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-08-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/385005v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/385005v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/385005v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/385005v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/117052" />
    <title>Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"w_9q8-uXVELAaSy02P65CttZyUFqZ9z446K1tUxes9w","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"117052","apath":"\/biorxiv\/early\/2018\/08\/06\/385005.atom","pisa":"biorxiv;385005v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;385005v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;385005\/DC2"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/385005v2","title":"Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis","identifier":"node\/117052"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022blLYwYwx9KsvJke4mpHTdhEZpH1e0YrDqLnAIR2nxyVYAfNI1jQrrIH1PbTO273ngIp6BDJtI29EE4AK8dAPjcb6Y6RLqqw\\\\\\\/\\\\\\\/FPx7u7CrCiJSJI7GLNsx25Tl\\\\\\\/3wkXG4lcOlLVMkOh\\\\\\\/CrQ4DzQBM\\\\\\\/+gIXrXhBEkjuksam7yIrZSOOFMO4c6DQ+Q+Spc6xWM6SLrAFWdud9yrBUUuQxjcQIuuhvw9QG2A0QzwVC4K1B6cQSs7s44ehmQTGVUNbGBx74zrNETAjEA+\\\\\\\/o0eXSQa+dXmPROwJGRwEJcMkeW6PR3dtq3UTC3Z7JTKngGeOs+KQ2SmD+XTt+lNvKTw28yTLvDbDhFVGnCPUFdE6GTFPOrf9eB6rJZi1auQaIAtyoC5motB8HjeebxVFVFlLSewwSBbR2w5OPELrKoMeuMXYI4+4zt\\\\\\\/FcoTUOgK5pKChb\\\\\\\/gUFwA9eygcqNFZF6XQ2rTwW7Z2rnjJK3rZLKXHohW+DfnJyrQaN7sx\\\\\\\/THBN3aeY+YNi4iwh5QSf5wZJqPVNqPmfbpGOBNf+N+NUUIh4M4xfqpkOQ6IMshhZtFuPKnP2Fz\\\\\\\/XhgsBq+I7rDkISt7BZGramod09LPJMVEZgBEkae\\\\\\\/rNfkxqWixUTcTEkoeKH5y8l7IRESRHgM\\\\\\\/i4mGP+dDcTuKS6HAub2pI\\\\\\\/\\\\\\\/Z2ZUXiGVjbiXSG2pOh9TLhHDsIM\\\\\\\/8J5+hv\\\\\\\/aVpPX95adt6btheplztnFE7Zzle\\\\\\\/f4CAdAIVLf0GvJUpI5SQtNmY2BlBiERxLTAtjI1YWEytkL0t+EveneqaYr1kRYNI3cJZMqulEQ5wQDONK6Z0PjVtyZ9d2vJSwf655REbA2Erq4B1sh+Al61MpnAaKj7yFPg02if96QYtTWH5CQNA79nUgymRxOg8+Fp4wrwRPCpn23rrnVZvKNhbmpzZvy6sfqOPidDESrBg+fu4AAlfaFB8Yrk8z6d1zQ7YLtVxCB8lbm7F7U\\\\\\\/EyAqwrCDKLvo6pUZ8lyOpPYX9gBwHMqFWYhqKvZ8M\\\\\\\/Pqp5n\\\\\\\/2GmX7ndJIFO+jM5RNocFqf1zsezAcVwM5Ry6VzRRhtQ7T2j+WDzkfHAglLKDtR1HMAaGlhyesl\\\\\\\/35z03HeL\\\\\\\/x7hDgIaoaSVAC7INymwWWTqYKFvEPbl0kiKwrvU\\\\\\\/BtnQIzQL+h+RYV8numuwauXIs\\\\\\\/1FYlBBxsvPUltwy\\\\\\\/BA8i6WEWu8Ypr9873ArMUqjE6fY3\\\\\\\/vBXDqPZvwtdut\\\\\\\/8Osl48CApOAeQsAos5+s\\\\\\\/uanb5Szizc3F5\\\\\\\/AitBBPVLHlDUG2XbaX9biex8crbOKO+6TCE1EfdjEzVYRxf3uEAA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022+L0enT92wzqbqFtCNUVmjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ae36df24e763ff5a4e02766653e56dfdfc7ae449699caa5e50b404b9a283ecf3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/385005v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-117052 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/385005v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xYS36pzc2aFhX_iQdZiNYWgMtUpQ-aA4yQ0NBM2lIHw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="117052" id="node117052" data-pisa="biorxiv;385005v2" data-pisa-master="biorxiv;385005" data-apath="/biorxiv/early/2018/08/06/385005.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Akshaya</span> <span class="nlm-surname">Ramesh</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sara</span> <span class="nlm-surname">Nakielny</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Hsu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Kyohere</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Oswald</span> <span class="nlm-surname">Byaruhanga</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Charles de</span> <span class="nlm-surname">Bourcy</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Egger</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Dimitrov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yun-Fang</span> <span class="nlm-surname">Juan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Sheu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">James</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Katrina</span> <span class="nlm-surname">Kalantar</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Charles</span> <span class="nlm-surname">Langelier</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Theodore</span> <span class="nlm-surname">Ruel</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Arthur</span> <span class="nlm-surname">Mpimbaza</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Philip J.</span> <span class="nlm-surname">Rosenthal</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Joseph L.</span> <span class="nlm-surname">DeRisi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/385005 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node117052" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Akshaya Ramesh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Weill Institute for Neurosciences, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Neurology, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Akshaya%2BRamesh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramesh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAkshaya%2BRamesh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sara Nakielny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Biophysics, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BNakielny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nakielny%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BNakielny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer Hsu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Division of Infectious Diseases, Department of Medicine, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BHsu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hsu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BHsu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mary Kyohere </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Infectious Diseases Research Collaboration</span>, Kampala, <span class='nlm-country'>Uganda</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BKyohere%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kyohere%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BKyohere%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Oswald Byaruhanga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Infectious Diseases Research Collaboration</span>, Kampala, <span class='nlm-country'>Uganda</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oswald%2BByaruhanga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byaruhanga%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOswald%2BByaruhanga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles de Bourcy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2Bde%2BBourcy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourcy%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2Bde%2BBourcy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rebecca Egger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BEgger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Egger%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BEgger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Boris Dimitrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boris%2BDimitrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dimitrov%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoris%2BDimitrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yun-Fang Juan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun-Fang%2BJuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juan%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYun-Fang%2BJuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jonathan Sheu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BSheu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sheu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BSheu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">James Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Katrina Kalantar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Biophysics, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrina%2BKalantar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalantar%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatrina%2BKalantar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Charles Langelier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Division of Infectious Diseases, Department of Medicine, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BLangelier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langelier%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BLangelier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Theodore Ruel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Division of Pediatric Infectious Diseases and Global Health, Department of Pediatrics, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theodore%2BRuel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruel%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheodore%2BRuel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Arthur Mpimbaza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Child Health and Development Centre, Makerere University</span>, Kampala, <span class='nlm-country'>Uganda</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arthur%2BMpimbaza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mpimbaza%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArthur%2BMpimbaza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Michael R. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Weill Institute for Neurosciences, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Neurology, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Philip J. Rosenthal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Division of Infectious Diseases, Department of Medicine, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BJ.%2BRosenthal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosenthal%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BJ.%2BRosenthal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Joseph L. DeRisi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Biophysics, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BL.%2BDeRisi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeRisi%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BL.%2BDeRisi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/385005v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:117052" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:117052/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/385005v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:117052" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:117052/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/385005v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:117052" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:117052/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/385005v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:117052" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:117052/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/385005v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:117052" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:117052/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/385005v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:117052" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:117052/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><div id="sec-1" class="subsection"><p id="p-4"><strong>Background</strong> Febrile illness is a major burden in African children, and non-malarial causes of fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics platform and service to identify microbes from metagenomic next-generation sequencing of tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, Uganda.</p></div><div id="sec-2" class="subsection"><p id="p-5"><strong>Results</strong> The most common pathogens identified were <em>Plasmodium falciparum</em> (51.1% of samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx of a child with rash and sepsis. We also identified two novel human rhinovirus C species.</p></div><div id="sec-3" class="subsection"><p id="p-6"><strong>Conclusions</strong> This exploratory pilot study demonstrates the utility of mNGS and the IDseq platform for defining the molecular landscape of febrile infectious diseases in resource limited areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases.</p></div></div><div class="section" id="sec-4"><h2 class="">Introduction</h2><p id="p-7">The evaluation of children with fever is challenging, particularly in the developing world. A febrile child in sub-Saharan Africa may have a mild self-resolving viral infection or may be suffering bacterial sepsis or malaria—major causes of disability and death [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>], [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. In the past, febrile illness in children under five years of age in most of Africa was treated empirically as malaria due to the limited availability of diagnostics and the risk of untreated malaria progressing to life-threatening illness. This strategy changed with new guidelines from the World Health Organization (WHO) in 2010, which recommend limiting malaria therapy to those with a confirmed diagnosis [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. However, standard recommendations for management of febrile children who do not have malaria are lacking. Increased knowledge about the prevalence of non-malarial pathogens associated with fever is needed to inform management strategies for febrile children who do not have malaria [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>].</p><p id="p-8">Advances in genome sequencing hold promise for addressing global infectious disease challenges by enabling unbiased detection of microbial pathogens without requirement for the extensive infrastructure of modern microbiology laboratories [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>], [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Although sequence-based diagnostics have not yet replaced most traditional microbiological assays, this situation is rapidly changing, as sequence-based strategies are incorporated for clinical care [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]–[<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], and costs have declined dramatically over the past decade [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. A significant roadblock toward implementation of sequence-based diagnostics is the extensive computational requirements and bioinformatics expertise required. In fact, as sequencing costs decline, computational expenses may proportionally increase due to inevitable expansion of existing genomic databases.</p><p id="p-9">To address these challenges, we developed IDseq, a cloud-based open-source bioinformatics platform and service for detection of microbial pathogens from metagenomic next-generation sequencing (mNGS) data. IDseq requires minimal computational hardware and is designed to enhance accessibility and build informatics capacity in resource limited regions. Here, we leveraged IDseq and mNGS data to perform an exploratory proof-of-concept molecular survey of febrile children in rural Uganda to characterize pathogens associated with fever, including both well-recognized and novel causes of illness.</p></div><div class="section" id="sec-5"><h2 class="">Analyses/Results</h2><div id="sec-6" class="subsection"><h3>Clinical presentations of children providing samples for analysis</h3><p id="p-10">From October to December, 2013, 94 children admitted to Tororo District Hospital were enrolled (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Their mean age was 16.4 (IQR: 8.0-12.0) months, and 66 (70.2%) were female. Chief symptoms reported in addition to fever were cough (88.3%), vomiting (56.4%), diarrhea (47.9%), and convulsions (27.7%). Top admitting diagnoses were respiratory tract infection (57.4%), gastroenteritis/diarrhea (29.8%), and septicemia (11.7%) (Data file 1a).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/567805/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/567805/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/567805" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-11" class="first-child">Overview of the patients enrolled in the study</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Serum and nasopharyngeal (NP) swab samples were collected from 90 children each; for four children, only one of the two sample types was successfully collected. Although 45 (47.9%) of the children had a presenting symptom of diarrhea, stool samples were available for only 10 due to logistical constraints. Blood smears identified <em>P. falciparum</em> in 12 of the 90 samples that underwent mNGS analysis.</p></div><div id="sec-7" class="subsection"><h3>Metagenomic sequencing findings</h3><p id="p-13">mNGS was performed on 90 serum, 90 NP swab, and 10 stool samples and analyses was done using the IDseq pipeline (see Methods section, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); detailed findings are reported in Data file S1b. For each sample type, RNA was extracted, libraries were prepared, samples were sequenced, and reads were analyzed using the IDseq Portal. A mean of 11.5 million (IQR 6.4 – 15.2 million) paired-end reads were obtained per sample; sequencing statistics are in Data file Table S2. For one batch of serum samples, only a single read, rather than paired-end reads, was produced.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of IDseq pipeline" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of IDseq pipeline&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F1.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F1.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-14" class="first-child">Schematic representation of IDseq pipeline</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>mNGS of serum</h3><p id="p-15">At least one microbial species was detected in 60 (66.7%) serum samples; more than one microbe was detected in 11 (12.2%) samples (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The most commonly identified microbes were <em>Plasmodium falciparum</em> (46, 51.1%) and parvovirus B19 (4, 4.4%). <em>P. falciparum</em> was detected in 10/12 samples from patients reported as smear-positive for malaria parasites. mNGS detected <em>Plasmodium spp.</em> in 37 additional samples that were smear negative (36 <em>P. falciparum</em>, 1 <em>P. malariae</em>). Viruses detected from serum included human immunodeficiency 1 virus (HIV-1), hepatitis A virus, rotavirus A, human herpesvirus (HHV) type 6, HHV type 4, HHV type 7, human rhinovirus (HRV)-C, HRV-A, enteroviruses (enterovirus A71, Coxsackievirus B2 and echovirus E30), human parechovirus 2, hepatitis B virus, a novel orthobunyavirus (described in greater detail below), human cardiovirus (Saffold virus), mamastrovirus 1 and Norwalk virus (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Microbes found in (A) serum and (B) nasopharyngeal (NP) swab samples. Note that bacterial species were not considered for NP samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Microbes found in (A) serum and (B) nasopharyngeal (NP) swab samples. Note that bacterial species were not considered for NP samples.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F2.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F2.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-16" class="first-child">Microbes found in (A) serum and (B) nasopharyngeal (NP) swab samples. Note that bacterial species were not considered for NP samples.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">Multiple viruses were detected from serum in patients with <em>Plasmodium</em> infections (10 of 46 (21.7%) samples; <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S1a</a>). Three of the four identified parvovirus B19 infections were associated with <em>P. falciparum</em>. Additionally, GB virus C and torque teno virus (TTV), which are of unknown clinical significance [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], were identified in the serum of 25 (27.8%) and 37 (41.1%) children, respectively.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/567826/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/567826/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/567826" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1:</span> <p id="p-88" class="first-child">(A) Co-infection table for <em>P. falciparum</em></p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/567807/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/567807/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/567807" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1:</span> <p id="p-89" class="first-child">B) Co-infection table for HRV</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>mNGS of NP swabs</h3><p id="p-18">A total of 90 NP swabs was collected and processed; 52 (57.7%) of these were from patients with admission diagnoses of pneumonia, respiratory tract infection, or bronchiolitis (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Chest imaging was not available to further assess these diagnoses. 73 NP samples (81.1%) contained at least 1 viral species (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). HRV-A and HRV-C were the most prevalent, followed by respiratory syncytial virus (RSV), cytomegalovirus (HHV-5), influenza B, and coronavirus OC43. Other respiratory viruses identified included influenza A, HRV-B, adenovirus B, 3 human parainfluenza virus types, metapneumovirus, coronavirus NL63, avian coronavirus, coxsackievirus A2, coxsackievirus B2, polyomaviruses (KI), HHV-6 and HHV-7. Other viruses identified that are not typically considered respiratory pathogens included hepatitis A virus, hepatitis B virus, parvovirus B19, mamastrovirus 1, Bwamba orthobunyavirus, betapapillomavirus 1, and rotavirus. Additionally, TTV was found in 49 (54.4%) NP swab samples, including one sample with both gemykrogvirus and TTV. For 26 (28.8%) patients, mNGS identified respiratory viral co-infections, most commonly with HRV-C (n=11) and HRV-A (n=5) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S1b</a>). The same microbial species was identified in the NP swab and serum samples in 6 patients, one each with HRV-A, HRV-C, hepatitis A virus, hepatitis B virus, rotavirus A, and parvovirus B19.</p><p id="p-19">Bacteria identified in NP samples included four dominant genera, which together comprised 79% of all microbial reads—<em>Moraxella</em> (39.4%), <em>Haemophilus</em> (16.7%), <em>Streptococcus</em> (16.2%), and <em>Corynebacterium</em> (6.6%). Given that diversity loss in the microbial flora in lower respiratory tract samples correlates with pneumonia [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>], [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>], we compared the Simpsons diversity Index (SDI) in patients with and without clinical diagnoses of respiratory tract infection. We found no significant difference between patients with (mean SDI = 0.51, IQR 0.37 −0.65) or without (mean SDI = 0.51, IQR = 0.42 - 0.65; p = 0.86) diagnoses of respiratory infection (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S1</a>). This finding is consistent with a growing body of work demonstrating that microbial composition of the nasopharynx may not correlate well with that of the lower respiratory tract in patients with pneumonia [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]–[<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>].</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simpsons diversity index (SDI) for samples with pneumonia versus other etiologies. Each triangle represents one sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simpsons diversity index (SDI) for samples with pneumonia versus other etiologies. Each triangle represents one sample.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F5.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F5.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1:</span> <p id="p-85" class="first-child">Simpsons diversity index (SDI) for samples with pneumonia versus other etiologies. Each triangle represents one sample.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>mNGS of stool</h3><p id="p-20">Among the 10 stool samples collected and sequenced, pathogens were detected in 9/10 samples. The three most common pathogens identified were rotavirus A (50%), <em>Cryptosporidium</em> (40%), and human parechovirus (40%). Seven children had co-infections: two rotavirus A and human parechovirus, and one each rotavirus A and <em>Cryptosporidium</em>, rotavirus A and enterovirus, <em>Cryptosporidium</em> and human parechovirus, <em>Cryptosporidium</em> and Norwalk virus, and <em>Giardia</em> and human parechovirus. Five samples also had <em>Blastocystis hominis</em>, a microbe of uncertain pathogenicity.</p></div><div id="sec-11" class="subsection"><h3>Orthobunyaviruses</h3><p id="p-21">Serum from one patient admitted with a clinical diagnosis of malaria and pneumonia contained a novel orthobunyavirus in addition to <em>P. falciparum</em>. Assembly of a draft genome and comparison with existing orthobunyavirus genomes indicated that this draft sequence includes 97.5%, 100% and 91% of the L, M, and S coding regions, respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3</a>). Average read coverage across the segments was 86-fold. Phylogenetic comparison showed that the novel virus was significantly divergent from known orthobunyaviruses, sharing 44.9-55.1% amino acid identity with the closest known relatives, Calchaqui virus, Kaeng Khoi virus, and Anopheles A virus (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figures S3a-c</a>). Of note, despite the divergence of this virus, 47.9% of the reads that belong to this new genome were detectable using default parameters in the IDseq pipeline, allowing for facile subsequent assembly. The virus was isolated from a patient from Nyangole village, Tororo District—hence, we propose the name “Nyangole virus”, consistent with nomenclature guidelines for the family <em>Bunyaviridae</em>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Antimicrobial resistant genes identified in (A) nasopharynx and (B) stool samples" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Antimicrobial resistant genes identified in (A) nasopharynx and (B) stool samples&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F6.medium.gif" width="315" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F6.medium.gif" width="315" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <p id="p-86" class="first-child">Antimicrobial resistant genes identified in (A) nasopharynx and (B) stool samples</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complete phylogenetic tree of (A) Large (L) or RNA dependent RNA polymerase, (B) Medium (M) or polyprotein of Gn, NSm and Gc proteins and (C) Small (S) segment encoding Nucleocapsid segments" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Complete phylogenetic tree of (A) Large (L) or RNA dependent RNA polymerase, (B) Medium (M) or polyprotein of Gn, NSm and Gc proteins and (C) Small (S) segment encoding Nucleocapsid segments&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-8.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-8.medium.gif" width="416" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complete phylogenetic tree of (A) Large (L) or RNA dependent RNA polymerase, (B) Medium (M) or polyprotein of Gn, NSm and Gc proteins and (C) Small (S) segment encoding Nucleocapsid segments" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Complete phylogenetic tree of (A) Large (L) or RNA dependent RNA polymerase, (B) Medium (M) or polyprotein of Gn, NSm and Gc proteins and (C) Small (S) segment encoding Nucleocapsid segments&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-9.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-9.medium.gif" width="401" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complete phylogenetic tree of (A) Large (L) or RNA dependent RNA polymerase, (B) Medium (M) or polyprotein of Gn, NSm and Gc proteins and (C) Small (S) segment encoding Nucleocapsid segments" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Complete phylogenetic tree of (A) Large (L) or RNA dependent RNA polymerase, (B) Medium (M) or polyprotein of Gn, NSm and Gc proteins and (C) Small (S) segment encoding Nucleocapsid segments&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-10.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-10.medium.gif" width="437" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F7/graphic-10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure S3:</span> <p id="p-87" class="first-child">Complete phylogenetic tree of (A) Large (L) or RNA dependent RNA polymerase, (B) Medium (M) or polyprotein of Gn, NSm and Gc proteins and (C) Small (S) segment encoding Nucleocapsid segments</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of the novel orthobunyavirus identified in a febrile child. (A) Schematic representation of the large (L) or RNA dependent RNA polymerase, medium (M) or polyprotein of Gn, NSm and Gc proteins and small (S) segment encoding the nucleocapsid (N) protein of Nyangole virus and percentage identity with the most closely related virus. Phylogenetic tree of all complete orthobunyavirus genome sequences along with Nyangole virus are represented in (B) for the RNA dependent RNA polymerase and (C) for the glycoprotein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characterization of the novel orthobunyavirus identified in a febrile child. (A) Schematic representation of the large (L) or RNA dependent RNA polymerase, medium (M) or polyprotein of Gn, NSm and Gc proteins and small (S) segment encoding the nucleocapsid (N) protein of Nyangole virus and percentage identity with the most closely related virus. Phylogenetic tree of all complete orthobunyavirus genome sequences along with Nyangole virus are represented in (B) for the RNA dependent RNA polymerase and (C) for the glycoprotein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F3.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F3.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-22" class="first-child">Characterization of the novel orthobunyavirus identified in a febrile child. (A) Schematic representation of the large (L) or RNA dependent RNA polymerase, medium (M) or polyprotein of Gn, NSm and Gc proteins and small (S) segment encoding the nucleocapsid (N) protein of Nyangole virus and percentage identity with the most closely related virus. Phylogenetic tree of all complete orthobunyavirus genome sequences along with Nyangole virus are represented in (B) for the RNA dependent RNA polymerase and (C) for the glycoprotein.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">In addition, a second orthobunyavirus, Bwamba virus, was identified in the NP swab sample from a patient admitted with rash, sepsis, and diarrhea. Insufficient sample and sequencing reads precluded genome assembly of this virus.</p></div><div id="sec-12" class="subsection"><h3>Genomic characterization of human rhinoviruses and influenza B in NP swabs</h3><div id="sec-13" class="subsection"><h4>Human rhinoviruses</h4><p id="p-24">Within the species rhinovirus, we assembled <em>de novo</em> a total of 13 HRV-C (mean coverage: 39-fold) and 13 HRV-A (mean coverage: 268-fold) genomes (&gt; 500 bp (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>)). Of these, 10 HRV-A and 9 HRV-C genomes had complete coverage of the VP1 region, which is used to define enterovirus types [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. HRV types are defined by divergence of &gt;73% in the VP1 gene. As such, we found three HRV-A and eight HRV-C types in this cohort. One individual harbored two distinct HRV-A types (genome pairwise identity=75.3%, VP1 pairwise identity=67.1%). Additionally, we assembled two novel HRV-C isolates from two patients admitted with gastroenteritis (patient ID: EOFI-014), and pneumonia, malaria and diarrhea (patient ID: EOFI-133), that shared 70.1% and 70.7% nucleotide sequence identity at VP1 compared to the closest known HRV-C (Accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JQ245968&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">JQ245968</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KF688606&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">KF688606</a>, respectively). The Picornavirus Working Group has established that novel HRV-Cs should exhibit at least 13% nucleotide sequence divergence in the VP1 gene [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>], qualifying these two isolates as novel.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of all complete HRV genomes from NCBI and HRV genomes assembled in this study (Purple &#x2013; Rhinovirus A, Orange &#x2013; Rhinovirus C)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949649374" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic tree of all complete HRV genomes from NCBI and HRV genomes assembled in this study (Purple &#x2013; Rhinovirus A, Orange &#x2013; Rhinovirus C)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F4.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F4.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/08/06/385005/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-25" class="first-child">Phylogenetic tree of all complete HRV genomes from NCBI and HRV genomes assembled in this study (Purple – Rhinovirus A, Orange – Rhinovirus C)</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h4>Influenza B virus</h4><p id="p-26">We assembled influenza B genome segments (&gt;500bp, mean-coverage: 41-fold) from six of seven samples containing influenza B virus (one sample had insufficient sequencing reads). The viruses assembled were &gt;99% similar to each other and &gt;99% identical to the B/Massachusetts/02/2012-like virus included in the vaccine recommended by WHO for the 2013-2014 northern hemisphere and 2014 southern hemisphere influenza seasons (Accession numbers: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_002204&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">NC_002204</a> to <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_002211&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">NC_002211</a>).</p></div><div id="sec-15" class="subsection"><h4>Antimicrobial resistance profiling in the nasopharynx and stool</h4><p id="p-27">We employed Short Read Sequencing Typing (SRST2) to survey the resistance gene landscape of our metagenomic dataset [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. In total, 86% of patients were found to harbor molecular evidence of resistant organisms in their nasopharynx and 50% in their stool. For both stool and NP swabs, genes conferring resistance to beta lactams were the most abundant, and included <em>ampC</em> (n=2), which confers resistance to first through third generation cephalosporins, and <em>IMP-1</em> (n=1) which confers broad spectrum resistance that includes carbapenems. In addition, genetic signatures of resistance to other antibacterial agents including aminoglycosides, macrolides/lincosamides, phenicols, sulfas, and tetracyclines were identified (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S2</a>).</p></div></div></div><div class="section" id="sec-16"><h2 class="">Discussion</h2><p id="p-28">The clinical management of children with fever is challenging in Africa, where clinicians often have access only to malaria diagnostics. A better understanding of the microbial agents causing fever in African children is needed to inform the development of better diagnostic algorithms and therapeutic guidelines. To address this unmet need, we developed and deployed IDseq in combination with unbiased mNGS to characterize the etiology of fever in Ugandan children admitted to a rural district hospital. We identified a wide range of pathogens in these children.</p><p id="p-29">Other studies evaluating causes of febrile illness in African children have focused on a limited number of pathogens [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]–[<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. In a study of febrile children in Tanzania utilizing rapid diagnostic tests, serologic tests, culture, and molecular tests, viruses accounted for 51% of lower respiratory infections, 78% of systemic infections, and 100% of nasopharyngeal infections. Additionally, 9% of the children had malaria and 4.2% bacteremia [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. In febrile children in Kenya, reported pathogens were spotted fever group <em>Rickettsia</em> (22.4%), influenza (22.4%), adenovirus (10.5%,) parainfluenza virus 1-3 (10.1%), Q fever (8.9%), RSV (5.3%), malaria (5.2%), scrub typhus (3.6%), human metapneumovirus (3.2%), group A <em>Streptococcus</em> (2.3)% and typhus group Rickettsiae (1.0%) [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>], [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Another study reported bacteremia in 19.1% of children admitted to a referral hospital in Uganda [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. Additionally, in patients (across all age groups) with severe febrile illness, bacteremia was detected in 10.1% in North Africa, 10.4% in East Africa, and 12.4% in West Africa [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>].</p><p id="p-30">Traditional pathogen detection methods, including culture, serology, and pathogen directed molecular methods, are logistically challenging in resource limited settings due to the need for extensive microbiology laboratory infrastructure. Unbiased sequencing approaches are designed to identify all potential pathogens, but have been limited by high cost. The costs of deep sequencing are rapidly decreasing, but the analysis of mNGS data necessarily incurs an increasingly large cost, as the available genomic databases to be searched continue to expand. Other major challenges for mNGS approaches include the need for better control datasets, in particular to enable discrimination of contaminants and commensal organisms from true pathogens. The IDseq platform was based on our prior experience with in-house pipelines for pathogen detection [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>], [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>], and it aims to address existing computational and bioinformatics barriers by providing facile cloud-based mNGS analysis without the requirement for significant on-premise computational infrastructure.</p><p id="p-31">Using mNGS and IDseq, the most common pathogen identified in the blood of febrile Ugandan children was <em>P. falciparum,</em> an expected result considering the high incidence of malaria in Tororo District at the time of this study [<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>]. Some discrepancies were seen compared to blood smear readings, with false positive smears probably due to errors in slide reading, a common problem in African clinics [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>], and false negative smears due to the expected greater sensitivity of mNGS for identification of <em>P. falciparum</em>. In children with only sub-microscopic parasitemia, it is uncertain whether fevers can be ascribed to malaria, and in fact many children had both <em>P. falciparum</em> and additional pathogens identified. Interestingly, three of the four cases of parvovirus B19 were found in association with <em>P. falciparum</em>; this co-infection has been associated with severe anemia with life-threatening consequences [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]–[<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>].</p><p id="p-32">HRV was the most commonly identified virus in NP swab samples, consistent with findings previously reported in sub-Saharan Africa and developed countries [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]–[<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. HRV-C was most frequently encountered (54.1%), followed by HRV-A (43.2%) and HRV-B (2.7%), similar to the distribution of HRVs previously reported in Kenya [<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>]. We identified two novel HRV-C species; these viruses were about 70% identical to the most closely related previously described HRV-C species [<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>]. Overall, we detected at least three HRV-A and eight HRV-C types co-circulating in Tororo District. Of note, during the same collection period, a lethal HRV-C outbreak was reported in chimpanzees in Kibale National Park, in western Uganda [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. The HRV-C reported in western Uganda was modestly related to an isolate observed in our study (74% nucleotide identity; 81% amino acid identity) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) [<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>]. Our results confirm that a wide spectrum of HRVs infects Ugandan children. In addition to HRV, we detected a spectrum of other known respiratory viruses, including RSV, human parainfluenza viruses, human coronaviruses, and adenovirus.</p><p id="p-33">Diarrheal disease is one of the leading causes of death in children in Africa [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. Approximately 48% of febrile children in our study presented with diarrhea, but due to logistical constraints stool specimens were available for only 10 cases. Rotavirus A, the leading cause of pediatric diarrhea worldwide [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], was the most commonly identified pathogen in this cohort. Rotavirus vaccination, known to be highly effective, is yet to be implemented in Uganda, but the need is clear [<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>]. In addition to rotavirus A, we detected <em>Cryptosporidium</em>, norovirus, <em>Giardia</em>, <em>B. hominis</em> and several enteroviruses in stool specimens. Enteroviruses, HRV-C, and mamastrovirus were also identified in the serum of three children with clinical diagnoses of gastroenteritis or diarrhea.</p><p id="p-34">Unbiased inspection of microbial sequences from sera revealed a novel member of the orthobunyavirus genus, tentatively named Nyangole virus, which was identified as a co-infection with <em>P. falciparum</em> in a child with clinical diagnoses of malaria and pneumonia. The virus was surprisingly divergent from known viruses, with an average amino acid similarity of 51.6% to its nearest known relatives including Calchaqui, Anopheles A and Kaeng Khoi viruses. Mosquitoes have been proposed as a vector for Calchaqui and Anopheles A viruses; Kaeng Khoi virus has been isolated from bedbugs [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]–[<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. Antibodies to these viruses have been detected in human sera, but their role as human pathogens is uncertain [<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>]–[<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. While the depth of coverage of Nyangole virus sequence in our patient suggests significant viremia, and other orthobunyaviruses are responsible for severe human illnesses (e.g., California encephalitis virus, La Crosse virus, Jamestown Canyon virus, and Cache Valley virus) [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>], it is unknown whether the identified virus was responsible for the presenting symptoms.</p><p id="p-35">NP swab analysis identified another Orthobunyavirus, Bwamba virus, in a child admitted with rash, sepsis and diarrhea. This virus has previously been described to cause fever in Uganda [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. Our identification of two orthobunyaviruses, including one novel virus, in a small sample of febrile Ugandan children suggests that the landscape of previously unidentified viruses that infect African children and potentially cause febrile illness, is significantly under explored.</p><p id="p-36">Molecular evidence of resistance to every available antibiotic except vancomycin in the WHO Essential Medicines List was identified in this study [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. Three children had genotypic evidence of ESBL producing organisms predicted to be resistant to ceftriaxone, a frontline antibiotic for severe infections in this region [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. In addition, one of these three children carried IMP-1, which also confers resistance to carbapenems, a class of antibiotics reserved for the most resistant infections.</p><p id="p-37">Our exploratory pilot study had important limitations. First, our samples were not collected randomly, but rather were a convenience sample due to the logistical constraints of our small clinical study; as such the results should not be seen as broadly representative of pathogens infecting Ugandan children. In particular, the lack of identification of bacteremia in study subjects may have been due to a relative paucity of severe illness, compared to that in other studies, in our cohort of admitted children. Second, clinical evaluation of children followed the standards of a rural African hospital, so diagnostic evaluation was limited to physical examination and malaria blood smears. Much more may be learned by linking rigorous clinical evaluation with mNGS results, and thereby more comprehensively assessing associations between clinical syndromes and specific pathogens.</p><p id="p-38">Despite these limitations, our study demonstrates the utility of the mNGS/IDseq platform, and provides an important cross sectional snapshot of causes of fever in African children. Combining unbiased mNGS and the IDseq platform enabled the identification of likely known and novel causes of pediatric illness, and makes available a powerful new open access tool for the characterization of infectious diseases in resource limited settings</p><div id="sec-17" class="subsection"><h3>Potential implications</h3><p id="p-39">This study provides a snapshot of pediatric fever in Tororo District, Uganda. mNGS permits universal pathogen detection using a single assay, and thus avoids the need for multiple independent, and often costly tests to determine disease etiology. Despite the promise and decreasing costs of this technology, the extensive computational and bioinformatic infrastructure needed to perform analysis of sequencing data remains a major barrier to implementation of mNGS in the developing world. Here we address the need for bioinformatic democratization and computational capacity building with IDseq, an open access platform to bring infectious disease surveillance to regions where it is needed most. As progress is made toward elimination of malaria in sub-Saharan Africa, it will be increasingly important to understand the landscape of pathogens that account for the remaining burden of morbidity and mortality. The use of mNGS can contribute importantly to this understanding, offering unbiased identification of infecting pathogens.</p></div></div><div class="section" id="sec-18"><h2 class="">Materials and Methods</h2><div id="sec-19" class="subsection"><h3>Enrollment of study subjects</h3><p id="p-40">We studied children admitted to Tororo District Hospital, Tororo, Uganda, with febrile illnesses. Potential subjects were identified by clinic staff, who notified study personnel, who subsequently evaluated the children for study eligibility. Inclusion criteria were: 1) age 2-60 months; 2) admission to Tororo District Hospital for acute illness; 3) documentation of axillary temperature &gt;38.0°C on admission or within 24 hours of admission; and 4) provision of informed consent from the parent or guardian for study procedures.</p></div><div id="sec-20" class="subsection"><h3>Study specimens</h3><p id="p-41">NP swabs and blood were collected from each enrolled subject within 24 hours of hospital admission. Approximately 5 ml of blood was collected by phlebotomy, the sample was centrifuged at room temperature, and serum was then stored at −80°C. NP swab samples collected with FLOQSwabs™ swabs (COPAN) were placed into cryovials with Trizol (Invitrogen), and stored at −80°C within ~5 min of collection. For subjects with acute diarrhea (≥ 3 loose or watery stools in 24 hours), stool was collected into clean plastic containers and stored at −80°C within ~5 min of collection. Samples were stored at −80°C until shipment on dry ice to UCSF for sequencing.</p></div><div id="sec-21" class="subsection"><h3>Clinical data</h3><p id="p-42">Clinical information was obtained from interviews with parents or guardians, with specific data entered onto a standardized case record form that included admission diagnosis and physical examination as well as malaria blood smear results. For malaria diagnosis, thick blood smears were Giemsa stained and evaluated by Tororo District Hospital laboratory personnel following routine standard-of-care practices. No efforts were made to improve on routine practice, so malaria smear readings represent routine standard-of-care rather than optimal quality controlled reads.</p></div><div id="sec-22" class="subsection"><h3>Metagenomic next-generation sequencing (mNGS)</h3><p id="p-43">After shipment to University of California, San Francisco, RNA was extracted from clinical samples as well as positive (HeLa cells) and negative (water) controls, and unbiased cDNA libraries were generated using previously described method [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. Barcoded samples were pooled, size selected (Blue Pippin), and run on an Illumina HiSeq2500 to obtain 135 base pair (bp) paired-end reads</p></div><div id="sec-23" class="subsection"><h3>Bioinformatic analysis and pathogen identification</h3><p id="p-44">Microbial pathogens were identified from raw sequencing reads using the IDseq Portal (<a href="https://idseq.net">https://idseq.net</a>), a novel cloud-based, open-source bioinformatics platform designed for detection of microbes from metagenomic data (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). IDseq scripts and user instructions are available at <a href="https://github.com/chanzuckerberg/idseq-dag">https://github.com/chanzuckerberg/idseq-dag</a> and the graphical user interface web application for sample upload is available at <a href="https://github.com/chanzuckerberg/idseq-web">https://github.com/chanzuckerberg/idseq-web</a>. IDseq is conceptually based on previously implemented platforms [<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>], [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>], [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>], but is optimized for scalable Amazon Web Services (AWS) cloud deployment. Bioinformatics data processing jobs are carried out on demand as Docker containers using AWS Batch. Alignments to the National Center for Biotechnology Information (NCBI) database are executed on dedicated auto scaling groups (ASG) of Amazon Elastic Compute Cloud (EC2) instances, with the number of server instances varied with job load. Fast downloads of the NCBI database from the Amazon Simple Storage Service to each new server instance are enabled by the open-source tool s3mi (<a href="https://github.com/chanzuckerberg/s3mi">https://github.com/chanzuckerberg/s3mi</a>). Initial alignment and removal of reads derived from the human genome is performed using the Spliced Transcripts Alignment to a Reference (STAR) algorithm [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]. Low-quality reads, duplicates, and low-complexity reads are then removed using the Paired-Read Iterative Contig Extension (PRICE) computational package [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>], the CD-HIT-DUP tool [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>], and a filter based on the Lempel-Ziv-Welch (LZW) compression score, respectively. A second round of human read filtering is carried out using bowtie2 [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]. Remaining reads are queried against the most recent version of the NCBI nucleotide and non-redundant protein databases (updated monthly) using GSNAPL and RAPSearch2 [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>], [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>], respectively. Reads matching GenBank records in the superphylum Deuterostomia are removed, given the high likelihood that such residual reads are of human origin. The relative abundance of microbial taxa is calculated based on reads per million (rpM) mapped at the genus level. To distinguish potential pathogens from ubiquitous environmental agents including commensal flora, a Z-score is calculated for the value observed for each genus relative to a background of healthy and no-template control samples [<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>]. An overview of this pipeline is represented in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>.</p><p id="p-45">IDseq can process 150 samples at a given time. As of this writing, the current version of the IDseq pipeline (IDseqv1.8) processes fastq files with approximately 70 million reads, typically containing &gt;99% host sequence, in 34 minutes. Run times may vary depending on demand, percentage of non-host sequence, and autoscaling parameters.</p><p id="p-46">For this study we report species greater than 0 rpM and Z-scores detected in the serum, stool, and NP samples. Consistent with previous studies, low levels of “index bleed through” or “barcode hopping” (assignment of sequencing reads to the wrong barcode/index) was observed within the non-templated control samples [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. To prevent mis-assignment, when a microbe found in more than one sample, it was reported only when present at levels at least four times the level of mis-assigned reads observed in the control samples. Given the extremely high levels of rotavirus found in stool samples, these samples were run in duplicate, and only microbes identified in both samples and present at levels at least four times the number of reads mis-assigned in the control samples were reported. If the reads identified for a given pathogen were not species-specific, we reported the corresponding genus. For NP and stool samples, because the nasopharynx and intestines are normally colonized with commensal bacteria [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]–[<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>], only non-bacterial species were reported.</p></div><div id="sec-24" class="subsection"><h3>Genome assembly, annotation and phylogenetic analysis</h3><p id="p-47">To more comprehensively characterize the genomes of identified microbes PRICE [<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a>] and St. Petersburg genome assembler (SPAdes) [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>] were used to <em>de novo</em> assemble short read sequences into larger contiguous sequences (contigs). Assembled contigs were queried against the NCBI nt database using BLAST to identify the closest related microbes. GenBank annotation files from genome sequence records corresponding to the highest scoring alignments were used to identify potential features within the <em>de novo</em> assembled genomes. Geneious v10.3.2 was used to annotate newly assembled genomes. Reference genomes for multiple sequence alignments and phylogenetic analyses were downloaded from NCBI. Multiple sequence alignments were generated using ClustalW in MEGA v6.0 and the Geneious aligner in Geneious v10.3.2. Neighbor-joining phylogenetic trees were generated using Geneious v10.3.2 and further assessed using FigTree v1.4.3. Annotation of protein domains in the novel orthobunyaviruses was performed using the InterPro webserver [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>] as well as direct alignment against previously known orthobunyaviruses. The TOPCONS webserver [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>] was used for the identification of transmembrane regions and signal peptides, and the NetNglyc 1.0 Server (<a href="http://www.cbs.dtu.dk/services/NetNGlyc/">http://www.cbs.dtu.dk/services/NetNGlyc/</a>) for the identification of glycosylation sites.</p></div><div id="sec-25" class="subsection"><h3>Evaluation of NP microbiome diversity</h3><p id="p-48">We applied SDI to evaluate alpha diversity of microbes identified in NP samples. For this analysis patients were stratified into two categories based on clinical assignment: respiratory infections (admitting diagnosis of pneumonia, respiratory tract infection, or bronchiolitis; n=52) and all other syndromes (n=39); cases with unknown admitting diagnosis were excluded. SDI was calculated in R using the Veganv2.4.4 package on genus-level reads per million values for all microbes, including bacteria. A Wilcox Rank Sum test was used to evaluate differences in SDI between patients in the two categories.</p></div><div id="sec-26" class="subsection"><h3>Antimicrobial resistance gene identification in nasopharynx and stool</h3><p id="p-49">The SRST2 computational package was used to identify antimicrobial resistance genes using the Argannot2 database as previously described [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]. We defined extended spectrum β-lactamase (ESBL) as a β-lactamase conferring resistance to the penicillins, first-, second-, and third-generation cephalosporins including the gene classes proposed by Giske et al [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>].</p></div></div><div class="section" id="sec-27"><h2 class="">Availability of data and code</h2><p id="p-50">All raw data have been deposited under Bioproject ID: PRJNA483304. Assembled genomes can be accessed in GenBank: Accession numbers: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH685676&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH685676</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH685701&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH685701</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH685703&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH685703</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH685719&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH685719</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH684286&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH684286</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH684293&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH684293</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH684298&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH684298</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH684334.&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom">MH684334.</a> All the raw data, intermediate data and IDseq reports can also be accessed at <a href="https://idseq.net">https://idseq.net</a> [Project ID: Uganda - all - 2]. All IDseq scripts and user instructions are available at <a href="https://github.com/chanzuckerberg/idseq-dag">https://github.com/chanzuckerberg/idseq-dag</a> and the graphical user interface web application for sample upload is available at <a href="https://github.com/chanzuckerberg/idseq-web">https://github.com/chanzuckerberg/idseq-web</a>. While IDseq is under continuous development and improvement, full version control of processing runs and reference databases are standard features.</p></div><div class="section" id="sec-28"><h2 class="">Declarations</h2><div class="section glossary"><h3>List of abbreviations</h3><dl id="def-list-1"><dt id="def-item-1">AWS<span class="def-item-dt-sep"></span></dt><dd>Amazon Web Services<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">ASG<span class="def-item-dt-sep"></span></dt><dd>Auto Scaling Groups<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">BLAST<span class="def-item-dt-sep"></span></dt><dd>Basic Local Alignment Search Tool<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">EC2<span class="def-item-dt-sep"></span></dt><dd>Elastic Compute Cloud<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">ESBL<span class="def-item-dt-sep"></span></dt><dd>Extended spectrum β-lactamase<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">HHV<span class="def-item-dt-sep"></span></dt><dd>human herpesvirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">HIV-1<span class="def-item-dt-sep"></span></dt><dd>human immunodeficiency 1 virus<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">HRV<span class="def-item-dt-sep"></span></dt><dd>Human Rhinovirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">IQR<span class="def-item-dt-sep"></span></dt><dd>Inter-quartile range<span class="def-item-dd-sep"></span></dd><dt id="def-item-10">L coding region<span class="def-item-dt-sep"></span></dt><dd>Large segment coding region (RNA-dependent RNA polymerase)<span class="def-item-dd-sep"></span></dd><dt id="def-item-11">LZW<span class="def-item-dt-sep"></span></dt><dd>Lempel-Ziv-Welch<span class="def-item-dd-sep"></span></dd><dt id="def-item-12">M coding region<span class="def-item-dt-sep"></span></dt><dd>Medium segment coding region (glycoprotein)<span class="def-item-dd-sep"></span></dd><dt id="def-item-13">MEGA<span class="def-item-dt-sep"></span></dt><dd>Molecular Evolutionary Genetics Analysis<span class="def-item-dd-sep"></span></dd><dt id="def-item-14">mNGS<span class="def-item-dt-sep"></span></dt><dd>metagenomic next-generation sequencing<span class="def-item-dd-sep"></span></dd><dt id="def-item-15">NCBI<span class="def-item-dt-sep"></span></dt><dd>National Center for Biotechnology Information<span class="def-item-dd-sep"></span></dd><dt id="def-item-16">NP swab<span class="def-item-dt-sep"></span></dt><dd>Nasopharyngeal swab<span class="def-item-dd-sep"></span></dd><dt id="def-item-17">nr database<span class="def-item-dt-sep"></span></dt><dd>non-redundant database<span class="def-item-dd-sep"></span></dd><dt id="def-item-18">nt database<span class="def-item-dt-sep"></span></dt><dd>nucleotide database<span class="def-item-dd-sep"></span></dd><dt id="def-item-19">PRICE<span class="def-item-dt-sep"></span></dt><dd>Paired-Read Iterative Contig Extension<span class="def-item-dd-sep"></span></dd><dt id="def-item-20">rpM<span class="def-item-dt-sep"></span></dt><dd>reads per million<span class="def-item-dd-sep"></span></dd><dt id="def-item-21">RSV<span class="def-item-dt-sep"></span></dt><dd>respiratory syncytial virus<span class="def-item-dd-sep"></span></dd><dt id="def-item-22">S coding region<span class="def-item-dt-sep"></span></dt><dd>Small segment coding region (nucleocapsid)<span class="def-item-dd-sep"></span></dd><dt id="def-item-23">SDI<span class="def-item-dt-sep"></span></dt><dd>Simpsons diversity Index<span class="def-item-dd-sep"></span></dd><dt id="def-item-24">SPAdes<span class="def-item-dt-sep"></span></dt><dd>St Petersburg genome assembler<span class="def-item-dd-sep"></span></dd><dt id="def-item-25">SRST2<span class="def-item-dt-sep"></span></dt><dd>Short Read Sequencing Typing<span class="def-item-dd-sep"></span></dd><dt id="def-item-26">STAR<span class="def-item-dt-sep"></span></dt><dd>Spliced Transcripts Alignment to a Reference<span class="def-item-dd-sep"></span></dd><dt id="def-item-27">TTV<span class="def-item-dt-sep"></span></dt><dd>torque teno virus<span class="def-item-dd-sep"></span></dd><dt id="def-item-28">VP1<span class="def-item-dt-sep"></span></dt><dd>Capsid protein VP1<span class="def-item-dd-sep"></span></dd><dt id="def-item-29">WHO<span class="def-item-dt-sep"></span></dt><dd>World Health Organization</dd></dl></div></div><div class="section" id="sec-29"><h2 class="">Ethics approval and consent to participate</h2><p id="p-80">The study protocol was approved by the Uganda National Council of Science and Technology and the Institutional Review Boards of the School of Medicine, Makerere University-College of Health Sciences and the University of California, San Francisco.</p></div><div class="section" id="sec-30"><h2 class="">Competing interests</h2><p id="p-81">None</p></div><div class="section" id="sec-31"><h2 class="">Funding</h2><p id="p-82">JLD is supported by the Chan Zuckerberg Biohub. CB, BD, YJ, JS, RE and JW are funded by the Chan Zuckerberg Initiative. SN was supported by Howard Hughes Medical Institute. PJR is funded by the National Institutes of Health, and work on this project by JH, MK, OB, and PJR was funded by the Doris Duke Charitable Foundation. MRW is supported by the Rachleff Foundation, NINDS K08NS096117 and the University of California, San Francisco Center for Next-Gen Precision Diagnostics, which is supported by the Sandler Foundation and the William K. Bowes, Jr. Foundation. AR is supported by the Rachleff Foundation. CL is supported by NHLBI K23HL138461-01A1, Nina Ireland Foundation and Marcus Program in Precision Medicine. KK is supported by the UC Berkeley UC San Francisco Joint Program in Bioengineering.</p></div><div class="section" id="sec-32"><h2 class="">Authors contributions</h2><p id="p-83">SN, JH, MK, OB, TR, AM, PR and JLD contributed to experimental design, data acquisition and sample processing. SN, JH, MK, OB, PR, TR, and AM, contributed to patient recruitment and clinical testing. CB, BD, YJ, JS, RE, and JW developed the IDseq pipeline. AR, KK, CL, SN, PR, MRW, and JLD contributed to data analysis. AR, MRW, PR, and JLD contributed to manuscript writing.</p></div><div class="section" id="sec-34"><h2 class="">Data files</h2><p id="p-90">Data file 1: (A) Clinical signs and symptoms of patients enrolled in the study. (B) mNGS findings in patients enrolled in the study</p><p id="p-91">Data file 2: Total number of sequencing reads and unique non-human reads in all samples analyzed</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-84">We thank the children and families in Tororo District, Uganda, who participated in this study. We thank Eric Chow, Jessica Lund, and the UCSF Center for Advanced Technology for assistance with sequencing. We recognize Amy Kistler. for intellectual discussions and comments on the paper. We thank Mark Stenglein for his helpful discussions and advice.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-2">e-mail address for authors: Akshaya Ramesh: <span class="em-link"><span class="em-addr">Akshaya.Ramesh{at}ucsf.edu</span></span>; Sara Nakielny: <span class="em-link"><span class="em-addr">ucsfsara{at}gmail.com</span></span>; Jennifer Hsu: <span class="em-link"><span class="em-addr">Jennifer.Hsu{at}kp.org</span></span>; Mary Kyohere: <span class="em-link"><span class="em-addr">pk_marx{at}yahoo.com</span></span>; Oswald Byaruhanga: <span class="em-link"><span class="em-addr">byaruos{at}gmail.com</span></span>; Charles de Bourcy: <span class="em-link"><span class="em-addr">cdebourcy{at}chanzuckerberg.com</span></span>; Rebecca Egger: <span class="em-link"><span class="em-addr">regger{at}chanzuckerberg.com</span></span>; Boris Dimitrov: <span class="em-link"><span class="em-addr">bdimitrov{at}chanzuckerberg.com</span></span>; Yun-Fang Juan: <span class="em-link"><span class="em-addr">yunfang{at}chanzuckerberg.com</span></span>; Jonathan Sheu: <span class="em-link"><span class="em-addr">jsheu{at}chanzuckerberg.com</span></span>; James Wang: <span class="em-link"><span class="em-addr">jwang{at}chanzuckerberg.com</span></span>; Katrina Kalantar: <span class="em-link"><span class="em-addr">kkalantar{at}berkeley.edu</span></span>; Charles Langelier: <span class="em-link"><span class="em-addr">Chaz.Langelier{at}ucsf.edu</span></span>; Theodore Ruel: <span class="em-link"><span class="em-addr">Theodore.Ruel{at}ucsf.edu</span></span>; Arthur Mpimbaza: <span class="em-link"><span class="em-addr">arthurwakg{at}yahoo.com</span></span>; Michael R. Wilson: <span class="em-link"><span class="em-addr">Michael.Wilson{at}ucsf.edu</span></span>; Philip J. Rosenthal: <span class="em-link"><span class="em-addr">Philip.Rosenthal{at}ucsf.edu</span></span>; Joseph L. DeRisi: <span class="em-link"><span class="em-addr">joe{at}derisilab.ucsf.edu</span></span></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.1" data-doi="10.1016/j.cmi.2018.02.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maze</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bassat</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feasey</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mandomando</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musicha</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">JA.</span></span> <span class="cit-article-title">The epidemiology of febrile illness in sub-Saharan Africa: implications for diagnosis and management</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Infect</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-vol">24</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">808</span>-<span class="cit-lpage">814</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cmi.2018.02.011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BMicrobiol%2BInfect%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmi.2018.02.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmi.2018.02.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.2" data-doi="10.4269/ajtmh.14-0646"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharples</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murdoch</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">JA.</span></span> <span class="cit-article-title">Community prevalence of fever and relationship with malaria among infants and children in low-resource areas</span>. <abbr class="cit-jnl-abbrev">Am J Trop Med Hyg</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">93</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">178</span>-<span class="cit-lpage">180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4269/ajtmh.14-0646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BTrop%2BMed%2BHyg%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.14-0646%26rft_id%253Dinfo%253Apmid%252F25918207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo4OiI5My8xLzE3OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA4LzA2LzM4NTAwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-385005v2.3"><div class="cit-metadata"><cite><span class="cit-chapter-title">Guidelines for the treatment of malaria</span>, <span class="cit-edition">2nd edition.</span> <span class="cit-publ-name">World Health Organization</span>. <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.4" data-doi="10.1016/j.tibtech.2013.01.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bibby</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Metagenomic identification of viral pathogens</span>. <abbr class="cit-jnl-abbrev">Trends Biotechnol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">31</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">275</span>-<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tibtech.2013.01.016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol%26rft.volume%253D31%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2013.01.016%26rft_id%253Dinfo%253Apmid%252F23415279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2013.01.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23415279&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.5" data-doi="10.1371/journal.pntd.0001485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yozwiak</span>  <span class="cit-name-given-names">NL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skewes-Cox</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenglein</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balmaseda</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">JL.</span></span> <span class="cit-article-title">Virus identification in unknown tropical febrile illness cases using deep sequencing</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">6</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">e1485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pntd.0001485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BNegl%2BTrop%2BDis%26rft.aulast%253DYozwiak%26rft.auinit1%253DN.%2BL.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1485%26rft.epage%253De1485%26rft.atitle%253DVirus%2Bidentification%2Bin%2Bunknown%2Btropical%2Bfebrile%2Billness%2Bcases%2Busing%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0001485%26rft_id%253Dinfo%253Apmid%252F22347512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0001485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22347512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.6" data-doi="10.1056/NEJMoa1401268"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Naccache</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samayoa</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Actionable Diagnosis of Neuroleptospirosis by Next-Generation Sequencing</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">370</span>(<span class="cit-issue">25</span>):<span class="cit-fpage">2408</span>-<span class="cit-lpage">2417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa1401268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D370%26rft.spage%253D2408%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1401268%26rft_id%253Dinfo%253Apmid%252F24896819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1401268&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24896819&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000337353000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.7" data-doi="10.1002/ana.24499"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shanbhag</span>  <span class="cit-name-given-names">NM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Diagnosing Balamuthia mandrillaris Encephalitis with Metagenomic Deep Sequencing</span>. <abbr class="cit-jnl-abbrev">Ann Neurol.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">78</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">722</span>-<span class="cit-lpage">730</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/ana.24499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BNeurol.%26rft.volume%253D78%26rft.spage%253D722%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fana.24499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ana.24499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.8" data-doi="10.1186/s13073-015-0257-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Illuminating uveitis: metagenomic deep sequencing identifies common and rare pathogens</span>. <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">8</span>:<span class="cit-fpage">1</span>-<span class="cit-lpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13073-016-0344-6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13073-015-0257-9%26rft_id%253Dinfo%253Apmid%252F26750923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13073-015-0257-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26750923&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.9" data-doi="10.3201/eid1606.091536"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quan</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briese</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Astrovirus encephalitis in boy with X-linked agammaglobulinemia</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">16</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">918</span>-<span class="cit-lpage">925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1606.091536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DQuan%26rft.auinit1%253DP.%2BL.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D918%26rft.epage%253D925%26rft.atitle%253DAstrovirus%2Bencephalitis%2Bin%2Bboy%2Bwith%2BX-linked%2Bagammaglobulinemia.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1606.091536%26rft_id%253Dinfo%253Apmid%252F20507741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1606.091536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20507741&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.10" data-doi="10.1056/NEJMoa073785"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palacios</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Druce</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A New Arenavirus in a Cluster of Fatal Transplant-Associated Diseases</span>. <abbr class="cit-jnl-abbrev">N Engl J Med.</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">358</span>:<span class="cit-fpage">991</span>-<span class="cit-lpage">998</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa073785.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa073785%26rft_id%253Dinfo%253Apmid%252F18256387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa073785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18256387&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253644700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.11" data-doi="10.1186/s13059-015-0866-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The real cost of sequencing: Scaling computation to keep pace with data generation</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">17</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>-<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13059-016-0917-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0866-z%26rft_id%253Dinfo%253Apmid%252F26753840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0866-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26753840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.12" data-doi="10.1016/S0168-8278(00)80173-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Naoumov</span>  <span class="cit-name-given-names">N V</span></span>. <span class="cit-article-title">TT virus - highly prevalent, but still in search of a disease</span>. <abbr class="cit-jnl-abbrev">J Hepatol</abbr>. <span class="cit-pub-date">2000</span>;<span class="cit-vol">33</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">157</span>-<span class="cit-lpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0168-8278(00)80173-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bhepatology%26rft.stitle%253DJ%2BHepatol%26rft.aulast%253DNaoumov%26rft.auinit1%253DN.%2BV.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D159%26rft.atitle%253DTT%2Bvirus--highly%2Bprevalent%252C%2Bbut%2Bstill%2Bin%2Bsearch%2Bof%2Ba%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-8278%252800%252980173-7%26rft_id%253Dinfo%253Apmid%252F10905600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-8278(00)80173-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10905600&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.13" data-doi="10.1186/1743-422X-4-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hafez</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaarawy</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hassan</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salim</span>  <span class="cit-name-given-names">RF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abd El Salam</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">AE.</span></span> <span class="cit-article-title">Prevalence of transfusion transmitted virus (TTV) genotypes among HCC patients in Qaluobia governorate</span>. <abbr class="cit-jnl-abbrev">Virol J.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">4</span>:<span class="cit-fpage">1</span>-<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1743-422X-4-135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DPatch%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D4%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DQuantitative%2Banalysis%2Bof%2BNipah%2Bvirus%2Bproteins%2Breleased%2Bas%2Bvirus-like%2Bparticles%2Breveals%2Bcentral%2Brole%2Bfor%2Bthe%2Bmatrix%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-4-1%26rft_id%253Dinfo%253Apmid%252F17204159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-4-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17204159&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.14" data-doi="10.1126/scitranslmed.3003783"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abreu</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagalingam</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Sinus Microbiome Diversity Depletion and Corynebacterium tuberculostearicum Enrichment Mediates Rhinosinusitis</span>. <abbr class="cit-jnl-abbrev">Sci Transl Med.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">4</span>(<span class="cit-issue">151</span>):<span class="cit-fpage">151ra124</span>-<span class="cit-lpage">151ra124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/scitranslmed.3003783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DAbreu%26rft.auinit1%253DN.%2BA.%26rft.volume%253D4%26rft.issue%253D151%26rft.spage%253D151ra124%26rft.epage%253D151ra124%26rft.atitle%253DSinus%2BMicrobiome%2BDiversity%2BDepletion%2Band%2BCorynebacterium%2Btuberculostearicum%2BEnrichment%2BMediates%2BRhinosinusitis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.3003783%26rft_id%253Dinfo%253Apmid%252F22972842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjE0OiI0LzE1MS8xNTFyYTEyNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA4LzA2LzM4NTAwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.15" data-doi="10.1164/rccm.201706-1097LE"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langelier</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zinter</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalantar</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Metagenomic Next-Generation Sequencing Detects Pulmonary Pathogens in Hematopoietic Cellular Transplant Patients with Acute Respiratory Illnesses</span>. <abbr class="cit-jnl-abbrev">Am J Respir Crit Care Med</abbr>. <span class="cit-pub-date">2018</span> Feb 15;<span class="cit-vol">197</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">524</span>-<span class="cit-lpage">528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1164/rccm.201706-1097LE</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BRespir%2BCrit%2BCare%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1164%252Frccm.201706-1097LE%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1164/rccm.201706-1097LE&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.16" data-doi="10.1093/cid/cix104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baggett</span>  <span class="cit-name-given-names">HC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">SRC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Colonization density of the upper respiratory tract as a predictor of pneumonia - Haemophilus influenzae, Moraxella catarrhalis, Staphylococcus aureus, and Pneumocystis jirovecii</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis.</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">64</span>:<span class="cit-fpage">S328</span>-<span class="cit-lpage">S336</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/cid/cix104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis.%26rft.volume%253D64%26rft.spage%253DS328%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fcix104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/cix104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.17" data-doi="10.1128/JCM.02037-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loens</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Heirstraeten</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malhotra-Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goossens</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ieven</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Optimal sampling sites and methods for detection of pathogen possibly causing community-acquired lower respiratory tract infections</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">47</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">21</span>-<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JCM.02037-08</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.02037-08%26rft_id%253Dinfo%253Apmid%252F19020070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjc6IjQ3LzEvMjEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wOC8wNi8zODUwMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.18" data-doi="10.1016/S2213-2600(16)00096-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zar</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barnett</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner-Lubbe</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myer</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">MP.</span></span> <span class="cit-article-title">Aetiology of childhood pneumonia in a well vaccinated South African birth cohort: A nested case-control study of the Drakenstein Child Health Study</span>. <abbr class="cit-jnl-abbrev">Lancet Respir Med</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">4</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">463</span>-<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S2213-2600(16)00096-5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BRespir%2BMed%26rft.volume%253D4%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS2213-2600%252816%252900096-5%26rft_id%253Dinfo%253Apmid%252F27117547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S2213-2600(16)00096-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27117547&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.19" data-doi="10.1099/jgv.0.000966"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lukashev</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vakulenko</span>  <span class="cit-name-given-names">YA.</span></span> <span class="cit-article-title">Molecular evolution of types in non-polio enteroviruses</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">98</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">2968</span>-<span class="cit-lpage">2981</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/jgv.0.000966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D98%26rft.spage%253D2968%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jgv.0.000966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.20" data-doi="10.1099/vir.0.053686-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Proposals for the classification of human rhinovirus species A, B and C into genotypically assigned types</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">94</span>(PART8):<span class="cit-fpage">1791</span>-<span class="cit-lpage">1806</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.053686-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.053686-0%26rft_id%253Dinfo%253Apmid%252F23677786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.053686-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23677786&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322850200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.21" data-doi="10.1186/s13073-014-0100-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Inouye</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dashnow</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raven</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">SRST2: Rapid genomic surveillance for public health and hospital microbiology labs</span>. <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">6</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">1</span>-<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13073-014-0090-6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13073-014-0100-8%26rft_id%253Dinfo%253Apmid%252F24433494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13073-014-0100-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24433494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.22" data-doi="10.1371/journal.pntd.0003335"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chipwaza</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mugasa</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selemani</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Dengue and Chikungunya Fever among Viral Diseases in Outpatient Febrile Children in Kilosa District Hospital, Tanzania</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">8</span>(<span class="cit-issue">11</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pntd.0003335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl%2BTrop%2BDis.%26rft.volume%253D8%26rft.spage%253De3335%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0003335%26rft_id%253Dinfo%253Apmid%252F25412076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0003335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25412076&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.23" data-doi="10.1371/journal.pone.0070305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">ST</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavlinac</span>  <span class="cit-name-given-names">PB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakiyingi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Mycobacterium tuberculosis Bacteremia in a Cohort of HIV-Infected Patients Hospitalized with Severe Sepsis in Uganda-High Frequency, Low Clinical Sand Derivation of a Clinical Prediction Score</span>. <abbr class="cit-jnl-abbrev">PLoS One.</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">8</span>(<span class="cit-issue">8</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0070305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne.%26rft.volume%253D8%26rft.spage%253De70305%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0070305%26rft_id%253Dinfo%253Apmid%252F23940557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0070305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23940557&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.24" data-doi="10.1371/journal.pntd.0002324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morrissey</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholson</span>  <span class="cit-name-given-names">WL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Etiology of Severe Non-malaria Febrile Illness in Northern Tanzania: A Prospective Cohort Study</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis.</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">7</span>(<span class="cit-issue">7</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pntd.0002324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl%2BTrop%2BDis.%26rft.volume%253D7%26rft.spage%253De2324%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0002324%26rft_id%253Dinfo%253Apmid%252F23875053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0002324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23875053&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.25" data-doi="10.1371/journal.pntd.0003750"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chipwaza</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mhamphi</span>  <span class="cit-name-given-names">GG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ngatunga</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Prevalence of Bacterial Febrile Illnesses in Children in Kilosa District, Tanzania</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">9</span>(<span class="cit-issue">5</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pntd.0003750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl%2BTrop%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0003750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0003750&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.26" data-doi="10.1056/NEJMoa1214482"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">D’Acremont</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kilowoko</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kyungu</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Beyond Malaria — Causes of Fever in Outpatient Tanzanian Children</span>. <abbr class="cit-jnl-abbrev">N Engl J Med.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">370</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">809</span>-<span class="cit-lpage">817</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa1214482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed.%26rft.volume%253D370%26rft.spage%253D809%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1214482%26rft_id%253Dinfo%253Apmid%252F24571753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1214482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571753&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332013000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.27" data-doi="10.4269/ajtmh.14-0560"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Meara</span>  <span class="cit-name-given-names">WP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laktabai</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Etiology of pediatric fever in Western Kenya: A case-control study of falciparum Malaria, Respiratory Viruses, and Streptococcal Pharyngitis</span>. <abbr class="cit-jnl-abbrev">Am J Trop Med Hyg.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">92</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1030</span>-<span class="cit-lpage">1037</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4269/ajtmh.14-0560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BTrop%2BMed%2BHyg.%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.14-0560%26rft_id%253Dinfo%253Apmid%252F25758648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo5OiI5Mi81LzEwMzAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wOC8wNi8zODUwMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.28" data-doi="10.3201/eid2205.150953"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maina</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farris</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Odhiambo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Q fever, scrub typhus, and rickettsial diseases in children, Kenya, 2011–2012</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis.</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">22</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">883</span>-<span class="cit-lpage">886</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid2205.150953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis.%26rft.volume%253D22%26rft.spage%253D883%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2205.150953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2205.150953&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.29" data-doi="10.4269/ajtmh.14-0494"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kibuuka</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byakika-Kibwika</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Achan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Bacteremia among febrile ugandan children treated with antimalarials despite a negative malaria test</span>. <abbr class="cit-jnl-abbrev">Am J Trop Med Hyg.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">93</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">276</span>-<span class="cit-lpage">280</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4269/ajtmh.14-0494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BTrop%2BMed%2BHyg.%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.14-0494%26rft_id%253Dinfo%253Apmid%252F26055736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo4OiI5My8yLzI3NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA4LzA2LzM4NTAwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.30" data-doi="10.1371/journal.pone.0131080"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murdoch</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reyburn</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">JA.</span></span> <span class="cit-article-title">Etiology of severe febrile illness in low- and middle-income countries: A systematic review</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">10</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1</span>-<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0127962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131080%26rft_id%253Dinfo%253Apmid%252F25919488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25919488&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.31" data-doi="10.1001/jamaneurol.2018.0463"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Donovan</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gelfand</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Chronic Meningitis Investigated via Metagenomic Next-Generation Sequencing</span>. <abbr class="cit-jnl-abbrev">JAMA Neurol.</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">94158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1001/jamaneurol.2018.0463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%2BNeurol.%26rft.volume%253D75%26rft.spage%253D947%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjamaneurol.2018.0463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jamaneurol.2018.0463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.32" data-doi="10.1101/gr.171934.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Naccache</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Federman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Veeraraghavan</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">24</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1180</span>-<span class="cit-lpage">1192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.171934.113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171934.113%26rft_id%253Dinfo%253Apmid%252F24899342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE4MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA4LzA2LzM4NTAwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.33" data-doi="10.1186/s12936-016-1650-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oguttu</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matovu</span>  <span class="cit-name-given-names">JKB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okumu</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid reduction of malaria following introduction of vector control interventions in Tororo District, Uganda: a descriptive study</span>. <abbr class="cit-jnl-abbrev">Malar J.</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">16</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>-<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12936-017-1871-3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalar%2BJ.%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12936-016-1650-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12936-016-1650-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.34" data-doi="10.1186/1475-2875-13-48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mekonnen</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aseffa</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Medhin</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berhe</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Velavan</span>  <span class="cit-name-given-names">TP.</span></span> <span class="cit-article-title">Re-evaluation of microscopy confirmed Plasmodium falciparum and Plasmodium vivax malaria by nested PCR detection in southern Ethiopia</span>. <abbr class="cit-jnl-abbrev">Malar J.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">13</span>(<span class="cit-issue">48</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1475-2875-13-48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalar%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1475-2875-13-48%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1475-2875-13-48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.35" data-doi="10.4103/2229-682-5070.202299"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baid</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Datta</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Falciparum malaria and parvovirus B19 coinfection: A rare entity</span>. <abbr class="cit-jnl-abbrev">Trop Parasitol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">7</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">47</span>-<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4103/2229-682-5070.202299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrop%2BParasitol%26rft.volume%253D7%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.4103%252F2229-682-5070.202299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4103/2229-682-5070.202299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.36" data-doi="10.1016/S2221-1691(13)60037-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duedu</span>  <span class="cit-name-given-names">KO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sagoe</span>  <span class="cit-name-given-names">KWC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ayeh-Kumi</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Affrim</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adiku</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">The effects of co-infection with human parvovirus B19 and Plasmodium falciparum on type and degree of anaemia in Ghanaian children</span>. <abbr class="cit-jnl-abbrev">Asian Pac J Trop Biomed</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">3</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">129</span>-<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S2221-1691(13)60037-4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsian%2BPac%2BJ%2BTrop%2BBiomed%26rft.volume%253D3%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS2221-1691%252813%252960037-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S2221-1691(13)60037-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.37" data-doi="10.1186/1471-2334-13-375"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Toan</span>  <span class="cit-name-given-names">NL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sy</span>  <span class="cit-name-given-names">BT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">LH</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Co-infection of human parvovirus B19 with Plasmodium falciparum contributes to malaria disease severity in Gabonese patients</span>. <abbr class="cit-jnl-abbrev">BMC Infect Dis.</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">13</span>(<span class="cit-issue">1</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2334-13-375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BInfect%2BDis.%26rft.volume%253D13%26rft.spage%253D375%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2334-13-375%26rft_id%253Dinfo%253Apmid%252F23945350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2334-13-375&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23945350&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.38" data-doi="10.1002/jmv.23251"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Onyango</span>  <span class="cit-name-given-names">CO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munywoki</span>  <span class="cit-name-given-names">PK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular epidemiology of human rhinovirus infections in Kilifi, coastal Kenya</span>. <abbr class="cit-jnl-abbrev">J Med Virol.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">84</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">823</span>-<span class="cit-lpage">831</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.23251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedical%2Bvirology%26rft.stitle%253DJ%2BMed%2BVirol%26rft.aulast%253DOnyango%26rft.auinit1%253DC.%2BO.%26rft.volume%253D84%26rft.issue%253D5%26rft.spage%253D823%26rft.epage%253D831%26rft.atitle%253DMolecular%2Bepidemiology%2Bof%2Bhuman%2Brhinovirus%2Binfections%2Bin%2BKilifi%252C%2Bcoastal%2BKenya.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.23251%26rft_id%253Dinfo%253Apmid%252F22431032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.23251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22431032&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.39" data-doi="10.1097/INF.0b013e3181f232fe"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Callaghan-Gordo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bassat</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morais</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Etiology and epidemiology of viral pneumonia among hospitalized children in rural mozambique: A malaria endemic area with high prevalence of human immunodeficiency virus</span>. <abbr class="cit-jnl-abbrev">Pediatr Infect Dis J.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">30</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">39</span>-<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1097/INF.0b013e3181f232fe</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPediatric%2Binfectious%2Bdisease%2Bjournal%26rft.stitle%253DPediatr%2BInfect%2BDis%2BJ%26rft.aulast%253DO%25CA%25BCcallaghan-Gordo%26rft.auinit1%253DC.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D44%26rft.atitle%253DEtiology%2Band%2Bepidemiology%2Bof%2Bviral%2Bpneumonia%2Bamong%2Bhospitalized%2Bchildren%2Bin%2Brural%2BMozambique%253A%2Ba%2Bmalaria%2Bendemic%2Barea%2Bwith%2Bhigh%2Bprevalence%2Bof%2Bhuman%2Bimmunodeficiency%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1097%252FINF.0b013e3181f232fe%26rft_id%253Dinfo%253Apmid%252F20805786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/INF.0b013e3181f232fe&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20805786&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.40" data-doi="10.1002/jmv.21665"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mbayame</span>  <span class="cit-name-given-names">Ndiaye Niang</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ousmane</span>  <span class="cit-name-given-names">M. Diop</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fatoumata</span>  <span class="cit-name-given-names">Diene Sarr</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deborah</span>  <span class="cit-name-given-names">Goudiaby</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">Malou-Sompy</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kader</span>  <span class="cit-name-given-names">Ndiaye</span></span>, <span class="cit-auth"><span class="cit-name-surname">Astrid Vabret</span>  <span class="cit-name-given-names">LB.</span></span> <span class="cit-article-title">Viral Etiology of Respiratory Infections in Children Under 5 Years Old Living in Tropical Rural Areas of Senegal: The EVIRA Project</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">82</span>:<span class="cit-fpage">866</span>-<span class="cit-lpage">872</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.21665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedical%2Bvirology%26rft.stitle%253DJ%2BMed%2BVirol%26rft.aulast%253DNiang%26rft.auinit1%253DM.%2BN.%26rft.volume%253D82%26rft.issue%253D5%26rft.spage%253D866%26rft.epage%253D872%26rft.atitle%253DViral%2Betiology%2Bof%2Brespiratory%2Binfections%2Bin%2Bchildren%2Bunder%2B5%2Byears%2Bold%2Bliving%2Bin%2Btropical%2Brural%2Bareas%2Bof%2BSenegal%253A%2BThe%2BEVIRA%2Bproject.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.21665%26rft_id%253Dinfo%253Apmid%252F20336732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.21665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336732&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.41" data-doi="10.1186/1471-2334-11-65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smuts</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zar</span>  <span class="cit-name-given-names">HJ.</span></span> <span class="cit-article-title">Human rhinovirus infection in young African children with acute wheezing</span>. <abbr class="cit-jnl-abbrev">BMC Infect Dis</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">11</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2334-11-65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Binfectious%2Bdiseases%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BInfect%2BDis%26rft.aulast%253DSmuts%26rft.auinit1%253DH.%2BE.%26rft.volume%253D11%26rft.spage%253D65%26rft.epage%253D65%26rft.atitle%253DHuman%2Brhinovirus%2Binfection%2Bin%2Byoung%2BAfrican%2Bchildren%2Bwith%2Bacute%2Bwheezing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2334-11-65%26rft_id%253Dinfo%253Apmid%252F21401965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2334-11-65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21401965&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.42" data-doi="10.1056/NEJMoa1500245"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Self</span>  <span class="cit-name-given-names">WH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wunderink</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Community-Acquired Pneumonia Requiring Hospitalization among U.S. Adults</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">373</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">415</span>-<span class="cit-lpage">427</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa1500245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D373%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1500245%26rft_id%253Dinfo%253Apmid%252F26172429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1500245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26172429&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.43" data-doi="10.3201/eid2402.170778"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scully</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Basnet</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrangham</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Lethal Respiratory Disease Associated with Human Rhinovirus C in Wild Chimpanzees, Uganda, 2013</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis.</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">24</span>(<span class="cit-issue">2</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid2402.170778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2402.170778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2402.170778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-web" id="cit-385005v2.44"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Diarrhoeal Disease</span>. <span class="cit-source">World Health Organization</span>. <span class="cit-pub-date">2017</span>. <a href="http://www.who.int/news-room/fact-sheets/detail/diarrhoeal-disease">http://www.who.int/news-room/fact-sheets/detail/diarrhoeal-disease</a>. <span class="cit-date-in-citation">Accessed 20 June 2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.45" data-doi="10.15585/mmwr.mm6643a7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mwenda</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaba</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Implementation of Rotavirus Surveillance and Vaccine Introduction - World Health Organization African Region, 2007-2016</span>. <abbr class="cit-jnl-abbrev">MMWR Morb Mortal Wkly Rep.</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">66</span>(<span class="cit-issue">43</span>):<span class="cit-fpage">1192</span>-<span class="cit-lpage">1196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.15585/mmwr.mm6643a7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMMWR%2BMorb%2BMortal%2BWkly%2BRep.%26rft.volume%253D66%26rft.spage%253D1192%26rft_id%253Dinfo%253Adoi%252F10.15585%252Fmmwr.mm6643a7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.15585/mmwr.mm6643a7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.46" data-doi="10.4269/ajtmh.1987.36.114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calisher</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monath</span>  <span class="cit-name-given-names">TP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabattini</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A newly recognized vesiculovirus, Calchaqui virus, and subtypes of Melao and Maguari viruses from Argentina, with serologic evidence for infections of humans and horses</span>. <abbr class="cit-jnl-abbrev">Am J Trop Med Hyg.</abbr> <span class="cit-pub-date">1987</span>;<span class="cit-vol">36</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">114</span>-<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BTropical%2BMedicine%2Band%2BHygiene%26rft.stitle%253DAm%2BJ%2BTrop%2BMed%2BHyg%26rft.aulast%253DCalisher%26rft.auinit1%253DC.%2BH.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D114%26rft.epage%253D119%26rft.atitle%253DA%2BNewly%2BRecognized%2BVesiculovirus%252C%2BCalchaqui%2BVirus%252C%2Band%2BSubtypes%2Bof%2BMelao%2Band%2BMaguari%2BViruses%2Bfrom%2BArgentina%252C%2Bwith%2BSerologic%2BEvidence%2Bfor%2BInfections%2Bof%2BHumans%2Band%2BHorses%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.1987.36.114%26rft_id%253Dinfo%253Apmid%252F2880522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo4OiIzNi8xLzExNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA4LzA2LzM4NTAwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">47.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.47" data-doi="10.1128/JVI.02080-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mohamed</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLees</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">RM.</span></span> <span class="cit-article-title">Viruses in the Anopheles A, Anopheles B, and Tete Serogroups in the Orthobunyavirus Genus (Family Bunyaviridae) Do Not Encode an NSs Protein</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">7612</span>-<span class="cit-lpage">7618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02080-08</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02080-08%26rft_id%253Dinfo%253Apmid%252F19439468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4My8xNS83NjEyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDgvMDYvMzg1MDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imlarp</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Top</span>  <span class="cit-name-given-names">FH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavanaugh</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">PK.</span></span> <span class="cit-article-title">Kaeng Khoi virus from naturally infected bedbugs (Cimicidae) and immature free tailed bats</span>. <abbr class="cit-jnl-abbrev">Bull World Health Organ</abbr>. <span class="cit-pub-date">1976</span>;<span class="cit-vol">53</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">365</span>-<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BWorld%2BHealth%2BOrganization%26rft.stitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.aulast%253DWilliams%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D53%26rft.issue%253D4%26rft.spage%253D365%26rft.epage%253D369%26rft.atitle%253DKaeng%2BKhoi%2Bvirus%2Bfrom%2Bnaturally%2Binfected%2Bbedbugs%2B%2528cimicidae%2529%2Band%2Bimmature%2Bfree-tailed%2Bbats.%26rft_id%253Dinfo%253Apmid%252F1086729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1086729&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.49" data-doi="10.4269/ajtmh.17-0810"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patroca da Silva</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marciel de Souza</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of the Gamboa Virus Serogroup (Orthobunyavirus Genus, Peribunyaviridae Family)</span>. <abbr class="cit-jnl-abbrev">Am J Trop Med Hyg.</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">98</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">tpmd170810</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4269/ajtmh.17-0810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BTrop%2BMed%2BHyg.%26rft.volume%253D98%26rft.spage%253Dtpmd170810%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.17-0810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4269/ajtmh.17-0810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plyusnin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">RM.</span></span> <abbr class="cit-jnl-abbrev">Bunyaviridae: Molecular and Cellular Biology</abbr>.; <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lutwama</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rwaguma</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nawanga</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mukuye</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Isolations of Bwamba virus from south central Uganda and north eastern Tanzania</span>. <abbr class="cit-jnl-abbrev">Afr Health Sci</abbr>. <span class="cit-pub-date">2002</span>;<span class="cit-vol">2</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">24</span>-<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAfrican%2Bhealth%2Bsciences%26rft.stitle%253DAfr%2BHealth%2BSci%26rft.aulast%253DLutwama%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D28%26rft.atitle%253DIsolations%2Bof%2BBwamba%2Bvirus%2Bfrom%2Bsouth%2Bcentral%2BUganda%2Band%2Bnorth%2Beastern%2BTanzania.%26rft_id%253Dinfo%253Apmid%252F12789111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12789111&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.52"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO Model List of Essential Medicines</span>. Edition 14, <span class="cit-pub-date">2017</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.53"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">UGANDA CLINICAL GUIDELINES</span>. <abbr class="cit-jnl-abbrev">Ministry of Health</abbr>. <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.54" data-doi="10.1128/JVI.00451-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stenglein</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanchez-Migallon Guzman</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">VE</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential Disease Susceptibilities in Experimentally Reptarenavirus-Infected Boa Constrictors and Ball Pythons</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">91</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">e00451</span>-<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00451-17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00451-17%26rft_id%253Dinfo%253Apmid%252F28515291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE1OiI5MS8xNS9lMDA0NTEtMTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wOC8wNi8zODUwMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.55" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">STAR: Ultrafast universal RNA-seq aligner</span>. <abbr class="cit-jnl-abbrev">Bioinformatics.</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">29</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">15</span>-<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bts635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.56" data-doi="10.1534/g3.113.005967"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruby</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellare</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Derisi</span>  <span class="cit-name-given-names">JL.</span></span> <span class="cit-article-title">PRICE: software for the targeted assembly of components of (Meta) genomic sequence data</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">3</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">865</span>-<span class="cit-lpage">880</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/g3.113.005967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.005967%26rft_id%253Dinfo%253Apmid%252F23550143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvNS84NjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wOC8wNi8zODUwMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.57" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Cd-hit: A fast program for clustering and comparing large sets of protein or nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">22</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">1658</span>-<span class="cit-lpage">1659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btl158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16731699&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238905700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.58" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL.</span></span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">9</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">357</span>-<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.1923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.59" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">26</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">873</span>-<span class="cit-lpage">881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btq057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20147302&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.60" data-doi="10.1186/1471-2105-12-159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">RAPSearch: a fast protein similarity search tool for short reads</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">12</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2105-12-159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DYe%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D159%26rft.epage%253D159%26rft.atitle%253DRAPSearch%253A%2Ba%2Bfast%2Bprotein%2Bsimilarity%2Bsearch%2Btool%2Bfor%2Bshort%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-159%26rft_id%253Dinfo%253Apmid%252F21575167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21575167&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.61" data-doi="10.1128/mSystems.00058-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fedewa</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenglein</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiplexed Metagenomic Deep Sequencing To Analyze the Composition of High-Priority Pathogen Reagents</span>. <abbr class="cit-jnl-abbrev">mSystems.</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">1</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">e00058</span>-<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/mSystems.00058-16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmSystems.%26rft.volume%253D1%26rft.spage%253De00058%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmSystems.00058-16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mSystems.00058-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.62" data-doi="10.1371/journal.pone.0017035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bogaert</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keijser</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huse</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Variability and diversity of nasopharyngeal microbiota in children: A metagenomic analysis</span>. <abbr class="cit-jnl-abbrev">PLoS One.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">6</span>(<span class="cit-issue">2</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0017035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBogaert%26rft.auinit1%253DD.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De17035%26rft.epage%253De17035%26rft.atitle%253DVariability%2Band%2Bdiversity%2Bof%2Bnasopharyngeal%2Bmicrobiota%2Bin%2Bchildren%253A%2Ba%2Bmetagenomic%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017035%26rft_id%253Dinfo%253Apmid%252F21386965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21386965&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.63" data-doi="10.1371/journal.pone.0179159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pérez-Losada</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alamri</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freishtat</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-article-title">Nasopharyngeal microbiome diversity changes over time in children with asthma</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">12</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>-<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0170543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0179159%26rft_id%253Dinfo%253Apmid%252F28957441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0179159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28957441&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">64.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-385005v2.64" data-doi="10.1126/science.1058709"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hooper</span>  <span class="cit-name-given-names">L V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">JI.</span></span> <span class="cit-article-title">Commensal Host-Bacterial Relationships in the Gut</span>. <abbr class="cit-jnl-abbrev">Science (80-)</abbr>. <span class="cit-pub-date">2001</span>;<span class="cit-vol">1115</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">1</span>-<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1058709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-%2529%26rft.volume%253D1115%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1058709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.1058709&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.65" data-doi="10.1152/physrev.00045.2009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sekirov</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antunes</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Gut microbiota in health and disease</span>. <abbr class="cit-jnl-abbrev">Physiol Rev</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">90</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">859</span>-<span class="cit-lpage">904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1152/physrev.00045.2009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiological%2BReviews%26rft.stitle%253DPhysiol.%2BRev.%26rft.aulast%253DSekirov%26rft.auinit1%253DI.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D859%26rft.epage%253D904%26rft.atitle%253DGut%2BMicrobiota%2Bin%2BHealth%2Band%2BDisease%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysrev.00045.2009%26rft_id%253Dinfo%253Apmid%252F20664075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physrev.00045.2009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20664075&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.66" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing</span>. <abbr class="cit-jnl-abbrev">J Comput Biol.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">19</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">455</span>-<span class="cit-lpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1089/cmb.2012.0021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.67" data-doi="10.1093/nar/gku1243"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">HY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The InterPro protein families database: The classification resource after 15 years</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">43</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D213</span>-<span class="cit-lpage">D221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gku1243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1243%26rft_id%253Dinfo%253Apmid%252F25428371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25428371&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.68" data-doi="10.1093/nar/gkv485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tsirigos</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Käll</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elofsson</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">The TOPCONS web server for consensus prediction of membrane protein topology and signal peptides</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">43</span>(<span class="cit-issue">W1</span>):<span class="cit-fpage">W401</span>-<span class="cit-lpage">W407</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkv485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv485%26rft_id%253Dinfo%253Apmid%252F25969446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25969446&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-385005v2.69" data-doi="10.1016/j.drup.2010.08.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giske</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornaglia</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Supranational surveillance of antimicrobial resistance: The legacy of the last decade and proposals for the future</span>. <abbr class="cit-jnl-abbrev">Drug Resist Updat</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">13</span>(4-5):<span class="cit-fpage">93</span>-<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.drup.2010.08.002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrug%2Bresistance%2Bupdates%2B%253A%2B%2Breviews%2Band%2Bcommentaries%2Bin%2Bantimicrobial%2Band%2Banticancer%2Bchemotherapy%26rft.stitle%253DDrug%2BResist%2BUpdat%26rft.aulast%253DGiske%26rft.auinit1%253DC.%2BG.%26rft.volume%253D13%26rft.issue%253D4-5%26rft.spage%253D93%26rft.epage%253D98%26rft.atitle%253DSupranational%2Bsurveillance%2Bof%2Bantimicrobial%2Bresistance%253A%2BThe%2Blegacy%2Bof%2Bthe%2Blast%2Bdecade%2Band%2Bproposals%2Bfor%2Bthe%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.drup.2010.08.002%26rft_id%253Dinfo%253Apmid%252F20864384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.drup.2010.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20864384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F08%2F06%2F385005.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/385005v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F385005v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/385047v1" title="A combination of transcription factors mediates inducible interchromosomal pairing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;August 06, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/385005v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/385005v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/385005v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/385005v2" class="active" data-icon-position="" data-hide-link-title="0">Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/117052" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-i-tse59pigUxuNipQVxHO-LA467mdS4zsEwWoZIKWl4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="117052" id="node117052--2" data-pisa="biorxiv;385005v2" data-pisa-master="biorxiv;385005" data-seqnum="117052" data-apath="/biorxiv/early/2018/08/06/385005.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Akshaya</span> <span class="nlm-surname">Ramesh</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sara</span> <span class="nlm-surname">Nakielny</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Hsu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Kyohere</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Oswald</span> <span class="nlm-surname">Byaruhanga</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Charles de</span> <span class="nlm-surname">Bourcy</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Egger</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Dimitrov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yun-Fang</span> <span class="nlm-surname">Juan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Sheu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">James</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Katrina</span> <span class="nlm-surname">Kalantar</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Charles</span> <span class="nlm-surname">Langelier</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Theodore</span> <span class="nlm-surname">Ruel</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Arthur</span> <span class="nlm-surname">Mpimbaza</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Philip J.</span> <span class="nlm-surname">Rosenthal</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Joseph L.</span> <span class="nlm-surname">DeRisi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385005; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/385005 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/385005v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;title=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;title=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;text=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;title=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;t=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;title=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;title=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="117052" id="node117052--3" data-pisa="biorxiv;385005v2" data-pisa-master="biorxiv;385005" data-seqnum="117052" data-apath="/biorxiv/early/2018/08/06/385005.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Akshaya</span> <span class="nlm-surname">Ramesh</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sara</span> <span class="nlm-surname">Nakielny</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Hsu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Kyohere</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Oswald</span> <span class="nlm-surname">Byaruhanga</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Charles de</span> <span class="nlm-surname">Bourcy</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Egger</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Dimitrov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yun-Fang</span> <span class="nlm-surname">Juan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Sheu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">James</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Katrina</span> <span class="nlm-surname">Kalantar</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Charles</span> <span class="nlm-surname">Langelier</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Theodore</span> <span class="nlm-surname">Ruel</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Arthur</span> <span class="nlm-surname">Mpimbaza</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Philip J.</span> <span class="nlm-surname">Rosenthal</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Joseph L.</span> <span class="nlm-surname">DeRisi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385005; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/385005 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Ramesh%2C+Akshaya&amp;rft.au=Nakielny%2C+Sara&amp;rft.au=Hsu%2C+Jennifer&amp;rft.au=Kyohere%2C+Mary&amp;rft.au=Byaruhanga%2C+Oswald&amp;rft.au=Bourcy%2C+Charles+de&amp;rft.au=Egger%2C+Rebecca&amp;rft.au=Dimitrov%2C+Boris&amp;rft.au=Juan%2C+Yun-Fang&amp;rft.au=Sheu%2C+Jonathan&amp;rft.au=Wang%2C+James&amp;rft.au=Kalantar%2C+Katrina&amp;rft.au=Langelier%2C+Charles&amp;rft.au=Ruel%2C+Theodore&amp;rft.au=Mpimbaza%2C+Arthur&amp;rft.au=Wilson%2C+Michael+R.&amp;rft.au=Rosenthal%2C+Philip+J.&amp;rft.au=DeRisi%2C+Joseph+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/117052/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/117052/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/117052/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/117052/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/117052/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/117052/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/117052/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/117052/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/117052/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/117052/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/117052/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/117052/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;count=horizontal&amp;via=&amp;text=Etiology%20of%20fever%20in%20Ugandan%20children%3A%20identification%20of%20microbial%20pathogens%20using%20metagenomic%20next-generation%20sequencing%20and%20IDseq%2C%20a%20platform%20for%20unbiased%20metagenomic%20analysis&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/385005v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/385005v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/385005v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
