<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/16/790444.atom" />
<meta name="HW.pisa" content="biorxiv;790444v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein" />
<meta name="DC.Identifier" content="10.1101/790444" />
<meta name="DC.Date" content="2020-01-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Viruses maximize their genetic coding capacity through a variety of biochemical mechanisms including programmed ribosomal frameshifting (PRF), which facilitates the production of multiple proteins from a single transcript. PRF is typically stimulated by structural elements within the mRNA that generate mechanical tension between the transcript and ribosome. However, in this work we show that the forces generated by the cotranslational folding of the nascent polypeptide chain can also enhance PRF. Using an array of biochemical, cellular, and computational techniques, we first demonstrate that the Sindbis virus structural polyprotein forms two competing topological isomers during biosynthesis at the ribosome-translocon complex. We then show that the formation of one of these topological isomers is linked to PRF. Coarse-grained molecular dynamic simulations reveal that the translocon-mediated membrane integration of a transmembrane domain upstream from the ribosomal slip-site generates a force on the nascent polypeptide chain that scales with observed frameshifting. Together, our results demonstrate that cotranslational folding of this protein generates a tension that stimulates PRF. To our knowledge, this constitutes the first example in which the conformational state of the nascent chain has been linked to PRF. These findings raise the possibility that, in addition to RNA-mediated translational recoding, a variety of cotranslational folding and/ or binding events may also stimulate PRF." />
<meta name="DC.Contributor" content="Haley R. Harrington" />
<meta name="DC.Contributor" content="Matthew H. Zimmer" />
<meta name="DC.Contributor" content="Laura M. Chamness" />
<meta name="DC.Contributor" content="Veronica Nash" />
<meta name="DC.Contributor" content="Wesley D. Penn" />
<meta name="DC.Contributor" content="Thomas F. Miller" />
<meta name="DC.Contributor" content="Suchetana Mukhopadhyay" />
<meta name="DC.Contributor" content="Jonathan P. Schlebach" />
<meta name="article:published_time" content="2020-01-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Viruses maximize their genetic coding capacity through a variety of biochemical mechanisms including programmed ribosomal frameshifting (PRF), which facilitates the production of multiple proteins from a single transcript. PRF is typically stimulated by structural elements within the mRNA that generate mechanical tension between the transcript and ribosome. However, in this work we show that the forces generated by the cotranslational folding of the nascent polypeptide chain can also enhance PRF. Using an array of biochemical, cellular, and computational techniques, we first demonstrate that the Sindbis virus structural polyprotein forms two competing topological isomers during biosynthesis at the ribosome-translocon complex. We then show that the formation of one of these topological isomers is linked to PRF. Coarse-grained molecular dynamic simulations reveal that the translocon-mediated membrane integration of a transmembrane domain upstream from the ribosomal slip-site generates a force on the nascent polypeptide chain that scales with observed frameshifting. Together, our results demonstrate that cotranslational folding of this protein generates a tension that stimulates PRF. To our knowledge, this constitutes the first example in which the conformational state of the nascent chain has been linked to PRF. These findings raise the possibility that, in addition to RNA-mediated translational recoding, a variety of cotranslational folding and/ or binding events may also stimulate PRF.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;790444v2" />
<meta name="citation_id" content="790444v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/790444v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/790444v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/790444v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444.full.pdf" />
<meta name="citation_doi" content="10.1101/790444" />
<meta name="citation_num_pages" content="37" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="790444" />
<meta name="citation_author" content="Haley R. Harrington" />
<meta name="citation_author_institution" content="Department of Chemistry, Indiana University" />
<meta name="citation_author" content="Matthew H. Zimmer" />
<meta name="citation_author_institution" content="Division of Chemistry and Chemical Engineering, California Institute of Technology" />
<meta name="citation_author" content="Laura M. Chamness" />
<meta name="citation_author_institution" content="Department of Chemistry, Indiana University" />
<meta name="citation_author" content="Veronica Nash" />
<meta name="citation_author_institution" content="Department of Chemistry, Indiana University" />
<meta name="citation_author" content="Wesley D. Penn" />
<meta name="citation_author_institution" content="Department of Chemistry, Indiana University" />
<meta name="citation_author" content="Thomas F. Miller" />
<meta name="citation_author_institution" content="Division of Chemistry and Chemical Engineering, California Institute of Technology" />
<meta name="citation_author_email" content="jschleba@indiana.edu" />
<meta name="citation_author_email" content="sumukhop@indiana.edu" />
<meta name="citation_author_email" content="tfm@caltech.edu" />
<meta name="citation_author" content="Suchetana Mukhopadhyay" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University" />
<meta name="citation_author_email" content="jschleba@indiana.edu" />
<meta name="citation_author_email" content="sumukhop@indiana.edu" />
<meta name="citation_author_email" content="tfm@caltech.edu" />
<meta name="citation_author" content="Jonathan P. Schlebach" />
<meta name="citation_author_institution" content="Department of Chemistry, Indiana University" />
<meta name="citation_author_email" content="jschleba@indiana.edu" />
<meta name="citation_author_email" content="sumukhop@indiana.edu" />
<meta name="citation_author_email" content="tfm@caltech.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0955-7633" />
<meta name="citation_reference" content="Caliskan, N., Peske, F. &amp; Rodnina, M. V. Changed in translation: mRNA recoding by −1 programmed ribosomal frameshifting. Trends Biochem Sci 40, 265–274 (2015)." />
<meta name="citation_reference" content="Halma, M. T., Ritchie, D. B, Capellano, T. R., Neupane, K. &amp; Woodside, M. T. Complex dynamics under tension in a high-efficiency frameshift stimulatory structure. Proc Natl Acad Sci U S A 116, 19500–19505 (2019)." />
<meta name="citation_reference" content="Namy, O., Moran, S. J., Stuart, D. I., Gilbert, R. J. &amp; Brierley, I. A mechanical explanation of RNA pseudoknot function in programmed ribosomal frameshifting. Nature 441, 244–247 (2006)." />
<meta name="citation_reference" content="Kim, H., et al. A frameshifting stimulatory stem loop destabilizes the hybrid state and impedes ribosomal translocation. Proc Natl Acad Sci U S A 111, 5538–5543 (2014)." />
<meta name="citation_reference" content="Caliskan, N., Katunin, V. I., Belardinelli, R., Peske, F. &amp; Rodnina, M. V. Programmed −1 frameshifting by kinetic partitioning during impeded translocation. Cell 157, 1619–1631 (2014)." />
<meta name="citation_reference" content="Chen et al. Dynamic pathways of −1 translational frameshifting. Nature 512, 328–332 (2014)." />
<meta name="citation_reference" content="Choi, J., OLoughlin, S., Atkins, J. F. &amp; Puglisi, J. D. The energy landscape of −1 ribosomal frameshifting. Sci Adv 6, eaax6969 (2020)." />
<meta name="citation_reference" content="Korniy, N., Samatova, E., Anokhina, M. M., Peske, F. &amp; Rodnina, M. V. Mechanisms and biomedical implications of −1 programmed ribosome frameshifting on viral and bacterial mRNAs. FEBS lett 593, 1468–1482 (2019)." />
<meta name="citation_reference" content="Bock et al. Thermodynamic control of −1 programmed ribosomal frameshifting. Nat Commun 10, 4598 (2019)." />
<meta name="citation_reference" content="Belew, A. T. et al. Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway. Nature 512, 265–269 (2014)." />
<meta name="citation_reference" content="Napthine, S. et al. Protein-directed ribosomal frameshifting temporally regulates gene expression. Nat Commun 8, 15582 (2017)." />
<meta name="citation_reference" content="Wang, X. et al. Regulation of HIV-1 Gag-Pol Expression by Shiftless, an Inhibitor of Programmed −1 Ribosomal Frameshifting. Cell 176, 625–635 e614 (2019)." />
<meta name="citation_reference" content="Ramsey, J. &amp; Mukhopadhyay, S. Disentangling the Frames, the State of Research on the Alphavirus 6K and TF Proteins. Viruses 9, 228 (2017)." />
<meta name="citation_reference" content="Brown, R. S., Wan, J. J. &amp; Kielian, M. The Alphavirus Exit Pathway: What We Know and What We Wish We Knew. Viruses 10, E89 (2018)." />
<meta name="citation_reference" content="Lobigs, M., Zhao, H. X. &amp; Garoff, H. Function of Semliki Forest virus E3 peptide in virus assembly: replacement of E3 with an artificial signal peptide abolishes spike heterodimerization and surface expression of E1. J Virol 64, 4346–4355 (1990)." />
<meta name="citation_reference" content="Mulvey, M. &amp; Brown, D. T. Assembly of the Sindbis virus spike protein complex. Virology 219, 125–132 (1996)." />
<meta name="citation_reference" content="Firth, A. E., Chung, B. Y., Fleeton, M. N. &amp; Atkins, J. F. Discovery of frameshifting in Alphavirus 6K resolves a 20-year enigma. Virol J 5, 108 (2008)." />
<meta name="citation_reference" content="Hallengard, D. et al. Novel attenuated Chikungunya vaccine candidates elicit protective immunity in C57BL/6 mice. J Virol 88, 2858–2866 (2014)." />
<meta name="citation_reference" content="Taylor, A. et al. Effects of an In-Frame Deletion of the 6k Gene Locus from the Genome of Ross River Virus. J Virol 90, 4150–4159, DOI:10.1128/JVI.03192-15 (2016)." />
<meta name="citation_reference" content="Snyder, J. E. et al. Functional characterization of the alphavirus TF protein. J Virol 87, 8511–8523 (2013)." />
<meta name="citation_reference" content="Rogers, K. J., Jones-Burrage, S., Maury, W. &amp; Mukhopadhyay, S. TF protein of Sindbis virus antagonizes host type I interferon responses in a palmitoylation-dependent manner. Virology, In press." />
<meta name="citation_reference" content="Chung, B. Y., Firth, A. E. &amp; Atkins, J. F. Frameshifting in alphaviruses: a diversity of 3’ stimulatory structures. J Mol Biol 397, 448–456 (2010)." />
<meta name="citation_reference" content="Kendra, J. A et al. Functional and structural characterization of the chickungunya virus translational recoding signals. J Biol Chem 293, 17536–17545 (2018)." />
<meta name="citation_reference" content="Snyder, A. J., Sokoloski, K. J. &amp; Mukhopadhyay, S. Mutating conserved cysteines in the alphavirus e2 glycoprotein causes virus-specific assembly defects. J Virol 86, 3100–3111 (2012)." />
<meta name="citation_reference" content="Ramsey, J., Renzi, E. C., Arnold, R. J., Trinidad, J. C. &amp; Mukhopadhyay, S. Palmitoylation of Sindbis Virus TF Protein Regulates Its Plasma Membrane Localization and Subsequent Incorporation into Virions. J Virol 91, e02000–e2016 (2017)." />
<meta name="citation_reference" content="Gaedigk-Nitschko, K. &amp; Schlesinger, M. J. The Sindbis virus 6K protein can be detected in virions and is acylated with fatty acids. Virology 175, 274–281 (1990)." />
<meta name="citation_reference" content="Politis, E. G., Roth, A. F. &amp; Davis, N. G. Transmembrane topology of the protein palmitoyl transferase Akr1. J Biol Chem 280, 10156–10163 (2005)." />
<meta name="citation_reference" content="Liljestrom, P. &amp; Garoff, H. Internally located cleavable signal sequences direct the formation of Semliki Forest virus membrane proteins from a polyprotein precursor. J Virol 65, 147–154 (1991)." />
<meta name="citation_reference" content="Cheng, R. H. et al. Nucleocapsid and glycoprotein organization in an enveloped virus. Cell 80, 621–630 (1995)." />
<meta name="citation_reference" content="Chen, L. et al. Implication for alphavirus host-cell entry and assembly indicated by a 3.5A resolution cryo-EM structure. Nat Commun 9, 5326 (2018)." />
<meta name="citation_reference" content="Gafvelin, G. &amp; von Heijne, G. Topological “frustration” in multispanning E. coli inner membrane proteins. Cell 77, 401–412 (1994)." />
<meta name="citation_reference" content="Ruch, T. R. &amp; Machamer, C. E. The coronavirus E protein: assembly and beyond. Viruses 4, 363–382 (2012)." />
<meta name="citation_reference" content="Hessa, T. et al. Molecular code for transmembrane-helix recognition by the Sec61 translocon. Nature 450, 1026–1030 (2007)." />
<meta name="citation_reference" content="Hessa, T. et al. Recognition of transmembrane helices by the endoplasmic reticulum translocon. Nature 433, 377–381 (2005)." />
<meta name="citation_reference" content="Ivanova, L. &amp; Schlesinger, M. J. Site-directed mutations in the Sindbis virus E2 glycoprotein identify palmitoylation sites and affect virus budding. J Virol 67, 2546–2551 (1993)." />
<meta name="citation_reference" content="Ryan, C., Ivanova, L. &amp; Schlesinger, M. J. Effects of site-directed mutations of transmembrane cysteines in sindbis virus E1 and E2 glycoproteins on palmitylation and virus replication. Virology 249, 62–67 (1998)." />
<meta name="citation_reference" content="Lee, H., Min, J., von Heijne, G. &amp; Kim, H. Glycosylatable GFP as a compartment-specific membrane topology reporter. Biochem Biophys Res Comm 427, 780–784 (2012)." />
<meta name="citation_reference" content="Virkki, M. T. et al. Folding of Aquaporin 1: Multiple evidence that helix 3 can shift out of the membrane core. Protein Sci 23, 981–992 (2014)." />
<meta name="citation_reference" content="Khan, Y. A., Loughran, G. &amp; Atkins, J. F. Contesting the Evidence for −1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5)-the HIV co-receptor. bioRxiv (2019)." />
<meta name="citation_reference" content="Ismail, N., Hedman, R., Schiller, N. &amp; von Heijne, G. A biphasic pulling force acts on transmembrane helices during translocon-mediated membrane integration. Nat Struct Mol Biol 19, 1018–1022 (2012)." />
<meta name="citation_reference" content="Cymer, F., Ismail, N. &amp; von Heijne, G. Weak pulling forces exerted on Nin-orientated transmembrane segments during co-translational insertion into the inner membrane of Escherichia coli. FEBS Lett 588, 1930–1934 (2014)." />
<meta name="citation_reference" content="Cymer, F. &amp; von Heijne, G. Cotranslational folding of membrane proteins probed by arrest-peptide-mediated force measurements. Proc Natl Acad Sci U S A 110, 14640–14645 (2013)." />
<meta name="citation_reference" content="Goldman, D. H. et al. Mechanical force releases nascent chain-mediated ribosome arrest in vitro and in vivo. Science 348, 457–460 (2015)." />
<meta name="citation_reference" content="Zhang, B. &amp; Miller, T. F., 3rd. Long-timescale dynamics and regulation of Sec-facilitated protein translocation. Cell rep 2, 927–937 (2012)." />
<meta name="citation_reference" content="Niesen, M. J., Wang, C. Y., Van Lehn, R. C. &amp; Miller, T. F., 3rd. Structurally detailed coarse-grained model for Sec-facilitated co-translational protein translocation and membrane integration. PLoS Comput Biol 13, e1005427 (2017)." />
<meta name="citation_reference" content="Van Lehn, R. C., Zhang, B. &amp; Miller, T. F., 3rd. Regulation of multispanning membrane protein topology via post-translational annealing. Elife 4, e08697 (2015)." />
<meta name="citation_reference" content="Niesen, M. J. M., Muller-Lucks, A., Hedman, R., von Heijne, G. &amp; Miller, T. F., 3rd. Forces on Nascent Polypeptides during Membrane Insertion and Translocation via the Sec Translocon. Biophys J 115, 1885–1894 (2018)." />
<meta name="citation_reference" content="Matlack, K. E., Misselwitz, B., Plath, K. &amp; Rapoport, T. A. BiP acts as a molecular ratchet during posttranslational transport of prepro-alpha factor across the ER membrane. Cell 97, 553–564 (1999)." />
<meta name="citation_reference" content="Brodsky, J. L., Goeckeler, J. &amp; Sheckman, R. BiP and Sec63p are Required for both co- and posttranslational protein translocation into the yeast endoplasmic reticulum. Proc Natl Acad Sci U S A 92, 9643–9646 (1995)." />
<meta name="citation_reference" content="Muldoon-Jacobs, K. L. &amp; Dinman, J. D. Specific Effects of Ribosome-Tethered Molecular Chaperones on Programmed −1 Ribosomal Frameshifting. Eukaryotic Cell 5, 762–770 (2006)." />
<meta name="citation_reference" content="Roushar, F. J. et al. Contribution of Cotranslational Folding Defects to Membrane Protein Homeostasis. J Am Chem Soc 141, 204–215 (2019)." />
<meta name="citation_reference" content="Voorhees, R. M., Fernandez, I. S., Scheres, S. H. &amp; Hegde, R. S. Structure of the mammalian ribosome-Sec61 complex to 3.4 A resolution. Cell 157, 1632–1643 (2014)." />
<meta name="citation_reference" content="Wu, S., Angelikopoulos, P., Tauriello, G., Papadimitriou, C. &amp; Koumoutsakos, P. Fusing heterogeneous data for the calibration of molecular dynamics force fields using hierarchical Bayesian models. J Chem Phys 145, 244112 (2016)." />
<meta name="twitter:title" content="Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Viruses maximize their genetic coding capacity through a variety of biochemical mechanisms including programmed ribosomal frameshifting (PRF), which facilitates the production of multiple proteins from a single transcript. PRF is typically stimulated by structural elements within the mRNA that generate mechanical tension between the transcript and ribosome. However, in this work we show that the forces generated by the cotranslational folding of the nascent polypeptide chain can also enhance PRF. Using an array of biochemical, cellular, and computational techniques, we first demonstrate that the Sindbis virus structural polyprotein forms two competing topological isomers during biosynthesis at the ribosome-translocon complex. We then show that the formation of one of these topological isomers is linked to PRF. Coarse-grained molecular dynamic simulations reveal that the translocon-mediated membrane integration of a transmembrane domain upstream from the ribosomal slip-site generates a force on the nascent polypeptide chain that scales with observed frameshifting. Together, our results demonstrate that cotranslational folding of this protein generates a tension that stimulates PRF. To our knowledge, this constitutes the first example in which the conformational state of the nascent chain has been linked to PRF. These findings raise the possibility that, in addition to RNA-mediated translational recoding, a variety of cotranslational folding and/ or binding events may also stimulate PRF." />
<meta name="og-title" property="og:title" content="Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/790444v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Viruses maximize their genetic coding capacity through a variety of biochemical mechanisms including programmed ribosomal frameshifting (PRF), which facilitates the production of multiple proteins from a single transcript. PRF is typically stimulated by structural elements within the mRNA that generate mechanical tension between the transcript and ribosome. However, in this work we show that the forces generated by the cotranslational folding of the nascent polypeptide chain can also enhance PRF. Using an array of biochemical, cellular, and computational techniques, we first demonstrate that the Sindbis virus structural polyprotein forms two competing topological isomers during biosynthesis at the ribosome-translocon complex. We then show that the formation of one of these topological isomers is linked to PRF. Coarse-grained molecular dynamic simulations reveal that the translocon-mediated membrane integration of a transmembrane domain upstream from the ribosomal slip-site generates a force on the nascent polypeptide chain that scales with observed frameshifting. Together, our results demonstrate that cotranslational folding of this protein generates a tension that stimulates PRF. To our knowledge, this constitutes the first example in which the conformational state of the nascent chain has been linked to PRF. These findings raise the possibility that, in addition to RNA-mediated translational recoding, a variety of cotranslational folding and/ or binding events may also stimulate PRF." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/790444v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/790444v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/790444v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/790444v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1107555" />
    <title>Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"V2YZet1As2bSSIOmuASKrxoUM0jR5pFhfohF-76KIZQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1107555","apath":"\/biorxiv\/early\/2020\/01\/16\/790444.atom","pisa":"biorxiv;790444v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;790444v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/790444v2","title":"Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein","identifier":"node\/1107555"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022s3XH6mkuEp0WMiWJq2kTDH3X1Livjg2zqJwY5ke9ybeT6E45y1e9\\\\\\\/UDRnqJkVsmj4MG\\\\\\\/NRZqHGqR0FR5FRZ+YdPw+XnqeeqFUmrKI6dE4tYwI8QLxLShF0HWQx1NR6xx7PCk5aQ5bRQTjx3rWLCeKUpcnBpG+T9esOV6XFrcVrx+ZrzdfcQYlSRU2Pc\\\\\\\/GqCE22oGo4lhAf2zbW\\\\\\\/jgYYT5cgQZUhY8RADqJ6HaBde6rW2Sw7RYENXRTDqJ9r2aGz0XEkgPuyOG2FHygrfxMN+Z+gA4r9klgu7SqVGutqj8KgguY6IJ7NNEeuD\\\\\\\/VTwT7cnfN49Wo+PfdQ7UejKOHvZNCDfT9qXozoyNsQw6MXLdDx+5Wsrjv6hxg\\\\\\\/kTGCPgsHi8Dm\\\\\\\/vZFL+Y\\\\\\\/H8lFOXhJ+HUfo5grBlPIgiCaQ8Vrjr6iGEx63\\\\\\\/qfxEz7QpBgAQJXQqZCxLs8doeIRqSa5MyC4HpOwaNKlESCzcrpPE\\\\\\\/rNY0+P6xB2sqTqv1xT6TP7adhltPIG5NX+0PNGpdCj3dGDfl7JIOAtflD2s6JV0MkPYPi+9lAeoURSJf1bapp4iMjF8Eqr\\\\\\\/ysUE55f7ivtwGK0fBFN0nQFCnecTWkLjc22plOjhp9g8LqSeooZs2jQlpKkC7d2BdEHD1Gn6IK6gZb4dE8\\\\\\\/9wK2TyjFRre2fKX89bxwNmaatvsg87ysWpa56Qo0b0kV2LmMn1adUlYzvCiAvjofaUth7T38pwkjHm7Ls9GRoSRTDV99kzSjhxQYsASx\\\\\\\/erd4kkbVkHP56yjXJl9B1taMT+qtn5TooL6KjNEPYYfcK8ItMfXDMXU24q6uzJ855aOqKb1kN6IKoOYqwLjV2CiMfdalOrFSjpf\\\\\\\/5yHdMDafbA3pRzZom+rd14On3uRYyB+SYfy5ug+virI9pQ6u9COvwLj5QGmWtwboZm+Um75k842qmouYpCwW0s9o\\\\\\\/9+Rt3ot2OSK5gRRqQJiq8\\\\\\\/AwjQ5YisShuuyQr1QHvK\\\\\\\/6Bxpb1a7sIfgg3DqmVkOIbUwIlrmysz+XRMwO2jHILZudql2l0DxlFiiaAF3y+69tFUF+CiDpsLoIPlCev\\\\\\\/6yUsjdjW9QPEjhwsbRgNZP0YSWwNY\\\\\\\/UDdjcswaUyBVrHymTmywUh334RrmkHFMvR5B8pg4wGAzre68+uo74kaUo6VY9+bQXx\\\\\\\/5bQVDnR2BBbC67o2Az+n5cdAsgmL3e2eZoQU\\\\\\\/j3qT4\\\\\\\/PfVnd5uswvFvJ8WtcghIdQ2g2180dAdbrxVlZmSkpY1CUPSg2MdSceqdlkufjW23dEZZBm2Kbam\\\\\\\/ulabzLM7cGBXGXjQVF2vtDBO+I2RmJiOEZj3fycJHgOq\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/rx2LX5C4qGWlEs6yb+GUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022217bc9c75762b0196a1aa2c8099cefe986b967b3a984daa2a56159b85e08973a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/790444v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1107555 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/790444v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SAKmXSxiLbSMbItOMGRV63IKyxi5ltOXm1xC2eBch1E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107555" id="node1107555" data-pisa="biorxiv;790444v2" data-pisa-master="biorxiv;790444" data-apath="/biorxiv/early/2020/01/16/790444.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Haley R.</span> <span class="nlm-surname">Harrington</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Matthew H.</span> <span class="nlm-surname">Zimmer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Laura M.</span> <span class="nlm-surname">Chamness</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Veronica</span> <span class="nlm-surname">Nash</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wesley D.</span> <span class="nlm-surname">Penn</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Thomas F.</span> <span class="nlm-surname">Miller</span> <span class="nlm-suffix">III</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Suchetana</span> <span class="nlm-surname">Mukhopadhyay</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0955-7633" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jonathan P.</span> <span class="nlm-surname">Schlebach</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/790444 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1107555" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haley R. Harrington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemistry, Indiana University</span>, Bloomington, IN <span class='nlm-country'>USA</span> 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haley%2BR.%2BHarrington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harrington%20HR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaley%2BR.%2BHarrington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew H. Zimmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Division of Chemistry and Chemical Engineering, California Institute of Technology</span>, Pasadena, CA <span class='nlm-country'>USA</span> 91125</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BH.%2BZimmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zimmer%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BH.%2BZimmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Laura M. Chamness </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemistry, Indiana University</span>, Bloomington, IN <span class='nlm-country'>USA</span> 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BM.%2BChamness%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chamness%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BM.%2BChamness%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Veronica Nash </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemistry, Indiana University</span>, Bloomington, IN <span class='nlm-country'>USA</span> 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Veronica%2BNash%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nash%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVeronica%2BNash%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wesley D. Penn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemistry, Indiana University</span>, Bloomington, IN <span class='nlm-country'>USA</span> 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wesley%2BD.%2BPenn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Penn%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWesley%2BD.%2BPenn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Thomas F. Miller <span class="nlm-suffix">III</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Division of Chemistry and Chemical Engineering, California Institute of Technology</span>, Pasadena, CA <span class='nlm-country'>USA</span> 91125</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BF.%2BMiller%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EIII%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20TF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BF.%2BMiller%2BIII" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jschleba@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">jschleba@indiana.edu</a>
<a href="mailto:sumukhop@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">sumukhop@indiana.edu</a>
<a href="mailto:tfm@caltech.edu" class="" data-icon-position="" data-hide-link-title="0">tfm@caltech.edu</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Suchetana Mukhopadhyay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biology, Indiana University</span>, Bloomington, IN <span class='nlm-country'>USA</span> 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchetana%2BMukhopadhyay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mukhopadhyay%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASuchetana%2BMukhopadhyay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jschleba@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">jschleba@indiana.edu</a>
<a href="mailto:sumukhop@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">sumukhop@indiana.edu</a>
<a href="mailto:tfm@caltech.edu" class="" data-icon-position="" data-hide-link-title="0">tfm@caltech.edu</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jonathan P. Schlebach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemistry, Indiana University</span>, Bloomington, IN <span class='nlm-country'>USA</span> 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BP.%2BSchlebach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schlebach%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BP.%2BSchlebach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0955-7633" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan P. Schlebach</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jschleba@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">jschleba@indiana.edu</a>
<a href="mailto:sumukhop@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">sumukhop@indiana.edu</a>
<a href="mailto:tfm@caltech.edu" class="" data-icon-position="" data-hide-link-title="0">tfm@caltech.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/790444v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1107555" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1107555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/790444v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1107555" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1107555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/790444v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1107555" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1107555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/790444v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1107555" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1107555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/790444v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1107555" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1107555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/790444v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1107555" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1107555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">Viruses maximize their genetic coding capacity through a variety of biochemical mechanisms including programmed ribosomal frameshifting (PRF), which facilitates the production of multiple proteins from a single transcript. PRF is typically stimulated by structural elements within the mRNA that generate mechanical tension between the transcript and ribosome. However, in this work we show that the forces generated by the cotranslational folding of the nascent polypeptide chain can also enhance PRF. Using an array of biochemical, cellular, and computational techniques, we first demonstrate that the Sindbis virus structural polyprotein forms two competing topological isomers during biosynthesis at the ribosome-translocon complex. We then show that the formation of one of these topological isomers is linked to PRF. Coarse-grained molecular dynamic simulations reveal that the translocon-mediated membrane integration of a transmembrane domain upstream from the ribosomal slip-site generates a force on the nascent polypeptide chain that scales with observed frameshifting. Together, our results demonstrate that cotranslational folding of this protein generates a tension that stimulates PRF. To our knowledge, this constitutes the first example in which the conformational state of the nascent chain has been linked to PRF. These findings raise the possibility that, in addition to RNA-mediated translational recoding, a variety of cotranslational folding and/ or binding events may also stimulate PRF.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Viruses have evolved numerous mechanisms to exploit the host machinery in order to increase the coding capacity of their highly constrained genomes. There are at least 27 viral genera that utilize programmed ribosomal frameshifting (PRF) in order to produce multiple proteins from a single transcript (<a href="https://viralzone.expasy.org/860">https://viralzone.expasy.org/860</a>). PRF is genetically encoded, and minimally requires a portion of the transcript that contains a repetitive “slippery” heptanucleotide sequence (slip-site) followed by a region that forms stimulatory RNA secondary structures (an ensemble of stem loops and/ or pseudoknots).<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup> A collision between the translating ribosome and the stimulatory secondary structure increases the kinetic barrier to translocation, which causes the ribosome to dwell on the slip-site.<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>-<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup> During this pause, the t-RNA that is annealed within the ribosomal P-site (and most often also the t-RNA in the A-site)<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup> begin to sample alternative base pairing interactions that shift the reading frame of the ribosme.<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup> Based on these mechanistic considerations, PRF is typically believed to be mediated at the level of RNA structure. Nevertheless, recent reports have also found that the efficiency of −1PRF can be tuned by a variety of regulatory proteins and/ or miRNA.<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup></p><p id="p-5">-1PRF is utilized to temporally and stoichiometrically regulate protein production during viral replication and assembly. For instance, the alphavirus structural proteins are most often produced from a single polyprotein that is cleaved into the capsid (CP), E3, E2, 6K, and E1 proteins (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> The E2 and E1 proteins are membrane glycoproteins that heterodimerize early in the assembly pathway. These dimeric units then form trimeric spike complexes, traffic to the plasma membrane, and initiate viral budding.<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>-<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup> −1PRF during the translation of the 6K protein gives rise to a secondary form of the polyprotein containing the TransFrame (TF) protein,<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup> a known virulence factor,<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>-<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup> in place of the 6K and E1 proteins (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Because −1PRF precludes the translation of E1, the efficiency of ribosomal frameshifting (1-48% in alphaviruses)<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> influences the stoichiometric ratio of the E1 and E2 glycoproteins and the net accumulation of spike complexes. Current evidence suggests −1PRF is stimulated by a canonical poly-U slip site and a downstream RNA hairpin.<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> However, an effort to map the stimulatory RNA structures within alphavirus polyproteins revealed that deletions within the predicted hairpin region are capable of reducing the efficiency of −1PRF, but appear to be insufficient to knock out frameshifting completely.<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup> This observation suggests there may be multiple regulatory mechanisms that mediate −1PRF within the alphavirus structural polyprotein.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure and Topological Properties of the Alphavirus Polyprotein. A) A cartoon depicts the relative size and orientation of proteins within the major form of the alphavirus structural polyprotein. B) A cartoon depicts the relative size and orientation of proteins within the frameshifted form of the alphavirus structural polyprotein. C) A portion of the SINV structural polyprotein spanning the E2, 6K, and E1 proteins was scanned with the &#x394;G predictor using a 23 residue window.33 The predicted free energy difference associated with the cotrantlational membrane integration of every possible 23 residue segment within the major form (black) and frameshifted form (orange) of the structural polyprotein is plotted as a function of the position of its central residue. The position of each predicted TM domain is indicated in purple. D) A cartoon depicts the manner in which the topological preferences of the guest TM domain (purple) influence the glycosylation state of the chimeric Lep protein. The glycosylation machinery is on the interior of the vesicle (the microsomal lumen). Membrane integration of the guest domain results in the production of a singly glycosylated product (bottom). The mis-integration of the guest domain results in the production of a doubly-glycosylated product (top). E) Chimeric Lep constructs bearing putative TM domains from the SINV structural polyprotein were produced by in vitro translation in the presence of canine rough microsomes, and analyzed by SDS-PAGE. A representative gel reveals the relative abundance of singly (G1) and doubly (G2) glycosylated translation products for each contruct. Control reactions contaning no RNA (no protein) and no rough microsomes (untargetted protein) are shown for the sake of comparison. These trends were consistently observed across five independent replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1949721169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Structure and Topological Properties of the Alphavirus Polyprotein.&lt;/span&gt; A) A cartoon depicts the relative size and orientation of proteins within the major form of the alphavirus structural polyprotein. B) A cartoon depicts the relative size and orientation of proteins within the frameshifted form of the alphavirus structural polyprotein. C) A portion of the SINV structural polyprotein spanning the E2, 6K, and E1 proteins was scanned with the &#x394;G predictor using a 23 residue window.&lt;sup&gt;33&lt;/sup&gt; The predicted free energy difference associated with the cotrantlational membrane integration of every possible 23 residue segment within the major form (black) and frameshifted form (orange) of the structural polyprotein is plotted as a function of the position of its central residue. The position of each predicted TM domain is indicated in purple. D) A cartoon depicts the manner in which the topological preferences of the guest TM domain (purple) influence the glycosylation state of the chimeric Lep protein. The glycosylation machinery is on the interior of the vesicle (the microsomal lumen). Membrane integration of the guest domain results in the production of a singly glycosylated product (bottom). The mis-integration of the guest domain results in the production of a doubly-glycosylated product (top). E) Chimeric Lep constructs bearing putative TM domains from the SINV structural polyprotein were produced by &lt;em&gt;in vitro&lt;/em&gt; translation in the presence of canine rough microsomes, and analyzed by SDS-PAGE. A representative gel reveals the relative abundance of singly (G1) and doubly (G2) glycosylated translation products for each contruct. Control reactions contaning no RNA (no protein) and no rough microsomes (untargetted protein) are shown for the sake of comparison. These trends were consistently observed across five independent replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F1.medium.gif" width="440" height="223"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F1.medium.gif" width="440" height="223"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Structure and Topological Properties of the Alphavirus Polyprotein.</span><p id="p-6" class="first-child">A) A cartoon depicts the relative size and orientation of proteins within the major form of the alphavirus structural polyprotein. B) A cartoon depicts the relative size and orientation of proteins within the frameshifted form of the alphavirus structural polyprotein. C) A portion of the SINV structural polyprotein spanning the E2, 6K, and E1 proteins was scanned with the ΔG predictor using a 23 residue window.<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> The predicted free energy difference associated with the cotrantlational membrane integration of every possible 23 residue segment within the major form (black) and frameshifted form (orange) of the structural polyprotein is plotted as a function of the position of its central residue. The position of each predicted TM domain is indicated in purple. D) A cartoon depicts the manner in which the topological preferences of the guest TM domain (purple) influence the glycosylation state of the chimeric Lep protein. The glycosylation machinery is on the interior of the vesicle (the microsomal lumen). Membrane integration of the guest domain results in the production of a singly glycosylated product (bottom). The mis-integration of the guest domain results in the production of a doubly-glycosylated product (top). E) Chimeric Lep constructs bearing putative TM domains from the SINV structural polyprotein were produced by <em>in vitro</em> translation in the presence of canine rough microsomes, and analyzed by SDS-PAGE. A representative gel reveals the relative abundance of singly (G1) and doubly (G2) glycosylated translation products for each contruct. Control reactions contaning no RNA (no protein) and no rough microsomes (untargetted protein) are shown for the sake of comparison. These trends were consistently observed across five independent replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">-1PRF occurs during synthesis and processing of the nascent alphavirus structural polyprotein at the endoplasmic reticulum (ER) membrane. Following autoproteolytic cleavage of CP in the cytosol, a signal peptide at the N-terminus of the E3 protein directs the nascent polyprotein to the endoplasmic reticulum (ER) lumen where processing of the downstream proteins occurs. Localization of these segments within the lumen is essential to ensure that the E3, E2, and E1 ectodomains form their native disulfides and undergo glycosylation.<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> Post-translational modifications are also critical for TF, which must be palmitoylated in order to reach the plasma membrane and incorporate into the viral envelope.<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> The palmitoylated cysteines in TF are positioned near the edge of a putative transmembrane (TM) domain that is found in both TF and 6K.<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup> Though these residues are present in both proteins, they are only palmitoylated in the context of the frameshifted polyprotein.<sup><a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a></sup> Considering palmitoylation only occurs on the cytosolic face of cellular membranes,<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> the distinct modification state of the two forms of the polyprotein is therefore suggestive of an underlying difference in their topologies. In this study, we set out to gain insight into the interplay between −1PRF and the topology of the structural polyprotein. We first mapped the topology of the Sindbis virus (SINV) structural polyprotein. Our results demonstrate that the structural polyprotein forms two topological isomers. The predominant topology features two TM domains upstream of the −1PRF site, and its formation coincides with production of the 6K protein. Alternatively, the minor topology contains an additional TM domain upstream from the −1PRF site that is linked to the production of TF. Using protein engineering in conjunction with coarse-grained molecular dynamics (CGMD) simulations, we demonstrate that the efficiency of −1PRF is dependent upon the force generated by the translocon-mediated membrane integration of the extra TM domain within the minor topomer. Together, our observations highlight novel connections between the cotranslational folding, biosynthesis, and processing of the alphavirus structural polyprotein. Moreover, our findings reveal a novel mechanism that regulates the overall efficiency of −1PRF.</p></div><div class="section" id="sec-2"><h2 class="">Results &amp; Discussion</h2><div id="sec-3" class="subsection"><h3>Topological Properties of the Alphavirus Structural Polyprotein</h3><p id="p-8">The current model of the alphavirus structural polyprotein suggests the E2 and 6K proteins each contain two TM domains.<sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> However, there are two caveats to this model. First, cryo-EM structures reveal that the E2 protein only contains a single TM domain in the context of the viral envelope.<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> Though it has been speculated that a second TM domain within E2 is somehow extruded from the membrane during processing, the marginal hydrophobicity of this segment also raises the possibility that it may fail to undergo translocon-mediated membrane integration in the first place. Second, the hydrophobic portion of the SINV 6K protein is only 35 residues in length, which is quite short for a segment containing two putative TM domains and a loop. These ambiguous topological signals suggest this portion of the polyprotein is frustrated and could potentially form multiple topological isomers,<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> as has been suggested for the coronavirus E protein.<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup></p><p id="p-9">To survey the topological preferences of the E2-6K region, we scanned its sequence using a knowledge-based algorithm that predicts the energetics associated with the transfer of polypeptide segments from the translocon to the ER membrane (ΔG predictor).<sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup> Energetic predictions suggest only the regions corresponding to the first hydrophobic segments within the E2 (TM1) and 6K (TM3) proteins are sufficiently hydrophobic to undergo robust membrane integration (Δ<em>G</em> &lt; 0 kcal/ mol, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>). In contrast, the translocon-mediated membrane integration of the second hydrophobic segment within E2 (TM2) is predicted to be inefficient (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a>). To test these predictions, we measured the translocon-mediated membrane integration of each putative TM domain using a glycosylation-based translocation assay.<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> Briefly, the sequences of each putative TM domain were cloned into a chimeric leader peptidase (Lep) reporter protein.<sup><a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a></sup> Membrane integration of the putative TM segment (purple helix, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1D</a>) results in the modification of only a single glycosylation site in Lep, whereas the passage of the segment into the lumen results in the modification of two glycosylation sites (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1D</a>). Chimeric Lep proteins were produced by <em>in vitro</em> translation in the presence of canine rough microsomes, which contain native ER membranes and translocons. Consistent with predictions, Lep proteins containing TMs 1 &amp; 3 acquire a single glycosyl modification, which suggests these segments undergo robust translocon-mediated membrane integration (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1E</a>). In contrast, the translocon-mediated membrane integration of the second putative TM domain of E2 (TM2) is significantly less efficient (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1E</a>). These observations suggest the E2 and 6K proteins are each likely to contain a single TM domain (TMs 1 &amp; 3, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). It should also be noted that −1PRF only modifies the sequence of the loops downstream from these TM domains (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1C</a>, orange line), and is therefore unlikely to impact their topological preferences.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Topological Properties of the Major Form of the SINV Structural Polyprotein. A) A cartoon depicts a putative topological model of most abundant topology of the structural polyprotein that is consistent with computational and biochemical data. The positions at which glycosylatable GFP (gGFP) reporter domains were fused to determine the cellular compartmentalization of the C-terminal portion of the segments corresponding to TMs 1 (green), 2 (yellow), 3 (orange), and the C-terminal edge of the hydrophobic portion of 6K (TM3+, red) are indicated with arrows. B) A cartoon depicts the manner in which the cellular compartmentalization of the gGFP reporter domain alters its fluorescence. Topological signals that direct the gGFP domain into the cytosol will generate a fluorescent gGFP (left), wheras topological signals that direct the gGFP domain into the lumen (right) generate a glycosylated, non-fluorescent fusion domain. C) Reporter constructs bearing a gGFP fusion downstream from TM1 (green), TM2 (yellow), TM3 (orange), and the hydrophobic portion of the 6K protein (TM3+, red) were expressed in HEK293T cells, and cellular fluorescence intensities were analyzed by flow cytometry. A histogram from a representative trial depicts the gGFP intensities assoicated with 3,000 transfected cells expressing each reporter construct at a consistent expression level as judged by the intensity of a bicistronic expression reporter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1949721169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Topological Properties of the Major Form of the SINV Structural Polyprotein.&lt;/span&gt; A) A cartoon depicts a putative topological model of most abundant topology of the structural polyprotein that is consistent with computational and biochemical data. The positions at which glycosylatable GFP (gGFP) reporter domains were fused to determine the cellular compartmentalization of the C-terminal portion of the segments corresponding to TMs 1 (green), 2 (yellow), 3 (orange), and the C-terminal edge of the hydrophobic portion of 6K (TM3+, red) are indicated with arrows. B) A cartoon depicts the manner in which the cellular compartmentalization of the gGFP reporter domain alters its fluorescence. Topological signals that direct the gGFP domain into the cytosol will generate a fluorescent gGFP (left), wheras topological signals that direct the gGFP domain into the lumen (right) generate a glycosylated, non-fluorescent fusion domain. C) Reporter constructs bearing a gGFP fusion downstream from TM1 (green), TM2 (yellow), TM3 (orange), and the hydrophobic portion of the 6K protein (TM3+, red) were expressed in HEK293T cells, and cellular fluorescence intensities were analyzed by flow cytometry. A histogram from a representative trial depicts the gGFP intensities assoicated with 3,000 transfected cells expressing each reporter construct at a consistent expression level as judged by the intensity of a bicistronic expression reporter.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F2.medium.gif" width="209" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F2.medium.gif" width="209" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Topological Properties of the Major Form of the SINV Structural Polyprotein.</span><p id="p-10" class="first-child">A) A cartoon depicts a putative topological model of most abundant topology of the structural polyprotein that is consistent with computational and biochemical data. The positions at which glycosylatable GFP (gGFP) reporter domains were fused to determine the cellular compartmentalization of the C-terminal portion of the segments corresponding to TMs 1 (green), 2 (yellow), 3 (orange), and the C-terminal edge of the hydrophobic portion of 6K (TM3+, red) are indicated with arrows. B) A cartoon depicts the manner in which the cellular compartmentalization of the gGFP reporter domain alters its fluorescence. Topological signals that direct the gGFP domain into the cytosol will generate a fluorescent gGFP (left), wheras topological signals that direct the gGFP domain into the lumen (right) generate a glycosylated, non-fluorescent fusion domain. C) Reporter constructs bearing a gGFP fusion downstream from TM1 (green), TM2 (yellow), TM3 (orange), and the hydrophobic portion of the 6K protein (TM3+, red) were expressed in HEK293T cells, and cellular fluorescence intensities were analyzed by flow cytometry. A histogram from a representative trial depicts the gGFP intensities assoicated with 3,000 transfected cells expressing each reporter construct at a consistent expression level as judged by the intensity of a bicistronic expression reporter.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Based on the computational and biochemical results in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>, we generated a topological model of the SINV structural polyprotein (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A</a>). This model correctly places the E2 and E1 ectodomains within the ER lumen, and places the two palmitoylated cysteine residues in E2 (C716 and C718) within the cytosol.<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>,<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> To probe the topological preferences of the SINV polyprotein in the cell, we produced and characterized a series of reporter constructs that begin with the E3 protein and end at the C-terminal edge of each of the three putative TM domains within E2 and 6K (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2A</a>, Supplemental Fig. 1). Each of these fragments was genetically fused to a C-terminal cassette containing a short GS linker and glycosylatable GFP (gGFP) gene, which contains two glycosylation sites within the core of the eGFP protein.<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> Topological signals that direct the gGFP protein into the cytosol will produce a fluorescent gGFP, whereas the glycosylation of gGFP within the ER lumen renders the protein non-fluorescent (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2B</a>). Each construct was then expressed in HEK293T cells, and flow cytometry was used to quantify the fluorescence intensity of the gGFP reporter at a consistent expression level as judged by the intensity of a bicistronic reporter protein (Supplemental Fig. 2). Expression of the reporter constructs containing gGFP downstream from TMs 1 &amp; 2 (see <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2A</a>) generates fluorescent gGFP (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2C</a>), which suggests the C-termini of these TM domains reside within the cytosol. In contrast, the reporter construct with gGFP downstream from TM3 (after R785) exhibits an attenuated GFP signal at an equivalent expression level (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2C</a>), which suggests the gGFP fused to the C-terminus of TM3 is projected into the ER lumen. Placement of the gGFP after the full stretch of hydrophobic amino acids in the 6K protein (after Y807, see <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2A</a>) also results in an attenuated gGFP signal (TM3+ in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2C</a>), which confirms the full stretch of hydrophobic residues near TM3 only spans the membrane once. Thus, results from this cellular reporter assay (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2C</a>) are consistent with predictions (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1C</a>), <em>in vitro</em> translation data (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig. 1E</a>), and the model shown in <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a>. These observations together confirm that the E1 and 6K proteins each contain a single TM domain in the most abundant form of the polyprotein.</p></div><div id="sec-4" class="subsection"><h3>Link between Topology and −1 Programmed Ribosomal Frameshifting</h3><p id="p-12">The topological properties of the structural polyprotein described above have implications for the manner in which it is processed at the ER membrane. Our model suggests the cluster of unmodified cysteines in the 6K protein (C786, C787, C789, and C790) reside at a C-terminal portion of TM3 that is projected into the ER lumen (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>) and is therefore inaccessible to palmitoylating enzymes. However, these same residues are palmitoylated in the TF protein,<sup><a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">25</a></sup> which suggests the orientation of TM3 must become inverted upon frameshifting in order to expose them to the cytosolic leaflet. Such an inversion could potentially occur as a consequence of the membrane integration of TM2 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>), which exhibits a weak propensity to undergo translocon-mediated membrane integration (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Fig. 1E</a>). Furthermore, the efficiency associated with the translocon-mediated membrane integration of TM2 (∼ 20 %, <a id="xref-fig-1-14" class="xref-fig" href="#F1">Fig. 1E</a>) is comparable to the frequency of - 1PRF in the SINV polyprotein (∼16%).<sup><a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a></sup> Taken together, these observations potentially suggest a connection between the formation of a secondary topomer and −1PRF.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Putative Model for the Interplay Between Translocon Mediated Membrane Integration and Ribosomal Frameshifting. A cartoon depicts the putative manner in which cotranslational folding of the nascent structural polyprotein is linked to &#x2212;1 programmed ribosomal frameshifting (&#x2212;1PRF). A) A cartoon depicts the topological properties of the major form of the nascent structural polyprotein. TM2 is too polar to robustly partition into the membrane during translation (left). As a result, the E2 protein only contains a single TM domain in the context of the major form of the structural polyprotein (right). Cysteine residues that are conditionally glycosylated in the frameshifted form of the polyprotein are shown in yellow. B) A cartoon depicts the topological properties of the frameshifted form of the nascent structural polyprotein. TM2 is hydrophobic enough to occaisionally partition into the membrane during translation (left), which imposes a tension on the ribosome that stimulates &#x2212;1PRF. As a result, the E2 protein contains two TM domains in the context of the frameshifted form of the structural polyprotein (right). Cysteine residues that are conditionally glycosylated in the frameshifted form of the polyprotein are shown in yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1949721169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Putative Model for the Interplay Between Translocon Mediated Membrane Integration and Ribosomal Frameshifting.&lt;/span&gt; A cartoon depicts the putative manner in which cotranslational folding of the nascent structural polyprotein is linked to &#x2212;1 programmed ribosomal frameshifting (&#x2212;1PRF). A) A cartoon depicts the topological properties of the major form of the nascent structural polyprotein. TM2 is too polar to robustly partition into the membrane during translation (left). As a result, the E2 protein only contains a single TM domain in the context of the major form of the structural polyprotein (right). Cysteine residues that are conditionally glycosylated in the frameshifted form of the polyprotein are shown in yellow. B) A cartoon depicts the topological properties of the frameshifted form of the nascent structural polyprotein. TM2 is hydrophobic enough to occaisionally partition into the membrane during translation (left), which imposes a tension on the ribosome that stimulates &#x2212;1PRF. As a result, the E2 protein contains two TM domains in the context of the frameshifted form of the structural polyprotein (right). Cysteine residues that are conditionally glycosylated in the frameshifted form of the polyprotein are shown in yellow.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F3.medium.gif" width="440" height="165"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F3.medium.gif" width="440" height="165"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Putative Model for the Interplay Between Translocon Mediated Membrane Integration and Ribosomal Frameshifting.</span><p id="p-13" class="first-child">A cartoon depicts the putative manner in which cotranslational folding of the nascent structural polyprotein is linked to −1 programmed ribosomal frameshifting (−1PRF). A) A cartoon depicts the topological properties of the major form of the nascent structural polyprotein. TM2 is too polar to robustly partition into the membrane during translation (left). As a result, the E2 protein only contains a single TM domain in the context of the major form of the structural polyprotein (right). Cysteine residues that are conditionally glycosylated in the frameshifted form of the polyprotein are shown in yellow. B) A cartoon depicts the topological properties of the frameshifted form of the nascent structural polyprotein. TM2 is hydrophobic enough to occaisionally partition into the membrane during translation (left), which imposes a tension on the ribosome that stimulates −1PRF. As a result, the E2 protein contains two TM domains in the context of the frameshifted form of the structural polyprotein (right). Cysteine residues that are conditionally glycosylated in the frameshifted form of the polyprotein are shown in yellow.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Based on these observations, we hypothesize that the translocon-mediated membrane integration of TM2 is mechanistically linked to −1PRF and the translation of the TF protein. Our model suggests mutations that alter the translocon-mediated membrane integration of TM2 should have a direct impact on −1PRF (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3</a>). To test this hypothesis, we assessed whether mutations that alter the hydrophobicity of TM2 also influence −1PRF. Both energetic predictions and <em>in vitro</em> translation measurements suggest the introduction of two non-native leucine residues into TM2 (T738L &amp; S739L, LL mutant) enhances the translocon-mediated membrane integration of TM2 (predicted ΔΔG = −1.7 kcal/ mol, Supplemental Fig. 3). In contrast, the introduction of two glutamate residues into TM2 (V735E &amp; I736E, EE mutant) is predicted to increase its transfer free energy by 3.3 kcal/ mol, which should significantly reduce its membrane integration efficiency (Supplemental Fig. 3A). The effects of the EE substitutions appear to be subtle in the context of the Lep protein (Supplemental Fig. 3B), though this likely reflects the limited dynamic range of this translocation assay.<sup><a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> Nevertheless, these results clearly show that the LL and EE mutations alter the translocon-mediated membrane integration of TM2.</p><p id="p-15">To determine whether the cotranslational membrane integration of TM2 impacts translational recoding, we measured the effects of these substitutions on ribosomal frameshifting. PRF is most commonly measured using dual-luciferase reporters, which fuse luciferase domains to the 5’ (renilla luciferase, 0-frame) and 3’ (firefly luciferase, −1 frame) of the gene of interest. The activity of firefly luciferase serves as a reporter for −1PRF and is normalized relative to translational efficiency based on the activity of renilla luciferase. Current versions of these reporter constructs contain self-cleaving 2A segments that release these luciferase domains from the polypeptide of interest.<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> While the 2A elements are likely to somewhat efficiently release each fusion domain at some point during translation, the introduction of a soluble N-terminal domain could potentially compromise the fidelity of SRP-mediated targeting of the nascent chain to the translocon. To preserve the integrity of the signal peptide, we generated a series of reporter constructs in which translation begins at the native E3 signal peptide and continues until the ribosome reaches a fluorescent mKate fusion domain that is encoded in the −1 reading frame downstream from the PRF site (Supplemental Fig. 4). To control for variations in transfection efficiency at the single-cell level, we included a downstream IRES cassette that drives the bicistronic expression of GFP from the reporter transcript. Reporter constructs encoding TM2 variants of the polyprotein were expressed in HEK293T cells, and cellular mKate intensities were quantitatively compared across cells within a discrete range of IRES-GFP intensities by flow cytometry (Supplemental Fig. 5). The average mKate intensity among cells expressing a reporter construct bearing mutations that knock out the native ribosomal slip-site (UUUUUUA→ GUUCCUA, SSKO) is 79 ± 5% lower than that that among cells expressing the WT form of the reporter (n = 3, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>), which confirms that mKate intensities reflect the efficiency of −1PRF. The EE substitutions in TM2 decrease mKate intensity by 61 ± 16% relative to WT (n = 3, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4A</a>). In contrast, the LL substitutions increase the mKate intensity by 30 ± 11% (n = 3, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4A</a>). It should be emphasized that each of these reporters contain both the native slip-site and stem loop regions, and that these mutations do not alter their sequences. Thus, these findings demonstrate that −1PRF is sensitive to mutations that impact the membrane integration efficiency of TM2. Together, biochemical evidence suggests that TM2 is inefficiently recognized by the translocon (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Fig. 1E</a>), and cellular topology reporters suggest that this segment is most often localized within the cytosol (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig. 2</a>). Nevertheless, mutagenesis reveals that the propensity of the nascent chain to form a secondary topomer is positively correlated with −1PRF. These results are therefore suggestive of a mechanistic link between topogenesis and −1PRF.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Influence of Sequence Modifications on &#x2212;1 Programmed Ribosomal Frameshifting. A fragment of the SINV structural polyprotein containing an mKate fused in the &#x2212;1 reading frame downstream from the ribosomal slip-site was used to compare the effects of sequence modifications on &#x2212;1PRF levels in HEK293T cells. A) &#x2212;1PRF reporter constructs containing the WT (black), the EE double mutant (red), and the LL double mutant (blue) TM2 seqeunce were expressed in HEK293T cells, and and cellular fluorescence intensities were analyzed by flow cytometry. A histogram depicts the mKate intensities assoicated with 3,000 cells expressing each reporter construct at a consistent expression level as judged by the intensity of a bicistronic expression reporter. These trends were consistently observed across three independent biological replicates. B) &#x2212;1PRF reporter constructs containing a series of deletions and G/S linker insertions within the loop between TMs 2 &amp; 3 were expressed in HEK293T cells, and and cellular fluorescence intensities were compared at a consistent expression level by flow cytometry. The mean fluorescence intensities associated with each construct were normalized relative to that of the WT construct, and the average relative fluorescence intensity values are plotted against the number of residues separating the C-terminal residue of TM2 from the slip site. Values represent the average relative fluorescence intensity from three independent biological replicates, and error bars reflect the standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1949721169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Influence of Sequence Modifications on &#x2212;1 Programmed Ribosomal Frameshifting.&lt;/span&gt; A fragment of the SINV structural polyprotein containing an mKate fused in the &#x2212;1 reading frame downstream from the ribosomal slip-site was used to compare the effects of sequence modifications on &#x2212;1PRF levels in HEK293T cells. A) &#x2212;1PRF reporter constructs containing the WT (black), the EE double mutant (red), and the LL double mutant (blue) TM2 seqeunce were expressed in HEK293T cells, and and cellular fluorescence intensities were analyzed by flow cytometry. A histogram depicts the mKate intensities assoicated with 3,000 cells expressing each reporter construct at a consistent expression level as judged by the intensity of a bicistronic expression reporter. These trends were consistently observed across three independent biological replicates. B) &#x2212;1PRF reporter constructs containing a series of deletions and G/S linker insertions within the loop between TMs 2 &amp; 3 were expressed in HEK293T cells, and and cellular fluorescence intensities were compared at a consistent expression level by flow cytometry. The mean fluorescence intensities associated with each construct were normalized relative to that of the WT construct, and the average relative fluorescence intensity values are plotted against the number of residues separating the C-terminal residue of TM2 from the slip site. Values represent the average relative fluorescence intensity from three independent biological replicates, and error bars reflect the standard deviation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F4.medium.gif" width="247" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F4.medium.gif" width="247" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Influence of Sequence Modifications on −1 Programmed Ribosomal Frameshifting.</span><p id="p-16" class="first-child">A fragment of the SINV structural polyprotein containing an mKate fused in the −1 reading frame downstream from the ribosomal slip-site was used to compare the effects of sequence modifications on −1PRF levels in HEK293T cells. A) −1PRF reporter constructs containing the WT (black), the EE double mutant (red), and the LL double mutant (blue) TM2 seqeunce were expressed in HEK293T cells, and and cellular fluorescence intensities were analyzed by flow cytometry. A histogram depicts the mKate intensities assoicated with 3,000 cells expressing each reporter construct at a consistent expression level as judged by the intensity of a bicistronic expression reporter. These trends were consistently observed across three independent biological replicates. B) −1PRF reporter constructs containing a series of deletions and G/S linker insertions within the loop between TMs 2 &amp; 3 were expressed in HEK293T cells, and and cellular fluorescence intensities were compared at a consistent expression level by flow cytometry. The mean fluorescence intensities associated with each construct were normalized relative to that of the WT construct, and the average relative fluorescence intensity values are plotted against the number of residues separating the C-terminal residue of TM2 from the slip site. Values represent the average relative fluorescence intensity from three independent biological replicates, and error bars reflect the standard deviation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Impact of Nascent Chain Forces on Ribosomal Frameshifting</h3><p id="p-17">The apparent link between cotranslational folding and ribosomal frameshifting has implications for the mechanism of −1PRF in the SINV structural polyprotein. The portion of the transcript containing the EE and LL mutations is over 100 nucleotides upstream from the ribosomal slip site, and should therefore not perturb the stimulatory RNA structures that are currently believed to modulate −1PRF.<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup> These mutations instead alter the portion of the nascent chain that falls just outside of the ribosomal exit tunnel during −1PRF, which suggests the nascent chain itself may stimulate frameshifting. Though it has yet to be implicated in ribosomal frameshifting, the cotranslational membrane integration and/ or folding of the nascent chain is known to generate tension on the ribosome.<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>-<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup> Furthermore, the C-terminal residue of TM2 is positioned 45 residues upstream of the slip site, which corresponds to a distance that should maximize the tension on the nascent chain at the moment the slip site occupies the ribosomal active site.<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> Previous investigations have demonstrated that the force generated by the membrane integration of the nascent chain is sharply dependent upon this spacing.<sup><a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a></sup> Therefore, to assess the potential role of this force in ribosomal frameshifting, we generated a set of SINV −1PRF reporter constructs (used in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4A</a>) containing a series of insertions and deletions that alter the distance between TM2 and the ribosomal slip site (see Supplemental Table 1). Reporter constructs were then expressed in HEK293T cells, and −1PRF reporter intensities were quantitatively compared at a uniform expression level by flow cytometry (Supplemental Fig. 5). A comparison of reporter intensities reveals that −1PRF is maximized at the WT distance of 45 residues (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4B</a>). In all cases, deletions and insertions that change the distance between TM2 and the slip site significantly reduce the efficiency of −1PRF (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4B</a>). Moreover, the insertion of a 10-residue G/S linker decreases the intensity of the frameshift reporter by 76 ± 8% (n = 3), which suggests the membrane integration of TM2 is likely to be the primary driver of −1PRF within the SINV structural polyprotein. Nevertheless, the deletion of the region containing the stimulatory RNA hairpin downstream of the slip-site abrogates PRF (Supplemental Fig. 6), which suggests that both the hairpin and TM2 are needed for efficient PRF. Together these findings suggest topological signals within the SINV structural polyprotein generate a mechanical force that stimulates −1PRF.</p><p id="p-18">To further explore the interplay between sequence, topology, and force, we carried out coarse-grained molecular dynamics (CGMD) simulations of the translation and translocon-mediated membrane integration of the nascent structural polyprotein.<sup><a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup> In these simulations, three-residue segments of the nascent chain are modeled as individual beads with physicochemical properties based on their constituent amino acids. New beads are translated at a rate of 5 amino acids per second, and emerge from the ribosome-translocon complex into an environment with an implicit representation of the bilayer and cytosol (Supplemental Movie 1).<sup><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup> These simulations were previously found to sufficiently recapitulate several aspects of cotranslational membrane protein folding including the formation of topological isomers and the generation of tension on the nascent chain.<sup><a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup> CGMD simulations of SINV polyprotein biosynthesis suggest the nascent chain samples several different topological isomers (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>), and that its topological heterogeneity persists after the polyprotein has cleared the translocon. TM2 undergoes translocon-mediated membrane integration (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>, right) in only 44 ± 4% of the CGMD trajectories in which TM1 is correctly integrated into the membrane. Consistent with expectations, CGMD simulations suggest that the membrane integration efficiency of TM2 is enhanced by the LL mutations (51 ± 4%) and reduced by the EE mutations (11 ± 3%). This finding provides additional evidence that the topological frustration within this domain (see <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3</a>) arises primarily from its marginal hydrophobicity.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coarse-Grained Molecular Dynamics Simulations of Polyprotein Biosynthesis. Coarse-grained molecular dynamics (CGMD) simulations were carried out to simulate biosynthesis of a series of SINV structural polyprotein variants, and the pulling force on the nascent chain was calculated while the ribosomal slip site occupies the ribosomal P-site. A) Representative snapshots from CGMD simulations are shown during translation at the slip-site, which is the point during elongation at which pulling forces on the nascent chain are measured. The ribosomal exit tunnel is shown in brown. The translocon is shown in gray and its lateral gate is highlighted in green. The nascent chain is shown in blue, except for the portions that correspond to TMs 1 &amp; 2, which are highlighted in orange and red, respectively. The snapshot on the left depicts a representative trajectory in which TM2 passes through the lateral gate and into the membrane. The snapshot on the right depicts a representative trajectory in which TM2 fails to enter the translocon. B) &#x2212;1PRF fluorescence reporter (mKate) intensity values for cells expressing a series of modified polyprotein variants were normalized relative to wild-type and plotted against the corresponding mean force measurements calculated from 560 CGMD trajectories. Error bars reflect the standard deviations from three independent biological replicates. The identity of each variant (see Supplemental Table 1) along with a linear fit of the data (dashes) are shown for reference (Pearson&#x2019;s R = 0.75). C) Pulling force measurements are compared among topological isomers for the WT (purple), LL (blue), and EE (red) polyprotein variants in which the number of TM2 residues (or beads) located within the translocon was found to vary. D) A histogram depicts the number of TM2 residues (or beads) within the translocon among the conformational trajectories sampled during biosynthesis of the WT (purple), LL (blue), and EE (red) variants of the SINV polyprotein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1949721169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Coarse-Grained Molecular Dynamics Simulations of Polyprotein Biosynthesis.&lt;/span&gt; Coarse-grained molecular dynamics (CGMD) simulations were carried out to simulate biosynthesis of a series of SINV structural polyprotein variants, and the pulling force on the nascent chain was calculated while the ribosomal slip site occupies the ribosomal P-site. A) Representative snapshots from CGMD simulations are shown during translation at the slip-site, which is the point during elongation at which pulling forces on the nascent chain are measured. The ribosomal exit tunnel is shown in brown. The translocon is shown in gray and its lateral gate is highlighted in green. The nascent chain is shown in blue, except for the portions that correspond to TMs 1 &amp; 2, which are highlighted in orange and red, respectively. The snapshot on the left depicts a representative trajectory in which TM2 passes through the lateral gate and into the membrane. The snapshot on the right depicts a representative trajectory in which TM2 fails to enter the translocon. B) &#x2212;1PRF fluorescence reporter (mKate) intensity values for cells expressing a series of modified polyprotein variants were normalized relative to wild-type and plotted against the corresponding mean force measurements calculated from 560 CGMD trajectories. Error bars reflect the standard deviations from three independent biological replicates. The identity of each variant (see Supplemental Table 1) along with a linear fit of the data (dashes) are shown for reference (Pearson&#x2019;s R = 0.75). C) Pulling force measurements are compared among topological isomers for the WT (purple), LL (blue), and EE (red) polyprotein variants in which the number of TM2 residues (or beads) located within the translocon was found to vary. D) A histogram depicts the number of TM2 residues (or beads) within the translocon among the conformational trajectories sampled during biosynthesis of the WT (purple), LL (blue), and EE (red) variants of the SINV polyprotein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F5.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F5.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Coarse-Grained Molecular Dynamics Simulations of Polyprotein Biosynthesis.</span><p id="p-19" class="first-child">Coarse-grained molecular dynamics (CGMD) simulations were carried out to simulate biosynthesis of a series of SINV structural polyprotein variants, and the pulling force on the nascent chain was calculated while the ribosomal slip site occupies the ribosomal P-site. A) Representative snapshots from CGMD simulations are shown during translation at the slip-site, which is the point during elongation at which pulling forces on the nascent chain are measured. The ribosomal exit tunnel is shown in brown. The translocon is shown in gray and its lateral gate is highlighted in green. The nascent chain is shown in blue, except for the portions that correspond to TMs 1 &amp; 2, which are highlighted in orange and red, respectively. The snapshot on the left depicts a representative trajectory in which TM2 passes through the lateral gate and into the membrane. The snapshot on the right depicts a representative trajectory in which TM2 fails to enter the translocon. B) −1PRF fluorescence reporter (mKate) intensity values for cells expressing a series of modified polyprotein variants were normalized relative to wild-type and plotted against the corresponding mean force measurements calculated from 560 CGMD trajectories. Error bars reflect the standard deviations from three independent biological replicates. The identity of each variant (see Supplemental Table 1) along with a linear fit of the data (dashes) are shown for reference (Pearson’s R = 0.75). C) Pulling force measurements are compared among topological isomers for the WT (purple), LL (blue), and EE (red) polyprotein variants in which the number of TM2 residues (or beads) located within the translocon was found to vary. D) A histogram depicts the number of TM2 residues (or beads) within the translocon among the conformational trajectories sampled during biosynthesis of the WT (purple), LL (blue), and EE (red) variants of the SINV polyprotein.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">To evaluate the connection between pulling forces on the nascent chain and frameshifting, we measured the tension on the nascent chain at the point of elongation when the slip site occupies the ribosomal active site (Supplemental Movie 1).<sup><a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a></sup> Pulling forces were highest for the LL variant, which averaged 4.2 pN higher than the WT. In contrast, the EE mutations reduce the pulling force on the nascent chain by an average 2.3 pN relative to WT. These results are consistent with the hypothesis that differences in frameshifting arise from the effects of these mutations on the pulling force on the nascent chain. Simulations of polyprotein variants bearing insertions or deletions that alter the distance between TM2 and the ribosomal slip-site indicate that the native distance (45 residues) is nearly optimal for the transmission of pulling force through the nascent chain (Supplemental Table 1), which is consistent with the observed patterns in frameshifting (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4B</a>). Overall, we find the −1PRF efficiency associated with each polyprotein variant roughly scales with corresponding mean pulling force measurements from CGMD simulations (Pearson’s R = 0.75, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>), which strongly suggests that the pulling forces generated by the translocon-mediated membrane integration of the nascent chain stimulate −1PRF.</p><p id="p-21">An analysis of the spectrum of topological states sampled during translation reveals that the magnitude of the pulling force transmitted to the ribosome scales with the number of beads that occupy the translocon (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5C</a>). This finding suggests pulling forces are generated by the movement of hydrophobic transmembrane segments from the protein-conducting channel of the translocon to the hydrophobic membrane core, as has been established previously.<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">47</a></sup> This interpretation suggests differences in pulling forces arise from variations in the distribution of topological isomers that form during translation of these variants (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5D</a>). The LL mutant predominately samples conformations where the majority of TM2 beads are in the translocon (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5A</a>, right), whereas the EE mutant almost exclusively adopts conformations in which the majority of TM2 beads fall outside of the translocon and within the cytosol (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5A</a>, left). As passage through the translocon is a prerequisite for membrane integration, the relationship between pulling forces and residence of the nascent chain within the translocon is consistent with our model for structural polyprotein biogenesis (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3</a>) and confirms that the translocon-mediated membrane integration of TM2 stimulates −1PRF.</p></div><div id="sec-6" class="subsection"><h3>Survey of Frameshifting Elements among Alphavirus Structural Polyproteins</h3><p id="p-22">Our model suggests that the hydrophobicity of TM2 and its distance from the slip site are the key determinants of the −1PRF efficiency within the SINV structural polyprotein. To assess whether this mechanism is likely to be operative within other alphaviruses, we surveyed six related structural polyproteins for similar sequence elements. Sequence scans carried out with the ΔG predictor reveal that each form of the alphavirus structural polyprotein contains a marginally hydrophobic TM domain upstream from the ribosomal slip site. Predicted transfer free energies associated with the translocon-mediated membrane integration of these putative TM domains range from +1.4 to +2.7 kcal/ mol (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), which suggests the translocon-mediated membrane integration of these segments is likely to be inefficient. Consistent with this notion, CGMD simulations of the translation of these polyproteins indicate that the membrane integration efficiency of these segments ranges from 33-64% (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Furthermore, these marginally hydrophobic TM domains reside 44-52 residues upstream of their corresponding −1PRF sites (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), which suggests that the tension generated by their translocon-mediated membrane integration is likely to be propagated back to the slip site.<sup><a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">41</a></sup> Force measurements derived from CGMD simulations of polyprotein synthesis suggest that the tension in the nascent chain when the slip-site occupies the ribosome is comparable to or greater than the tension generated during translation of the SINV variants characterized herein (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Taken together, these findings suggest that this −1PRF mechanism is likely to be conserved across the alphavirus genus. Additional investigations are needed to determine how nascent chain- and RNA-mediated −1PRF mechanisms are balanced against one another, and how this mechanistic diversity ultimately influences viral evolution.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1112008/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1112008/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1112008" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Comparison of the Topological Properties of Alphavirus Structural Polyproteins</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-7"><h2 class="">Conclusions</h2><p id="p-23">Using an array of biochemical, cellular, and computational methods we show that the nascent SINV structural polyprotein forms a spectrum of topological intermediates during biosynthesis, and that −1PRF is primarily driven by the translocon-mediated membrane integration of a marginally hydrophobic TM domain within the E2 protein. We also provide evidence to suggest this mechanism is likely to be conserved across the alphavirus genus. To date, the mechanistic basis of −1PRF has been generally attributed to the kinetic effects of mechanochemical forces that are generated by structural elements within the mRNA. Indeed, we do find that PRF in the SINV structural polyprotein is dependent upon the RNA stem loop downstream of the slip-site (Supplemental Fig. 6). Nevertheless, it is clear that the forces generated by the translocon-mediated membrane integration of TM2 dramatically enhance the frameshifting efficiency. To our knowledge, the findings reported herein constitute the first instance in which forces generated by conformational transitions in the nascent polypeptide chain have been implicated in the efficiency of PRF. Though additional investigations are needed to elucidate how pulling forces in the nascent chain physically stimulate −1PRF, a causative role for tension in both the transcript and nascent chain seems plausible given that ribosomal frameshifting fundamentally arises from the movement of the tRNA with respect to the mRNA. It seems likely that cotranslational folding is one of many regulators, which include both host and viral effectors, that tune the net efficiency of PRF. This creates the potential for mechanistic diversity that could provide an evolutionary benefit for alphaviruses, as −1PRF is rendered tunable by either downstream or upstream mutations that impact the stability of the mRNA hairpin or the conformational ensemble of the nascent chain, respectively. This flexibility could also potentially provide the virus with a means of maintaining desired −1PRF levels in the presence host factors that globally regulate −1PRF through mRNA interactions.<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup></p><p id="p-24">It should be noted that the implications of these findings potentially extend beyond the realm of viral proteins. A wide variety of molecular transitions have been found to generate tension within the nascent chain including the folding of soluble domains near the ribosomal exit tunnel<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup> and the translocon-mediated membrane integration of nascent TM domains.<sup><a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">41</a></sup> These observations suggest the tension in the nascent chain should fluctuate as the structural features emerge from the ribosome (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a>), which may therefore provide the ribosome with a readout for the progress of cotranslational folding. In the case of the SINV structural polyprotein, the topological frustration within the nascent chain leads to the production of two competing topomers that generate distinct pulling forces on the ribosome in a manner that ultimately impacts the fidelity and processivity of translation (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6 B &amp; C</a>). Additional investigations are needed to explore the potential relevance of this type of cotranslational feedback to protein homeostasis. Indeed, interactions between the nascent chain and molecular chaperones are known to ratchet polypeptides across the membrane,<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup> and may therefore contribute to pulling forces. This could potentially account for the fact that the deletion of components of the ribosome-associated chaperone complex has been found to attenuate −1PRF in yeast.<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup> Future investigations are needed to evaluate the full range of −1PRF effectors and how these are potentially exploited for regulatory purposes.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interplay between Topology, Pulling Force, and Programmed Ribosomal Frameshifting. Cartoons depict the manner in which the translocon-mediated membrane integration of the nascent chain generates a fluctuation in pulling force that triggers PRF during synthesis of the SINV structural polyprotein. A) The pulling force generated by the translocon-mediated membrane integration of each TM domain generates a pulling force on the nascent chain that is maximized during the conjugation of the amino acid that lies &#x223C;45 residues upstream of the C-terminal residue of the TM domain. B) The translocon-mediated membrane integration of TM2 is marginally efficient, which results in the formation of two topologies during translation of the SINV polyprotein. TM2 most often fails to undergo translocon-mediated membrane integration, which results in the formation of a topology featuring only two TM domains (TMs 1 &amp; 3) in the form of the polyprotein containing the 6K protein. However, the translocon-mediated membrane integration of TM2 generates an alternative topology in the frameshifted form of the polyprotein containing the TF protein. C) A hypothetical plot of the fluctuations in the nascent chain pulling force during the translation of the two forms of the SINV structural polyprotein is shown. The translocon-mediated membrane integration of TM2 generates an extra pulling force on the nascent chain while the slip-site occupies the ribosomal P-site, which stimulates &#x2212;1PRF." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1949721169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Interplay between Topology, Pulling Force, and Programmed Ribosomal Frameshifting.&lt;/span&gt; Cartoons depict the manner in which the translocon-mediated membrane integration of the nascent chain generates a fluctuation in pulling force that triggers PRF during synthesis of the SINV structural polyprotein. A) The pulling force generated by the translocon-mediated membrane integration of each TM domain generates a pulling force on the nascent chain that is maximized during the conjugation of the amino acid that lies &#x223C;45 residues upstream of the C-terminal residue of the TM domain. B) The translocon-mediated membrane integration of TM2 is marginally efficient, which results in the formation of two topologies during translation of the SINV polyprotein. TM2 most often fails to undergo translocon-mediated membrane integration, which results in the formation of a topology featuring only two TM domains (TMs 1 &amp; 3) in the form of the polyprotein containing the 6K protein. However, the translocon-mediated membrane integration of TM2 generates an alternative topology in the frameshifted form of the polyprotein containing the TF protein. C) A hypothetical plot of the fluctuations in the nascent chain pulling force during the translation of the two forms of the SINV structural polyprotein is shown. The translocon-mediated membrane integration of TM2 generates an extra pulling force on the nascent chain while the slip-site occupies the ribosomal P-site, which stimulates &#x2212;1PRF.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F6.medium.gif" width="263" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F6.medium.gif" width="263" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/790444/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Interplay between Topology, Pulling Force, and Programmed Ribosomal Frameshifting.</span><p id="p-25" class="first-child">Cartoons depict the manner in which the translocon-mediated membrane integration of the nascent chain generates a fluctuation in pulling force that triggers PRF during synthesis of the SINV structural polyprotein. A) The pulling force generated by the translocon-mediated membrane integration of each TM domain generates a pulling force on the nascent chain that is maximized during the conjugation of the amino acid that lies ∼45 residues upstream of the C-terminal residue of the TM domain. B) The translocon-mediated membrane integration of TM2 is marginally efficient, which results in the formation of two topologies during translation of the SINV polyprotein. TM2 most often fails to undergo translocon-mediated membrane integration, which results in the formation of a topology featuring only two TM domains (TMs 1 &amp; 3) in the form of the polyprotein containing the 6K protein. However, the translocon-mediated membrane integration of TM2 generates an alternative topology in the frameshifted form of the polyprotein containing the TF protein. C) A hypothetical plot of the fluctuations in the nascent chain pulling force during the translation of the two forms of the SINV structural polyprotein is shown. The translocon-mediated membrane integration of TM2 generates an extra pulling force on the nascent chain while the slip-site occupies the ribosomal P-site, which stimulates −1PRF.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-8"><h2 class="">Methods</h2><div id="sec-9" class="subsection"><h3>Computational Predictions of Topological Energetics</h3><p id="p-26">The energetics associated with the translocon-mediated membrane integration of the nascent structural polyprotein were carried out using the ΔG predictor (<a href="http://dgpred.cbr.su.se/">http://dgpred.cbr.su.se/</a>).<sup><a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">34</a></sup> These predictions are generated using a window scan function that sums depth-dependent free energies associated with the transfer of amino acids from the translocon to the ER membrane.<sup><a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a></sup> Full sequence scans of varying window size were used to compare the predicted transfer free energies associated with each segment within the polyprotein in order to identify the segments that are most likely to undergo translocon-mediated membrane integration.</p></div><div id="sec-10" class="subsection"><h3>Plasmid Preparation and Mutagenesis</h3><p id="p-27">Chimeric Lep genes were generated in the context of a pGEM-based Lep expression vector<sup><a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">34</a></sup> that was kindly provided by the laboratory of Gunnar von Heijne. Putative TM domains of interest were introduced into the H-segment position of this Lep construct using Gibson assembly. To probe the topology of and ribosomal frameshifting within the SINV structural polyprotein, a portion of the polyprotein containing the E3, E2, and 6K/ TF proteins was first introduced downstream from the cytomegalovirus (CMV) promoter within a pcDNA5 vector using Gibson assembly. To produce a series of reporter constructs for polyprotein topology, Gibson assembly was then used to replace the portion of the polyprotein gene downstream from each putative TM domain with a genetic cassette containing a 10-residue G/S linker, a glycosylatable eGFP gene,<sup><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a></sup> an internal ribosomal entry site (IRES), and an mKate gene, respectively (Supplemental Fig. 1).</p><p id="p-28">To produce a series of reporter constructs for ribosomal frameshifting, Gibson assembly was used to replace the portion of the polyprotein gene that falls 100 basepairs downstream from the ribosomal slips site in the 6K gene with a cassette containing an mKate gene in the −1 reading frame followed by an IRES and a GFP gene (Supplemental Fig. 4). The frameshift reporter (mKate) was fused 100 nucleotides downstream from the slip site in order to avoid disrupting the stimulatory RNA hairpin downstream from the slip sites.<sup><a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></sup> In addition to avoiding potential issues related to the impact of fusion domains on SRP-targeting of the nascent chain, this design also avoids recently described artefacts associated with previous generations of the dual-luciferase reporter system in two ways.<sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup> First, the transcripts of our fluorescent expression reporter (eGFP) does not contain any cryptic splice sites. Second, the fluorescent −1PRF reporter protein (mKate) is liberated from the mutated portion of the nascent polypeptide through a native proteolytic cleavage site between the E2 and 6K/TF protein. Thus, mutations within TM2 should not impact the stability and/ or turnover of mKate. An IRES-eGFP cassette was also incorporated into the downstream portion of the reporter transcript in order to facilitate comparisons of reporter intensities across cells with uniform expression level. Site directed mutagenesis was used to introduce mutations into these constructs. Insertions and deletions were introduced using In-Fusion cloning (Takara, Mountain View, CA).</p></div><div id="sec-11" class="subsection"><h3>In vitro Translation of Chimeric Lep Proteins</h3><p id="p-29">Chimeric Lep proteins were generated by <em>in vitro</em> translation as was described previously.<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup> Briefly, mRNA for each chimeric Lep protein was produced from plasmids using the RiboMAX RNA production system in accordance with the manufacturer’s instructions (Promega, Madison, WI). Lep proteins were then produced from mRNA by <em>in vitro</em> translation using rabbit reticulocyte lysate (Promega, Madison, WI) supplemented with canine pancreatic rough microsomes (tRNA probes, College Station, TX) and EasyTag <sup><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></sup>S methionine (PerkinElmer, Waltham, MA). <em>In vitro</em> translation reactions were then diluted 1:4 into 1X SDS PAGE sample buffer and separated using a 12% SDS PAGE gel. To image radioactive translation products, PAGE gels were then dried, exposed to a phosphorimaging screen, and imaged using a Typhoon Imager (GE Healthcare, New York, NY). Image J software was then employed to process the data quantify the glycosylation state of each construct by densitometry.</p></div><div id="sec-12" class="subsection"><h3>Expression of Fluorescent Reporter Constructs</h3><p id="p-30">Flow cytometry was used to compare the fluorescence intensity profiles of HEK293T cells expressing topology and frameshifting reporter constructs. Briefly, HEK293T cells were grown in DMEM (Gibco, Grand Island, NY) containing 10% fetal bovine serum (Corning, Corning, NY) and a penicillin/ streptomycin antibiotic supplement (Gibco, Grand Island, NY) in an incubator containing 5% CO2 at 37° C. Reporter constructs were transiently expressed using lipofectamine 3000 (Invitrogen, Carlsbad CA) in accordance with the manufacturer’s instructions. Cells were harvested two days post-transfection and analyzed using a BD LSRII flow cytometer (BD Biosciences, Franklin Lakes, NJ). Cellular fluorescence profiles were analyzed using FlowJo software (Treestar, Ashland, OR). To compare cellular reporter intensities among cells with a uniform expression level, analysis of reporter intensity levels was restricted to cells that fell within a defined, uniform range of IRES-mKate or IRES-GFP expression reporters intensities. An example of the hierarchical gating strategy employed herein is shown in Supplemental Figs. 2 &amp; 5.</p></div><div id="sec-13" class="subsection"><h3>Course Grained Simulations of Polyprotein Translation</h3><p id="p-31">Coarse-grained molecular dynamics (CGMD) simulations are based on a previously developed and tested approach.<sup><a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">45</a></sup> Briefly, simulations are carried out in the context of a coarsened representation of the ribosome exit tunnel, Sec translocon, and nascent chain. The nascent chain is composed of beads that each represent three amino acids, and new beads are sequentially added to the nascent chain in order to explicitly simulate translation. Translation occurs at a rate of 5 aa/s, which mimics the rate of translation by eukaryotic ribosomes. Each bead interacts with the translocon, ribosome, and other beads in a manner that is dependent on the hydrophobicity and charge of its composite amino acids. Interactions with the solvent and lipid bilayer are modelled implicitly. The ribosome and Sec translocon are fixed in place, with the exception of the lateral gate of the translocon, which stochastically switches between the open and closed conformations in a manner dependent on the free energy difference between the two configurations.</p><p id="p-32">The geometries of the ribosome and translocon are based on cryo-EM structures.<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup> Residue-specific interactions have been parameterized based on over 200 µs of simulations with the MARTINI forcefield. Fitting is performed using a Bayesian uncertainty quantification framework.<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup> This approach represents an update relative previous published methodology, and the new parameters utilized herein are included in Supplemental Table 2. All other parameters necessary to describe the system are available in previous published work.<sup><a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">45</a></sup> Integration is performed using overdamped Langevin dynamics with a diffusion coefficient of 253 nm<sup>2</sup>/s and a time step of 300 ns. Despite the significant simplifications involved in this model, the CGMD model has proven accurate in capturing the integration probabilities, topology distributions, and forces experienced in previous studies.<sup><a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">45</a>,<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">47</a></sup></p><p id="p-33">In order to obtain the distribution of topologies for various polyprotein mutants, the translation and integration of each sequence was simulated 560 times. In order to reduce computational cost, simulations only included the first three TMDs of the alphavirus polyprotein. In order to focus on the topological preferences of TM2, restraints were applied to enforce that TM1 adopts its native topological orientation. Pulling force measurements were performed by pausing translation when the −1PRF site resides within the ribosomal peptidyl transfer center. During this pause, the final bead was fixed in place and the force on the bead exerted by the nascent chain was measured along the translocon channel axis. Due to the truncation of the exit tunnel in our model, the final bead corresponds to the amino acids 27 residues N-terminal of the −1PRF site. Translation is paused for 3 seconds, which is equivalent to the time it would take to translate 5 beads. This relatively short time window ensures that the distribution of polyprotein topologies is not affected by the pause. During this window, pulling forces were measured at a rate of 333 frames per second. In order to sample a wide range of topologies and conformations, each mutant was independently simulated 560 times. This protocol is analogous to force measurements performed in previous work, with exception of a shortened pausing duration.<sup><a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">47</a></sup></p></div></div><div class="section" id="sec-15"><h2 class="">Author Contributions</h2><p id="p-35">J.P.S, S.M., and T.F.M. III designed the experiments. H.R.H., L.M.C., W.D.P, and V.N. produced the genetic constructs and carried out the biochemical and cellular experiments. M.H.Z. carried out and analyzed CGMD simulations. J.P.S. wrote the manuscript with editorial input from S.M. and T.F.M. III.</p></div><div class="section" id="sec-16"><h2 class="">Competing Interests</h2><p id="p-36">The authors declare no conflict of interest.</p></div><div class="section" id="sec-17"><h2 class="">Data Availability</h2><p id="p-37">The datasets described herein are available from the corresponding authors upon request. Accession numbers are provided in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>.</p></div><div class="section" id="sec-18"><h2 class="">Code Availability</h2><p id="p-38">All code relating to the CGMD simulations detailed herein have been previously detailed in references 39 and 41, and are freely available upon request.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">We thank Renuka Kudva and Gunnar von Heijne for scientific input. We thank David Giedroc and Jonathan Dinman for editorial input. We thank Christiane Hassel and the Indiana University Bloomington Flow Cytometry Core Facility for their experimental support. This research was supported in part by grant from the National Institute of Allergy and Infectious Diseases (NIAID) to J.P.S. and S. M. (R21AI142383) as well as a grant from the National Institute of General Medical Sciences to T. F. M. (R01GM125063).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">Text has been updated for clarification.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.1" data-doi="10.1016/j.tibs.2015.03.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caliskan</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M. V.</span></span> <span class="cit-article-title">Changed in translation: mRNA recoding by −1 programmed ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Trends Biochem Sci</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">274</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem%2BSci%26rft.volume%253D40%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2015.03.006%26rft_id%253Dinfo%253Apmid%252F25850333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2015.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25850333&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.2" data-doi="10.1073/pnas.1905258116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halma</span>,  <span class="cit-name-given-names">M. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">D. B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Capellano</span>,  <span class="cit-name-given-names">T. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neupane</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Woodside</span>,  <span class="cit-name-given-names">M. T.</span></span> <span class="cit-article-title">Complex dynamics under tension in a high-efficiency frameshift stimulatory structure</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">116</span>, <span class="cit-fpage">19500</span>–<span class="cit-lpage">19505</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1905258116%26rft_id%253Dinfo%253Apmid%252F31409714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzM5LzE5NTAwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNzkwNDQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.3" data-doi="10.1038/nature04735"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Namy</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moran</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stuart</span>,  <span class="cit-name-given-names">D. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">R. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">A mechanical explanation of RNA pseudoknot function in programmed ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">441</span>, <span class="cit-fpage">244</span>–<span class="cit-lpage">247</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNamy%26rft.auinit1%253DO.%26rft.volume%253D441%26rft.issue%253D7090%26rft.spage%253D244%26rft.epage%253D247%26rft.atitle%253DA%2Bmechanical%2Bexplanation%2Bof%2BRNA%2Bpseudoknot%2Bfunction%2Bin%2Bprogrammed%2Bribosomal%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04735%26rft_id%253Dinfo%253Apmid%252F16688178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16688178&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237418800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-790444v2.4" data-doi="10.1073/pnas.1403457111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A frameshifting stimulatory stem loop destabilizes the hybrid state and impedes ribosomal translocation</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">5538</span>–<span class="cit-lpage">5543</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1403457111%26rft_id%253Dinfo%253Apmid%252F24706807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE1LzU1MzgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi83OTA0NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-790444v2.5" data-doi="10.1016/j.cell.2014.04.041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caliskan</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katunin</span>,  <span class="cit-name-given-names">V. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belardinelli</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M. V.</span></span> <span class="cit-article-title">Programmed −1 frameshifting by kinetic partitioning during impeded translocation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>, <span class="cit-fpage">1619</span>–<span class="cit-lpage">1631</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1619%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.04.041%26rft_id%253Dinfo%253Apmid%252F24949973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.04.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24949973&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340941900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-790444v2.6" data-doi="10.1038/nature13428"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Chen</span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dynamic pathways of −1 translational frameshifting</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>, <span class="cit-fpage">328</span>–<span class="cit-lpage">332</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13428%26rft_id%253Dinfo%253Apmid%252F24919156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24919156&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340508200039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.7" data-doi="10.1126/sciadv.aax6969"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">OLoughlin</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Puglisi</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">The energy landscape of −1 ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Sci Adv</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">eaax6969</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DChoi%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253Deaax6969%26rft.epage%253Deaax6969%26rft.atitle%253DThe%2Benergy%2Blandscape%2Bof%2B-1%2Bribosomal%2Bframeshifting%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aax6969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNi8xL2VhYXg2OTY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNzkwNDQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korniy</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samatova</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anokhina</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M. V.</span></span> <span class="cit-article-title">Mechanisms and biomedical implications of −1 programmed ribosome frameshifting on viral and bacterial mRNAs</span>. <abbr class="cit-jnl-abbrev">FEBS lett</abbr> <span class="cit-vol">593</span>, <span class="cit-fpage">1468</span>–<span class="cit-lpage">1482</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2Blett%26rft.volume%253D593%26rft.spage%253D1468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.9"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Bock</span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Thermodynamic control of −1 programmed ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">4598</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D4598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.10" data-doi="10.1038/nature13429"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belew</span>,  <span class="cit-name-given-names">A. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13429%26rft_id%253Dinfo%253Apmid%252F25043019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25043019&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340508200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-790444v2.11" data-doi="10.1038/ncomms15582"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Napthine</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Protein-directed ribosomal frameshifting temporally regulates gene expression</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">15582</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D15582%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15582%26rft_id%253Dinfo%253Apmid%252F28593994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28593994&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Regulation of HIV-1 Gag-Pol Expression by Shiftless, an Inhibitor of Programmed −1 Ribosomal Frameshifting</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">625</span>–<span class="cit-lpage">635 e614</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramsey</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Disentangling the Frames, the State of Research on the Alphavirus 6K and TF Proteins</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">228</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D9%26rft.spage%253D228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.14" data-doi="10.3390/v10020089"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">R. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">J. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kielian</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">The Alphavirus Exit Pathway: What We Know and What We Wish We Knew</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">E89</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253DE89%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv10020089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v10020089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lobigs</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">H. X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Garoff</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Function of Semliki Forest virus E3 peptide in virus assembly: replacement of E3 with an artificial signal peptide abolishes spike heterodimerization and surface expression of E1</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">64</span>, <span class="cit-fpage">4346</span>–<span class="cit-lpage">4355</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLobigs%26rft.auinit1%253DM%26rft.volume%253D64%26rft.issue%253D9%26rft.spage%253D4346%26rft.epage%253D4355%26rft.atitle%253DFunction%2Bof%2BSemliki%2BForest%2Bvirus%2BE3%2Bpeptide%2Bin%2Bvirus%2Bassembly%253A%2Breplacement%2Bof%2BE3%2Bwith%2Ban%2Bartificial%2Bsignal%2Bpeptide%2Babolishes%2Bspike%2Bheterodimerization%2Band%2Bsurface%2Bexpression%2Bof%2BE1.%26rft_id%253Dinfo%253Apmid%252F2200886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY0LzkvNDM0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2Lzc5MDQ0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.16" data-doi="10.1006/viro.1996.0229"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mulvey</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">D. T.</span></span> <span class="cit-article-title">Assembly of the Sindbis virus spike protein complex</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">219</span>, <span class="cit-fpage">125</span>–<span class="cit-lpage">132</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DMulvey%26rft.auinit1%253DM.%26rft.volume%253D219%26rft.issue%253D1%26rft.spage%253D125%26rft.epage%253D132%26rft.atitle%253DAssembly%2Bof%2Bthe%2BSindbis%2Bvirus%2Bspike%2Bprotein%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1996.0229%26rft_id%253Dinfo%253Apmid%252F8623521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1996.0229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8623521&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UJ53200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.17" data-doi="10.1186/1743-422X-5-108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>,  <span class="cit-name-given-names">A. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">B. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleeton</span>,  <span class="cit-name-given-names">M. N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-article-title">Discovery of frameshifting in Alphavirus 6K resolves a 20-year enigma</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">108</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DFirth%26rft.auinit1%253DA.%2BE.%26rft.volume%253D5%26rft.spage%253D108%26rft.epage%253D108%26rft.atitle%253DDiscovery%2Bof%2Bframeshifting%2Bin%2BAlphavirus%2B6K%2Bresolves%2Ba%2B20-year%2Benigma.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-5-108%26rft_id%253Dinfo%253Apmid%252F18822126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-5-108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18822126&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.18" data-doi="10.1128/JVI.03453-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hallengard</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Novel attenuated Chikungunya vaccine candidates elicit protective immunity in C57BL/6 mice</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">2858</span>–<span class="cit-lpage">2866</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03453-13%26rft_id%253Dinfo%253Apmid%252F24371047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzUvMjg1OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2Lzc5MDQ0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-790444v2.19" data-doi="10.1128/JVI.03192-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Effects of an In-Frame Deletion of the 6k Gene Locus from the Genome of Ross River Virus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>, <span class="cit-fpage">4150</span>–<span class="cit-lpage">4159</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.03192-15</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03192-15%26rft_id%253Dinfo%253Apmid%252F26865723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjkwLzgvNDE1MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2Lzc5MDQ0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-790444v2.20" data-doi="10.1128/JVI.00449-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Functional characterization of the alphavirus TF protein</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">8511</span>–<span class="cit-lpage">8523</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00449-13%26rft_id%253Dinfo%253Apmid%252F23720714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xNS84NTExIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNzkwNDQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">K. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones-Burrage</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maury</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">TF protein of Sindbis virus antagonizes host type I interferon responses in a palmitoylation-dependent manner</span>. <abbr class="cit-jnl-abbrev">Virology, In press</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.22" data-doi="10.1016/j.jmb.2010.01.044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">B. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firth</span>,  <span class="cit-name-given-names">A. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-article-title">Frameshifting in alphaviruses: a diversity of 3’ stimulatory structures</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">397</span>, <span class="cit-fpage">448</span>–<span class="cit-lpage">456</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DChung%26rft.auinit1%253DB.%2BY.%26rft.volume%253D397%26rft.issue%253D2%26rft.spage%253D448%26rft.epage%253D456%26rft.atitle%253DFrameshifting%2Bin%2Balphaviruses%253A%2Ba%2Bdiversity%2Bof%2B3%2527%2Bstimulatory%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2010.01.044%26rft_id%253Dinfo%253Apmid%252F20114053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2010.01.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20114053&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.23" data-doi="10.1074/jbc.RA118.005606"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kendra</span>,  <span class="cit-name-given-names">J. A</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Functional and structural characterization of the chickungunya virus translational recoding signals</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">293</span>, <span class="cit-fpage">17536</span>–<span class="cit-lpage">17545</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.RA118.005606%26rft_id%253Dinfo%253Apmid%252F30242123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTMvNDUvMTc1MzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi83OTA0NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.24" data-doi="10.1128/JVI.06615-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">A. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sokoloski</span>,  <span class="cit-name-given-names">K. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Mutating conserved cysteines in the alphavirus e2 glycoprotein causes virus-specific assembly defects</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">3100</span>–<span class="cit-lpage">3111</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06615-11%26rft_id%253Dinfo%253Apmid%252F22238319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzYvMzEwMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2Lzc5MDQ0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.25" data-doi="10.1128/JVI.02000-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramsey</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Renzi</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trinidad</span>,  <span class="cit-name-given-names">J. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Palmitoylation of Sindbis Virus TF Protein Regulates Its Plasma Membrane Localization and Subsequent Incorporation into Virions</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">91</span>, <span class="cit-fpage">e02000</span>–<span class="cit-lpage">e2016</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D91%26rft.spage%253De02000%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02000-16%26rft_id%253Dinfo%253Apmid%252F27852864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/JVI.02000-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27852864&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.26" data-doi="10.1016/0042-6822(90)90209-A"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaedigk-Nitschko</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">The Sindbis virus 6K protein can be detected in virions and is acylated with fatty acids</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">175</span>, <span class="cit-fpage">274</span>–<span class="cit-lpage">281</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DGaedigk-Nitschko%26rft.auinit1%253DK.%26rft.volume%253D175%26rft.issue%253D1%26rft.spage%253D274%26rft.epage%253D281%26rft.atitle%253DThe%2BSindbis%2Bvirus%2B6K%2Bprotein%2Bcan%2Bbe%2Bdetected%2Bin%2Bvirions%2Band%2Bis%2Bacylated%2Bwith%2Bfatty%2Bacids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252890%252990209-A%26rft_id%253Dinfo%253Apmid%252F2408229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(90)90209-A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2408229&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.27" data-doi="10.1074/jbc.M411946200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Politis</span>,  <span class="cit-name-given-names">E. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roth</span>,  <span class="cit-name-given-names">A. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">N. G.</span></span> <span class="cit-article-title">Transmembrane topology of the protein palmitoyl transferase Akr1</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">280</span>, <span class="cit-fpage">10156</span>–<span class="cit-lpage">10163</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M411946200%26rft_id%253Dinfo%253Apmid%252F15632165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMTEvMTAxNTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi83OTA0NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liljestrom</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Garoff</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Internally located cleavable signal sequences direct the formation of Semliki Forest virus membrane proteins from a polyprotein precursor</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">65</span>, <span class="cit-fpage">147</span>–<span class="cit-lpage">154</span> (<span class="cit-pub-date">1991</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLiljestrom%26rft.auinit1%253DP%26rft.volume%253D65%26rft.issue%253D1%26rft.spage%253D147%26rft.epage%253D154%26rft.atitle%253DInternally%2Blocated%2Bcleavable%2Bsignal%2Bsequences%2Bdirect%2Bthe%2Bformation%2Bof%2BSemliki%2BForest%2Bvirus%2Bmembrane%2Bproteins%2Bfrom%2Ba%2Bpolyprotein%2Bprecursor.%26rft_id%253Dinfo%253Apmid%252F1985194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjY1LzEvMTQ3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNzkwNDQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.29" data-doi="10.1016/0092-8674(95)90516-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">R. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nucleocapsid and glycoprotein organization in an enveloped virus</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">80</span>, <span class="cit-fpage">621</span>–<span class="cit-lpage">630</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCheng%26rft.auinit1%253DR.%2BH.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D621%26rft.epage%253D630%26rft.atitle%253DNucleocapsid%2Band%2Bglycoprotein%2Borganization%2Bin%2Ban%2Benveloped%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990516-2%26rft_id%253Dinfo%253Apmid%252F7867069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90516-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7867069&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QJ60200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Implication for alphavirus host-cell entry and assembly indicated by a 3.5A resolution cryo-EM structure</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">5326</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D5326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.31" data-doi="10.1016/0092-8674(94)90155-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gafvelin</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">von Heijne</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Topological “frustration” in multispanning E. coli inner membrane proteins</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">77</span>, <span class="cit-fpage">401</span>–<span class="cit-lpage">412</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGafvelin%26rft.auinit1%253DG.%26rft.volume%253D77%26rft.issue%253D3%26rft.spage%253D401%26rft.epage%253D412%26rft.atitle%253DTopological%2B%2526quot%253Bfrustration%2526quot%253B%2Bin%2Bmultispanning%2BE.%2Bcoli%2Binner%2Bmembrane%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990155-4%26rft_id%253Dinfo%253Apmid%252F8181060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90155-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8181060&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NK97000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.32" data-doi="10.3390/v4030363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruch</span>,  <span class="cit-name-given-names">T. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Machamer</span>,  <span class="cit-name-given-names">C. E.</span></span> <span class="cit-article-title">The coronavirus E protein: assembly and beyond</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">363</span>–<span class="cit-lpage">382</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DRuch%26rft.auinit1%253DT.%2BR.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D382%26rft.atitle%253DThe%2Bcoronavirus%2BE%2Bprotein%253A%2Bassembly%2Band%2Bbeyond.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4030363%26rft_id%253Dinfo%253Apmid%252F22590676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4030363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22590676&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.33" data-doi="10.1038/nature06387"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hessa</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular code for transmembrane-helix recognition by the Sec61 translocon</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>, <span class="cit-fpage">1026</span>–<span class="cit-lpage">1030</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHessa%26rft.auinit1%253DT.%26rft.volume%253D450%26rft.issue%253D7172%26rft.spage%253D1026%26rft.epage%253D1030%26rft.atitle%253DMolecular%2Bcode%2Bfor%2Btransmembrane-helix%2Brecognition%2Bby%2Bthe%2BSec61%2Btranslocon.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06387%26rft_id%253Dinfo%253Apmid%252F18075582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18075582&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251579900075&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.34" data-doi="10.1038/nature03216"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hessa</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Recognition of transmembrane helices by the endoplasmic reticulum translocon</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">433</span>, <span class="cit-fpage">377</span>–<span class="cit-lpage">381</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHessa%26rft.auinit1%253DT.%26rft.volume%253D433%26rft.issue%253D7024%26rft.spage%253D377%26rft.epage%253D381%26rft.atitle%253DRecognition%2Bof%2Btransmembrane%2Bhelices%2Bby%2Bthe%2Bendoplasmic%2Breticulum%2Btranslocon.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03216%26rft_id%253Dinfo%253Apmid%252F15674282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15674282&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226546200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanova</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Site-directed mutations in the Sindbis virus E2 glycoprotein identify palmitoylation sites and affect virus budding</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">67</span>, <span class="cit-fpage">2546</span>–<span class="cit-lpage">2551</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DIvanova%26rft.auinit1%253DL%26rft.volume%253D67%26rft.issue%253D5%26rft.spage%253D2546%26rft.epage%253D2551%26rft.atitle%253DSite-directed%2Bmutations%2Bin%2Bthe%2BSindbis%2Bvirus%2BE2%2Bglycoprotein%2Bidentify%2Bpalmitoylation%2Bsites%2Band%2Baffect%2Bvirus%2Bbudding.%26rft_id%253Dinfo%253Apmid%252F8474160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY3LzUvMjU0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2Lzc5MDQ0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.36" data-doi="10.1006/viro.1998.9281"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ryan</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ivanova</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Effects of site-directed mutations of transmembrane cysteines in sindbis virus E1 and E2 glycoproteins on palmitylation and virus replication</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">249</span>, <span class="cit-fpage">62</span>–<span class="cit-lpage">67</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DRyan%26rft.auinit1%253DC.%26rft.volume%253D249%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D67%26rft.atitle%253DEffects%2Bof%2Bsite-directed%2Bmutations%2Bof%2Btransmembrane%2Bcysteines%2Bin%2Bsindbis%2Bvirus%2BE1%2Band%2BE2%2Bglycoproteins%2Bon%2Bpalmitylation%2Band%2Bvirus%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1998.9281%26rft_id%253Dinfo%253Apmid%252F9740777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1998.9281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9740777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.37" data-doi="10.1016/j.bbrc.2012.09.138"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Min</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Heijne</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Glycosylatable GFP as a compartment-specific membrane topology reporter</span>. <abbr class="cit-jnl-abbrev">Biochem Biophys Res Comm</abbr> <span class="cit-vol">427</span>, <span class="cit-fpage">780</span>–<span class="cit-lpage">784</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DLee%26rft.auinit1%253DH.%26rft.volume%253D427%26rft.issue%253D4%26rft.spage%253D780%26rft.epage%253D784%26rft.atitle%253DGlycosylatable%2BGFP%2Bas%2Ba%2Bcompartment-specific%2Bmembrane%2Btopology%2Breporter.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2012.09.138%26rft_id%253Dinfo%253Apmid%252F23047006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2012.09.138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23047006&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.38" data-doi="10.1002/pro.2483"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Virkki</span>,  <span class="cit-name-given-names">M. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Folding of Aquaporin 1: Multiple evidence that helix 3 can shift out of the membrane core</span>. <abbr class="cit-jnl-abbrev">Protein Sci</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">981</span>–<span class="cit-lpage">992</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BSci%26rft.volume%253D23%26rft.spage%253D981%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.2483%26rft_id%253Dinfo%253Apmid%252F24777974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pro.2483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24777974&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khan</span>,  <span class="cit-name-given-names">Y. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loughran</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-article-title">Contesting the Evidence for −1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5)-the HIV co-receptor</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.40" data-doi="10.1038/nsmb.2376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ismail</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hedman</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schiller</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">von Heijne</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">A biphasic pulling force acts on transmembrane helices during translocon-mediated membrane integration</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1018</span>–<span class="cit-lpage">1022</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DIsmail%26rft.auinit1%253DN.%26rft.volume%253D19%26rft.issue%253D10%26rft.spage%253D1018%26rft.epage%253D1022%26rft.atitle%253DA%2Bbiphasic%2Bpulling%2Bforce%2Bacts%2Bon%2Btransmembrane%2Bhelices%2Bduring%2Btranslocon-mediated%2Bmembrane%2Bintegration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2376%26rft_id%253Dinfo%253Apmid%252F23001004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23001004&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.41" data-doi="10.1016/j.febslet.2014.03.050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cymer</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ismail</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">von Heijne</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Weak pulling forces exerted on Nin-orientated transmembrane segments during co-translational insertion into the inner membrane of Escherichia coli</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr> <span class="cit-vol">588</span>, <span class="cit-fpage">1930</span>–<span class="cit-lpage">1934</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLett%26rft.volume%253D588%26rft.spage%253D1930%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2014.03.050%26rft_id%253Dinfo%253Apmid%252F24726730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2014.03.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24726730&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.42" data-doi="10.1073/pnas.1306787110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cymer</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">von Heijne</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Cotranslational folding of membrane proteins probed by arrest-peptide-mediated force measurements</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">110</span>, <span class="cit-fpage">14640</span>–<span class="cit-lpage">14645</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1306787110%26rft_id%253Dinfo%253Apmid%252F23959879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzM2LzE0NjQwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNzkwNDQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.43" data-doi="10.1126/science.1261909"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldman</span>,  <span class="cit-name-given-names">D. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mechanical force releases nascent chain-mediated ribosome arrest in vitro and in vivo</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>, <span class="cit-fpage">457</span>–<span class="cit-lpage">460</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoldman%26rft.auinit1%253DD.%2BH.%26rft.volume%253D348%26rft.issue%253D6233%26rft.spage%253D457%26rft.epage%253D460%26rft.atitle%253DMechanical%2Bforce%2Breleases%2Bnascent%2Bchain-mediated%2Bribosome%2Barrest%2Bin%2Bvitro%2Band%2Bin%2Bvivo%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1261909%26rft_id%253Dinfo%253Apmid%252F25908824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzMy80NTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi83OTA0NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.44" data-doi="10.1016/j.celrep.2012.08.039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">T. F.</span></span>, <span class="cit-edition">3rd</span>. <span class="cit-article-title">Long-timescale dynamics and regulation of Sec-facilitated protein translocation</span>. <abbr class="cit-jnl-abbrev">Cell rep</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">927</span>–<span class="cit-lpage">937</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BRep%26rft.aulast%253DZhang%26rft.auinit1%253DB.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D927%26rft.epage%253D937%26rft.atitle%253DLong-timescale%2Bdynamics%2Band%2Bregulation%2Bof%2Bsec-facilitated%2Bprotein%2Btranslocation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.08.039%26rft_id%253Dinfo%253Apmid%252F23084746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.08.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23084746&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Niesen</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Lehn</span>,  <span class="cit-name-given-names">R. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">T. F.</span></span>, <span class="cit-edition">3rd</span>. <span class="cit-article-title">Structurally detailed coarse-grained model for Sec-facilitated co-translational protein translocation and membrane integration</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e1005427</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D13%26rft.spage%253De1005427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.46" data-doi="10.7554/eLife.08697"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Lehn</span>,  <span class="cit-name-given-names">R. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">T. F.</span></span>, <span class="cit-edition">3rd</span>. <span class="cit-article-title">Regulation of multispanning membrane protein topology via post-translational annealing</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e08697</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.08697%26rft_id%253Dinfo%253Apmid%252F26408961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.08697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26408961&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47 in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Niesen</span>,  <span class="cit-name-given-names">M. J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller-Lucks</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hedman</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Heijne</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">T. F.</span></span>, <span class="cit-edition">3rd</span>. <span class="cit-article-title">Forces on Nascent Polypeptides during Membrane Insertion and Translocation via the Sec Translocon</span>. <abbr class="cit-jnl-abbrev">Biophys J</abbr> <span class="cit-vol">115</span>, <span class="cit-fpage">1885</span>–<span class="cit-lpage">1894</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys%2BJ%26rft.volume%253D115%26rft.spage%253D1885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48 in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.48" data-doi="10.1016/S0092-8674(00)80767-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matlack</span>,  <span class="cit-name-given-names">K. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Misselwitz</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plath</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rapoport</span>,  <span class="cit-name-given-names">T. A.</span></span> <span class="cit-article-title">BiP acts as a molecular ratchet during posttranslational transport of prepro-alpha factor across the ER membrane</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">97</span>, <span class="cit-fpage">553</span>–<span class="cit-lpage">564</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMatlack%26rft.auinit1%253DK.%2BE.%26rft.volume%253D97%26rft.issue%253D5%26rft.spage%253D553%26rft.epage%253D564%26rft.atitle%253DBiP%2Bacts%2Bas%2Ba%2Bmolecular%2Bratchet%2Bduring%2Bposttranslational%2Btransport%2Bof%2Bprepro-alpha%2Bfactor%2Bacross%2Bthe%2BER%2Bmembrane.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980767-9%26rft_id%253Dinfo%253Apmid%252F10367885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80767-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10367885&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080672100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49 in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.49" data-doi="10.1073/pnas.92.21.9643"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goeckeler</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sheckman</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">BiP and Sec63p are Required for both co- and posttranslational protein translocation into the yeast endoplasmic reticulum</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">92</span>, <span class="cit-fpage">9643</span>–<span class="cit-lpage">9646</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBrodsky%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D92%26rft.issue%253D21%26rft.spage%253D9643%26rft.epage%253D9646%26rft.atitle%253DBiP%2Band%2BSec63p%2Bare%2Brequired%2Bfor%2Bboth%2Bco-%2Band%2Bposttranslational%2Bprotein%2Btranslocation%2Binto%2Bthe%2Byeast%2Bendoplasmic%2Breticulum.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.21.9643%26rft_id%253Dinfo%253Apmid%252F7568189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTIvMjEvOTY0MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2Lzc5MDQ0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50 in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.50" data-doi="10.1128/EC.5.4.762-770.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muldoon-Jacobs</span>,  <span class="cit-name-given-names">K. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">Specific Effects of Ribosome-Tethered Molecular Chaperones on Programmed −1 Ribosomal Frameshifting</span>. <abbr class="cit-jnl-abbrev">Eukaryotic Cell</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">762</span>–<span class="cit-lpage">770</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DMuldoon-Jacobs%26rft.auinit1%253DK.%2BL.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D762%26rft.epage%253D770%26rft.atitle%253DSpecific%2BEffects%2Bof%2BRibosome-Tethered%2BMolecular%2BChaperones%2Bon%2BProgrammed%2B-1%2BRibosomal%2BFrameshifting%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.5.4.762-770.2006%26rft_id%253Dinfo%253Apmid%252F16607023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI1LzQvNzYyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNzkwNDQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51 in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roushar</span>,  <span class="cit-name-given-names">F. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Contribution of Cotranslational Folding Defects to Membrane Protein Homeostasis</span>. <abbr class="cit-jnl-abbrev">J Am Chem Soc</abbr> <span class="cit-vol">141</span>, <span class="cit-fpage">204</span>–<span class="cit-lpage">215</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAm%2BChem%2BSoc%26rft.volume%253D141%26rft.spage%253D204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52 in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.52" data-doi="10.1016/j.cell.2014.05.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Voorhees</span>,  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandez</span>,  <span class="cit-name-given-names">I. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheres</span>,  <span class="cit-name-given-names">S. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hegde</span>,  <span class="cit-name-given-names">R. S.</span></span> <span class="cit-article-title">Structure of the mammalian ribosome-Sec61 complex to 3.4 A resolution</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>, <span class="cit-fpage">1632</span>–<span class="cit-lpage">1643</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1632%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.05.024%26rft_id%253Dinfo%253Apmid%252F24930395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.05.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24930395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F790444.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340941900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53 in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-790444v2.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Angelikopoulos</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tauriello</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papadimitriou</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Koumoutsakos</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Fusing heterogeneous data for the calibration of molecular dynamics force fields using hierarchical Bayesian models</span>. <abbr class="cit-jnl-abbrev">J Chem Phys</abbr> <span class="cit-vol">145</span>, <span class="cit-fpage">244112</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BChem%2BPhys%26rft.volume%253D145%26rft.spage%253D244112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/790444v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F790444v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/376590v3" title="Fungal-bacterial interaction selects for quorum sensing mutants and a metabolic shift towards the production of natural antifungal compounds" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 16, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/790444v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/790444v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/790444v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/790444v2" class="active" data-icon-position="" data-hide-link-title="0">Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1107555" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gZTgS7shWREF9HGoK-IEEVnnMhSIPhUexcOmpfRyFjY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107555" id="node1107555" data-pisa="biorxiv;790444v2" data-pisa-master="biorxiv;790444" data-seqnum="1107555" data-apath="/biorxiv/early/2020/01/16/790444.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Haley R.</span> <span class="nlm-surname">Harrington</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Matthew H.</span> <span class="nlm-surname">Zimmer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Laura M.</span> <span class="nlm-surname">Chamness</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Veronica</span> <span class="nlm-surname">Nash</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wesley D.</span> <span class="nlm-surname">Penn</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Thomas F.</span> <span class="nlm-surname">Miller</span> <span class="nlm-suffix">III</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Suchetana</span> <span class="nlm-surname">Mukhopadhyay</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Jonathan P.</span> <span class="nlm-surname">Schlebach</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">790444; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/790444 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/790444v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;title=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;title=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;text=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;title=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;t=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;title=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;title=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107555" id="node1107555--2" data-pisa="biorxiv;790444v2" data-pisa-master="biorxiv;790444" data-seqnum="1107555" data-apath="/biorxiv/early/2020/01/16/790444.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Haley R.</span> <span class="nlm-surname">Harrington</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Matthew H.</span> <span class="nlm-surname">Zimmer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Laura M.</span> <span class="nlm-surname">Chamness</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Veronica</span> <span class="nlm-surname">Nash</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wesley D.</span> <span class="nlm-surname">Penn</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Thomas F.</span> <span class="nlm-surname">Miller</span> <span class="nlm-suffix">III</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Suchetana</span> <span class="nlm-surname">Mukhopadhyay</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Jonathan P.</span> <span class="nlm-surname">Schlebach</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">790444; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/790444 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Harrington%2C+Haley+R.&amp;rft.au=Zimmer%2C+Matthew+H.&amp;rft.au=Chamness%2C+Laura+M.&amp;rft.au=Nash%2C+Veronica&amp;rft.au=Penn%2C+Wesley+D.&amp;rft.au=Miller%2C+Thomas+F.&amp;rft.au=Mukhopadhyay%2C+Suchetana&amp;rft.au=Schlebach%2C+Jonathan+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1107555/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1107555/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1107555/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1107555/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1107555/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1107555/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1107555/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1107555/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1107555/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1107555/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1107555/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1107555/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;count=horizontal&amp;via=&amp;text=Cotranslational%20Folding%20Stimulates%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Alphavirus%20Structural%20Polyprotein&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/790444v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/790444v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/790444v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biochemistry" class="highlight" data-icon-position="" data-hide-link-title="0">Biochemistry<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
