<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/11/2020.03.10.986109.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.10.986109v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Larger viral genome size facilitates emergence of zoonotic diseases" />
<meta name="DC.Identifier" content="10.1101/2020.03.10.986109" />
<meta name="DC.Date" content="2020-03-11" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Emergence of new viral diseases is linked to mutation or recombination events. The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses." />
<meta name="DC.Contributor" content="Richard E. Grewelle" />
<meta name="article:published_time" content="2020-03-11" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Larger viral genome size facilitates emergence of zoonotic diseases" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Emergence of new viral diseases is linked to mutation or recombination events.  The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny.  Herein a model is described connecting the production of genetic variability with increasing genome size.  Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes.  These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.10.986109v1" />
<meta name="citation_id" content="2020.03.10.986109v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.10.986109" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.10.986109" />
<meta name="citation_author" content="Richard E. Grewelle" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University" />
<meta name="citation_author_institution" content="Hopkins Marine Station" />
<meta name="citation_author_email" content="regrew@stanford.edu" />
<meta name="citation_reference" content="Taylor, L. H., Latham, S. M. &amp; Woolhouse, M. E. Risk factors for human disease emergence. Philosophical Transactions of the Royal Society of London. Series B: Biological Sciences 356, 983–989 (2001)." />
<meta name="citation_reference" content="Gebreyes, W. A. et al. The global one health paradigm: challenges and opportunities for tackling infectious diseases at the human, animal, and environment interface in low-resource settings. PLoS neglected tropical diseases 8 (2014)." />
<meta name="citation_reference" content="Hampson, K. et al. Estimating the global burden of endemic canine rabies. PLoS neglected tropical diseases 9 (2015)." />
<meta name="citation_reference" content="World Bank. Update on the economic impact of the 2014 Ebola epidemic on Liberia, Sierra Leone, and Guinea (English) Accessed: 2020-02-07. 2014. http://documents.worldbank.org/curated/en/577631468167373122/Update-on-the-economic-impact-of-the-2014-Ebola-epidemic-on-Liberia-Sierra-Leone-and-Guinea." />
<meta name="citation_reference" content="Jones, K. E. et al. Global trends in emerging infectious diseases. Nature 451, 990–993 (2008)." />
<meta name="citation_reference" content="Li, Q. et al. Early transmission dynamics in Wuhan, China, of novel coronavirus–infected pneumonia. New England Journal of Medicine (2020)." />
<meta name="citation_reference" content="Worldometer. COVID-19 CORONAVIRUS OUTBREAK Last accessed: 2020-02-28. 2020. https://www.worldometers.info/coronavirus/." />
<meta name="citation_reference" content="De Wit, E., van Doremalen, N., Falzarano, D. &amp; Munster, V. J. SARS and MERS: recent insights into emerging coronaviruses. Nature Reviews Microbiology 14, 523 (2016)." />
<meta name="citation_reference" content="Chu, D. K. et al. Avian coronavirus in wild aquatic birds. Journal of virology 85, 12815–12820 (2011)." />
<meta name="citation_reference" content="Rest, J. S. &amp; Mindell, D. P. SARS associated coronavirus has a recombinant polymerase and coronaviruses have a history of host-shifting. Infection, Genetics and Evolution 3, 219–225 (2003)." />
<meta name="citation_reference" content="Makino, S., Keck, J. G., Stohlman, S. A. &amp; Lai, M. High-frequency RNA recombination of murine coronaviruses. Journal of Virology 57, 729–737 (1986)." />
<meta name="citation_reference" content="NCBI Viral Genome Library Last accessed: 2020-02-06. 2020. https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi." />
<meta name="citation_reference" content="Parrish, C. R. et al. Cross-species virus transmission and the emergence of new epidemic diseases. Microbiol. Mol. Biol. Rev. 72, 457–470 (2008)." />
<meta name="citation_reference" content="Drake, J. W. &amp; Holland, J. J. Mutation rates among RNA viruses. Proceedings of the National Academy of Sciences 96, 13910–13913 (1999)." />
<meta name="citation_reference" content="Pèrez-Losada, M., Arenas, M., Galán, J. C., Palero, F. &amp; González-Candelas, F. Recombination in viruses: mechanisms, methods of study, and evolutionary consequences. Infection, Genetics and Evolution 30, 296–307 (2015)." />
<meta name="citation_reference" content="Jensen-Seaman, M. I. et al. Comparative recombination rates in the rat, mouse, and human genomes. Genome research 14, 528–538 (2004)." />
<meta name="citation_reference" content="Drummond, D. A., Silberg, J. J., Meyer, M. M., Wilke, C. O. &amp; Arnold, F. H. On the conservative nature of intragenic recombination. Proceedings of the National Academy of Sciences 102, 5380–5385 (2005)." />
<meta name="citation_reference" content="Bull, J. J., Sanjuan, R. &amp; Wilke, C. O. Theory of lethal mutagenesis for viruses. Journal of virology 81, 2930–2939 (2007)." />
<meta name="citation_reference" content="Johnson, C. K. et al. Spillover and pandemic properties of zoonotic viruses with high host plasticity. Scientific reports 5, 14830 (2015)." />
<meta name="citation_reference" content="Ecker, S., Pancaldi, V., Valencia, A., Beck, S. &amp; Paul, D. S. Epigenetic and transcriptional variability shape phenotypic plasticity. Bioessays 40, 1700148 (2018)." />
<meta name="citation_reference" content="Lauber, C. et al. The footprint of genome architecture in the largest genome expansion in RNA viruses. PLoS pathogens 9 (2013)." />
<meta name="citation_reference" content="Patton, J. T. &amp; Spencer, E. Genome replication and packaging of segmented double-stranded RNA viruses. Virology 277, 217–225 (2000)." />
<meta name="citation_reference" content="Rivera-Mulia, J. C. &amp; Gilbert, D. M. Replicating large genomes: divide and conquer. Molecular cell 62, 756–765 (2016)." />
<meta name="citation_reference" content="Minskaia, E. et al. Discovery of an RNA virus 3 5 exoribonuclease that is critically involved in coronavirus RNA synthesis. Proceedings of the National Academy of Sciences 103, 5108–5113 (2006)." />
<meta name="citation_reference" content="Lu, R. et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. The Lancet (2020)." />
<meta name="citation_reference" content="Nagy, P. D. &amp; Pogany, J. The dependence of viral RNA replication on co-opted host factors. Nature Reviews Microbiology 10, 137–149 (2012)." />
<meta name="citation_reference" content="Simon-Loriere, E. &amp; Holmes, E. C. Why do RNA viruses recombine? Nature Reviews Microbiology 9, 617–626 (2011)." />
<meta name="citation_reference" content="Rector, A. &amp; Van Ranst, M. Animal papillomaviruses. Virology 445, 213–223 (2013)." />
<meta name="citation_reference" content="McVean, G., Awadalla, P. &amp; Fearnhead, P. A coalescent-based method for detecting and estimating recombination from gene sequences. Genetics 160, 1231–1241 (2002)." />
<meta name="citation_reference" content="Archer, J. et al. Identifying the important HIV-1 recombination breakpoints. PLoS computational biology 4 (2008)." />
<meta name="citation_reference" content="Carroll, D. et al. The global virome project. Science 359, 872–874 (2018)." />
<meta name="twitter:title" content="Larger viral genome size facilitates emergence of zoonotic diseases" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Emergence of new viral diseases is linked to mutation or recombination events. The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses." />
<meta name="og-title" property="og:title" content="Larger viral genome size facilitates emergence of zoonotic diseases" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Emergence of new viral diseases is linked to mutation or recombination events. The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-11" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.10.986109v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.10.986109v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.10.986109v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1191318" />
    <title>Larger viral genome size facilitates emergence of zoonotic diseases | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2NzRehSx3dV52h_kOTXSByw46wba1DkSifGBwezoYHg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1191318","apath":"\/biorxiv\/early\/2020\/03\/11\/2020.03.10.986109.atom","pisa":"biorxiv;2020.03.10.986109v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.10.986109v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.10.986109v1","title":"Larger viral genome size facilitates emergence of zoonotic diseases","identifier":"node\/1191318"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oX3Sd\\\\\\\/oqoJp7HalGrzkgg4IeF68TQSyoO9+x9nPiYgW8aE4iogk9eRkXUb5GO6YY\\\\\\\/XGw20nsilS9aaZHTfpqDIq201xMNligK1ld9ERP2AW6SA96z5H83EMrHLRcvnvUv7YSdIwJGBzAYiWoBHTky+oMv9eI7xsiFd3vBNAr3KLugKDPxd7u8Sc4sCP+mdJXXzKXku\\\\\\\/wZdZ49Bm6FuMyAC\\\\\\\/9SESf6n7La6H\\\\\\\/XGcyJIRdGq4s3VnoSGPTjPYs9WSb4eIYLHbHiXvdFZvmSBZtdTL3Ef0H0VfribatBO6eVLhGol60BlfW4hqYQV\\\\\\\/YF6ONg23AadrZpkVx0A4cLT6M07c7SfSkX8FrrTM5zZ+5TJbkwhxsN6FILgB0xgEnPImFf2SG+imzagMJehXZaRv8sAHkgIADsdc6kjIBelQtmVUWa8vjQqulPrDuvIB0sGbdBlRg+W9a9uJu2Ec736ousbz+S9ZojGk9l3qmCg74ddW40j9P7rI4OIxne4n9dZRBjZooPMfDebXRoc0OM0sthGxJF7hQgq9Ma1HUOgLZAzcNc2pOWfQr7OewdcVIiNJW48jC28qMlu16DGgZofhwRQE\\\\\\\/aI0eJa9OnIiaNk7VC9yc2e1ZfQYk4YuKh6A5rjh\\\\\\\/VcWOYnnJrQFWY5X2ssLBZFddzORT9K8401NRQxLUiIy0gGkGbwVpTX20o6A9Cv+cuZHmSfTvgpEK46CPZ5uxE+lvXGV54LP7Fy483OjsHGMNejQZbUwSjLY+k8RInktxzd14uFP+EkAlou4CiRJ\\\\\\\/GFxZ5WU8or9QysNF94FPuc4JvRGXu1AWhXMlodJP8ydZX0AcXvjOSb3gikySczRTEjp0jK\\\\\\\/Rr8Wt7KKKmVw2wMGJ+F7nTdwAXjBGgLZpy\\\\\\\/8GHVTFUgCkeBQCviqHl88O5zFUDV6r\\\\\\\/cRhcnMBkTI2sqWBSg7qzdum1DlNXLJYcRQNZ0katC5DBmPqF6LxkRxCcDcAx3OSQoxH79QW9b7\\\\\\\/ox6DatPM16rA2Rx1NTfmsmZgqfuFk0AB51ff2GC+mP2P4\\\\\\\/qVLnWBXM8rTbx2fEPKJDATPcXhom9rlpdZCznJAEoF+1z7aeZWhEbx4A\\\\\\\/z1IATb0LqlTK0F5ok7NcBdwVVHFQTzkgwW8lsYiB3OPw9FCBDuWKQrm+UtlJa5RImTfn+bdyl4YECWA0E5J6mzbm15HvTvKJPxaV1ESn6Z0Ez\\\\\\\/vUB77u9IIErFQSU7hbtWnEUvTlfZS2cN0UfrpCuKVjzyH77hDTf\\\\\\\/GfpV2Dj670yI3STbWn8ikfQOBfHDYKVhVRmSww8B1nCSlATFGPmt505FM0Fqolm6t8\\\\\\\/N7WrjrmQvI+O\\\u0022,\\\u0022iv\\\u0022:\\\u0022fWGkl9sXQpbMv\\\\\\\/w691YTHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224e52082fa11d9ab4ebabf4cb98329f76601b99a18f46f3b84e07ed3c97856736\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.10.986109v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1191318 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.10.986109v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xVHwW8NDg78ifX4bYMmlPF3HOk13pIGCMFjjOri7inQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1191318" id="node1191318" data-pisa="biorxiv;2020.03.10.986109v1" data-pisa-master="biorxiv;2020.03.10.986109" data-apath="/biorxiv/early/2020/03/11/2020.03.10.986109.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Larger viral genome size facilitates emergence of zoonotic diseases</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Richard E.</span> <span class="nlm-surname">Grewelle</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.10.986109 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1191318" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Richard E. Grewelle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, Stanford University</span>, 371 Serra Mall, Stanford, CA 94305, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Hopkins Marine Station</span>, Pacific Grove, CA 93950, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BE.%2BGrewelle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grewelle%20RE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BE.%2BGrewelle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:regrew@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">regrew@stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.10.986109v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1191318" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1191318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986109v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1191318" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1191318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986109v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1191318" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1191318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986109v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1191318" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1191318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986109v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1191318" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1191318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986109v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1191318" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1191318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.10.986109v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1191318" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1191318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Emergence of new viral diseases is linked to mutation or recombination events. The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Human population expansion continually redefines the interactions between humans and wildlife. Zoonotic diseases that emerge from these interactions account for over 60% of human infectious diseases [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Although this figure may represent over 2 billion new cases annually, zoonoses are characteristically underreported and misdiagnosed [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Because zoonoses affect humans and other vertebrates, they also generate substantial economic losses, as livestock death and human disability lead to loss of productivity [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. For example, economic losses due to Ebola in 2014 were $2.2 billion [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Zoonotic outbreaks also present an increasingly pressing risk to human health due to higher densities of people with little to no immunity to novel pathogens. They are often unpredictable in occurrence and are caused by new species against which there are no developed vaccines, treatment, or public health protocol. This is particularly true for viruses that have great capacity to mutate, even during the course of a pandemic [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. This century has seen numerous pandemics caused by zoonotic viruses like Ebola, Nipah, Avian Influenza, and Coronavirus [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>].</p><p id="p-5">In late 2019, a novel Coronavirus (COVID-19) emerged and has since killed over 3000 people, making it the deadliest Coronavirus pandemic in recent history [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. The Wuhan province in China, where the virus emerged, effectively quarantined its populace, and many countries restricted travel of their own citizens to China. Although human to human transmission is driving current transmission, its emergence was facilitated by close human-animal contact, much like its predecessors SARS and MERS [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Members of the Coronavididae family are widely known to jump across host taxa to become infectious in new populations [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Non-pathogenic varieties in a reservoir population can become pathogenic in new species. This phenomenon is well documented in many viral zoonoses, yet Coronaviruses pose exceptional risk to human populations due to the variety of potential reservoirs and the virulence of the emergent pandemics they cause [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>]. The ability of Coronaviruses to shift to new species is due in part to the mutation rate they exhibit, but they are not exceptional among viruses in this way. They are exceptional, however, in their high frequency of recombination and in possessing the largest of the ssRNA viral genomes [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. These two characteristics provide ample opportunity for Coronaviruses to evolve new traits and establish themselves in new host species.</p><p id="p-6">The capacity to produce genetic variability is a defining feature among viruses that are able to jump to new host species. Mutations and recombination events have been explored as a source of this variability. Both processes have been implicated in producing new viral strains or species responsible for observed pandemics [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. The rates of mutation and recombination are known to vary across viral taxa and within viral genomes [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. For example, RNA viruses characteristically mutate faster than DNA viruses due to polymerase infidelity, and hypermutable regions within some viral genomes have been demonstrated. Likewise, recombination processes vary considerably by viral type: reassortment is common in viruses with multi-segmented genomes, while homologous or non-homologous recombination is predominant in viruses with non-segmented genomes [<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>].</p><p id="p-7">A primary challenge for virologists and public health professionals is identifying characteristics of viruses that lead to higher likelihoods of new outbreaks [<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>]. The variety of mechanisms by which genetic variation can be produced makes finding clear patterns in zoonotic risk difficult [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>]. One yet unconsidered viral characteristic likely to facilitate cross-species infections is the size of the viral genome. Both the total number of mutations accumulated during replication and the probability of a recombination event increase with the size of a genome when replication mechanisms are conserved, though nucleotide substitutions from either process often lead to net reduction in fitness of progeny. The number of substitutions, thus potential for reduction in host-specific fitness, increases with genome size [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. If genetic variability is instrumental in producing cross-species infections, genome size should be positively related to the likelihood of viral spillover events provided lethal substitutions do not substantially reduce the mean fitness of the viral population.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-8">Cross-species transmission for each viral particle is assumed to be a function of the phenotypic plasticity of the viral genome and the genetic variability produced during replication [<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. The probability of a transmission event occurring can be represented as:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-1.gif"/></noscript></span>

</span>
where <em>λ</em> ∈ [0, ∞) is the weighted contribution of plasticity over variability in promoting cross-species transmission and <em>L</em> is the length of the viral genome. Although a relationship between genome size and phenotypic plasticity is probable, little work has enlightened the nature of this relationship. What is clear is that transcription and gene regulation networks can become more complex with increasing number of nucleotides [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Variability produced with each replication event can be more accurately approximated. Variability is produced via mutation and recombination (reassortment, non-homologous and homologous). Given per nucleotide rates of mutation (<em>µ</em>) and recombination (<em>R</em>), the accumulation of variability per genome should increase linearly as (<em>µ</em> + <em>R</em>(1 − <em>µ</em>))<em>L</em>. When <em>µ</em> and R are small, this variability can be expressed as (<em>µ</em> + <em>R</em>)<em>L</em>. Lethal mutagenesis and incompatible recombinants reduce viability of viral products. Viability converges to <em>e</em><sup>−<em>µγL</em></sup> across mutational models [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>] and is calculated as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-1.gif"/></noscript></span></span> for recombination [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] (see supplement), where <em>γ</em> is the proportion of lethal mutations, <em>ρ</em> is the genome-specific tolerance to recombination events, and <em>D</em> ∈ [0, 1] is the sequence divergence between co-infecting viruses. Per nucleotide variability (<em>dV/dN</em>) produced with each replication event can be given as:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-2.gif"/></noscript></span>

</span>
</p><p id="p-9">Per genome variability (<em>dV/dG</em>) can be given as:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-3.gif"/></noscript></span>

</span>
</p><p id="p-10">Assuming the number of polymerase enzymes scales with genome size [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], a rate of production of variability can be derived.
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-4.gif"/></noscript></span>

</span>
</p><p id="p-11">The efficiency of replication of each genome can be represented as <em>L/</em>(<em>I</em> + <em>L</em>), where <em>I</em> is the product of the time required for replication initiation and termination, and <em>r</em>, the rate of polymerase-mediated replication. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> displays <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">equations 2</a>–<a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">4</a> parameterized to represent positive sense ssRNA viruses. Per nucleotide variability production decreases with genome size due to reduced fitness with accumulated substitutions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>). However, genome-level variability production increases almost linearly with genome size, revealing that the rate of fitness reduction from the accumulation of deleterious substitutions does not outpace the increased variability associated with increase in genome size (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>). Because more genomes can be replicated in a fixed amount of time for smaller genomes, the total variability produced in unit time gives indication of whether population-level variability increases with genome size. In fact, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a> shows that population-level variability is maximized within the known size range of ssRNA(+) viral genomes. Variability production per unit time continues to decline for genome sizes larger than 50,000 nt, though no known ssRNA viruses have genomes larger than 35,000 nt [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]. The maximum variability production per unit time is dependent on the efficiency of replication. Because Coronaviruses have much larger genomes than other ssRNA(+) viruses, they have evolved 3′ → 5′exoribonuclease repair and have lower mutation rates than other ssRNA(+) viruses [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Substituting this mutation rate in <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">equation 4</a> produces a curve (Fig. S2) with a maximum shifted to the right, reflecting that production of genetic variability is maximized at a larger genome size for Coronaviruses than for other ssRNA(+) viruses. Considering these theoretical results, variability is greater per virion produced for viruses with larger genomes. Because population-level production of variability has been shown to improve viral survival via immune evasion [<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>], there is likely strong selection toward viral genome sizes close to the peak described by <a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">equation 4</a> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1C</a>). Even in the absence of phenotypic plasticity conferred by large genomes, viruses with large genomes have high potential to generate variability that promotes cross-species transmission events. Zoonotic viruses should, therefore, possess larger genomes than other viruses. To examine the relationship between viral genome size and likelihood of zoonosis, this study compiles a comprehensive list of all known viral zoonoses and leverages the entire NCBI database of viral genomes [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>].</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The rate at which genetic variability is produced per virion depends on genome size. The relationships are shown as substitutions per nucleotide (A), per genome (B), and per second (C) for a positive sense ssRNA virus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-368715731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The rate at which genetic variability is produced per virion depends on genome size. The relationships are shown as substitutions per nucleotide (A), per genome (B), and per second (C) for a positive sense ssRNA virus.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F1.medium.gif" width="294" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F1.medium.gif" width="294" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-12" class="first-child">The rate at which genetic variability is produced per virion depends on genome size. The relationships are shown as substitutions per nucleotide (A), per genome (B), and per second (C) for a positive sense ssRNA virus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">Peer-reviewed literature from 1920 to 2019 was searched via the Web of Science electronic library for viral zoonoses. Search key words included “zoonotic”, “zoonoses”, “emerging disease”, “viral spillover”, “animal to human transmission”, and “animal reservoir virus”. After cross examination of all viruses yielded through this search, all confirmed zoonotic viruses transmitted from animals to humans in the past 100 years that also had a complete genome sequence were included in the analysis. This set encompasses 126 viral species from 40 genera in groups 1, 3, 4, 5, and 6 of the Baltimore classification system, grouped by genome structure. Genome sizes for all viruses were retrieved from the NCBI genome database or from peer-reviewed reports of fully sequenced genomes. The entire NCBI viral genome database of 9238 sequenced genomes was downloaded and filtered to remove incomplete, prokaryote, fungus, protist, and plant associated viral sequences. 2088 animal associated virus species in the five evaluated Baltimore groups remained after removal of zoonotic virus sequences. All species were categorized by genome structure according to the Baltimore classification system. One-tailed Welch’s T-tests for sample means were performed for each Baltimore group, comparing the genome sizes of zoonotic viruses with all other non-zoonotic viruses. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> gives the results of these analyses, showing that zoonotic virus genomes are significantly larger than non-zoonotic virus genomes across viral groups. Resolution of this relationship is strong except for viral group 4, where the mean genome size for zoonotic viruses approaches being larger than the mean genome size for other animal viruses (at <em>α</em> = 0.05). The distributions of genome sizes are visualized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1195595/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1195595/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1195595" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-14" class="first-child">Comparison of viral genome size between known zoonotic viruses and all other animal associated viruses. Comparisons are made within Baltimore groups classified according to genomic structure. Mean and standard errors for each group are reported.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Violin plots comparing genome sizes between non-zoonotic animal viruses and zoonotic viruses. Zoonotic virus genome sizes inhabit the upper ranges of values in each group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-368715731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Violin plots comparing genome sizes between non-zoonotic animal viruses and zoonotic viruses. Zoonotic virus genome sizes inhabit the upper ranges of values in each group.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F2.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F2.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/11/2020.03.10.986109/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-15" class="first-child">Violin plots comparing genome sizes between non-zoonotic animal viruses and zoonotic viruses. Zoonotic virus genome sizes inhabit the upper ranges of values in each group.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-3"><h2 class="">Discussion</h2><p id="p-16">This finding strongly suggests that large genome size facilitates emergence of viral zoonoses. The signal is clear despite the large diversity of viruses within each Baltimore group and despite not accounting for viruses that share or jump between animal host species. Viruses with a wide host range likely share characteristics with zoonotic viruses, and based on these results, should have large genomes as well. For example, numerous species of Coronavirus are included in the non-zoonotic list (n=42) because they are not directly implicated in zoonoses, yet they are known to exhibit wide host ranges [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>]. Specifically, several bat Coronaviruses are in the non-zoonotic list, though researchers largely regard these viruses as the close relatives to SARS and MERS CoV [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. This fact likely impaired the resolution of the relationship between genome size and cross-species transmission, resulting in the single non-significant p-value across evaluated groups. Large genomes are likely to facilitate zoonotic transmission via high mutational load per genome replication or high probability of recombination; both processes produce variability by which cross-species infection may occur. Genome sizes for some viral groups, especially dsDNA viruses, are tightly correlated with increasing number of genes (<em>R</em><sup>2</sup> = 0.91, Fig. S2). More coding regions within the genome could lead to greater plasticity and expanded host range, as reliance on host replication machinery is reduced [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>].</p><p id="p-17">Among the viral groups there exists much variation in replication strategy, thus there is potential for differences in the predominant strategy for producing genetic variability leading to zoonotic transmission events [<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>]. Though mutation rate is 100 fold lower per nucleotide for dsDNA viruses than for RNA viruses, their much larger genomes enable a comparable likeli-hood of accumulating at least one mutation during replication. A negative correlation between mutation rate and genome size has been noted for a small set of viruses for which mutation rate has been directly measured [<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a>]. Much of the correlation for RNA viruses results from measurement of bacteriophage mutation rate, and this interesting relationship should be further explored. Coronaviruses have evolved 3′→5′proofreading capabilities to accommodate mutagenesis associated with a large genome [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>]. As recombination is generally associated with lower reduction in fitness compared to mutation, recombination may be the preferred mechanism to generate genetic variability for the viruses with the largest genomes. dsDNA viruses are known to have high per nucleotide rates of non-homologous recombination. Because of the large genome sizes of dsDNA viruses, the likelihood of at least one recombination event occurring during replication of the genome is high [<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. The many coded genes of ds-DNA viruses allow more easily incorporation of recombinant DNA without deleterious effects and may reduce host specificity through phenotypic plasticity [<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>]. In contrast, Papillo-maviruses are dsDNA viruses that are characteristically species-specific [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. These viruses have small genomes with little plasticity to accommodate recombinant DNA or expand host range. dsRNA viruses are highly segmented, and the number of segments is highly correlated with the size of the genome (<em>R</em><sup>2</sup> = 0.84, Fig. S2), which allows for a high degree of recombination via reassortment of segments during a coinfection [<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">15</a>]. Gene expression is regulated by translation for ssRNA(+) viruses. This group includes Coronaviruses and is known for high levels of homologous recombination because of frequent template switching during discontinuous transcription. Having the largest RNA viral genomes, Coronaviruses can produce over 25% recombinant progeny with their high rates of recombination [<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">15</a>]. ssRNA(-) viruses are not known to recombine with high frequency, so their genetic variation relies on high mutation rates alone [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. ssRNA-RT viruses are known for maintaining high mutation rates characteristic of RNA viruses as well as high recombination rates for a subset of these viruses, including HIV. Pseudodiploidy in retroviruses can promote recombination due to the physical proximity of RNA strands in the virion as well as partial masking of deleterious mutations [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. Large genomes are more likely to contain intergenic, non-coding regions that assist in gene regulation but also serve as hotspots for recombination and mutation due to weaker selection to conserve these regions [<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>].</p><p id="p-18">Viral zoonoses can emerge as a result of large genetic changes associated with recombination or significant mutation–both more likely to occur in large genomes. Phenotypic plasticity is hypothesized to be positively related to genome size and may facilitate cross-species transmission, though this notion requires future exploration. The relationship between likelihood of zoonotic transmission and genome size has implications for predicting future zoonoses and for understanding general principles about viral life history. Host specificity may be considered a function of genome reduction. Guiding principles like the relationship between genome size and host range should be explored to broaden scientific understanding of the viral world and its role in ecosystems. Predicting future epidemics requires knowledge of potential zoonotic viruses. Far less than 1% of viruses have been discovered, and the potential host interactions of each viral species are even less well-characterized, making zoonotic risks difficult to estimate [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. These data demonstrate that viral genome size is an effective indicator of zoonotic risk across viral genome subtypes. In an age of high-throughput genomics and rapid discovery of new viruses, characterization of recombination and mutation rates is far slower and more labor-intensive than genome sequencing, making genome size an appealing first step in zoonotic risk evaluation. Analysis of viral genome size and structure has the potential to provide deep insights in predicting future pandemics.</p></div><div class="section" id="sec-4"><h2 class="">Methods</h2><div id="sec-5" class="subsection"><h3>Derivation of Equation 2</h3><p id="p-19">The accumulation of substitutions in a genome occurs via mutation and recombination, with associated per nucleotide rates <em>µ</em> and <em>R</em>. These rates can be interpreted as the per nucleotide probability of a substitution occurring in a replication event. Assuming mutation and recombination are independent processes, the probability of a substitution (variability) at each nucleotide can be represented as:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-8.gif"/></noscript></span>

</span>
</p><p id="p-20">The probability that a substitution occurs for each nucleotide can be interpreted as a rate of substitution per replication event. Substitution events are assumed to be deleterious. Each event contributes to the reduction in fitness of the viral progeny, and these events accumulate in a genome, creating non-linear declines in fitness. Viral genome size is constrained by this reduction in fitness: larger genomes accumulate more substitutions and potential for fitness reduction is greater than for small genomes [<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>]. Nucleotide-level fitness thus corresponds to genome-level fitness, and successful production of nucleotide variability is constrained by the fitness decay that occurs with increasing genome size. For purposes here, fitness will be synonymous with viability of each replicated genome. Setting the fitness of the parental virion or template genome at 1, fitness of progeny takes values between 0 and 1. Because a mutation and recombination event at a given nucleotide are mutually exclusive, the probability of each event occurring separately can be calculated as
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-9.gif"/></noscript></span>

</span>
</p><p id="p-21">Across mutational models, fitness (infectivity) due to lethal mutagenesis is given as <em>ω</em><sub><em>mut</em></sub> = <em>e</em><sup>−<em>µγ</em>(<em>nt</em>)</sup> [<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a>]. The fitness in the event of recombination is demonstrated to be <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-2.gif"/></noscript></span></span> 17, where <em>m</em> is the number of amino acid substitution events due to recombination and <em>d</em> is the maximum number of amino acid substitutions possible given the genetic dissimilarity of the coinfecting viral complement. Genome-level fitness for mutagenesis can be calculated as <em>ω</em><sub><em>mut</em></sub> = <em>e</em><sup>−<em>µγL</em></sup>. Because the probability of a random nucleotide substitution resulting in an amino acid substitution is 415<em>/</em>549 ≈ 3<em>/</em>4, the number of amino acid substitutions in a genome replication event is <em>m</em> = 3<em>RL/</em>4. Denoting <em>D</em> as proportional nucleotide dissimilarity in a complementary strand from another virus, <em>d</em> = 3<em>DL/</em>4. This yields <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-3.gif"/></noscript></span></span>. Because mutation and recombination are components of fitness, the probability of each occurring with exclusion of the other at each nucleotide must be accounted. This can be done by replacing <em>µ</em> with <em>Pr</em>(<em>µ</em>\<em>R</em>) and <em>R</em> with <em>Pr</em>(<em>R</em>\<em>µ</em>). Genome-level fitness for substitution events can be calculated as
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-10.gif"/></noscript></span>

</span>
which simplifies to
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-11.gif"/></noscript></span>

</span>
</p><p id="p-22">The resulting rate of per nucleotide variability produced is the product of variability accumulation and the average fitness of each genome replicated.
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-12.gif"/></noscript></span>

</span>
</p><p id="p-23">When <em>µ</em> and <em>R</em> are sufficiently small such that <em>µR</em> « <em>µ, R</em> (i.e. <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/inline-graphic-4.gif"/></noscript></span></span>), then <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">equation S5</a> can be approximated as
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/11/2020.03.10.986109/embed/graphic-13.gif"/></noscript></span>

</span>
</p><p id="p-24">This is given by <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">equation 2</a> in the main text.</p></div><div id="sec-6" class="subsection"><h3>Parameter values for figure 1</h3><p id="p-25"><a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> represents a general simulation for ssRNA(+) viruses. To understand the mutation and recombination consequences for this group of viruses, parameter values were approximated from the literature on ssRNA(+) viruses. Because specific estimates for parameter values for each family of viruses within this group is sparse or non-existent, approximate values are given when possible. Qualitative outcomes for each of the panels in <a id="xref-fig-1-7" class="xref-fig" href="#F1">figure 1</a> (main text) are similar across a broad range of possible parameter values, but relationships may differ for other groups of viruses. The parameter values used in the simulation are given in table S1. <em>r</em> is the rate constant (<em>k</em><sub><em>pol</em></sub>) for RNA dependent RNA polymerase in nucleotides per second. This is the speed at which the replicase complex can replicate RNA. <em>I</em> is the product of <em>r</em> and the amount of additional time (in seconds) required to replicate a strand of RNA outside of the elongation phase plus any additional delay associated with replicating multiple, short strands of RNA rather than fewer, long strands on which multiple replicase complexes could be active simultaneously. The value for <em>I</em> chosen was simply an order of magnitude smaller than the large Coronavirus genomes, though this delay has not been quantified in scientific literature. This value is important in determining the genome size at which the maximum rate of production of genetic variability (per second) occurs. <em>µ</em> is the mutation rate per nucleotide per replication event by RNA dependent RNA polymerase. This quantity can vary around the value given in table S1, but the value chosen is an appropriate approximation for ssRNA(+) viruses. <em>R</em> is the recombination rate per nucleotide per replication event. This value varies considerably in RNA viruses but tends to be high for many ssRNA(+) viruses. <em>γ</em> gives the proportion of lethal mutations of the total number of mutations. The vast majority of mutations fall into three categories: neutral, deleterious, or lethal. There is likely variation in the respective proportions of each for viruses, though few attempts have been made to quantify these proportions. In the study referenced, one-third of mutations resulted in viral particles that were not capable of in-fecting (infectivity assay). This constitutes a lethal mutation and is a good estimate of fitness reduction for cross-species transmission events. <em>ρ</em> represents the effective tolerance of proteins to a single recombination event. Misfolding proteins likely constitute lethality or severe reduction in fitness in viruses, so this measure is used to simulate the reduction in fitness in the event of amino acid substitutions due to recombination. <em>D</em> is the proportion of dissimilar nucleotides in a co-infecting viral strain or species. Higher dissimilarity results in higher probability of amino acid substitutions due to recombination. 12% dissimilarity is the nucleotide dissimilarity between COVID-19 and SARS-COV.</p></div><div id="sec-7" class="subsection"><h3>Literature review</h3><p id="p-26">Peer-reviewed scientific literature was searched for reports on zoonotic viruses transmitted from animals to humans using the Web of Science electronic library database for published reports through 2019. Keywords were used in the search as specified in the main text. Centers for Disease Control and Prevention and World Health Organization reports as well as published compilations of human infectious diseases were also used as references, but reported viruses were cross-validated as zoonotic with other supporting peer-reviewed literature [<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>]. Zoonotic viruses were identified as having known non-human and human hosts via molecular or serological assay (seroconversion). Viral infections supported by circumstantial evidence sans assay verification were not included. The list of zoonotic viruses was formed after searching for complete genome sequences on the NCBI database [<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>] then extending to other peer-reviewed full genome sequences if the virus was not found in the NCBI database. This final list included 126 unique viral species. Each viral species was classified according to the Baltimore classification system by mRNA production/genome structure. The compiled list is included as table S3.</p></div><div id="sec-8" class="subsection"><h3>Non-zoonotic virus filtering</h3><p id="p-27">The entire NCBI database on viral genomes was downloaded and screened for animal-associated viruses. The category of unidentified viruses was excluded from the screening process. All incomplete sequences were excluded as well. The majority of viruses are labeled with host associations, though in many instances, these labels were found to be inaccurate. Viruses were organized by families and screened initially by known host range. Viral families associated with prokaryotes, archaea, plants, or fungi exclusively were not included. Satellite viruses and viroids were also excluded. Remaining genera and species were manually cross-checked with existing literature to verify the belonging as animal associated. If species were recovered from an animal or had known relatives recovered from an animal, the species was included in the non-zoonotic, animal associated virus list. The animal grouping was assigned due to the difficulty of parsing associations further to vertebrates. Many viruses are recovered from vectors or other invertebrates, yet have known or unknown association with vertebrates. Sometimes viruses have no known association outside of an insect vector, though relatives of the virus are known to infect vertebrate hosts. As an exercise to ensure genome size did not differ substantially between vertebrate and invertebrate-associated viruses, another filtering process was conducted manually whereby each species was cross-validated with existing literature to remove known or likely known to not be associated with a vertebrate host. The total number of non-zoonotic viruses was reduced from 2088 to 1533 in this process. Performing one-tailed Welch’s T-tests with this new data set revealed little departure from previous conclusions. Results are presented in table S2 and Fig. S1.</p></div></div><div class="section" id="sec-10"><h2 class="">Funding</h2><p id="p-29">REG is funded by the Stanford Graduate Fellowship.</p></div><div class="section" id="sec-11"><h2 class="">Author Contributions</h2><p id="p-30">REG contributed to each section of this manuscript.</p></div><div class="section COI-statement" id="sec-12"><h2 class="">Competing Interests</h2><p id="p-31">REG declares no competing interest.</p></div><div class="section" id="sec-13"><h2 class="">Materials and Correspondence</h2><p id="p-32">Viral genomes were retrieved from the NCBI viral genome library <a href="https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi">https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi</a>. The list of zoonotic virus species with associated genome sizes is provided in the supplement. Python scripts to reproduce the figures is available from the author upon request.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-28">I would like to thank Kalin Wilson and Paul Bump for providing insightful feedback on this manuscript. I would also like to thank my advisor, Giulio De Leo, for continued support and advice through my PhD.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-3"><a href="https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi">https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi</a></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.1" data-doi="10.1098/rstb.2001.0888"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">L. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Latham</span>,  <span class="cit-name-given-names">S. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Woolhouse</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-article-title">Risk factors for human disease emergence</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society of London. Series B: Biological Sciences</abbr> <span class="cit-vol">356</span>, <span class="cit-fpage">983</span>–<span class="cit-lpage">989</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DTaylor%26rft.auinit1%253DL.%2BH.%26rft.volume%253D356%26rft.issue%253D1411%26rft.spage%253D983%26rft.epage%253D989%26rft.atitle%253DRisk%2Bfactors%2Bfor%2Bhuman%2Bdisease%2Bemergence%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2001.0888%26rft_id%253Dinfo%253Apmid%252F11516376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2001.0888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11516376&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F11%2F2020.03.10.986109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170315900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gebreyes</span>,  <span class="cit-name-given-names">W. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The global one health paradigm: challenges and opportunities for tackling infectious diseases at the human, animal, and environment interface in low-resource settings</span>. <abbr class="cit-jnl-abbrev">PLoS neglected tropical diseases</abbr> <span class="cit-vol">8</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hampson</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Estimating the global burden of endemic canine rabies</span>. <abbr class="cit-jnl-abbrev">PLoS neglected tropical diseases</abbr> <span class="cit-vol">9</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.10.986109v1.4"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Bank</span>. <span class="cit-source">Update on the economic impact of the 2014 Ebola epidemic on Liberia, Sierra Leone, and Guinea (English)</span> Accessed: <span class="cit-date-in-citation">2020-02-07</span>. <span class="cit-pub-date">2014</span>. <a href="http://documents.worldbank.org/curated/en/577631468167373122/Update-on-the-economic-impact-of-the-2014-Ebola-epidemic-on-Liberia-Sierra-Leone-and-Guinea">http://documents.worldbank.org/curated/en/577631468167373122/Update-on-the-economic-impact-of-the-2014-Ebola-epidemic-on-Liberia-Sierra-Leone-and-Guinea</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.5" data-doi="10.1038/nature06536"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">K. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global trends in emerging infectious diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>, <span class="cit-fpage">990</span>–<span class="cit-lpage">993</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DK.%2BE.%26rft.volume%253D451%26rft.issue%253D7181%26rft.spage%253D990%26rft.epage%253D993%26rft.atitle%253DGlobal%2Btrends%2Bin%2Bemerging%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06536%26rft_id%253Dinfo%253Apmid%252F18288193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288193&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F11%2F2020.03.10.986109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253313100048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Q.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Early transmission dynamics in Wuhan, China, of novel coronavirus–infected pneumonia</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.10.986109v1.7"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Worldometer</span>. <span class="cit-source">COVID-19 CORONAVIRUS OUTBREAK</span> Last accessed: <span class="cit-date-in-citation">2020-02-28</span>. <span class="cit-pub-date">2020</span>. <a href="https://www.worldometers.info/coronavirus/">https://www.worldometers.info/coronavirus/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.8" data-doi="10.1038/nrmicro.2016.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Wit</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falzarano</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V. J.</span></span> <span class="cit-article-title">SARS and MERS: recent insights into emerging coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">523</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BMicrobiology%26rft.volume%253D14%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2016.81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2016.81&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.9" data-doi="10.1128/JVI.05838-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">D. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Avian coronavirus in wild aquatic birds</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">85</span>, <span class="cit-fpage">12815</span>–<span class="cit-lpage">12820</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05838-11%26rft_id%253Dinfo%253Apmid%252F21957308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yMy8xMjgxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzExLzIwMjAuMDMuMTAuOTg2MTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rest</span>,  <span class="cit-name-given-names">J. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mindell</span>,  <span class="cit-name-given-names">D. P.</span></span> <span class="cit-article-title">SARS associated coronavirus has a recombinant polymerase and coronaviruses have a history of host-shifting</span>. <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">219</span>–<span class="cit-lpage">225</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Makino</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keck</span>,  <span class="cit-name-given-names">J. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>,  <span class="cit-name-given-names">S. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">High-frequency RNA recombination of murine coronaviruses</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">57</span>, <span class="cit-fpage">729</span>–<span class="cit-lpage">737</span> (<span class="cit-pub-date">1986</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMakino%26rft.auinit1%253DS%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D729%26rft.epage%253D737%26rft.atitle%253DHigh-frequency%2BRNA%2Brecombination%2Bof%2Bmurine%2Bcoronaviruses.%26rft_id%253Dinfo%253Apmid%252F3005623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjU3LzMvNzI5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTEvMjAyMC4wMy4xMC45ODYxMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.10.986109v1.12"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">NCBI Viral Genome Library</span> Last accessed: <span class="cit-date-in-citation">2020-02-06</span>. <span class="cit-pub-date">2020</span>. <a href="https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi">https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.13" data-doi="10.1128/MMBR.00004-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parrish</span>,  <span class="cit-name-given-names">C. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cross-species virus transmission and the emergence of new epidemic diseases</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev</abbr>. <span class="cit-vol">72</span>, <span class="cit-fpage">457</span>–<span class="cit-lpage">470</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DParrish%26rft.auinit1%253DC.%2BR.%26rft.volume%253D72%26rft.issue%253D3%26rft.spage%253D457%26rft.epage%253D470%26rft.atitle%253DCross-Species%2BVirus%2BTransmission%2Band%2Bthe%2BEmergence%2Bof%2BNew%2BEpidemic%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00004-08%26rft_id%253Dinfo%253Apmid%252F18772285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Mi8zLzQ1NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzExLzIwMjAuMDMuMTAuOTg2MTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.14" data-doi="10.1073/pnas.96.24.13910"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-article-title">Mutation rates among RNA viruses</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">96</span>, <span class="cit-fpage">13910</span>–<span class="cit-lpage">13913</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDrake%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D96%26rft.issue%253D24%26rft.spage%253D13910%26rft.epage%253D13913%26rft.atitle%253DMutation%2Brates%2Bamong%2BRNA%2Bviruses%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.24.13910%26rft_id%253Dinfo%253Apmid%252F10570172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjQvMTM5MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMS8yMDIwLjAzLjEwLjk4NjEwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pèrez-Losada</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arenas</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galán</span>,  <span class="cit-name-given-names">J. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palero</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">González-Candelas</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Recombination in viruses: mechanisms, methods of study, and evolutionary consequences</span>. <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">296</span>–<span class="cit-lpage">307</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D30%26rft.spage%253D296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.16" data-doi="10.1101/gr.1970304"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jensen-Seaman</span>,  <span class="cit-name-given-names">M. I.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative recombination rates in the rat, mouse, and human genomes</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">528</span>–<span class="cit-lpage">538</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DJensen-Seaman%26rft.auinit1%253DM.%2BI.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D528%26rft.epage%253D538%26rft.atitle%253DComparative%2BRecombination%2BRates%2Bin%2Bthe%2BRat%252C%2BMouse%252C%2Band%2BHuman%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1970304%26rft_id%253Dinfo%253Apmid%252F15059993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzQvNTI4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTEvMjAyMC4wMy4xMC45ODYxMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.17" data-doi="10.1073/pnas.0500729102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silberg</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilke</span>,  <span class="cit-name-given-names">C. O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Arnold</span>,  <span class="cit-name-given-names">F. H.</span></span> <span class="cit-article-title">On the conservative nature of intragenic recombination</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">5380</span>–<span class="cit-lpage">5385</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0500729102%26rft_id%253Dinfo%253Apmid%252F15809422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE1LzUzODAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMS8yMDIwLjAzLjEwLjk4NjEwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.18" data-doi="10.1128/JVI.01624-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bull</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuan</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wilke</span>,  <span class="cit-name-given-names">C. O.</span></span> <span class="cit-article-title">Theory of lethal mutagenesis for viruses</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">2930</span>–<span class="cit-lpage">2939</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01624-06%26rft_id%253Dinfo%253Apmid%252F17202214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzYvMjkzMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzExLzIwMjAuMDMuMTAuOTg2MTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">C. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Spillover and pandemic properties of zoonotic viruses with high host plasticity</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">14830</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D5%26rft.spage%253D14830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ecker</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pancaldi</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valencia</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beck</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Paul</span>,  <span class="cit-name-given-names">D. S.</span></span> <span class="cit-article-title">Epigenetic and transcriptional variability shape phenotypic plasticity</span>. <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">1700148</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioessays%26rft.volume%253D40%26rft.spage%253D1700148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauber</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The footprint of genome architecture in the largest genome expansion in RNA viruses</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr> <span class="cit-vol">9</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.22" data-doi="10.1006/viro.2000.0645"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patton</span>,  <span class="cit-name-given-names">J. T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Spencer</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Genome replication and packaging of segmented double-stranded RNA viruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">277</span>, <span class="cit-fpage">217</span>–<span class="cit-lpage">225</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DPatton%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D277%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D225%26rft.atitle%253DGenome%2Breplication%2Band%2Bpackaging%2Bof%2Bsegmented%2Bdouble-stranded%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.2000.0645%26rft_id%253Dinfo%253Apmid%252F11080470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.2000.0645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11080470&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F11%2F2020.03.10.986109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165628300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.23" data-doi="10.1016/j.molcel.2016.05.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rivera-Mulia</span>,  <span class="cit-name-given-names">J. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">D. M.</span></span> <span class="cit-article-title">Replicating large genomes: divide and conquer</span>. <abbr class="cit-jnl-abbrev">Molecular cell</abbr> <span class="cit-vol">62</span>, <span class="cit-fpage">756</span>–<span class="cit-lpage">765</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.volume%253D62%26rft.spage%253D756%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2016.05.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2016.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.24" data-doi="10.1073/pnas.0508200103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minskaia</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of an RNA virus 3 5 exoribonuclease that is critically involved in coronavirus RNA synthesis</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">103</span>, <span class="cit-fpage">5108</span>–<span class="cit-lpage">5113</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508200103%26rft_id%253Dinfo%253Apmid%252F16549795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEzLzUxMDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMS8yMDIwLjAzLjEwLjk4NjEwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.26" data-doi="10.1038/nrmicro2692"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagy</span>,  <span class="cit-name-given-names">P. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pogany</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">The dependence of viral RNA replication on co-opted host factors</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">137</span>–<span class="cit-lpage">149</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DNagy%26rft.auinit1%253DP.%2BD.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D137%26rft.epage%253D149%26rft.atitle%253DThe%2Bdependence%2Bof%2Bviral%2BRNA%2Breplication%2Bon%2Bco-opted%2Bhost%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2692%26rft_id%253Dinfo%253Apmid%252F22183253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22183253&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F11%2F2020.03.10.986109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.27" data-doi="10.1038/nrmicro2614"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simon-Loriere</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span> <span class="cit-article-title">Why do RNA viruses recombine?</span> <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">617</span>–<span class="cit-lpage">626</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DSimon-Loriere%26rft.auinit1%253DE.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D617%26rft.epage%253D626%26rft.atitle%253DWhy%2Bdo%2BRNA%2Bviruses%2Brecombine%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2614%26rft_id%253Dinfo%253Apmid%252F21725337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21725337&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F11%2F2020.03.10.986109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.28" data-doi="10.1016/j.virol.2013.05.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rector</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Van Ranst</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Animal papillomaviruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">445</span>, <span class="cit-fpage">213</span>–<span class="cit-lpage">223</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D445%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2013.05.007%26rft_id%253Dinfo%253Apmid%252F23711385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2013.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23711385&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F11%2F2020.03.10.986109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Awadalla</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fearnhead</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">A coalescent-based method for detecting and estimating recombination from gene sequences</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>, <span class="cit-fpage">1231</span>–<span class="cit-lpage">1241</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1231%26rft.epage%253D1241%26rft.atitle%253DA%2BCoalescent-Based%2BMethod%2Bfor%2BDetecting%2Band%2BEstimating%2BRecombination%2BFrom%2BGene%2BSequences%26rft_id%253Dinfo%253Apmid%252F11901136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzEyMzEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMS8yMDIwLjAzLjEwLjk4NjEwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Archer</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identifying the important HIV-1 recombination breakpoints</span>. <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr> <span class="cit-vol">4</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986109v1.31" data-doi="10.1126/science.aap7463"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carroll</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The global virome project</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">359</span>, <span class="cit-fpage">872</span>–<span class="cit-lpage">874</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCarroll%26rft.auinit1%253DD.%26rft.volume%253D359%26rft.issue%253D6378%26rft.spage%253D872%26rft.epage%253D874%26rft.atitle%253DThe%2BGlobal%2BVirome%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap7463%26rft_id%253Dinfo%253Apmid%252F29472471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTkvNjM3OC84NzIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMS8yMDIwLjAzLjEwLjk4NjEwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.10.986109v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.10.981001v1" title="Non-canonical open reading frames encode functional proteins essential for cancer cell survival" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.974246v2" title="Multi-dimensional predictions of psychotic symptoms via machine learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 11, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.10.986109v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.10.986109v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/03/11/2020.03.10.986109.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.10.986109v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.10.986109v1" class="active" data-icon-position="" data-hide-link-title="0">Larger viral genome size facilitates emergence of zoonotic diseases</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1191318" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZKI_phvBr0qgbskKNBjHCXpEM2SK-YgHLcoVfGkagog" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1191318" id="node1191318--2" data-pisa="biorxiv;2020.03.10.986109v1" data-pisa-master="biorxiv;2020.03.10.986109" data-seqnum="1191318" data-apath="/biorxiv/early/2020/03/11/2020.03.10.986109.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Larger viral genome size facilitates emergence of zoonotic diseases</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Richard E.</span> <span class="nlm-surname">Grewelle</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.10.986109; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.10.986109 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;title=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;title=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;text=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;title=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;t=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;title=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;title=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1191318" id="node1191318--3" data-pisa="biorxiv;2020.03.10.986109v1" data-pisa-master="biorxiv;2020.03.10.986109" data-seqnum="1191318" data-apath="/biorxiv/early/2020/03/11/2020.03.10.986109.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Larger viral genome size facilitates emergence of zoonotic diseases</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Richard E.</span> <span class="nlm-surname">Grewelle</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.10.986109; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.10.986109 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Grewelle%2C+Richard+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1191318/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1191318/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1191318/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1191318/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1191318/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1191318/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1191318/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1191318/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1191318/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1191318/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1191318/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1191318/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;count=horizontal&amp;via=&amp;text=Larger%20viral%20genome%20size%20facilitates%20emergence%20of%20zoonotic%20diseases&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986109v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.10.986109v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
