<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Bioinformatics</span><span class="iso-abbrev" title="iso-abbrev">Bioinformatics</span><span class="publisher-id" title="publisher-id">bioinformatics</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Bioinformatics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1367-4803</span><span class="issn-epub" tagx="issn" title="issn-epub">1367-4811</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6247931">6247931</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/bioinformatics/bty407">10.1093/bioinformatics/bty407</a></span><span class="publisher-id" title="publisher-id">bty407</span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Applications Notes</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Nextstrain: real-time tracking of pathogen evolution</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">James</span></span><a href="#bty407-aff1">1</a><a href="#bty407-cor1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megill</span><span class="given-names" tagx="given-names" title="given-names">Colin</span></span><a href="#bty407-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">Sidney M</span></span><a href="#bty407-aff1">1</a><a href="#bty407-aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huddleston</span><span class="given-names" tagx="given-names" title="given-names">John</span></span><a href="#bty407-aff1">1</a><a href="#bty407-aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potter</span><span class="given-names" tagx="given-names" title="given-names">Barney</span></span><a href="#bty407-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callender</span><span class="given-names" tagx="given-names" title="given-names">Charlton</span></span><a href="#bty407-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sagulenko</span><span class="given-names" tagx="given-names" title="given-names">Pavel</span></span><a href="#bty407-aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">Trevor</span></span><a href="#bty407-aff1">1</a><a href="#bty407-FM2" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">Richard A</span></span><a href="#bty407-aff3">3</a><a href="#bty407-aff4">4</a><a href="#bty407-aff5">5</a><a href="#bty407-FM2" /></span></div><span class="citation_author_institution" id="bty407-aff1">[1], </span><span class="citation_author_institution" id="bty407-aff2">[2], </span><span class="citation_author_institution" id="bty407-aff3">[3], </span><span class="citation_author_institution" id="bty407-aff4">[4], </span><span class="citation_author_institution" id="bty407-aff5">[5], </span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelso</span><span class="given-names" tagx="given-names" title="given-names">Janet</span></span><span class="role" tagx="role" title="role">Associate Editor</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <p>The authors wish it to be known that, in their opinion, the Trevor Bedford and Richard A. Neher authors should be regarded as Joint last Authors.</p> </div> <div class="corresp" title="corresp">
To whom correspondence should be addressed. E-mail: <span class="email" tagx="email" title="email">jhadfiel@fredhutch.org</span></div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-12-12</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-5-5</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-5-5</span></span><span class="volume" tagx="volume" title="volume">34</span><span class="issue" tagx="issue" title="issue">23</span><span class="fpage" tagx="fpage" title="fpage">4121</span><span class="lpage" tagx="lpage" title="lpage">4123</span><span class="history" title="history"><span class="received" title="received">received: 2017-10-11</span><span class="rev-recd" title="rev-recd">rev-recd: 2018-4-20</span><span class="accepted" title="accepted">accepted: 2018-5-16</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a href="http://creativecommons.org/licenses/by-nc/4.0/">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</span></span></div><a class="self-uri" href="bty407.pdf" title="self-uri">bty407.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <div class="summary" title="sec"> <div class="title" tagx="title" title="title">
Summary</div> <p>Understanding the spread and evolution of pathogens is important for effective public health measures and surveillance. Nextstrain consists of a database of viral genomes, a bioinformatics pipeline for phylodynamics analysis, and an interactive visualization platform. Together these present a real-time view into the evolution and spread of a range of viral pathogens of high public health importance. The visualization integrates sequence data with other data types such as geographic information, serology, or host species. Nextstrain compiles our current understanding into a single accessible location, open to health professionals, epidemiologists, virologists and the public alike.</p> </div> <div class="availabilityandimplementation" title="sec"> <div class="title" tagx="title" title="title">
Availability and implementation</div> <p>All code (predominantly JavaScript and Python) is freely available from github.com/nextstrain and the web-application is available at nextstrain.org.</p> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Open Science Prize</span></div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">NSF</span><span class="named-content" title="named-content">10.13039/100000001</span></div><span class="award-id" tagx="award-id" title="award-id">DGE-1256082</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">SMB</span><span class="named-content" title="named-content">10.13039/100011211</span></div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">ERC</span><span class="named-content" title="named-content">10.13039/100010663</span></div><span class="award-id" tagx="award-id" title="award-id">StG-260686</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">RAN</span></div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">NIH</span><span class="named-content" title="named-content">10.13039/100000002</span></div><span class="award-id" tagx="award-id" title="award-id">R35 GM119774-01</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Pew Biomedical Scholar</span></div> </div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="1introduction" title="sec"> <div class="title" tagx="title" title="title">
1 Introduction</div> <p>Viral pathogens pose an ever-present danger to global human health, highlighted by recent events such as the West African Ebola epidemic and the ongoing Zika epidemic in the Americas. The rapid evolution of these viruses allows inference of epidemic history from genomic data. Such analyses are often done in isolation, and may lack the spatial or temporal context in which to best interpret the results (<a href="#bty407-B6">Pybus <i>et al.</i>, 2013</a>). Furthermore, the results of analyses are rarely made available to the public or health bodies until after publication, which may be too late to aid understanding or effect change in policy. We have developed Nextstrain to visualize outbreaks in as close to real time as possible. Whilst currently encompassing a selection of viruses, extension to non-viral pathogens is forthcoming.</p> <p>The regularly updated nature and rapidity of these analyses is crucial to the monitoring and understanding of pathogen epidemiology and evolution. Sequencing times and costs are continually dropping, with on-the-ground sequencing used during recent epidemics (<a href="#bty407-B3">Faria <i>et al.</i>, 2017</a>; <a href="#bty407-B7">Quick <i>et al.</i>, 2016</a>). Rapid methods by which to analyze, interpret, and disseminate results must complement this speed of sequencing.</p> <p>Nextstrain consists of data curation, analysis and visualization components: Python scripts maintain a database of available sequences and related metadata, sourced from public repositories such as NCBI (<a href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</a>), GISAID (<a href="http://www.gisaid.org">www.gisaid.org</a>) and ViPR (<a href="http://www.viprbrc.org">www.viprbrc.org</a>), as well as GitHub repositories and other sources of genomic data. A suite of tools perform phylodynamic analysis (<a href="#bty407-B10">Volz <i>et al.</i>, 2013</a>), including subsampling, alignment, phylogenetic inference, temporal dating of ancestral nodes and discrete trait geographic reconstruction, including inference of the most likely transmission events. This leverages the maximum likelihood analyses implemented in TreeTime (<a href="#bty407-B8">Sagulenko <i>et al.</i>, 2018</a>), allowing a full analysis of the entire Ebola epidemic (<i>n </i>=<i> </i>1581 genomes) in under 2 h on a modern laptop. These scripts separate generic core functionality from a light pathogen-specific layer such that they are easily adapted to different pathogens. Visualization is available through nextstrain.org. This approach is similar in concept to Nextflu (<a href="#bty407-B5">Neher and Bedford, 2015</a>) however extended and generalized to different viral pathogens. There is a growing need for surveillance of non-influenza viruses (<a href="#bty407-B9">Tang <i>et al.</i>, 2017</a>), and Nextstrain is able to be extended to most outbreaks with readily accessible genomic data, although we note the potential for recombination or low mutation rate to confound phylogenetic signal.</p> </div> <div class="2jointtemporalandspatialvisualization" title="sec"> <div class="title" tagx="title" title="title">
2 Joint temporal and spatial visualization</div> <p>Conveying understanding of pathogen evolution through space and time involves filtering large amounts of data into forms that can be easily reasoned with. Multiple views into different facets of the data are presented and remain in sync as one interacts with the data. This allows simultaneous interrogation of phylogenetic and spatial relationships, with additional data such as genotype or serotype expressed through colourings (<a href="#bty407-F1">Fig. 1</a>). This is coupled with an interactive time slider to see how the pathogen has evolved and spread over the course of the epidemic. By animating the temporal dimension, a high level overview of how the entire outbreak unfolded is quickly gained. This approach both communicates the geographical spread of the epidemic alongside the underlying genomic data that supports this geographic reconstruction. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1.</span></a></div>   <p>Genomic epidemiology of Zika virus as of Oct 2017 (live display at nextstrain.org/zika). The main interface consists of three linked panels—a phylogenetic tree, geographic transmissions and the genetic diversity across the genome</p>   </div> <p>Maximum likelihood ancestral state reconstruction of discrete traits such as country or region of isolation allows identification of probable transmission events given the sampled data, together with inferred probability distributions of ancestral state at each node. Internal node colours indicate ancestral state and shifts are drawn as links between demes on the map. Confidence is conveyed by matching colour saturation to the confidence of that trait, and by displaying all relevant information when one hovers over the corresponding branch or isolate on the tree. Sampling bias and lack of data can obscure transmission links, and in certain cases we have chosen not to display the inferred states.</p> </div> <div class="3monitoringofevolutionandadaptation" title="sec"> <div class="title" tagx="title" title="title">
3 Monitoring of evolution and adaptation</div> <p>Nextstrain tracks and reconstructs mutations across the tree and displays this information as a bar chart of entropy at each position in the genome, as well as showing the mutations inferred to occur on each branch by hovering over the tree. Selecting a position in the genome with non-zero entropy reveals the distribution of the segregating variant in the phylogeny and on the map. This allows interrogation of genetic change which may be adaptive or underlying a change in disease dynamics.</p> <p>For many pathogens, the emergence and spread of gain-of-function variants is a grave concern. For instance, China has experienced seasonal epidemics of influenza A/H7N9 over the past five years. Despite no known human-to human-transmission events, the high mortality rate of 30% (<a href="#bty407-B4">Li <i>et al.</i>, 2014</a>) makes the threat of mutations which facilitate human-to-human transmission of extreme concern. For example, mutations identified by <a href="#bty407-B2">de Vries <i>et al.</i> (2017)</a> are readily visible at nextstrain.org/avian/h7n9. Continual monitoring of such putatively adaptive mutations is critical.</p> </div> <div class="4amodelforpublicsharingofdata" title="sec"> <div class="title" tagx="title" title="title">
4 A model for public sharing of data</div> <p>Nextstrain presents a single, continuously updated overview of both endemic viral disease (seasonal influenza, dengue) as well as emergent viral outbreaks (avian influenza, Zika, Ebola), all based upon the same underlying bioinformatics architecture. This architecture is well positioned to respond to future outbreaks, be they viral or bacterial.</p> <p>Analysis of such outbreaks relies on public sharing of data, and Nextstrain has the ability to automatically update as new sequences from a range of public databases and repositories appear. Scientists are justifiably hesitant to cede control of their data, and we try to address these concerns by preventing access to the raw genome sequences, and by clearly indicating the source of each sequence. Derived data, such as phylogenetic trees, metadata and screenshots are available, and one can append private metadata via CSV files. We believe this strikes a compromise between keeping certain data private and allowing the dissemination of results important to the wider scientific community, thereby encouraging collaboration between scientists. Genomic epidemiology has the potential to inform the public, health organisations and scientists alike, a potential realized by sharing of data in real-time rather than retrospectively (<a href="#bty407-B1">Croucher and Didelot, 2015</a>).</p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by the Open Science Prize to TB and RAN, by the NSF through DGE-1256082 to SMB, by the ERC through StG-260686 to RAN and by NIH R35 GM119774-01 to TB. TB is a Pew Biomedical Scholar.</p> <p><i>Conflict of Interest</i>: none declared.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bty407-B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croucher</span><span class="given-names" tagx="given-names" title="given-names">N.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Didelot</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">The application of genomics to tracing bacterial pathogen transmission</span>. <span class="source" tagx="source" title="source">Curr. Opin. Microbiol</span>., <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">62</span>–<span class="lpage" tagx="lpage" title="lpage">67</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25461574">25461574</a></span></span></li> <li tag="ref"><a name="bty407-B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vries</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Three mutations switch H7N9 influenza to human-type receptor specificity</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>., <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">e1006390</span>–<span class="lpage" tagx="lpage" title="lpage">e1006316</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28617868">28617868</a></span></span></li> <li tag="ref"><a name="bty407-B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">N.R.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Epidemic establishment and cryptic transmission of Zika virus in Brazil and the Americas</span>. <span class="source" tagx="source" title="source">Nature</span>, <span class="volume" tagx="volume" title="volume">546</span>, <span class="fpage" tagx="fpage" title="fpage">406</span>–<span class="lpage" tagx="lpage" title="lpage">410</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28538727">28538727</a></span></span></li> <li tag="ref"><a name="bty407-B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Epidemiology of human infections with avian influenza A(H7N9) virus in China</span>. <span class="source" tagx="source" title="source">N. Engl. J. Med</span>., <span class="volume" tagx="volume" title="volume">370</span>, <span class="fpage" tagx="fpage" title="fpage">520</span>–<span class="lpage" tagx="lpage" title="lpage">532</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23614499">23614499</a></span></span></li> <li tag="ref"><a name="bty407-B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">nextflu: real-time tracking of seasonal influenza virus evolution in humans</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">3546</span>–<span class="lpage" tagx="lpage" title="lpage">3548</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26115986">26115986</a></span></span></li> <li tag="ref"><a name="bty407-B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Evolutionary epidemiology: preparing for an age of genomic plenty</span>. <span class="source" tagx="source" title="source">Phil. Trans. R Soc. B</span>, <span class="volume" tagx="volume" title="volume">368</span>, <span class="fpage" tagx="fpage" title="fpage">20120193</span>–<span class="lpage" tagx="lpage" title="lpage">20120193</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382418">23382418</a></span></span></li> <li tag="ref"><a name="bty407-B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Real-time, portable genome sequencing for Ebola surveillance</span>. <span class="source" tagx="source" title="source">Nature</span>, <span class="volume" tagx="volume" title="volume">530</span>, <span class="fpage" tagx="fpage" title="fpage">228</span>–<span class="lpage" tagx="lpage" title="lpage">232</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26840485">26840485</a></span></span></li> <li tag="ref"><a name="bty407-B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sagulenko</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">Treetime: maximum-likelihood phylodynamic analysis</span>. <span class="source" tagx="source" title="source">Virus Evol</span>., <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">vex042.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29340210">29340210</a></span></span></li> <li tag="ref"><a name="bty407-B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Global epidemiology of non-influenza RNA respiratory viruses: data gaps and a growing need for surveillance</span>. <span class="source" tagx="source" title="source">Lancet Infect. Dis</span>., <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">e320</span>–<span class="lpage" tagx="lpage" title="lpage">e327</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28457597">28457597</a></span></span></li> <li tag="ref"><a name="bty407-B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E.M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Viral phylodynamics</span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol</span>., <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e1002947.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555203">23555203</a></span></span></li> </ul> </div> </div>  </body></html>