<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/21/2020.02.21.959817.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.21.959817v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.21.959817"/>
  <meta name="DC.Date" content="2020-02-21"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Reverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in E. coli hosts due to size and occasional instability[1][1]-[3][2]. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the Coronaviridae, Flaviviridae and Paramyxoviridae families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in Saccharomyces cerevisiae using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-2[4][3] in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak. [1]: #ref-1 [2]: #ref-3 [3]: #ref-4"/>
  <meta name="DC.Contributor" content="Tran Thi Nhu Thao"/>
  <meta name="DC.Contributor" content="Fabien Labroussaa"/>
  <meta name="DC.Contributor" content="Nadine Ebert"/>
  <meta name="DC.Contributor" content="Philip V’kovski"/>
  <meta name="DC.Contributor" content="Hanspeter Stalder"/>
  <meta name="DC.Contributor" content="Jasmine Portmann"/>
  <meta name="DC.Contributor" content="Jenna Kelly"/>
  <meta name="DC.Contributor" content="Silvio Steiner"/>
  <meta name="DC.Contributor" content="Melle Holwerda"/>
  <meta name="DC.Contributor" content="Annika Kratzel"/>
  <meta name="DC.Contributor" content="Mitra Gultom"/>
  <meta name="DC.Contributor" content="Laura Laloli"/>
  <meta name="DC.Contributor" content="Linda Hüsser"/>
  <meta name="DC.Contributor" content="Manon Wider"/>
  <meta name="DC.Contributor" content="Stephanie Pfaender"/>
  <meta name="DC.Contributor" content="Dagny Hirt"/>
  <meta name="DC.Contributor" content="Valentina Cippà"/>
  <meta name="DC.Contributor" content="Silvia Crespo-Pomar"/>
  <meta name="DC.Contributor" content="Simon Schröder"/>
  <meta name="DC.Contributor" content="Doreen Muth"/>
  <meta name="DC.Contributor" content="Daniela Niemeyer"/>
  <meta name="DC.Contributor" content="Marcel A. Müller"/>
  <meta name="DC.Contributor" content="Christian Drosten"/>
  <meta name="DC.Contributor" content="Ronald Dijkman"/>
  <meta name="DC.Contributor" content="Joerg Jores"/>
  <meta name="DC.Contributor" content="Volker Thiel"/>
  <meta name="article:published_time" content="2020-02-21"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Reverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in &lt;;i&gt;;E. coli&lt;;/i&gt;; hosts due to size and occasional instability&lt;;sup&gt;;1-3&lt;;/sup&gt;;. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the &lt;;i&gt;;Coronaviridae, Flaviviridae&lt;;/i&gt;; and &lt;;i&gt;;Paramyxoviridae&lt;;/i&gt;; families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in &lt;;i&gt;;Saccharomyces cerevisiae&lt;;/i&gt;; using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-2&lt;;sup&gt;;4&lt;;/sup&gt;; in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.21.959817v1"/>
  <meta name="citation_id" content="2020.02.21.959817v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959817v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959817v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959817v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.21.959817"/>
  <meta name="citation_num_pages" content="29"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.21.959817"/>
  <meta name="citation_author" content="Tran Thi Nhu Thao"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Graduate School for Biomedical Science, University of Bern"/>
  <meta name="citation_author" content="Fabien Labroussaa"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Institute of Veterinary Bacteriology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Nadine Ebert"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Philip V’kovski"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Hanspeter Stalder"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Jasmine Portmann"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Jenna Kelly"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Silvio Steiner"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Graduate School for Biomedical Science, University of Bern"/>
  <meta name="citation_author" content="Melle Holwerda"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Graduate School for Biomedical Science, University of Bern"/>
  <meta name="citation_author_institution" content="Insitute for Infectious Diseases, University of Bern"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9814-7793"/>
  <meta name="citation_author" content="Annika Kratzel"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Graduate School for Biomedical Science, University of Bern"/>
  <meta name="citation_author" content="Mitra Gultom"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Graduate School for Biomedical Science, University of Bern"/>
  <meta name="citation_author_institution" content="Insitute for Infectious Diseases, University of Bern"/>
  <meta name="citation_author" content="Laura Laloli"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Graduate School for Biomedical Science, University of Bern"/>
  <meta name="citation_author_institution" content="Insitute for Infectious Diseases, University of Bern"/>
  <meta name="citation_author" content="Linda Hüsser"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Manon Wider"/>
  <meta name="citation_author_institution" content="Insitute for Infectious Diseases, University of Bern"/>
  <meta name="citation_author" content="Stephanie Pfaender"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Dagny Hirt"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Valentina Cippà"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Institute of Veterinary Bacteriology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Silvia Crespo-Pomar"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Institute of Veterinary Bacteriology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author" content="Simon Schröder"/>
  <meta name="citation_author_institution" content="Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health"/>
  <meta name="citation_author" content="Doreen Muth"/>
  <meta name="citation_author_institution" content="Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health"/>
  <meta name="citation_author_institution" content="German Centre for Infection Research, associated partner Charité"/>
  <meta name="citation_author" content="Daniela Niemeyer"/>
  <meta name="citation_author_institution" content="Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health"/>
  <meta name="citation_author_institution" content="German Centre for Infection Research, associated partner Charité"/>
  <meta name="citation_author" content="Marcel A. Müller"/>
  <meta name="citation_author_institution" content="Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health"/>
  <meta name="citation_author_institution" content="German Centre for Infection Research, associated partner Charité"/>
  <meta name="citation_author_institution" content="Martsinovsky Institute of Medical Parasitology, Tropical and Vector Borne Diseases, Sechenov University"/>
  <meta name="citation_author" content="Christian Drosten"/>
  <meta name="citation_author_institution" content="Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health"/>
  <meta name="citation_author_institution" content="German Centre for Infection Research, associated partner Charité"/>
  <meta name="citation_author" content="Ronald Dijkman"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Insitute for Infectious Diseases, University of Bern"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0320-2743"/>
  <meta name="citation_author" content="Joerg Jores"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_institution" content="Institute of Veterinary Bacteriology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_email" content="volker.thiel@vetsuisse.unibe.ch"/>
  <meta name="citation_author_email" content="joerg.jores@vetsuisse.unibe.ch"/>
  <meta name="citation_author" content="Volker Thiel"/>
  <meta name="citation_author_institution" content="Institute of Virology and Immunology (IVI)"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern"/>
  <meta name="citation_author_email" content="volker.thiel@vetsuisse.unibe.ch"/>
  <meta name="citation_author_email" content="joerg.jores@vetsuisse.unibe.ch"/>
  <meta name="citation_reference" content="Almazan, F. et al. Engineering the largest RNA virus genome as an infectious bacterial artificial chromosome. Proc Natl Acad Sci U S A 97, 5516–5521, doi:10.1073/pnas.97.10.5516 (2000)."/>
  <meta name="citation_reference" content="Thiel, V., Herold, J., Schelle, B. &amp;; Siddell, S. G. Infectious RNA transcribed in vitro from a cDNA copy of the human coronavirus genome cloned in vaccinia virus. J Gen Virol 82, 1273–1281 (2001)."/>
  <meta name="citation_reference" content="Yount, B., Curtis, K. M. &amp;; Baric, R. S. Strategy for systematic assembly of large RNA and DNA genomes: transmissible gastroenteritis virus model. J Virol 74, 10600–10611, doi:10.1128/jvi.74.22.10600-10611.2000 (2000)."/>
  <meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med, doi:10.1056/NEJMoa2001017 (2020)."/>
  <meta name="citation_reference" content="Zaki, A. M., van Boheemen, S., Bestebroer, T. M., Osterhaus, A. D. &amp;; Fouchier, R. A. Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N Engl J Med 367, 1814–1820, doi:10.1056/NEJMoa1211721 (2012)."/>
  <meta name="citation_reference" content="Cao-Lormeau, V. M. et al. Zika virus, French polynesia, South pacific, 2013. Emerg Infect Dis 20, 1085–1086, doi:10.3201/eid2006.140138 (2014)."/>
  <meta name="citation_reference" content="Baize, S. et al. Emergence of Zaire Ebola virus disease in Guinea. N Engl J Med 371, 1418–1425, doi:10.1056/NEJMoa1404505 (2014)."/>
  <meta name="citation_reference" content="Gibson, D. G. et al. Creation of a Bacterial Cell Controlled by a Chemically Synthesized Genome. Science 329, 52–56, doi:science.1190719 [pii] 10.1126/science.1190719 (2010)."/>
  <meta name="citation_reference" content="Lartigue, C. et al. Creating bacterial strains from genomes that have been cloned and engineered in yeast. Science 325, 1693–1696, doi:1173759 [pii] 10.1126/science.1173759 (2009)."/>
  <meta name="citation_reference" content="Benders, G. A. et al. Cloning whole bacterial genomes in yeast. Nucleic Acids Res 38, 2558–2569, doi:10.1093/nar/gkq119 (2010)."/>
  <meta name="citation_reference" content="Hutchison, C. A., 3rd et al. Design and synthesis of a minimal bacterial genome. Science 351, aad6253, doi:10.1126/science.aad6253 (2016)."/>
  <meta name="citation_reference" content="Kouprina, N. &amp;; Larionov, V. Selective isolation of genomic loci from complex genomes by transformation-associated recombination cloning in the yeast Saccharomyces cerevisiae. Nat Protoc 3, 371–377, doi:10.1038/nprot.2008.5 (2008)."/>
  <meta name="citation_reference" content="Vashee, S. et al. Cloning, Assembly, and Modification of the Primary Human Cytomegalovirus Isolate Toledo by Yeast-Based Transformation-Associated Recombination. mSphere 2, doi:10.1128/mSphereDirect.00331-17 (2017)."/>
  <meta name="citation_reference" content="Oldfield, L. M. et al. Genome-wide engineering of an infectious clone of herpes simplex virus type 1 using synthetic genomics assembly methods. Proc Natl Acad Sci U S A 114, E8885–E8894, doi:10.1073/pnas.1700534114 (2017)."/>
  <meta name="citation_reference" content="Coley, S. E. et al. Recombinant mouse hepatitis virus strain A59 from cloned, full-length cDNA replicates to high titers in vitro and is fully pathogenic in vivo. J Virol 79, 3097–3106, doi:10.1128/JVI.79.5.3097-3106.2005 (2005)."/>
  <meta name="citation_reference" content="Zust, R. et al. Coronavirus non-structural protein 1 is a major pathogenicity factor: implications for the rational design of coronavirus vaccines. PLoS Pathog 3, e109, doi:10.1371/journal.ppat.0030109 (2007)."/>
  <meta name="citation_reference" content="Muth, D. et al. Transgene expression in the genome of Middle East respiratory syndrome coronavirus based on a novel reverse genetics system utilizing Red-mediated recombination cloning. J Gen Virol 98, 2461–2469, doi:10.1099/jgv.0.000919 (2017)."/>
  <meta name="citation_reference" content="Hu, D. et al. Genomic characterization and infectivity of a novel SARS-like coronavirus in Chinese bats. Emerg Microbes Infect 7, 154, doi:10.1038/s41426-018-0155-5 (2018)."/>
  <meta name="citation_reference" content="Thiel, V. et al. Mechanisms and enzymes involved in SARS coronavirus genome expression. J Gen Virol 84, 2305–2315, doi:10.1099/vir.0.19424-0 (2003)."/>
  <meta name="citation_reference" content="van den Worm, S. H. et al. Reverse genetics of SARS-related coronavirus using vaccinia virus-based recombination. PLoS One 7, e32857, doi:10.1371/journal.pone.0032857 (2012)."/>
  <meta name="citation_reference" content="Polo, S., Ketner, G., Levis, R. &amp;; Falgout, B. Infectious RNA transcripts from full-length dengue virus type 2 cDNA clones made in yeast. J Virol 71, 5366–5374 (1997)."/>
  <meta name="citation_reference" content="Chan, J. F. et al. Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan. Emerg Microbes Infect 9, 221–236, doi:10.1080/22221751.2020.1719902 (2020)."/>
  <meta name="twitter:title" content="Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Reverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in E. coli hosts due to size and occasional instability[1][1]-[3][2]. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the Coronaviridae, Flaviviridae and Paramyxoviridae families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in Saccharomyces cerevisiae using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-2[4][3] in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak. [1]: #ref-1 [2]: #ref-3 [3]: #ref-4"/>
  <meta name="og-title" property="og:title" content="Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.21.959817v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Reverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in E. coli hosts due to size and occasional instability[1][1]-[3][2]. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the Coronaviridae, Flaviviridae and Paramyxoviridae families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in Saccharomyces cerevisiae using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-2[4][3] in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak. [1]: #ref-1 [2]: #ref-3 [3]: #ref-4"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-21"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1162361 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Reverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in 
     <em>E. coli</em> hosts due to size and occasional instability
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the 
     <em>Coronaviridae, Flaviviridae</em> and 
     <em>Paramyxoviridae</em> families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in 
     <em>Saccharomyces cerevisiae</em> using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-2
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup> in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Main Text</h2>
    <p id="p-4">Population growth, increased travel and climate change foster epidemic and pandemic threats by (re)emerging RNA viruses. Although black swan events cannot be predicted, our medical sector should be ready to use the latest advancements in science to combat emerging viruses. Within the past decade we have seen outbreaks of Middle-East Respiratory Syndrome coronavirus (MERS-CoV)
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>, ZIKA virus (ZIKV)
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>, Ebola virus
     <sup>
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup>, and at the end of 2019, SARS-CoV-2 that was first detected in Wuhan, Hubei province, China
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup>, but meanwhile spread throughout China and beyond. Until now SARS-CoV-2 isolates from China have not been shared with health authorities and the scientific community, although they are urgently needed to build up serological diagnostics, to develop and assess antivirals and vaccines, and to establish appropriate 
     <em>in vivo</em> models. Generation of SARS-CoV-2 from chemically synthesized DNA could bypass the limited availability of virus isolates and would furthermore allow genetic modifications and functional characterization of individual genes. However, although the workhorse 
     <em>E. coli</em> did prove very useful for cloning many viral genomes, it has drawbacks for a number of emerging RNA viruses, including coronaviruses, in terms of assembling and stably maintaining full-length molecular clones.
    </p>
    <p id="p-5">Synthetic genomics is a field that was fuelled by the efforts to create a bacterial cell that is controlled by a synthetic genome 
     <sup>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>. Genome-wide reassembly of the ∼1.1Mb 
     <em>Mycoplasma</em> genome was first attempted using 
     <em>E. coli</em> as an intermediate host 
     <sup>
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>
     </sup>, but the maintenance of these 100-kbp DNA fragments appeared to be very difficult in this host. Therefore, the yeast 
     <em>Saccharomyces cerevisiae</em> was chosen to clone, assemble and even mutagenize entire 
     <em>Mycoplasma</em> genomes 
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup>. This ultimately led to the construction of the Mycoplasma JCVI Syn3.0, a cell carrying the minimal set of genes allowing its autonomous replication in auxotrophic medium 
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>. The rationale for using a yeast cloning system is the ability of yeast to recombine overlapping DNA fragments 
     <em>in cellulo</em>, which led to the development of a technique called “transformation-associated recombination” (TAR) cloning
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup>.
    </p>
    <p id="p-6">More recently, TAR cloning was successfully applied for the assembly, genetic engineering and rescue of large DNA viruses such as cytomegalovirus and herpes simplex virus 1
     <sup>
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup>. While large DNA viruses such as herpesviruses can be rescued by transfecting the entire viral DNA genome into eukaryotic cells, the rescue of RNA viruses requires the transcription of a viral genomic RNA from the cloned DNA template. For coronaviruses that belong to a family of positive-stranded RNA viruses termed 
     <em>Coronaviridae</em>, the generation of full-length molecular clones has long been hampered by the extraordinary genome size (27-31 kb) and occasional instability of cloned DNA in 
     <em>E. coli</em>. It took until 2000/2001 before unconventional approaches, such as cloning in low copy bacterial artificial chromosomes (BACs) or vaccinia virus, or cloning of subgenomic DNA fragments followed by 
     <em>in-vitro</em> ligation, were successful
     <sup>
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>-
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
     </sup>. However, each system has caveats leaving the generation of recombinant coronaviruses cumbersome. We therefore aimed to assess the use of the yeast 
     <em>S. cerevisiae</em> to assemble and maintain genomes of diverse RNA viruses in order to establish a rapid, stable, and universal reverse genetics pipeline for RNA viruses.
    </p>
    <p id="p-7">In an effort to adopt a yeast-based reverse genetics platform to RNA viruses we first tested the Murine hepatitis virus strain A59 containing the gene for the green fluorescent protein (MHV-GFP), which is routinely used in our laboratory and has an established vaccinia virus-based reverse genetics platform
     <sup>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
     </sup>. The overall strategy is shown in 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a>. Viral RNA was prepared from MHV-GFP-infected murine 17Cl-1 cells and used to amplify 7 overlapping DNA fragments by RT-PCR together spanning the MHV-GFP genome from nt 2024 to nt 29672. Fragments containing the 5’- and 3’-termini were PCR-amplified from the vaccinia virus-cloned genome to include a T7-RNA polymerase promoter and a cleavage site (
     <em>Pac</em>I) following the polyA sequence. Overlaps to the TAR vector pVC604 were included in the primers used to amplify the 5’- and 3’-terminal fragments and the pVC604 backbone that was also generated by PCR (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b</a>, 
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, Extended Data Table 1, Extended Data Figure 1a). All DNA fragments were used to transform 
     <em>Saccharomyces cerevisiae</em> (strain VL6-48N) and resulting colonies were screened for correct assembly of the YAC containing the MHV genome by multiplex PCRs that covered the junctions of the different fragments. This screen revealed that &gt;; 90 % of the clones tested were positive indicating a highly efficient assembly in yeast (Extended Data Figure 1a). To rescue MHV-GFP we randomly chose two clones, purified the YAC, linearized the plasmid using the restriction endonuclease 
     <em>Pac</em>I (Extended Data Table 2) and subjected it to T7 RNA polymerase-based 
     <em>in vitro</em> transcription to generate capped viral genomic RNA. This RNA was transfected together with an 
     <em>in vitro</em> transcribed mRNA encoding the MHV nucleocapsid (N) protein into BHK-MHV-N cells as previously described
     <sup>
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
     </sup>. Transfected BHK-MHV-N cells were mixed with MHV-susceptible 17Cl-1 cells and cytopathic effect (CPE), virus-induced syncytia, and GFP-expressing cells were readily detectable for both clones within 48 hours, indicating successful recovery of infectious virus (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1c</a>). Finally, we assessed the replication kinetics of the recovered viruses, which were indistinguishable compared to the parental MHV-GFP (
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1d</a>).
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1166038/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1166038/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1166038" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1:</span>
      <p id="p-8" class="first-child">RNA virus genomes cloned using the synthetic genomics platform</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Application of yeast-based TAR cloning to generate viral cDNA clones and the recovery of recombinant MHV-GFP. a, Schematic representation of the general workflow of TAR cloning and virus rescue. In-yeast assembly require the delivery of overlapping DNA fragments covering the viral genome and a TAR vector into yeast in one transformation event. Transformed DNA fragments are joined by homologous recombination in yeast generating a YAC-cloned full-length viral genomic cDNA. The in vitro production of infectious capped viral RNA starts with the isolation of the YAC, followed by plasmid linearization to provide DNA template for run-off T7 polymerase-based transcription. Virus rescue starts with electroporation of BHK-21 and/or BHK-21 cells expressing the corresponding coronavirus N protein, followed by co-cultivation of electroporated cells with susceptible cell line for virus production and amplification. b, Schematic representation of MHV-GFP genome organisation with 9 viral subgenomic overlapping cDNA fragments used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. c, Recovery of infectious recombinant MHV-GFP. Cell culture supernatants containing viruses produced after virus rescue from two MHV-GFP YAC clones (Clone1, Clone 2) were used to infect 17Cl-1 cells. At 48 hours post-infection, infected cells were visualised for GFP expression (left panels) and by bright field microscopy (right panels). Mock represents 17Cl-1 cells inoculated with supernatant from BHK-MHV-N cells electroporated without viral RNAs. d, Replication kinetics of parental MHV-GFP and recombinant MHV-GFP Clone 1 and Clone 2. L929 cells were infected at an MOI of 0.1. Cell culture supernatants were harvested at indicated time points post-infection and titrated by plaque assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. pfu/ml, plaque forming units per millilitre." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-345331397" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Application of yeast-based TAR cloning to generate viral cDNA clones and the recovery of recombinant MHV-GFP.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, Schematic representation of the general workflow of TAR cloning and virus rescue. In-yeast assembly require the delivery of overlapping DNA fragments covering the viral genome and a TAR vector into yeast in one transformation event. Transformed DNA fragments are joined by homologous recombination in yeast generating a YAC-cloned full-length viral genomic cDNA. The &lt;;em&gt;;in vitro&lt;;/em&gt;; production of infectious capped viral RNA starts with the isolation of the YAC, followed by plasmid linearization to provide DNA template for run-off T7 polymerase-based transcription. Virus rescue starts with electroporation of BHK-21 and/or BHK-21 cells expressing the corresponding coronavirus N protein, followed by co-cultivation of electroporated cells with susceptible cell line for virus production and amplification. &lt;;strong&gt;;b&lt;;/strong&gt;;, Schematic representation of MHV-GFP genome organisation with 9 viral subgenomic overlapping cDNA fragments used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. &lt;;strong&gt;;c&lt;;/strong&gt;;, Recovery of infectious recombinant MHV-GFP. Cell culture supernatants containing viruses produced after virus rescue from two MHV-GFP YAC clones (Clone1, Clone 2) were used to infect 17Cl-1 cells. At 48 hours post-infection, infected cells were visualised for GFP expression (left panels) and by bright field microscopy (right panels). Mock represents 17Cl-1 cells inoculated with supernatant from BHK-MHV-N cells electroporated without viral RNAs. &lt;;strong&gt;;d&lt;;/strong&gt;;, Replication kinetics of parental MHV-GFP and recombinant MHV-GFP Clone 1 and Clone 2. L929 cells were infected at an MOI of 0.1. Cell culture supernatants were harvested at indicated time points post-infection and titrated by plaque assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. pfu/ml, plaque forming units per millilitre.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F1.medium.gif" width="440" height="283"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F1.medium.gif" width="440" height="283"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <span class="caption-title">Application of yeast-based TAR cloning to generate viral cDNA clones and the recovery of recombinant MHV-GFP.</span>
      <p id="p-9" class="first-child">
       <strong>a</strong>, Schematic representation of the general workflow of TAR cloning and virus rescue. In-yeast assembly require the delivery of overlapping DNA fragments covering the viral genome and a TAR vector into yeast in one transformation event. Transformed DNA fragments are joined by homologous recombination in yeast generating a YAC-cloned full-length viral genomic cDNA. The 
       <em>in vitro</em> production of infectious capped viral RNA starts with the isolation of the YAC, followed by plasmid linearization to provide DNA template for run-off T7 polymerase-based transcription. Virus rescue starts with electroporation of BHK-21 and/or BHK-21 cells expressing the corresponding coronavirus N protein, followed by co-cultivation of electroporated cells with susceptible cell line for virus production and amplification. 
       <strong>b</strong>, Schematic representation of MHV-GFP genome organisation with 9 viral subgenomic overlapping cDNA fragments used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. 
       <strong>c</strong>, Recovery of infectious recombinant MHV-GFP. Cell culture supernatants containing viruses produced after virus rescue from two MHV-GFP YAC clones (Clone1, Clone 2) were used to infect 17Cl-1 cells. At 48 hours post-infection, infected cells were visualised for GFP expression (left panels) and by bright field microscopy (right panels). Mock represents 17Cl-1 cells inoculated with supernatant from BHK-MHV-N cells electroporated without viral RNAs. 
       <strong>d</strong>, Replication kinetics of parental MHV-GFP and recombinant MHV-GFP Clone 1 and Clone 2. L929 cells were infected at an MOI of 0.1. Cell culture supernatants were harvested at indicated time points post-infection and titrated by plaque assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. pfu/ml, plaque forming units per millilitre.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-10">To address if the synthetic genomics platform can be applied to other coronaviruses, and if it can be used for rapid mutagenesis, we used a molecular BAC clone of MERS-CoV
     <sup>
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
     </sup>. We PCR-amplified eight subgenomic overlapping DNA fragments covering the entire MERS-CoV genome (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>, Extended Data Figure 1b, Extended Data Table 1). The 5’- and 3’- terminal DNA fragments contained the T7-RNA polymerase promoter upstream the MERS-CoV 5’-end and a restriction endonuclease cleavage site 
     <em>MluI</em> downstream the polyA sequence, and overlapping sequences with the TAR plasmid pVC604. To mutagenize the MERS-CoV clone and introduce the GFP gene, fragment 7 was modified to overlap with the PCR-amplified GFP gene. Following transformation almost all YAC clones were positive for the junctions indicating a successful assembly (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a>, Extended Data Figure 1b). Virus rescue from cloned DNA was performed as described previously
     <sup>
      <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>
     </sup> and virus-induced CPE became apparent in VeroB4 cells inoculated with recovered recombinant MERS-CoV (
     <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2b</a>). Importantly, we could readily detect green fluorescent VeroB4 cells that had been infected with recovered MERS-CoV-GFP demonstrating that the synthetic genomics platform is suitable to genetically modify coronavirus genomes (
     <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2b</a>). As expected replication kinetics of recombinant MERS-CoV and MERS-CoV-GFP were slightly reduced compared to cell-culture-adapted MERS-CoV-EMC (
     <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2c</a>).
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Generation of viral cDNA clones and recovery of recombinant MERS-CoV and MERS-CoV-GFP. a, Schematic representation of MERS-CoV (upper panel) and MERS-CoV-GFP (lower panel) genome organization with 8 and 10 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. b, Rescue of recombinant MERS-CoV and MERS-CoV-GFP. Following delivery of viral RNAs into BHK-21 cells via electroporation the cells were co-cultivated with VeroB4 cells and supernatants containing recombinant viruses produced were used to infect new VeroB4 cells. Infected cells were visualised by bright field microscopy for recombinant MERS-CoV (upper panel; 5 days post infection), and for GFP expression for recombinant MERS-CoV-GFP (lower panel, 3 days post infection). Mock represents VeroB4 cells inoculated with supernatant from BHK-21 cells electroporated without viral RNAs. c, Replication kinetics of the MERS-CoV-EMC (laboratory adapted isolate), rMERS-CoV and rMERS-CoV-GFP. VeroB4 cells were infected at an MOI of 0.01. Cell culture supernatants were harvested at indicated time points post infection and titrated by TCID50 assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. TCID50/ml, 50% tissue culture infectious dose per millilitre." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-345331397" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Generation of viral cDNA clones and recovery of recombinant MERS-CoV and MERS-CoV-GFP.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, Schematic representation of MERS-CoV (upper panel) and MERS-CoV-GFP (lower panel) genome organization with 8 and 10 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. &lt;;strong&gt;;b&lt;;/strong&gt;;, Rescue of recombinant MERS-CoV and MERS-CoV-GFP. Following delivery of viral RNAs into BHK-21 cells via electroporation the cells were co-cultivated with VeroB4 cells and supernatants containing recombinant viruses produced were used to infect new VeroB4 cells. Infected cells were visualised by bright field microscopy for recombinant MERS-CoV (upper panel; 5 days post infection), and for GFP expression for recombinant MERS-CoV-GFP (lower panel, 3 days post infection). Mock represents VeroB4 cells inoculated with supernatant from BHK-21 cells electroporated without viral RNAs. &lt;;strong&gt;;c&lt;;/strong&gt;;, Replication kinetics of the MERS-CoV-EMC (laboratory adapted isolate), rMERS-CoV and rMERS-CoV-GFP. VeroB4 cells were infected at an MOI of 0.01. Cell culture supernatants were harvested at indicated time points post infection and titrated by TCID50 assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. TCID50/ml, 50% tissue culture infectious dose per millilitre.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F2.medium.gif" width="440" height="233"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F2.medium.gif" width="440" height="233"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <span class="caption-title">Generation of viral cDNA clones and recovery of recombinant MERS-CoV and MERS-CoV-GFP.</span>
      <p id="p-11" class="first-child">
       <strong>a</strong>, Schematic representation of MERS-CoV (upper panel) and MERS-CoV-GFP (lower panel) genome organization with 8 and 10 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. 
       <strong>b</strong>, Rescue of recombinant MERS-CoV and MERS-CoV-GFP. Following delivery of viral RNAs into BHK-21 cells via electroporation the cells were co-cultivated with VeroB4 cells and supernatants containing recombinant viruses produced were used to infect new VeroB4 cells. Infected cells were visualised by bright field microscopy for recombinant MERS-CoV (upper panel; 5 days post infection), and for GFP expression for recombinant MERS-CoV-GFP (lower panel, 3 days post infection). Mock represents VeroB4 cells inoculated with supernatant from BHK-21 cells electroporated without viral RNAs. 
       <strong>c</strong>, Replication kinetics of the MERS-CoV-EMC (laboratory adapted isolate), rMERS-CoV and rMERS-CoV-GFP. VeroB4 cells were infected at an MOI of 0.01. Cell culture supernatants were harvested at indicated time points post infection and titrated by TCID50 assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. TCID50/ml, 50% tissue culture infectious dose per millilitre.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-12">Having shown that the TAR cloning approach is robust and reproducible, we aimed to thoroughly evaluate the system concerning stability of cloned sequences, the range of applicability to other virus genomes, and if molecular clones can even be generated from clinical samples. The yeast clones containing YACs encoding MHV-GFP and MERS-CoV were passaged 15-17 times followed by re-sequencing. Results of this experiment were very supportive of the yeast-based platform (Extended Data Table 3), since the genomes could be stably maintained and no mutations arose. We then embarked on cloning a number of other coronaviruses (HCoV-229E
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>, HCoV-HKU1; GenBank: 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NC_006577&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom">NC_006577</a>, MERS-CoV-Riyadh-1734-2015; GenBank: 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN481979&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom">MN481979</a>) and viruses of other families, such as ZIKV (family 
     <em>Flaviviridae</em>, GenBank: 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=KX377337&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom">KX377337</a>) and human respiratory-syncytial-virus (hRSV; family 
     <em>Paramyxoviridae</em>) (
     <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) that are known to be difficult to clone and stably maintain in 
     <em>E. coli</em>. As shown in Extended data Figure 1d-h, we followed the same strategy to prepare the genome termini in cloned plasmids to include genetic elements that are later needed for virus rescue (e.g. T7-RNA polymerase promoter, cleavages sites or ribozyme elements) and produced a set of overlapping DNA fragments covering the rest of the genome. This strategy proved to be successful irrespectively of the source of virus or nucleic acid template or the number of DNA fragments that were used for the one-step assembly in yeast. Of note, we were also able to clone hRSV-B, without any prior information on the virus genotype, directly from a clinical sample (nasopharyngeal aspirate) by designing RSV consensus primers to amplify 4 overlapping DNA fragments (sequence submitted to GenBank). Collectively, these results demonstrate the synthetic genomics platform provides the technical advance to rapidly generate molecular clones of diverse RNA viruses by using viral isolates, cloned DNA, synthetic DNA, and even clinical samples as starting material.
    </p>
    <p id="p-13">The appearance of a novel CoV in China at the end of 2019 prompted us to go a step further and to test the applicability of our synthetic genomics platform to reconstruct the virus based on its released genome sequence and chemically synthesized DNA fragments. Notably, at the time when the first genome sequences of SARS-CoV-2 were released (Jan 10/11
     <sup>th</sup>, 2020) no virus isolate has been made available and it took until the end of January 2020 when successful isolation of SARS-CoV-2 was reported from patients in Australia (
     <a href="https://www.the-scientist.com/news-opinion/australian-lab-cultures-new-coronavirus-as-infections-climb-67031">https://www.the-scientist.com/news-opinion/australian-lab-cultures-new-coronavirus-as-infections-climb-67031</a>). We fragmented the genome into 12 subgenomic DNA fragments ranging in size between 0.5-3.4 kbp (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>, Extended Data Figure 1i, Extended Data Tables 1, 4). In parallel, we also aimed to generate a SARS-CoV-2 expressing GFP that could be valuable to facilitate the establishment of serological diagnostics (e.g. virus neutralization assay) and detection in cell culture. To do this we divided fragment 11 into 3 sub-fragments (
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3a</a>, Extended Data Figure 1i, Extended Data Tables 1, 4) to include the GFP sequence, which was inserted in frame within the ORF7a. We noticed at the 5’ end of the reported SARS-CoV-2 sequence that nucleotides 3-5 (5’-AU
     <span class="underline">UAA</span>AGG; Genbank 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN996528.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom">MN996528.1</a>) were different compared to the closely related SARS-CoV (5’-AU
     <span class="underline">AUU</span>AGG; GenBank 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=AY291315&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom">AY291315</a>) and to the even more closely related bat SARS-related CoVs ZXC21 and ZC45 (5’-AU
     <span class="underline">AUU</span>AGG)
     <sup>
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
     </sup>, and that for all three the 5’-end has been experimentally determined by 5’-RACE
     <sup>
      <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
     </sup>. Therefore, we designed three 5’-end versions of fragment 1 containing the reported SARS-CoV-2 sequence (fragment 1.3; 5’-AU
     <span class="underline">UAA</span>AGG), a version modified by 3 nucleotides (fragment 1.1; 5’-AU
     <span class="underline">AUU</span>AGG), and a version containing the 124 5’-terminal nucleotides spanning the first four 5’-terminal stem-loop structures of SARS-CoV (fragment 1.2; 
     <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3b</a>). Notably, nucleotide differences between SARS-CoV-2 and SARS-CoV within this region are in agreement with the predicted RNA secondary structures (
     <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3b</a>).
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Generation of viral cDNA clones and recovery of recombinant SARS-CoV-2 and SARS-CoV-2-GFP. a, Schematic representation of SARS-CoV-2 (upper panel) and SARS-CoV-2-GFP (lower panel) genome organisation with 12 and 14 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. b, Representation of predicted RNA stem-loop (SL) secondary structures formed in the 5[[#x2019]]-UTR of SARS-CoV-2. The SARS-CoV-2 RNA secondary structures were manually adjusted based on RNA structure predictions by Chan et al.22. Black letters and numbering represents the SARS-CoV-2 5[[#x2019]]-terminal sequence. Red letters depict nucleotides that are different within the SARS-CoV 5[[#x2019]]-terminal sequence ([[#x201C]]-[[#x201D]] indicates nucleotide deletion in SARS-CoV versus SARS-CoV-2). N20 indicates 20 nucleotides. SL structures 1, 2, and 4 are shown according to Chan et al.22. Note that recombinant virus construct 1 (rSARS-CoV-2 clone 1) and construct 4 (rSARS-CoV-GFP clone 4) contain the SARS-CoV-2 5[[#x2019]]-terminus with nucleotides 3-5 (UAA) exchanged by nucleotides 3-5 (AUU) of SARS-CoV; recombinant virus construct 2 (rSARS-CoV-2 clone 2) and construct 5 (rSARS-CoV-2-GFP clone 5) contain a 5[[#x2019]]-terminus with the first 124 nucleotides from SARS-CoV; and recombinant virus construct 3 (rSARS-CoV-2 clone 3) and construct 6 (rSARS-CoV-GFP clone 6) contain the authentic SARS-CoV-2 5[[#x2019]]-terminus. c, Rescue of recombinant rSARS-CoV-2 and rSARS-CoV-2-GFP. The experimental setting is illustrated in the upper panel. Two clones of each construct 1-6 were used to prepare YAC DNAs and in-vitro transcribed viral genome RNAs were electroporated into BHK-21 cells and/or BHK-SARS-N together with an mRNA encoding the SARS-CoV-2 N protein. Electroporated cells were co-cultivated with VeroE6 cells in a T25 flask and cell culture supernatants were transferred to 12-well plates to perform plaque assay for rSARS-CoV-2 constructs 1-3 and fluorescence microscopy for rSARS-CoV-2-GFP constructs 4-6. Plaque assays are shown in the lower panel for SARS-CoV-2 clones 1.1, 2.2, and 3.1. VeroE6 cells were infected with 10[[#x2212]]1, 10[[#x2212]]2, and 10[[#x2212]]3 dilutions of 1 ml supernatant from the individual rescue experiments and are compared to non-infected cells (Mock). The expression of GFP in SARS-CoV-2-GFP infected VeroE6 cells is shown for clones 4.1, 5.2, and 6.1. Mock represents non-infected VeroE6 cells. d, Summary of the timeline of events of the SARS-CoV-2 outbreak and the work leading to the reconstruction and recovery of rSARS-CoV-2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-345331397" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Generation of viral cDNA clones and recovery of recombinant SARS-CoV-2 and SARS-CoV-2-GFP.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, Schematic representation of SARS-CoV-2 (upper panel) and SARS-CoV-2-GFP (lower panel) genome organisation with 12 and 14 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. &lt;;strong&gt;;b&lt;;/strong&gt;;, Representation of predicted RNA stem-loop (SL) secondary structures formed in the 5[[#x2019]]-UTR of SARS-CoV-2. The SARS-CoV-2 RNA secondary structures were manually adjusted based on RNA structure predictions by Chan et al.&lt;;sup&gt;;22&lt;;/sup&gt;;. Black letters and numbering represents the SARS-CoV-2 5[[#x2019]]-terminal sequence. Red letters depict nucleotides that are different within the SARS-CoV 5[[#x2019]]-terminal sequence ([[#x201C]]-[[#x201D]] indicates nucleotide deletion in SARS-CoV versus SARS-CoV-2). N20 indicates 20 nucleotides. SL structures 1, 2, and 4 are shown according to Chan &lt;;em&gt;;et al.&lt;;/em&gt;;&lt;;sup&gt;;22&lt;;/sup&gt;;. Note that recombinant virus construct 1 (rSARS-CoV-2 clone 1) and construct 4 (rSARS-CoV-GFP clone 4) contain the SARS-CoV-2 5[[#x2019]]-terminus with nucleotides 3-5 (UAA) exchanged by nucleotides 3-5 (AUU) of SARS-CoV; recombinant virus construct 2 (rSARS-CoV-2 clone 2) and construct 5 (rSARS-CoV-2-GFP clone 5) contain a 5[[#x2019]]-terminus with the first 124 nucleotides from SARS-CoV; and recombinant virus construct 3 (rSARS-CoV-2 clone 3) and construct 6 (rSARS-CoV-GFP clone 6) contain the authentic SARS-CoV-2 5[[#x2019]]-terminus. &lt;;strong&gt;;c&lt;;/strong&gt;;, Rescue of recombinant rSARS-CoV-2 and rSARS-CoV-2-GFP. The experimental setting is illustrated in the upper panel. Two clones of each construct 1-6 were used to prepare YAC DNAs and &lt;;em&gt;;in-vitro&lt;;/em&gt;; transcribed viral genome RNAs were electroporated into BHK-21 cells and/or BHK-SARS-N together with an mRNA encoding the SARS-CoV-2 N protein. Electroporated cells were co-cultivated with VeroE6 cells in a T25 flask and cell culture supernatants were transferred to 12-well plates to perform plaque assay for rSARS-CoV-2 constructs 1-3 and fluorescence microscopy for rSARS-CoV-2-GFP constructs 4-6. Plaque assays are shown in the lower panel for SARS-CoV-2 clones 1.1, 2.2, and 3.1. VeroE6 cells were infected with 10&lt;;sup&gt;;[[#x2212]]1&lt;;/sup&gt;;, 10&lt;;sup&gt;;[[#x2212]]2&lt;;/sup&gt;;, and 10&lt;;sup&gt;;[[#x2212]]3&lt;;/sup&gt;; dilutions of 1 ml supernatant from the individual rescue experiments and are compared to non-infected cells (Mock). The expression of GFP in SARS-CoV-2-GFP infected VeroE6 cells is shown for clones 4.1, 5.2, and 6.1. Mock represents non-infected VeroE6 cells. &lt;;strong&gt;;d&lt;;/strong&gt;;, Summary of the timeline of events of the SARS-CoV-2 outbreak and the work leading to the reconstruction and recovery of rSARS-CoV-2.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F3.medium.gif" width="318" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F3.medium.gif" width="318" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/21/2020.02.21.959817/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <span class="caption-title">Generation of viral cDNA clones and recovery of recombinant SARS-CoV-2 and SARS-CoV-2-GFP.</span>
      <p id="p-14" class="first-child">
       <strong>a</strong>, Schematic representation of SARS-CoV-2 (upper panel) and SARS-CoV-2-GFP (lower panel) genome organisation with 12 and 14 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. 
       <strong>b</strong>, Representation of predicted RNA stem-loop (SL) secondary structures formed in the 5’-UTR of SARS-CoV-2. The SARS-CoV-2 RNA secondary structures were manually adjusted based on RNA structure predictions by Chan et al.
       <sup>
        <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
       </sup>. Black letters and numbering represents the SARS-CoV-2 5’-terminal sequence. Red letters depict nucleotides that are different within the SARS-CoV 5’-terminal sequence (“-” indicates nucleotide deletion in SARS-CoV versus SARS-CoV-2). N20 indicates 20 nucleotides. SL structures 1, 2, and 4 are shown according to Chan 
       <em>et al.</em>
       <sup>
        <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>
       </sup>. Note that recombinant virus construct 1 (rSARS-CoV-2 clone 1) and construct 4 (rSARS-CoV-GFP clone 4) contain the SARS-CoV-2 5’-terminus with nucleotides 3-5 (UAA) exchanged by nucleotides 3-5 (AUU) of SARS-CoV; recombinant virus construct 2 (rSARS-CoV-2 clone 2) and construct 5 (rSARS-CoV-2-GFP clone 5) contain a 5’-terminus with the first 124 nucleotides from SARS-CoV; and recombinant virus construct 3 (rSARS-CoV-2 clone 3) and construct 6 (rSARS-CoV-GFP clone 6) contain the authentic SARS-CoV-2 5’-terminus. 
       <strong>c</strong>, Rescue of recombinant rSARS-CoV-2 and rSARS-CoV-2-GFP. The experimental setting is illustrated in the upper panel. Two clones of each construct 1-6 were used to prepare YAC DNAs and 
       <em>in-vitro</em> transcribed viral genome RNAs were electroporated into BHK-21 cells and/or BHK-SARS-N together with an mRNA encoding the SARS-CoV-2 N protein. Electroporated cells were co-cultivated with VeroE6 cells in a T25 flask and cell culture supernatants were transferred to 12-well plates to perform plaque assay for rSARS-CoV-2 constructs 1-3 and fluorescence microscopy for rSARS-CoV-2-GFP constructs 4-6. Plaque assays are shown in the lower panel for SARS-CoV-2 clones 1.1, 2.2, and 3.1. VeroE6 cells were infected with 10
       <sup>−1</sup>, 10
       <sup>−2</sup>, and 10
       <sup>−3</sup> dilutions of 1 ml supernatant from the individual rescue experiments and are compared to non-infected cells (Mock). The expression of GFP in SARS-CoV-2-GFP infected VeroE6 cells is shown for clones 4.1, 5.2, and 6.1. Mock represents non-infected VeroE6 cells. 
       <strong>d</strong>, Summary of the timeline of events of the SARS-CoV-2 outbreak and the work leading to the reconstruction and recovery of rSARS-CoV-2.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-15">We ordered the synthetic DNA constructs on January 14
     <sup>th</sup> (
     <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3d</a>, Extended data Table 4, Extended Data Document 1), and until February 4
     <sup>th</sup> 12 of 14 constructs were delivered as sequence-confirmed plasmids. However, fragments 5 and 7 turned out to be problematic to clone in 
     <em>E. coli</em> and could not be delivered. Since we received at the same time SARS-CoV-2 viral RNA obtained from an isolate of a Munich patient (SARS-CoV-2/München-1.1/2020/929), we decided to amplify the regions of fragments 5 and 7 by RT-PCR. TAR cloning was immediately initiated, and for all 6 SARS-CoV-2 constructs we obtained correctly assembled molecular clones (3 versions of the 5’-end and each version with and without GFP; Extended Data Fig. 2). Since sequence verification was not possible within this short time frame, we randomly selected two clones for each construct for rescuing recombinant rSARS-CoV-2 and rSARS-CoV-2-GFP. YAC DNA were isolated, 
     <em>in vitro</em> transcribed and the resulting RNAs electroporated together with an mRNA encoding the SARS-CoV-2 N protein into BHK-21 and, in parallel, into BHK-SARS-N cells expressing the SARS-CoV N protein
     <sup>
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
     </sup>. The BHK-21 and BHK-SARS-N cells were seeded over VeroE6 cells and at two days post electroporation we observed green fluorescent signals in cells that received the GFP-encoding SARS-CoV-2 RNAs. Indeed supernatants transferred to fresh VeroE6 cells contained infectious recombinant viruses for almost all recombinant rSARS-CoV-2 and rSARS-CoV-GFP constructs (Extended Data Fig. 2). As shown in 
     <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3c</a> for SARS-CoV-2 clones 1.1, 2.2, and 3.1, plaques were detectable on VeroE6 cells that were inoculated with supernatants containing the RNA-electroporated cells, demonstrating the infectious virus has been recovered for recombinant rSARS-CoV-2 and for the variants containing modified 5’-termini. Similarly, we could detect green fluorescent cells on VeroE6 monolayers that were inoculated with supernatants from RNA-electroporated cells of rSARS-CoV-2-GFP clones 4.1, 5.2 and 6.1, demonstrating that recombinant GFP-expressing viruses have been recoverd for all 5’-end variants. This result demonstrates the full functionality of the SARS-CoV-2 reverse genetics system and also shows that the 5’-ends of SARS-CoV and bat SARS-related CoVs ZXC21 and ZC45 are compatible with the replication machinery of SARS-CoV-2.
    </p>
    <p id="p-16">We expect that the fast, robust and versatile synthetic genomics platform we describe here will provide new insights into the molecular biology and pathogenesis of a number of emerging RNA viruses and will specifically facilitate the molecular characterization of the novel coronavirus SARS-CoV-2. Although homologous recombination in yeast has already been used for the generation of a number of molecular virus clones in the past
     <sup>
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
     </sup>, we present here a thorough evaluation of the feasibility of this approach to rapidly generate full-length cDNAs for large RNA viruses that have a known history of instability in 
     <em>E. coli</em>. We show that one main advantage of the TAR cloning system is that the viral genomes can be fragmented to at least 14 overlapping fragments and re-assembled with remarkable efficacy (usually &gt;;90% of the clones are correct). This allowed us to complete the cloning and rescue of rSARS-CoV-2 and rSARS-CoV-2-GFP within one week. It should also be noted that we see considerable potential to reduce the time of DNA synthesis. Currently, synthetic DNA fragments are still routinely cloned in 
     <em>E. coli</em>, which turned out to be problematic for SARS-CoV-2 fragments 5 and 7. We could however use 4 and 3 shorter dsDNA parts of fragments 5 and 7, respectively, that we received, to assemble the full-length fragments 5 and 7 by TAR cloning which is an additional proof of the superior cloning efficiency of our yeast system versus 
     <em>E. coli</em>-based systems. This allowed us to generate a molecular clone of SARS-CoV-2 by using exclusively chemically synthesized DNA, and the rescue of this version is currently in progress.
    </p>
    <p id="p-17">The appearance of SARS-CoV-2 emphasizes the need for preparedness to rapidly respond to emerging virus threats. While the availability of SARS-CoV-2 isolates is still limited, full virus genome sequences were disclosed very early during the outbreak. The rapidity of our synthetic genomics approach to generate SARS-CoV-2 and the applicability to other emerging RNA viruses make this system an attractive alternative to provide infectious virus to health authorities and diagnostic laboratories without the need to have access to clinical samples. Finally, as the SARS-CoV-2 outbreak is ongoing, we expect to see sequence variations and possibly phenotypic changes of evolving SARS-CoV-2 in the human host. With the synthetic genomics platform it is now possible to introduce such sequence variations into the infectious clone and to functionally characterize SARS-CoV-2 evolution in real-time.</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Authors contributions</h2>
    <p id="p-19">VT and JJ conceived the study. TT, NE and FL performed most of the experiments. PV, HS, JP, SSt, MH, AK, MG, LL, LH, MW, SP, DH, VC, SC, SSch, DM, DN, MM, CD, RD, did experimental work and/or provided essential experimental systems and reagents. TT, NE, FL, HS, JK, and RD performed sequencing including computational analyses. VT, JJ, TT, FL, NE wrote the manuscript and made the figures. All authors read and approved the final manuscript.</p>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Ethical statement</h2>
    <p id="p-20">The authors are aware that this work contains aspects of Dual Use Research of Concern (DURC). The benefits were carefully balanced against the risks and the benefits outweigh the risks. Permission to generate and work with recombinant SARS-CoV-2 and SARS-CoV-2-GFP was granted by the Swiss Federal Office of Public Health (#A131191/3) with consultation of the Federal Office for Environment, Federal Food Safety and Veterinary Office, and the Swiss Expert Committee for Biosafety.</p>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Competing interests</h2>
    <p id="p-21">The authors declare no competing interests</p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Additional information</h2>
    <p id="p-22">Correspondence and requests for material should be addressed to V.T. or J.J.</p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Figure Legends</h2>
    <p id="p-23">
     <strong>Extended Data Figure 1</strong>
    </p>
    <p id="p-24">
     <strong>a</strong>, Schematic representation of the MHV-GFP genome organisation (upper panel) and nine viral subgenomic cDNA fragments (F1-9) used for TAR cloning. Viral open reading frames (ORFs), the ORF for GFP and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-9 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J10 represent junctions with the TAR vector. Gel images show results from two multiplex PCRs that were designed to detect the presence of correctly recombined junctions J1-10, confirming the proper assembly of the YAC containing the viral genome in 12 out of 12 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-25">
     <strong>b</strong>, Schematic representation of the MERS-CoV genome organisation (upper panel) and eight viral subgenomic cDNA fragments (F1-8) used for TAR cloning. Viral open reading frames (ORFs) and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-8 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J9 represent junctions with the TAR vector. Gel image shows results from a multiplex PCR that was designed to detect the presence of correctly recombined junctions J1-9, confirming the proper assembly of the YAC containing the viral genome in 6 out of 8 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-26">
     <strong>c</strong>, Schematic representation of the MERS-CoV-GFP genome organisation (upper panel) and ten viral subgenomic cDNA fragments (F1-10 and F13) used for TAR cloning. Viral open reading frames (ORFs), ORF for GFP and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-8 and J10-11 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J9 represent junctions with the TAR vector. Gel images show results from two multiplex PCRs that were designed to detect the presence of correctly recombined junctions J1-11, confirming the proper assembly of the YAC containing the viral genome in 5 out of 8 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-27">
     <strong>d</strong>, Schematic representation of the HCoV-229E genome organisation (upper panel) and thirteen viral subgenomic cDNA fragments (F1-13) used for TAR cloning. Viral open reading frames (ORFs) and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-13 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J14 represent junctions with the TAR vector. Gel images show results from two multiplex PCRs that were designed to detect the presence of correctly recombined junctions J1-14, confirming the proper assembly of the YAC containing the viral genome in 7 out of 10 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-28">
     <strong>d</strong>, Schematic representation of the HCoV-HKU1 genome organisation (upper panel) and eleven viral subgenomic cDNA fragments (F1-11) used for TAR cloning. Viral open reading frames (ORFs) and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-11 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J12 represent junctions with the TAR vector. Gel images show results from two multiplex PCRs that were designed to detect the presence of correctly recombined junctions J1-12, confirming the proper assembly of the YAC containing the viral genome in 14 out of 14 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-29">
     <strong>e</strong>, Schematic representation of the MERS-CoV-Riadh-1734-2015 genome organisation (upper panel) and eight viral subgenomic cDNA fragments (F1-8) used for TAR cloning. Viral open reading frames (ORFs) and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-8 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J9 represent junctions with the TAR vector. Gel images show results from a multiplex PCR and a simplex PCR, both of which were designed to detect the presence of correctly recombined junctions J1-9, confirming the proper assembly of the YAC containing the viral genome in 5 out of 8 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-30">
     <strong>f</strong>, Schematic representation of the ZIKA virus genome organisation (upper panel) and six viral subgenomic cDNA fragments (F1-6) used for TAR cloning. Viral open reading frames (ORFs) and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-5 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J6 represent junctions with the TAR vector. Gel image shows results from a multiplex PCR that was designed to detect the presence of correctly recombined junctions J1-6, confirming the proper assembly of the YAC containing the viral genome in 3 out of 15 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; HDR, hepatitis delta virus ribozyme.
    </p>
    <p id="p-31">
     <strong>g</strong>, Schematic representation of the human RSV-B virus genome organisation (upper panel) and six viral subgenomic cDNA fragments (F1-6) used for TAR cloning. Viral open reading frames (ORFs) and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J1-5 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J5 contain junctions with the TAR vector. Gel image shows results from a multiplex PCR that was designed to detect the presence of correctly recombined junctions J1-5, confirming the proper assembly of the YAC containing the viral genome in 7 out of 8 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; HHrb, hammerhead ribozyme, Rb-T7ter, Ribozyme and T7 RNA polymerase terminator seqeunce.
    </p>
    <p id="p-32">
     <strong>h</strong>, Schematic representation of the SARS-CoV-2 genome organisation (upper panel) and twelve viral subgenomic cDNA fragments (F1-12) used for TAR cloning. Viral open reading frames (ORFs) and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-12 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J13 represent junctions with the TAR vector. Gel images show results from two multiplex PCRs that were designed to detect the presence of correctly recombined junctions J1-13, confirming the proper assembly of the YAC containing the viral genome in 6 out of 6 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-33">
     <strong>i</strong>, Schematic representation of the SARS-CoV-2 genome organisation (upper panel) and fourteen viral subgenomic cDNA fragments (F1-10 and F12-15) used for TAR cloning. Viral open reading frames (ORFs), ORF for GFP and sequence elements at the 5’- and 3’-untranslated regions (UTRs) are indicated. Primers used to generate the fragments are listed in Extended Data Table 1. J2-12 and J14 represent the junctions, i.e. overlapping regions, between the subgenomic cDNA fragments. J1 and J13 represent junctions with the TAR vector. Gel images show results from two multiplex PCRs and a simplex PCR, all of which were designed to detect the presence of correctly recombined junctions J1-13 and J14 respectively, confirming the proper assembly of the YAC containing the viral genome in 6 out of 6 clones. kb, kilobases; pT7, T7 RNA polymerase promoter; An, poly (A) tail.
    </p>
    <p id="p-34">
     <strong>Extended Data Figure 2</strong>
    </p>
    <p id="p-35">
     <strong>a</strong>, Six constructs were initially designed based on 3 different 5’-UTR regions. These regions contain i) a modified sequence of the 5’-UTR region of the SARS-CoV-2 (5’-AT
     <span class="underline">AUU</span>AGG; 5’-terminal 8 nucleotides are shown, modified nucleotides underlined) were nucleotides 3-5 (UAA) of SARS-CoV-2 were changed to AUU to match the SARS-CoV nucleotides 3-5 (synthetic fragment 1.1 for constructs 1 and 4), ii) a SARS-CoV-2 5’-terminus with the first 124 nucleotides exchanged by the corresponding 5’-terminal sequence of SARS-CoV (synthetic fragment 1.2; construct 2 and 5) and iii) the reported sequence of the SARS-CoV-2 (GISAID: EPI_ISL_402119) (synthetic fragment 1.3; constructs 3 and 6). After yeast transformation, 10 colonies were randomly picked for each of the six constructs and all the junctions bridging the overlapping fragments were verified by multiplex PCR. For each construct, two clones (x.1 and x.2) were randomly selected and YAC DNAs were isolated (12 clones in total). RNAs were generated by 
     <em>in vitro</em> transcription and electroporated together with an mRNA encoding the SARS-CoV-2 N protein either into BHK-21 cells (12 clones) or BHK-SARS-N cells (cells expressing the SARS-CoV N protein) (6 clones). Electroporated cells were then co-cultivated with susceptible VeroE6 cells to rescue the recombinant viruses. P.0 supernatants were collected at different time points after electroporation (from 2 dpe to 5 dpe) and transferred to VeroE6 cells to generate P.1 virus stocks, and in parallel to demonstrate presence of infectious virus by plaque assay (virus clones that do not encode GFP) or fluorescence microscopy (GFP-encoding virus clones). YAC, yeast artificial chromosome; h.p.e, hours post-electroporation; d.p.e, days post-electroporation; CPE, cytopathic effects; GFP, green fluorescent protein; pfu/ml, plaque forming units per millilitre.
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-18">This work was supported by the European Commission (Marie Sklodowska-Curie Innovative Training Network “HONOURS”; grant agreement No 721367), the Swiss National Science Foundation (SNF; grants CRSII3_160780, and 310030_173085), the German Research Council (DFG; grants SFB-TR84 (TRR 84/3, A07), and DR 772/7-2), the Federal Ministry of Education and Research (BMBF; grant RAPID, #01KI1723A) and by core funds of the University of Bern. We thank the J. Craig Venter Institute (JCVI) for provision of the TAR vectors. We thank specifically Eric Sun and Tibbers Zhang and are grateful for the timely provision of the synthetic DNA fragments from GenScript. We thank Jeanne Peters Zocher from VETCOM Zürich for her advice in generating the figures and we thank Franziska Suter-Riniker and Pascal Bittel, Institute for Infectious Diseases, University of Bern, for providing clinical samples. We thank Philippe Plattet, Markus Gerber, Martina Friesland, Marco Alves, Nathalie Vielle, Beatrice Zumkehr, Melanie Brügger, Daniel Brechbühl for reagents, technical advise and helpful discussions.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list">
     <li>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.1" data-doi="10.1073/pnas.97.10.5516">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Almazan</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Engineering the largest RNA virus genome as an infectious bacterial artificial chromosome</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">97</span>, 
         <span class="cit-fpage">5516</span>–
         <span class="cit-lpage">5521</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.97.10.5516</span> (
         <span class="cit-pub-date">2000</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAlmazan%26rft.auinit1%253DF.%26rft.volume%253D97%26rft.issue%253D10%26rft.spage%253D5516%26rft.epage%253D5521%26rft.atitle%253DEngineering%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenome%2Bas%2Ban%2Binfectious%2Bbacterial%2Bartificial%2Bchromosome%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.10.5516%26rft_id%253Dinfo%253Apmid%252F10805807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTAvNTUxNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIxLzIwMjAuMDIuMjEuOTU5ODE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.2" data-doi="10.1099/0022-1317-82-6-1273">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thiel</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Herold</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schelle</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Siddell</span>, 
          <span class="cit-name-given-names">S. G.</span>
         </span>
         <span class="cit-article-title">Infectious RNA transcribed in vitro from a cDNA copy of the human coronavirus genome cloned in vaccinia virus</span>. 
         <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">1273</span>–
         <span class="cit-lpage">1281</span> (
         <span class="cit-pub-date">2001</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DThiel%26rft.auinit1%253DV.%26rft.volume%253D82%26rft.issue%253D6%26rft.spage%253D1273%26rft.epage%253D1281%26rft.atitle%253DInfectious%2BRNA%2Btranscribed%2Bin%2Bvitro%2Bfrom%2Ba%2BcDNA%2Bcopy%2Bof%2Bthe%2Bhuman%2Bcoronavirus%2Bgenome%2Bcloned%2Bin%2Bvaccinia%2Bvirus%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-82-6-1273%26rft_id%253Dinfo%253Apmid%252F11369870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/0022-1317-82-6-1273&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11369870&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000168896600003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.3" data-doi="10.1128/JVI.74.22.10600-10611.2000">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yount</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Curtis</span>, 
          <span class="cit-name-given-names">K. M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Baric</span>, 
          <span class="cit-name-given-names">R. S.</span>
         </span>
         <span class="cit-article-title">Strategy for systematic assembly of large RNA and DNA genomes: transmissible gastroenteritis virus model</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">74</span>, 
         <span class="cit-fpage">10600</span>–
         <span class="cit-lpage">10611</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.74.22.10600-10611.2000</span> (
         <span class="cit-pub-date">2000</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DYount%26rft.auinit1%253DB.%26rft.volume%253D74%26rft.issue%253D22%26rft.spage%253D10600%26rft.epage%253D10611%26rft.atitle%253DStrategy%2Bfor%2BSystematic%2BAssembly%2Bof%2BLarge%2BRNA%2Band%2BDNA%2BGenomes%253A%2BTransmissible%2BGastroenteritis%2BVirus%2BModel%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.74.22.10600-10611.2000%26rft_id%253Dinfo%253Apmid%252F11044104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3NC8yMi8xMDYwMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIxLzIwMjAuMDIuMjEuOTU5ODE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.4" data-doi="10.1056/NEJMoa2001017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China</span>, 2019. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001017</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252F31978945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31978945&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.5" data-doi="10.1056/NEJMoa1211721">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zaki</span>, 
          <span class="cit-name-given-names">A. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Boheemen</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bestebroer</span>, 
          <span class="cit-name-given-names">T. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Osterhaus</span>, 
          <span class="cit-name-given-names">A. D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fouchier</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>
         <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">367</span>, 
         <span class="cit-fpage">1814</span>–
         <span class="cit-lpage">1820</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa1211721</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23075143&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000310773200008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.6" data-doi="10.3201/eid2011.141380">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cao-Lormeau</span>, 
          <span class="cit-name-given-names">V. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Zika virus, French polynesia, South pacific, 2013</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">1085</span>–
         <span class="cit-lpage">1086</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.3201/eid2006.140138</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D1085%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2011.141380%26rft_id%253Dinfo%253Apmid%252F24856001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid2011.141380&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24856001&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.7" data-doi="10.1056/NEJMoa1404505">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baize</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Emergence of Zaire Ebola virus disease in Guinea</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">371</span>, 
         <span class="cit-fpage">1418</span>–
         <span class="cit-lpage">1425</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa1404505</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1404505%26rft_id%253Dinfo%253Apmid%252F24738640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1404505&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24738640&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.8" data-doi="10.1126/science.1190719">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gibson</span>, 
          <span class="cit-name-given-names">D. G.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Creation of a Bacterial Cell Controlled by a Chemically Synthesized Genome</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">329</span>, 
         <span class="cit-fpage">52</span>–
         <span class="cit-lpage">56</span>, doi:science.1190719 [pii] 
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.1190719</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190719%26rft_id%253Dinfo%253Apmid%252F20488990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjkvNTk4Ny81MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIxLzIwMjAuMDIuMjEuOTU5ODE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.9" data-doi="10.1126/science.1173759">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lartigue</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Creating bacterial strains from genomes that have been cloned and engineered in yeast</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">325</span>, 
         <span class="cit-fpage">1693</span>–
         <span class="cit-lpage">1696</span>, doi:1173759 [pii] 
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.1173759</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1173759%26rft_id%253Dinfo%253Apmid%252F19696314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjUvNTk0OC8xNjkzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjEvMjAyMC4wMi4yMS45NTk4MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.10" data-doi="10.1093/nar/gkq119">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Benders</span>, 
          <span class="cit-name-given-names">G. A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cloning whole bacterial genomes in yeast</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">38</span>, 
         <span class="cit-fpage">2558</span>–
         <span class="cit-lpage">2569</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkq119</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq119%26rft_id%253Dinfo%253Apmid%252F20211840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkq119&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20211840&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000277238900006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.11" data-doi="10.1126/science.aad6253">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hutchison</span>, 
          <span class="cit-name-given-names">C. A.</span>
         </span>, 
         <span class="cit-edition">3rd</span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Design and synthesis of a minimal bacterial genome</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">351</span>, 
         <span class="cit-fpage">aad6253</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.aad6253</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHutchison%26rft.auinit1%253DC.%2BA.%26rft.volume%253D351%26rft.issue%253D6280%26rft.spage%253Daad6253%26rft.epage%253Daad6253%26rft.atitle%253DDesign%2Band%2Bsynthesis%2Bof%2Ba%2Bminimal%2Bbacterial%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad6253%26rft_id%253Dinfo%253Apmid%252F27013737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTEvNjI4MC9hYWQ2MjUzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjEvMjAyMC4wMi4yMS45NTk4MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.12" data-doi="10.1038/nprot.2008.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kouprina</span>, 
          <span class="cit-name-given-names">N.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Larionov</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-article-title">Selective isolation of genomic loci from complex genomes by transformation-associated recombination cloning in the yeast Saccharomyces cerevisiae</span>. 
         <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">371</span>–
         <span class="cit-lpage">377</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nprot.2008.5</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKouprina%26rft.auinit1%253DN.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D371%26rft.epage%253D377%26rft.atitle%253DSelective%2Bisolation%2Bof%2Bgenomic%2Bloci%2Bfrom%2Bcomplex%2Bgenomes%2Bby%2Btransformation-associated%2Brecombination%2Bcloning%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.5%26rft_id%253Dinfo%253Apmid%252F18323808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nprot.2008.5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18323808&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000254137100005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.13" data-doi="10.1128/mSphereDirect.00331-17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vashee</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cloning, Assembly, and Modification of the Primary Human Cytomegalovirus Isolate Toledo by Yeast-Based Transformation-Associated Recombination</span>. 
         <abbr class="cit-jnl-abbrev">mSphere</abbr>
         <span class="cit-vol">2</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/mSphereDirect.00331-17</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DmSphere%26rft.stitle%253DmSphere%26rft.aulast%253DVashee%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253De00331-17%26rft.epage%253De00331-17%26rft.atitle%253DCloning%252C%2BAssembly%252C%2Band%2BModification%2Bof%2Bthe%2BPrimary%2BHuman%2BCytomegalovirus%2BIsolate%2BToledo%2Bby%2BYeast-Based%2BTransformation-Associated%2BRecombination%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmSphereDirect.00331-17%26rft_id%253Dinfo%253Apmid%252F28989973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibXNwaCI7czo1OiJyZXNpZCI7czoxMzoiMi81L2UwMDMzMS0xNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIxLzIwMjAuMDIuMjEuOTU5ODE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.14" data-doi="10.1073/pnas.1700534114">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Oldfield</span>, 
          <span class="cit-name-given-names">L. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genome-wide engineering of an infectious clone of herpes simplex virus type 1 using synthetic genomics assembly methods</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">114</span>, 
         <span class="cit-fpage">E8885</span>–
         <span class="cit-lpage">E8894</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1700534114</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1700534114%26rft_id%253Dinfo%253Apmid%252F28928148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQyL0U4ODg1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjEvMjAyMC4wMi4yMS45NTk4MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.15" data-doi="10.1128/JVI.79.5.3097-3106.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Coley</span>, 
          <span class="cit-name-given-names">S. E.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Recombinant mouse hepatitis virus strain A59 from cloned, full-length cDNA replicates to high titers in vitro and is fully pathogenic in vivo</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">79</span>, 
         <span class="cit-fpage">3097</span>–
         <span class="cit-lpage">3106</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.79.5.3097-3106.2005</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DColey%26rft.auinit1%253DS.%2BE.%26rft.volume%253D79%26rft.issue%253D5%26rft.spage%253D3097%26rft.epage%253D3106%26rft.atitle%253DRecombinant%2BMouse%2BHepatitis%2BVirus%2BStrain%2BA59%2Bfrom%2BCloned%252C%2BFull-Length%2BcDNA%2BReplicates%2Bto%2BHigh%2BTiters%2BIn%2BVitro%2Band%2BIs%2BFully%2BPathogenic%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.5.3097-3106.2005%26rft_id%253Dinfo%253Apmid%252F15709029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzUvMzA5NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIxLzIwMjAuMDIuMjEuOTU5ODE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.16" data-doi="10.1371/journal.ppat.0030109">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zust</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Coronavirus non-structural protein 1 is a major pathogenicity factor: implications for the rational design of coronavirus vaccines</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">e109</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.0030109</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DZust%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D8%26rft.spage%253De109%26rft.epage%253De109%26rft.atitle%253DCoronavirus%2Bnon-structural%2Bprotein%2B1%2Bis%2Ba%2Bmajor%2Bpathogenicity%2Bfactor%253A%2Bimplications%2Bfor%2Bthe%2Brational%2Bdesign%2Bof%2Bcoronavirus%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0030109%26rft_id%253Dinfo%253Apmid%252F17696607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0030109&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17696607&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.17" data-doi="10.1099/jgv.0.000919">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Muth</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Transgene expression in the genome of Middle East respiratory syndrome coronavirus based on a novel reverse genetics system utilizing Red-mediated recombination cloning</span>. 
         <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>
         <span class="cit-vol">98</span>, 
         <span class="cit-fpage">2461</span>–
         <span class="cit-lpage">2469</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1099/jgv.0.000919</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D98%26rft.spage%253D2461%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/jgv.0.000919&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.18" data-doi="10.1038/s41426-018-0155-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic characterization and infectivity of a novel SARS-like coronavirus in Chinese bats</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">154</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41426-018-0155-5</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D7%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41426-018-0155-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41426-018-0155-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.19" data-doi="10.1099/vir.0.19424-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thiel</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Mechanisms and enzymes involved in SARS coronavirus genome expression</span>. 
         <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>
         <span class="cit-vol">84</span>, 
         <span class="cit-fpage">2305</span>–
         <span class="cit-lpage">2315</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.19424-0</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DThiel%26rft.auinit1%253DV.%26rft.volume%253D84%26rft.issue%253D9%26rft.spage%253D2305%26rft.epage%253D2315%26rft.atitle%253DMechanisms%2Band%2Benzymes%2Binvolved%2Bin%2BSARS%2Bcoronavirus%2Bgenome%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19424-0%26rft_id%253Dinfo%253Apmid%252F12917450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/vir.0.19424-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12917450&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000184993100003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.20" data-doi="10.1371/journal.pone.0032857">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">van den Worm</span>, 
          <span class="cit-name-given-names">S. H.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Reverse genetics of SARS-related coronavirus using vaccinia virus-based recombination</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">e32857</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0032857</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253Dvan%2Bden%2BWorm%26rft.auinit1%253DS.%2BH.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De32857%26rft.epage%253De32857%26rft.atitle%253DReverse%2Bgenetics%2Bof%2BSARS-related%2Bcoronavirus%2Busing%2Bvaccinia%2Bvirus-based%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032857%26rft_id%253Dinfo%253Apmid%252F22412934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032857&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22412934&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F21%2F2020.02.21.959817.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Polo</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ketner</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Levis</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Falgout</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-article-title">Infectious RNA transcripts from full-length dengue virus type 2 cDNA clones made in yeast</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">71</span>, 
         <span class="cit-fpage">5366</span>–
         <span class="cit-lpage">5374</span> (
         <span class="cit-pub-date">1997</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DPolo%26rft.auinit1%253DS.%26rft.volume%253D71%26rft.issue%253D7%26rft.spage%253D5366%26rft.epage%253D5374%26rft.atitle%253DInfectious%2BRNA%2Btranscripts%2Bfrom%2Bfull-length%2Bdengue%2Bvirus%2Btype%2B2%2BcDNA%2Bclones%2Bmade%2Bin%2Byeast%26rft_id%253Dinfo%253Apmid%252F9188607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcxLzcvNTM2NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIxLzIwMjAuMDIuMjEuOTU5ODE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.21.959817v1.22" data-doi="10.1080/22221751.2020.1719902">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">J. F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">221</span>–
         <span class="cit-lpage">236</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1080/22221751.2020.1719902</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1719902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1719902&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
