<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/06/14/670398.atom"/>
  <meta name="HW.pisa" content="biorxiv;670398v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="In-vivo targeted tagging of RNA isolates cell specific transcriptional responses to environmental stimuli and identifies liver-to-adipose RNA transfer"/>
  <meta name="DC.Identifier" content="10.1101/670398"/>
  <meta name="DC.Date" content="2019-06-14"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Bio-fluids contain various circulating cell-free RNA transcripts (ccfRNAs). The composition of these ccfRNAs varies between bio-fluids and constitute tantalizing biomarker candidates for several pathologies. ccfRNAs have also been demonstrated as mediators of cellular communication, yet little is known about their function in physiological and developmental settings and most works are limited to in-vitro studies. Here, we have developed iTAG-RNA, a novel method for the unbiased tagging of RNA transcripts in mice in-vivo. We used this method to isolate hepatocytes and kidney proximal epithelial cells-specific transcriptional response to a dietary challenge without interfering with the tissue architecture, and to identify multiple hepatocyte-secreted ccfRNAs in plasma. We also identified transfer of these hepatic derived ccfRNAs to adipose tissue, where they likely serve as a buffering mechanism to maintain cholesterol and lipid homeostasis. Our findings directly demonstrate in-vivo transfer of RNAs between tissues and highlight its implications for endocrine signaling and homeostasis."/>
  <meta name="DC.Contributor" content="J. Darr"/>
  <meta name="DC.Contributor" content="M. Lassi"/>
  <meta name="DC.Contributor" content="Archana Tomar"/>
  <meta name="DC.Contributor" content="R. Gerlini"/>
  <meta name="DC.Contributor" content="F. Scheid"/>
  <meta name="DC.Contributor" content="M. Hrabě de Angelis"/>
  <meta name="DC.Contributor" content="M. Witting"/>
  <meta name="DC.Contributor" content="R. Teperino"/>
  <meta name="article:published_time" content="2019-06-14"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="In-vivo targeted tagging of RNA isolates cell specific transcriptional responses to environmental stimuli and identifies liver-to-adipose RNA transfer"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Bio-fluids contain various circulating cell-free RNA transcripts (ccfRNAs). The composition of these ccfRNAs varies between bio-fluids and constitute tantalizing biomarker candidates for several pathologies. ccfRNAs have also been demonstrated as mediators of cellular communication, yet little is known about their function in physiological and developmental settings and most works are limited to in-vitro studies. Here, we have developed iTAG-RNA, a novel method for the unbiased tagging of RNA transcripts in mice in-vivo. We used this method to isolate hepatocytes and kidney proximal epithelial cells-specific transcriptional response to a dietary challenge without interfering with the tissue architecture, and to identify multiple hepatocyte-secreted ccfRNAs in plasma. We also identified transfer of these hepatic derived ccfRNAs to adipose tissue, where they likely serve as a buffering mechanism to maintain cholesterol and lipid homeostasis. Our findings directly demonstrate in-vivo transfer of RNAs between tissues and highlight its implications for endocrine signaling and homeostasis.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;670398v2"/>
  <meta name="citation_id" content="670398v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/670398v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/670398v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/670398v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398.full.pdf"/>
  <meta name="citation_doi" content="10.1101/670398"/>
  <meta name="citation_num_pages" content="30"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="670398"/>
  <meta name="citation_author" content="J. Darr"/>
  <meta name="citation_author_institution" content="Institute of Experimental Genetics, Helmholtz Zentrum München, German Research center for Environmental Health – Neuherberg"/>
  <meta name="citation_author_institution" content="German Center for Diabetes Research (DZD) – Neuherberg"/>
  <meta name="citation_author" content="M. Lassi"/>
  <meta name="citation_author_institution" content="Institute of Experimental Genetics, Helmholtz Zentrum München, German Research center for Environmental Health – Neuherberg"/>
  <meta name="citation_author_institution" content="German Center for Diabetes Research (DZD) – Neuherberg"/>
  <meta name="citation_author" content="Archana Tomar"/>
  <meta name="citation_author_institution" content="Institute of Experimental Genetics, Helmholtz Zentrum München, German Research center for Environmental Health – Neuherberg"/>
  <meta name="citation_author_institution" content="German Center for Diabetes Research (DZD) – Neuherberg"/>
  <meta name="citation_author" content="R. Gerlini"/>
  <meta name="citation_author_institution" content="Institute of Experimental Genetics, Helmholtz Zentrum München, German Research center for Environmental Health – Neuherberg"/>
  <meta name="citation_author_institution" content="German Center for Diabetes Research (DZD) – Neuherberg"/>
  <meta name="citation_author" content="F. Scheid"/>
  <meta name="citation_author_institution" content="Institute of Experimental Genetics, Helmholtz Zentrum München, German Research center for Environmental Health – Neuherberg"/>
  <meta name="citation_author_institution" content="German Center for Diabetes Research (DZD) – Neuherberg"/>
  <meta name="citation_author" content="M. Hrabě de Angelis"/>
  <meta name="citation_author_institution" content="Institute of Experimental Genetics, Helmholtz Zentrum München, German Research center for Environmental Health – Neuherberg"/>
  <meta name="citation_author_institution" content="German Center for Diabetes Research (DZD) – Neuherberg"/>
  <meta name="citation_author_institution" content="Experimental Genetics, Faculty of Life and Food Sciences Weihenstephan, Technische Universität München"/>
  <meta name="citation_author" content="M. Witting"/>
  <meta name="citation_author_institution" content="Research Unit Analytical BioGeoChemistry"/>
  <meta name="citation_author_institution" content="Chair of Analytical Food Chemistry, Technische Universität München"/>
  <meta name="citation_author_email" content="raffaele.teperino@helmholtz-muenchen.de"/>
  <meta name="citation_author_email" content="Michael.witting@helmholtz-muenchen.de"/>
  <meta name="citation_author" content="R. Teperino"/>
  <meta name="citation_author_institution" content="Institute of Experimental Genetics, Helmholtz Zentrum München, German Research center for Environmental Health – Neuherberg"/>
  <meta name="citation_author_institution" content="German Center for Diabetes Research (DZD) – Neuherberg"/>
  <meta name="citation_author_email" content="raffaele.teperino@helmholtz-muenchen.de"/>
  <meta name="citation_author_email" content="Michael.witting@helmholtz-muenchen.de"/>
  <meta name="citation_reference" content="Murillo, O.D., et al., exRNA Atlas Analysis Reveals Distinct Extracellular RNA Cargo Types and Their Carriers Present across Human Biofluids. Cell, 2019. 177(2): p. 463–477 e15."/>
  <meta name="citation_reference" content="Yeri, A., et al., Total Extracellular Small RNA Profiles from Plasma, Saliva, and Urine of Healthy Subjects. Sci Rep, 2017. 7: p. 44061."/>
  <meta name="citation_reference" content="Dror, S., et al., Melanoma miRNA trafficking controls tumour primary niche formation. Nat Cell Biol, 2016. 18(9): p. 1006–17."/>
  <meta name="citation_reference" content="Castano, C., et al., Obesity-associated exosomal miRNAs modulate glucose and lipid metabolism in mice. Proc Natl Acad Sci U S A, 2018. 115(48): p. 12158–12163."/>
  <meta name="citation_reference" content="Wortzel, I., et al., Exosome-Mediated Metastasis: Communication from a Distance. Dev Cell, 2019. 49(3): p. 347–360."/>
  <meta name="citation_reference" content="Schwarzenbach, H., et al., Clinical relevance of circulating cell-free microRNAs in cancer. Nature Reviews Clinical Oncology, 2014. 11: p. 145."/>
  <meta name="citation_reference" content="Gilad, S., et al., Serum microRNAs are promising novel biomarkers. PLoS One, 2008. 3(9): p. e3148."/>
  <meta name="citation_reference" content="Pegtel, D.M., et al., Functional delivery of viral miRNAs via exosomes. Proc Natl Acad Sci U S A, 2010. 107(14): p. 6328–33."/>
  <meta name="citation_reference" content="Vickers, K.C., et al., MicroRNAs are transported in plasma and delivered to recipient cells by high-density lipoproteins. Nat Cell Biol, 2011. 13(4): p. 423–33."/>
  <meta name="citation_reference" content="Pastuzyn, E.D., et al., The Neuronal Gene Arc Encodes a Repurposed Retrotransposon Gag Protein that Mediates Intercellular RNA Transfer. Cell, 2018. 172(1-2): p. 275–288 e18."/>
  <meta name="citation_reference" content="Kosaka, N., et al., Secretory mechanisms and intercellular transfer of microRNAs in living cells. J Biol Chem, 2010. 285(23): p. 17442–52."/>
  <meta name="citation_reference" content="Thomou, T., et al., Adipose-derived circulating miRNAs regulate gene expression in other tissues. Nature, 2017. 542(7642): p. 450–455."/>
  <meta name="citation_reference" content="Sharma, U., et al., Small RNAs Are Trafficked from the Epididymis to Developing Mammalian Sperm. Dev Cell, 2018. 46(4): p. 481–494 e6."/>
  <meta name="citation_reference" content="Gay, L., et al., Mouse TU tagging: a chemical/genetic intersectional method for purifying cell type-specific nascent RNA. Genes Dev, 2013. 27(1): p. 98–115."/>
  <meta name="citation_reference" content="Herzog, V.A., et al., Thiol-linked alkylation of RNA to assess expression dynamics. Nat Methods, 2017. 14(12): p. 1198–1204."/>
  <meta name="citation_reference" content="Chatzi, C., et al., Transcriptional Profiling of Newly Generated Dentate Granule Cells Using TU Tagging Reveals Pattern Shifts in Gene Expression during Circuit Integration. eNeuro, 2016. 3(1)."/>
  <meta name="citation_reference" content="Miller, M.R., et al., TU-tagging: cell type-specific RNA isolation from intact complex tissues. Nat Methods, 2009. 6(6): p. 439–41."/>
  <meta name="citation_reference" content="Ghosh, A.C., et al., UPRT, a suicide-gene therapy candidate in higher eukaryotes, is required for Drosophila larval growth and normal adult lifespan. Sci Rep, 2015. 5: p. 13176."/>
  <meta name="citation_reference" content="Maquat, L.E. and M. Kiledjian, RNA turnover in eukaryotes: nucleases, pathways and analysis of mRNA decay. Preface. Methods Enzymol, 2008. 448: p. xxi–xxii."/>
  <meta name="citation_reference" content="Hida, N., et al., EC-tagging allows cell type-specific RNA analysis. Nucleic Acids Res, 2017. 45(15): p. e138."/>
  <meta name="citation_reference" content="Song, A.J. and R.D. Palmiter, Detecting and Avoiding Problems When Using the Cre-lox System. Trends Genet, 2018. 34(5): p. 333–340."/>
  <meta name="citation_reference" content="Alberti, C., et al., Cell-type specific sequencing of microRNAs from complex animal tissues. Nat Methods, 2018. 15(4): p. 283–289."/>
  <meta name="citation_reference" content="Marquart, T.J., et al., miR-33 links SREBP-2 induction to repression of sterol transporters. Proc Natl Acad Sci U S A, 2010. 107(27): p. 12228–32."/>
  <meta name="citation_reference" content="Rayner, K.J., et al., MiR-33 contributes to the regulation of cholesterol homeostasis. Science, 2010. 328(5985): p. 1570–3."/>
  <meta name="citation_reference" content="Zheng, L., et al., Effect of miRNA-10b in regulating cellular steatosis level by targeting PPAR-alpha expression, a novel mechanism for the pathogenesis of NAFLD. J Gastroenterol Hepatol, 2010. 25(1): p. 156–63."/>
  <meta name="citation_reference" content="Lee, E.K., et al., miR-130 suppresses adipogenesis by inhibiting peroxisome proliferator-activated receptor gamma expression. Mol Cell Biol, 2011. 31(4): p. 626–38."/>
  <meta name="citation_reference" content="Jao, C.Y. and A. Salic, Exploring RNA transcription and turnover in vivo by using click chemistry. Proc Natl Acad Sci U S A, 2008. 105(41): p. 15779–84."/>
  <meta name="citation_reference" content="Best, M.D., Click chemistry and bioorthogonal reactions: unprecedented selectivity in the labeling of biological molecules. Biochemistry, 2009. 48(28): p. 6571–84."/>
  <meta name="citation_reference" content="Hagemeijer, M.C., et al., Visualizing coronavirus RNA synthesis in time by using click chemistry. J Virol, 2012. 86(10): p. 5808–16."/>
  <meta name="citation_reference" content="Gierlich, J., et al., Click chemistry as a reliable method for the high-density postsynthetic functionalization of alkyne-modified DNA. Org Lett, 2006. 8(17): p. 3639–42."/>
  <meta name="citation_reference" content="Meyer, J.P., et al., Click Chemistry and Radiochemistry: The First 10 Years. Bioconjug Chem, 2016. 27(12): p. 2791–2807."/>
  <meta name="citation_reference" content="Pradere, U., et al., Synthesis of nucleoside phosphate and phosphonate prodrugs. Chem Rev, 2014. 114(18): p. 9154–218."/>
  <meta name="citation_reference" content="Erion, M.D., et al., Design, synthesis, and characterization of a series of cytochrome P(450) 3A-activated prodrugs (HepDirect prodrugs) useful for targeting phosph(on)ate-based drugs to the liver. J Am Chem Soc, 2004. 126(16): p. 5154–63."/>
  <meta name="citation_reference" content="Boyer, S.H., et al., Synthesis and characterization of a novel liver-targeted prodrug of cytosine-1-beta-D-arabinofuranoside monophosphate for the treatment of hepatocellular carcinoma. J Med Chem, 2006. 49(26): p. 7711–20."/>
  <meta name="citation_reference" content="Erion, M.D., et al., Targeting thyroid hormone receptor-beta agonists to the liver reduces cholesterol and triglycerides and improves the therapeutic index. Proc Natl Acad Sci U S A, 2007. 104(39): p. 15490–5."/>
  <meta name="citation_reference" content="Reddy, K.R., et al., Pradefovir: a prodrug that targets adefovir to the liver for the treatment of hepatitis B. J Med Chem, 2008. 51(3): p. 666–76."/>
  <meta name="citation_reference" content="van Herwaarden, A.E., et al., Midazolam and cyclosporin a metabolism in transgenic mice with liver-specific expression of human CYP3A4. Drug Metab Dispos, 2005. 33(7): p. 892–5."/>
  <meta name="citation_reference" content="van Herwaarden, A.E., et al., Knockout of cytochrome P450 3A yields new mouse models for understanding xenobiotic metabolism. J Clin Invest, 2007. 117(11): p. 3583–92."/>
  <meta name="citation_reference" content="Simonet, W.S., et al., A far-downstream hepatocyte-specific control region directs expression of the linked human apolipoprotein E and C-I genes in transgenic mice. J Biol Chem, 1993. 268(11): p. 8221–9."/>
  <meta name="citation_reference" content="Stresser, D.M., et al., Highly selective inhibition of human CYP3Aa in vitro by azamulin and evidence that inhibition is irreversible. Drug Metab Dispos, 2004. 32(1): p. 105–12."/>
  <meta name="citation_reference" content="Su, D., et al., Quantitative analysis of ribonucleoside modifications in tRNA by HPLC-coupled mass spectrometry. Nat Protoc, 2014. 9(4): p. 828–41."/>
  <meta name="citation_reference" content="Williams, L.M., et al., The development of diet-induced obesity and glucose intolerance in C57BL/6 mice on a high-fat diet consists of distinct phases. PLoS One, 2014. 9(8): p. e106159."/>
  <meta name="citation_reference" content="Dobin, A., et al., STAR: ultrafast universal RNA-seq aligner. Bioinformatics, 2013. 29(1): p. 15–21."/>
  <meta name="citation_reference" content="Tarazona, S., et al., Data quality aware analysis of differential expression in RNA-seq with NOISeq R/Bioc package. Nucleic Acids Res, 2015. 43(21): p. e140."/>
  <meta name="citation_reference" content="Chen, E.Y., et al., Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC Bioinformatics, 2013. 14: p. 128."/>
  <meta name="citation_reference" content="Coudrier, E., D. Kerjaschki, and D. Louvard, Cytoskeleton organization and submembranous interactions in intestinal and renal brush borders. Kidney Int, 1988. 34(3): p. 309–20."/>
  <meta name="citation_reference" content="Lin, G., E. Finger, and J.C. Gutierrez-Ramos, Expression of CD34 in endothelial cells, hematopoietic progenitors and nervous cells in fetal and adult mouse tissues. Eur J Immunol, 1995. 25(6): p. 1508–16."/>
  <meta name="citation_reference" content="Fina, L., et al., Expression of the CD34 gene in vascular endothelial cells. Blood, 1990. 75(12): p. 2417–26."/>
  <meta name="citation_reference" content="Liu, Y., D.K. Qiu, and X. Ma, Liver X receptors bridge hepatic lipid metabolism and inflammation. J Dig Dis, 2012. 13(2): p. 69–74."/>
  <meta name="citation_reference" content="Kersten, S., et al., Peroxisome proliferator-activated receptor alpha mediates the adaptive response to fasting. J Clin Invest, 1999. 103(11): p. 1489–98."/>
  <meta name="citation_reference" content="Tyagi, S., et al., The peroxisome proliferator-activated receptor: A family of nuclear receptors role in various diseases. J Adv Pharm Technol Res, 2011. 2(4): p. 236–40."/>
  <meta name="citation_reference" content="Everett, L., A. Galli, and D. Crabb, The role of hepatic peroxisome proliferator-activated receptors (PPARs) in health and disease. Liver, 2000. 20(3): p. 191–199."/>
  <meta name="citation_reference" content="Park, J., et al., Single-cell transcriptomics of the mouse kidney reveals potential cellular targets of kidney disease. Science, 2018. 360(6390): p. 758–763."/>
  <meta name="citation_reference" content="MacParland, S.A., et al., Single cell RNA sequencing of human liver reveals distinct intrahepatic macrophage populations. Nat Commun, 2018. 9(1): p. 4383."/>
  <meta name="citation_reference" content="Love, M.I., W. Huber, and S. Anders, Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol, 2014. 15(12): p. 550."/>
  <meta name="citation_reference" content="Srinivasan, S., et al., Small RNA Sequencing across Diverse Biofluids Identifies Optimal Methods for exRNA Isolation. Cell, 2019. 177(2): p. 446–462 e16."/>
  <meta name="citation_reference" content="Wang, M., Y. Zhao, and B. Zhang, Efficient Test and Visualization of Multi-Set Intersections. Sci Rep, 2015. 5: p. 16923."/>
  <meta name="citation_reference" content="Liu, H., et al., Escherichia coli noncoding RNAs can affect gene expression and physiology of Caenorhabditis elegans. Nat Commun, 2012. 3: p. 1073."/>
  <meta name="citation_reference" content="Weiberg, A., et al., Fungal small RNAs suppress plant immunity by hijacking host RNA interference pathways. Science, 2013. 342(6154): p. 118–23."/>
  <meta name="citation_reference" content="Timmons, L. and A. Fire, Specific interference by ingested dsRNA. Nature, 1998. 395(6705): p. 854."/>
  <meta name="citation_reference" content="Rivkin, M., et al., Inflammation-Induced Expression and Secretion of MicroRNA 122 Leads to Reduced Blood Levels of Kidney-Derived Erythropoietin and Anemia. Gastroenterology, 2016. 151(5): p. 999–1010 e3."/>
  <meta name="citation_reference" content="Chai, C., et al., Metabolic Circuit Involving Free Fatty Acids, microRNA 122, and Triglyceride Synthesis in Liver and Muscle Tissues. Gastroenterology, 2017. 153(5): p. 1404–1415."/>
  <meta name="citation_reference" content="Rechavi, O., et al., Cell contact-dependent acquisition of cellular and viral nonautonomously encoded small RNAs. Genes Dev, 2009. 23(16): p. 1971–9."/>
  <meta name="citation_reference" content="Lee, Y.S., et al., Inflammation is necessary for long-term but not short-term high-fat diet-induced insulin resistance. Diabetes, 2011. 60(10): p. 2474–83."/>
  <meta name="citation_reference" content="Hotamisligil, G.S., Inflammation and metabolic disorders. Nature, 2006. 444(7121): p. 860–7."/>
  <meta name="citation_reference" content="Chou, C.H., et al., miRTarBase update 2018: a resource for experimentally validated microRNA-target interactions. Nucleic Acids Res, 2018. 46(D1): p. D296–D302."/>
  <meta name="citation_reference" content="Portius, D., C. Sobolewski, and M. Foti, MicroRNAs-Dependent Regulation of PPARs in Metabolic Diseases and Cancers. PPAR Res, 2017. 2017: p. 7058424."/>
  <meta name="citation_reference" content="Horie, T., et al., MicroRNA-33 regulates sterol regulatory element-binding protein 1 expression in mice. Nat Commun, 2013. 4: p. 2883."/>
  <meta name="citation_reference" content="Shimano, H., SREBPs: physiology and pathophysiology of the SREBP family. FEBS J, 2009. 276(3): p. 616–21."/>
  <meta name="citation_reference" content="Kirchner, S. and Z. Ignatova, Emerging roles of tRNA in adaptive translation, signalling dynamics and disease. Nat Rev Genet, 2015. 16(2): p. 98–112."/>
  <meta name="citation_reference" content="Enache, L.S., et al., Circulating RNA molecules as biomarkers in liver disease. Int J Mol Sci, 2014. 15(10): p. 17644–66."/>
  <meta name="citation_reference" content="Thul, P.J., et al., A subcellular map of the human proteome. Science, 2017. 356(6340)."/>
  <meta name="citation_reference" content="Balcells, I., S. Cirera, and P.K. Busk, Specific and sensitive quantitative RT-PCR of miRNAs with DNA primers. BMC Biotechnol, 2011. 11: p. 70."/>
  <meta name="citation_reference" content="Busk, P.K., A tool for design of primers for microRNA-specific quantitative RT-qPCR. BMC Bioinformatics, 2014. 15: p. 29."/>
  <meta name="citation_reference" content="Chan, P.P. and T.M. Lowe, GtRNAdb 2.0: an expanded database of transfer RNA genes identified in complete and draft genomes. Nucleic Acids Res, 2016. 44(D1): p. D184–9."/>
  <meta name="twitter:title" content="In-vivo targeted tagging of RNA isolates cell specific transcriptional responses to environmental stimuli and identifies liver-to-adipose RNA transfer"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Bio-fluids contain various circulating cell-free RNA transcripts (ccfRNAs). The composition of these ccfRNAs varies between bio-fluids and constitute tantalizing biomarker candidates for several pathologies. ccfRNAs have also been demonstrated as mediators of cellular communication, yet little is known about their function in physiological and developmental settings and most works are limited to in-vitro studies. Here, we have developed iTAG-RNA, a novel method for the unbiased tagging of RNA transcripts in mice in-vivo. We used this method to isolate hepatocytes and kidney proximal epithelial cells-specific transcriptional response to a dietary challenge without interfering with the tissue architecture, and to identify multiple hepatocyte-secreted ccfRNAs in plasma. We also identified transfer of these hepatic derived ccfRNAs to adipose tissue, where they likely serve as a buffering mechanism to maintain cholesterol and lipid homeostasis. Our findings directly demonstrate in-vivo transfer of RNAs between tissues and highlight its implications for endocrine signaling and homeostasis."/>
  <meta name="og-title" property="og:title" content="In-vivo targeted tagging of RNA isolates cell specific transcriptional responses to environmental stimuli and identifies liver-to-adipose RNA transfer"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/670398v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Bio-fluids contain various circulating cell-free RNA transcripts (ccfRNAs). The composition of these ccfRNAs varies between bio-fluids and constitute tantalizing biomarker candidates for several pathologies. ccfRNAs have also been demonstrated as mediators of cellular communication, yet little is known about their function in physiological and developmental settings and most works are limited to in-vitro studies. Here, we have developed iTAG-RNA, a novel method for the unbiased tagging of RNA transcripts in mice in-vivo. We used this method to isolate hepatocytes and kidney proximal epithelial cells-specific transcriptional response to a dietary challenge without interfering with the tissue architecture, and to identify multiple hepatocyte-secreted ccfRNAs in plasma. We also identified transfer of these hepatic derived ccfRNAs to adipose tissue, where they likely serve as a buffering mechanism to maintain cholesterol and lipid homeostasis. Our findings directly demonstrate in-vivo transfer of RNAs between tissues and highlight its implications for endocrine signaling and homeostasis."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-06-14"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>In-vivo targeted tagging of RNA isolates cell specific transcriptional responses to environmental stimuli and identifies liver-to-adipose RNA transfer | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-776165 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Bio-fluids contain various circulating cell-free RNA transcripts (ccfRNAs). The composition of these ccfRNAs varies between bio-fluids and constitute tantalizing biomarker candidates for several pathologies. ccfRNAs have also been demonstrated as mediators of cellular communication, yet little is known about their function in physiological and developmental settings and most works are limited to in-vitro studies. Here, we have developed iTAG-RNA, a novel method for the unbiased tagging of RNA transcripts in mice in-vivo. We used this method to isolate hepatocytes and kidney proximal epithelial cells-specific transcriptional response to a dietary challenge without interfering with the tissue architecture, and to identify multiple hepatocyte-secreted ccfRNAs in plasma. We also identified transfer of these hepatic derived ccfRNAs to adipose tissue, where they likely serve as a buffering mechanism to maintain cholesterol and lipid homeostasis. Our findings directly demonstrate in-vivo transfer of RNAs between tissues and highlight its implications for endocrine signaling and homeostasis.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Little is known about the biological function of circulating cell-free RNAs (
     <strong>ccfRNA</strong>). Found to be associated with exosomes, lipoproteins, ribonucleoproteins and more, these transcripts can be isolated and sequenced from multiple bio-fluids such as plasma, lymph, cerebral fluids, breast milk and more [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. ccfRNAs are directly implicated in the development of several pathologies including cancer and obesity [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>-
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>] and are intensively studied as disease biomarkers [
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, 
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Despite this, the role they play in physiological and developmental settings and in mediating cell-to-cell communication remains largely unknown. In-vitro, a growing number of works demonstrate the relevance of RNA based cellular communication [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>-
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>], however in-vivo evidence is still limited. This discrepancy is partly due to the difficulties posed to tracking ccfRNAs from transcriptional source to potential sites of action in-vivo. Indeed the tools available to study ccfRNAs in physiological settings are limited and very few studies attempt to tackle this problem directly. One work found evidence to suggest that the majority of circulating miRNAs originate in adipose tissue and that some of the adipose derived miRNAs may play a role in the regulation of liver Fgf21 levels [
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. However, this work focuses on miRNA and does not directly demonstrate transfer of RNAs between tissues nor directly identify adipose secreted RNAs.
    </p>
    <p id="p-5">Transfer of miRNAs was also demonstrated between epithelial cells of the caput epididymis to maturing spermatozoa, leading to a shift in sperm RNA content during its maturation [
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. This study made use of 4-thiouracil-tagging (TU-tagging) [
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>] combined with SLAM-Seq [
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>] to demonstrate loading of miRNAs transcribed in caput epididymis into maturing spermatozoa. TU-tagging entails cell-type specific expression of uracil phosphoribosyltransferase (UPRT) and administration of 4-thiouracil, with the assumption that only cells expressing UPRT would incorporate 4-thiouracil into transcribing RNA. Thio-RNA can then be purified and used for downstream gene expression analyses, or alternatively combined with SLAM-Seq to identify labeled transcripts. TU-tagging has proven useful in several additional systems [
     <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, 
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, 
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], however, given endogenous [
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] and alternative [
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] pathways for uracil incorporation, the labeling specificity in this method remains unclear. In addition, as is demonstrated in Herzog et. al [
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>] and by Sharma et. al [
     <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>], labeling with TU-tagging of PolI and PolIII transcripts is inefficient, rendering tRNAs and ribosomal transcripts unlabeled.
    </p>
    <p id="p-6">Indeed there are only a limited number of techniques enabling in-vivo targeted labeling of RNAs. In addition to TU-tagging, 5-ethynylcytosine-tagging (EC-tagging) [
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] is a new method, which utilizes cell-type specific co-expression of cytosine deaminase (CD) with UPRT to achieve RNA labeling with 5-ethynyluridine (5EU) following administration of 5-ethynylcytosine. Both TU and EC tagging use cre-recombination to express the relevant enzymes in a tissue specific manner and stochastic expression from the cre-promoter may lead to unwanted expression of the enzymes in different tissues [
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Finally, one recently developed method called Mime-seq allows for cell type specific labeling of microRNA [
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. In this method, tissue specific expression of a plant derived methyltransferase mediates a 3′-terminal 2′-O-methylation of microRNAs that, when combined with a methylation dependent library construction, allows for sequencing of tissue specific microRNAs. Mime-seq allows labeling of miRNAs alone leaving other RNA biotypes unlabeled. Given the need for a technique that allows for a Cre-independent and unbiased labeling of total RNA transcription in-vivo, we developed iTAG-RNA 
     <strong>[For In-vivo Targeted Tagging of RNA]</strong>. This method incorporates mouse genetics with a novel uridine analog and an established RNA labeling chemistry to allow tagging of total RNA in target cells in-vivo. Using iTAG-RNA we are able to identify transcriptional re-programming of hepatocytes in-vivo following an acute high fat diet stress and to enrich for and identify hepatocyte derived plasma ccfRNAs. Moreover, we are able to identify RNA-based liver-to-adipose RNA transfer. These liver derived ccfRNAs include variable coding and non-coding RNAs such as miRNAs and tRNAs. Among the miRNAs transferred from liver to adipose tissue we find mir-33, mir-10b and mir130a, which target major regulators of cholesterol and lipid efflux and bio-synthesis such as Srebf1 [
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], Abca1 [
     <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>, 
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], Ppara [
     <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] and Pparg [
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] respectively.
    </p>
    <p id="p-7">Our study demonstrates for the first time an unbiased technique that allows labelling, tracking and quantification of variable types of ccfRNAs from their transcriptional source to downstream tissues, in which they can potentially act to regulate expression of target genes. We demonstrate RNA-based liver-to-adipose transfer of a myriad of RNA transcripts and their response to an environmental challenge. The continued identification and characterization of RNA based signaling in-vivo is imperative for the understanding of developmental, physiological and pathological processes, and can aid in the future development of relevant disease biomarkers.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Small molecule design and genetic approach for targeted in-vivo labeling of RNA</h3>
     <p id="p-8">5-Ethynyl Uridine 
      <strong>(5EU)</strong> is a synthetic uridine analogue extensively used in RNA turnover studies [
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>-
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. The nitrogenous base contains an alkyne group that can be covalently linked to an azide group using a simple copper mediated reaction called click chemistry [
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, 
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. This synthetic base has been demonstrated to incorporate into transcribing RNA in place of uridine and have little to no biological effects thereafter [
      <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>]. Following administration to mice, 5EU is readily taken up by cells with no regard to cell identity, depending to some extent on the administration method and dosage used [
      <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>]. Here, we present a novel method for the targeted in-vivo delivery of 5EU. To achieve this, we designed a ‘pro-drug’ of the 5EU base 
      <strong>(HD5EU)</strong> that is based on the ‘Hep-Direct’ pro-drug design [
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, 
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>] (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). This design was developed to target small molecules and nucleotide analogues to the human CYP3A4 enzyme and several small molecules of this design have been or are currently in clinical studies [
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>-
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. The human CYP3A4 enzyme catalyzes an oxidative cleavage of the HD5EU small molecule which, following a spontaneous beta-elimination, results in the formation of 5EU mono-phosphate that can then be incorporated into transcribing RNA (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1a</a>). HD5EU was synthesized by Chiroblock GmbH, the identity of the final product was validated using MS, p-NMR and h-NMR and the molecule’s purity was assessed at over 98% (
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Sup. Figure 1a-d</a>).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="HD5EU small molecule design and CYP3A4 expression pattern. A) HD5EU small molecule and metabolite structure relative to 5EU. B) W.B. depicting tissue expression pattern of CYP3A4 in humanize CYP3A4 cluster deleted mice. HSP90 serves as loading control. GAS = Gastrocnemius muscle; CM = Cardiac Muscle; BAT = Brown adipose tissue; ScA = Subcutaneous white adipose tissue; VsA = Visceral white adipose tissue; S.I. = Small intestine; L.I. = Large intestine. C) qRT-PCR validating liver and kidney specific expression of CYP3A4. Error-bars for standard-deviation of 3 biological repeats. D) Administration of the HD5EU small molecule to cells expressing CYP3A4 allows metabolism to 5EU and labelling of total RNA. Labeled transcripts are then secreted to the extracellular matrix and can be identified in bio-fluids and recipient cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-909241570" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;HD5EU small molecule design and CYP3A4 expression pattern.&lt;;/span&gt;; &lt;;strong&gt;;A)&lt;;/strong&gt;; HD5EU small molecule and metabolite structure relative to 5EU. &lt;;strong&gt;;B)&lt;;/strong&gt;; W.B. depicting tissue expression pattern of CYP3A4 in humanize CYP3A4 cluster deleted mice. HSP90 serves as loading control. GAS = Gastrocnemius muscle; CM = Cardiac Muscle; BAT = Brown adipose tissue; ScA = Subcutaneous white adipose tissue; VsA = Visceral white adipose tissue; S.I. = Small intestine; L.I. = Large intestine. &lt;;strong&gt;;C)&lt;;/strong&gt;; qRT-PCR validating liver and kidney specific expression of CYP3A4. Error-bars for standard-deviation of 3 biological repeats. &lt;;strong&gt;;D)&lt;;/strong&gt;; Administration of the HD5EU small molecule to cells expressing CYP3A4 allows metabolism to 5EU and labelling of total RNA. Labeled transcripts are then secreted to the extracellular matrix and can be identified in bio-fluids and recipient cells.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F1.medium.gif" width="382" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F1.medium.gif" width="382" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">HD5EU small molecule design and CYP3A4 expression pattern.</span>
       <p id="p-9" class="first-child">
        <strong>A)</strong> HD5EU small molecule and metabolite structure relative to 5EU. 
        <strong>B)</strong> W.B. depicting tissue expression pattern of CYP3A4 in humanize CYP3A4 cluster deleted mice. HSP90 serves as loading control. GAS = Gastrocnemius muscle; CM = Cardiac Muscle; BAT = Brown adipose tissue; ScA = Subcutaneous white adipose tissue; VsA = Visceral white adipose tissue; S.I. = Small intestine; L.I. = Large intestine. 
        <strong>C)</strong> qRT-PCR validating liver and kidney specific expression of CYP3A4. Error-bars for standard-deviation of 3 biological repeats. 
        <strong>D)</strong> Administration of the HD5EU small molecule to cells expressing CYP3A4 allows metabolism to 5EU and labelling of total RNA. Labeled transcripts are then secreted to the extracellular matrix and can be identified in bio-fluids and recipient cells.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-10">In addition to the HD5EU small molecule we took advantage of the published humanized liver specific CYP3A4 mouse line 
      <strong>FVB/129P2-
       <em>Cyp3a13</em>
      </strong>
      <sup>
       <strong>tm1Ahs</strong>
      </sup>
      <strong>Del(
       <em>5Cyp3a57-Cyp3a59</em>)
      </strong>
      <sup>
       <strong>1Ahs</strong>
      </sup>
      <strong>Tg(
       <em>APOE</em>-CYP3A4)
      </strong>
      <sup>
       <strong>A1Ahs</strong>
      </sup> obtained from Taconic [
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, 
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. These humanized mice 
      <strong>(hCYP3A4)</strong> express the human CYP3A4 enzyme under a modified Apolipoprotein E (APOE) promoter and are stably knocked out for nine homologous murine genes, thus leaving the human enzyme as the sole member of the enzyme family to be expressed in a Cre-independent, tissue-specific manner in-vivo. In keeping with published data on the activity of the modified ApoE promoter [
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>], qRT-PCR and WB analyses demonstrate restricted expression of the human Cyp3a4 enzyme to liver and kidney (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1b-c</a>). As such, upon administration of the HD5EU small molecule to the humanized CYP3A4 mice, we expect the molecule to be metabolized to bioavailable 5EU mono-phosphate exclusively in cells expressing CYP3A4, namely hepatocytes and kidney proximal renal epithelial cells, thus allowing in-vivo targeted labelling of transcription and identification of secreted transcripts in bio-fluids upon pull-down of 5EU labelled RNA (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1d</a>).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>CYP3A4 is necessary for in-vitro and in-vivo metabolism of the HD5EU small molecule</h3>
     <p id="p-11">To test and validate the metabolism of the HD5EU small molecule, we first isolated primary hepatocytes from hCYP3A4 mice. Following an 8 hour treatment of primary hepatocytes with 1mM HD5EU or 5EU, nuclear staining similar to 5EU labeling is clearly evident following click-it fluorescent staining (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>). When pre-treated with Azamulin, a highly selective CYP3A4 inhibitor [
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>], HD5EU treated primary hepatocytes no longer demonstrate nuclear staining, in contrast to 5EU treated primary hepatocytes, where nuclear staining is unaffected by Azamulin pretreatment (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a>). These results indicate that while 5EU is still readily incorporated into transcribing RNA in the nucleus, nuclear staining in HD5EU treated cells is dependent upon CYP3A4 activity.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Tissue specific staining evident with HD5EU is dependent on CYP3A4 activity. A) Primary hepatocytes demonstrate fluorescent nuclear staining of nascent RNA transcription following 8 hours of 5EU and HD5EU treatment. Azamulin treatment hinders fluorescent nuclear staining in HD5EU treated cells but not 5EU treated cells. 594-azide used for click-it staining. B) Following 2 hours of HD5EU administration to mice, in-vivo fluorescent nuclear staining is restricted to hepatocytes and epithelial cells in the kidney and absent from other tissue as evident in panel C and supplementary figure 2. Sol. = solvent for HD5EU. C) A similar treatment with 5EU in-vivo results in fluorescent nuclear staining of cells in multiple tissue including; Spleen, Pancreas, Muscle, Small Intestine, Brain and VsWAT. Consecutive injections did not change the observed staining pattern as no tissue apart from liver and kidney demonstrated positive staining. Scale Bar = 50[[#xB5]]M. For negative controls see supplementary figure 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-909241570" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Tissue specific staining evident with HD5EU is dependent on CYP3A4 activity.&lt;;/span&gt;; &lt;;strong&gt;;A)&lt;;/strong&gt;; Primary hepatocytes demonstrate fluorescent nuclear staining of nascent RNA transcription following 8 hours of 5EU and HD5EU treatment. Azamulin treatment hinders fluorescent nuclear staining in HD5EU treated cells but not 5EU treated cells. 594-azide used for click-it staining. &lt;;strong&gt;;B)&lt;;/strong&gt;; Following 2 hours of HD5EU administration to mice, in-vivo fluorescent nuclear staining is restricted to hepatocytes and epithelial cells in the kidney and absent from other tissue as evident in panel C and supplementary figure 2. Sol. = solvent for HD5EU. &lt;;strong&gt;;C)&lt;;/strong&gt;; A similar treatment with 5EU in-vivo results in fluorescent nuclear staining of cells in multiple tissue including; Spleen, Pancreas, Muscle, Small Intestine, Brain and VsWAT. Consecutive injections did not change the observed staining pattern as no tissue apart from liver and kidney demonstrated positive staining. Scale Bar = 50[[#xB5]]M. For negative controls see supplementary figure 2.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F2.medium.gif" width="440" height="248"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F2.medium.gif" width="440" height="248"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Tissue specific staining evident with HD5EU is dependent on CYP3A4 activity.</span>
       <p id="p-12" class="first-child">
        <strong>A)</strong> Primary hepatocytes demonstrate fluorescent nuclear staining of nascent RNA transcription following 8 hours of 5EU and HD5EU treatment. Azamulin treatment hinders fluorescent nuclear staining in HD5EU treated cells but not 5EU treated cells. 594-azide used for click-it staining. 
        <strong>B)</strong> Following 2 hours of HD5EU administration to mice, in-vivo fluorescent nuclear staining is restricted to hepatocytes and epithelial cells in the kidney and absent from other tissue as evident in panel C and 
        <a id="xref-fig-8-1" class="xref-fig" href="#F8">supplementary figure 2</a>. Sol. = solvent for HD5EU. 
        <strong>C)</strong> A similar treatment with 5EU in-vivo results in fluorescent nuclear staining of cells in multiple tissue including; Spleen, Pancreas, Muscle, Small Intestine, Brain and VsWAT. Consecutive injections did not change the observed staining pattern as no tissue apart from liver and kidney demonstrated positive staining. Scale Bar = 50µM. For negative controls see 
        <a id="xref-fig-8-2" class="xref-fig" href="#F8">supplementary figure 2</a>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-13">We administered HD5EU to humanized CYP3A4 mice and already 2 hours following administration found robust nuclear staining in hepatocytes and kidney epithelial cells but in no other tissue examined. This in contrast to mice administered with 5EU where nuclear staining was evident in multiple tissues (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2b-c</a> and 
      <a id="xref-fig-8-3" class="xref-fig" href="#F8">Sup. Figure 2a</a>). Of note, animals administered with HD5EU did not demonstrate any visible side effects. In addition we could not detect any signs of DNA damage or apoptosis in the liver during the different treatment regimens as demonstrated by staining for phosph-P53 and cleaved Caspase-3, supporting HD5EU as a non-toxic agent (
      <a id="xref-fig-8-4" class="xref-fig" href="#F8">Sup Figure 2b</a>).
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Mass-Spectrometric validation and quantification of 5EU incorporation into RNA following HD5EU treatment</h3>
     <p id="p-14">To validate that nuclear staining evident in-vitro and in-vivo following HD5EU treatment is indeed indicative of 5EU incorporation into transcribing RNA, we adopted the mass-spectrometry method described by 
      <strong>Su et. al [
       <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]
      </strong>. Using a column with a smaller inner diameter and lower flow rates to improve the response of individual nucleotides, we were able to identify a wide range of unmodified and modified nucleotides 
      <strong>(Sup. Table 1)</strong>. 5EU (m/z 269.0768) was well separated from the potential interfering 13C-Adenosine isotope (Adenosine m/z 268.1040, 13C-Adenosine m/z 269.106541) in standard samples (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). Due to mass spectrometric settings all nucleotides show a prominent in-source fragment, which corresponds to the neutral loss of the ribose (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3b</a>, denoted as [M-132+H]
      <sup>+</sup>).
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="MS validation and quantification of 5EU in liver RNA of HD5EU treated mice. A) Extracted ion chromatograms from A, C, G, U and 5EU and gradient slope. B) Close up of extracted ion chromatograms of 5EU [M+H]+ and 5EU in-source fragment [M-132+H]+. C) Tandem MS spectra at 20 eV of standard and sample indicating main fragment [M-132+H]+. D) Further fragments at 40eV used for identification of 5EU in RNA samples. A-D upper panels depict standard, lower panels depict liver long RNA from HD5EU treated mice. E) Relative quantification of 5EU in primary hepatocytes treated for 8h with the indicated compounds. Azamulin treatment inhibits CYP3A4 mediated HD5EU metabolism to 5EUMP and its subsequent incorporation into RNA. F) Bioanalyzer image depicting isolation of liver long and short RNA. G) Relative quantification of 5EU in liver derived long / small RNA following indicated time after 5EU / HD5EU administration to mice. Error bars indicated standard deviation calculated for 3 biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-909241570" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;MS validation and quantification of 5EU in liver RNA of HD5EU treated mice.&lt;;/span&gt;; &lt;;strong&gt;;A)&lt;;/strong&gt;; Extracted ion chromatograms from A, C, G, U and 5EU and gradient slope. &lt;;strong&gt;;B)&lt;;/strong&gt;; Close up of extracted ion chromatograms of 5EU [M+H]&lt;;sup&gt;;+&lt;;/sup&gt;; and 5EU in-source fragment [M-132+H]&lt;;sup&gt;;+&lt;;/sup&gt;;. &lt;;strong&gt;;C)&lt;;/strong&gt;; Tandem MS spectra at 20 eV of standard and sample indicating main fragment [M-132+H]&lt;;sup&gt;;+&lt;;/sup&gt;;. &lt;;strong&gt;;D)&lt;;/strong&gt;; Further fragments at 40eV used for identification of 5EU in RNA samples. &lt;;strong&gt;;A-D&lt;;/strong&gt;; upper panels depict standard, lower panels depict liver long RNA from HD5EU treated mice. &lt;;strong&gt;;E)&lt;;/strong&gt;; Relative quantification of 5EU in primary hepatocytes treated for 8h with the indicated compounds. Azamulin treatment inhibits CYP3A4 mediated HD5EU metabolism to 5EUMP and its subsequent incorporation into RNA. &lt;;strong&gt;;F)&lt;;/strong&gt;; Bioanalyzer image depicting isolation of liver long and short RNA. &lt;;strong&gt;;G)&lt;;/strong&gt;; Relative quantification of 5EU in liver derived long / small RNA following indicated time after 5EU / HD5EU administration to mice. Error bars indicated standard deviation calculated for 3 biological replicates.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F3.medium.gif" width="440" height="269"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F3.medium.gif" width="440" height="269"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">MS validation and quantification of 5EU in liver RNA of HD5EU treated mice.</span>
       <p id="p-15" class="first-child">
        <strong>A)</strong> Extracted ion chromatograms from A, C, G, U and 5EU and gradient slope. 
        <strong>B)</strong> Close up of extracted ion chromatograms of 5EU [M+H]
        <sup>+</sup> and 5EU in-source fragment [M-132+H]
        <sup>+</sup>. 
        <strong>C)</strong> Tandem MS spectra at 20 eV of standard and sample indicating main fragment [M-132+H]
        <sup>+</sup>. 
        <strong>D)</strong> Further fragments at 40eV used for identification of 5EU in RNA samples. 
        <strong>A-D</strong> upper panels depict standard, lower panels depict liver long RNA from HD5EU treated mice. 
        <strong>E)</strong> Relative quantification of 5EU in primary hepatocytes treated for 8h with the indicated compounds. Azamulin treatment inhibits CYP3A4 mediated HD5EU metabolism to 5EUMP and its subsequent incorporation into RNA. 
        <strong>F)</strong> Bioanalyzer image depicting isolation of liver long and short RNA. 
        <strong>G)</strong> Relative quantification of 5EU in liver derived long / small RNA following indicated time after 5EU / HD5EU administration to mice. Error bars indicated standard deviation calculated for 3 biological replicates.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-16">Tandem mass spectrometry validated that 5EU is present in RNA extracted from the liver of HD5EU treated mice (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3a-d</a>). The main fragment at 20eV collision energy was the neutral loss of ribose ([M-132+H]
      <sup>+</sup>) consistent with the observed in-source fragment (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3b-c</a>). Further fragments of the remaining nucleoside fragment were observed under higher collision energy of 40 eV (
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3d</a>). To quantify 5EU incorporation into RNA we used the prominent in-source fragment of 5EU due to the low abundance of 5EU in biological samples, as this in-source fragment was up to 3-5-fold higher than the intact molecule. In-vitro, Azamulin treatment of primary hepatocytes inhibited HD5EU metabolism and incorporation into transcribing RNA (
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3e</a>), in-line with observed fluorescent staining (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2a</a>). In-vivo, we could detect and quantify 5EU incorporation into both short (less than 200bp) and long RNA isolated from the liver of HD5EU treated mice, 2 hours following the administration of the compound (
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3f-g</a>). 8h following HD5EU administration 5EU was still detectable in long RNA (though it could not be accurately quantified as it was below quantification limit), whilst only a moderate reduction was detected in short RNA.
     </p>
     <p id="p-17">Taken together, these results confirm that HD5EU is metabolized in a CYP3A4 dependent manner to 5EU, which is then incorporated into transcribing RNA.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Robustness and reproducibility of RNA precipitation</h3>
     <p id="p-18">2 hours following administration of HD5EU, 5EU-containing liver and kidney transcripts can be biotinylated and pulled-down for next generation sequencing. We persistently failed to generate any amplified libraries following pull-down of unlabeled RNA isolated from liver, plasma or kidney of saline treated control mice (
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Sup. Figure 3a-e</a>). In addition, following a 2 hour treatment with HD5EU we failed to generate libraries following biotinylation and pull-down from plasma and from additional tissues of the HD5EU-treated mice, this in contrast to liver and kidney where expected library amplicons were generated (
      <a id="xref-fig-9-2" class="xref-fig" href="#F9">Sup. Figure 3a-e</a>). This result was consistent for both poly-A enriched and small RNA libraries and suggests biotinylation to be specific for 5EU containing transcripts. Technical replicates of pull-down libraries demonstrated a high degree of correlation between themselves, supporting the technical robustness and reproducibility of the method (
      <a id="xref-fig-9-3" class="xref-fig" href="#F9">Sup. Figure 3f</a>, spearman correlation coefficient = 0.95
      <strong>)</strong>.
     </p>
     <p id="p-19">To further assess the levels of non-specific RNA pull-down, we prepared a 10:1 mixture of non-labeled small RNAs from 
      <em>S. Cerevisiae</em> with labeled small RNAs derived from mouse liver. Library construction and sequencing of this mixture following RNA pull-down demonstrated highly effective depletion of yeast RNA compared to input (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>). These results demonstrate RNA pull-down to be highly selective to biotinylated RNA and non-specific RNA precipitation to be extremely low to not detectable.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="mRNA pull-down enriches for specific cell populations in vivo. A) Pull-down of a 10:1 mixture of unlabeled yeast RNA with labeled RNA from mouse liver specifically depletes unlabeled yeast RNA (gray) whilst enriching for the labelled mouse RNA (blue). Compare observed (mix O.) ratios of genomic alignment to expected ratios (mix E.) B) Scatter plot visualizing pulled-down (green) and depleted (purple) poly-A RNA from mouse kidney. C) GO and gene set enrichment analysis for pulled-down (green) and depleted (purple) protein coding transcripts in kidney. An enrichment for brush boarder membrane proteins, a unique feature in the kidney of proximal epithelial cells, is evident in pulled-down transcripts. Bubble size proportional to [[#x2013]]log10 of adj. PV. D) Scatter plot visualizing labelled pulled-down (green) and depleted (purple) poly-A RNA from mouse Liver. E) GO and gene set enrichment analysis for pulled-down (green) and depleted (purple) protein coding transcripts in liver. Bubble size proportional to [[#x2013]]log10 of adj. PV. F) Markers for various cell types found in the kidney and their relative expression in our input and pull-down poly-A RNA libraries. Markers adopted from Park J. et. al. [53]. PT = proximal tubule; endo = endothelial cells; fib = fibroblasts; LOH = Loop of Henle; lymp = lymphocytes; macro = macrophages; neutro = neutrophils; nkC = natural killer cells; novel = novel cell type; podo = podocytes; T-lymp = T-lymphocytes; CD = collecting duct; DCT = distal convoluted tubule. PD = pulled-down, DP = depleted. IN = Input. G) Markers for various cell types found in the liver and their relative expression in our input and pull-down poly-A RNA libraries. tC = t-cells; stC = stellate cells; kuf = kuffer cells; hep = hepatocytes; ery = erythrocytes; end = endothelial cells; cho = cholangiocytes; bC = b-cells. Markers adopted from MacParland et. al. [54]. PD = pulled-down, DP = depleted. IN = Input. H) Venn diagram demonstrating the overlap between differentially expressed genes in input liver mRNA (orange) and pull-down liver mRNA (Blue) with the corresponding scatterplot. HFD normalized mean expression on the X-axis, LFD normalized mean expression on the Y-axis. I) Differentially expressed genes in input kidney mRNA (orange) with the corresponding scatterplot. HFD normalized mean expression on the X-axis, LFD normalized mean expression on the Y-axis. No DEG detected in pull-down mRNA. Bubble size proportional to [[#x2013]]log10 of adj. PV. J) GO and gene set enrichment analysis for dietary induced differentially expressed protein coding genes identified in pull-down (blue) and input (orange) liver libraries. K) GO and gene set enrichment analysis for dietary induced differentially expressed protein coding genes identified in input kidney libraries. Bubble size proportional to [[#x2013]] log10 of adj. PV." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-909241570" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;mRNA pull-down enriches for specific cell populations in vivo.&lt;;/span&gt;; &lt;;strong&gt;;A)&lt;;/strong&gt;; Pull-down of a 10:1 mixture of unlabeled yeast RNA with labeled RNA from mouse liver specifically depletes unlabeled yeast RNA (gray) whilst enriching for the labelled mouse RNA (blue). Compare observed (mix O.) ratios of genomic alignment to expected ratios (mix E.) &lt;;strong&gt;;B)&lt;;/strong&gt;; Scatter plot visualizing pulled-down (green) and depleted (purple) poly-A RNA from mouse kidney. &lt;;strong&gt;;C)&lt;;/strong&gt;; GO and gene set enrichment analysis for pulled-down (green) and depleted (purple) protein coding transcripts in kidney. An enrichment for brush boarder membrane proteins, a unique feature in the kidney of proximal epithelial cells, is evident in pulled-down transcripts. Bubble size proportional to [[#x2013]]log&lt;;sub&gt;;10&lt;;/sub&gt;; of adj. PV. &lt;;strong&gt;;D)&lt;;/strong&gt;; Scatter plot visualizing labelled pulled-down (green) and depleted (purple) poly-A RNA from mouse Liver. &lt;;strong&gt;;E)&lt;;/strong&gt;; GO and gene set enrichment analysis for pulled-down (green) and depleted (purple) protein coding transcripts in liver. Bubble size proportional to [[#x2013]]log&lt;;sub&gt;;10&lt;;/sub&gt;; of adj. PV. &lt;;strong&gt;;F)&lt;;/strong&gt;; Markers for various cell types found in the kidney and their relative expression in our input and pull-down poly-A RNA libraries. Markers adopted from Park J. et. al. [53]. PT = proximal tubule; endo = endothelial cells; fib = fibroblasts; LOH = Loop of Henle; lymp = lymphocytes; macro = macrophages; neutro = neutrophils; nkC = natural killer cells; novel = novel cell type; podo = podocytes; T-lymp = T-lymphocytes; CD = collecting duct; DCT = distal convoluted tubule. PD = pulled-down, DP = depleted. IN = Input. &lt;;strong&gt;;G)&lt;;/strong&gt;; Markers for various cell types found in the liver and their relative expression in our input and pull-down poly-A RNA libraries. tC = t-cells; stC = stellate cells; kuf = kuffer cells; hep = hepatocytes; ery = erythrocytes; end = endothelial cells; cho = cholangiocytes; bC = b-cells. Markers adopted from MacParland et. al. [54]. PD = pulled-down, DP = depleted. IN = Input. &lt;;strong&gt;;H)&lt;;/strong&gt;; Venn diagram demonstrating the overlap between differentially expressed genes in input liver mRNA (orange) and pull-down liver mRNA (Blue) with the corresponding scatterplot. HFD normalized mean expression on the X-axis, LFD normalized mean expression on the Y-axis. &lt;;strong&gt;;I)&lt;;/strong&gt;; Differentially expressed genes in input kidney mRNA (orange) with the corresponding scatterplot. HFD normalized mean expression on the X-axis, LFD normalized mean expression on the Y-axis. No DEG detected in pull-down mRNA. Bubble size proportional to [[#x2013]]log&lt;;sub&gt;;10&lt;;/sub&gt;; of adj. PV. &lt;;strong&gt;;J)&lt;;/strong&gt;; GO and gene set enrichment analysis for dietary induced differentially expressed protein coding genes identified in pull-down (blue) and input (orange) liver libraries. &lt;;strong&gt;;K)&lt;;/strong&gt;; GO and gene set enrichment analysis for dietary induced differentially expressed protein coding genes identified in input kidney libraries. Bubble size proportional to [[#x2013]] log&lt;;sub&gt;;10&lt;;/sub&gt;; of adj. PV.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F4.medium.gif" width="440" height="360"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F4.medium.gif" width="440" height="360"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">mRNA pull-down enriches for specific cell populations in vivo.</span>
       <p id="p-20" class="first-child">
        <strong>A)</strong> Pull-down of a 10:1 mixture of unlabeled yeast RNA with labeled RNA from mouse liver specifically depletes unlabeled yeast RNA (gray) whilst enriching for the labelled mouse RNA (blue). Compare observed (mix O.) ratios of genomic alignment to expected ratios (mix E.) 
        <strong>B)</strong> Scatter plot visualizing pulled-down (green) and depleted (purple) poly-A RNA from mouse kidney. 
        <strong>C)</strong> GO and gene set enrichment analysis for pulled-down (green) and depleted (purple) protein coding transcripts in kidney. An enrichment for brush boarder membrane proteins, a unique feature in the kidney of proximal epithelial cells, is evident in pulled-down transcripts. Bubble size proportional to –log
        <sub>10</sub> of adj. PV. 
        <strong>D)</strong> Scatter plot visualizing labelled pulled-down (green) and depleted (purple) poly-A RNA from mouse Liver. 
        <strong>E)</strong> GO and gene set enrichment analysis for pulled-down (green) and depleted (purple) protein coding transcripts in liver. Bubble size proportional to –log
        <sub>10</sub> of adj. PV. 
        <strong>F)</strong> Markers for various cell types found in the kidney and their relative expression in our input and pull-down poly-A RNA libraries. Markers adopted from Park J. et. al. [
        <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. PT = proximal tubule; endo = endothelial cells; fib = fibroblasts; LOH = Loop of Henle; lymp = lymphocytes; macro = macrophages; neutro = neutrophils; nkC = natural killer cells; novel = novel cell type; podo = podocytes; T-lymp = T-lymphocytes; CD = collecting duct; DCT = distal convoluted tubule. PD = pulled-down, DP = depleted. IN = Input. 
        <strong>G)</strong> Markers for various cell types found in the liver and their relative expression in our input and pull-down poly-A RNA libraries. tC = t-cells; stC = stellate cells; kuf = kuffer cells; hep = hepatocytes; ery = erythrocytes; end = endothelial cells; cho = cholangiocytes; bC = b-cells. Markers adopted from MacParland et. al. [
        <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. PD = pulled-down, DP = depleted. IN = Input. 
        <strong>H)</strong> Venn diagram demonstrating the overlap between differentially expressed genes in input liver mRNA (orange) and pull-down liver mRNA (Blue) with the corresponding scatterplot. HFD normalized mean expression on the X-axis, LFD normalized mean expression on the Y-axis. 
        <strong>I)</strong> Differentially expressed genes in input kidney mRNA (orange) with the corresponding scatterplot. HFD normalized mean expression on the X-axis, LFD normalized mean expression on the Y-axis. No DEG detected in pull-down mRNA. Bubble size proportional to –log
        <sub>10</sub> of adj. PV. 
        <strong>J)</strong> GO and gene set enrichment analysis for dietary induced differentially expressed protein coding genes identified in pull-down (blue) and input (orange) liver libraries. 
        <strong>K)</strong> GO and gene set enrichment analysis for dietary induced differentially expressed protein coding genes identified in input kidney libraries. Bubble size proportional to – log
        <sub>10</sub> of adj. PV.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-7" class="subsection">
     <h3>RNA labelling in-vivo uncovers tissue architecture and stress induced transcriptional reprogramming</h3>
     <p id="p-21">We continued to examine if in-vivo labeling enriches for transcriptional programs of specific cellular populations within complex tissues in-vivo, such as the proximal renal epithelial cells in the kidney and hepatocytes in liver, and whether detection of environmentally induced transcriptional reprogramming is possible. To this end we fed mice with high fat (
      <strong>HFD</strong>) or control low fat (
      <strong>LFD</strong>) diets for two weeks. Following this acute HFD exposure, which is expected to alter the transcriptional program in the liver [
      <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>], we administered HD5EU 2 hours before sacrificing. We then continued to generate poly-A RNA libraries from kidney and liver input and pull-down RNA.
     </p>
     <p id="p-22">Following mapping with the STAR aligner [
      <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>], transcript quantification using HTSeq-count and differential pull-down analyses using the NOISeq package [
      <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>], we defined pulled-down transcripts as those whose abundance can be estimated with a high degree of confidence to be at least half of the abundance observed in input (i.e. at least 50% of the gene’s transcripts are labeled with a probability cut-off of 0.975) (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b-g</a>, 
      <strong>Sup. table 2</strong>).
     </p>
     <p id="p-23">In kidney, where proximal renal epithelial cells are labelled, GO annotation and gene set enrichment analysis using Enrichr [
      <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>] demonstrated an enrichment for genes localized to the brush boarder membrane, along with a few more general terms found enriched also in input such as mitochondria, focal adhesion and genes specific to or highly expressed in the kidney (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4b-c</a>, 
      <strong>Sup. table 3</strong>). The brush boarder membrane in the kidney is a unique feature of proximal renal epithelial cells [
      <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>], and among pulled-down transcripts, the Solute Carrier Family 9 member A3 (Slc9a3) is one of its specific markers. Slc9a3 is the sodium–hydrogen antiporter 3, which is highly expressed in the proximal tubule and allows active transport of sodium to the cell. Terms uniquely enriched in genes depleted following pull-down include genes associated with ribosomal function, genes specific to or highly expressed in CD34 positive cells, dendritic cells as well as genes associated to neutrophil ficolin granules. CD34 positive cells are likely endothelial cells found in glomeruli and blood vessels in both human and mice but absent from tubules [
      <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>, 
      <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. In liver, enrichment for identified liver targets of the nuclear receptors PPARA, LXR and RXR is evident in both depleted and pulled-down transcripts. Pulled-down transcripts demonstrate additional enrichment for identified liver targets of transcription factors such as Foxo1, Clock and Nucks1 and for transcripts localizing to nuclear speckles and nucleoli (
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4d-e</a>, 
      <strong>Sup. table 4)</strong>. LXR and RXR are implicated in lipid metabolism and heterodimerize to regulate gene expression. Their transcriptional upregulation is associated with increased hepatic lipogenesis [
      <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. PPARA instead, binds long chain free fatty acids and is a central regulator of lipid metabolism. It heterodimerizes with RXR or LXR to regulate mitochondrial and peroxisomal fatty acid oxidation [
      <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>-
      <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>].
     </p>
     <p id="p-24">Published single-cell sequencing from kidney [
      <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a>] and liver [
      <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>] supports depletion of genes associated with irrelevant cell types in both organs. Using published single-cell data, we compared the top ranking genes defined in each study to be cluster specific markers to our pull-down enrichment results. In kidney, we find the majority of cluster markers to be depleted following pull-down of kidney poly-A RNA, apart from a small subset of markers for proximal tubule cells (
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4f</a>). In liver, this trend continued and from the list of cluster specific markers, 80% of those identified as enriched following liver poly-A RNA pull-down were defined as markers of hepatocyte clusters.
     </p>
     <p id="p-25">Taken together these results support specific labelling of renal proximal tubule epithelial cells in the kidney and of hepatocytes in the liver, with enrichment of their transcriptomes in pulled-down RNA and specific depletion for genes associated with other cell types found in these organs. To assess the feasibility of detecting dynamic transcriptional responses using iTag-RNA, we examined whether diet-induced transcriptional reprogramming can be identified in pulled-down poly-A RNA, and to what extent it reflects transcriptional changes observed in whole-tissue input RNA. Differential gene expression analyses using the DEseq package [
      <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>] revealed a substantial overlap between diet induced transcriptional reprogramming in the liver as observed in input mRNA to transcriptional reprogramming observed in pull-down libraries (
      <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4g</a>, 
      <strong>Sup. table 5)</strong>. Though the total amount of differentially expressed genes 
      <strong>(DEG)</strong> was roughly 4 fold lower in pull-down vs. input libraries (157 vs. 636 DEG, with an FDR cutoff of less than 0.05 and absolute log2 fold change greater than 1), a 61% overlap (96 DEG) between the two sample sets was detected. This overlap is much higher than expected by chance (chi test &lt;;0.0001). Diet induced DEG in both input and pull-down liver RNA demonstrated a significant enrichment for genes regulated by PPARA, LXR and RXR (
      <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4h</a> and 
      <strong>Sup. Table 7)</strong>.
     </p>
     <p id="p-26">As opposed to liver, diet-induced differential expression in the kidney was limited to 108 transcripts in input poly-A RNA enriched for mitochondrial and ribosomal proteins, whilst pulled-down RNA demonstrated no transcriptional reprograming (
      <a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4i-j</a>, 
      <strong>Sup. table 6-7).</strong> These findings may reflect the more complex cellular composition of the kidney and a lack of transcriptional reprogramming in proximal renal epithelial cells.
     </p>
     <p id="p-27">Taken together, these results provide a proof-of-concept that iTAG-RNA allows isolation of cell-type specific transcriptional responses to environmental challenges. Importantly, with no need for the disruption of the tissue architecture or interference with the cellular microenvironment.</p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Hepatocyte derived ccfRNA are detected in Plasma</h3>
     <p id="p-28">Given the observed hepatic transcriptional reprogramming following a HFD challenge, we examined whether we can detect labeled hepatocyte derived plasma ccfRNAs, and if the profile of these secreted transcripts changes in response to the dietary challenge. Plasma isolated ccfRNAs are predominantly short / fragmented RNA transcripts with a bi-modal distribution and a major peak smaller than 200bp ([
      <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>] 
      <strong>and</strong>
      <a id="xref-fig-9-4" class="xref-fig" href="#F9">sup figure 3g</a>). As already described, with a single dose of HD5EU 2 hours before blood collection we failed to generate libraries from plasma ccfRNA following biotinylation and pull-down. However, with multiple doses of HD5EU administration 6, 4 and 2 hours before blood collection, we were able to generate small RNA libraries following pull-down of plasma ccfRNAs.
     </p>
     <p id="p-29">Multiple short RNAs were pulled-down in both liver and plasma under HFD and LFD. (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5a</a>, 459/992 for HFD and 234/700 for LFD). This co-occurrence rate is greater than expected by chance for HFD and LFD (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5a</a>, P.V calculated using the SuperExactTest package in R [
      <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>], Fold enrichment = HFD: 2.7; LFD = 1.8; P.V. = HFD: 4.3e-104; LFD = 6.42e-21). The identity of pulled-down plasma ccfRNAs varied between dietary challenges (
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5b</a>
      <strong>and Sup. tables 8-10)</strong>, with multiple reads found enriched only under a specific dietary challenge, suggesting that liver secreted ccfRNAs indeed change with dietary interventions.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Identification of hepatocyte secreted circulating transcripts in Plasma. A) Overlap between pulled-down small RNA transcripts in liver and plasma under HFD and LFD challenge. Number of transcripts per overlap indicated in the y-axis. Color corresponds to [[#x2013]]log10 of calculated P.V and is annotated in the chart. Gene sets included in each comparison are indicated in green. B) Scatter plot annotating selected miRNAs and mt-tRNAs identified as pulled-down in circulating plasma RNA under HFD and LFD conditions. Transcripts are either depleted following pull-down regardless of dietary regime (purple), pulled-down regardless of dietary regime (green) or pulled-down under HFD (red) or LFD (blue) dietary regimes only. C) Bubble plot representing the relative percentage of pulled-down transcripts per biotype in liver and plasma libraries, averaged across HFD and LFD regimes. Bubble size represents log10 of actual number of transcripts per bio-type. Error bars for standard deviation between sets. D) Bubble plot of GO and gene set enrichment analysis for constitutively pulled-down (green) and depleted (purple) cell-free circulating transcripts. Bubble size proportional to [[#x2013]]log10 of adj. PV. E) Bubble plot of GO and gene set enrichment analysis for cell-free circulating transcripts pulled-down in HFD (red) or LFD (blue). Bubble size proportional to [[#x2013]]log10 of adj. PV." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-909241570" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Identification of hepatocyte secreted circulating transcripts in Plasma.&lt;;/span&gt;; &lt;;strong&gt;;A)&lt;;/strong&gt;; Overlap between pulled-down small RNA transcripts in liver and plasma under HFD and LFD challenge. Number of transcripts per overlap indicated in the y-axis. Color corresponds to [[#x2013]]log&lt;;sub&gt;;10&lt;;/sub&gt;; of calculated P.V and is annotated in the chart. Gene sets included in each comparison are indicated in green. &lt;;strong&gt;;B)&lt;;/strong&gt;; Scatter plot annotating selected miRNAs and mt-tRNAs identified as pulled-down in circulating plasma RNA under HFD and LFD conditions. Transcripts are either depleted following pull-down regardless of dietary regime (purple), pulled-down regardless of dietary regime (green) or pulled-down under HFD (red) or LFD (blue) dietary regimes only. &lt;;strong&gt;;C)&lt;;/strong&gt;; Bubble plot representing the relative percentage of pulled-down transcripts per biotype in liver and plasma libraries, averaged across HFD and LFD regimes. Bubble size represents log10 of actual number of transcripts per bio-type. Error bars for standard deviation between sets. &lt;;strong&gt;;D)&lt;;/strong&gt;; Bubble plot of GO and gene set enrichment analysis for constitutively pulled-down (green) and depleted (purple) cell-free circulating transcripts. Bubble size proportional to [[#x2013]]log&lt;;sub&gt;;10&lt;;/sub&gt;; of adj. PV. &lt;;strong&gt;;E)&lt;;/strong&gt;; Bubble plot of GO and gene set enrichment analysis for cell-free circulating transcripts pulled-down in HFD (red) or LFD (blue). Bubble size proportional to [[#x2013]]log&lt;;sub&gt;;10&lt;;/sub&gt;; of adj. PV.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F5.medium.gif" width="440" height="238"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F5.medium.gif" width="440" height="238"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <span class="caption-title">Identification of hepatocyte secreted circulating transcripts in Plasma.</span>
       <p id="p-30" class="first-child">
        <strong>A)</strong> Overlap between pulled-down small RNA transcripts in liver and plasma under HFD and LFD challenge. Number of transcripts per overlap indicated in the y-axis. Color corresponds to –log
        <sub>10</sub> of calculated P.V and is annotated in the chart. Gene sets included in each comparison are indicated in green. 
        <strong>B)</strong> Scatter plot annotating selected miRNAs and mt-tRNAs identified as pulled-down in circulating plasma RNA under HFD and LFD conditions. Transcripts are either depleted following pull-down regardless of dietary regime (purple), pulled-down regardless of dietary regime (green) or pulled-down under HFD (red) or LFD (blue) dietary regimes only. 
        <strong>C)</strong> Bubble plot representing the relative percentage of pulled-down transcripts per biotype in liver and plasma libraries, averaged across HFD and LFD regimes. Bubble size represents log10 of actual number of transcripts per bio-type. Error bars for standard deviation between sets. 
        <strong>D)</strong> Bubble plot of GO and gene set enrichment analysis for constitutively pulled-down (green) and depleted (purple) cell-free circulating transcripts. Bubble size proportional to –log
        <sub>10</sub> of adj. PV. 
        <strong>E)</strong> Bubble plot of GO and gene set enrichment analysis for cell-free circulating transcripts pulled-down in HFD (red) or LFD (blue). Bubble size proportional to –log
        <sub>10</sub> of adj. PV.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-31">Various biotypes are identified in pulled-down liver and plasma short RNA libraries, with the relative proportion of pulled-down transcripts varying between the two. tRNAs, mitochondrial tRNAs and mitochondrial genes are found to be overly represented in pull-down RNA from plasma relative to liver (
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5c</a>). This result suggests that the majority of mitochondrial transcripts found in plasma originate predominantly in the liver.
     </p>
     <p id="p-32">Additional support in favor of the hepatic origin of plasma labeled transcripts can be found in fragments originating from protein coding genes. These protein coding fragments demonstrate a significant enrichment for liver specific and highly expressed genes, whilst transcripts constitutively depleted in pull-down RNA demonstrate an enrichment for bone marrow specific protein coding fragments, genes related to hematopoietic differentiation and genes specific to neutrophil function. (
      <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5d</a>
      <strong>and sup. table 11)</strong>. This result may suggests that the hematopoietic system is one of the major contributors of circulating RNAs. HFD and LFD specific pulled-down protein coding transcripts demonstrate differential enrichment for annotations including adipocytokine signaling and mitochondrial electron transport respectively (
      <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5e</a>). The significant enrichment found for liver specific protein coding genes among pulled-down transcripts supports the hypothesis that pulled-down ccfRNA transcripts originate in hepatocytes, where Cyp3a4 expression metabolizes HD5EU to 5EU, allowing its incorporation into nascent transcribing RNA.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Hepatic derived ccfRNA are found in visceral white adipose tissue where they contribute to the small RNA pool and potentially regulate lipid storage</h3>
     <p id="p-33">In worms, plants and prokaryotes, extracellular RNA signaling was described to modulate host/pathogen interactions and to orchestrate an adaptive response to environmental stimuli [
      <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>-
      <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>], and demonstrated the idea that biological systems can exist as holobionts characterized by continuous exchange of genetic (DNA, RNA) material. In mammals, RNA-based intercellular signaling has been described in several settings [
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, 
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, 
      <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>-
      <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>], using mostly in-vitro systems or ectopic administration of RNAs to demonstrate RNA transfer and signal transduction. Little is known on the extent of RNA transfer in-vivo and on the role it may play in physiological settings. Two weeks of high-fat diet feeding are sufficient to impair metabolic homeostasis [
      <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>], and induce morpho-functional alterations in both liver and visceral adipose tissue [
      <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">64</a>]. Given the central role liver and adipose tissue play in metabolic control [
      <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>], and existing evidence suggesting RNA-based signaling between the tissues [
      <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>], we used iTAG-RNA to identify diet-sensitive RNA-based liver-to-adipose signals.
     </p>
     <p id="p-34">The number of transcripts found to be enriched following pull-down in liver, plasma and VsWAT was 9.7 folds greater than expected by chance for HFD and 6.8 folds greater than expected for LFD (HFD: 264/27, PV=6.34e-180; LFD: 116/17, PV = 3.7e-59 as calculated using the SuperExactTest package in R [
      <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>]) 
      <strong>(Figure 6a Sup tables 8-10)</strong>. Focusing on identified hepatic transcribed plasma ccfRNAs, multiple small RNAs could be enriched for in VsWAT. These include miRNAs such as mir-33, mir-10b and mir-130a, in addition to mt-tRNAs (
      <strong>Figure 6b)</strong>. Among experimentally validated targets of the identified miRNAs (as annotated by miRTarBase [
      <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]), an enrichment is found for proteins associated with the GO term negative regulation of lipid storage (GO:0010888, adj. P.V. = 0.0078, enrichment score = 26.68) such as Abca1, Ppara and Pparg which are regulated by mir-33 [
      <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>, 
      <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>], mir-10b [
      <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>] and mir-130a [
      <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>, 
      <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>] respectively. Mir-33 is also an identified regulator of the Srebp family of transcription factors that are central in cholesterol and fatty acid synthesis [
      <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>, 
      <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>] and is in fact encoded within the intron of Srebf2. Total RNA sequencing of VsWAT identified 100 genes to be differentially expressed following acute HFD feeding (28 upregulated / 72 downregulated, 
      <strong>Sup table 12</strong>). As expected given the increased dietary intake of free fatty acids following HFD feeding, downregulation of Srebf1 and of several target genes involved in fatty acid biosynthesis such as FasN, Acaca, and Scd2 is evident (
      <strong>Figure 6c-d</strong>).
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Plasma ccfRNAs can be detected in VsWAT. A) Venn diagram demonstrating the overlap between pulled-down transcripts in expression sets from liver plasma and VsWAT for HFD and LFD (Blue = Liver; Red = Plasma; Yellow = VsWAT). B) miRNAs and mt-tRNAs that are pulled-down in both plasma and VsWAT (Red = HFD; Blue = LFD). C) Volcano plot demonstrating DEG in total RNA sequencing of VsWAT following 2 weeks of HFD or LFD challenge. Red [[#x2013]] downregulated in HFD, Green [[#x2013]] Upregulated in HFD. D) qRT-PCR estimations for differential expression of Pparg and Fasn in VsWAT. # = significant fold change, n=3. E) qRT-PCR estimations for differential expression of selected miRNAs. # = significant fold change, n=3. F) Western blot for Pparg, modest upregulation of Pparg levels is evident following 2 weeks of HFD. G) Depiction of small RNA transfer between liver and adipose tissue. H) A model suggesting that the small RNA pool within a cell results from a balance between transcription and degradation on the one hand and influx and efflux on the other." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-909241570" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Plasma ccfRNAs can be detected in VsWAT.&lt;;/span&gt;; &lt;;strong&gt;;A)&lt;;/strong&gt;; Venn diagram demonstrating the overlap between pulled-down transcripts in expression sets from liver plasma and VsWAT for HFD and LFD (Blue = Liver; Red = Plasma; Yellow = VsWAT). &lt;;strong&gt;;B)&lt;;/strong&gt;; miRNAs and mt-tRNAs that are pulled-down in both plasma and VsWAT (Red = HFD; Blue = LFD). &lt;;strong&gt;;C)&lt;;/strong&gt;; Volcano plot demonstrating DEG in total RNA sequencing of VsWAT following 2 weeks of HFD or LFD challenge. Red [[#x2013]] downregulated in HFD, Green [[#x2013]] Upregulated in HFD. &lt;;strong&gt;;D)&lt;;/strong&gt;; qRT-PCR estimations for differential expression of Pparg and Fasn in VsWAT. # = significant fold change, n=3. &lt;;strong&gt;;E)&lt;;/strong&gt;; qRT-PCR estimations for differential expression of selected miRNAs. # = significant fold change, n=3. &lt;;strong&gt;;F)&lt;;/strong&gt;; Western blot for Pparg, modest upregulation of Pparg levels is evident following 2 weeks of HFD. &lt;;strong&gt;;G)&lt;;/strong&gt;; Depiction of small RNA transfer between liver and adipose tissue. H) A model suggesting that the small RNA pool within a cell results from a balance between transcription and degradation on the one hand and influx and efflux on the other.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F6.medium.gif" width="440" height="369"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F6.medium.gif" width="440" height="369"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/14/670398/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <span class="caption-title">Plasma ccfRNAs can be detected in VsWAT.</span>
       <p id="p-35" class="first-child">
        <strong>A)</strong> Venn diagram demonstrating the overlap between pulled-down transcripts in expression sets from liver plasma and VsWAT for HFD and LFD (Blue = Liver; Red = Plasma; Yellow = VsWAT). 
        <strong>B)</strong> miRNAs and mt-tRNAs that are pulled-down in both plasma and VsWAT (Red = HFD; Blue = LFD). 
        <strong>C)</strong> Volcano plot demonstrating DEG in total RNA sequencing of VsWAT following 2 weeks of HFD or LFD challenge. Red – downregulated in HFD, Green – Upregulated in HFD. 
        <strong>D)</strong> qRT-PCR estimations for differential expression of Pparg and Fasn in VsWAT. # = significant fold change, n=3. 
        <strong>E)</strong> qRT-PCR estimations for differential expression of selected miRNAs. # = significant fold change, n=3. 
        <strong>F)</strong> Western blot for Pparg, modest upregulation of Pparg levels is evident following 2 weeks of HFD. 
        <strong>G)</strong> Depiction of small RNA transfer between liver and adipose tissue. H) A model suggesting that the small RNA pool within a cell results from a balance between transcription and degradation on the one hand and influx and efflux on the other.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Discussion</h2>
    <p id="p-36">In this work we present 
     <strong>iTAG-RNA</strong>, a novel method for targeted in-vivo labelling of global RNA transcription, and use it to identify hepatocyte secreted ccfRNA and their uptake by adipose tissue in-vivo. iTAG-RNA allows labelling of total RNA transcripts in-vivo using two main components: 
     <strong>HD5EU</strong>, a newly designed small molecule that serves as a metabolite for the human CYP3A4 enzyme; and 
     <strong>an existing humanized transgenic mouse model</strong> expressing the human CYP3A4 enzyme under a modified APOE promoter (
     <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). CYP3A4 catalyzes the oxidative cleavage of an aryl group of the HD5EU molecule, which in turn undergoes spontaneous beta elimination to produce a bio-available 5EU-monophosphate. The existing transgenic mouse model allows in-vivo labelling of hepatocytes and kidney proximal renal epithelial cells (
     <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), as the tissue expression pattern of the enzyme dictates the site of the small molecule’s metabolism and subsequent RNA labeling. To validate the specificity of HD5EU metabolism, we demonstrate that HD5EU is indeed metabolized in a CYP3A4 dependent manner to 5EU-monophosphate. 5EU is then incorporated in place of uridine into transcribing RNA (
     <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>-
     <a id="xref-fig-3-8" class="xref-fig" href="#F3">3</a>) and allows highly selective RNA precipitation and sequencing of labeled RNAs (
     <a id="xref-fig-9-5" class="xref-fig" href="#F9">Sup. Figure 3</a> and 
     <a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4a</a>). Development of new transgenic models would allow for labeling of multiple tissues of choice. Administration of HD5EU allows enrichment for the transcriptional program of proximal renal epithelial cells and hepatocytes in-situ without disruption of the kidney or liver architecture (
     <a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4b-g</a>). In addition, environmentally induced transcriptional reprogramming is evident following labelling (
     <a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4h-k</a>). As opposed to recently described methods [
     <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>, 
     <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, 
     <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>, 
     <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>] and in keeping with the literature where POLI, POLII and POLIII are demonstrated to incorporate 5EU [
     <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a>], mRNA and small RNAs of various types including rRNA, tRNA and miRNA are found to be labeled and enriched in pulled-down RNA.
    </p>
    <p id="p-37">Critically, and uniquely to iTag-RNA, we are also able to enrich for liver derived plasma ccfRNA following an administration of multiple doses of HD5EU (
     <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>). Pulled-down plasma ccfRNAs demonstrate an enrichment for liver derived RNA fragments of protein-coding genes, whilst depleted transcripts demonstrate an enrichment for annotation relating to function and differentiation of the hematopoietic system. Apart from fragments of protein coding genes, liver secreted ccfRNA include various small RNA transcripts such as miRNA, mt-tRNAs and tRNAs. Given the evident enrichment for mitochondrial transcripts following pull-down, our results suggest that hepatocytes are the main source of mitochondrially encoded ccfRNA transcripts in plasma. The functional significance of these tRNA fragments is unclear, but several studies have suggested tRNAs and tRNA fragments can mediate cellular signaling, and that the overall tRNA pool and composition within a cell has functional significance [
     <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>].
    </p>
    <p id="p-38">We continued to explore the possibility that ccfRNAs are taken up by tissues in-vivo to mediate cell-to-cell communication. Focusing on VsWAT we pulled-down and amplified labeled RNA, again following multiple doses of HD5EU 
     <strong>(Figure 6a)</strong>. The transcripts we identified as labeled in VsWAT demonstrated a greater overlap than expected by chance with labeled plasma ccfRNAs or indeed with labeled transcripts in the liver, supporting their hepatic transcriptional origin. These transcripts including miRNAs and mt-tRNAs 
     <strong>(Figure 6b)</strong>, with the identified miRNAs implicated in post-transcriptional regulation of proteins involved in lipid, cholesterol and fatty acid pathways. Together our results directly demonstrate for the first time, in-vivo transfer of a large variety of RNAs including miRNAs between hepatocytes and visceral adipose tissue (
     <strong>Figure 6g)</strong>, with the identity of transferred RNAs varying following an acute dietary challenge. In light of the observed diversity and scope of RNA transfer between hepatocytes and adipocytes, our results suggest that the pool of small RNAs in a cell in-vivo results from a balance not only between transcription and degradation, but also influx and efflux of small RNAs from the extracellular environment (
     <strong>Figure 6h)</strong>, with the latter serving as a buffering mechanism for transcriptional and physiological responses in target cells.
    </p>
    <p id="p-39">To date, the majority of potential ccfRNA biomarkers associated with liver pathologies have been miRNAs [
     <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>]. Our findings suggest that fragments of protein coding genes together with mitochondrial tRNAs and mitochondria encoded transcripts can also serve as useful biomarkers for hepatic function. Development of additional genetic models similar to the one used can allow for better transcriptional characterization of distinct cell populations in-vivo, with the added benefit of labelling endogenous ccfRNAs in-vivo. Lastly, HD5EU may prove beneficial in clinical settings. As metabolic activation of HD5EU requires the human Cyp3a4 enzyme, whose expression under physiological conditions is largely limited to the liver (The human protein atlas. 2019. CYP3A4. [ONLINE] Available at: 
     <a href="https://www.proteinatlas.org/ENSG00000160868-CYP3A4/tissue/primary+data">https://www.proteinatlas.org/ENSG00000160868-CYP3A4/tissue/primary+data</a>
     <strong>[
      <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>])
     </strong>, administration of this small molecule to humans may constitute a novel diagnostic tool allowing assessment of hepatic function by means of a liquid biopsy.
    </p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Methods</h2>
    <div id="sec-12" class="subsection">
     <h3>Mice handling</h3>
     <p id="p-40">Mice were purchased from Taconic (Taconic USA). All mice were kept in a SPF facility in accordance with the Bavarian Animal law. Mice were fed with a chow / high fat diet / low fat diet as indicated (Rodent Diet with 60 kcal% from fat - Research Diet D12492i, Rodent Diet with 10 kcal% from fat – Research Diet D12450B). 5EU (7848.2, Carl Roth) was solubilized in saline 0.9% NaCl, HD5EU in a 25 % PEG-400, 5% DMSO saline solution. Compounds were administered intraperitoneally at a dose of 0.15 / 0.3
      <sub>mg/g</sub> in a total volume of 200µl. First administration was always carried out at ZG-3 to avoid circadian effects. For blood and organ collection, mice were terminally anesthetized with Ketamin/Xylazine at indicated times following drug administration. Heart puncture was performed to collect blood in EDTA coated syringes. Blood was centrifuged at 4.8K rpm for 10’ followed by 12K rpm at 20’ and filtration through a 22uM PES filter (PA59.1, Carl Roth). For isolation of primary hepatocytes, mice were anesthetized and liver perfused through the vena cava with Gibco’s liver perfusion buffer (17701-038, Gibco) and liver digestion buffer (17703-034, Gibco) in accordance with manufacturer’s instructions.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Cell culture</h3>
     <p id="p-41">Isolated primary hepatocytes were counted using the countess automated cell counter (C10227, invitrogen), and plated to a density of 75k/Cm
      <sup>2</sup> on Geltrex (A1413201, ThermoFisher) coated coverslips (200 µg/cm
      <sup>2</sup>) in Williams’ Medium E (A12176, Gibco) supplemented with Gibco’s Primary Hepatocyte Maintenance Supplements (CM4000, Gibco). 24 hours following plating, cells were treated with 1mM of 5EU or HD5EU for 8 hours. Azamulin (SML0485, Sigma Aldrich) was added at a concentration of 20µM for 30 minutes before addition of indicated compounds to a final concentration of 10µM for the length of the treatment.
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Tissue processing and imaging</h3>
     <p id="p-42">Tissues were fixed in a neutral buffered 10% formalin solution (HT501128, Sigma) for 48 hours before dehydration and embedding in paraffin in accordance with published protocols. 4µm sections were cut on a Leica microtome (RM2165, Institute of Experimental Genetics), rehydrated and stained using the Click-iT™ RNA Alexa Fluor™ 594 Imaging Kit (C10330, ThermoFisher) in accordance with manufacturer’s instructions. Mounting was done with Vectashield hardset antifade mounting medium with DAPI (H-1500, Vector Laboratories). Imagining was done using a Laser Scanning Confocal Microscope (Olympus Fluoview 1200, Institute for Diabetes and Cancer, Neuherberg, Germany) equipped with an Olympus UPlanSApo 60x 1.35 Oil immersion objective.</p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Western Blot</h3>
     <p id="p-43">Tissues were homogenized using a Miltenyi gentleMACS Dissociator (Miltenyi biotec) in RIPA buffer supplemented with protease (S8820, Sigma Aldrich) and phosphatase (88667, Thermo Fisher) inhibitors. Protein concentration was measured using a standard Bradford assay reagent (B6916, Sigma Aldrich).40µg total protein were loaded per samples on a pre-cast gradient 4-12% gel (NW04120, Invitrogen). Proteins were transferred to a PVDF membrane (ISEQ00010, Merck Millipore) blocked and blotted using the iBind system (SLF1020, Invitrogen) with primary anti-CYP3A4 (MA5-17064, Thermo Fisher), anti-Phospho-p53 S392 (#9281, cell signaling), anti-total-p53 (#2524, cell signaling), anti-cleaved Caspase-3 (ab214430, abcam), anti-total Caspase-3 (#ab184787, abcam), anti-pparg (MA5-14889, Thermo Fisher), anti-histone H3 (4499s, cell signalling) and anti-HSP90 (SC-7949, Santa-Cruz), and secondary IgG HRP (7076 and 7074, Cell Signaling)</p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>RNA extraction, qRT-PCR, pull-down and library construction</h3>
     <p id="p-44">Plasma RNA was extracted using TRI Reagent BD (T3809, Sigma Aldrich), in accordance with manufacturer’s instructions. RNA from tissues was extracted using NucleoZOL (740404.200, Macherey-Nagel) reagent, in accordance with manufacturer’s instructions. For qRT-PCR, reverse transcription was conducted using the high-Capacity cDNA Reverse Transcription Kit (4368814, Applied Biosystems), in accordance with the manufacturer’s instructions. Real-time was carried out on a quant-studio 6 flex (applied biosystems) with SYBR Green PCR Master Mix (#4309155 applied biosystems) and primers; hCyp3-F: TTGGCATGAGGTTTGCTCTC; hCyp3-R: ACAACGGGTTTTTCTGGTTG; Pparg-F: AGATTCTCCTGTTGACCCAGAG; Pparg-R: AGCTGATTCCGAAGTTGGTG; Fasn-F: CTGCTGTTGGAAGTCAGCTATG; Fasn-R: ATGCCTCTGAACCACTCACAC; Actin-F: CACAGCTTCTTTGCAGCTCCT; Actin-R: CAGCAGTGCAATGTTAAAAGG; qRT-PCR for miRNAs was conducted as previously described [
      <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>] and primers designed with miRprimer2 [
      <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>]. miR-7f-5p-F: CGCAGTGAGGTAGTAGATTG; miR-7f-5p-R: CAGGTCCAGTTTTTTTTTTTTTTTAAC; miR-130a-3p-F: CAGCAGTGCAATGTTAAAAGG; miR-130a-3p-R: CAGGTCCAGTTTTTTTTTTTTTTTATG; miR-33-5p-F: CGCAGGTGCATTGTAGT; miR-33-5p-F:GTCCAGTTTTTTTTTTTTTTTGCAAT; miR-10b-5p-F CAGTACCCTGTAGAACCGA; miR-10b-5p-R:GGTCCAGTTTTTTTTTTTTTTTCAG; Pull-down of 5EU labeled RNA was done using the Click-it Nascent RNA Capture Kit (C10365, Thermo Fisher). 10µg total / small RNA was used as input from tissues, 200ng plasma RNA was used as input for RNA pull-down from blood. RNA was used as template for library construction using the CATS mRNA/small RNA kit (C05010043 and C05010040, Diagenode) with slight modifications to protocol i.e.; Poly-A selection and RNA fragmentation were performed before biotinylation and pull-down of 5EU. 14 cycles of amplification for RNA from tissues, 20 for RNA from blood. 10ng of RNA was used for Input. For MS analysis, 10µg long / short RNA was used. RNA was digested as described in Sue et. al. [
      <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>].
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>UPLC-UHR-ToF-MS analysis</h3>
     <p id="p-45">Mass spectrometric analysis of nucleotides from RNA was performed on a Waters Acquity UPLC (Waters, Eschborn, Germany) coupled to a Bruker maXis UHR-ToF-MS (Bruker Daltonic, Bremen, Germany). Separation was performed on a Thermo Hypersil Gold column (150 x 1.0 mm, 3 µm, 25003-151030, Thermo Fisher) using a multistep gradient with 100% water and 100% ACN, both with 0.1% formic acid. Gradient conditions were as followed: 0-6 min 0% B, 6-7.65 min linear increase to 1% B, 7.65 to 10 min linear increase to 6% B, 10 to 12 min linear increase to 50% B, 12 to 14 min linear increase to 75% B, 14 to 17 min isocratic hold of 75% B, 17 to 17.5 min return to initial conditions. Column temperature was 36°C and flow rate was set to 0.09 ml/min. Before each run the column was re-equilibrated for 3 minutes with starting conditions. High mass accuracy was achieved by infusion of 1:4 diluted ESI low concentration tune mix (Agilent Technologies, Waldbronn, Germany) at the start of each chromatographic run. Each analysis was internally recalibrated using the tune mix peak at the beginning of the chromatogram using a custom VB script within Bruker DataAnalysis 4.0 (Bruker Daltonic, Bremen, Germany). Quantitative analysis was performed in Bruker QuantAnalysis 4.0 (Bruker Daltonic, Bremen, Germany). High Resolution-Extracted Ion Chromatograms (HR-EICs) were created around each precursor mass +/- 0.005 Da. Chromatograms were smoothed and peak areas were used for quantification. In case of 5EU additional quantification was performed on a validated in-source fragment [M-132+H]
      <sup>+</sup>.
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Bioinformatic analysis</h3>
     <p id="p-46">RNA Libraries were sequenced on an Illumina HiSeq 2500 instrument (
      <strong>IGA Technology Services Srl, Italy)</strong> at 75bp single-ended. Adaptors were trimmed in accordance with the CATS sequencing kit manual. Reads were aligned to the mouse mm10 genome using the STAR aligner [
      <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>] and a reference transcript gtf file from ensamble modified to contain tRNA transcripts as annotated by GtRNAdb [
      <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>].
     </p>
     <p id="p-47">For the detection of differentially regulated genes between HFD and LFD the Deseq2 package was used [
      <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a>]. Pull-down enrichment analysis was conducted using the NOISeq package. Transcripts with expression values smaller than a cpm of 1 and a coefficient of variation greater than 300 were filtered out prior to tmm normalization and enrichment analysis. GO and gene set enrichment was calculated using the Enrichr tool [
      <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>].
     </p>
    </div>
   </div>
   <div class="section COI-statement" id="sec-20">
    <h2 class="">Conflict of interest</h2>
    <p id="p-48">The authors declare no conflict of interest.</p>
   </div>
   <div class="section" id="sec-21">
    <h2 class="">Author contributions</h2>
    <p id="p-50">Conceptualization D.J.; Methodology D.J. and W.M.; Investigation D.J., L.M., G.R., A.T., S.F. and W.M.; Writing – Original Draft D.J., W.M., and T.R.; Writing – Review &amp;; Editing D.J., H.A.M, W.M. and T.R.; Funding Acquisition H.A.M and T.R.; Resources; H.A.M, W.M. and T.R.; Supervision T.R.</p>
   </div>
   <div class="section" id="sec-23">
    <h2 class="">Supplementary Tables</h2>
    <p id="p-54">
     <strong>Supplementary table 1.</strong> Identification and separation of various modified and unmodified transcripts in LC-MS.
    </p>
    <p id="p-55">
     <strong>Supplementary table 2.</strong> Noiseq-bio analysis for the detection of enriched and depleted transcripts following pull-down of labelled poly-A RNA in liver and kidney.
    </p>
    <p id="p-56">
     <strong>Supplementary table 3.</strong> GO and gene set enrichment analysis for kidney pulled-down and depleted poly-A RNA.
    </p>
    <p id="p-57">
     <strong>Supplementary table 4.</strong> GO and gene set enrichment analysis for liver pulled-down and depleted poly-A RNA.
    </p>
    <p id="p-58">
     <strong>Supplementary table 5.</strong> Differential expression analysis using Deseq2, for the detection of dietary induced DEG in liver, input and pull-down poly-A RNA libraries.
    </p>
    <p id="p-59">
     <strong>Supplementary table 6.</strong> Differential expression analysis using Deseq2, for the detection of dietary induced DEG in kidney, input and pull-down poly-A RNA libraries.
    </p>
    <p id="p-60">
     <strong>Supplementary table 7.</strong> GO and gene set enrichment analysis on dietary induced differentially expressed genes in liver and kidney pull-down or depleted poly-A RNA.
    </p>
    <p id="p-61">
     <strong>Supplementary table 8.</strong> Noiseqbio differential pull-down analysis for liver, plasma and VsWAT under HFD regime. TMM normalized mean expression values.
    </p>
    <p id="p-62">
     <strong>Supplementary table 9.</strong> Noiseqbio differential pull-down analysis for liver, plasma and VsWAT under LFD regime. TMM normalized mean expression values.
    </p>
    <p id="p-63">
     <strong>Supplementary table 10.</strong> SuperExacttest package output. Overlaps between different sets and calculated P.V.
    </p>
    <p id="p-64">
     <strong>Supplementary table 11</strong> GO and gene set enrichment analysis on plasma pulled-down protein coding genes.
    </p>
    <p id="p-65">
     <strong>Supplementary table 12</strong> Differential expression analysis using Deseq2 package, for the detection of dietary induced DEG in total VsWAT RNA.
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-49">We would like to thank Dr. Julia Calzada-Wack, Jacqueline Mueller and Marion Fisch for their kind assistance with tissue processing and sectioning. We thank Dr. Anja Zeigerer for access to and assistance with the confocal microscope. This work has been supported by the German Diabetes Research Center (DZD NEXT Grant 2019) to T.R., and by the ERC Recognition Award from the Helmholtz Research Center Munich to T.R. The authors thank the Helmholtz Association and the German Diabetes Research Center for funding the positions of G.R., D.J., L.M., S.F., T.A. and T.R.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-1">
      <p id="p-3">This revised version includes fig.6 which was mistakenly missing from the original submission.</p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">Reference</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Murillo</span>, 
          <span class="cit-name-given-names">O.D.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">exRNA Atlas Analysis Reveals Distinct Extracellular RNA Cargo Types and Their Carriers Present across Human Biofluids</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-vol">177</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">463</span>–
         <span class="cit-lpage">477 e15</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yeri</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Total Extracellular Small RNA Profiles from Plasma, Saliva, and Urine of Healthy Subjects</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">7</span>: p. 
         <span class="cit-fpage">44061</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D44061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.3" data-doi="10.1038/ncb3399">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dror</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Melanoma miRNA trafficking controls tumour primary niche formation</span>. 
         <abbr class="cit-jnl-abbrev">Nat Cell Biol</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">18</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">1006</span>–
         <span class="cit-lpage">17</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCell%2BBiol%26rft.volume%253D18%26rft.spage%253D1006%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb3399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncb3399&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.4" data-doi="10.1073/pnas.1808855115">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Castano</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Obesity-associated exosomal miRNAs modulate glucose and lipid metabolism in mice</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">115</span>(
         <span class="cit-issue">48</span>): p. 
         <span class="cit-fpage">12158</span>–
         <span class="cit-lpage">12163</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1808855115%26rft_id%253Dinfo%253Apmid%252F30429322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzQ4LzEyMTU4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wortzel</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Exosome-Mediated Metastasis: Communication from a Distance</span>. 
         <abbr class="cit-jnl-abbrev">Dev Cell</abbr>, 
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-vol">49</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">347</span>–
         <span class="cit-lpage">360</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDev%2BCell%26rft.volume%253D49%26rft.spage%253D347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schwarzenbach</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Clinical relevance of circulating cell-free microRNAs in cancer</span>. 
         <abbr class="cit-jnl-abbrev">Nature Reviews Clinical Oncology</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">11</span>: p. 
         <span class="cit-fpage">145</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BClinical%2BOncology%26rft.volume%253D11%26rft.spage%253D145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.7" data-doi="10.1371/journal.pone.0003148">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gilad</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Serum microRNAs are promising novel biomarkers</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">3</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">e3148</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGilad%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253De3148%26rft.epage%253De3148%26rft.atitle%253DSerum%2BmicroRNAs%2Bare%2Bpromising%2Bnovel%2Bbiomarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003148%26rft_id%253Dinfo%253Apmid%252F18773077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003148&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18773077&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.8" data-doi="10.1073/pnas.0914843107">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pegtel</span>, 
          <span class="cit-name-given-names">D.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Functional delivery of viral miRNAs via exosomes</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">107</span>(
         <span class="cit-issue">14</span>): p. 
         <span class="cit-fpage">6328</span>–
         <span class="cit-lpage">33</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0914843107%26rft_id%253Dinfo%253Apmid%252F20304794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzE0LzYzMjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNi8xNC82NzAzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.9" data-doi="10.1038/ncb2210">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vickers</span>, 
          <span class="cit-name-given-names">K.C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">MicroRNAs are transported in plasma and delivered to recipient cells by high-density lipoproteins</span>. 
         <abbr class="cit-jnl-abbrev">Nat Cell Biol</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">13</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">423</span>–
         <span class="cit-lpage">33</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DVickers%26rft.auinit1%253DK.%2BC.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D423%26rft.epage%253D433%26rft.atitle%253DMicroRNAs%2Bare%2Btransported%2Bin%2Bplasma%2Band%2Bdelivered%2Bto%2Brecipient%2Bcells%2Bby%2Bhigh-density%2Blipoproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2210%26rft_id%253Dinfo%253Apmid%252F21423178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncb2210&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21423178&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000289070900014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-670398v2.10" data-doi="10.1016/j.cell.2018.03.024">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pastuzyn</span>, 
          <span class="cit-name-given-names">E.D.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The Neuronal Gene Arc Encodes a Repurposed Retrotransposon Gag Protein that Mediates Intercellular RNA Transfer</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">172</span>(
         <span class="cit-issue">1-2</span>): p. 
         <span class="cit-fpage">275</span>–
         <span class="cit-lpage">288 e18</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D172%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.03.024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.03.024&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.11" data-doi="10.1074/jbc.M110.107821">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kosaka</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Secretory mechanisms and intercellular transfer of microRNAs in living cells</span>. 
         <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">285</span>(
         <span class="cit-issue">23</span>): p. 
         <span class="cit-fpage">17442</span>–
         <span class="cit-lpage">52</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.107821%26rft_id%253Dinfo%253Apmid%252F20353945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMjMvMTc0NDIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNi8xNC82NzAzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.12" data-doi="10.1038/nature21365">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thomou</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Adipose-derived circulating miRNAs regulate gene expression in other tissues</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">542</span>(
         <span class="cit-issue">7642</span>): p. 
         <span class="cit-fpage">450</span>–
         <span class="cit-lpage">455</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D542%26rft.spage%253D450%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature21365%26rft_id%253Dinfo%253Apmid%252F28199304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature21365&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28199304&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sharma</span>, 
          <span class="cit-name-given-names">U.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Small RNAs Are Trafficked from the Epididymis to Developing Mammalian Sperm</span>. 
         <abbr class="cit-jnl-abbrev">Dev Cell</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">46</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">481</span>–
         <span class="cit-lpage">494 e6</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDev%2BCell%26rft.volume%253D46%26rft.spage%253D481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.14" data-doi="10.1101/gad.205278.112">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gay</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Mouse TU tagging: a chemical/genetic intersectional method for purifying cell type-specific nascent RNA</span>. 
         <abbr class="cit-jnl-abbrev">Genes Dev</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">27</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">98</span>–
         <span class="cit-lpage">115</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGay%26rft.auinit1%253DL.%26rft.volume%253D27%26rft.issue%253D1%26rft.spage%253D98%26rft.epage%253D115%26rft.atitle%253DMouse%2BTU%2Btagging%253A%2Ba%2Bchemical%252Fgenetic%2Bintersectional%2Bmethod%2Bfor%2Bpurifying%2Bcell%2Btype-specific%2Bnascent%2BRNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.205278.112%26rft_id%253Dinfo%253Apmid%252F23307870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMjcvMS85OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE0LzY3MDM5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.15" data-doi="10.1038/nmeth.4435">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Herzog</span>, 
          <span class="cit-name-given-names">V.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Thiol-linked alkylation of RNA to assess expression dynamics</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">14</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">1198</span>–
         <span class="cit-lpage">1204</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D1198%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4435%26rft_id%253Dinfo%253Apmid%252F28945705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.4435&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28945705&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chatzi</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Transcriptional Profiling of Newly Generated Dentate Granule Cells Using TU Tagging Reveals Pattern Shifts in Gene Expression during Circuit Integration</span>. 
         <abbr class="cit-jnl-abbrev">eNeuro</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">3</span>(
         <span class="cit-issue">1</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.17" data-doi="10.1038/nmeth.1329">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">TU-tagging: cell type-specific RNA isolation from intact complex tissues</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">6</span>): p. 
         <span class="cit-fpage">439</span>–
         <span class="cit-lpage">41</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMiller%26rft.auinit1%253DM.%2BR.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253D439%26rft.epage%253D441%26rft.atitle%253DTU-tagging%253A%2Bcell%2Btype-specific%2BRNA%2Bisolation%2Bfrom%2Bintact%2Bcomplex%2Btissues.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1329%26rft_id%253Dinfo%253Apmid%252F19430475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.1329&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19430475&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000266493300015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ghosh</span>, 
          <span class="cit-name-given-names">A.C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">UPRT, a suicide-gene therapy candidate in higher eukaryotes, is required for Drosophila larval growth and normal adult lifespan</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">5</span>: p. 
         <span class="cit-fpage">13176</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D13176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Maquat</span>, 
          <span class="cit-name-given-names">L.E.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Kiledjian</span>
         </span>, 
         <span class="cit-article-title">RNA turnover in eukaryotes: nucleases, pathways and analysis of mRNA decay. Preface</span>. 
         <abbr class="cit-jnl-abbrev">Methods Enzymol</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">448</span>: p. 
         <span class="cit-fpage">xxi</span>–
         <span class="cit-lpage">xxii</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol%26rft.volume%253D448%26rft.spage%253Dxxi%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.20" data-doi="10.1093/nar/gkx551">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hida</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">EC-tagging allows cell type-specific RNA analysis</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">45</span>(
         <span class="cit-issue">15</span>): p. 
         <span class="cit-fpage">e138</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D45%26rft.spage%253De138%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkx551&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.21" data-doi="10.1016/j.tig.2017.12.008">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">A.J.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.D.</span>
          <span class="cit-name-surname">Palmiter</span>
         </span>, 
         <span class="cit-article-title">Detecting and Avoiding Problems When Using the Cre-lox System</span>. 
         <abbr class="cit-jnl-abbrev">Trends Genet</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">34</span>(
         <span class="cit-issue">5</span>): p. 
         <span class="cit-fpage">333</span>–
         <span class="cit-lpage">340</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet%26rft.volume%253D34%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.12.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.12.008&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.22" data-doi="10.1038/nmeth.4610">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Alberti</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Cell-type specific sequencing of microRNAs from complex animal tissues</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">15</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">283</span>–
         <span class="cit-lpage">289</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D15%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.4610&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.23" data-doi="10.1073/pnas.1005191107">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Marquart</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">miR-33 links SREBP-2 induction to repression of sterol transporters</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">107</span>(
         <span class="cit-issue">27</span>): p. 
         <span class="cit-fpage">12228</span>–
         <span class="cit-lpage">32</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1005191107%26rft_id%253Dinfo%253Apmid%252F20566875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzI3LzEyMjI4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.24" data-doi="10.1126/science.1189862">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rayner</span>, 
          <span class="cit-name-given-names">K.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">MiR-33 contributes to the regulation of cholesterol homeostasis</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">328</span>(
         <span class="cit-issue">5985</span>): p. 
         <span class="cit-fpage">1570</span>–
         <span class="cit-lpage">3</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1189862%26rft_id%253Dinfo%253Apmid%252F20466885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjgvNTk4NS8xNTcwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.25" data-doi="10.1111/j.1440-1746.2009.05949.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Effect of miRNA-10b in regulating cellular steatosis level by targeting PPAR-alpha expression, a novel mechanism for the pathogenesis of NAFLD</span>. 
         <abbr class="cit-jnl-abbrev">J Gastroenterol Hepatol</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">156</span>–
         <span class="cit-lpage">63</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgastroenterology%2Band%2Bhepatology%26rft.stitle%253DJ%2BGastroenterol%2BHepatol%26rft.aulast%253DZheng%26rft.auinit1%253DL.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D163%26rft.atitle%253DEffect%2Bof%2BmiRNA-10b%2Bin%2Bregulating%2Bcellular%2Bsteatosis%2Blevel%2Bby%2Btargeting%2BPPAR-alpha%2Bexpression%252C%2Ba%2Bnovel%2Bmechanism%2Bfor%2Bthe%2Bpathogenesis%2Bof%2BNAFLD.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1440-1746.2009.05949.x%26rft_id%253Dinfo%253Apmid%252F19780876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1440-1746.2009.05949.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19780876&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.26" data-doi="10.1128/MCB.00894-10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">E.K.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">miR-130 suppresses adipogenesis by inhibiting peroxisome proliferator-activated receptor gamma expression</span>. 
         <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">31</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">626</span>–
         <span class="cit-lpage">38</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BCell%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00894-10%26rft_id%253Dinfo%253Apmid%252F21135128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjMxLzQvNjI2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.27" data-doi="10.1073/pnas.0808480105">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jao</span>, 
          <span class="cit-name-given-names">C.Y.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Salic</span>
         </span>, 
         <span class="cit-article-title">Exploring RNA transcription and turnover in vivo by using click chemistry</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">105</span>(
         <span class="cit-issue">41</span>): p. 
         <span class="cit-fpage">15779</span>–
         <span class="cit-lpage">84</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808480105%26rft_id%253Dinfo%253Apmid%252F18840688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQxLzE1Nzc5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-670398v2.28" data-doi="10.1021/bi9007726">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Best</span>, 
          <span class="cit-name-given-names">M.D.</span>
         </span>, 
         <span class="cit-article-title">Click chemistry and bioorthogonal reactions: unprecedented selectivity in the labeling of biological molecules</span>. 
         <abbr class="cit-jnl-abbrev">Biochemistry</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">48</span>(
         <span class="cit-issue">28</span>): p. 
         <span class="cit-fpage">6571</span>–
         <span class="cit-lpage">84</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DBest%26rft.auinit1%253DM.%2BD.%26rft.volume%253D48%26rft.issue%253D28%26rft.spage%253D6571%26rft.epage%253D6584%26rft.atitle%253DClick%2Bchemistry%2Band%2Bbioorthogonal%2Breactions%253A%2Bunprecedented%2Bselectivity%2Bin%2Bthe%2Blabeling%2Bof%2Bbiological%2Bmolecules.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi9007726%26rft_id%253Dinfo%253Apmid%252F19485420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/bi9007726&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19485420&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268137800003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.29" data-doi="10.1128/JVI.07207-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hagemeijer</span>, 
          <span class="cit-name-given-names">M.C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Visualizing coronavirus RNA synthesis in time by using click chemistry</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">86</span>(
         <span class="cit-issue">10</span>): p. 
         <span class="cit-fpage">5808</span>–
         <span class="cit-lpage">16</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.07207-11%26rft_id%253Dinfo%253Apmid%252F22438542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xMC81ODA4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.30" data-doi="10.1021/ol0610946">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gierlich</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Click chemistry as a reliable method for the high-density postsynthetic functionalization of alkyne-modified DNA</span>. 
         <abbr class="cit-jnl-abbrev">Org Lett</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">17</span>): p. 
         <span class="cit-fpage">3639</span>–
         <span class="cit-lpage">42</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DOrganic%2Bletters%26rft.stitle%253DOrg%2BLett%26rft.aulast%253DGierlich%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D17%26rft.spage%253D3639%26rft.epage%253D3642%26rft.atitle%253DClick%2Bchemistry%2Bas%2Ba%2Breliable%2Bmethod%2Bfor%2Bthe%2Bhigh-density%2Bpostsynthetic%2Bfunctionalization%2Bof%2Balkyne-modified%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fol0610946%26rft_id%253Dinfo%253Apmid%252F16898780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ol0610946&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16898780&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000239655500005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Meyer</span>, 
          <span class="cit-name-given-names">J.P.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Click Chemistry and Radiochemistry: The First 10 Years</span>. 
         <abbr class="cit-jnl-abbrev">Bioconjug Chem</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">27</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">2791</span>–
         <span class="cit-lpage">2807</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioconjug%2BChem%26rft.volume%253D27%26rft.spage%253D2791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.32" data-doi="10.1021/cr5002035">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pradere</span>, 
          <span class="cit-name-given-names">U.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Synthesis of nucleoside phosphate and phosphonate prodrugs</span>. 
         <abbr class="cit-jnl-abbrev">Chem Rev</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">114</span>(
         <span class="cit-issue">18</span>): p. 
         <span class="cit-fpage">9154</span>–
         <span class="cit-lpage">218</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChem%2BRev%26rft.volume%253D114%26rft.spage%253D9154%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fcr5002035%26rft_id%253Dinfo%253Apmid%252F25144792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/cr5002035&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25144792&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000342328500009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.33" data-doi="10.1021/ja031818y">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Erion</span>, 
          <span class="cit-name-given-names">M.D.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Design, synthesis, and characterization of a series of cytochrome P(450) 3A-activated prodrugs (HepDirect prodrugs) useful for targeting phosph(on)ate-based drugs to the liver</span>. 
         <abbr class="cit-jnl-abbrev">J Am Chem Soc</abbr>, 
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-vol">126</span>(
         <span class="cit-issue">16</span>): p. 
         <span class="cit-fpage">5154</span>–
         <span class="cit-lpage">63</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253DErion%26rft.auinit1%253DM.%2BD.%26rft.volume%253D126%26rft.issue%253D16%26rft.spage%253D5154%26rft.epage%253D5163%26rft.atitle%253DDesign%252C%2Bsynthesis%252C%2Band%2Bcharacterization%2Bof%2Ba%2Bseries%2Bof%2Bcytochrome%2BP%2528450%2529%2B3A-activated%2Bprodrugs%2B%2528HepDirect%2Bprodrugs%2529%2Buseful%2Bfor%2Btargeting%2Bphosph%2528on%2529ate-based%2Bdrugs%2Bto%2Bthe%2Bliver.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja031818y%26rft_id%253Dinfo%253Apmid%252F15099098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ja031818y&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15099098&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220957900043&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.34" data-doi="10.1021/jm0607449">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Boyer</span>, 
          <span class="cit-name-given-names">S.H.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Synthesis and characterization of a novel liver-targeted prodrug of cytosine-1-beta-D-arabinofuranoside monophosphate for the treatment of hepatocellular carcinoma</span>. 
         <abbr class="cit-jnl-abbrev">J Med Chem</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">49</span>(
         <span class="cit-issue">26</span>): p. 
         <span class="cit-fpage">7711</span>–
         <span class="cit-lpage">20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DBoyer%26rft.auinit1%253DS.%2BH.%26rft.volume%253D49%26rft.issue%253D26%26rft.spage%253D7711%26rft.epage%253D7720%26rft.atitle%253DSynthesis%2Band%2Bcharacterization%2Bof%2Ba%2Bnovel%2Bliver-targeted%2Bprodrug%2Bof%2Bcytosine-1-beta-D-arabinofuranoside%2Bmonophosphate%2Bfor%2Bthe%2Btreatment%2Bof%2Bhepatocellular%2Bcarcinoma.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm0607449%26rft_id%253Dinfo%253Apmid%252F17181153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm0607449&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17181153&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-670398v2.35" data-doi="10.1073/pnas.0702759104">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Erion</span>, 
          <span class="cit-name-given-names">M.D.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Targeting thyroid hormone receptor-beta agonists to the liver reduces cholesterol and triglycerides and improves the therapeutic index</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-vol">104</span>(
         <span class="cit-issue">39</span>): p. 
         <span class="cit-fpage">15490</span>–
         <span class="cit-lpage">5</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0702759104%26rft_id%253Dinfo%253Apmid%252F17878314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzM5LzE1NDkwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.36" data-doi="10.1021/jm7012216">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Reddy</span>, 
          <span class="cit-name-given-names">K.R.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Pradefovir: a prodrug that targets adefovir to the liver for the treatment of hepatitis B</span>. 
         <abbr class="cit-jnl-abbrev">J Med Chem</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">51</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">666</span>–
         <span class="cit-lpage">76</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DReddy%26rft.auinit1%253DK.%2BR.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D666%26rft.epage%253D676%26rft.atitle%253DPradefovir%253A%2Ba%2Bprodrug%2Bthat%2Btargets%2Badefovir%2Bto%2Bthe%2Bliver%2Bfor%2Bthe%2Btreatment%2Bof%2Bhepatitis%2BB.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm7012216%26rft_id%253Dinfo%253Apmid%252F18173234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm7012216&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18173234&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.37" data-doi="10.1124/dmd.105.004721">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">van Herwaarden</span>, 
          <span class="cit-name-given-names">A.E.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Midazolam and cyclosporin a metabolism in transgenic mice with liver-specific expression of human CYP3A4</span>. 
         <abbr class="cit-jnl-abbrev">Drug Metab Dispos</abbr>, 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-vol">33</span>(
         <span class="cit-issue">7</span>): p. 
         <span class="cit-fpage">892</span>–
         <span class="cit-lpage">5</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDrug%2BMetab%2BDispos%26rft_id%253Dinfo%253Adoi%252F10.1124%252Fdmd.105.004721%26rft_id%253Dinfo%253Apmid%252F15845749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZG1kIjtzOjU6InJlc2lkIjtzOjg6IjMzLzcvODkyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.38" data-doi="10.1172/JCI33435">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">van Herwaarden</span>, 
          <span class="cit-name-given-names">A.E.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Knockout of cytochrome P450 3A yields new mouse models for understanding xenobiotic metabolism</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Invest</abbr>, 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-vol">117</span>(
         <span class="cit-issue">11</span>): p. 
         <span class="cit-fpage">3583</span>–
         <span class="cit-lpage">92</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.aulast%253Dvan%2BHerwaarden%26rft.auinit1%253DA.%2BE.%26rft.volume%253D117%26rft.issue%253D11%26rft.spage%253D3583%26rft.epage%253D3592%26rft.atitle%253DKnockout%2Bof%2Bcytochrome%2BP450%2B3A%2Byields%2Bnew%2Bmouse%2Bmodels%2Bfor%2Bunderstanding%2Bxenobiotic%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI33435%26rft_id%253Dinfo%253Apmid%252F17975676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1172/JCI33435&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17975676&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000250676000050&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Simonet</span>, 
          <span class="cit-name-given-names">W.S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A far-downstream hepatocyte-specific control region directs expression of the linked human apolipoprotein E and C-I genes in transgenic mice</span>. 
         <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, 
         <span class="cit-pub-date">1993</span>. 
         <span class="cit-vol">268</span>(
         <span class="cit-issue">11</span>): p. 
         <span class="cit-fpage">8221</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DSimonet%26rft.auinit1%253DW.%2BS.%26rft.volume%253D268%26rft.issue%253D11%26rft.spage%253D8221%26rft.epage%253D8229%26rft.atitle%253DA%2Bfar-downstream%2Bhepatocyte-specific%2Bcontrol%2Bregion%2Bdirects%2Bexpression%2Bof%2Bthe%2Blinked%2Bhuman%2Bapolipoprotein%2BE%2Band%2BC-I%2Bgenes%2Bin%2Btransgenic%2Bmice%26rft_id%253Dinfo%253Apmid%252F7681840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNjgvMTEvODIyMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE0LzY3MDM5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.40" data-doi="10.1124/dmd.32.1.105">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Stresser</span>, 
          <span class="cit-name-given-names">D.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Highly selective inhibition of human CYP3Aa in vitro by azamulin and evidence that inhibition is irreversible</span>. 
         <abbr class="cit-jnl-abbrev">Drug Metab Dispos</abbr>, 
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-vol">32</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">105</span>–
         <span class="cit-lpage">12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDrug%2BMetabolism%2Band%2BDisposition%26rft.stitle%253DDrug%2BMetab.%2BDispos.%26rft.aulast%253DStresser%26rft.auinit1%253DD.%2BM.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D112%26rft.atitle%253DHIGHLY%2BSELECTIVE%2BINHIBITION%2BOF%2BHUMAN%2BCYP3A%2BIN%2BVITRO%2BBY%2BAZAMULIN%2BAND%2BEVIDENCE%2BTHAT%2BINHIBITION%2BIS%2BIRREVERSIBLE%26rft_id%253Dinfo%253Adoi%252F10.1124%252Fdmd.32.1.105%26rft_id%253Dinfo%253Apmid%252F14709627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZG1kIjtzOjU6InJlc2lkIjtzOjg6IjMyLzEvMTA1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTQvNjcwMzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.41" data-doi="10.1038/nprot.2014.047">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Su</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Quantitative analysis of ribonucleoside modifications in tRNA by HPLC-coupled mass spectrometry</span>. 
         <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">828</span>–
         <span class="cit-lpage">41</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D9%26rft.spage%253D828%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2014.047%26rft_id%253Dinfo%253Apmid%252F24625781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nprot.2014.047&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24625781&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.42" data-doi="10.1371/journal.pone.0106159">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Williams</span>, 
          <span class="cit-name-given-names">L.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The development of diet-induced obesity and glucose intolerance in C57BL/6 mice on a high-fat diet consists of distinct phases</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">8</span>): p. 
         <span class="cit-fpage">e106159</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De106159%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0106159%26rft_id%253Dinfo%253Apmid%252F25170916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0106159&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25170916&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.43" data-doi="10.1093/bioinformatics/bts635">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dobin</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">29</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">15</span>–
         <span class="cit-lpage">21</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23104886&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000312654600003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">44.</span>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.44" data-doi="10.1093/nar/gkv711">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tarazona</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Data quality aware analysis of differential expression in RNA-seq with NOISeq R/Bioc package</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">43</span>(
         <span class="cit-issue">21</span>): p. 
         <span class="cit-fpage">e140</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv711%26rft_id%253Dinfo%253Apmid%252F26184878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkv711&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26184878&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">45.</span>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.45" data-doi="10.1186/1471-2105-14-128">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">E.Y.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">14</span>: p. 
         <span class="cit-fpage">128</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D128%26rft.atitle%253DENRICHR%253A%2BINTERACTIVE%2BAND%2BCOLLABORATIVE%2BHTML5%2BGENE%2BLIST%2BENRICHMENT%2BANALYSIS%2BTOOL%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-128%26rft_id%253Dinfo%253Apmid%252F23586463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-128&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23586463&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">46.</span>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.46" data-doi="10.1038/ki.1988.183">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Coudrier</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Kerjaschki</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Louvard</span>
         </span>, 
         <span class="cit-article-title">Cytoskeleton organization and submembranous interactions in intestinal and renal brush borders</span>. 
         <abbr class="cit-jnl-abbrev">Kidney Int</abbr>, 
         <span class="cit-pub-date">1988</span>. 
         <span class="cit-vol">34</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">309</span>–
         <span class="cit-lpage">20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DKidney%2Binternational%26rft.stitle%253DKidney%2BInt%26rft.aulast%253DCoudrier%26rft.auinit1%253DE.%26rft.volume%253D34%26rft.issue%253D3%26rft.spage%253D309%26rft.epage%253D320%26rft.atitle%253DCytoskeleton%2Borganization%2Band%2Bsubmembranous%2Binteractions%2Bin%2Bintestinal%2Band%2Brenal%2Bbrush%2Bborders.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fki.1988.183%26rft_id%253Dinfo%253Apmid%252F3050247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ki.1988.183&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=3050247&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1988P732800001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">47.</span>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.47" data-doi="10.1002/eji.1830250606">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Finger</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.C.</span>
          <span class="cit-name-surname">Gutierrez-Ramos</span>
         </span>, 
         <span class="cit-article-title">Expression of CD34 in endothelial cells, hematopoietic progenitors and nervous cells in fetal and adult mouse tissues</span>. 
         <abbr class="cit-jnl-abbrev">Eur J Immunol</abbr>, 
         <span class="cit-pub-date">1995</span>. 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">6</span>): p. 
         <span class="cit-fpage">1508</span>–
         <span class="cit-lpage">16</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bimmunology%26rft.stitle%253DEur%2BJ%2BImmunol%26rft.aulast%253DLin%26rft.auinit1%253DG.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D1508%26rft.epage%253D1516%26rft.atitle%253DExpression%2Bof%2BCD34%2Bin%2Bendothelial%2Bcells%252C%2Bhematopoietic%2Bprogenitors%2Band%2Bnervous%2Bcells%2Bin%2Bfetal%2Band%2Badult%2Bmouse%2Btissues.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Feji.1830250606%26rft_id%253Dinfo%253Apmid%252F7542195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/eji.1830250606&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=7542195&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1995RG46500005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">48.</span>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.48">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fina</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Expression of the CD34 gene in vascular endothelial cells</span>. 
         <abbr class="cit-jnl-abbrev">Blood</abbr>, 
         <span class="cit-pub-date">1990</span>. 
         <span class="cit-vol">75</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">2417</span>–
         <span class="cit-lpage">26</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBlood%26rft.stitle%253DBlood%26rft.aulast%253DFina%26rft.auinit1%253DL.%26rft.volume%253D75%26rft.issue%253D12%26rft.spage%253D2417%26rft.epage%253D2426%26rft.atitle%253DExpression%2Bof%2Bthe%2BCD34%2Bgene%2Bin%2Bvascular%2Bendothelial%2Bcells%26rft_id%253Dinfo%253Apmid%252F1693532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiNzUvMTIvMjQxNyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE0LzY3MDM5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">49.</span>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.49" data-doi="10.1111/j.1751-2980.2011.00554.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.K.</span>
          <span class="cit-name-surname">Qiu</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Ma</span>
         </span>, 
         <span class="cit-article-title">Liver X receptors bridge hepatic lipid metabolism and inflammation</span>. 
         <abbr class="cit-jnl-abbrev">J Dig Dis</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">13</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">69</span>–
         <span class="cit-lpage">74</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DJ%2BDig%2BDis%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D69%26rft.epage%253D74%26rft.atitle%253DLiver%2BX%2Breceptors%2Bbridge%2Bhepatic%2Blipid%2Bmetabolism%2Band%2Binflammation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1751-2980.2011.00554.x%26rft_id%253Dinfo%253Apmid%252F22257474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1751-2980.2011.00554.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22257474&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">50.</span>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.50" data-doi="10.1172/JCI6223">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kersten</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Peroxisome proliferator-activated receptor alpha mediates the adaptive response to fasting</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Invest</abbr>, 
         <span class="cit-pub-date">1999</span>. 
         <span class="cit-vol">103</span>(
         <span class="cit-issue">11</span>): p. 
         <span class="cit-fpage">1489</span>–
         <span class="cit-lpage">98</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.aulast%253DKersten%26rft.auinit1%253DS.%26rft.volume%253D103%26rft.issue%253D11%26rft.spage%253D1489%26rft.epage%253D1498%26rft.atitle%253DPeroxisome%2Bproliferator%2596activated%2Breceptor%2B%257Balpha%257D%2Bmediates%2Bthe%2Badaptive%2Bresponse%2Bto%2Bfasting%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI6223%26rft_id%253Dinfo%253Apmid%252F10359558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1172/JCI6223&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10359558&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000083468100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">51.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-670398v2.51" data-doi="10.4103/2231-4040.90879">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tyagi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The peroxisome proliferator-activated receptor: A family of nuclear receptors role in various diseases</span>. 
         <abbr class="cit-jnl-abbrev">J Adv Pharm Technol Res</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">2</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">236</span>–
         <span class="cit-lpage">40</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BAdv%2BPharm%2BTechnol%2BRes%26rft.volume%253D2%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.4103%252F2231-4040.90879%26rft_id%253Dinfo%253Apmid%252F22247890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.4103/2231-4040.90879&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22247890&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">52.</span>
      <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.52" data-doi="10.1034/j.1600-0676.2000.020003191.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Everett</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Galli</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Crabb</span>
         </span>, 
         <span class="cit-article-title">The role of hepatic peroxisome proliferator-activated receptors (PPARs) in health and disease</span>. 
         <abbr class="cit-jnl-abbrev">Liver</abbr>, 
         <span class="cit-pub-date">2000</span>. 
         <span class="cit-vol">20</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">191</span>–
         <span class="cit-lpage">199</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLiver%26rft.stitle%253DLiver%26rft.aulast%253DEverett%26rft.auinit1%253DL.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D191%26rft.epage%253D199%26rft.atitle%253DThe%2Brole%2Bof%2Bhepatic%2Bperoxisome%2Bproliferator-activated%2Breceptors%2B%2528PPARs%2529%2Bin%2Bhealth%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1600-0676.2000.020003191.x%26rft_id%253Dinfo%253Apmid%252F10902968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1034/j.1600-0676.2000.020003191.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10902968&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000087559000001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">53.</span>
      <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.53" data-doi="10.1126/science.aar2131">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Single-cell transcriptomics of the mouse kidney reveals potential cellular targets of kidney disease</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">360</span>(
         <span class="cit-issue">6390</span>): p. 
         <span class="cit-fpage">758</span>–
         <span class="cit-lpage">763</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aar2131%26rft_id%253Dinfo%253Apmid%252F29622724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjAvNjM5MC83NTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNi8xNC82NzAzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">54.</span>
      <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.54">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">MacParland</span>, 
          <span class="cit-name-given-names">S.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Single cell RNA sequencing of human liver reveals distinct intrahepatic macrophage populations</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">4383</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D4383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">55.</span>
      <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.55" data-doi="10.1186/s13059-014-0550-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Love</span>, 
          <span class="cit-name-given-names">M.I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Huber</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Anders</span>
         </span>, 
         <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">15</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">550</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25516281&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">56.</span>
      <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.56">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Srinivasan</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Small RNA Sequencing across Diverse Biofluids Identifies Optimal Methods for exRNA Isolation</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-vol">177</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">446</span>–
         <span class="cit-lpage">462 e16</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">57.</span>
      <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.57" data-doi="10.1038/srep16923">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Zhao</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-article-title">Efficient Test and Visualization of Multi-Set Intersections</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">5</span>: p. 
         <span class="cit-fpage">16923</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D16923%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep16923%26rft_id%253Dinfo%253Apmid%252F26603754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/srep16923&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26603754&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">58.</span>
      <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.58" data-doi="10.1038/ncomms2071">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Escherichia coli noncoding RNAs can affect gene expression and physiology of Caenorhabditis elegans</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">3</span>: p. 
         <span class="cit-fpage">1073</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D3%26rft.spage%253D1073%26rft.epage%253D1073%26rft.atitle%253DEscherichia%2Bcoli%2Bnoncoding%2BRNAs%2Bcan%2Baffect%2Bgene%2Bexpression%2Band%2Bphysiology%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2071%26rft_id%253Dinfo%253Apmid%252F23011127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms2071&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23011127&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">59.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-670398v2.59" data-doi="10.1126/science.1239705">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Weiberg</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Fungal small RNAs suppress plant immunity by hijacking host RNA interference pathways</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">342</span>(
         <span class="cit-issue">6154</span>): p. 
         <span class="cit-fpage">118</span>–
         <span class="cit-lpage">23</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWeiberg%26rft.auinit1%253DA.%26rft.volume%253D342%26rft.issue%253D6154%26rft.spage%253D118%26rft.epage%253D123%26rft.atitle%253DFungal%2BSmall%2BRNAs%2BSuppress%2BPlant%2BImmunity%2Bby%2BHijacking%2BHost%2BRNA%2BInterference%2BPathways%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1239705%26rft_id%253Dinfo%253Apmid%252F24092744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE1NC8xMTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNi8xNC82NzAzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">60.</span>
      <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.60" data-doi="10.1038/27579">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Timmons</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Fire</span>
         </span>, 
         <span class="cit-article-title">Specific interference by ingested dsRNA</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">1998</span>. 
         <span class="cit-vol">395</span>(
         <span class="cit-issue">6705</span>): p. 
         <span class="cit-fpage">854</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTimmons%26rft.auinit1%253DL.%26rft.volume%253D395%26rft.issue%253D6705%26rft.spage%253D854%26rft.epage%253D854%26rft.atitle%253DSpecific%2Binterference%2Bby%2Bingested%2BdsRNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F27579%26rft_id%253Dinfo%253Apmid%252F9804418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/27579&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9804418&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000076713400039&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">61.</span>
      <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.61">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rivkin</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Inflammation-Induced Expression and Secretion of MicroRNA 122 Leads to Reduced Blood Levels of Kidney-Derived Erythropoietin and Anemia</span>. 
         <abbr class="cit-jnl-abbrev">Gastroenterology</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">151</span>(
         <span class="cit-issue">5</span>): p. 
         <span class="cit-fpage">999</span>–
         <span class="cit-lpage">1010 e3</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGastroenterology%26rft.volume%253D151%26rft.spage%253D999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">62.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-670398v2.62">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chai</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Metabolic Circuit Involving Free Fatty Acids, microRNA 122, and Triglyceride Synthesis in Liver and Muscle Tissues</span>. 
         <abbr class="cit-jnl-abbrev">Gastroenterology</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">153</span>(
         <span class="cit-issue">5</span>): p. 
         <span class="cit-fpage">1404</span>–
         <span class="cit-lpage">1415</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGastroenterology%26rft.volume%253D153%26rft.spage%253D1404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">63.</span>
      <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.63" data-doi="10.1101/gad.1789609">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rechavi</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Cell contact-dependent acquisition of cellular and viral nonautonomously encoded small RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Genes Dev</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">23</span>(
         <span class="cit-issue">16</span>): p. 
         <span class="cit-fpage">1971</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRechavi%26rft.auinit1%253DO.%26rft.volume%253D23%26rft.issue%253D16%26rft.spage%253D1971%26rft.epage%253D1979%26rft.atitle%253DCell%2Bcontact-dependent%2Bacquisition%2Bof%2Bcellular%2Band%2Bviral%2Bnonautonomously%2Bencoded%2Bsmall%2BRNAs%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1789609%26rft_id%253Dinfo%253Apmid%252F19684116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIzLzE2LzE5NzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNi8xNC82NzAzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">64.</span>
      <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.64" data-doi="10.2337/db11-0194">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">Y.S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Inflammation is necessary for long-term but not short-term high-fat diet-induced insulin resistance</span>. 
         <abbr class="cit-jnl-abbrev">Diabetes</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">60</span>(
         <span class="cit-issue">10</span>): p. 
         <span class="cit-fpage">2474</span>–
         <span class="cit-lpage">83</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDiabetes%26rft_id%253Dinfo%253Adoi%252F10.2337%252Fdb11-0194%26rft_id%253Dinfo%253Apmid%252F21911747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZGlhYmV0ZXMiO3M6NToicmVzaWQiO3M6MTA6IjYwLzEwLzI0NzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNi8xNC82NzAzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">65.</span>
      <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.65" data-doi="10.1038/nature05485">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hotamisligil</span>, 
          <span class="cit-name-given-names">G.S.</span>
         </span>, 
         <span class="cit-article-title">Inflammation and metabolic disorders</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">444</span>(
         <span class="cit-issue">7121</span>): p. 
         <span class="cit-fpage">860</span>–
         <span class="cit-lpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHotamisligil%26rft.auinit1%253DG.%2BS.%26rft.volume%253D444%26rft.issue%253D7121%26rft.spage%253D860%26rft.epage%253D867%26rft.atitle%253DInflammation%2Band%2Bmetabolic%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05485%26rft_id%253Dinfo%253Apmid%252F17167474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature05485&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17167474&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000242805400043&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">66.</span>
      <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.66" data-doi="10.1093/nar/gkx1067">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chou</span>, 
          <span class="cit-name-given-names">C.H.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">miRTarBase update 2018: a resource for experimentally validated microRNA-target interactions</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">46</span>(
         <span class="cit-issue">D1</span>): p. 
         <span class="cit-fpage">D296</span>–
         <span class="cit-lpage">D302</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1067%26rft_id%253Dinfo%253Apmid%252F29126174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkx1067&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29126174&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">67.</span>
      <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.67">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Portius</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Sobolewski</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Foti</span>
         </span>, 
         <span class="cit-article-title">MicroRNAs-Dependent Regulation of PPARs in Metabolic Diseases and Cancers</span>. 
         <abbr class="cit-jnl-abbrev">PPAR Res</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">2017</span>: p. 
         <span class="cit-fpage">7058424</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPPAR%2BRes%26rft.volume%253D2017%26rft.spage%253D7058424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">68.</span>
      <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.68" data-doi="10.1038/ncomms3883">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Horie</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">MicroRNA-33 regulates sterol regulatory element-binding protein 1 expression in mice</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">4</span>: p. 
         <span class="cit-fpage">2883</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D4%26rft.spage%253D2883%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3883%26rft_id%253Dinfo%253Apmid%252F24300912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms3883&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24300912&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">69.</span>
      <a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.69" data-doi="10.1111/j.1742-4658.2008.06806.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shimano</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-article-title">SREBPs: physiology and pathophysiology of the SREBP family</span>. 
         <abbr class="cit-jnl-abbrev">FEBS J</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">276</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">616</span>–
         <span class="cit-lpage">21</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DShimano%26rft.auinit1%253DH.%26rft.volume%253D276%26rft.issue%253D3%26rft.spage%253D616%26rft.epage%253D621%26rft.atitle%253DSREBPs%253A%2Bphysiology%2Band%2Bpathophysiology%2Bof%2Bthe%2BSREBP%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2008.06806.x%26rft_id%253Dinfo%253Apmid%252F19143830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2008.06806.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19143830&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">70.</span>
      <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.70" data-doi="10.1038/nrg3861">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kirchner</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Ignatova</span>
         </span>, 
         <span class="cit-article-title">Emerging roles of tRNA in adaptive translation, signalling dynamics and disease</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">16</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">98</span>–
         <span class="cit-lpage">112</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D16%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3861%26rft_id%253Dinfo%253Apmid%252F25534324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrg3861&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25534324&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">71.</span>
      <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.71">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Enache</span>, 
          <span class="cit-name-given-names">L.S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Circulating RNA molecules as biomarkers in liver disease</span>. 
         <abbr class="cit-jnl-abbrev">Int J Mol Sci</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">15</span>(
         <span class="cit-issue">10</span>): p. 
         <span class="cit-fpage">17644</span>–
         <span class="cit-lpage">66</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BMol%2BSci%26rft.volume%253D15%26rft.spage%253D17644%26rft_id%253Dinfo%253Apmid%252F25272224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=25272224&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">72.</span>
      <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.72">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thul</span>, 
          <span class="cit-name-given-names">P.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A subcellular map of the human proteome</span>. 
         <abbr class="cit-jnl-abbrev">Science, 2017</abbr>. 
         <span class="cit-vol">356</span>(
         <span class="cit-pub-date">6340</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">73.</span>
      <a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.73" data-doi="10.1186/1472-6750-11-70">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Balcells</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Cirera</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.K.</span>
          <span class="cit-name-surname">Busk</span>
         </span>, 
         <span class="cit-article-title">Specific and sensitive quantitative RT-PCR of miRNAs with DNA primers</span>. 
         <abbr class="cit-jnl-abbrev">BMC Biotechnol</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">11</span>: p. 
         <span class="cit-fpage">70</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbiotechnology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBiotechnol%26rft.aulast%253DBalcells%26rft.auinit1%253DI.%26rft.volume%253D11%26rft.spage%253D70%26rft.epage%253D70%26rft.atitle%253DSpecific%2Band%2Bsensitive%2Bquantitative%2BRT-PCR%2Bof%2BmiRNAs%2Bwith%2BDNA%2Bprimers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6750-11-70%26rft_id%253Dinfo%253Apmid%252F21702990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1472-6750-11-70&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21702990&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">74.</span>
      <a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.74" data-doi="10.1186/1471-2105-15-29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Busk</span>, 
          <span class="cit-name-given-names">P.K.</span>
         </span>, 
         <span class="cit-article-title">A tool for design of primers for microRNA-specific quantitative RT-qPCR</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">15</span>: p. 
         <span class="cit-fpage">29</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-29%26rft_id%253Dinfo%253Apmid%252F24472427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-29&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24472427&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">75.</span>
      <a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-670398v2.75" data-doi="10.1093/nar/gkv1309">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">P.P.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.M.</span>
          <span class="cit-name-surname">Lowe</span>
         </span>, 
         <span class="cit-article-title">GtRNAdb 2.0: an expanded database of transfer RNA genes identified in complete and draft genomes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">44</span>(
         <span class="cit-issue">D1</span>): p. 
         <span class="cit-fpage">D184</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1309%26rft_id%253Dinfo%253Apmid%252F26673694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkv1309&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26673694&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F14%2F670398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
