<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/10/21/810697.atom"/>
  <meta name="HW.pisa" content="biorxiv;810697v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations"/>
  <meta name="DC.Identifier" content="10.1101/810697"/>
  <meta name="DC.Date" content="2019-10-21"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Phylogenetic methods can use the sampling times of molecular sequence data to calibrate the molecular clock, enabling the estimation of substitution rates and time scales for rapidly evolving pathogens and data sets containing ancient DNA samples. A key aspect of such calibrations is whether a sufficient amount of molecular evolution has occurred over the sampling time window, that is, whether the data can be treated as being from a measurably evolving population. Here we investigate the performance of a fully Bayesian evaluation of temporal signal (BETS) in molecular sequence data. The method involves comparing the fit of two models: a model in which the data are accompanied by the actual (heterochronous) sampling times, and a model in which the samples are constrained to be contemporaneous (isochronous). We conduct extensive simulations under a range of conditions to demonstrate that BETS accurately classifies data sets according to whether they contain temporal signal or not, even when there is substantial among-lineage rate variation. We explore the behaviour of this classification in analyses of five data sets: modern samples of A/H1N1 influenza virus , the bacterium Bordetella pertussis , and coronaviruses from mammalian hosts, and ancient DNA data sets of Hepatitis B virus and of mitochondrial genomes of dog species. Our results indicate that BETS is an effective alternative to other measures of temporal signal. In particular, this method has the key advantage of allowing a coherent assessment of the entire model, including the molecular clock and tree prior which are essential aspects of Bayesian phylodynamic analyses."/>
  <meta name="DC.Contributor" content="Sebastian Duchene"/>
  <meta name="DC.Contributor" content="Philippe Lemey"/>
  <meta name="DC.Contributor" content="Tanja Stadler"/>
  <meta name="DC.Contributor" content="Simon YW Ho"/>
  <meta name="DC.Contributor" content="David A Duchene"/>
  <meta name="DC.Contributor" content="Vijaykrishna Dhanasekaran"/>
  <meta name="DC.Contributor" content="Guy Baele"/>
  <meta name="article:published_time" content="2019-10-21"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Phylogenetic methods can use the sampling times of molecular sequence data to calibrate the molecular clock, enabling the estimation of substitution rates and time scales for rapidly evolving pathogens and data sets containing ancient DNA samples. A key aspect of such calibrations is whether a sufficient amount of molecular evolution has occurred over the sampling time window, that is, whether the data can be treated as being from a measurably evolving population. Here we investigate the performance of a fully Bayesian evaluation of temporal signal (BETS) in molecular sequence data. The method involves comparing the fit of two models: a model in which the data are accompanied by the actual (heterochronous) sampling times, and a model in which the samples are constrained to be contemporaneous (isochronous). We conduct extensive simulations under a range of conditions to demonstrate that BETS accurately classifies data sets according to whether they contain temporal signal or not, even when there is substantial among-lineage rate variation. We explore the behaviour of this classification in analyses of five data sets: modern samples of &lt;;i&gt;;A/H1N1 influenza virus&lt;;/i&gt;;, the bacterium &lt;;i&gt;;Bordetella pertussis&lt;;/i&gt;;, and coronaviruses from mammalian hosts, and ancient DNA data sets of &lt;;i&gt;;Hepatitis B virus&lt;;/i&gt;; and of mitochondrial genomes of dog species. Our results indicate that BETS is an effective alternative to other measures of temporal signal. In particular, this method has the key advantage of allowing a coherent assessment of the entire model, including the molecular clock and tree prior which are essential aspects of Bayesian phylodynamic analyses.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;810697v1"/>
  <meta name="citation_id" content="810697v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/810697v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/810697v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/810697v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697.full.pdf"/>
  <meta name="citation_doi" content="10.1101/810697"/>
  <meta name="citation_num_pages" content="30"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="810697"/>
  <meta name="citation_author" content="Sebastian Duchene"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Peter Doherty Institute for Infection and Immunity, University of Melbourne"/>
  <meta name="citation_author_email" content="sebastian.duchene@unimelb.edu.au"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2863-0907"/>
  <meta name="citation_author" content="Philippe Lemey"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Transplantation, Rega Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2826-5353"/>
  <meta name="citation_author" content="Tanja Stadler"/>
  <meta name="citation_author_institution" content="Department of Biosystems Science and Engineering, ETH Zürich; and Swiss Institute of Bioinformatics; Basel"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6431-535X"/>
  <meta name="citation_author" content="Simon YW Ho"/>
  <meta name="citation_author_institution" content="School of Life and Environmental Sciences, University of Sydney"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0361-2307"/>
  <meta name="citation_author" content="David A Duchene"/>
  <meta name="citation_author_institution" content="Research School of Biology, Australian National University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5993-978X"/>
  <meta name="citation_author" content="Vijaykrishna Dhanasekaran"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Biomedicine Discovery Institute, Monash University"/>
  <meta name="citation_author" content="Guy Baele"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Transplantation, Rega Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1915-7732"/>
  <meta name="citation_reference" content="citation_title=Bayesian Phylogenetics, Methods, Algorithms, and Applications;citation_pages=59-93;citation_year=2014"/>
  <meta name="citation_reference" content="Baele G., Lemey P., Bedford T., Rambaut A., Suchard M.A., Alekseyenko A. V. 2012. Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty. Mol. Biol. Evol. 29:2157–2167."/>
  <meta name="citation_reference" content="Baele G., Lemey P., Suchard M.A. 2016. Genealogical working distributions for Bayesian model testing with phylogenetic uncertainty. Syst. Biol. 65:250–264."/>
  <meta name="citation_reference" content="Baele G., Li W.L.S., Drummond A.J., Suchard M.A., Lemey P. 2013. Accurate model selection of relaxed molecular clocks in bayesian phylogenetics. Mol. Biol. Evol. 30:239–243."/>
  <meta name="citation_reference" content="Bart M.J., Harris S.R., Advani A., Arakawa Y., Bottero D., Bouchez V., Cassiday P.K., Chiang C.-S., Dalby T., Fry N.K. 2014. Global population structure and evolution of Bordetella pertussis and their relationship with vaccination. MBio. 5:e01074–14."/>
  <meta name="citation_reference" content="Biek R., Pybus O.G., Lloyd-Smith J.O., Didelot X. 2015. Measurably evolving pathogens in the genomic era. Trends Ecol. Evol. 30:306–313."/>
  <meta name="citation_reference" content="Bouckaert R., Vaughan T.G., Barido-Sottani J., Duchene S., Fourment M., Gavryushkina A., Heled J., Jones G., Kuhnert D., de Maio N. 2019. BEAST 2.5: An Advanced Software Platform for Bayesian Evolutionary Analysis. PLOS Comput. Biol. 15:e1006650."/>
  <meta name="citation_reference" content="Bromham L., Duchêne S., Hua X., Ritchie A.M., Duchêne D.A., Ho S.Y.W. 2018. Bayesian molecular dating: Opening up the black box. Biol. Rev. 93:1165–1191."/>
  <meta name="citation_reference" content="Drummond A.J., Ho S.Y.W., Phillips M.J., Rambaut A. 2006. Relaxed phylogenetics and dating with confidence. PLOS Biol. 4:e88."/>
  <meta name="citation_reference" content="Drummond A.J., Pybus O.G., Rambaut A., Forsberg R., Rodrigo A.G. 2003. Measurably evolving populations. Trends Ecol. Evol. 18:481–488."/>
  <meta name="citation_reference" content="Duchene S., Duchene D.A., Geoghegan J.L., Dyson Z.A., Hawkey J., Holt K.E. 2018. Inferring demographic parameters in bacterial genomic data using Bayesian and hybrid phylogenetic methods. BMC Evol. Biol. 18:95."/>
  <meta name="citation_reference" content="Duchene S., Duchene D.A., Holmes E.C., Ho S.Y.W. 2015. The performance of the daterandomization test in phylogenetic analyses of time-structured virus data. Mol. Biol. Evol. 32:1895–1906."/>
  <meta name="citation_reference" content="Duchene S., Holmes E.C., Ho S.Y.W. 2014. Analyses of evolutionary dynamics in viruses are hindered by a time-dependent bias in rate estimates. Proc. R. Soc. London B. 281:20140732."/>
  <meta name="citation_reference" content="Duchene S., Holt K.E., Weill F.-X., Le Hello S., Hawkey J., Edwards D.J., Fourment M., Holmes E.C. 2016. Genome-scale rates of evolutionary change in bacteria. Microb. Genomics. 2:e000094."/>
  <meta name="citation_reference" content="Fan Y., Wu R., Chen M.-H., Kuo L., Lewis P.O. 2011. Choosing among partition models in Bayesian phylogenetics. Mol. Biol. Evol. 28:523–532."/>
  <meta name="citation_reference" content="Ferreira M.A.R., Suchard M.A. 2008. Bayesian analysis of elapsed times in continuous-time Markov chains. Can. J. Stat. 36:355–368."/>
  <meta name="citation_reference" content="Fourment M., Magee A.F., Whidden C., Bilge A., Matsen I. V, Frederick A., Minin V.N. 2019. 19 dubious ways to compute the marginal likelihood of a phylogenetic tree topology. Syst. Biol.:syz046."/>
  <meta name="citation_reference" content="Guindon S., Dufayard J.-F., Lefort V., Anisimova M., Hordijk W., Gascuel O. 2010. New algorithms and methods to estimate maximum likelihood phylogenies: assessing the performance of PhyML 3.0. Syst. Biol. 59:307–321."/>
  <meta name="citation_reference" content="Hasegawa M., Kishino H., Yano T. 1985. Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. J. Mol. Evol. 22:160–174."/>
  <meta name="citation_reference" content="Hedge J., Lycett S.J., Rambaut A. 2013. Real-time characterization of the molecular epidemiology of an influenza pandemic. Biol. Lett. 9:20130331."/>
  <meta name="citation_reference" content="Hedge J., Wilson D.J. 2014. Bacterial phylogenetic reconstruction from whole genomes is robust to recombination but demographic inference is not. MBio. 5:e02158–14."/>
  <meta name="citation_reference" content="Hipsley C.A., Müller J. 2014. Beyond fossil calibrations: realities of molecular clock practices in evolutionary biology. Front. Genet. 5:138."/>
  <meta name="citation_reference" content="Ho S.Y.W., Duchene S. 2014. Molecular-clock methods for estimating evolutionary rates and time scales. Mol. Ecol. 23:5947–5975."/>
  <meta name="citation_reference" content="Kass R.E., Raftery A.E. 1995. Bayes factors. J. Am. Stat. Assoc. 90:773–795."/>
  <meta name="citation_reference" content="Korber B., Muldoon M., Theiler J., Gao F., Gupta R., Lapedes A., Hahn B.H., Wolinsky S., Bhattacharya T. 2000. Timing the ancestor of the HIV-1 pandemic strains. Science. 288:1789–1796."/>
  <meta name="citation_reference" content="Lartillot N., Philippe H. 2006. Computing Bayes factors using thermodynamic integration. Syst. Biol. 55:195–207."/>
  <meta name="citation_reference" content="Maturana P., Brewer B.J., Klaere S., Bouckaert R. 2019. Model selection and parameter inference in phylogenetics using Nested Sampling. Syst. Biol. 68:219–233."/>
  <meta name="citation_reference" content="Möller S., du Plessis L., Stadler T. 2018. Impact of the tree prior on estimating clock rates during epidemic outbreaks. Proc. Natl. Acad. Sci. USA. 115:4200–4205."/>
  <meta name="citation_reference" content="Murray G.G.R., Wang F., Harrison E.M., Paterson G.K., Mather A.E., Harris S.R., Holmes M.A., Rambaut A., Welch J.J. 2015. The effect of genetic structure on molecular dating and tests for temporal signal. Methods Ecol. Evol. 7:80–89."/>
  <meta name="citation_reference" content="Nascimento F.F., dos Reis M., Yang Z. 2017. A biologist’s guide to Bayesian phylogenetic analysis. Nat. Ecol. Evol. 1:1446."/>
  <meta name="citation_reference" content="Newton M.A., Raftery A.E. 1994. Approximate Bayesian inference with the weighted likelihood bootstrap. J. R. Stat. Soc. Ser. B. 56:3–26."/>
  <meta name="citation_reference" content="Oaks J.R., Cobb K.A., Minin V.N., Leaché A.D. 2019. Marginal likelihoods in phylogenetics: a review of methods and applications. Syst. Biol. 68:681–697."/>
  <meta name="citation_reference" content="Patterson Ross Z., Klunk J., Fornaciari G., Giuffra V., Duchene S., Duggan A.T., Poinar D., Douglas M.W., Eden J.-S., Holmes E.C., Poinar H.N. 2018. The paradox of HBV evolution as revealed from a 16thcentury mummy. PLoS Pathog. 14:e1006887."/>
  <meta name="citation_reference" content="Plummer M., Best N., Cowles K., Vines K. 2006. CODA: Convergence diagnosis and output analysis for MCMC. R News. 6:7–11."/>
  <meta name="citation_reference" content="Raftery A., Newton M., Satagopan J., Krivitsky P. 2007. Estimating the integrated likelihood via posterior simulation using the harmonic mean identity. Bayesian Statistics. Oxford University Press. p. 1–45."/>
  <meta name="citation_reference" content="Rambaut A., Lam T.T., Carvalho L.M., Pybus O.G. 2016. Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen). Virus Evol. 2:vew007."/>
  <meta name="citation_reference" content="Ramsden C., Holmes E.C.C., Charleston M.A.A. 2009. Hantavirus evolution in relation to its rodent and insectivore hosts: no evidence for codivergence. Mol. Biol. Evol. 26:143–153."/>
  <meta name="citation_reference" content="Rieux A., Balloux F. 2016. Inferences from tip-calibrated phylogenies: a review and a practical guide. Mol. Ecol. 25:1911–1924."/>
  <meta name="citation_reference" content="Ritchie A.M., Lo N., Ho S.Y.W. 2017. The impact of the tree prior on molecular dating of data sets containing a mixture of inter-and intraspecies sampling. Syst. Biol. 66:413–425."/>
  <meta name="citation_reference" content="Schliep K.P. 2011. phangorn: Phylogenetic analysis in R. Bioinformatics. 27:592–593."/>
  <meta name="citation_reference" content="Shapiro B., Ho S.Y.W., Drummond A.J., Suchard M.A., Pybus O.G., Rambaut A. 2011. A Bayesian phylogenetic method to estimate unknown sequence ages. Mol. Biol. Evol. 28:879–887."/>
  <meta name="citation_reference" content="Shapiro B., Rambaut A., Drummond A.J. 2006. Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences. Mol. Biol. Evol. 23:7–9."/>
  <meta name="citation_reference" content="Skilling J. 2006. Nested sampling for general Bayesian computation. Bayesian Anal. 1:833–859."/>
  <meta name="citation_reference" content="Suchard M.A., Lemey P., Baele G., Ayres D.L., Drummond A.J., Rambaut A. 2018. Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10. Virus Evol. 4:vey016. Tavaré S. 1986. Some probabilistic and statistical problems in the analysis of DNA sequences. Lect. Math. Life Sci. 17:57–86."/>
  <meta name="citation_reference" content="Thalmann O., Shapiro B., Cui P., Schuenemann V.J., Sawyer S.K., Greenfield D.L., Germonpré M.B., Sablin M. V, López-Giráldez F., Domingo-Roura X. 2013. Complete mitochondrial genomes of ancient canids suggest a European origin of domestic dogs. Science. 342:871–874."/>
  <meta name="citation_reference" content="Tong K.J., Duchêne D.A., Duchêne S., Geoghegan J.L., Ho S.Y.W. 2018. A comparison of methods for estimating substitution rates from ancient DNA sequence data. BMC Evol. Biol. 18:70."/>
  <meta name="citation_reference" content="Trovão N.S., Baele G., Vrancken B., Bielejec F., Suchard M.A., Fargette D., Lemey P. 2015. Host ecology determines the dispersal patterns of a plant virus. Virus Evol. 1:vev016."/>
  <meta name="citation_reference" content="Vaughan T.G., Drummond A.J. 2013. A stochastic simulator of birth–death master equations with application to phylodynamics. Mol. Biol. Evol. 30:1480–1493."/>
  <meta name="citation_reference" content="Wertheim J.O., Chu D.K.W., Peiris J.S.M., Pond S.L.K., Poon L.L.M. 2013. A case for the ancient origin of coronaviruses. J. Virol. 87:7039–7045."/>
  <meta name="citation_reference" content="Xie W., Lewis P.O., Fan Y., Kuo L., Chen M.H. 2011. Improving marginal likelihood estimation for Bayesian phylogenetic model selection. Syst. Biol. 60:150–160."/>
  <meta name="citation_reference" content="Yang Z. 1994. Estimating the pattern of nucleotide substitution. J. Mol. Evol. 39:105–111."/>
  <meta name="citation_reference" content="Yang Z. 1996. Among-site rate variation and its impact on phylogenetic analyses. Trends Ecol. Evol. 11:367–372."/>
  <meta name="citation_reference" content="citation_title=Evolving Genes and Proteins;citation_pages=97-166;citation_year=1965"/>
  <meta name="twitter:title" content="Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Phylogenetic methods can use the sampling times of molecular sequence data to calibrate the molecular clock, enabling the estimation of substitution rates and time scales for rapidly evolving pathogens and data sets containing ancient DNA samples. A key aspect of such calibrations is whether a sufficient amount of molecular evolution has occurred over the sampling time window, that is, whether the data can be treated as being from a measurably evolving population. Here we investigate the performance of a fully Bayesian evaluation of temporal signal (BETS) in molecular sequence data. The method involves comparing the fit of two models: a model in which the data are accompanied by the actual (heterochronous) sampling times, and a model in which the samples are constrained to be contemporaneous (isochronous). We conduct extensive simulations under a range of conditions to demonstrate that BETS accurately classifies data sets according to whether they contain temporal signal or not, even when there is substantial among-lineage rate variation. We explore the behaviour of this classification in analyses of five data sets: modern samples of A/H1N1 influenza virus , the bacterium Bordetella pertussis , and coronaviruses from mammalian hosts, and ancient DNA data sets of Hepatitis B virus and of mitochondrial genomes of dog species. Our results indicate that BETS is an effective alternative to other measures of temporal signal. In particular, this method has the key advantage of allowing a coherent assessment of the entire model, including the molecular clock and tree prior which are essential aspects of Bayesian phylodynamic analyses."/>
  <meta name="og-title" property="og:title" content="Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/810697v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Phylogenetic methods can use the sampling times of molecular sequence data to calibrate the molecular clock, enabling the estimation of substitution rates and time scales for rapidly evolving pathogens and data sets containing ancient DNA samples. A key aspect of such calibrations is whether a sufficient amount of molecular evolution has occurred over the sampling time window, that is, whether the data can be treated as being from a measurably evolving population. Here we investigate the performance of a fully Bayesian evaluation of temporal signal (BETS) in molecular sequence data. The method involves comparing the fit of two models: a model in which the data are accompanied by the actual (heterochronous) sampling times, and a model in which the samples are constrained to be contemporaneous (isochronous). We conduct extensive simulations under a range of conditions to demonstrate that BETS accurately classifies data sets according to whether they contain temporal signal or not, even when there is substantial among-lineage rate variation. We explore the behaviour of this classification in analyses of five data sets: modern samples of A/H1N1 influenza virus , the bacterium Bordetella pertussis , and coronaviruses from mammalian hosts, and ancient DNA data sets of Hepatitis B virus and of mitochondrial genomes of dog species. Our results indicate that BETS is an effective alternative to other measures of temporal signal. In particular, this method has the key advantage of allowing a coherent assessment of the entire model, including the molecular clock and tree prior which are essential aspects of Bayesian phylodynamic analyses."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-10-21"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-959076 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Phylogenetic methods can use the sampling times of molecular sequence data to calibrate the molecular clock, enabling the estimation of substitution rates and time scales for rapidly evolving pathogens and data sets containing ancient DNA samples. A key aspect of such calibrations is whether a sufficient amount of molecular evolution has occurred over the sampling time window, that is, whether the data can be treated as being from a measurably evolving population. Here we investigate the performance of a fully Bayesian evaluation of temporal signal (BETS) in molecular sequence data. The method involves comparing the fit of two models: a model in which the data are accompanied by the actual (heterochronous) sampling times, and a model in which the samples are constrained to be contemporaneous (isochronous). We conduct extensive simulations under a range of conditions to demonstrate that BETS accurately classifies data sets according to whether they contain temporal signal or not, even when there is substantial among-lineage rate variation. We explore the behaviour of this classification in analyses of five data sets: modern samples of 
     <em>A/H1N1 influenza virus</em>, the bacterium 
     <em>Bordetella pertussis</em>, and coronaviruses from mammalian hosts, and ancient DNA data sets of 
     <em>Hepatitis B virus</em> and of mitochondrial genomes of dog species. Our results indicate that BETS is an effective alternative to other measures of temporal signal. In particular, this method has the key advantage of allowing a coherent assessment of the entire model, including the molecular clock and tree prior which are essential aspects of Bayesian phylodynamic analyses.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">The molecular clock has become a ubiquitous tool for studying evolutionary processes in rapidly evolving organisms and in data sets that include ancient DNA. In its simplest form, the molecular clock posits that evolutionary change occurs at a predictable rate over time (
     <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zuckerkandl and Pauling 1965</a>). The molecular clock can be calibrated to estimate divergence times by using sampling time information, the timing of known divergence events, or a previous estimate of the substitution rate (
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hipsley and Müller 2014</a>). For example, 
     <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Korber et al. (2000)</a> used sampling times to calibrate the molecular clock and to infer the time of origin of HIV group 1. Their approach consisted of estimating a phylogenetic tree and conducting a regression of the distance from the root to each of the tips as a function of sequence sampling time. In this method, the slope of the regression is an estimate of the substitution rate in substitutions per site per unit of time, the intercept with the time axis is the age of the root node, and the coefficient of determination (
     <em>R</em>
     <sup>2</sup>) is the degree to which the data exhibit clocklike behaviour (
     <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rambaut et al. 2016</a>). Despite the practicality of root-to-tip regression, its use as a statistical tool for molecular dating has several well-known limitations. In particular, data points are not independent because they have shared ancestry (i.e., internal branches are traversed multiple times) and a strict clocklike behaviour is assumed by necessity.
    </p>
    <p id="p-5">The past few decades have seen a surge in molecular clock models that explicitly use phylogenetic information. Bayesian methods have gained substantial popularity, largely due to the wide array of complex models that can be implemented and the fact that independent information, including calibrations, can be specified via prior distributions (
     <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nascimento et al. 2017</a>). Of particular importance is the availability of molecular clock models that relax the assumption of strict clock behaviour by explicitly modelling rate variation among lineages (reviewed by 
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ho and Duchene (2014)</a> and by 
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bromham et al. (2018)</a>).
    </p>
    <p id="p-6">Regardless of the methodology used to analyse time-stamped sequence data, a sufficient amount of molecular evolution must have occurred over the sampling time window to allow reliable estimates of substitution rates and timescales. In such cases, the population can be considered to be ‘measurably evolving’ (
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Drummond et al. 2003</a>). The degree of ‘temporal information’ in sequence data is determined by the sequence length, the substitution rate, and the range of available sampling times. Some viruses evolve at a rate of around 5×10
     <sup>−3</sup> subs/site/year (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Duchene et al. 2014</a>), such that samples collected over a few weeks can be sufficient to calibrate the molecular clock. In more slowly evolving organisms, such as mammals, a sampling window of tens of thousands of years might be necessary; this can be achieved by including ancient DNA sequences (
     <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Drummond et al. 2003</a>; 
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Biek et al. 2015</a>).
    </p>
    <p id="p-7">Testing for temporal signal is an important step for verifying that the molecular clock can be calibrated using the sampling times (
     <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rieux and Balloux 2016</a>). For this purpose a date-randomization test has been proposed that compares actual substitution rate estimates to those obtained by repeatedly permuting the sequence sampling times (
     <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ramsden et al. 2009</a>). A data set is considered to have strong temporal signal if the rate estimated using the correct sampling times does not overlap with those of the permutation replicates (
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Duchene et al. 2015</a>, 2018; 
     <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Murray et al. 2015</a>). An implementation of this test is also available that performs the permutation during a single Bayesian inference (
     <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Trovão et al. 2015</a>). The interpretation of the date-randomization test is essentially frequentist in nature, which leads to an inconsistent mixture of statistical frameworks when Bayesian phylogenetic methods are used. Moreover, the procedure is not applicable in cases with small numbers of sampling times, owing to the limited number of possible permutations (
     <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Duchene et al. 2015</a>).
    </p>
    <p id="p-8">We propose a full Bayesian model test, which we refer to as BETS (Bayesian Evaluation of Temporal Signal), to assess temporal signal based on previous analyses by 
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baele et al. (2012)</a>. The approach involves quantifying statistical support for two competing models: a model in which the data are accompanied by the actual sampling times (i.e., the data are treated as heterochronous) and a model in which the sampling times are contemporaneous (i.e., the data are treated as isochronous). Therefore, the sampling times are treated as part of the model and the test can be understood as a test of ultrametricity of the phylogenetic tree. If incorporating sampling times improves the statistical fit, then their use for clock calibration is warranted. The crux of BETS, as with Bayesian model selection, is that it requires calculating the marginal likelihood of the model in question. The marginal likelihood measures the evidence for a model given the data, and calculating it requires integration of its likelihood across all parameter values, weighted by the prior (
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kass and Raftery 1995</a>).
    </p>
    <p id="p-9">Because the marginal likelihood is a measure of model evidence, the ratio of the marginal likelihoods of two competing models, known as the Bayes factor, is used to assess support for one model relative to the other. In the case of applying BETS, let 
     <em>M</em>
     <sub>het</sub> represent the heterochronous model, 
     <em>M</em>
     <sub>iso</sub> the isochronous model, and 
     <em>Y</em> the sequence data, such that P(
     <em>Y</em>|
     <em>M</em>
     <sub>het</sub>) and P(
     <em>Y</em>|
     <em>M</em>
     <sub>iso</sub>) are their respective marginal likelihoods. These models differ in the number of parameters; in 
     <em>M</em>
     <sub>iso</sub> the substitution rates and times are nonidentifiable, so the rate is fixed to an arbitrary value, while in 
     <em>M</em>
     <sub>het</sub> it is a free parameter. Differences in the number of parameters do not need to be taken into account separately, because the marginal likelihood naturally penalizes excessive parameterization. 
     <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kass and Raftery (1995)</a> gave guidelines to interpreting Bayes factors, where a (log) Bayes factor log(P(
     <em>Y</em>|
     <em>M</em>
     <sub>het</sub>)) – log(P(
     <em>Y</em>|
     <em>M</em>
     <sub>iso</sub>)) of at least 5 indicates ‘very strong’ support for 
     <em>M</em>
     <sub>het</sub> over 
     <em>M</em>
     <sub>iso</sub>, a value of 3 indicates ‘strong’ support, and a value of 1 is considered as positive evidence for 
     <em>M</em>
     <sub>het</sub> over 
     <em>M</em>
     <sub>iso</sub>.
    </p>
    <p id="p-10">The importance of model selection in Bayesian phylogenetics has prompted the development of various techniques to calculate marginal likelihoods (reviewed by Baele et al. (2014) and by 
     <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Oaks et al. (2019)</a>). These techniques can be broadly classified into prior-based and/or posterior-based estimators and path-sampling approaches. Prior- and posterior-based estimators, also known as importance sampling, include the widely used harmonic-mean estimator (
     <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Newton and Raftery 1994</a>) and the AICM and BICM (Bayesian analogues to the Akaike information criterion and the Bayesian information criterion, respectively) (
     <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Raftery et al. 2007</a>). These scores are easy to compute because they only require samples from the posterior distribution as obtained through Markov chain Monte Carlo (MCMC) integration. However, the harmonic-mean estimator has been shown to have unacceptably high variance when the prior is diffuse relative to the posterior, and, together with the AICM, has shown poor performance in practical settings (
     <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baele et al. 2012</a>, 
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2013</a>). The BICM requires a sample size to be specified for each parameter, which is far from trivial for phylogenetic inference and therefore remains unexplored for such applications.
    </p>
    <p id="p-11">Path-sampling approaches include path sampling (originally introduced in phylogenetics as ‘thermodynamic integration’) (
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lartillot and Philippe 2006</a>), stepping-stone sampling (
     <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Xie et al. 2011</a>), and generalized stepping-stone (GSS) sampling (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fan et al. 2011</a>; 
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baele et al. 2016</a>). These methods depend on drawing samples using MCMC from a range of power posterior distributions that represent the path from the posterior to the (working) prior, and therefore require additional computation. Another numerical technique that was recently introduced to phylogenetics is nested sampling (NS) (
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Maturana et al. 2019</a>), which approximates the marginal likelihood by simplifying the marginal-likelihood function from a multi-dimensional to a one-dimensional integral over the cumulative distribution function of the marginal likelihood (
     <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Skilling 2006</a>). 
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fourment et al. (2019)</a> recently compared the accuracy of a range of marginal-likelihood estimation methods and found GSS to be the most accurate, albeit at increased computational cost. Clearly, the reliability of the marginal-likelihood estimator is a key consideration for applying BETS.
    </p>
    <p id="p-12">We conducted a simulation study to assess the reliability of BETS under a range of conditions that are typical for data sets of rapidly evolving organisms and of those involving ancient DNA. We also analysed five empirical data sets to showcase the performance of the test in practice. Our analyses demonstrate the utility of BETS to provide accurate evaluation of temporal signal across a wide range of conditions.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Simulations of Measurably Evolving Populations</h3>
     <p id="p-13">In our simulations we considered sequence data from heterochronous and isochronous trees. Heterochronous trees represent a situation where there is sufficient temporal signal, whereas isochronous trees lack temporal signal altogether. We simulated heterochronous phylogenetic trees under a stochastic birth-death process with between 90 and 110 tips. To generate isochronous trees we used similar settings, but we assumed a single sampling time. We then simulated substitution rates along the trees according to an uncorrelated relaxed clock with an underlying lognormal distribution with a mean of 5×10
      <sup>−3</sup> subs/site/unit time and a standard deviation, 
      <em>σ</em>, of 0, 0.1, 0.5, or 1, where 
      <em>σ</em>=0 is equivalent to simulating under a strict clock. We then simulated sequence evolution using an HKY+Γ substitution model, with parameter values similar to those estimated for influenza virus (
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hedge et al. 2013</a>), to generate alignments of 10,000 nucleotides.
     </p>
     <p id="p-14">Our main simulation conditions produced data sets in which about 50% of the sites were variable. We refer to this simulation scenario as (i) ‘high substitution rate and wide sampling window’, and we considered three other simulation scenarios that involved (ii) a lower substitution rate of 10
      <sup>−5</sup> subs/site/unit time, (iii) a narrower sampling window, and (iv) both of the last two conditions. We analysed the sequence data using a strict clock and an uncorrelated relaxed clock with an underlying lognormal distribution (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Drummond et al. 2006</a>). We considered three configurations for sampling times: birth-death sampling times, which are correct for the heterochronous data but not for the isochronous data; identical sampling times, which is correct for isochronous data but not for the heterochronous data; and permuted birth-death sampling times, which are incorrect for both heterochronous and isochronous data.
     </p>
     <p id="p-15">We estimated the log marginal likelihoods of these six combinations of sampling times and clock models using NS and GSS as implemented in BEAST 2.5 (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bouckaert et al. 2019</a>) and BEAST 1.10 (
      <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Suchard et al. 2018</a>), respectively. Our BETS approach ranked the models according to their log marginal likelihoods and computed log Bayes factors of the best heterochronous model (
      <em>M</em>
      <sub>het</sub>) compared with the best isochronous model (
      <em>M</em>
      <sub>iso</sub>).
     </p>
     <div id="sec-4" class="subsection">
      <h4>(i) Simulations with High Substitution Rate and Wide Sampling Window</h4>
      <p id="p-16">Both NS and GSS correctly classified data as being heterochronous or isochronous in 10 out of 10 simulations, including in the presence of a high degree of among-lineage rate variation (i.e., 
       <em>σ</em>=1; 
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">fig. 1</a> for heterochronous data and 
       <a id="xref-fig-8-1" class="xref-fig" href="#F8">supplementary fig. S1</a>, Supplementary Material online, for isochronous data). Although both marginal-likelihood estimators detected temporal signal, NS supported the relaxed clock over the strict clock for three heterochronous data sets simulated without among-lineage rate variation (
       <em>σ</em>=0) and for six data sets simulated with low among-lineage rate variation (
       <em>σ</em>=0.1). In the simulations of isochronous data, NS often favoured the relaxed clock over the strict clock when there was low among-lineage rate variation (
       <em>σ</em>=0.0 and 
       <em>σ</em>=0.1), albeit mostly with log Bayes factors below 5 (
       <a id="xref-fig-8-2" class="xref-fig" href="#F8">supplementary fig. S1</a>, Supplementary Material online). In contrast, GSS always selected the strict clock under these conditions (
       <a id="xref-fig-1-2" class="xref-fig" href="#F1">fig. 1</a> and 
       <a id="xref-fig-8-3" class="xref-fig" href="#F8">supplementary fig. S1</a>, Supplementary Material online).
      </p>
      <div id="F8" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log Bayes factors of isochronous data simulated with a high substitution rate. Each panel shows the results for data sets simulated with a different degree of among-lineage rate variation, governed by the standard deviation [[#x3C3]] of a lognormal distribution. The x-axis depicts six analysis settings, with two molecular clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). The points have been jittered to facilitate visualization. The y-axis shows log Bayes factors relative to the best model. Red points correspond to estimates using generalized stepping-stone sampling and blue points correspond to estimates using nested sampling. We conducted 10 simulation replicates, with each replicate data set analysed under the six analysis settings and two marginal-likelihood estimators, such that stochastic error might cause differences in the preferred model. The number next to each cloud of points denotes the number of times (out of 10) that the corresponding model had the highest marginal likelihood with generalized stepping-stone sampling (red) and nested sampling (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Log Bayes factors of isochronous data simulated with a high substitution rate. Each panel shows the results for data sets simulated with a different degree of among-lineage rate variation, governed by the standard deviation &lt;;em&gt;;[[#x3C3]]&lt;;/em&gt;; of a lognormal distribution. The &lt;;em&gt;;x&lt;;/em&gt;;-axis depicts six analysis settings, with two molecular clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). The points have been jittered to facilitate visualization. The &lt;;em&gt;;y&lt;;/em&gt;;-axis shows log Bayes factors relative to the best model. Red points correspond to estimates using generalized stepping-stone sampling and blue points correspond to estimates using nested sampling. We conducted 10 simulation replicates, with each replicate data set analysed under the six analysis settings and two marginal-likelihood estimators, such that stochastic error might cause differences in the preferred model. The number next to each cloud of points denotes the number of times (out of 10) that the corresponding model had the highest marginal likelihood with generalized stepping-stone sampling (red) and nested sampling (blue).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F8.medium.gif" width="440" height="341"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. S1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F8.medium.gif" width="440" height="341"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. S1.</span>
        <p id="p-61" class="first-child">Log Bayes factors of isochronous data simulated with a high substitution rate. Each panel shows the results for data sets simulated with a different degree of among-lineage rate variation, governed by the standard deviation 
         <em>σ</em> of a lognormal distribution. The 
         <em>x</em>-axis depicts six analysis settings, with two molecular clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). The points have been jittered to facilitate visualization. The 
         <em>y</em>-axis shows log Bayes factors relative to the best model. Red points correspond to estimates using generalized stepping-stone sampling and blue points correspond to estimates using nested sampling. We conducted 10 simulation replicates, with each replicate data set analysed under the six analysis settings and two marginal-likelihood estimators, such that stochastic error might cause differences in the preferred model. The number next to each cloud of points denotes the number of times (out of 10) that the corresponding model had the highest marginal likelihood with generalized stepping-stone sampling (red) and nested sampling (blue).
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="F1" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log Bayes factors of heterochronous data simulated with a high substitution rate and wide sampling window. Each panel shows the results for data sets simulated with a different degree of among-lineage rate variation, governed by the standard deviation [[#x3C3]] of a lognormal distribution. In each plot the x-axis depicts six analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). The points have been jittered along the x-axis to facilitate visualization. The y-axis shows log Bayes factors relative to the best model. Red points correspond to estimates using generalized stepping-stone sampling and blue points correspond to estimates using nested sampling. We conducted 10 simulation replicates, with each replicate data set analysed under the six analysis settings and two marginal-likelihood estimators, such that stochastic error might cause differences in the preferred model. The number next to each cloud of points denotes the number of times (out of 10) that the corresponding model had the highest marginal likelihood with generalized stepping-stone sampling (red) and nested sampling (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Log Bayes factors of heterochronous data simulated with a high substitution rate and wide sampling window. Each panel shows the results for data sets simulated with a different degree of among-lineage rate variation, governed by the standard deviation &lt;;em&gt;;[[#x3C3]]&lt;;/em&gt;; of a lognormal distribution. In each plot the &lt;;em&gt;;x&lt;;/em&gt;;-axis depicts six analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). The points have been jittered along the &lt;;em&gt;;x&lt;;/em&gt;;-axis to facilitate visualization. The &lt;;em&gt;;y&lt;;/em&gt;;-axis shows log Bayes factors relative to the best model. Red points correspond to estimates using generalized stepping-stone sampling and blue points correspond to estimates using nested sampling. We conducted 10 simulation replicates, with each replicate data set analysed under the six analysis settings and two marginal-likelihood estimators, such that stochastic error might cause differences in the preferred model. The number next to each cloud of points denotes the number of times (out of 10) that the corresponding model had the highest marginal likelihood with generalized stepping-stone sampling (red) and nested sampling (blue).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F1.medium.gif" width="440" height="342"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F1.medium.gif" width="440" height="342"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. 1.</span>
        <p id="p-17" class="first-child">Log Bayes factors of heterochronous data simulated with a high substitution rate and wide sampling window. Each panel shows the results for data sets simulated with a different degree of among-lineage rate variation, governed by the standard deviation 
         <em>σ</em> of a lognormal distribution. In each plot the 
         <em>x</em>-axis depicts six analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). The points have been jittered along the 
         <em>x</em>-axis to facilitate visualization. The 
         <em>y</em>-axis shows log Bayes factors relative to the best model. Red points correspond to estimates using generalized stepping-stone sampling and blue points correspond to estimates using nested sampling. We conducted 10 simulation replicates, with each replicate data set analysed under the six analysis settings and two marginal-likelihood estimators, such that stochastic error might cause differences in the preferred model. The number next to each cloud of points denotes the number of times (out of 10) that the corresponding model had the highest marginal likelihood with generalized stepping-stone sampling (red) and nested sampling (blue).
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-18">For the heterochronous data sets, NS and GSS always displayed very strong support for 
       <em>M</em>
       <sub>het</sub> over 
       <em>M</em>
       <sub>iso</sub>, with log Bayes factors of at least 90. For the isochronous data sets, the log Bayes factors for 
       <em>M</em>
       <sub>iso</sub> relative to 
       <em>M</em>
       <sub>het</sub> were overall much lower, but still decisive, ranging from 30 to 50. Furthermore, log Bayes factors tended to decline with an increasing degree of among-lineage rate variation in the data. Another important observation is that in the heterochronous data, the relaxed clock was consistently selected over the strict clock when assuming that the data were isochronous, or when the sampling times had been permuted (
       <a id="xref-fig-1-3" class="xref-fig" href="#F1">fig. 1</a> and 
       <a id="xref-fig-8-4" class="xref-fig" href="#F8">supplementary fig. S1</a>, Supplementary Material online). Moreover, the strict clock with permuted sampling times yielded the lowest log marginal likelihoods for heterochronous data. Both of these patterns are likely to be due to an apparently higher degree of among-lineage rate variation when sampling times are misspecified.
      </p>
     </div>
     <div id="sec-5" class="subsection">
      <h4>(ii) Simulations with Low Substitution Rate and Wide Sampling Window</h4>
      <p id="p-19">Our simulations with a low substitution rate of 10
       <sup>−5</sup> subs/site/unit time produced data sets that each had about 10 variable sites, which provides very little information for the estimation of evolutionary parameters. Additionally, due to the stochasticity of the simulation process, increased estimator variance between replicates is to be expected given the small number of variable sites. For the heterochronous data sets, GSS selected the heterochronous model with correct dates in at least 7 out of 10 simulation replicates (
       <a id="xref-fig-2-1" class="xref-fig" href="#F2">fig. 2</a>). Across the simulations with different clock models (40 in total), only in five heterochronous data sets did we find models with permuted sampling times to have the highest log marginal likelihoods. For NS, in 11 out of 40 simulations, either isochronous models or those with random sampling times were incorrectly selected when heterochronous data sets were analysed.
      </p>
      <div id="F2" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log Bayes factors of heterochronous data simulated under a low substitution rate and a wide sampling window. Symbols and colours are the same as those in figure 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Log Bayes factors of heterochronous data simulated under a low substitution rate and a wide sampling window. Symbols and colours are the same as those in figure 1.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F2.medium.gif" width="440" height="341"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F2.medium.gif" width="440" height="341"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. 2.</span>
        <p id="p-20" class="first-child">Log Bayes factors of heterochronous data simulated under a low substitution rate and a wide sampling window. Symbols and colours are the same as those in 
         <a id="xref-fig-1-4" class="xref-fig" href="#F1">figure 1</a>.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-21">Log marginal likelihoods calculated using GSS tended to support models with sampling times (either permuted or those from the birth-death) for the isochronous data, whereas NS appeared to support all models with similar frequencies (
       <a id="xref-fig-9-1" class="xref-fig" href="#F9">supplementary fig. S2</a>, Supplementary Material online). However, a critical feature of the results from the data sets with a low substitution rate is that the log marginal likelihoods for all models were more similar to one another than those for the data sets with high substitution rate (note that the log marginal likelihood scale in 
       <a id="xref-fig-2-2" class="xref-fig" href="#F2">fig. 2</a> is smaller than that in 
       <a id="xref-fig-1-5" class="xref-fig" href="#F1">fig. 1</a>). As a case in point, for the isochronous data with 
       <em>σ</em>=0.1 there were log Bayes factors of about 0.1 for the best model with birth-death sampling times relative to those with permuted sampling times. This result indicates that comparing models with permuted sampling times might be useful for determining whether the data are informative about a particular set of sampling times.
      </p>
      <div id="F9" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log Bayes factors of isochronous data simulated with a low substitution rate. Symbols and colours are the same as those in figure 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Log Bayes factors of isochronous data simulated with a low substitution rate. Symbols and colours are the same as those in figure 1.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F9.medium.gif" width="440" height="338"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. S2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F9.medium.gif" width="440" height="338"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. S2.</span>
        <p id="p-62" class="first-child">Log Bayes factors of isochronous data simulated with a low substitution rate. Symbols and colours are the same as those in 
         <a id="xref-fig-1-10" class="xref-fig" href="#F1">figure 1</a>.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-6" class="subsection">
      <h4>(iii)Simulations with High Substitution Rate and Narrow Sampling Window</h4>
      <p id="p-22">We conducted a set of simulations similar to those described in scenario (i) but where sequence sampling spanned only the last 10% of the age of the tree (0.5 units of time, compared with 5 units of time for the simulations with a wide sampling window). These conditions reflect those of organisms with deep evolutionary histories and for which samples are available for only a small portion of this time. Since in these trees the samples were collected over a narrower time window, we used a higher sampling probability to obtain about 100 samples, as in our other simulations (see examples of trees in 
       <a id="xref-fig-10-1" class="xref-fig" href="#F10">supplementary fig. S3</a>, Supplementary Material online). For these analyses we only considered heterochronous data because the isochronous case is the same as that in scenario (i).
      </p>
      <div id="F10" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Example of three phylogenetic trees used in simulations. Red dashed lines indicate the times of each of the tips and therefore represent the sampling process over time. All trees are simulated under a birth-death process with time of origin of 5, such that the sum of the tree height and the length of the stem branch leading to the root is always 5. In all trees, we set the birth rate [[#x3BB]]=1.5, and become uninfectious rate [[#x3B4]]=1, where [[#x3B4]]=[[#x3BC]]+[[#x3D5]], where [[#x3BC]] is the death rate and [[#x3D5]] is the sampling rate upon death. Thus, the population growth rate is constant and the same across all trees. The top tree assumes a constant sampling process and a wide sampling window ([[#x3D5]]=0.5 throughout the whole process), whereas in the second tree sampling starts after 4.5. Before this time the sampling rate, [[#x3D5]]0, is zero. After 4.5 time units the sampling rate [[#x3D5]]1 is 0.9 (and thus mu_1 = 0.1), resulting in a narrow sampling window. The bottom tree has samples drawn at a single point in time with a sampling probability at present, [[#x3C1]], of 0.5 (and thus phi=0)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Example of three phylogenetic trees used in simulations. Red dashed lines indicate the times of each of the tips and therefore represent the sampling process over time. All trees are simulated under a birth-death process with time of origin of 5, such that the sum of the tree height and the length of the stem branch leading to the root is always 5. In all trees, we set the birth rate &lt;;em&gt;;[[#x3BB]]&lt;;/em&gt;;=1.5, and become uninfectious rate &lt;;em&gt;;[[#x3B4]]&lt;;/em&gt;;=1, where &lt;;em&gt;;[[#x3B4]]&lt;;/em&gt;;=&lt;;em&gt;;[[#x3BC]]&lt;;/em&gt;;+&lt;;em&gt;;[[#x3D5]]&lt;;/em&gt;;, where &lt;;em&gt;;[[#x3BC]]&lt;;/em&gt;; is the death rate and &lt;;em&gt;;[[#x3D5]]&lt;;/em&gt;; is the sampling rate upon death. Thus, the population growth rate is constant and the same across all trees. The top tree assumes a constant sampling process and a wide sampling window (&lt;;em&gt;;[[#x3D5]]&lt;;/em&gt;;=0.5 throughout the whole process), whereas in the second tree sampling starts after 4.5. Before this time the sampling rate, &lt;;em&gt;;[[#x3D5]]&lt;;/em&gt;;&lt;;sub&gt;;0&lt;;/sub&gt;;, is zero. After 4.5 time units the sampling rate &lt;;em&gt;;[[#x3D5]]&lt;;/em&gt;;&lt;;sub&gt;;1&lt;;/sub&gt;; is 0.9 (and thus mu_1 = 0.1), resulting in a narrow sampling window. The bottom tree has samples drawn at a single point in time with a sampling probability at present, &lt;;em&gt;;[[#x3C1]]&lt;;/em&gt;;, of 0.5 (and thus phi=0).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F10.medium.gif" width="440" height="337"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. S3." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F10.medium.gif" width="440" height="337"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. S3." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. S3.</span>
        <p id="p-63" class="first-child">Example of three phylogenetic trees used in simulations. Red dashed lines indicate the times of each of the tips and therefore represent the sampling process over time. All trees are simulated under a birth-death process with time of origin of 5, such that the sum of the tree height and the length of the stem branch leading to the root is always 5. In all trees, we set the birth rate 
         <em>λ</em>=1.5, and become uninfectious rate 
         <em>δ</em>=1, where 
         <em>δ</em>=
         <em>μ</em>+
         <em>ϕ</em>, where 
         <em>μ</em> is the death rate and 
         <em>ϕ</em> is the sampling rate upon death. Thus, the population growth rate is constant and the same across all trees. The top tree assumes a constant sampling process and a wide sampling window (
         <em>ϕ</em>=0.5 throughout the whole process), whereas in the second tree sampling starts after 4.5. Before this time the sampling rate, 
         <em>ϕ</em>
         <sub>0</sub>, is zero. After 4.5 time units the sampling rate 
         <em>ϕ</em>
         <sub>1</sub> is 0.9 (and thus mu_1 = 0.1), resulting in a narrow sampling window. The bottom tree has samples drawn at a single point in time with a sampling probability at present, 
         <em>ρ</em>, of 0.5 (and thus phi=0).
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-23">Both GSS and NS showed excellent performance in detecting temporal signal in this scenario, almost always selecting models with correct sampling times. The exceptions to this pattern occurred for one data set with 
       <em>σ</em>=0.5 and for two data sets with 
       <em>σ</em>=1.0 for NS (
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">fig. 3</a>). Differentiating between the strict clock and relaxed clock appeared somewhat more difficult, particularly for NS, where the relaxed clock with correct sampling times yielded log marginal likelihoods very similar to those for the strict clock for data with low among-lineage rate variation (
       <em>σ</em> of 0.0 or 0.1). Although NS and GSS performed well in these simulations, the log Bayes factors for 
       <em>M</em>
       <sub>het</sub> relative to 
       <em>M</em>
       <sub>iso</sub> were much lower than those for data with a high substitution rate and a wide sampling window in (i). One obvious example is in the data with 
       <em>σ</em>=0.0, where the mean log Bayes factors for 
       <em>M</em>
       <sub>het</sub> over 
       <em>M</em>
       <sub>iso</sub> using GSS was 203.15 with a wide sampling window (
       <a id="xref-fig-1-6" class="xref-fig" href="#F1">fig. 1</a>), but only 35.77 when sampling spanned a narrow time window (
       <a id="xref-fig-3-2" class="xref-fig" href="#F3">fig. 3</a>).
      </p>
      <div id="F3" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log Bayes factors of heterochronous data simulated under a high substitution rate and narrow sampling window. Symbols and colours are the same as those in figure 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Log Bayes factors of heterochronous data simulated under a high substitution rate and narrow sampling window. Symbols and colours are the same as those in figure 1.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F3.medium.gif" width="440" height="338"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F3.medium.gif" width="440" height="338"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. 3.</span>
        <p id="p-24" class="first-child">Log Bayes factors of heterochronous data simulated under a high substitution rate and narrow sampling window. Symbols and colours are the same as those in 
         <a id="xref-fig-1-7" class="xref-fig" href="#F1">figure 1</a>.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-7" class="subsection">
      <h4>(iv) Simulations with Low Substitution Rate and Narrow Sampling Window</h4>
      <p id="p-25">We considered data sets with a narrow sampling window, as in scenario (iii), and with a low substitution rate of 10
       <sup>−5</sup> subs/site/unit time, as in scenario (ii). We generated only heterochronous trees under these conditions, because the isochronous case would be the same as that in (ii).
      </p>
      <p id="p-26">Estimates of log marginal likelihoods with GSS and NS were very similar among models, with mean log Bayes factors among data sets of less than 1 for the two models with highest marginal likelihoods for GSS (
       <a id="xref-fig-4-1" class="xref-fig" href="#F4">fig. 4</a>). In the data sets with 
       <em>σ</em>=0.0, GSS and NS always preferred a heterochronous model. However, in a few cases (three for GSS and one for NS) the model with permuted sampling times was selected, indicating that temporal signal was not detected. As with the data sets with low substitution rate and constant sampling (ii), the relaxed clock was sometimes preferred over the strict clock, even when the data sets had no rate variation among lineages.
      </p>
      <div id="F4" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log Bayes factors of heterochronous data simulated under a low substitution rate and narrow sampling window. Symbols and colours are the same as those in figure 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Log Bayes factors of heterochronous data simulated under a low substitution rate and narrow sampling window. Symbols and colours are the same as those in figure 1.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F4.medium.gif" width="440" height="337"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F4.medium.gif" width="440" height="337"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. 4.</span>
        <p id="p-27" class="first-child">Log Bayes factors of heterochronous data simulated under a low substitution rate and narrow sampling window. Symbols and colours are the same as those in 
         <a id="xref-fig-1-8" class="xref-fig" href="#F1">figure 1</a>.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-8" class="subsection">
      <h4>Comparison with Root-to-tip Regression</h4>
      <p id="p-28">Using a subset of the heterochronous data sets, we conducted root-to-tip regression using phylogenetic trees inferred using maximum likelihood in PhyML 3.1 (
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guindon et al. 2010</a>) with the same substitution model as in our BEAST analyses, and with the placement of the root chosen to maximize 
       <em>R</em>
       <sup>2</sup> in TempEst (
       <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Rambaut et al. 2016</a>). We selected data sets generated with a high substitution rate and with both constant and narrow sampling windows. Because GSS and NS correctly detected temporal signal under these conditions, these regressions demonstrate the extent to which this informal regression assessment matches the BETS approach. We did not attempt to provide a thorough benchmarking of the two methods here.
      </p>
      <p id="p-29">All regressions had 
       <em>R</em>
       <sup>2</sup> values that matched our expectation from the degree of among-lineage rate variation, that is, higher values of 
       <em>σ</em> corresponded to lower values of 
       <em>R</em>
       <sup>2</sup> (
       <a id="xref-fig-5-1" class="xref-fig" href="#F5">fig. 5</a>). The data with a wide sampling window yielded regression slopes ranging from 7.3×10
       <sup>−3</sup> to 5.4×10
       <sup>−3</sup> subs/site/unit time, which is similar to the substitution rate values used to generate the data. Although the root- to-tip regression is sometimes used to assess temporal signal, it has no cut-off values to confirm temporal signal. This becomes critical when considering the data with a narrow sampling window, for which the 
       <em>R</em> was between 0.13 and 0.02. For example, the regression for a data set with 
       <em>σ</em>=1 and narrow sampling window had an 
       <em>R</em>
       <sup>2</sup> of 0.02, which is sometimes considered sufficiently low as to preclude molecular clock analyses (
       <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Rieux and Balloux 2016</a>). However, BETS supported strong temporal structure under a relaxed clock in this data set, with log Bayes factors of 5.48 for this particular data set, which matches the simulation conditions. More importantly, even with such high rate variation, the substitution rate estimated using a relaxed clock and the correct sampling times included the true value used to generate the data (5×10
       <sup>−3</sup> subs/site/unit time), with a 95% highest posterior density (HPD) of between 2.15×10
       <sup>−3</sup> and 1.90×10
       <sup>−2</sup> subs/site/unit time, while the regression slope was 2.22×10
       <sup>−2</sup> subs/site/unit time. A key implication of these comparisons is that BETS provides a formal assessment of temporal signal, unlike statistics computed from the regression. Moreover, the root-to-tip regression appears uninformative when the data have been sampled over a narrow time window and there is some rate variation among lineages.
      </p>
      <div id="F5" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Root-to-tip regressions for a subset of data sets simulated with varying degrees of among-lineage rate variation (governed by the standard deviation [[#x3C3]] of a lognormal distribution), using a high substitution rate and either a wide or narrow sampling window. The y-axis is the root-to-tip distance and the x-axis is the time from the youngest tip, where 0 is the present. Each point corresponds to a tip in the tree and the solid line is the best-fit linear regression using least-squares. The coefficient of determination, R2, is shown in each case. For comparison, the log Bayes factors of the best heterochronous model relative the best isochronous model, BF(Mhet -Miso), are also shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Root-to-tip regressions for a subset of data sets simulated with varying degrees of among-lineage rate variation (governed by the standard deviation &lt;;em&gt;;[[#x3C3]]&lt;;/em&gt;; of a lognormal distribution), using a high substitution rate and either a wide or narrow sampling window. The &lt;;em&gt;;y&lt;;/em&gt;;-axis is the root-to-tip distance and the &lt;;em&gt;;x&lt;;/em&gt;;-axis is the time from the youngest tip, where 0 is the present. Each point corresponds to a tip in the tree and the solid line is the best-fit linear regression using least-squares. The coefficient of determination, &lt;;em&gt;;R&lt;;/em&gt;;&lt;;sup&gt;;2&lt;;/sup&gt;;, is shown in each case. For comparison, the log Bayes factors of the best heterochronous model relative the best isochronous model, BF(&lt;;em&gt;;M&lt;;/em&gt;;&lt;;sub&gt;;het&lt;;/sub&gt;; -&lt;;em&gt;;M&lt;;/em&gt;;&lt;;sub&gt;;iso&lt;;/sub&gt;;), are also shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F5.medium.gif" width="440" height="438"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F5.medium.gif" width="440" height="438"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. 5.</span>
        <p id="p-30" class="first-child">Root-to-tip regressions for a subset of data sets simulated with varying degrees of among-lineage rate variation (governed by the standard deviation 
         <em>σ</em> of a lognormal distribution), using a high substitution rate and either a wide or narrow sampling window. The 
         <em>y</em>-axis is the root-to-tip distance and the 
         <em>x</em>-axis is the time from the youngest tip, where 0 is the present. Each point corresponds to a tip in the tree and the solid line is the best-fit linear regression using least-squares. The coefficient of determination, 
         <em>R</em>
         <sup>2</sup>, is shown in each case. For comparison, the log Bayes factors of the best heterochronous model relative the best isochronous model, BF(
         <em>M</em>
         <sub>het</sub> -
         <em>M</em>
         <sub>iso</sub>), are also shown.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-9" class="subsection">
      <h4>Analyses of Empirical Data Sets</h4>
      <p id="p-31">We analysed five empirical data sets with similar configurations of sampling times as in our simulation study (
       <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Two data sets consisted of rapidly evolving pathogens: 
       <em>A/H1N1 influenza virus</em> (
       <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hedge et al. 2013</a>) and 
       <em>Bordetella pertussis</em> (
       <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bart et al. 2014</a>). We also analysed a data set with highly divergent sequences of coronaviruses (
       <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wertheim et al. 2013</a>), and two data sets with ancient DNA: 
       <em>Hepatitis B virus</em> (
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Patterson Ross et al. 2018</a>), and mitochondrial genomes of dog species (
       <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Thalmann et al. 2013</a>). Due to the demonstrated higher accuracy of GSS over NS (
       <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fourment et al. 2019</a>), we applied the BETS approach using the former method only.
      </p>
      <div id="T1" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/961556/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/961556/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/961556" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 1.</span>
        <span class="caption-title">Details of empirical data sets used in this study.</span>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-32">The 
       <em>A/H1N1 influenza virus</em> data demonstrated clear temporal signal, with the strict clock and relaxed clock with the correct sampling times having the highest log marginal likelihoods, and a log Bayes factor of 
       <em>M</em>
       <sub>het</sub> with respect to 
       <em>M</em>
       <sub>iso</sub> of 150 (
       <a id="xref-fig-6-1" class="xref-fig" href="#F6">fig. 6</a>). The strict clock had higher support than the relaxed clock for the correct sampling times (log Bayes factor 3.41). Broadly, this result is consistent with previous evidence of strong temporal signal and clocklike behaviour in this data set (
       <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hedge et al. 2013</a>). Using the strict clock with correct sampling times we estimated a substitution rate of 3.37×10
       <sup>−3</sup> subs/site/year (HPD: 2.98×10
       <sup>−3</sup> to 3.78×10
       <sup>−3</sup>).
      </p>
      <div id="F6" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log marginal likelihoods estimated using generalized stepping-stone sampling for six analysis settings for sequence data from rapidly evolving pathogens, A/H1N1 Human influenza virus and Bordetella pertussis. The y-axis is the marginal likelihood and the x-axis shows the analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). Solid points and dashed lines correspond to the log marginal-likelihood estimates. The asterisk denotes the model with the highest marginal likelihood." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Log marginal likelihoods estimated using generalized stepping-stone sampling for six analysis settings for sequence data from rapidly evolving pathogens, &lt;;em&gt;;A/H1N1 Human influenza virus&lt;;/em&gt;; and &lt;;em&gt;;Bordetella pertussis&lt;;/em&gt;;. The &lt;;em&gt;;y&lt;;/em&gt;;-axis is the marginal likelihood and the &lt;;em&gt;;x&lt;;/em&gt;;-axis shows the analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). Solid points and dashed lines correspond to the log marginal-likelihood estimates. The asterisk denotes the model with the highest marginal likelihood.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F6.medium.gif" width="437" height="440"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F6.medium.gif" width="437" height="440"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. 6.</span>
        <p id="p-33" class="first-child">Log marginal likelihoods estimated using generalized stepping-stone sampling for six analysis settings for sequence data from rapidly evolving pathogens, 
         <em>A/H1N1 Human influenza virus</em> and 
         <em>Bordetella pertussis</em>. The 
         <em>y</em>-axis is the marginal likelihood and the 
         <em>x</em>-axis shows the analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and permuted (Permuted). Solid points and dashed lines correspond to the log marginal-likelihood estimates. The asterisk denotes the model with the highest marginal likelihood.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-34">We detected temporal signal in the 
       <em>Bordetella pertussis</em> data set (
       <a id="xref-fig-6-2" class="xref-fig" href="#F6">fig. 6</a>). The relaxed clock with the correct sampling times had the highest log marginal likelihood, with a log Bayes factor relative to the strict clock of 28.86. The log Bayes factor for 
       <em>M</em>
       <sub>het</sub> relative to 
       <em>M</em>
       <sub>iso</sub> was 47.40. These results echo previous assessments of these data using a date-randomization test (
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Duchene et al. 2016</a>). We estimated a mean substitution rate using the best model of 1.65×10
       <sup>−7</sup> subs/site/year (95% HPD: 1.36×10
       <sup>−7</sup> to 2.00×10
       <sup>−7</sup>).
      </p>
      <p id="p-35">Our analyses did not detect temporal signal in the coronavirus data, for which the strict clock and relaxed clock with no sampling times had the highest log marginal likelihoods. The log Bayes factor of 
       <em>M</em>
       <sub>het</sub> relative to 
       <em>M</em>
       <sub>iso</sub> was -16.82, indicating strong support for the isochronous model. The relaxed clock was supported over the strict clock, with a log Bayes factor of 19.25 (
       <a id="xref-fig-7-1" class="xref-fig" href="#F7">fig. 7</a>). Previous analyses of this data set suggested an ancient origin for this group of viruses, but here the lack of temporal signal precludes any interpretation of our estimates of substitution rates and timescales.
      </p>
      <div id="F7" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log marginal likelihoods estimated using generalized stepping-stone sampling for six analysis settings for data sets with ancient DNA or highly divergent sequences. The y-axis is the marginal likelihood and the x-axis shows the analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and randomized (Random). Solid points and dashed lines correspond to the log marginal-likelihood estimates. The asterisk denotes the model with the highest marginal likelihood." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991507801" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Log marginal likelihoods estimated using generalized stepping-stone sampling for six analysis settings for data sets with ancient DNA or highly divergent sequences. The &lt;;em&gt;;y&lt;;/em&gt;;-axis is the marginal likelihood and the &lt;;em&gt;;x&lt;;/em&gt;;-axis shows the analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and randomized (Random). Solid points and dashed lines correspond to the log marginal-likelihood estimates. The asterisk denotes the model with the highest marginal likelihood.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="FIG. 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F7.medium.gif" width="375" height="440"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="FIG. 7." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F7.medium.gif" width="375" height="440"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/21/810697/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">FIG. 7.</span>
        <p id="p-36" class="first-child">Log marginal likelihoods estimated using generalized stepping-stone sampling for six analysis settings for data sets with ancient DNA or highly divergent sequences. The 
         <em>y</em>-axis is the marginal likelihood and the 
         <em>x</em>-axis shows the analysis settings, with two clock models, strict clock (SC) and the uncorrelated relaxed clock with an underlying lognormal distribution (UCLN), and three settings for sampling times: generated under the birth-death process (BD), identical sampling times (Isochronous), and randomized (Random). Solid points and dashed lines correspond to the log marginal-likelihood estimates. The asterisk denotes the model with the highest marginal likelihood.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-37">The 
       <em>Hepatitis B viru</em>s data set included several human genotypes with complete genomes, where 135 were modern sequences collected from 1963 to 2013 and two were ancient samples from human mummies from the 16
       <sup>th</sup> century. Previous studies have not found any temporal signal in these data using different approaches, despite the inclusion of ancient sequences. Our estimates of log marginal likelihoods were consistent with a lack of temporal signal, with a log Bayes factor of - 101.51 for 
       <em>M</em>
       <sub>het</sub> relative to 
       <em>M</em>
       <sub>iso</sub>.
      </p>
      <p id="p-38">The dog mitochondrial genome data contained samples from up to 36,000 years before the present. BETS detected temporal signal in these data, with a log Bayes factor of 38.77 for 
       <em>M</em>
       <sub>het</sub> relative to 
       <em>M</em>
       <sub>iso</sub>; this result is consistent with that of a date-randomization test in a previous study (
       <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tong et al. 2018</a>). The estimated substitution rate for these data using the best model had a mean of 1.08×10
       <sup>−7</sup> subs/site/year (95% HPD: 7.49×10
       <sup>−8</sup> to 1.52×10
       <sup>−7</sup>).
      </p>
     </div>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Discussion</h2>
    <p id="p-39">We have proposed BETS, a method that explicitly assesses the statistical support for including sequence sampling times in a Bayesian framework. It is a test of the strength of the temporal signal in a data set, which is an important prerequisite for obtaining reliable inferences in phylodynamic analyses. BETS considers the model ensemble, such that the method can detect temporal signal using models that account for substitution rate variation among lineages. The results of our analyses demonstrate that our method is effective in a range of conditions, including when the substitution rate is low or when the sampling window represents a small portion of the timespan of the tree.</p>
    <p id="p-40">BETS does not require date permutations, which differentiates it from the widely used date-randomization test for temporal structure. Date-randomization tests address the question of whether a particular association between sequences and sampling times produces estimates different from those obtained from data sets with permuted sampling times (
     <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Duchene et al. 2015</a>; 
     <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Murray et al. 2015</a>). However, such an approach is not a formal test of temporal signal in the data because the permutations do not necessarily constitute an appropriate null model. In contrast, our method does not require permutations and so has the benefit of being robust to using a small number of sampling times.
    </p>
    <p id="p-41">Accurate calculations of marginal likelihoods are essential for BETS. In our simulation study, we found that GSS and NS correctly assessed the presence and absence of temporal signal in the data under most conditions. The correct clock model was also identified, although in a few instances NS preferred an overparameterized model. Conceivably, using different marginal-likelihood estimators might affect the actual model selected. 
     <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Murray et al. (2015)</a> also employed a Bayesian model-testing approach using the AICM to assess temporal signal. In their study, the AICM performed well in simulations, but failed to detect temporal signal in empirical data. We attribute this finding to the low accuracy of AICM relative to path-sampling methods (
     <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Baele et al. 2012</a>, 
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">2013</a>), and suggest careful consideration of the marginal-likelihood estimator for tests of temporal signal.
    </p>
    <p id="p-42">A key advantage of BETS is that the complete model is considered, unlike in simpler data-exploration methods such as root-to-tip regression. Specifically, root-to-tip regression is a visual tool for uncovering problems with data quality and to inspect clocklike behaviour, but the absence of appropriate statistics means that there is no clear way of determining whether the data contains temporal information. Consider the regressions in 
     <a id="xref-fig-5-2" class="xref-fig" href="#F5">figure 5</a> for data with a high substitution rate and narrow sampling window. Even when among-lineage rate variation is low (
     <em>σ</em>=0.1), the data points form a cloud, with a low 
     <em>R</em>
     <sup>2</sup> of 0.09. However, the apparent ‘noise’ around the regression line is probably the result of stochasticity in sequence evolution and of the narrow sampling window relative to the age of the root of the tree. In fact, for this particular data set the model with the highest log marginal likelihood is the strict clock with correct sampling times.
    </p>
    <p id="p-43">In all of our analyses, we ensured that the priors for different models and configurations of sampling times were identical because, as with all Bayesian analyses, model comparison using marginal likelihoods can depend on the choice of prior (
     <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Oaks et al. 2019</a>). For example, the tree prior can affect inferences of temporal signal, as it is part of the full model specification. Here we used an exponential-growth coalescent tree prior, which closely matches the demographic dynamics of the birth-death process under which the data were simulated. The effect of using an inappropriate tree prior on tests of temporal signal requires further investigation, but previous studies have suggested that there is only a small impact on estimates of rates and times if the sequence data are informative (
     <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ritchie et al. 2017</a>; 
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Möller et al. 2018</a>).
    </p>
    <p id="p-44">An interesting finding is that statistical support for isochronous sampling times in truly isochronous data is lower than that for the correct sampling times in truly heterochronous data. This can potentially lead to an increased risk of incorrectly concluding the presence of temporal signal, but we only found this to be a problem in a small number of cases. In particular, in isochronous data simulated with a low substitution rate, and with very few variable sites, the best models were sometimes those that included sampling times, albeit with very low log Bayes factors (e.g., 
     <a id="xref-fig-9-2" class="xref-fig" href="#F9">supplementary fig. S2</a>, Supplementary Material online). This probably occurs because stochastic error associated with a small amount of evolution leads to low power for model selection.
    </p>
    <p id="p-45">Permuting sampling times led to poor model fit, as expected. This procedure requires substantial computing requirements, depending on the number of permutations that are performed, and we find that such date permutations are of limited value for model testing when the data are highly informative (e.g., 
     <a id="xref-fig-1-9" class="xref-fig" href="#F1">figs. 1</a> and 
     <a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>). However, in data sets with very low information content, such as those that were produced by simulation with a low substitution rate here, conducting a small number of date permutations might offer a conservative approach to determining whether model fit and parameter estimates are driven by a particular set of sampling times, as one would expect in the presence of temporal signal.
    </p>
    <p id="p-46">The nature of the BETS approach means that every parameter in the model has a prior probability, including the substitution rate. Because substitution rates and times are nonidentifiable, it is conceivable that an informative prior on the rate or on the age of an internal node might have a stronger effect than the sampling times on the posterior, for example if the samples span a very short window of time. Such analyses with informative substitution rate priors effectively include several simultaneous sources of calibrating information (i.e., sampling times, internal nodes, and an informative rate prior). Using sampling times in addition to other sources of calibration information might still be warranted if it improves the fit of the model, which can be tested using our proposed method.</p>
    <p id="p-47">Analyses with multiple calibrations can also allow uncertainty in sequence sampling times, especially in data sets that include ancient DNA, where sampling times can be treated as parameters in the model (
     <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shapiro et al. 2011</a>). BETS provides a coherent approach to assess temporal structure in these circumstances, unlike date-randomization tests that typically use point values for sampling times. In fact, BETS can be used as a means to validate whether a sample is modern or ancient.
    </p>
    <p id="p-48">In general, the uptake of Bayesian model testing in phylogenetics has great potential for improving our confidence in estimates of substitution rates and timescales. The test that we have proposed here, BETS, provides a coherent and intuitive framework to test for temporal information in the data.</p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-12" class="subsection">
     <h3>Simulations</h3>
     <p id="p-49">We simulated phylogenetic trees under a stochastic birth-death process using MASTER v6.1 (
      <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vaughan and Drummond 2013</a>), by specifying birth rate 
      <em>λ</em>=1.5, death rate 
      <em>μ</em>=0.5, and sampling rate 
      <em>ϕ</em>=0.5. This corresponds to an exponentially growing infectious outbreak with reproductive number 
      <em>R</em>
      <sub>0</sub>=1.5 and a wide sampling window. We set the simulation time to 5 units of time, which corresponds to the time of origin of the process. For isochronous trees, we used similar settings, but instead of using the sampling rate, we sampled each tip with probability
      <em>ρ</em>=0.5 when the process was stopped after 5 units of time (i.e. 
      <em>μ</em>=1 and 
      <em>ϕ</em>=0). Some of our analyses consisted of artificially specifying sampling times for isochronous trees, which we set to those that we would have obtained from a birth-death process with 
      <em>μ</em>=0.5 and 
      <em>ϕ</em>=0.5.
     </p>
     <p id="p-50">In a second set of simulations of heterochronous trees, we generated trees with a narrow sampling window. We specified two intervals for 
      <em>μ</em> and
      <em>ϕ</em>. The first interval spanned 4.5 units of time with 
      <em>μ</em>=1.0 and 
      <em>ϕ</em>=0, and the second interval 0.5 units of time with 
      <em>μ</em>=0.1 and 
      <em>ϕ</em>=0.9. As a result, the process still had a constant become uninfectious rate (
      <em>μ</em>+
      <em>ϕ</em>), but samples were only collected in the second interval. The high sampling rate in the second interval resulted in trees with similar numbers of tips to those with a wide sampling window, but where their ages only spanned 0.5 units of time.
     </p>
     <p id="p-51">We only considered the simulated trees that contained between 90 and 110 tips. The trees generated in MASTER are chronograms (with branch lengths in units of time), so we simulated substitution rates to generate phylograms (with branch lengths in units of subs/site). To do this we specified the uncorrelated lognormal relaxed clock with a mean rate of 5×10
      <sup>−3</sup> or 10
      <sup>−5</sup> subs/site/unit time and a standard deviation 
      <em>σ</em> of 0 (corresponding to a strict clock), 0.1, 0.5, or 1. We simulated sequence evolution along these phylograms under the HKY nucleotide substitution model (
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hasegawa et al. 1985</a>). We added among-site rate variation using a discretized gamma distribution (
      <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Yang 1994</a>, 
      <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">1996</a>) using Phangorn v2.5 (
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schliep 2011</a>) to generate sequence alignments of 10,000 nucleotides. We set the transition-to-transversion ratio of the HKY model to 10 and the shape of the gamma distribution to 1, which is similar to estimates of these parameters in influenza viruses (
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Duchene et al. 2014</a>; 
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hedge and Wilson 2014</a>). For each simulation scenario we generated 10 sequence alignments.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Estimation of Marginal Likelihoods Using Nested Sampling</h3>
     <p id="p-52">We analysed the data in BEAST 2.5 using the matching substitution model, the exponential-growth coalescent tree prior, the strict clock or relaxed clock, and different configurations of sampling times. We chose the exponential-growth coalescent tree prior, instead of the birth-death tree prior, because it is conditioned on the samples instead of assuming a sampling process; this ensures that the marginal likelihoods for isochronous and heterochronous trees are comparable.</p>
     <p id="p-53">We specified proper priors on all parameters, which is essential for accurate estimation of marginal likelihoods (
      <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Baele et al., 2013</a>). In our heterochronous analyses the prior on the substitution rate had a uniform distribution bounded between 0 and 1. We made this arbitrary choice to set a somewhat uninformative prior and because the default prior in BEAST 2.5 is a uniform distribution between 0 and infinity, which is improper. Owing to the nonidentifiability of substitution rates and times, neither can be inferred in the absence of calibrating information, so in our isochronous analyses we fixed the value of the substitution rate to 1. The initial NS chain length was chosen so as to draw 20,000 samples, with 20,000 steps, 32 particles, and a subchain length of 5,000 (note that NS is not equivalent to standard MCMC, nor is the definition of an iteration/step). The chain length and its accompanying sampling frequency were adjusted to obtain effective sample sizes for key parameters of at least 200 (computed in the NS output in BEAST 2.5). Examples of MASTER files and BEAST input files for NS are available online (supplementary data, Supplementary Material online).
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Estimation of Marginal Likelihoods Using Generalized Stepping-Stone Sampling</h3>
     <p id="p-54">We used BEAST 1.10 with the same model specifications and priors as in BEAST2, except for the prior on the substitution rate, for which we used the approximate continuous-time Markov chain reference prior (
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ferreira and Suchard 2008</a>). Because our simulation analyses of GSS and NS differ in this prior, the marginal-likelihood estimates are not directly comparable, so for each simulation we report log Bayes factors of competing models instead of the log marginal likelihoods. The GSS implementation in BEAST 1.10 has two different working priors for the tree generative process: a matching tree prior and a product of exponentials. The latter approach is the most generally applicable and is the one that we used here (
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baele et al. 2016</a>).
     </p>
     <p id="p-55">We used an initial MCMC chain length of 5×10
      <sup>7</sup> steps sampling every 5000 steps. After discarding 10% of the samples obtained, the remaining samples were used to construct the working distributions for the GSS analysis. These comprised 100 path steps distributed according to quantiles from a 
      <em>β</em> distribution with 
      <em>α</em>=0.3, with each of the 101 resulting power posterior inferences running for 5×10
      <sup>5</sup> iterations. We assessed sufficient sampling for the initial MCMC analysis by verifying that the effective sample sizes for key parameters were at least 200 in Coda v0.19 (
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Plummer et al. 2006</a>). If this condition was not met, we doubled the length of the MCMC and reduced sampling frequency accordingly. Examples of MASTER files and BEAST input files for GSS are available online (supplementary data, Supplementary Material online).
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Analyses of Empirical Data Sets</h3>
     <p id="p-56">We downloaded sequence alignments from their original publications (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>): complete genomes of 
      <em>A/H1N1 influenza virus</em> (
      <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Hedge et al. 2013</a>), whole genome sequences of 
      <em>B. pertussis</em> (
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bart et al. 2014</a>; 
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Duchene et al. 2016</a>), RdRP sequences of coronaviruses (
      <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Wertheim et al. 2013</a>), complete genomes of 
      <em>Hepatitis B virus</em> (
      <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Patterson Ross et al. 2018</a>), and dog mitochondrial genomes (
      <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Thalmann et al. 2013</a>). The data and BEAST input files are available in the Supplementary Material online.
     </p>
     <p id="p-57">Briefly, we used similar settings as in our simulations to estimate marginal likelihoods using GSS. For sequence sampling times we considered the correct sampling times, no sampling times (i.e., isochronous), and permuted sampling times. We also specified tree priors as follows: an exponential-growth coalescent for the 
      <em>A/H1N2 influenza virus, Bordetella pertussis</em>, coronaviruses, and 
      <em>Hepatitis B virus</em> data sets, and a constant-size coalescent for the dog mitochondrial genomes as used by 
      <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Tong et al. (2018)</a>. We again chose the HKY+Γ substitution model, except in the analysis of 
      <em>Hepatitis B virus</em> data, for which we used the GTR+Γ model (Tavaré 1986), and in the analysis of the dog data set for which we used the SRD06 substitution model (
      <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shapiro et al. 2006</a>) for coding regions and the GTR+Γ for noncoding regions.
     </p>
    </div>
   </div>
   <div class="section" id="sec-17">
    <h2 class="">Supplementary Material</h2>
    <p id="p-58">Supplementary data are available online.</p>
   </div>
   <div class="section" id="sec-18">
    <h2 class="">Funding</h2>
    <p id="p-59">SD was supported by an Australian Research Council Discovery Early Career Researcher Award (DE190100805) and an Australian National Health and Medical Research Council grant (APP1157586). PL acknowledges funding from the European Research Council under the European Union’s Horizon 2020 research and innovation programme (grant agreement no. 725422-ReservoirDOCS) and the Research Foundation -- Flanders (‘Fonds voor Wetenschappelijk Onderzoek -- Vlaanderen’, G066215N, G0D5117N and G0B9317N). SYWH was funded by the Australian Research Council (FT160100167). GB acknowledges support from the Interne Fondsen KU Leuven / Internal Funds KU Leuven under grant agreement C14/18/094. VD was supported by contract HHSN272201400006C from the National Institute of Allergy and Infectious Diseases, National Institutes of Health, U.S. Department of Health and Human Services, USA.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-60">Pending.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-3">
       <a href="https://github.com/sebastianduchene/marginal_likelihood_tempest">https://github.com/sebastianduchene/marginal_likelihood_tempest</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-book no-rev-xref" id="cit-810697v1.1">
       <div class="cit-metadata">
        <ol class="duplicate">
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Chen</span>
           <span class="cit-name-given-names">M.-H.</span>
          </span>, 
         </li>
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Kuo</span>
           <span class="cit-name-given-names">L.</span>
          </span>, 
         </li>
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Lewis</span>
           <span class="cit-name-given-names">P.O.</span>
          </span>
         </li>
        </ol>
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-chapter-title">Bayesian model selection in phylogenetics and genealogy-based population genetics</span>. In: 
         <span class="cit-ed">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">M.-H.</span>
         </span>, 
         <span class="cit-ed">
          <span class="cit-name-surname">Kuo</span>
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-ed">
          <span class="cit-name-surname">Lewis</span>
          <span class="cit-name-given-names">P.O.</span>
         </span>, editors. 
         <span class="cit-source">Bayesian Phylogenetics, Methods, Algorithms, and Applications</span>. 
         <span class="cit-publ-loc">Boca Raton, Florida</span>: 
         <span class="cit-publ-name">CPC Press</span>. p. 
         <span class="cit-fpage">59</span>–
         <span class="cit-lpage">93</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.2" data-doi="10.1093/molbev/mss084">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alekseyenko</span>
          <span class="cit-name-given-names">A. V.</span>
         </span>
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">29</span>:
         <span class="cit-fpage">2157</span>–
         <span class="cit-lpage">2167</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss084%26rft_id%253Dinfo%253Apmid%252F22403239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/mss084&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22403239&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000308851600009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.3" data-doi="10.1093/sysbio/syv083">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Genealogical working distributions for Bayesian model testing with phylogenetic uncertainty</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">65</span>:
         <span class="cit-fpage">250</span>–
         <span class="cit-lpage">264</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyv083%26rft_id%253Dinfo%253Apmid%252F26526428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/sysbio/syv083&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26526428&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.4" data-doi="10.1093/molbev/mss243">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">W.L.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">A.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Accurate model selection of relaxed molecular clocks in bayesian phylogenetics</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">30</span>:
         <span class="cit-fpage">239</span>–
         <span class="cit-lpage">243</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss243%26rft_id%253Dinfo%253Apmid%252F23090976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/mss243&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23090976&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000314122000001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.5" data-doi="10.1128/mBio.01074-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bart</span>
          <span class="cit-name-given-names">M.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Harris</span>
          <span class="cit-name-given-names">S.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Advani</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Arakawa</span>
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bottero</span>
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bouchez</span>
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cassiday</span>
          <span class="cit-name-given-names">P.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chiang</span>
          <span class="cit-name-given-names">C.-S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dalby</span>
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fry</span>
          <span class="cit-name-given-names">N.K.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Global population structure and evolution of Bordetella pertussis and their relationship with vaccination</span>. 
         <abbr class="cit-jnl-abbrev">MBio</abbr>. 
         <span class="cit-vol">5</span>:
         <span class="cit-fpage">e01074</span>–
         <span class="cit-lpage">14</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D5%26rft.spage%253De01074%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01074-14%26rft_id%253Dinfo%253Apmid%252F24757216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1128/mBio.01074-14&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24757216&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.6" data-doi="10.1016/j.tree.2015.03.009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Biek</span>
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>
          <span class="cit-name-given-names">O.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lloyd-Smith</span>
          <span class="cit-name-given-names">J.O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Didelot</span>
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Measurably evolving pathogens in the genomic era</span>. 
         <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. 
         <span class="cit-vol">30</span>:
         <span class="cit-fpage">306</span>–
         <span class="cit-lpage">313</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol%26rft.volume%253D30%26rft.spage%253D306%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2015.03.009%26rft_id%253Dinfo%253Apmid%252F25887947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tree.2015.03.009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25887947&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bouckaert</span>
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vaughan</span>
          <span class="cit-name-given-names">T.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barido-Sottani</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fourment</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gavryushkina</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Heled</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jones</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuhnert</span>
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Maio</span>
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-article-title">BEAST 2.5: An Advanced Software Platform for Bayesian Evolutionary Analysis</span>. 
         <abbr class="cit-jnl-abbrev">PLOS Comput. Biol</abbr>. 
         <span class="cit-vol">15</span>:
         <span class="cit-fpage">e1006650</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol%26rft.volume%253D15%26rft.spage%253De1006650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.8" data-doi="10.1111/brv.12390">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bromham</span>
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchêne</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hua</span>
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ritchie</span>
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchêne</span>
          <span class="cit-name-given-names">D.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">Bayesian molecular dating: Opening up the black box</span>. 
         <abbr class="cit-jnl-abbrev">Biol. Rev</abbr>. 
         <span class="cit-vol">93</span>:
         <span class="cit-fpage">1165</span>–
         <span class="cit-lpage">1191</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev%26rft.volume%253D93%26rft.spage%253D1165%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbrv.12390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/brv.12390&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.9" data-doi="10.1371/journal.pbio.0040088">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">A.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Phillips</span>
          <span class="cit-name-given-names">M.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">Relaxed phylogenetics and dating with confidence</span>. 
         <abbr class="cit-jnl-abbrev">PLOS Biol</abbr>. 
         <span class="cit-vol">4</span>:
         <span class="cit-fpage">e88</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DRelaxed%2Bphylogenetics%2Band%2Bdating%2Bwith%2Bconfidence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040088%26rft_id%253Dinfo%253Apmid%252F16683862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040088&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16683862&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.10" data-doi="10.1016/S0169-5347(03)00216-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">A.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>
          <span class="cit-name-given-names">O.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Forsberg</span>
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rodrigo</span>
          <span class="cit-name-given-names">A.G.</span>
         </span>
         <span class="cit-pub-date">2003</span>. 
         <span class="cit-article-title">Measurably evolving populations</span>. 
         <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. 
         <span class="cit-vol">18</span>:
         <span class="cit-fpage">481</span>–
         <span class="cit-lpage">488</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol%26rft.volume%253D18%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252803%252900216-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0169-5347(03)00216-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000185311800014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-810697v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">D.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Geoghegan</span>
          <span class="cit-name-given-names">J.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dyson</span>
          <span class="cit-name-given-names">Z.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hawkey</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holt</span>
          <span class="cit-name-given-names">K.E.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">Inferring demographic parameters in bacterial genomic data using Bayesian and hybrid phylogenetic methods</span>. 
         <abbr class="cit-jnl-abbrev">BMC Evol. Biol</abbr>. 
         <span class="cit-vol">18</span>:
         <span class="cit-fpage">95</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol%26rft.volume%253D18%26rft.spage%253D95%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.12" data-doi="10.1093/molbev/msv056">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">D.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">E.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">The performance of the daterandomization test in phylogenetic analyses of time-structured virus data</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">32</span>:
         <span class="cit-fpage">1895</span>–
         <span class="cit-lpage">1906</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv056%26rft_id%253Dinfo%253Apmid%252F25771196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msv056&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25771196&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.13" data-doi="10.1098/rspb.2014.0732">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">E.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Analyses of evolutionary dynamics in viruses are hindered by a time-dependent bias in rate estimates</span>. 
         <abbr class="cit-jnl-abbrev">Proc. R. Soc. London B</abbr>. 
         <span class="cit-vol">281</span>:
         <span class="cit-fpage">20140732</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BLondon%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.0732%26rft_id%253Dinfo%253Apmid%252F24850916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1098/rspb.2014.0732&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24850916&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holt</span>
          <span class="cit-name-given-names">K.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weill</span>
          <span class="cit-name-given-names">F.-X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Le Hello</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hawkey</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Edwards</span>
          <span class="cit-name-given-names">D.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fourment</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">E.C.</span>
         </span>
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Genome-scale rates of evolutionary change in bacteria. Microb</span>. 
         <abbr class="cit-jnl-abbrev">Genomics</abbr>. 
         <span class="cit-vol">2</span>:
         <span class="cit-fpage">e000094</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D2%26rft.spage%253De000094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.15" data-doi="10.1093/molbev/msq224">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">M.-H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuo</span>
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewis</span>
          <span class="cit-name-given-names">P.O.</span>
         </span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Choosing among partition models in Bayesian phylogenetics</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">28</span>:
         <span class="cit-fpage">523</span>–
         <span class="cit-lpage">532</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq224%26rft_id%253Dinfo%253Apmid%252F20801907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msq224&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20801907&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000285418600052&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ferreira</span>
          <span class="cit-name-given-names">M.A.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-article-title">Bayesian analysis of elapsed times in continuous-time Markov chains</span>. 
         <abbr class="cit-jnl-abbrev">Can. J. Stat</abbr>. 
         <span class="cit-vol">36</span>:
         <span class="cit-fpage">355</span>–
         <span class="cit-lpage">368</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BStat%26rft.volume%253D36%26rft.spage%253D355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fourment</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Magee</span>
          <span class="cit-name-given-names">A.F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Whidden</span>
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bilge</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Matsen</span>
          <span class="cit-name-given-names">I. V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Frederick</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Minin</span>
          <span class="cit-name-given-names">V.N.</span>
         </span>
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-article-title">19 dubious ways to compute the marginal likelihood of a phylogenetic tree topology</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>.:
         <span class="cit-vol">syz046</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.18" data-doi="10.1093/sysbio/syq010">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Guindon</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dufayard</span>
          <span class="cit-name-given-names">J.-F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lefort</span>
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anisimova</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hordijk</span>
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gascuel</span>
          <span class="cit-name-given-names">O.</span>
         </span>
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">New algorithms and methods to estimate maximum likelihood phylogenies: assessing the performance of PhyML 3.0</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">59</span>:
         <span class="cit-fpage">307</span>–
         <span class="cit-lpage">321</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20525638&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000276528300006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.19" data-doi="10.1007/BF02101694">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hasegawa</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kishino</span>
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yano</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-pub-date">1985</span>. 
         <span class="cit-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA</span>. 
         <abbr class="cit-jnl-abbrev">J. Mol. Evol</abbr>. 
         <span class="cit-vol">22</span>:
         <span class="cit-fpage">160</span>–
         <span class="cit-lpage">174</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHasegawa%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D174%26rft.atitle%253DDating%2Bof%2Bthe%2Bhuman-ape%2Bsplitting%2Bby%2Ba%2Bmolecular%2Bclock%2Bof%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02101694%26rft_id%253Dinfo%253Apmid%252F3934395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/BF02101694&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=3934395&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1985AWB1600007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.20" data-doi="10.1098/rsbl.2013.0331">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hedge</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lycett</span>
          <span class="cit-name-given-names">S.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Real-time characterization of the molecular epidemiology of an influenza pandemic</span>. 
         <abbr class="cit-jnl-abbrev">Biol. Lett</abbr>. 
         <span class="cit-vol">9</span>:
         <span class="cit-fpage">20130331</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2013.0331%26rft_id%253Dinfo%253Apmid%252F23883574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1098/rsbl.2013.0331&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23883574&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.21" data-doi="10.1128/mBio.02158-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hedge</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilson</span>
          <span class="cit-name-given-names">D.J.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Bacterial phylogenetic reconstruction from whole genomes is robust to recombination but demographic inference is not</span>. 
         <abbr class="cit-jnl-abbrev">MBio</abbr>. 
         <span class="cit-vol">5</span>:
         <span class="cit-fpage">e02158</span>–
         <span class="cit-lpage">14</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D5%26rft.spage%253De02158%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.02158-14%26rft_id%253Dinfo%253Apmid%252F25425237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1128/mBio.02158-14&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25425237&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hipsley</span>
          <span class="cit-name-given-names">C.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Müller</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Beyond fossil calibrations: realities of molecular clock practices in evolutionary biology</span>. 
         <abbr class="cit-jnl-abbrev">Front. Genet</abbr>. 
         <span class="cit-vol">5</span>:
         <span class="cit-fpage">138</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet%26rft.volume%253D5%26rft.spage%253D138%26rft_id%253Dinfo%253Apmid%252F24904638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=24904638&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.23" data-doi="10.1111/mec.12953">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Molecular-clock methods for estimating evolutionary rates and time scales</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. 
         <span class="cit-vol">23</span>:
         <span class="cit-fpage">5947</span>–
         <span class="cit-lpage">5975</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft.volume%253D23%26rft.spage%253D5947%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/mec.12953&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000346771900005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.24" data-doi="10.2307/2291091">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kass</span>
          <span class="cit-name-given-names">R.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Raftery</span>
          <span class="cit-name-given-names">A.E.</span>
         </span>
         <span class="cit-pub-date">1995</span>. 
         <span class="cit-article-title">Bayes factors</span>. 
         <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc</abbr>. 
         <span class="cit-vol">90</span>:
         <span class="cit-fpage">773</span>–
         <span class="cit-lpage">795</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291091%26rft_id%253Dinfo%253Apmid%252F10783804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2307/2291091&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10783804&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1995RA10400045&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.25" data-doi="10.1126/science.288.5472.1789">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Korber</span>
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muldoon</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Theiler</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gupta</span>
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lapedes</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hahn</span>
          <span class="cit-name-given-names">B.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolinsky</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhattacharya</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-pub-date">2000</span>. 
         <span class="cit-article-title">Timing the ancestor of the HIV-1 pandemic strains</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-vol">288</span>:
         <span class="cit-fpage">1789</span>–
         <span class="cit-lpage">1796</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKorber%26rft.auinit1%253DB.%26rft.volume%253D288%26rft.issue%253D5472%26rft.spage%253D1789%26rft.epage%253D1796%26rft.atitle%253DTiming%2Bthe%2BAncestor%2Bof%2Bthe%2BHIV-1%2BPandemic%2BStrains%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5472.1789%26rft_id%253Dinfo%253Apmid%252F10846155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODgvNTQ3Mi8xNzg5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjEvODEwNjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.26" data-doi="10.1080/10635150500433722">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lartillot</span>
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Philippe</span>
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">Computing Bayes factors using thermodynamic integration</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">55</span>:
         <span class="cit-fpage">195</span>–
         <span class="cit-lpage">207</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DLartillot%26rft.auinit1%253DN.%26rft.volume%253D55%26rft.issue%253D2%26rft.spage%253D195%26rft.epage%253D207%26rft.atitle%253DComputing%2BBayes%2Bfactors%2Busing%2Bthermodynamic%2Bintegration.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150500433722%26rft_id%253Dinfo%253Apmid%252F16522570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/10635150500433722&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16522570&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Maturana</span>
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brewer</span>
          <span class="cit-name-given-names">B.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Klaere</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bouckaert</span>
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-article-title">Model selection and parameter inference in phylogenetics using Nested Sampling</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">68</span>:
         <span class="cit-fpage">219</span>–
         <span class="cit-lpage">233</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft.volume%253D68%26rft.spage%253D219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.28" data-doi="10.1073/pnas.1713314115">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Möller</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">du Plessis</span>
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stadler</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">Impact of the tree prior on estimating clock rates during epidemic outbreaks</span>. 
         <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr>. 
         <span class="cit-vol">115</span>:
         <span class="cit-fpage">4200</span>–
         <span class="cit-lpage">4205</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1713314115%26rft_id%253Dinfo%253Apmid%252F29610334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzE2LzQyMDAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yMS84MTA2OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Murray</span>
          <span class="cit-name-given-names">G.G.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Harrison</span>
          <span class="cit-name-given-names">E.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Paterson</span>
          <span class="cit-name-given-names">G.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mather</span>
          <span class="cit-name-given-names">A.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Harris</span>
          <span class="cit-name-given-names">S.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Welch</span>
          <span class="cit-name-given-names">J.J.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">The effect of genetic structure on molecular dating and tests for temporal signal</span>. 
         <abbr class="cit-jnl-abbrev">Methods Ecol. Evol</abbr>. 
         <span class="cit-vol">7</span>:
         <span class="cit-fpage">80</span>–
         <span class="cit-lpage">89</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol%26rft.volume%253D7%26rft.spage%253D80%26rft_id%253Dinfo%253Apmid%252F27110344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=27110344&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nascimento</span>
          <span class="cit-name-given-names">F.F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">dos Reis</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-article-title">A biologist’s guide to Bayesian phylogenetic analysis</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol</abbr>. 
         <span class="cit-vol">1</span>:
         <span class="cit-fpage">1446</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol%26rft.volume%253D1%26rft.spage%253D1446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Newton</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Raftery</span>
          <span class="cit-name-given-names">A.E.</span>
         </span>
         <span class="cit-pub-date">1994</span>. 
         <span class="cit-article-title">Approximate Bayesian inference with the weighted likelihood bootstrap</span>. 
         <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Ser. B</abbr>. 
         <span class="cit-vol">56</span>:
         <span class="cit-fpage">3</span>–
         <span class="cit-lpage">26</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSer.%2BB%26rft.volume%253D56%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Oaks</span>
          <span class="cit-name-given-names">J.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cobb</span>
          <span class="cit-name-given-names">K.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Minin</span>
          <span class="cit-name-given-names">V.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leaché</span>
          <span class="cit-name-given-names">A.D.</span>
         </span>
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-article-title">Marginal likelihoods in phylogenetics: a review of methods and applications</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">68</span>:
         <span class="cit-fpage">681</span>–
         <span class="cit-lpage">697</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft.volume%253D68%26rft.spage%253D681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Patterson Ross</span>
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Klunk</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fornaciari</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Giuffra</span>
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchene</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duggan</span>
          <span class="cit-name-given-names">A.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poinar</span>
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Douglas</span>
          <span class="cit-name-given-names">M.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eden</span>
          <span class="cit-name-given-names">J.-S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">E.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poinar</span>
          <span class="cit-name-given-names">H.N.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">The paradox of HBV evolution as revealed from a 16thcentury mummy</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. 
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">e1006887</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1006887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Plummer</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Best</span>
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cowles</span>
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vines</span>
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">CODA: Convergence diagnosis and output analysis for MCMC</span>. 
         <abbr class="cit-jnl-abbrev">R News</abbr>. 
         <span class="cit-vol">6</span>:
         <span class="cit-fpage">7</span>–
         <span class="cit-lpage">11</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-book" id="cit-810697v1.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Raftery</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Newton</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Satagopan</span>
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Krivitsky</span>
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-chapter-title">Estimating the integrated likelihood via posterior simulation using the harmonic mean identity</span>. 
         <span class="cit-source">Bayesian Statistics</span>. 
         <span class="cit-publ-name">Oxford University Press</span>. p. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">45</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.36" data-doi="10.1093/ve/vew007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lam</span>
          <span class="cit-name-given-names">T.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carvalho</span>
          <span class="cit-name-given-names">L.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>
          <span class="cit-name-given-names">O.G.</span>
         </span>
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. 
         <abbr class="cit-jnl-abbrev">Virus Evol</abbr>. 
         <span class="cit-vol">2</span>:
         <span class="cit-fpage">vew007</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvew007%26rft_id%253Dinfo%253Apmid%252F27774300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/ve/vew007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27774300&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.37" data-doi="10.1093/molbev/msn234">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ramsden</span>
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">E.C.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Charleston</span>
          <span class="cit-name-given-names">M.A.A.</span>
         </span>
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Hantavirus evolution in relation to its rodent and insectivore hosts: no evidence for codivergence</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">26</span>:
         <span class="cit-fpage">143</span>–
         <span class="cit-lpage">153</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn234%26rft_id%253Dinfo%253Apmid%252F18922760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msn234&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18922760&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000261681900014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.38" data-doi="10.1111/mec.13586">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rieux</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Balloux</span>
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Inferences from tip-calibrated phylogenies: a review and a practical guide</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. 
         <span class="cit-vol">25</span>:
         <span class="cit-fpage">1911</span>–
         <span class="cit-lpage">1924</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft.volume%253D25%26rft.spage%253D1911%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13586%26rft_id%253Dinfo%253Apmid%252F26880113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/mec.13586&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26880113&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ritchie</span>
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lo</span>
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-article-title">The impact of the tree prior on molecular dating of data sets containing a mixture of inter-and intraspecies sampling</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">66</span>:
         <span class="cit-fpage">413</span>–
         <span class="cit-lpage">425</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft.volume%253D66%26rft.spage%253D413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.40" data-doi="10.1093/bioinformatics/btq706">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schliep</span>
          <span class="cit-name-given-names">K.P.</span>
         </span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">phangorn: Phylogenetic analysis in R</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. 
         <span class="cit-vol">27</span>:
         <span class="cit-fpage">592</span>–
         <span class="cit-lpage">593</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq706%26rft_id%253Dinfo%253Apmid%252F21169378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq706&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21169378&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000287246000025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.41" data-doi="10.1093/molbev/msq262">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shapiro</span>
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">A.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>
          <span class="cit-name-given-names">O.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">A Bayesian phylogenetic method to estimate unknown sequence ages</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">28</span>:
         <span class="cit-fpage">879</span>–
         <span class="cit-lpage">887</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq262%26rft_id%253Dinfo%253Apmid%252F20889726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msq262&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20889726&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000287112300002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.42" data-doi="10.1093/molbev/msj021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shapiro</span>
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">A.J.</span>
         </span>
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">23</span>:
         <span class="cit-fpage">7</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj021%26rft_id%253Dinfo%253Apmid%252F16177232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msj021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16177232&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000233843900003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.43" data-doi="10.1214/06-BA127">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Skilling</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">Nested sampling for general Bayesian computation</span>. 
         <abbr class="cit-jnl-abbrev">Bayesian Anal</abbr>. 
         <span class="cit-vol">1</span>:
         <span class="cit-fpage">833</span>–
         <span class="cit-lpage">859</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal%26rft.volume%253D1%26rft.spage%253D833%26rft_id%253Dinfo%253Adoi%252F10.1214%252F06-BA127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1214/06-BA127&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000207447100012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.44">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ayres</span>
          <span class="cit-name-given-names">D.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">A.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10. Virus Evol. 4:vey016. Tavaré S. 1986. Some probabilistic and statistical problems in the analysis of DNA sequences</span>. 
         <abbr class="cit-jnl-abbrev">Lect. Math. Life Sci</abbr>. 
         <span class="cit-vol">17</span>:
         <span class="cit-fpage">57</span>–
         <span class="cit-lpage">86</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLect.%2BMath.%2BLife%2BSci%26rft.volume%253D17%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.45" data-doi="10.1126/science.1243650">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thalmann</span>
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shapiro</span>
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cui</span>
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schuenemann</span>
          <span class="cit-name-given-names">V.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sawyer</span>
          <span class="cit-name-given-names">S.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greenfield</span>
          <span class="cit-name-given-names">D.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Germonpré</span>
          <span class="cit-name-given-names">M.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sablin</span>
          <span class="cit-name-given-names">M. V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">López-Giráldez</span>
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Domingo-Roura</span>
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Complete mitochondrial genomes of ancient canids suggest a European origin of domestic dogs</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-vol">342</span>:
         <span class="cit-fpage">871</span>–
         <span class="cit-lpage">874</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThalmann%26rft.auinit1%253DO.%26rft.volume%253D342%26rft.issue%253D6160%26rft.spage%253D871%26rft.epage%253D874%26rft.atitle%253DComplete%2BMitochondrial%2BGenomes%2Bof%2BAncient%2BCanids%2BSuggest%2Ba%2BEuropean%2BOrigin%2Bof%2BDomestic%2BDogs%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1243650%26rft_id%253Dinfo%253Apmid%252F24233726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE2MC84NzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yMS84MTA2OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tong</span>
          <span class="cit-name-given-names">K.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchêne</span>
          <span class="cit-name-given-names">D.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duchêne</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Geoghegan</span>
          <span class="cit-name-given-names">J.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">S.Y.W.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">A comparison of methods for estimating substitution rates from ancient DNA sequence data</span>. 
         <abbr class="cit-jnl-abbrev">BMC Evol. Biol</abbr>. 
         <span class="cit-vol">18</span>:
         <span class="cit-fpage">70</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol%26rft.volume%253D18%26rft.spage%253D70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.47" data-doi="10.1093/ve/vev016">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Trovão</span>
          <span class="cit-name-given-names">N.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vrancken</span>
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bielejec</span>
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fargette</span>
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Host ecology determines the dispersal patterns of a plant virus</span>. 
         <abbr class="cit-jnl-abbrev">Virus Evol</abbr>. 
         <span class="cit-vol">1</span>:
         <span class="cit-fpage">vev016</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvev016%26rft_id%253Dinfo%253Apmid%252F27774287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/ve/vev016&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27774287&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.48" data-doi="10.1093/molbev/mst057">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vaughan</span>
          <span class="cit-name-given-names">T.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">A.J.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">A stochastic simulator of birth–death master equations with application to phylodynamics</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. 
         <span class="cit-vol">30</span>:
         <span class="cit-fpage">1480</span>–
         <span class="cit-lpage">1493</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst057%26rft_id%253Dinfo%253Apmid%252F23505043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/mst057&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23505043&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000318812000023&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.49" data-doi="10.1128/JVI.03273-12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wertheim</span>
          <span class="cit-name-given-names">J.O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>
          <span class="cit-name-given-names">D.K.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peiris</span>
          <span class="cit-name-given-names">J.S.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pond</span>
          <span class="cit-name-given-names">S.L.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>
          <span class="cit-name-given-names">L.L.M.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">A case for the ancient origin of coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">J. Virol</abbr>. 
         <span class="cit-vol">87</span>:
         <span class="cit-fpage">7039</span>–
         <span class="cit-lpage">7045</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03273-12%26rft_id%253Dinfo%253Apmid%252F23596293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xMi83MDM5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjEvODEwNjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.50" data-doi="10.1093/sysbio/syq085">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewis</span>
          <span class="cit-name-given-names">P.O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuo</span>
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">M.H.</span>
         </span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Improving marginal likelihood estimation for Bayesian phylogenetic model selection</span>. 
         <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. 
         <span class="cit-vol">60</span>:
         <span class="cit-fpage">150</span>–
         <span class="cit-lpage">160</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq085%26rft_id%253Dinfo%253Apmid%252F21187451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/sysbio/syq085&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21187451&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000287255100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference in text" id="ref-51">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.51" data-doi="10.1007/BF00178256">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-pub-date">1994</span>. 
         <span class="cit-article-title">Estimating the pattern of nucleotide substitution</span>. 
         <abbr class="cit-jnl-abbrev">J. Mol. Evol</abbr>. 
         <span class="cit-vol">39</span>:
         <span class="cit-fpage">105</span>–
         <span class="cit-lpage">111</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D111%26rft.atitle%253DEstimating%2Bthe%2Bpattern%2Bof%2Bnucleotide%2Bsubstitution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00178256%26rft_id%253Dinfo%253Apmid%252F8064867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/BF00178256&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=8064867&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1994NQ00800014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference in text" id="ref-52">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-810697v1.52" data-doi="10.1016/0169-5347(96)10041-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-pub-date">1996</span>. 
         <span class="cit-article-title">Among-site rate variation and its impact on phylogenetic analyses</span>. 
         <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. 
         <span class="cit-vol">11</span>:
         <span class="cit-fpage">367</span>–
         <span class="cit-lpage">372</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D367%26rft.epage%253D372%26rft.atitle%253DAmong-site%2Brate%2Bvariation%2Band%2Bits%2Bimpact%2Bon%2Bphylogenetic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252896%252910041-0%26rft_id%253Dinfo%253Apmid%252F21237881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0169-5347(96)10041-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21237881&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F21%2F810697.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1996VC82300007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference in text" id="ref-53">↵</a>
      <div class="cit ref-cit ref-book" id="cit-810697v1.53">
       <div class="cit-metadata">
        <ol class="duplicate">
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Bryson</span>
           <span class="cit-name-given-names">V.</span>
          </span>, 
         </li>
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Vogel</span>
           <span class="cit-name-given-names">H.</span>
          </span>
         </li>
        </ol>
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zuckerkandl</span>
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pauling</span>
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-pub-date">1965</span>. 
         <span class="cit-chapter-title">Evolutionary divergence and convergence in proteins</span>. In: 
         <span class="cit-ed">
          <span class="cit-name-surname">Bryson</span>
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-ed">
          <span class="cit-name-surname">Vogel</span>
          <span class="cit-name-given-names">H.</span>
         </span>, editors. 
         <span class="cit-source">Evolving Genes and Proteins</span>. 
         <span class="cit-publ-loc">New York</span>: 
         <span class="cit-publ-name">Academic press</span>. p. 
         <span class="cit-fpage">97</span>–
         <span class="cit-lpage">166</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
