<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/08/2020.04.06.026765.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.06.026765v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Noisy Pooled PCR for Virus Testing"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.06.026765"/>
  <meta name="DC.Date" content="2020-04-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Fast testing can help mitigate the coronavirus disease 2019 (COVID-19) pandemic. Despite their accuracy for single sample analysis, infectious diseases diagnostic tools, like RT-PCR, require substantial resources to test large populations. We develop a scalable approach for determining the viral status of pooled patient samples. Our approach converts group testing to a linear inverse problem, where false positives and negatives are interpreted as generated by a noisy communication channel, and a message passing algorithm estimates the illness status of patients. Numerical results reveal that our approach estimates patient illness using fewer pooled measurements than existing noisy group testing algorithms. Our approach can easily be extended to various applications, including where false negatives must be minimized. Finally, in a Utopian world we would have collaborated with RT-PCR experts; it is difficult to form such connections during a pandemic. We welcome new collaborators to reach out and help improve this work!"/>
  <meta name="DC.Contributor" content="Junan Zhu"/>
  <meta name="DC.Contributor" content="Kristina Rivera"/>
  <meta name="DC.Contributor" content="Dror Baron"/>
  <meta name="article:published_time" content="2020-04-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Noisy Pooled PCR for Virus Testing"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Fast testing can help mitigate the coronavirus disease 2019 (COVID-19) pandemic. Despite their accuracy for single sample analysis, infectious diseases diagnostic tools, like RT-PCR, require substantial resources to test large populations. We develop a scalable approach for determining the viral status of pooled patient samples. Our approach converts group testing to a linear inverse problem, where false positives and negatives are interpreted as generated by a noisy communication channel, and a message passing algorithm estimates the illness status of patients. Numerical results reveal that our approach estimates patient illness using fewer pooled measurements than existing noisy group testing algorithms. Our approach can easily be extended to various applications, including where false negatives must be minimized. Finally, in a Utopian world we would have collaborated with RT-PCR experts; it is difficult to form such connections during a pandemic. We welcome new collaborators to reach out and help improve this work!&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.06.026765v1"/>
  <meta name="citation_id" content="2020.04.06.026765v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.026765v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.026765v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.026765v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.06.026765"/>
  <meta name="citation_num_pages" content="8"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.06.026765"/>
  <meta name="citation_author" content="Junan Zhu"/>
  <meta name="citation_author_institution" content="Harvest Fund Management"/>
  <meta name="citation_author" content="Kristina Rivera"/>
  <meta name="citation_author_institution" content="North Carolina State University"/>
  <meta name="citation_author" content="Dror Baron"/>
  <meta name="citation_author_institution" content="North Carolina State University"/>
  <meta name="citation_author_email" content="barondror@ncsu.edu"/>
  <meta name="citation_reference" content="T. Nolan, R. Hands, and S. Bustin, “Quantification of mRNA using real-time rt-PCR,” Nature Prot., vol. 1, pp. 1559–82, Feb 2006."/>
  <meta name="citation_reference" content="I. Yelin, N. Aharony, E. Shaer-Tamar, A. Argoetti, E. Messer, Berenbaum, E. Shafran, A. Kuzli, N. Gandali, T. Hashimshony, Y. Mandel-Gutfreund, M. Halberthal, Y. Geffen, M. Szwarcwort-Cohen, and R. Kishony, “Evaluation of COVID-19 RT-qPCR test in multi-sample pools,” medRxiv, 2020. [Online]. Available: https://www.medrxiv.org/content/early/2020/03/27/2020.03.26.20039438"/>
  <meta name="citation_reference" content="S. L. Stramer, D. E. Krysztof, J. P. Brodsky, T. A. Fickett, B. Reynolds, R. Y. Dodd, and S. H. Kleinman, “Comparative analysis of triplex nucleic acid test assays in United States blood donors,” Transfusion, vol. 53, no. 10pt2, pp. 2525–2537, 2013."/>
  <meta name="citation_reference" content="S. R. Ayaan Hossain, Alexander C. Reis and H. M. Salis, “A Massively Parallel COVID-19 Diagnostic Assay for Simultaneous Testing of 19200 Patient Samples,” Google Docs, Mar 2020."/>
  <meta name="citation_reference" content="M. Baym, S. Kryazhimskiy, T. D. Lieberman, H. Chung, M. M. Desai, and R. Kishony, “Inexpensive multiplexed library preparation for megabase-sized genomes,” PloS one, vol. 10, no. 5, p. e0128036, 2015."/>
  <meta name="citation_reference" content="S. Rangan, “Generalized approximate message passing for estimation with random linear mixing,” in Proc. IEEE Int. Symp. Inf. Theory, 2011, pp. 2168–2172."/>
  <meta name="citation_reference" content="T. M. Cover and J. A. Thomas, Elements of Information Theory. New York, NY, USA: Wiley-Interscience, 2006."/>
  <meta name="citation_reference" content="D. L. Donoho, A. Maleki, and A. Montanari, “Message passing algorithms for compressed sensing,” Proc. Nat. Academy Sci., vol. 106, no. 45, pp. 18 914–18 919, Nov. 2009."/>
  <meta name="citation_reference" content="J. Tan, D. Carmon, and D. Baron, “Signal estimation with additive error metrics in compressed sensing,” IEEE Trans. Inf. Theory, vol. 60, no. 1, pp. 150–158, Jan. 2014."/>
  <meta name="citation_reference" content="J. Zhu and D. Baron, “Performance limits with additive error metrics in noisy multimeasurement vector problems,” IEEE Trans. Signal Proc., vol. 66, no. 20, pp. 5338–5348, Oct 2018."/>
  <meta name="citation_reference" content="L. Châtel, X. Yang, F. Cholette, H. Soudeyns, P. Sandstrom, and C. Lavigne, “Impact of pre-amplification conditions on sensitivity of the tat/rev induced limiting dilution assay,” Archives of virology, vol. 163, no. 10, pp. 2701–2710, 2018."/>
  <meta name="citation_reference" content="Y. Ma, D. Baron, and D. Needell, “Two-part reconstruction with noisy-sudocodes,” IEEE Trans. Signal Process., vol. 62, no. 23, pp. 6323–6334, Dec. 2014."/>
  <meta name="citation_reference" content="J. D. Haupt and R. Nowak, “Adaptive sensing for sparse recovery,” in Compressed Sensing: Theory and Applications. Cambridge University Press, 2012."/>
  <meta name="twitter:title" content="Noisy Pooled PCR for Virus Testing"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Fast testing can help mitigate the coronavirus disease 2019 (COVID-19) pandemic. Despite their accuracy for single sample analysis, infectious diseases diagnostic tools, like RT-PCR, require substantial resources to test large populations. We develop a scalable approach for determining the viral status of pooled patient samples. Our approach converts group testing to a linear inverse problem, where false positives and negatives are interpreted as generated by a noisy communication channel, and a message passing algorithm estimates the illness status of patients. Numerical results reveal that our approach estimates patient illness using fewer pooled measurements than existing noisy group testing algorithms. Our approach can easily be extended to various applications, including where false negatives must be minimized. Finally, in a Utopian world we would have collaborated with RT-PCR experts; it is difficult to form such connections during a pandemic. We welcome new collaborators to reach out and help improve this work!"/>
  <meta name="og-title" property="og:title" content="Noisy Pooled PCR for Virus Testing"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.026765v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Fast testing can help mitigate the coronavirus disease 2019 (COVID-19) pandemic. Despite their accuracy for single sample analysis, infectious diseases diagnostic tools, like RT-PCR, require substantial resources to test large populations. We develop a scalable approach for determining the viral status of pooled patient samples. Our approach converts group testing to a linear inverse problem, where false positives and negatives are interpreted as generated by a noisy communication channel, and a message passing algorithm estimates the illness status of patients. Numerical results reveal that our approach estimates patient illness using fewer pooled measurements than existing noisy group testing algorithms. Our approach can easily be extended to various applications, including where false negatives must be minimized. Finally, in a Utopian world we would have collaborated with RT-PCR experts; it is difficult to form such connections during a pandemic. We welcome new collaborators to reach out and help improve this work!"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-08"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Noisy Pooled PCR for Virus Testing | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1235189 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-4">Fast testing can help mitigate the coronavirus disease 2019 (COVID-19) pandemic. Despite their accuracy for single sample analysis, infectious diseases diagnostic tools, like RT-PCR, require substantial resources to test large populations. We develop a scalable approach for determining the viral status of pooled patient samples. Our approach converts group testing to a linear inverse problem, where false positives and negatives are interpreted as generated by a noisy communication channel, and a message passing algorithm estimates the illness status of patients. Numerical results reveal that our approach estimates patient illness using fewer pooled measurements than existing noisy group testing algorithms. Our approach can easily be extended to various applications, including where false negatives must be minimized. Finally, in a Utopian world we would have collaborated with RT-PCR experts; it is difficult to form such connections during a pandemic. 
     <em>We welcome new collaborators to reach out and help improve this work!</em>
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">I. I
     <span class="sc">ntroduction</span>
    </h2>
    <div id="sec-2" class="subsection">
     <h3>Motivation</h3>
     <p id="p-5">
      <em>Reverse transcription polymerase chain reaction</em> (RT-PCR) is a prevalent diagnostic tool for infectious diseases, like 
      <em>coronavirus disease 2019</em> (COVID-19). RT-PCR is a labor intensive technical procedure that requires numerous trained laboratory personnel to analyze one patient sample [
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Briefly, 
      <em>ribonucleic acid</em> (RNA) is isolated from a patient’s respiratory tract and purified for reverse transcription, a process where the RNA template is turned into 
      <em>complementary deoxyribonucleic acid</em> (cDNA). cDNA, along with specific viral primers, is loaded into a machine, where cDNA is amplified and annealed to the target sequence. While extending through each PCR cycle, a reporter dye is cleaved or broken from a probe to amplify fluorescence intensity and reveal a positive sample.
     </p>
     <p id="p-6">Despite its accuracy for single sample analysis, RT-PCR requires substantial resources to test a large number of samples. Instead, we aim to develop a scalable testing procedure that allows for patient samples to be combined before PCR.</p>
    </div>
    <div id="sec-3" class="subsection">
     <h3>Main idea</h3>
     <p id="p-7">Noisy group testing is used to analyze RT-PCR data from mixed or 
      <em>pooled</em> samples, as recently demonstrated for COVID-19 [
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. The goal of group testing is twofold. First, to increase the accuracy of testing for each individual patient by combining information from multiple pooled measurements that sample genetic material from that same individual. Second, to use a reduced number of measurements, especially in settings where a large population is being tested, most patients are healthy, and so many individual measurements will come out negative and thus show that multiple patients are healthy. In summary, group testing allows to evaluate large populations at high throughput, low per-patient diagnostic costs, and low false positive and negative probabilities.
     </p>
     <p id="p-8">Noiseless group testing has been established, but noisy group testing algorithms are less mature. For example, recent work on COVID-19 [
      <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>], [
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>] uses pooled tests to rule out patients corresponding to negative pooled measurements. Their approach implicitly relies on false negatives being rare in RT-PCR, but diluting many samples may increase false negatives [
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Additionally, patients corresponding to positive pooled measurements are later tested individually [
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>], which does not benefit from pooling. Our algorithm (
      <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">Sec. III</a>) applies pooling to identify individual sick patients.
     </p>
     <p id="p-9">Recently, researchers across the world have been looking to increase the sample size per PCR run by using custom barcodes for each sample and then pooling them together [
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Custom barcoding is not new in terms of multiplexed genetic sequencing [
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. Briefly, custom barcodes for each patients RNA sample are designed by an algorithm and substituted in as the 
      <em>reverse transcriptase</em> (RT) primers to generate barcoded cDNA. Next-generation sequencing is performed after a single pooled PCR reaction, and then demultiplexed to determine each samples viral content. Our method of pooling samples before adding barcodes could be used by researchers for a quicker time to analysis and also as a complementary method to reanalyze barcoded samples.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Contributions and organization</h3>
     <p id="p-10">We focus on a simple pooled testing model for RT-PCR (
      <a id="xref-sec-5-1" class="xref-sec" href="#sec-5">Sec. II</a>). This model is converted to a linear inverse problem (
      <a id="xref-sec-8-1" class="xref-sec" href="#sec-8">Sec. II-A</a>), and our goal is to estimate a vector of patient illness status, 
      <em>x</em>, from a vector of noisy RT-PCR measurements, 
      <em>y</em>, a matrix 
      <em>A</em> relating patients and measurements, and statistical information about false positives and negatives (
      <a id="xref-sec-9-1" class="xref-sec" href="#sec-9">Sec. II-B</a> and 
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). This estimation problem is solved using 
      <em>generalized approximate message passing</em> (GAMP) [
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>] in 
      <a id="xref-sec-10-2" class="xref-sec" href="#sec-10">Sec. III</a>. Promising numerical results are provided in 
      <a id="xref-sec-11-1" class="xref-sec" href="#sec-11">Sec. IV</a>, and 
      <a id="xref-sec-15-1" class="xref-sec" href="#sec-15">Sec. V</a> discusses how our GAMP-based approach can be extended.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="System model. A Bernoulli process with probability [[#x3C1]] of sickness generates an input vector x [[#x2208]] {0, 1}N, which reflects patient illness status. The input is multiplied by a measurement matrix, A [[#x2208]] {0, 1}M[[#xD7]]N, resulting in noiseless measurements, w = Ax [[#x2208]] [[#x2115]]M (1), which are processed by an RT-PCR channel, resulting in noisy measurements, y [[#x2208]] {0, 1}M (2). GAMP [7] processes the input channel relating [[#x3C1]] and x with gin([[#xB7]]) (4), and the output channel relating w and x with gout([[#xB7]]) (5) (details in Sec. III)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1959036521" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;System model. A Bernoulli process with probability &lt;;em&gt;;[[#x3C1]]&lt;;/em&gt;; of sickness generates an input vector &lt;;em&gt;;x&lt;;/em&gt;; [[#x2208]] {0, 1}&lt;;sup&gt;;&lt;;em&gt;;N&lt;;/em&gt;;&lt;;/sup&gt;;, which reflects patient illness status. The input is multiplied by a measurement matrix, &lt;;em&gt;;A&lt;;/em&gt;; [[#x2208]] {0, 1}&lt;;sup&gt;;&lt;;em&gt;;M[[#xD7]]N&lt;;/em&gt;;&lt;;/sup&gt;;, resulting in noiseless measurements, &lt;;em&gt;;w&lt;;/em&gt;; = &lt;;em&gt;;Ax&lt;;/em&gt;; [[#x2208]] [[#x2115]]&lt;;sup&gt;;&lt;;em&gt;;M&lt;;/em&gt;;&lt;;/sup&gt;; (1), which are processed by an RT-PCR channel, resulting in noisy measurements, &lt;;em&gt;;y&lt;;/em&gt;; [[#x2208]] {0, 1}&lt;;sup&gt;;&lt;;em&gt;;M&lt;;/em&gt;;&lt;;/sup&gt;; (2). GAMP [7] processes the input channel relating &lt;;em&gt;;[[#x3C1]]&lt;;/em&gt;; and &lt;;em&gt;;x&lt;;/em&gt;; with &lt;;em&gt;;g&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;in&lt;;/em&gt;;&lt;;/sub&gt;;([[#xB7]]) (4), and the output channel relating &lt;;em&gt;;w&lt;;/em&gt;; and &lt;;em&gt;;x&lt;;/em&gt;; with &lt;;em&gt;;g&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;out&lt;;/em&gt;;&lt;;/sub&gt;;([[#xB7]]) (5) (details in Sec. III).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F1.medium.gif" width="440" height="220"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F1.medium.gif" width="440" height="220"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 1.</span>
       <p id="p-11" class="first-child">System model. A Bernoulli process with probability 
        <em>ρ</em> of sickness generates an input vector 
        <em>x</em> ∈ {0, 1}
        <sup>
         <em>N</em>
        </sup>, which reflects patient illness status. The input is multiplied by a measurement matrix, 
        <em>A</em> ∈ {0, 1}
        <sup>
         <em>M×N</em>
        </sup>, resulting in noiseless measurements, 
        <em>w</em> = 
        <em>Ax</em> ∈ ℕ
        <sup>
         <em>M</em>
        </sup>
        <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">(1)</a>, which are processed by an RT-PCR channel, resulting in noisy measurements, 
        <em>y</em> ∈ {0, 1}
        <sup>
         <em>M</em>
        </sup>
        <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">(2)</a>. GAMP [
        <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>] processes the input channel relating 
        <em>ρ</em> and 
        <em>x</em> with 
        <em>g</em>
        <sub>
         <em>in</em>
        </sub>(·) 
        <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">(4)</a>, and the output channel relating 
        <em>w</em> and 
        <em>x</em> with 
        <em>g</em>
        <sub>
         <em>out</em>
        </sub>(·) 
        <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">(5)</a> (details in 
        <a id="xref-sec-10-3" class="xref-sec" href="#sec-10">Sec. III</a>).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">II. M
     <span class="sc">odel</span>
    </h2>
    <div id="sec-6" class="subsection">
     <h3>Conventional RT-PCR</h3>
     <p id="p-12">RT-PCR has a binary outcome. That is, once the sample is amplified, there is plenty of genetic material available for identification. Prior to amplification, there can be problems during pre-processing to isolate purified RNA. Either genetic material can be damaged, in which case all further tests with this material are negative, or the sample is contaminated, in which case further tests are positive. However, such pre-processing problems essentially flip the sample’s condition permanently from negative to positive or vice versa, because all tests of the patient will be using flawed genetic material from this patient. Therefore, such problems will not be discussed further.</p>
     <p id="p-13">Once the sample has been pre-processed, there are two further problems that could arise once we partition the sample into multiple group test measurements. One possible outcome is a 
      <em>false negative</em>, meaning not enough genetic material from a sick patient and, therefore, insufficient amplification. It is also possible to have a 
      <em>false positive</em>, meaning the sample was contaminated by viral matter, and the test is positive although the patient is healthy. We focus on these two problems, as a well-designed group testing procedure mitigates their effects.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Group testing</h3>
     <p id="p-14">Instead of sampling genetic material from one patient, we will pool material from multiple patients. In principle, if any of the patients is sick, and there is sufficient genetic material, then the pooled group test will come out positive. However, it is possible that group testing will use less genetic material per patient, meaning that the measurement is diluted, and the probability of false negatives (per sick patient) might be larger than conventional (unpooled) measurements [
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>].
     </p>
     <div id="sec-8" class="subsection">
      <h4>A. Number of sick patients per measurement</h4>
      <p id="p-15">We now express the number of sick patients pooled per measurement as a product between a binary matrix representing what patients are pooled in different measurements, and a binary vector representing patient illness status. Later, 
       <a id="xref-sec-9-2" class="xref-sec" href="#sec-9">Sec. II-B</a> forms a noisy probabilistic outcome, where the RT-PCR test being positive or negative depends probabilistically on the number of sick patients pooled per measurement.
      </p>
      <p id="p-16">Our system is illustrated in 
       <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>. We have 
       <em>N</em> patients. The status of each patient is given by 
       <em>x</em>
       <sub>
        <em>n</em>
       </sub>, where 
       <em>n</em> ∈{1, 
       <em>…, N</em>}. If patient 
       <em>n</em> is sick, then 
       <em>x</em>
       <sub>
        <em>n</em>
       </sub> = 1, else 
       <em>x</em>
       <sub>
        <em>n</em>
       </sub> = 0. The 
       <em>N</em> entries are modeled as 
       <em>independent and identically distributed</em> (i.i.d.), and we model 
       <em>x</em>
       <sub>
        <em>n</em>
       </sub> using a 
       <em>random variable</em> (RV), 
       <em>X</em>
       <sub>
        <em>n</em>
       </sub>. These 
       <em>N</em> RVs follow a Bernoulli 
       <em>probability mass function</em> (pmf), 
       <em>X</em>
       <sub>
        <em>n</em>
       </sub>
       <em>∼</em> Ber(
       <em>ρ</em>), 
       <span class="disp-formula" id="disp-formula-1">
        <span class="highwire-responsive-lazyload">
         <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-2.gif"/>
         <noscript>
          <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-2.gif"/>
         </noscript>
        </span>
       </span> where Pr(·) denotes probability, and 
       <em>ρ</em> is the percentage of sick patients.
      </p>
      <p id="p-17">Next, the vector 
       <em>x</em> ∈ {0, 1}
       <sup>
        <em>N</em>
       </sup> is multiplied by a binary 
       <em>measurement matrix A</em> ∈ {0, 1}
       <sup>
        <em>M×N</em>
       </sup>. Because 
       <em>x</em> and 
       <em>A</em> are binary, the matrix vector product, 
       <span class="disp-formula" id="disp-formula-2">
        <span class="highwire-responsive-lazyload">
         <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-3.gif"/>
         <noscript>
          <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-3.gif"/>
         </noscript>
        </span>
       </span> is a length-
       <em>M</em> vector of natural (non-negative) numbers.
      </p>
      <p id="p-18">The matrix 
       <em>A</em> is interpreted as follows. Row 
       <em>m</em> corresponds to measurement 
       <em>m</em>, and column 
       <em>n</em> to patient 
       <em>n</em>, where 
       <em>m</em> ∈ {1, 
       <em>…, M</em>} and 
       <em>n</em> ∈ {1, 
       <em>…, N</em>}. If patient 
       <em>n</em> is not measured in measurement 
       <em>m</em>, then 
       <em>A</em>
       <sub>
        <em>mn</em>
       </sub>, the matrix entry in row 
       <em>m</em> and column 
       <em>n</em>, is zero; such patients do not affect the outcome of measurement 
       <em>m</em>. In contrast, 
       <em>A</em>
       <sub>
        <em>mn</em>
       </sub> = 1 when genetic material from patient 
       <em>n</em> appears in measurement 
       <em>m</em>. It can be seen that 
       <em>w</em>
       <sub>
        <em>m</em>
       </sub> counts the number of sick patients evaluated by measurement 
       <em>m</em>.
      </p>
      <p id="p-19">In noiseless group testing, the RT-PCR measurement is positive if and only if 
       <em>w</em>
       <sub>
        <em>m</em>
       </sub>
       <em>&gt;;</em> 0. However, RT-PCR suffers from false positives and negatives.
      </p>
     </div>
     <div id="sec-9" class="subsection">
      <h4>B. Noisy model</h4>
      <p id="p-20">We now account for these false positives and negatives. The noisy measurement 
       <em>y</em>
       <sub>
        <em>m</em>
       </sub> depends on 
       <em>w</em>
       <sub>
        <em>m</em>
       </sub> through a conditional probability, Pr(
       <em>Y</em>
       <sub>
        <em>m</em>
       </sub>|
       <em>W</em>
       <sub>
        <em>m</em>
       </sub>), where 
       <em>Y</em>
       <sub>
        <em>m</em>
       </sub> and 
       <em>W</em>
       <sub>
        <em>m</em>
       </sub> are RVs. To evaluate Pr(
       <em>Y</em>
       <sub>
        <em>m</em>
       </sub>|
       <em>W</em>
       <sub>
        <em>m</em>
       </sub>), we denote the probability of an individual patient being sick yet not having enough genetic material in one of the measurements by 
       <em>p</em>
       <sub>1</sub>. This is the probability of a false negative caused by 
       <em>one</em> patient; with 
       <em>w</em>
       <sub>
        <em>m</em>
       </sub> sick patients, the probability that all of them have false negatives is (
       <em>p</em>
       <sub>1</sub>)
       <sup>
        <em>w</em>
       </sup>
       <em>m</em>. (We note in passing that false negatives could be modeled as independent of 
       <em>w</em>
       <sub>
        <em>m</em>
       </sub> [
       <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>].)
      </p>
      <p id="p-21">Similarly, the probability of a false positive is denoted by 
       <em>p</em>
       <sub>2</sub>. If there is a false positive in 
       <em>y</em>
       <sub>
        <em>m</em>
       </sub>, then 
       <em>y</em>
       <sub>
        <em>m</em>
       </sub> = 1, irrespective of the status of the patients being evaluated by measurement 
       <em>m</em>. On the other hand, 
       <em>y</em>
       <sub>
        <em>m</em>
       </sub> = 0 means that there was no false positive, and all the patients evaluated that were actually sick resulted in false negatives. Based on this discussion, we can express the probability for 
       <em>y</em>
       <sub>
        <em>m</em>
       </sub> to be 0 or 1 given 
       <em>w</em>
       <sub>
        <em>m</em>
       </sub>, 
       <span class="disp-formula" id="disp-formula-3">
        <span class="highwire-responsive-lazyload">
         <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-4.gif"/>
         <noscript>
          <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-4.gif"/>
         </noscript>
        </span>
       </span>
      </p>
      <p id="p-22">Then, we compute Pr(
       <em>Y</em>
       <sub>
        <em>m</em>
       </sub> = 1|
       <em>W</em>
       <sub>
        <em>m</em>
       </sub>) = 1 
       <em>−</em> Pr(
       <em>Y</em>
       <sub>
        <em>m</em>
       </sub> = 0|
       <em>W</em>
       <sub>
        <em>m</em>
       </sub>). In communication and information theory, such a probabilistic relationship is known as a 
       <em>channel</em> [
       <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]; the output channel relating the vectors 
       <em>w</em> and 
       <em>y</em> appears in 
       <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>.
      </p>
      <p id="p-23">We now have a linear relationship from 
       <em>x</em> to 
       <em>w</em>, and the noiseless measurements vector 
       <em>w</em>, which contains the number of sick patients per measurement, is then processed by a probabilistic channel to yield the noisy measurements vector, 
       <em>y</em>. Our goal is to estimate 
       <em>x</em> from 
       <em>y, A</em>, and statistical information about the channel. Other group testing approaches often perform pooled measurements in a first part, and positives are tested individually in a second part [
       <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>]; our method can improve both parts by pooling all measurements and accounting for all available information. In the following section, we describe our algorithmic framework in detail.
      </p>
      <div id="statement-1" class="statement">
       <span class="statement-label">Algorithm 1</span>
       <h3>GAMP</h3>
       <p id="p-24"> </p>
       <div id="F2" class="fig pos-float type-figure odd">
        <div class="highwire-figure">
         <div class="fig-inline-img-wrapper">
          <div class="fig-inline-img">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1959036521" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
            <span class="hw-responsive-img">
             <img class="highwire-fragment fragment-image lazyload" alt="Figure2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F2.medium.gif" width="369" height="440"/>
             <noscript>
              <img class="highwire-fragment fragment-image" alt="Figure2" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F2.medium.gif" width="369" height="440"/>
             </noscript>
            </span>
           </a>
          </div>
         </div>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure2" data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
       </div>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">III. A
     <span class="sc">lgorithmic framework</span>
    </h2>
    <p id="p-25">We estimate 
     <em>x</em> from 
     <em>y, A</em>, and statistical information about the channel by applying 
     <em>generalized approximate messsage passing</em> (GAMP) [
     <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>], which is an iterative signal estimation al-M gorithm. GAMP is preferred, because it achieves best-possible estimation-theoretic performance asymptotically, in the limit of large linear estimation problems.
    </p>
    <p id="p-26">Our approach focuses on the 
     <em>large system limit</em>, where 
     <em>N → ∞, M</em> (
     <em>N</em>) depends on 
     <em>N</em>, and 
     <span class="inline-formula" id="inline-formula-1">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-1.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-1.gif"/>
       </noscript>
      </span>
     </span>, where we call 
     <em>R</em> the measurement rate. GAMP relies on the large system limit for various summations in the derivation steps of the algorithm to be well-approximated as Gaussian under the central limit theorem [
     <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>]. Note that running our algorithm for small problem sizes such as 
     <em>N</em> = 100 patients and 
     <em>M</em> = 30 measurements may result in poor estimation quality.
    </p>
    <p id="p-27">The GAMP algorithm is listed in 
     <a id="xref-statement-1-1" class="xref-statement" href="#statement-1">Algorithm 1</a>. For a detailed derivation, we refer the reader to Rangan [
     <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>]. An intuitive and less formal explanation is provided below.
    </p>
    <p id="p-28">GAMP is comprised of two parts. The first part involves the input channel relating 
     <em>ρ</em> and 
     <em>x</em> (
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>) [
     <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">7</a>], where 
     <em>x</em> is estimated from an auxiliary vector 
     <em>q</em> ∈ ℝ
     <sup>
      <em>N</em>
     </sup> (cf. Line 10 of 
     <a id="xref-statement-1-2" class="xref-statement" href="#statement-1">Algorithm 1</a>) through a function 
     <em>g</em>
     <sub>
      <em>in</em>
     </sub>(·). The auxiliary vector, 
     <em>q</em>, is known in the AMP literature
     <sup>
      <a id="xref-fn-3-1" class="xref-fn" href="#fn-3">1</a>
     </sup> as the pseudo data, which can be treated as a noisy version of the true signal 
     <em>x</em>, 
     <span class="disp-formula" id="disp-formula-4">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-6.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-6.gif"/>
       </noscript>
      </span>
     </span> where 
     <em>v</em> ∈ ℝ
     <sup>
      <em>N</em>
     </sup> is 
     <em>additive white Gaussian noise</em> (AWGN) with zero mean, where Δ
     <sub>
      <em>vn</em>
     </sub> is the variance of 
     <em>v</em>
     <sub>
      <em>n</em>
     </sub>. Hence, 
     <em>g</em>
     <sub>
      <em>in</em>
     </sub>(·) can be interpreted as a denoising function, 
     <span class="disp-formula" id="disp-formula-5">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-7.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-7.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-29">While other denoising functions can be used, conditional expectation, i.e., E[X|Q], minimizes the mean squared error (MSE) in each GAMP iteration, and so it reduces the error as quickly as possible.</p>
    <p id="p-30">The second part of GAMP involves the output channel (cf. 
     <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1</a>), where 
     <em>y</em>
     <sub>
      <em>m</em>
     </sub> depends probabilistically on 
     <em>w</em>
     <sub>
      <em>m</em>
     </sub>. We estimate 
     <em>w</em>
     <sub>
      <em>m</em>
     </sub> from 
     <em>y</em>
     <sub>
      <em>m</em>
     </sub> using a second denoising function, 
     <span class="disp-formula" id="disp-formula-6">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-8.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-8.gif"/>
       </noscript>
      </span>
     </span> where the expectation is taken over the pmf, 
     <span class="disp-formula" id="disp-formula-7">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-9.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-9.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-31">In this expression, 
     <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">(5)</a>, we have mean and variance values for 
     <em>w</em>
     <sub>
      <em>m</em>
     </sub>, and can interpret 
     <em>g</em>
     <sub>
      <em>out</em>
     </sub>(
     <em>k</em>
     <sub>
      <em>m</em>
     </sub>; 
     <em>y</em>
     <sub>
      <em>m</em>
     </sub>,Θ
     <sub>
      <em>m</em>
     </sub>) as a correction term that reflects residual information, which is provided by the noisy measurements vector, y, but is not yet reflected in our estimates, 
     <em>k</em> for A
     <em>x</em>, and 
     <span class="inline-formula" id="inline-formula-2">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-2.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-2.gif"/>
       </noscript>
      </span>
     </span>. The correction term is used in later iterations to compute 
     <em>q</em> and 
     <em>g</em>
     <sub>
      <em>in</em>
     </sub>(·).
    </p>
    <p id="p-32">GAMP uses these two scalar functions, 
     <em>g</em>
     <sub>
      <em>in</em>
     </sub>(·) 
     <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">(4)</a> and 
     <em>g</em>
     <sub>
      <em>out</em>
     </sub>(·) 
     <a id="xref-disp-formula-6-3" class="xref-disp-formula" href="#disp-formula-6">(5)</a>, to estimate 
     <em>x</em> and 
     <em>w</em> = 
     <em>Ax</em>
     <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">(1)</a> from 
     <em>q</em>
     <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">(3)</a> and 
     <em>y</em>
     <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">(2)</a>, respectively. That is, GAMP iteratively cleans the input and output channels. A numerical illustration is provided in 
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 2</a>; 
     <a id="xref-sec-11-2" class="xref-sec" href="#sec-11">Sec. IV</a> discusses this figure in detail. GAMP also uses derivatives of these scalar functions to estimate the variance. In words, knowing not only the mean but also the variance around the mean allows GAMP to judiciously use information from 
     <span class="inline-formula" id="inline-formula-3">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-3.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-3.gif"/>
       </noscript>
      </span>
     </span> when estimating 
     <span class="inline-formula" id="inline-formula-4">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-4.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-4.gif"/>
       </noscript>
      </span>
     </span> and vice versa.
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Top: [[#x1D4C1]]2 norms of the input (dashed red line; associated with right vertical axis) and output (solid blue; left) channel noise as functions of the GAMP iteration. Bottom: The first 1000 entries of the unknown patient illness status vector x, and their estimates. (N = 5000 patients; [[#x3C1]] = 0.01 percentage of sick patients; measurement rate R = M/N = 0.3; npos = 0.5 average sick patients per measurement; false negative probability p1 = 0.02; false positive p2 = 0.001.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1959036521" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Top: &lt;;em&gt;;[[#x1D4C1]]&lt;;/em&gt;;&lt;;sub&gt;;2&lt;;/sub&gt;; norms of the input (dashed red line; associated with right vertical axis) and output (solid blue; left) channel noise as functions of the GAMP iteration. Bottom: The first 1000 entries of the unknown patient illness status vector &lt;;em&gt;;x&lt;;/em&gt;;, and their estimates. (&lt;;em&gt;;N&lt;;/em&gt;; = 5000 patients; &lt;;em&gt;;[[#x3C1]]&lt;;/em&gt;; = 0.01 percentage of sick patients; measurement rate &lt;;em&gt;;R&lt;;/em&gt;; = &lt;;em&gt;;M/N&lt;;/em&gt;; = 0.3; &lt;;em&gt;;n&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;pos&lt;;/em&gt;;&lt;;/sub&gt;; = 0.5 average sick patients per measurement; false negative probability &lt;;em&gt;;p&lt;;/em&gt;;&lt;;sub&gt;;1&lt;;/sub&gt;; = 0.02; false positive &lt;;em&gt;;p&lt;;/em&gt;;&lt;;sub&gt;;2&lt;;/sub&gt;; = 0.001.)&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F3.medium.gif" width="440" height="310"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F3.medium.gif" width="440" height="310"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 2.</span>
      <p id="p-33" class="first-child">Top: 
       <em>𝓁</em>
       <sub>2</sub> norms of the input (dashed red line; associated with right vertical axis) and output (solid blue; left) channel noise as functions of the GAMP iteration. Bottom: The first 1000 entries of the unknown patient illness status vector 
       <em>x</em>, and their estimates. (
       <em>N</em> = 5000 patients; 
       <em>ρ</em> = 0.01 percentage of sick patients; measurement rate 
       <em>R</em> = 
       <em>M/N</em> = 0.3; 
       <em>n</em>
       <sub>
        <em>pos</em>
       </sub> = 0.5 average sick patients per measurement; false negative probability 
       <em>p</em>
       <sub>1</sub> = 0.02; false positive 
       <em>p</em>
       <sub>2</sub> = 0.001.)
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">IV. N
     <span class="sc">umerical</span> R
     <span class="sc">esults</span>
    </h2>
    <div id="sec-12" class="subsection">
     <h3>A. GAMP illustration</h3>
     <p id="p-34">This section provides numerical results showing how GAMP solves noisy group testing problem. For readers who are new to GAMP, we begin by illustrating how GAMP cleans up the input and output channels iteratively.</p>
     <p id="p-35">We evaluate 
      <em>N</em> = 5000 patients at a time, where the fraction of infected patients is 
      <em>ρ</em> = 0.01. The measurement rate is 
      <em>R</em> = 
      <em>M/N</em> = 0.3, meaning that we take 
      <em>M</em> = 
      <em>NR</em> = 1500 RT-PCR measurements.
      <sup>
       <a id="xref-fn-4-1" class="xref-fn" href="#fn-4">2</a>
      </sup> The matrix 
      <em>A</em> is designed to pick up 
      <em>n</em>
      <sub>
       <em>pos</em>
      </sub> sick patients per measurement on average; we let 
      <em>n</em>
      <sub>
       <em>pos</em>
      </sub> = 0.5. The numbers of ones per row and column are kept close to 
      <em>n</em>
      <sub>
       <em>pos</em>
      </sub>
      <em>/ρ</em> and 
      <em>Rn</em>
      <sub>
       <em>pos</em>
      </sub>
      <em>/ρ</em>, respectively. For the RT-PCR channel, we assume a false negative probability, 
      <em>p</em>
      <sub>1</sub> = 0.02, and false positive probability, 
      <em>p</em>
      <sub>2</sub> = 0.001.
      <sup>
       <a id="xref-fn-5-1" class="xref-fn" href="#fn-5">3</a>
      </sup> We quantify GAMP signal estimation quality using the 
      <em>area under the receiver operating curve</em> (AUC-ROC). In words, the ROC captures trade-offs between false positives and negatives, and increasing the AUC reflects better estimation. While standard GAMP minimizes the MSE [
      <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">7</a>], other error metrics can be minimized [
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], [
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>].
     </p>
     <p id="p-36">The top pannel of 
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 2</a> plots the 
      <em>𝓁</em>
      <sub>2</sub> norms of the input channel noise (dashed red line) and output channel noise (solid blue). We can see that the input and output channels improve over iterations. The bottom panel shows the first 1000 entries of the input signal vector 
      <em>x</em> and their estimates. We can see that patient illness status is estimated well.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>B. Group testing under various conditions</h3>
     <p id="p-37">We investigate the impact of the percentage of sick patients, 
      <em>ρ</em>, and measurement rate, 
      <em>R</em> = 
      <em>M/N</em>, on estimation accuracy in 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 3</a>. As before, 
      <em>N</em> = 5000, 
      <em>n</em>
      <sub>
       <em>pos</em>
      </sub> = 0.5, 
      <em>p</em>
      <sub>1</sub> = 0.02, and 
      <em>p</em>
      <sub>2</sub> = 0.001. We run our algorithm on 
      <em>ρ</em> ∈ {0.005, 0.01, 0.015, …, 0.05} and 
      <em>R</em> ∈ {0.1, 0.15, 0.2, …, 0.5}. For each setting, we randomly generate 20 different triples of (
      <em>x, A, y</em>), and record the AUC for every triple. The performance for each setting is evaluated by averaging AUC values. Our results show that the AUC increases with the measurement rate, 
      <em>R</em>, and larger 
      <em>ρ</em> requires larger 
      <em>R</em> to yield an AUC near 1. These results align with our expectation that more measurements improve estimation, while more sick patients require more measurements.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Estimation accuracy in AUC (vertical axis) as a function of the percentage of sick patients, [[#x3C1]], and measurement rate, R = M/N. (N = 5000; npos = 0.5; p1 = 0.02; p2 = 0.001.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1959036521" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Estimation accuracy in AUC (vertical axis) as a function of the percentage of sick patients, &lt;;em&gt;;[[#x3C1]]&lt;;/em&gt;;, and measurement rate, &lt;;em&gt;;R&lt;;/em&gt;; = &lt;;em&gt;;M/N&lt;;/em&gt;;. (&lt;;em&gt;;N&lt;;/em&gt;; = 5000; &lt;;em&gt;;n&lt;;/em&gt;;&lt;;sub&gt;;&lt;;em&gt;;pos&lt;;/em&gt;;&lt;;/sub&gt;; = 0.5; &lt;;em&gt;;p&lt;;/em&gt;;&lt;;sub&gt;;1&lt;;/sub&gt;; = 0.02; &lt;;em&gt;;p&lt;;/em&gt;;&lt;;sub&gt;;2&lt;;/sub&gt;; = 0.001.)&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F4.medium.gif" width="440" height="322"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F4.medium.gif" width="440" height="322"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.026765/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 3.</span>
       <p id="p-38" class="first-child">Estimation accuracy in AUC (vertical axis) as a function of the percentage of sick patients, 
        <em>ρ</em>, and measurement rate, 
        <em>R</em> = 
        <em>M/N</em>. (
        <em>N</em> = 5000; 
        <em>n</em>
        <sub>
         <em>pos</em>
        </sub> = 0.5; 
        <em>p</em>
        <sub>1</sub> = 0.02; 
        <em>p</em>
        <sub>2</sub> = 0.001.)
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-14" class="subsection">
     <h3>C. Two part approach</h3>
     <p id="p-39">Recent work Hanel and Thurner [
      <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">3</a>] analyzes a two part group testing approach. Their model for PCR uses Pr(
      <em>Y</em>
      <sub>
       <em>m</em>
      </sub> = 0|
      <em>W</em>
      <sub>
       <em>m</em>
      </sub>
      <em>&gt;;</em> 0) = (1 
      <em>− p</em>
      <sub>2</sub>)
      <em>p</em>
      <sub>1</sub>, while we use 
      <span class="inline-formula" id="inline-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-5.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-5.gif"/>
        </noscript>
       </span>
      </span>; we evaluated their approach using 
      <em>ρ</em> = 0.01, 
      <em>p</em>
      <sub>1</sub> = 0.02, and 
      <em>p</em>
      <sub>2</sub> = 0.001. Hanel and Thurner’s Part 1 pools a block of 
      <em>B</em> = 11 patients at a time. If a pool is negative, all patients in the block are declared healthy; else Part 2 measures them individually. The measurement rate is 
      <span class="disp-formula" id="disp-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-12.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-12.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-40">We compute Pr(
      <em>Y</em>
      <sub>
       <em>m</em>
      </sub> = 1), 
      <span class="disp-formula" id="disp-formula-9">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-13.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-13.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-41">Note that Pr(
      <em>W</em>
      <sub>
       <em>m</em>
      </sub> = 0) = (1 
      <em>− ρ</em>)
      <sup>
       <em>B</em>
      </sup>, Pr(
      <em>Y</em>
      <sub>
       <em>m</em>
      </sub> = 1|
      <em>W</em>
      <sub>
       <em>m</em>
      </sub> = 0) = 
      <em>p</em>
      <sub>2</sub>, Pr(
      <em>W</em>
      <sub>
       <em>m</em>
      </sub>
      <em>&gt;;</em> 0) = 1 
      <em>−</em> Pr(
      <em>W</em>
      <sub>
       <em>m</em>
      </sub> = 0), and 
      <span class="disp-formula" id="disp-formula-10">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-14.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-14.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-42">Combining these results, 
      <span class="disp-formula" id="disp-formula-11">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-15.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/graphic-15.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-43">A simulation over 
      <em>N</em> = 10
      <sup>7</sup> patients had 935 false positives and 4038 false negatives.
     </p>
     <p id="p-44">Our two part approach modifies Part 2. Instead of testing patients within each positive block individually, we combine all patients within all positive blocks into a new linear inverse problem, and solve the resulting estimation problem 
      <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">(1)</a> with GAMP. For example, let the block size be 
      <em>B</em> = 25 patients in Part 1. In Part 2, we combine all positive blocks and apply 
      <em>R</em> = 0.5 and 
      <em>n</em>
      <sub>
       <em>pos</em>
      </sub> = 0.5 to the linear inverse problem. Note that (
      <em>i</em>) positive measurements from Part 1 are reused in the matrix 
      <em>A</em> and measurement vector 
      <em>y</em> of Part 2, because they contain information that helps GAMP; (
      <em>ii</em>) we decide whether a patient is sick or not by thresholding 
      <span class="inline-formula" id="inline-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-6.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-6.gif"/>
        </noscript>
       </span>
      </span>. Combining Parts 1 and 2, the measurement rate is 
      <em>R</em> = 0.149. We randomly generate 100 (
      <em>x, y, A</em>) triples for 
      <em>N</em> = 10000 patients in Part 1, Among 100
      <em>N</em> = 10
      <sup>6</sup> patients, there are 92 false positives and 366 false negatives. Our false positive and negative rates are both lower than those of Hanel and Thurner [
      <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">3</a>].
     </p>
    </div>
   </div>
   <div class="section" id="sec-15">
    <h2 class="">V. D
     <span class="sc">iscussion</span>
    </h2>
    <p id="p-45">Our current approach relies on various assumptions. Below are issues that can be considered in ongoing and future work.</p>
    <div id="sec-16" class="subsection">
     <h3>Challenges</h3>
     <p id="p-46">Some of the challenges we expect involve better modeling of RT-PCR, in particular how pooling multiple samples dilutes the genetic material and may increase false positives and negatives [
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Other challenges involve matrix design; better matrices will improve estimation quality.
     </p>
     <p id="p-47">One question is whether 
      <em>p</em>
      <sub>1</sub> is the same for each patient 
      <em>n</em> in each measurement 
      <em>m</em> where 
      <em>A</em>
      <sub>
       <em>mn</em>
      </sub> = 1. It might be possible to use different matrix entry values (not just 0 and 1) and thus sample more genetic material in some cases, and less in others. This will likely result in 
      <em>p</em>
      <sub>1</sub> depending on the amount of genetic matter being sampled. Therefore, genetic material will have to be measured before samples are pooled to ensure the same concentration of genetic material is loaded for each sample. Limiting the amount of genetic matter being processed may reduce the costs of the overall measurement process. On the other hand, if the amount of genetic material per patient per measurement varies, a sophisticated channel could be supported by having nonzeros in 
      <em>A</em>
      <sub>
       <em>mn</em>
      </sub> take different values.
     </p>
     <p id="p-48">Another question is whether the false positive probability, 
      <em>p</em>
      <sub>2</sub>, is identical for all measurements. Alternately, 
      <em>p</em>
      <sub>2</sub> may depend on the measurement system, for example the number of samples pooled together, or the number of RT-PCR iterations. If individual RT-PCR iterations are costly, then the number of iterations can be reduced, resulting in larger 
      <em>p</em>
      <sub>2</sub>. Our experience with AMP-based algorithms suggests that a modest increase in 
      <em>R</em> = 
      <em>M/N</em> will compensate for the degraded individual measurements. Cost effectiveness of each iteration will determine the number of times each sample can be run. This value should be weighed against the number of times individual samples are pooled, allowing to optimize the number of samples pooled per measurement à la [
      <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">3</a>].
     </p>
     <p id="p-49">A third challenge pertains to the measurement matrix, 
      <em>A</em>. In our current design, rows and columns contain similar numbers of nonzeros (
      <a id="xref-sec-11-3" class="xref-sec" href="#sec-11">Sec. IV</a>). Therefore, each measurement provides the same 
      <em>signal to noise ratio</em> (SNR). One matrix design option is to allow different rows and columns to have different numbers of nonzeros. Another is to prevent any pair of patients, 
      <em>n</em>
      <sub>1</sub> and 
      <em>n</em>
      <sub>2</sub>, from both having nonzero matrix entries in different rows, 
      <em>m</em>
      <sub>1</sub> and 
      <em>m</em>
      <sub>2</sub>, i.e., 
      <span class="inline-formula" id="inline-formula-7">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-7.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-7.gif"/>
        </noscript>
       </span>
      </span>, and 
      <span class="inline-formula" id="inline-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.026765/embed/inline-graphic-8.gif"/>
        </noscript>
       </span>
      </span> cannot all be nonzero. Refinements in matrix design will improve our estimation quality.
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Applications</h3>
     <p id="p-50">Improvements in testing accuracy can be used in different ways in different applications.</p>
     <ul class="list-unord " id="list-1">
      <li id="list-item-1">
       <p id="p-51">False positive and negative rates of individual RT-PCR measurements can be reduced by pooling together samples, and using GAMP-based algorithms. This can help decide when it is safe to release a COVID-19 patient from quarantine.</p>
      </li>
      <li id="list-item-2">
       <p id="p-52">Latency reduction. As the first RT-PCR measurements from a batch of patients arrive, all patients corresponding to positive measurements can be quarantined. As more RT-PCR measurements come in, GAMP can determine which of the individual patients in the positive pooled samples are actually healthy.</p>
      </li>
      <li id="list-item-3">
       <p id="p-53">Throughput can be drastically increased for fixed target levels of false positives and negatives. This can be useful for testing large populations with minimal cost.</p>
      </li>
      <li id="list-item-4">
       <p id="p-54">False negatives must be low to prevent a few sick patients from infecting many others. Low false negative probabilities can be provided by a two-part signal estimation approach [
        <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Part 1 will use a conventional measurement matrix 
        <em>A</em>. Part 2 takes extra measurements only for patients deemed healthy in Part 1, thus reducing false negatives. Similar ideas have been proposed for adaptive sensing [
        <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>].
       </p>
      </li>
     </ul>
     <p id="p-55">Finally, our GAMP-based approach uses statistical information about the probability of a patient being sick, 
      <em>ρ</em>, and probabilities governing false positives and negatives, 
      <em>p</em>
      <sub>1</sub> and 
      <em>p</em>
      <sub>2</sub>. The algorithm can be improved using more information, for example statistical dependencies between household members being sick. We will integrate more predictive information to further improve estimation quality.
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">VI. A
     <span class="sc">cknowledgments</span>
    </h2>
    <p id="p-56">The authors thank Michael Daniele for RT-PCR modeling discussions, and Ahmad Beirami, Yitzhak (Tsahi) Birk, Igor Carron, Steven Cotten, Florent Krzakala, John Muth, and Lenka Zdeborova for discussions of group testing. Baron also thanks numerous colleagues at North Carolina State University for putting him in touch with various specialists.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-supported-by" id="fn-1">
      <p id="p-1">The work of Baron was supported in part by NSF EECS #1611112. Rivera was supported by the National Institutes of Health under award number F31DK118859-02 and by the 2019 Howard Hughes Medical Institute Gilliam Fellowship Award.</p>
     </li>
     <li class="fn-other" id="fn-2">
      <p id="p-2">Zhu is with Harvest Fund Management, Beijing, China, email 
       <span class="em-link">
        <span class="em-addr">junan.zhu{at}gmail.com</span>
       </span>. Rivera and Baron are with North Carolina State University, Raleigh, NC, email fkrrivera, 
       <span class="em-link">
        <span class="em-addr">barondrorg{at}ncsu.edu</span>
       </span>.
      </p>
     </li>
     <li class="fn" id="fn-3">
      <p id="p-57">
       <a class="rev-xref" href="#xref-fn-3-1">↵</a>
       <span class="fn-label">1</span> AMP can be derived from GAMP for a specific setting [
       <a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">7</a>]. While AMP [
       <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>] requires the matrices it processes to have zero mean, GAMP is less restrictive.
      </p>
     </li>
     <li class="fn" id="fn-4">
      <p id="p-58">
       <a class="rev-xref" href="#xref-fn-4-1">↵</a>
       <span class="fn-label">2</span> Our GAMP-based algorithm is relatively fast; problems of size (
       <em>M</em> = 1500, 
       <em>N</em> = 5000) take a few seconds to run on a laptop computer.
      </p>
     </li>
     <li class="fn" id="fn-5">
      <p id="p-59">
       <a class="rev-xref" href="#xref-fn-5-1">↵</a>
       <span class="fn-label">3</span> The parameters 
       <em>p</em>
       <sub>1</sub> and 
       <em>p</em>
       <sub>2</sub> resemble Hanel and Thurner [
       <a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">3</a>]; other sources suggest larger false positive and negative probabilities. For our software, these are merely parameters that are easily modified.
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">R
     <span class="sc">eferences</span>
    </h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">[1].</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Nolan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Hands</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Bustin</span>
         </span>, “
         <span class="cit-article-title">Quantification of mRNA using real-time rt-PCR</span>,” 
         <abbr class="cit-jnl-abbrev">Nature Prot</abbr>., vol. 
         <span class="cit-vol">1</span>, pp. 
         <span class="cit-fpage">1559</span>–
         <span class="cit-lpage">82</span>, 
         <span class="cit-month">Feb</span>
         <span class="cit-pub-date">2006</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BProt%26rft.volume%253D1%26rft.spage%253D1559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[2].</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.04.06.026765v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">I.</span>
          <span class="cit-name-surname">Yelin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Aharony</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Shaer-Tamar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Argoetti</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E. Messer</span>, 
          <span class="cit-name-surname">Berenbaum</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Shafran</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Kuzli</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Gandali</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Hashimshony</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Mandel-Gutfreund</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Halberthal</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Geffen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Szwarcwort-Cohen</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Kishony</span>
         </span>, “
         <span class="cit-article-title">Evaluation of COVID-19 RT-qPCR test in multi-sample pools</span>,” 
         <span class="cit-source">medRxiv</span>, 
         <span class="cit-pub-date">2020</span>. [Online]. Available: 
         <a href="https://www.medrxiv.org/content/early/2020/03/27/2020.03.26.20039438">https://www.medrxiv.org/content/early/2020/03/27/2020.03.26.20039438</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[3].</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Hanel</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Thurner</span>
         </span>, “
         <abbr class="cit-jnl-abbrev">Boosting test-efficiency by pooled testing strategies for SARS-CoV-2</abbr>,” arXiv preprint 
         <span class="cit-pub-id cit-pub-id-arxiv">2003.09944</span>, 
         <span class="cit-month">Mar</span>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[4].</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S. L.</span>
          <span class="cit-name-surname">Stramer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. E.</span>
          <span class="cit-name-surname">Krysztof</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. P.</span>
          <span class="cit-name-surname">Brodsky</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. A.</span>
          <span class="cit-name-surname">Fickett</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Reynolds</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. Y.</span>
          <span class="cit-name-surname">Dodd</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. H.</span>
          <span class="cit-name-surname">Kleinman</span>
         </span>, “
         <span class="cit-article-title">Comparative analysis of triplex nucleic acid test assays in United States blood donors</span>,” 
         <abbr class="cit-jnl-abbrev">Transfusion</abbr>, vol. 
         <span class="cit-vol">53</span>, no. 
         <span class="cit-issue">10pt2</span>, pp. 
         <span class="cit-fpage">2525</span>–
         <span class="cit-lpage">2537</span>, 
         <span class="cit-pub-date">2013</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTransfusion%26rft.volume%253D53%26rft.spage%253D2525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[5].</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S. R. Ayaan</span>
          <span class="cit-name-surname">Hossain</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Alexander C.</span>
          <span class="cit-name-surname">Reis</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">H. M.</span>
          <span class="cit-name-surname">Salis</span>
         </span>, “
         <span class="cit-article-title">A Massively Parallel COVID-19 Diagnostic Assay for Simultaneous Testing of 19200 Patient Samples</span>,” 
         <abbr class="cit-jnl-abbrev">Google Docs</abbr>, 
         <span class="cit-month">Mar</span>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[6].</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.6" data-doi="10.1371/journal.pone.0128036">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Baym</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Kryazhimskiy</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. D.</span>
          <span class="cit-name-surname">Lieberman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Chung</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. M.</span>
          <span class="cit-name-surname">Desai</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Kishony</span>
         </span>, “
         <span class="cit-article-title">Inexpensive multiplexed library preparation for megabase-sized genomes</span>,” 
         <abbr class="cit-jnl-abbrev">PloS one</abbr>, vol. 
         <span class="cit-vol">10</span>, no. 
         <span class="cit-issue">5</span>, p. 
         <span class="cit-fpage">e0128036</span>, 
         <span class="cit-pub-date">2015</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D10%26rft.spage%253De0128036%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0128036%26rft_id%253Dinfo%253Apmid%252F26000737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0128036&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26000737&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.026765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[7].</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Rangan</span>
         </span>, “
         <span class="cit-article-title">Generalized approximate message passing for estimation with random linear mixing</span>,” in 
         <abbr class="cit-jnl-abbrev">Proc. IEEE Int. Symp. Inf. Theory</abbr>, 
         <span class="cit-pub-date">2011</span>, pp. 
         <span class="cit-fpage">2168</span>–
         <span class="cit-lpage">2172</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[8].</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.04.06.026765v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">T. M.</span>
          <span class="cit-name-surname">Cover</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. A.</span>
          <span class="cit-name-surname">Thomas</span>
         </span>, 
         <span class="cit-source">Elements of Information Theory</span>. 
         <span class="cit-publ-loc">New York, NY, USA</span>: 
         <span class="cit-publ-name">Wiley-Interscience</span>, 
         <span class="cit-pub-date">2006</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[9].</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D. L.</span>
          <span class="cit-name-surname">Donoho</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Maleki</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Montanari</span>
         </span>, “
         <span class="cit-article-title">Message passing algorithms for compressed sensing</span>,” 
         <abbr class="cit-jnl-abbrev">Proc. Nat. Academy Sci</abbr>., vol. 
         <span class="cit-vol">106</span>, no. 
         <span class="cit-issue">45</span>, pp. 
         <span class="cit-fpage">18 914</span>–
         <span class="cit-lpage">18 919</span>, 
         <span class="cit-month">Nov</span>. 
         <span class="cit-pub-date">2009</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc.%2BNat.%2BAcademy%2BSci%26rft.volume%253D106%26rft.spage%253D18914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[10].</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Tan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Carmon</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Baron</span>
         </span>, “
         <span class="cit-article-title">Signal estimation with additive error metrics in compressed sensing</span>,” 
         <abbr class="cit-jnl-abbrev">IEEE Trans. Inf. Theory</abbr>, vol. 
         <span class="cit-vol">60</span>, no. 
         <span class="cit-issue">1</span>, pp. 
         <span class="cit-fpage">150</span>–
         <span class="cit-lpage">158</span>, 
         <span class="cit-month">Jan</span>. 
         <span class="cit-pub-date">2014</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BInf.%2BTheory%26rft.volume%253D60%26rft.spage%253D150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[11].</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Zhu</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Baron</span>
         </span>, “
         <span class="cit-article-title">Performance limits with additive error metrics in noisy multimeasurement vector problems</span>,” 
         <abbr class="cit-jnl-abbrev">IEEE Trans. Signal Proc</abbr>., vol. 
         <span class="cit-vol">66</span>, no. 
         <span class="cit-issue">20</span>, pp. 
         <span class="cit-fpage">5338</span>–
         <span class="cit-lpage">5348</span>, 
         <span class="cit-month">Oct</span>
         <span class="cit-pub-date">2018</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BSignal%2BProc%26rft.volume%253D66%26rft.spage%253D5338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[12].</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Châtel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Yang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Cholette</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Soudeyns</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Sandstrom</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Lavigne</span>
         </span>, “
         <span class="cit-article-title">Impact of pre-amplification conditions on sensitivity of the tat/rev induced limiting dilution assay</span>,” 
         <abbr class="cit-jnl-abbrev">Archives of virology</abbr>, vol. 
         <span class="cit-vol">163</span>, no. 
         <span class="cit-issue">10</span>, pp. 
         <span class="cit-fpage">2701</span>–
         <span class="cit-lpage">2710</span>, 
         <span class="cit-pub-date">2018</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.volume%253D163%26rft.spage%253D2701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[13].</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.026765v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Ma</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Baron</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Needell</span>
         </span>, “
         <span class="cit-article-title">Two-part reconstruction with noisy-sudocodes</span>,” 
         <abbr class="cit-jnl-abbrev">IEEE Trans. Signal Process</abbr>., vol. 
         <span class="cit-vol">62</span>, no. 
         <span class="cit-issue">23</span>, pp. 
         <span class="cit-fpage">6323</span>–
         <span class="cit-lpage">6334</span>, 
         <span class="cit-month">Dec</span>. 
         <span class="cit-pub-date">2014</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BSignal%2BProcess%26rft.volume%253D62%26rft.spage%253D6323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[14].</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.04.06.026765v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J. D.</span>
          <span class="cit-name-surname">Haupt</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Nowak</span>
         </span>, “
         <span class="cit-chapter-title">Adaptive sensing for sparse recovery</span>,” in 
         <span class="cit-source">Compressed Sensing: Theory and Applications</span>. 
         <span class="cit-publ-name">Cambridge University Press</span>, 
         <span class="cit-pub-date">2012</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
