<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7125191">7125191</a></span><span class="publisher-id" title="publisher-id">62597</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-020-62597-5">10.1038/s41598-020-62597-5</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Epidemics with mutating infectivity on small-world networks</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RÃ¼diger</span><span class="given-names" tagx="given-names" title="given-names">Sten</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">sten.ruediger@gmail.com</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plietzsch</span><span class="given-names" tagx="given-names" title="given-names">Anton</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SaguÃ©s</span><span class="given-names" tagx="given-names" title="given-names">Francesc</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokolov</span><span class="given-names" tagx="given-names" title="given-names">Igor M.</span></span><a href="#Aff1">1</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurths</span><span class="given-names" tagx="given-names" title="given-names">JÃ¼rgen</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a><a href="#Aff5">5</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2248 7639</span><span class="institution-id" title="institution-id">grid.7468.d</span><span class="institution" title="institution">Department of Physics, </span><span class="institution" title="institution">Humboldt-UniversitÃ¤t zu Berlin, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0493 9031</span><span class="institution-id" title="institution-id">grid.4556.2</span><span class="institution" title="institution">Potsdam Institute for Climate Impact Research (PIK), </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 1937 0247</span><span class="institution-id" title="institution-id">grid.5841.8</span><span class="institution" title="institution">Departament de QuÃ­mica FÃ­sica, </span><span class="institution" title="institution">Universitat de Barcelona, </span></div></span><span class="citation_author_institution" id="Aff4">[4], </span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2179 0417</span><span class="institution-id" title="institution-id">grid.446088.6</span><span class="institution" title="institution">Saratov State University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">5919</span><span class="history" title="history"><span class="received" title="received">received: 2019-10-10</span><span class="accepted" title="accepted">accepted: 2020-3-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articleâ€™s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Epidemics and evolution of many pathogens occur on similar timescales so that their dynamics are often entangled. Here, in a first step to study this problem theoretically, we analyze mutating pathogens spreading on simple SIR networks with grid-like connectivity. We have in mind the spatial aspect of epidemics, which often advance on transport links between hosts or groups of hosts such as cities or countries. We focus on the case of mutations that enhance an agentâ€™s infection rate. We uncover that the small-world property, i.e., the presence of long-range connections, makes the network very vulnerable, supporting frequent supercritical mutations and bringing the network from disease extinction to full blown epidemic. For very large numbers of long-range links, however, the effect reverses and we find a reduced chance for large outbreaks. We study two cases, one with discrete number of mutational steps and one with a continuous genetic variable, and we analyze various scaling regimes. For the continuous case we derive a Fokker-Planck-like equation for the probability density and solve it for small numbers of shortcuts using the WKB approximation. Our analysis supports the claims that a potentiating mutation in the transmissibility might occur during an epidemic wave and not necessarily before its initiation.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Complex networks</div> <div class="kwd" title="kwd">
Nonlinear phenomena</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100001659</span><span class="institution" title="institution">Deutsche Forschungsgemeinschaft (German Research Foundation)</span></div> </div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">In the human history, epidemic diseases such as the acquired immune deficiency syndrome (AIDS) and the pandemic influenza have killed millions of people worldwide. Recently, Influenza A (H1N1) spread in 2009, and Ebola spread in the years 2013 to 2016. Not as catastrophic as originally feared, Influenza A, which is genetically close to the Spanish flu of the early 20th century, caused 284,000 deaths<sup><a href="#CR1">1</a></sup>, while Ebola caused 11,300 deaths<sup><a href="#CR2">2</a></sup>. The Influenza epidemics, particularly, proved that a highly contagious new virus could rapidly spread to all continents of the earth in less than a year. Among animals, diseases including the African swine fever have been rapidly expanding and have required expensive steps by the responsible authorities.</p> <p id="Par3">To understand disease spreading and identify possible outbreak control methods, it is valuable to study contact networks and the interaction of the network structure with epidemiology<sup><a href="#CR3">3</a></sup>. The mathematical modeling of dynamical processes on networks has, in the recent years, produced an abundance of results that help to unravel the complexities of epidemic waves and predict the outcomes and risks of transmitting diseases<sup><a href="#CR4">4</a>â€"<a href="#CR7">7</a></sup>. Thus, the physical theory of percolation on networks<sup><a href="#CR8">8</a></sup> was mapped to the spreading of diseases<sup><a href="#CR9">9</a></sup>. This knowledge has been used to determine transmission rates for which a disease becomes endemic<sup><a href="#CR10">10</a></sup> by relating the critical infection rate to the threshold of percolation on networks. Moreover, effective immunization strategies to fight the spreading of a disease have been found by identifying individual nodes that should be made immune based on certain network measures<sup><a href="#CR7">7</a></sup>.</p> <p id="Par4">One important attribute of networks, which appears in many socio-spatial real-world networks, is the small-world property. With it, individual nodes are connected frequently in their local spatial areas<sup><a href="#CR11">11</a>,<a href="#CR12">12</a></sup>, yet they are also connected to distant nodes via a small number of intermediated nodes. Transport networks, which are common substrates for infection spreading, were found to possess the small-world property; particularly those that developed in modern times. For instance, for the spread of human diseases it may be important that individuals fly long distances by plane, which makes the human transport grid to be of small-world type<sup><a href="#CR13">13</a></sup>. Similarly, transmission networks of viruses such as the West Nile virus, which are transmitted in short-range by mosquitos and long-range by birds<sup><a href="#CR14">14</a></sup>, likely exhibit the small-world property.</p> <p id="Par5">Since small-world networks are so ubiquitous in our world, interest in dynamical processes for such networks has grown quickly. This particularly holds for epidemics where spatial relations between hosts (e.g., transport between villages, city, or countries) are relevant. Importantly, it was found that generally small-worldness lowers the threshold for percolation and thus also promotes the global contagion on networks with this property<sup><a href="#CR5">5</a></sup>. We here built on this earlier modeling work for epidemics on small-world networks. We assume a network of nodes which are inhabited by individuals (hosts of the pathogen). Because of transport of individuals between nodes, infecting agents can travel with them to a connected node and infect the nodeâ€™s community. This process is modeled here by the established Susceptible-Infected-Recovered (SIR) model in which, for simplicity, individuals at each node are collectively in one of three states: susceptible (S), infected (I), or recovered or removed (R).</p> <p id="Par6">Additionally, we take into account the genetic mutations and selectivity that a spreading pathogen experiences. Viruses have very high mutation rates<sup><a href="#CR15">15</a></sup> and several mutations can accumulate leading to substantial displacements in their genotype. In fact, mutation rates are so high that the evolution of a virus is interacting with its epidemiological behavior<sup><a href="#CR16">16</a></sup>. We here investigate the case that pathogens, by a series of random mutations, undergo changes of their genes that affect their infection or transmission probability. Such mutations of the infectivity occur for instance during adaptation to new environments or as response to antiviral or antibiotic drugs<sup><a href="#CR17">17</a></sup>. Recently, the possibility that an adaptive mutation during the West African Ebola epidemic increased transmission between humans was described<sup><a href="#CR18">18</a>,<a href="#CR19">19</a></sup>.</p> <p id="Par7">We specifically study the situation where the agentâ€™s infection probability is initially subcritical (in the sense of percolation theory) but may mutate into a state with higher, supercritical infection probability. We find that in networks without small-world links, the probability of an escape to the supercritical state remains small, even if the mutation probability is large. In contrast, if small-world links are present, a supercritical mutation occurs frequently even for moderate mutation rate. The genetic change then entails a transition to a supercritical infection probability, which in turn causes a global infection of almost all nodes of the network. In contrast to the former case of epidemics assisted by small-world links, we here obtain a regime with a high chance of complete infestation of the host species due to the escape process driven by mutations. For larger numbers of long-range links, i.e., for networks approaching the global mixing of random graphs, the sweeping outbreaks disappear again.</p> </div> <div class="mathematicalmodel" title="sec"> <div class="title" tagx="title" title="title">
Mathematical Model</div> <p id="Par8">Our model comprises a two-dimensional spatial network of nodes that adopt one of the three SIR states. A one-dimensional fitness variable is attached to each node that hosts theÂ virus, i.e., that is in the I-state. As motivated above, we have chosen a genetically dependent fitness variable that directly affects the node-to-node transmission probability.</p> <p id="Par9">Generally, the concept of the fitness landscape relates the genotype of an organism to its survival and reproductive success. If the fitness landscape is flat, all genotypes of a population have the same chance of replication. If however, the landscape exhibits regions of different height, with peaks surrounded by deep valleys, a population usually climbs uphill by series of small genetic changes. In our model we specifically consider the generic case where we have two peaks and one valley in the fitness space, i.e., states between the maxima are selectively disadvantageous (Fig.Â <a href="#Fig1">1a</a>). Thus the non-spatial part of our model is similar to point models of stochastic tunneling through a fitness valley<sup><a href="#CR20">20</a>â€"<a href="#CR22">22</a></sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>(<b>a</b>) Functional relation between the infection probability <i>Î»</i> and the mutating variable <i>Î³</i>. The dashed blue line shows the location of the percolation point for a square grid network of nodes. (<b>b</b>) Scheme showing the possible transitions during a time step. Every infected node recovers after a time step (left). A susceptible node that is linked to an infected node becomes infected with probability <i>Î»</i>(<i>Î³</i>), where <i>Î³</i> is the genetic variable of the infected node. The infected node inherits the value <i>Î³</i> that is then mutated to the new value <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\gamma {\prime} $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
â€²</div> </div></div><span class="inline-graphic" title="inline-graphic" /></div> in the same time step. (<b>c</b>) A two-dimensional square lattice where a fraction <i>p</i> of regular links has been replaced by long-distance links. Infections (filled circles) can travel to close neighbors but also to distant nodes. (<b>d</b>) Snapshots of the evolution of the <i>Î³</i> variable on a network with 200 Ã— 200 nodes for p = 0.01 and <i>Ï‡</i> = 0.004 (blue - not infected, green - infected, <i>Î³</i>Â =Â âˆ’1, yellow - infected with <i>Î³</i>Â =Â 1). After 180 iterations the supercritical mutation covers almost all of the area.<p />   </div> <p /> <p id="Par10">In the case of viral epidemics, the two peaks could, for example, correspond to different specified routes of transmission between individual hosts. For instance, in<sup><a href="#CR23">23</a></sup> the possibility of an evolution to a respiratory route in mammals has been studied for the potentially dangerous A/H5N1 Influenza virus. Here, as is typical for viruses<sup><a href="#CR24">24</a></sup>, the number of genetic substitutions that are required for this virus to adapt is low (at 3 to 5). In contrast, bacteria are adapting to new environments slower, which may be linked to the fact that individual mutations have a smaller effect and many mutations may be required<sup><a href="#CR25">25</a></sup>.</p> <p id="Par11">Spatial effects on evolution in fitness landscapes have been considered in<sup><a href="#CR26">26</a></sup> and were generally found to support generation of mutants even if they are less fit. This may also lead to faster adaptation in multi-peaked fitness landscapes since the mutants explore the fitness space quicker. We here propose a model where the spatial structure is more complex by allowing local spatial contacts and, additionally, far-reaching contacts.</p> <div class="sirmodelofepidemicspreading" title="sec"> <div class="title" tagx="title" title="title">
SIR model of epidemic spreading</div> <p id="Par12">The SIR model is one of the simplest models for transmission of diseases in a population of individuals. Every individual is in either one of three SIR states. The transitions between the states are </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">1</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$S\mathop{\to }\limits^{\lambda }I\mathop{\to }\limits^{\mu }R,$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mover_UNKNOWN"> <div class="mo_UNKNOWN">
â†’</div> <div class="mi_UNKNOWN">
Î»</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mover_UNKNOWN"> <div class="mo_UNKNOWN">
â†’</div> <div class="mi_UNKNOWN">
Î¼</div> </div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
,</div></div><div class="graphic" position="anchor" /> </div> </div>where <i>Î»</i> and <i>Î¼</i> are the probabilities of infection and recovery per time step, respectively. Consider now a network of locations on which the infection spreads by specified contacts. Every node of the network contains individuals that are collectively in one of the three states. If a node is susceptible and at least one of its connected nodes is infected, it becomes infected as well with probability <i>Î»</i>, whereas an infected node recovers with probability <i>Î¼</i>. The value of <i>Î»</i> is determined by the states <i>Î³</i> of the infected nodes connected to a susceptible node, see Fig.Â <a href="#Fig1">1b</a>. During each time step, for every S-node we calculate the probability <i>Î»</i>(<i>Î³</i>) for any connected I-node and let the S-node become infected according to this probability. If several connected nodes are infected, we randomly shuffle the order by which the transition is tested. Individuals in state R are immune and cannot be infected again. Note that for simplicity we consider <i>Î»</i> and <i>Î¼</i> as probabilities for a (relatively large) discrete time step, which means that we do not obtain exponential waiting time distribution for the transitions.<p /> <p id="Par13">We initially start with a network of susceptible nodes and a single randomly chosen infected site. After a certain time, the system reaches a stable equilibrium where no infected nodes remain. The fraction of recovered nodes depends on the infection and recovery probabilities <i>Î»</i> and <i>Î¼</i> as well as on the topology of the network. For simplicity, we set the recovery probability to <i>Î¼</i>Â =Â 1 so that an infected node is recovered or removed after one time step.</p> </div> <div class="sirmodelinawatts-strogatz-network" title="sec"> <div class="title" tagx="title" title="title">
SIR Model in a Watts-Strogatz-Network</div> <p id="Par14">To take into account the geographical space om which the disease spreads, we consider a two-dimensional regular square lattice with <i>N</i> nodes and periodic boundary conditions. In such a grid every node has four nearest neighbors and the disease would spread spatially like a wave front. It is known that, in analogy to the problem of percolation on lattices<sup><a href="#CR27">27</a></sup>, for a supercritical infection probability <i>Î»</i>Â &amp;gt;Â 0.5 a nonzero probability for the spreading of the infection exists<sup><a href="#CR28">28</a></sup>.</p> <p id="Par15">Next, every link is rewired to a random node with probability <i>p</i> which generates shortcuts and decreases the average path length. The resulting network is a generalized Watts-Strogatz-Graph<sup><a href="#CR29">29</a></sup> with mean degree <i>k</i>Â =Â 4 (Fig.Â <a href="#Fig1">1c</a>). For very large <i>p</i> close to 1 we obtain a well-mixed random network. It was shown in<sup><a href="#CR5">5</a></sup> that rewiring reduces the threshold of percolation in the network so that the critical <i>Î»</i> is smaller than 0.5.</p> </div> <div class="mutationsoftheinfectionprobability" title="sec"> <div class="title" tagx="title" title="title">
Mutations of the Infection Probability</div> <p id="Par16">In many investigations of SIR on networks, the infection probability <i>Î»</i> is a constant of the dynamical process. Now, incorporating genetic adaptation, we assume that the infection probability can be varied by mutations. The disease could for instance be a virus that spreads from node to node on the network and can undergo mutations of its genome.</p> <p id="Par17">Here we assume that the individuals at each node are collectively at the same genetic state. This may appear unrealistic at first; however, it depends indeed on the respective time scales of mutations, fixations and transmissions to determine how close a population is to this scenario. Given the phylogenetic data for instance for the case of Ebola epidemic in 2013-2016, it appears that the genetic variability of the entire virus population of the outbreak is much larger than the variability at each infection site or country<sup><a href="#CR30">30</a></sup>. Following this line of thought we consider the limit in which establishment of a strain at each node occurs quickly, so that the local population is collectively characterized by a single virus strain. We will study two scenarios, one for pathogens requiring a small number of mutational steps for adaptation and one for pathogens requiring a large number of steps, where a continuous approximation may be valid:</p> <p id="Par18">i) In a first case, we introduce a relation <i>Î»</i><sub>d</sub>(<i>Î³</i>) with the mutating genetic variable <i>Î³</i>, which can have three values: -1 in the initial state (no mutation), 0 in a neutral state (one mutation), 1 in the supercritical state (two mutations). The transmission probability <i>Î»</i><sub>d</sub> depends on <i>Î³</i> as <div class="table-wrap_UNKNOWN" id="Taba"> </div></p><table frame="hsides" rules="groups"> <tbody> <tr> <td><i>Î³</i></td> <td>âˆ’1</td> <td>0</td> <td>1</td> </tr> <tr> <td><i>Î»</i><sub>d</sub></td> <td>0.45</td> <td>0.35</td> <td>0.95</td> </tr> </tbody> </table>  <p /> <p id="Par19">This mapping is chosen so that it has two maxima of optimal fitness. One peak is set slightly below the critical infection probability of 0.5 so that the epidemic wave typically covers at most a smallÂ finite domain. A second peak is chosen well above the critical infection probability corresponding to a well adapted genotype of the species that may typically lead to a global epidemic wave. In keeping with the valley crossing mentioned above, the intermediate state <i>Î³</i>Â =Â 0 presents a minimum in the fitness, with which many of the nodes do notÂ pass on the pathogen. As an initial condition we choose <i>Î³</i>Â =Â âˆ’1.0 at all network nodes and initially infect one randomly chosen node. Then the infection probability is initially below the critical value at <i>Î»</i>Â =Â 0.45. If a node <i>i</i> is in state <i>I</i>, the sequence variable <i>Î³</i><sub><i>i</i></sub> mutates in each iteration step with the rates given by the scheme </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">2</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$-1\underset{{\rm{\chi /2}}}{\overset{{\rm{\chi }}}{\rightleftarrows }}0\underset{{\rm{\chi }}}{\overset{{\rm{\chi /2}}}{\rightleftarrows }}1$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1</div> <div class="munderover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
â‡"</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡/2</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> </div> <div class="mn_UNKNOWN">
0</div> <div class="munderover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
â‡"</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡/2</div> </div> </div> <div class="mn_UNKNOWN">
1</div></div><div class="graphic" position="anchor" /> </div> </div>where the -1, 0, 1 are the state values of <i>Î³</i>. The parameter <i>Ï‡</i> can be interpreted as a mutation rate parameter that quantifies the probability per time step of the respective transitions. The transmission probability <i>Î»</i>(<i>Î³</i><sub><i>i</i></sub>) changes according to the table above. If a node <i>i</i> infects one of its connected sites, the genetic variable <i>Î³</i><sub><i>i</i></sub> is inherited. This value is then, during the same iteration step, mutated. After an infected node transitions to the R-state, its <i>Î³</i><sub><i>i</i></sub> value remains unchanged.<p /> <p id="Par20">ii) In a second scenario we assume a continuous variable <i>Î³</i> and a corresponding function <i>Î»</i><sub>c</sub>(<i>Î³</i>), see Fig.Â <a href="#Fig1">1a</a>. In the literature on evolutionary dynamics, <i>Î³</i> is called the sequence variable and the function <i>Î»</i> can be viewed as the fitness landscape in the sequence space<sup><a href="#CR31">31</a>,<a href="#CR32">32</a></sup>. Usually, the sequence space is high-dimensional, but for simplicity we consider here only a one-dimensional one.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="discretecase" title="sec"> <div class="title" tagx="title" title="title">
Discrete case</div> <p id="Par21">Given the initial infected site and its infection probability <i>Î»</i>, an infection can generally spread through parts of the network. If <i>Î»</i> stays constant, there is a chance that the pathogen infects small or large spatial domains, which can be mathematically described by the established percolation theory. However, as a consequence of the mutations, the infection probability changes and enables more substantial evolutions of the epidemic. If, in this process, the genetic variable <i>Î³</i> evolves towards 1, the epidemic may be able to spread throughout the entire network since <i>Î»</i> is then larger than the critical value of 0.5. Fig.Â <a href="#Fig1">1d</a> shows an exemplary evolution of the genetic variable <i>Î³</i> where most of the network is eventually infected by the mutated pathogen (green and yellow).</p> <p id="Par22">To quantify the effect, we let the system evolve for 10<sup>3</sup> to 10<sup>6</sup> timesteps for each parameter set. For each run we generate a new network with the given parameters. We stop a simulation when the number of R nodes does not change anymore. We then determine the fraction of those runs in which the R-state <i>covers more than</i> 90% of the network, i.e., the outbreak leads to almost complete coverage. This event will be called a sweep in the following.</p> <p id="Par23">We find that the capability to sweep the network depends strongly on the structure of the network (Fig.Â <a href="#Fig2">2a</a>). If no or few long-range links are present and the genetic mutation rate is weak or moderate, the infection wave quickly dies out and overall infection is rare. However, our simulations reveal that for a rewiring rate <i>p</i>Â &amp;gt;Â 0.005 a moderate mutation rate suffices to obtain a large probability of full epidemics. In these epidemics almost all nodes of the network become infected. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Epidemics of the SIR model with discrete mutations of the infection probability <i>Î»</i><sub>d</sub>(<i>Î³</i>) in a square lattice (<i>N</i> = 200 Ã— 200) and a Watts-Strogatz-graph â€" dependence on two parameters: mutation rate <i>Ï‡</i> and rewiring fraction <i>p</i>. (<b>a</b>) The z-axis is the shareÂ of runs in which almost complete coverage of the graph is obtained (number of R-nodes is larger than 90% at final state). (<b>b</b>) ShareÂ of R nodes averaged for 10,000 runs for each parameter set.</p>   </div> <p /> <p id="Par24">Note that this does not mean that there are no infections spreading for small rates or without mutations. In fact, without mutations (i.e., for a constant infection probability <i>Î»</i>Â â‰ˆÂ 0.45) infections can spread. This can happen if, as was mentioned above, shortcuts enable the spreading of infections even if the <i>Î»</i> value predicts a subcritical behavior for a pure grid. To discuss this in detail we now determine the shareÂ of infected nodes per run and average it over all runs. Fig.Â <a href="#Fig2">2b</a> shows that the average number of infected nodes is substantially larger than 0 for <i>p</i>Â &amp;gt;Â 0.005 and all mutation rates. It contrast to the fraction of sweeps, it does not vanish with decreasing <i>Ï‡</i>.</p> <p id="Par25">The difference of our scenario with mutations to this former mutationless case of SIR in small world networks is the generation of an explosive giant component in a fraction of the runs. For the usual percolation, close to the transition point, the size of a giant component is smaller than the total population. Accordingly, for small mutation rate it is found that the distribution of the fraction of infected nodes per run has a peak at a value between 0 and 1, see Fig.Â <a href="#Fig3">3a</a>. This means that in a given run the infection either hits the giant component at, e.g., 50% of all nodes, or the contagion dies out. However, for sufficiently frequent mutations, in each of our small world networks there is either an almost complete coverage (&amp;gt;90%) or only a tiny fraction of the nodes becomes infected and removed, see Fig.Â <a href="#Fig3">3c</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>(<b>a</b>â€"<b>c</b>) Histograms showing the distribution of the number of R nodes in the final steady state per run for a total of 1,000 runs: <i>Ï‡</i> = 0.001 (<b>a</b>), 0.004 (<b>b</b>), 0.01 (<b>c</b>). (<b>d</b>) Histogram of the <i>Î³</i> variable (âˆ’1 or 1, no nodes reside in the 0 state in the final configuration) for the three values of mutation rate <i>Ï‡</i> (<i>p</i> = 0.01).</p>   </div> <p /> <p id="Par26">This points to a very distinctive feature of epidemic outbreaksÂ with critical mutations: If the average fraction of infected nodes is, e.g., 30%, it could either mean that in each run around 30% of the nodes are infected, or, in an extreme case, in 30% of the runs almost all nodes are infected, while in 70% of the runs most nodes stay healthy. From a public health point of view we think that the second case is the more alarming one, since an outbreak of a deadly virus could lead to extinction of the entire population. Therefore, in our model, with short-cuts and random mutations, the outcomes and assessments of risks are different exactly in this respect that the chance of extinction rises dramatically.</p> <p id="Par27">The reason for this different behavior in our mutating SIR model is the fact that for sufficiently large mutation rate the sequence variable <i>Î³</i> can become 1 and settle at the global extremum of the fitness landscape. Fig.Â <a href="#Fig3">3d</a> demonstrates this by comparing a case of small <i>Ï‡</i> (blue bar), moderate <i>Ï‡</i> (green) and large <i>Ï‡</i> (red). Only in the last two cases, we obtain escape from the fitness maximum at <i>Î³</i>Â =Â âˆ’1.</p> <p id="Par28">To understand the causes of the described phenomenon, we now analyze the scaling behavior of the percentage of sweeps in dependence on the mutation rate, <i>Ï‡</i>, and the fraction of shortcuts, <i>p</i>. Fig.Â <a href="#Fig4">4a</a> presents a double-logarithmic plot in which the sweep fraction is plotted versus 1âˆ•<i>Ï‡</i>. There is a clear linear relation on the log-log scales for small <i>Ï‡</i>, which is characterized by a slope independent on <i>p</i>. With increasing <i>p</i>, the lines are shifted to larger values in sweep fraction. For larger mutation rates, the linear dependence is lost and the fraction of sweeps saturates for all <i>p</i>. For very large <i>p</i>, we note that the trend reverses and much a smaller sweep fraction results for a large range in <i>Ï‡</i> (red curve, <i>p</i>Â =Â 0.5). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Discrete mutation model. (<b>a</b>) Percentage of sweeps in its dependence on <i>Ï‡</i> (for indicated <i>p</i> values). (<b>b</b>) The same plot for the mutation probability. The solid line shows the slope of -2 for small <i>Ï‡</i>.</p>   </div> <p /> <p id="Par29">We will now present a theoretical explanation for the observed scaling behavior. This is made easier by splitting the phenomenology of a sweep into occurrence of a critical mutation and a following epidemic wave. We now only consider the probability that during a run at least one critical mutation occurs. Plots of this probability (i.e., the probability that during a run at least one node reaches <i>Î³</i>Â â‰¥Â 0) appear similar to those for the sweep fraction for small <i>p</i>, see Fig.Â <a href="#Fig4">4b</a>. For small <i>Ï‡</i> the slopes of the curves apparently approach -2 (see solid black line for comparison).</p> <p id="Par30">This quadratic dependence on <i>Ï‡</i> can be reasoned in the following way: The probability that at least one of the sites mutates twice to reach <i>Î³</i>Â =Â 1 is </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">3</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${P}_{{\rm{mut}}}=1-{\left(1-\frac{{\chi }^{2}}{2}\right)}^{s}=1-{e}^{s{\rm{ln}}(1-{\chi }^{2}/2)}\approx 1-{e}^{-s{\chi }^{2}/2},$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
mut</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
ln</div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
â‰ˆ</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
s</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
,</div></div><div class="graphic" position="anchor" /> </div> </div>where <i>s</i> is the number of sites belonging to the finite cluster activated by the first infected node. The size of the cluster is distributed according to <i>n</i>(<i>s</i>)â€‰~â€‰<i>s</i><sup>âˆ’<i>Ï"</i></sup> with a universal scaling exponent <i>Ï"</i>Â â‰ˆÂ 2.05<sup><a href="#CR33">33</a></sup>. For very small <i>Ï‡</i> we can further approximate <i>P</i><sub>mut</sub>Â â‰ˆÂ <i>s</i><i>Ï‡</i><sup>2</sup>âˆ•2 and we obtain the mean probability for at least one supercritical mutation: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">4</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${P}_{{\rm{mut}}}\approx \mathop{\sum }\limits_{s=1}^{\infty }sn(s)\frac{s{\chi }^{2}}{2} \sim \frac{{\chi }^{2}}{2}\mathop{\sum }\limits_{s=1}^{\infty }{s}^{2-\tau }.$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
mut</div> </div> </div> <div class="mo_UNKNOWN">
â‰ˆ</div> <div class="munderover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆž</div> </div> </div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
n</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
~</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="munderover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆž</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
Ï"</div> </div> </div> <div class="mo_UNKNOWN">
.</div></div><div class="graphic" position="anchor" /> </div> </div>This scaling with <i>Ï‡</i> is the one observed in Fig.Â <a href="#Fig4">4b</a> (solid black line) for small rates. For larger rates we can use the approximation in Eq. (<a href="#Equ3">3</a>) and evaluate the sum up to a cut-off: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">5</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${P}_{{\rm{m}}{\rm{u}}{\rm{t}}}\sim \mathop{\sum }\limits_{s=1}^{{\rm{\infty }}}sn(s)(1-{e}^{-s{\chi }^{2}/2})$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
âˆ¼</div> <div class="munderover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆž</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
s</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
/</div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
)</div></div><div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">6</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\approx (2-\tau ){\int }_{2/{\chi }^{2}}^{{\rm{\infty }}}{s}^{1-\tau }{\rm{d}}s$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
â‰ˆ</div> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
Ï"</div> <div class="mo_UNKNOWN">
)</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ«</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
/</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆž</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
Ï"</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mi_UNKNOWN">
s</div></div><div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">7</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$={\left(\frac{2}{{\chi }^{2}}\right)}^{2-\tau }.$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‡</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
Ï"</div> </div> </div> <div class="mo_UNKNOWN">
.</div></div><div class="graphic" position="anchor" /> </div> </div>Since 4Â âˆ’Â 2<i>Ï"</i>Â â‰ˆÂ âˆ’Â 0.1 we expect a very weak dependence for larger <i>Ï‡</i>, which is indeed the case in our simulation data. The fact that the percolation behavior predicts the <i>Ï‡</i> dependence of the mutation fraction suggests that the increase of <i>P</i><sub>mut</sub> with <i>p</i> is driven by the percolation transition.<p /> <p id="Par31">Finally, discussing the probability of a sweep following a mutation, Fig.Â <a href="#Fig4">4a</a> shows that the fraction of sweeps declines again for <i>p</i> larger than 0.1 even if the number of critical mutations does not shrink. An obvious scenario is that for large <i>p</i> the giant cluster formed by the non-mutated pathogens (<i>Î³</i>Â =Â âˆ’1) is large and spanning the entire domain quick enough so that much of the network is sufficiently covered with the subcritical infection before the supercritical form can spread. This would particularly effect those cases in which the mutation rate is small, such that a critical mutation needs much time to develop.</p> </div> <div class="continuouscase" title="sec"> <div class="title" tagx="title" title="title">
Continuous case</div> <p id="Par32">For this case, we introduce a functional relation <i>Î»</i>(<i>Î³</i>) with the continuous mutating genetic variable <i>Î³</i>: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">8</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\lambda (\gamma )=\exp (-{\gamma }^{2})(0.55{\gamma }^{4}+0.7{\gamma }^{3}+1.1{\gamma }^{2}+0.25).$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
exp</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0.55</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
4</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.7</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1.1</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.25</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
.</div></div><div class="graphic" position="anchor" /> </div> </div>This exemplary function is shown in Fig.Â <a href="#Fig1">1a</a> and is chosen so that it has two maxima, one below and one above the critical infection probability.<p /> <p id="Par33">As before, we start with <i>Î³</i>Â =Â âˆ’1.0 at all network nodes and initially infect one randomly chosen node. If a node <i>i</i> is in state <i>I</i>, the sequence variable <i>Î³</i><sub><i>i</i></sub> mutates in each iteration step by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">9</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M19">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\gamma }_{i}\to {\gamma }_{i}+{\xi }_{i}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
â†’</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î¾</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div>where <i>Î¾</i><sub><i>i</i></sub> is a Gaussian variable with mean zero andÂ standard deviation <i>Ïƒ</i>.<p /> <p id="Par34">At first glance, we find a similar qualitative behavior as in the discrete case: sweeps in general occur for larger \sigma and <i>p</i> values, see Fig.Â <a href="#Fig5">5a</a>. The mutation probability scales now exponentially with several regimes. For very small noise, the mutation probability disappears exponentially with <i>Ïƒ</i> (data not shown). Similar to the discrete case, the slopes of these graphs do not depend on <i>p</i> in this regime. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Continuous mutation model. (<b>a</b>) Percentage of sweeps in its dependence on <i>Ïƒ</i> (for indicated <i>p</i> values), continuous case. (<b>b</b>) The same plot for the mutation probability. Dashed lines are linear fits. (<b>c</b>) A finite-size effect is visible at large <i>p</i>Â =Â 0.07 comparing a lattice of 300 by 300 points (black) compared to 200 by 200 points (green) and 100 by 100 points (brown) (for <i>p</i>Â = 0, 0.015 and 0.07, from bottom to top).</p>   </div> <p /> <p id="Par35">For larger noise, the scaling depends on the fraction of long bonds. Fig.Â <a href="#Fig5">5a</a> presents the logarithm of the sweep fraction plotted versus the inverse of <i>Ïƒ</i>. There appears an exponential relation for a range of approximately 4Â <div class="â_UNKNOWN">
ÏƒÂ <div class="â_UNKNOWN">
p. For larger rewiring probabilities <i>p</i> there emerges a slight kink for higher values of 1âˆ•<i>Ïƒ</i>. This effect is due to the finite grid size. Simulations with lattices of different sizes show that the large-noise regime extends to higher 1âˆ•<i>Ïƒ</i> for larger lattices (Fig.Â <a href="#Fig5">5c</a>). Results for lattice sizes 200 Ã— 200 and 300 Ã— 300 agree for 1âˆ•<i>Ïƒ</i> values up to 7, which also appear to limit the range of linear fitting.</div></div></p> <p id="Par36">Plots of theÂ mutation probability <i>P</i><sub>mut</sub> exhibit exponential scaling, see Fig.Â <a href="#Fig5">5b</a>. Since we discovered a scaling different from the discrete case, an explanation of the phenomenon should be based on a more complex model. We will approximate the active cluster as a regular tree with a coordination number <i>C</i> (the number of subbranches attached to each node). The evolution of the distribution of <i>Î³</i> values on the tree for a given time can then be approximated by the birth-death equation </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">10</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M21">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\frac{\partial }{\partial t}P={\sigma }^{2}\frac{{\partial }^{2}}{\partial {\gamma }^{2}}P+[C\lambda (\gamma )-1]P$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆ‚</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆ‚</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆ‚</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
âˆ‚</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
Î»</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
]</div> </div> <div class="mi_UNKNOWN">
P</div></div><div class="graphic" position="anchor" /> </div> </div>with <i>P</i>Â =Â <i>P</i>(<i>Î³</i>,Â <i>t</i>). The expression in the square brackets denotes the local proliferation rate. As shown in the SI AppendixÂ <a href="#MOESM1">1</a>, the total probability <i>s</i> of escaping through the value of <i>Î³</i>Â =Â 0 can be obtained by means of a Wentzel-Kramers-Brillouin (WKB) approximation <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">11</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M23">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\rm{\log }}\,s \sim -\frac{1}{\sigma }{\int }_{-1+\epsilon }^{0}\sqrt{r(x)}dx$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
log</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
~</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ«</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Ïµ</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
x</div></div><div class="graphic" position="anchor" /> </div> </div>with <i>r</i>(<i>Î³</i>)Â =Â 1Â âˆ’Â <i>C</i><i>Î»</i>(<i>Î³</i>) being the local decay rate. We recover the observed 1âˆ•<i>Ïƒ</i> dependence with the slope <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M25">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\int }_{-1+\epsilon }^{0}\sqrt{r(x)}dx$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
 âˆ«</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Ïµ</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
x</div></div><span class="inline-graphic" title="inline-graphic" /></div></span>. For small <i>p</i> the coordination number <i>C</i> can be determined by equating the integral to the slopes in Fig.Â <a href="#Fig5">5b</a>. We obtain, for instance, <i>C</i>Â =Â 2.06 for <i>p</i>Â =Â 0.<p /> <p id="Par37">Our numerical simulations of Eq. (<a href="#Equ10">10</a>) showed that the exponential scaling found in the WKB approximation survives for larger <i>C</i> values (data not shown) even if the corresponding physically relevant WKB solution cannot be found. Thus, the model tree network can give a good explanation for the exponential scaling with mutation rate <i>Ïƒ</i>.</p> <p id="Par38">A few words on the limitations of this approximation are in order.Â The dependence of <i>C</i> on the network properties or, in our case, on <i>p</i>, however, remains phenomenological. Additional numerical analysis shows that the described prevalence of supercritical mutations for larger <i>p</i> does not occur in a grid-like network. SI Appendix, Fig. S<a href="#MOESM1">1</a> shows that the probability for supercritical mutations is much smaller for a local network (without short-cuts) even if the mean size of the infected clusters is similarly large as in the small-world case. This suggests that increasing the number of long links in the network helps the mutations by way of (i) increasing the percolation cluster (i.e., lowering the percolation threshold) and (ii) by lifting local blocking of mutated nodes. Thus, we conclude that, besides the increasing percolation cluster, it is the topology of the small-world network that drives the occurrence of mutations andÂ global epidemics in our continuous model.</p> </div> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par39">We have shown that the interaction of epidemic transmissions of a mutating pathogen with the structure of a network can produce a transition to a regime of complete infection of a host species spatiallyÂ distributed on the network. The problem has been framed in terms of stochastic escape from a subcritical transmission probability to a supercritical transmission probability. The escape to the potentially disastrous supercritical regime is facilitated by a small number of long-range links on the spatial grid of hosts. We are thus closing a loop in which transmission enables mutation and mutation enables the spreading of disease on the complete network.</p> <p id="Par40">Our work suggests that risk assessment should be substantially different depending on whether or not mutations are occurring. Under noisy conditions (i.e., allowing mutations) infections are not extensively worse (in a statistical sense) compared to noise-free condition, as the average percentages of infected sites do not change substantially. But mutations are potentially risky, since they are likely to produce a global outbreak in a given population, which is explosive and fast. This makes the latter situation far more difficult and challenging to control epidemiologically.</p> <p id="Par41">The effect that we have described rests qualitatively on two aspects. First, the small-world links drive the network close to or beyond the critical point for percolation. Thus, for <i>p</i> large enough, the infected node often initiates a large cluster, where each node can undergo a mutation and the total probability for a supercritical mutation is high.</p> <p id="Par42">Second, the long-range connections allow the mutated form to emerge and to move freely to uninfected parts of the network, so that the mutation can quickly spread and cover large parts of the network (sweep). The latter, however, only holds if the original type has not covered the entire domain before the mutated type appears. Typically, if <i>p</i> is large, the speed of an epidemic wave is increased. In this situation, the non-mutated type indeed covers large parts of the network before the supercritical pathogens can propagate. Oftentimes, in this situation the non-mutated form remains sparse while covering the network so that the probability of full blown epidemics is much reduced, see Figs.Â <a href="#Fig4">4a,b</a>.</p> <p id="Par43">Qualitatively the behavior we describe is similar for the cases of discrete and continuous mutational spaces. The scalings of mutation probabilities, however, are different for the two cases. While in the discrete case the probability to a supercritical mutant state scales with the mutation rate in power law form, we found exponential behavior for the continuous case. For both models, we presented analytical derivations of these relations. The scalings with the small-worldness parameter <i>p</i>, however, are more complex and our discussion in Appendix S2 gives a first hint on the differences of the continuous and discrete cases. For the limit of a continuous mutation space, our simulations suggest that long-range links additionally help to overcome a local blocking to the spreading of pathogens in intermediate genetic states (see Appendix S2 for more details).</p> <p id="Par44">To fend off disease spreading, our results suggest several approaches. First, in the early stage of the epidemics, one should prevent the critical mutation. This could be achieved by a suppression of long-range links for instance by shutdown of transport hubs or preferential vaccination of regions around long-range connected nodes. In this way the development of isolated regions of the network, where the mutation can proceed without competition, is diminished. Second, after a supercritical pathogen has established, one needs to prevent spreading by removal of links around the supercritical nucleation site. Third, it is also conceivable, that, depending on the topology, an increase of <i>p</i> can result in the spreading of the non-mutated form, thus preventing the infection by the mutated form. This and other questions regarding, for instance, the influence of vaccinations are subject of current work on the problem.</p> <p id="Par45">It is interesting to compare our results with the recent debate whether a potentiating mutation could occur during an epidemic or whether it usually occurs before the start of the epidemic<sup><a href="#CR34">34</a></sup>. For instance, in the case of the Ebola outbreak 2013-2016 it was suggested that one or several mutations identified during the outbreak caused the strong infectivity of the virus<sup><a href="#CR18">18</a>,<a href="#CR19">19</a></sup>. Potentially this question might also determine the chances of a severe mutation of the Ebola virus from a droplet to an airborne transmission route<sup><a href="#CR35">35</a></sup>. It has been argued that within-host selectivity provides a much quicker adaptation than between-host selectivity since the time-scale of selectivity is much shorter within a host than between hosts. In the context of our study, the within-host scenario is one of well-mixed dynamics (large <i>p</i>, see Fig.Â <a href="#Fig4">4a</a>), where we found a domination of the non-mutated form. Only for intermediate small-worldness do we find the mutated form to be competitive. Our results suggest that the decisive mutation benefits from the small-worldness and may thus arise during the epidemic wave on a network with the suitable properties. This finding substantiates the claims for a mutation during the epidemic<sup><a href="#CR18">18</a>,<a href="#CR19">19</a></sup>.</p> <p id="Par46">Finally, the relevance of our work for other network types should be studied in the future. Besides the small-worldness, other topological properties as scale-freeness and degree correlation have been shown to play an important role for epidemic spreading<sup><a href="#CR36">36</a>,<a href="#CR37">37</a></sup> and should therefore be taken into account. We expect that a similar mechanism to the one described here for the case of epidemics may also be relevant for the related problems of spreading on networks of other phenomena such as rumors and influences.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62597_MOESM1_ESM.pdf">LINK</a> <p>Supplement file.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p>is available for this paper at 10.1038/s41598-020-62597-5.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This paper was developed within the scope of the IRTG 1740/TRP 2015/50122-0, funded by the DFG/FAPESP. J.K. was supported by the project RF Goverment Grant 075-15-2019-1885.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>S.R., A.P., F.S., I.M.S. and J.K. designed the study, interpreted the data and wrote the main manuscript text. S.R. and A.P. created the software. S.R. F.S. and I.M.S. derived the equations and analyzed the solutions. I.M.S. and S.R. wrote the supplementary text.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par48">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dawood</span><span class="given-names" tagx="given-names" title="given-names">FS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Estimated global mortality associated with the first 12 months of 2009 pandemic influenza A H1N1 virus circulation: a modelling study</span><span class="source" tagx="source" title="source">The Lancet infectious diseases</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">687</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(12)70121-4">10.1016/S1473-3099(12)70121-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22738893">22738893</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><i>WHO: Ebola situation report 30 March 2016</i> (2016).</span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Danon, L.Â <i>et al</i>. Networks and the epidemiology of infectious disease.Â <b>2011</b> (2011).</span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newman</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemics and percolation in small-world networks</span><span class="source" tagx="source" title="source">Physical Review E. 61</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">5678</span><span class="pub-id"><a href="https://dx.doi.org/10.1103/PhysRevE.61.5678">10.1103/PhysRevE.61.5678</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sander</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warren</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokolov</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simon</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koopman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Percolation on heterogeneous networks as a model for epidemics</span><span class="source" tagx="source" title="source">Mathematical Biosciences</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">180</span><span class="fpage" tagx="fpage" title="fpage">293</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0025-5564(02)00117-7">10.1016/S0025-5564(02)00117-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12387929">12387929</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colizza</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pastor-Satorras</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reaction-diffusion processes and metapopulation models in heterogeneous networks</span><span class="source" tagx="source" title="source">Nature Physics</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">276</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nphys560">10.1038/nphys560</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pastor-Satorras</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castellano</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Mieghem</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemic processes in complex networks</span><span class="source" tagx="source" title="source">Reviews of Modern Physics</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">925</span><span class="pub-id"><a href="https://dx.doi.org/10.1103/RevModPhys.87.925">10.1103/RevModPhys.87.925</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">R. Cohen &amp;amp; S. Havlin, <i>Complex networks: structure, robustness and function</i> (Cambridge university press, 2010).</span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanley</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braunstein</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Unification of theoretical approaches for epidemic spreading on complex networks</span><span class="source" tagx="source" title="source">Reports on Progress in Physics</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">80</span><span class="fpage" tagx="fpage" title="fpage">036603</span><span class="pub-id"><a href="https://dx.doi.org/10.1088/1361-6633/aa5398">10.1088/1361-6633/aa5398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28176679">28176679</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">M. Newman, <i>Networks: an introduction</i> (Oxford university press, 2010).</span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BarthÃ©lemy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spatial networks</span><span class="source" tagx="source" title="source">Physics Reports</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">499</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.physrep.2010.11.002">10.1016/j.physrep.2010.11.002</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">G-Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jusup</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pattern transitions in spatial epidemics: Mechanisms and emergent properties</span><span class="source" tagx="source" title="source">Physics of Life Reviews</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plrev.2016.08.002">10.1016/j.plrev.2016.08.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27567502">27567502</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaral</span><span class="given-names" tagx="given-names" title="given-names">LAN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scala</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barthelemy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanley</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Classes of small-world networks</span><span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">11149</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.200327197">10.1073/pnas.200327197</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sule</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology and ecology of West Nile virus in sub-Saharan Africa</span><span class="source" tagx="source" title="source">Parasites &amp;amp; Vectors</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">414</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13071-018-2998-y">10.1186/s13071-018-2998-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30005653">30005653</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duffy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Why are RNA virus mutation rates so damn high?</span><span class="source" tagx="source" title="source">PLoS Biology</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">e3000003</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.3000003">10.1371/journal.pbio.3000003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30102691">30102691</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis of the dynamics of viral infectious disease</span><span class="source" tagx="source" title="source">Nature Reviews Genetics</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">540</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg2583">10.1038/nrg2583</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatem</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virus evolution and transmission in an ever more connected world</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">282</span><span class="fpage" tagx="fpage" title="fpage">20142878</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2014.2878">10.1098/rspb.2014.2878</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diehl</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ebola virus glycoprotein with increased infectivity dominated the 2013â€"2016 epidemic</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">167</span><span class="fpage" tagx="fpage" title="fpage">1088</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2016.10.014">10.1016/j.cell.2016.10.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27814506">27814506</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urbanowicz</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Human adaptation of Ebola virus during the West African outbreak</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">167</span><span class="fpage" tagx="fpage" title="fpage">1079</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2016.10.013">10.1016/j.cell.2016.10.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27814505">27814505</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwasa</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michor</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Stochastic tunnels in evolutionary dynamics</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">166</span><span class="fpage" tagx="fpage" title="fpage">1571</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.166.3.1571">10.1534/genetics.166.3.1571</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15082570">15082570</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinreich</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid evolutionary escape by large populations from local fitness peaks is likely in nature</span><span class="source" tagx="source" title="source">Evolution</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">1175</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.0014-3820.2005.tb01769.x">10.1111/j.0014-3820.2005.tb01769.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16050095">16050095</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altland</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krug</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szendro</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rare events in population genetics: stochastic tunneling in a two-locus model with recombination</span><span class="source" tagx="source" title="source">Physical Review Letters</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">088101</span><span class="pub-id"><a href="https://dx.doi.org/10.1103/PhysRevLett.106.088101">10.1103/PhysRevLett.106.088101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21405603">21405603</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The potential for respiratory droplet-transmissible A/H5N1 influenza virus to evolve in a mammalian host</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">336</span><span class="fpage" tagx="fpage" title="fpage">1541</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1222526">10.1126/science.1222526</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22723414">22723414</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockhurst</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The evolution and genetics of virus host shifts</span><span class="source" tagx="source" title="source">PLoS Pathogens</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e1004395</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004395">10.1371/journal.ppat.1004395</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25375777">25375777</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonneaud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinert</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuijper</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Understanding the emergence of bacterial pathogens in novel hosts</span><span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society B</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">374</span><span class="fpage" tagx="fpage" title="fpage">20180328</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2018.0328">10.1098/rstb.2018.0328</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neuhauser</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dean</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerr</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tipping the mutation-selection balance: Limited migration increases the frequency of deleterious mutants</span><span class="source" tagx="source" title="source">Journal of Theoretical Biology</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">380</span><span class="fpage" tagx="fpage" title="fpage">123</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2015.05.003">10.1016/j.jtbi.2015.05.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25983046">25983046</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">A. Bunde &amp;amp; Havlin S.Â Percolation I inÂ <i>Fractals and Disordered Systems</i> (Springer, 1996), pp. 59â€"114.</span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newman</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spread of epidemic disease on networks</span><span class="source" tagx="source" title="source">Physical Review E</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">016128</span><span class="pub-id"><a href="https://dx.doi.org/10.1103/PhysRevE.66.016128">10.1103/PhysRevE.66.016128</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watts</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strogatz</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Collective dynamics of small-world networks</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">393</span><span class="fpage" tagx="fpage" title="fpage">440</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/30918">10.1038/30918</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9623998">9623998</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudas</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span></span><span class="mixed-article-title" title="mixed-article-title">The evolution of Ebola virus: Insights from the 2013â€"2016 epidemic</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">538</span><span class="fpage" tagx="fpage" title="fpage">193</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature19790">10.1038/nature19790</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27734858">27734858</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">M. A. Nowak, <i>Evolutionary dynamics</i> (Harvard University Press, 2006).</span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Visser</span><span class="given-names" tagx="given-names" title="given-names">JAG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krug</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Empirical fitness landscapes and the predictability of evolution</span><span class="source" tagx="source" title="source">Nature Reviews Genetics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">480</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg3744">10.1038/nrg3744</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">D. Stauffer &amp;amp; A. Aharony, <i>Introduction to percolation theory</i> (Taylor &amp;amp; Francis, 2014).</span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malik</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Did a single amino acid change make Ebola virus more virulent?</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">167</span><span class="fpage" tagx="fpage" title="fpage">892</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2016.10.032">10.1016/j.cell.2016.10.032</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27814518">27814518</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Judson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prescott</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munster</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">Understanding ebola virus transmission</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">511</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v7020511">10.3390/v7020511</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25654239">25654239</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Infection dynamics on scale-free networks</span><span class="source" tagx="source" title="source">Physical Review E</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">066112</span><span class="pub-id"><a href="https://dx.doi.org/10.1103/PhysRevE.64.066112">10.1103/PhysRevE.64.066112</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Edge-based epidemic spreading in degree-correlated complex networks</span><span class="source" tagx="source" title="source">Journal of Theoretical Biology</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">454</span><span class="fpage" tagx="fpage" title="fpage">164</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2018.06.006">10.1016/j.jtbi.2018.06.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29885412">29885412</a></span></span></li> </ul> </div> </div>  </body></html>