<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-2.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/05/2020.04.02.019075.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.02.019075v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rapid community-driven development of a SARS-CoV-2 tissue simulator" />
<meta name="DC.Identifier" content="10.1101/2020.04.02.019075" />
<meta name="DC.Date" content="2020-04-05" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The 2019 novel coronavirus, SARS-CoV-2, is an emerging pathogen of critical significance to international public health. Knowledge of the interplay between molecular-scale virus-receptor interactions, single-cell viral replication, intracellular-scale viral transport, and emergent tissue-scale viral propagation is limited. Moreover, little is known about immune system-virus-tissue interactions and how these can result in low-level (asymptomatic) infections in some cases and acute respiratory distress syndrome (ARDS) in others, particularly with respect to presentation in different age groups or pre-existing inflammatory risk factors like diabetes. A critical question for treatment and protection is why it appears that the severity of infection may correlate with the initial level of virus exposure. Given the nonlinear interactions within and among each of these processes, multiscale simulation models can shed light on the emergent dynamics that lead to divergent outcomes, identify actionable “choke points” for pharmacologic interactions, screen potential therapies, and identify potential biomarkers that differentiate response dynamics. Given the complexity of the problem and the acute need for an actionable model to guide therapy discovery and optimization, we introduce a prototype of a multiscale model of SARS-CoV-2 dynamics in lung and intestinal tissue that will be iteratively refined. The first prototype model was built and shared internationally as open source code and interactive, cloud-hosted executables in under 12 hours. In a sustained community effort, this model will integrate data and expertise across virology, immunology, mathematical biology, quantitative systems physiology, cloud and high performance computing, and other domains to accelerate our response to this critical threat to international health.

Note This is a rapid prototyping project. For the very latest, see &lt;http://covid19.physicell.org&gt;" />
<meta name="DC.Contributor" content="Yafei Wang" />
<meta name="DC.Contributor" content="Randy Heiland" />
<meta name="DC.Contributor" content="Morgan Craig" />
<meta name="DC.Contributor" content="Courtney Davis" />
<meta name="DC.Contributor" content="Ashlee N. Ford Versypt" />
<meta name="DC.Contributor" content="Adrianne Jenner" />
<meta name="DC.Contributor" content="Jonathan Ozik" />
<meta name="DC.Contributor" content="Nicholson Collier" />
<meta name="DC.Contributor" content="Chase Cockrell" />
<meta name="DC.Contributor" content="Andrew Becker" />
<meta name="DC.Contributor" content="Gary An" />
<meta name="DC.Contributor" content="James A. Glazier" />
<meta name="DC.Contributor" content="Aarthi Narayanan" />
<meta name="DC.Contributor" content="Amber M. Smith" />
<meta name="DC.Contributor" content="Paul Macklin" />
<meta name="article:published_time" content="2020-04-05" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Rapid community-driven development of a SARS-CoV-2 tissue simulator" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The 2019 novel coronavirus, SARS-CoV-2, is an emerging pathogen of critical significance to international public health. Knowledge of  the interplay between molecular-scale virus-receptor interactions, single-cell viral replication, intracellular-scale viral transport, and emergent tissue-scale viral propagation is limited. Moreover, little is known about immune system-virus-tissue interactions and how these can result in low-level (asymptomatic) infections in some cases and acute respiratory distress syndrome (ARDS) in others, particularly with respect to presentation in different age groups or pre-existing inflammatory risk factors like diabetes. A critical question for treatment and protection is why it appears that the severity of infection may correlate with the initial level of virus exposure. Given the nonlinear interactions within and among each of these processes, multiscale simulation models can shed light on the emergent dynamics that lead to divergent outcomes, identify actionable &quot;choke points&quot; for pharmacologic interactions, screen potential therapies, and identify potential biomarkers that differentiate response dynamics. Given the complexity of the problem and the acute need for an actionable model to guide therapy discovery and optimization, we introduce a prototype of a multiscale model of SARS-CoV-2 dynamics in lung and intestinal tissue that will be iteratively refined. The first prototype model was built and shared internationally as open source code and interactive, cloud-hosted executables in under 12 hours. In a sustained community effort, this model will integrate data and expertise across virology, immunology, mathematical biology, quantitative systems physiology, cloud and high performance computing, and other domains to accelerate our response to this critical threat to international health.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.02.019075v1" />
<meta name="citation_id" content="2020.04.02.019075v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.02.019075" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.02.019075" />
<meta name="citation_author" content="Yafei Wang" />
<meta name="citation_author_institution" content="Department of Intelligent Systems Engineering, Indiana University. Bloomington" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6594-0608" />
<meta name="citation_author" content="Randy Heiland" />
<meta name="citation_author_institution" content="Department of Intelligent Systems Engineering, Indiana University. Bloomington" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7440-2905" />
<meta name="citation_author" content="Morgan Craig" />
<meta name="citation_author_institution" content="Department of Mathematics, University of Montreal. Montreal" />
<meta name="citation_author_institution" content="CHU Sainte-Justine Research Centre" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4852-4770" />
<meta name="citation_author" content="Courtney Davis" />
<meta name="citation_author_institution" content="Natural Science Division, Pepperdine University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3274-5707" />
<meta name="citation_author" content="Ashlee N. Ford Versypt" />
<meta name="citation_author_institution" content="School of Chemical Engineering, Oklahoma State University" />
<meta name="citation_author_institution" content="Oklahoma Center for Respiratory and Infectious Diseases" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9059-5703" />
<meta name="citation_author" content="Adrianne Jenner" />
<meta name="citation_author_institution" content="Department of Mathematics, University of Montreal. Montreal" />
<meta name="citation_author_institution" content="CHU Sainte-Justine Research Centre" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9103-7092" />
<meta name="citation_author" content="Jonathan Ozik" />
<meta name="citation_author_institution" content="Decision and Infrastructure Sciences, Argonne National Laboratory" />
<meta name="citation_author_institution" content="Consortium for Advanced Science and Engineering, University of Chicago" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3495-6735" />
<meta name="citation_author" content="Nicholson Collier" />
<meta name="citation_author_institution" content="Decision and Infrastructure Sciences, Argonne National Laboratory" />
<meta name="citation_author_institution" content="Consortium for Advanced Science and Engineering, University of Chicago" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2376-4156" />
<meta name="citation_author" content="Chase Cockrell" />
<meta name="citation_author_institution" content="The University of Vermont Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3224-7617" />
<meta name="citation_author" content="Andrew Becker" />
<meta name="citation_author_institution" content="The University of Vermont Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5406-5545" />
<meta name="citation_author" content="Gary An" />
<meta name="citation_author_institution" content="The University of Vermont Medical Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4549-9004" />
<meta name="citation_author" content="James A. Glazier" />
<meta name="citation_author_institution" content="Department of Intelligent Systems Engineering, Indiana University. Bloomington" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3634-190X" />
<meta name="citation_author" content="Aarthi Narayanan" />
<meta name="citation_author_institution" content="National Center for Biodefense and Infectious Disease, George Mason University. Manassas" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4889-4416" />
<meta name="citation_author" content="Amber M. Smith" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Tennessee Health Science Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7092-6904" />
<meta name="citation_author" content="Paul Macklin" />
<meta name="citation_author_institution" content="Department of Intelligent Systems Engineering, Indiana University. Bloomington" />
<meta name="citation_author_email" content="macklinp@iu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9925-0151" />
<meta name="citation_reference" content="Dong, E., Du, H. &amp; Gardner, L. An interactive web-based dashboard to track COVID-19 in real time. The Lancet Infectious Diseases, doi:10.1016/s1473-3099(20)30120-1 (2020)." />
<meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature 579, 265–269, doi:10.1038/s41586-020-2008-3 (2020)." />
<meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273, doi:10.1038/s41586-020-2012-7 (2020)." />
<meta name="citation_reference" content="Bouadma, L., Lescure, F.-X., Lucet, J.-C., Yazdanpanah, Y. &amp; Timsit, J.-F. Severe SARS-CoV-2 infections: practical considerations and management strategy for intensivists. Intensive Care Medicine 46, 579–582, doi:10.1007/s00134-020-05967-x (2020)." />
<meta name="citation_reference" content="Yang, X. et al. Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study. The Lancet Respiratory Medicine, doi:10.1016/s2213-2600(20)30079-5 (2020)." />
<meta name="citation_reference" content="Zhou, F. et al. Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study. The Lancet 395, 1054–1062, doi:10.1016/s0140-6736(20)30566-3 (2020)." />
<meta name="citation_reference" content="Zhang, J.-j. et al. Clinical characteristics of 140 patients infected with SARS-CoV-2 in Wuhan, China. Allergy, doi:10.1111/all.14238 (2020)." />
<meta name="citation_reference" content="Liang, W. et al. Cancer patients in SARS-CoV-2 infection: a nationwide analysis in China. The Lancet Oncology 21, 335–337, doi:10.1016/s1470-2045(20)30096-6 (2020)." />
<meta name="citation_reference" content="Macklin, P. When Seeing Isn’t Believing: How Math Can Guide Our Interpretation of Measurements and Experiments. Cell Systems 5, 92–94, doi:10.1016/j.cels.2017.08.005 (2017)." />
<meta name="citation_reference" content="Macklin, P. Key challenges facing data-driven multicellular systems biology. GigaScience 8, doi:10.1093/gigascience/giz127 (2019)." />
<meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 367, 1260–1263, doi:10.1126/science.abb2507 (2020)." />
<meta name="citation_reference" content="Sahin, A. R. 2019 Novel Coronavirus (COVID-19) Outbreak: A Review of the Current Literature. Eurasian Journal of Medicine and Oncology, doi:10.14744/ejmo.2020.12220 (2020)." />
<meta name="citation_reference" content="White, K. A., Enjuanes, L. &amp; Berkhout, B. RNA virus replication, transcription and recombination. RNA Biology 8, 182–183, doi:10.4161/rna.8.2.15663 (2014)." />
<meta name="citation_reference" content="Stark, G. R., Kerr, I. M., Williams, B. R. G., Silverman, R. H. &amp; Schreiber, R. D. How Cells Respond to Interferons. Annual Review of Biochemistry 67, 227–264, doi:10.1146/annurev.biochem.67.1.227 (1998)." />
<meta name="citation_reference" content="Perry, A. K., Chen, G., Zheng, D., Tang, H. &amp; Cheng, G. The host type I interferon response to viral and bacterial infections. Cell Research 15, 407–422, doi:10.1038/sj.cr.7290309 (2005)." />
<meta name="citation_reference" content="Danthi, P. Viruses and the Diversity of Cell Death. Annual Review of Virology 3, 533–553, doi:10.1146/annurev-virology-110615-042435 (2016)." />
<meta name="citation_reference" content="Yue, Y. et al. SARS-Coronavirus Open Reading Frame-3a drives multimodal necrotic cell death. Cell Death &amp; Disease 9, doi:10.1038/s41419-018-0917-y (2018)." />
<meta name="citation_reference" content="Keck, F. et al. Altered mitochondrial dynamics as a consequence of Venezuelan Equine encephalitis virus infection. Virulence 8, 1849–1866, doi:10.1080/21505594.2016.1276690 (2017)." />
<meta name="citation_reference" content="Keck, F. et al. Mitochondrial-Directed Antioxidant Reduces Microglial-Induced Inflammation in Murine In Vitro Model of TC-83 Infection. Viruses 10, doi:10.3390/v10110606 (2018)." />
<meta name="citation_reference" content="Thiel, V. et al. Lack of Innate Interferon Responses during SARS Coronavirus Infection in a Vaccination and Reinfection Ferret Model. PLoS ONE 7, e45842, doi:10.1371/journal.pone.0045842 (2012)." />
<meta name="citation_reference" content="Widagdo, W., Okba, N. M. A., Stalin Raj, V. &amp; Haagmans, B. L. MERS-coronavirus: From discovery to intervention. One Health 3, 11–16, doi:10.1016/j.onehlt.2016.12.001 (2017)." />
<meta name="citation_reference" content="Channappanavar, R. et al. Dysregulated Type I Interferon and Inflammatory Monocyte-Macrophage Responses Cause Lethal Pneumonia in SARS-CoV-Infected Mice. Cell Host &amp; Microbe 19, 181–193, doi:10.1016/j.chom.2016.01.007 (2016)." />
<meta name="citation_reference" content="Channappanavar, R. et al. IFN-I response timing relative to virus replication determines MERS coronavirus infection outcomes. Journal of Clinical Investigation 129, 3625–3639, doi:10.1172/jci126363 (2019)." />
<meta name="citation_reference" content="Al-Hazmi, A. Challenges presented by MERS corona virus, and SARS corona virus to global health. Saudi Journal of Biological Sciences 23, 507–511, doi:10.1016/j.sjbs.2016.02.019 (2016)." />
<meta name="citation_reference" content="Qin, C. et al. Dysregulation of Immune Response in Patients with COVID-19 in Wuhan, China. SSRN Electronic Journal, doi:10.2139/ssrn.3541136 (2020)." />
<meta name="citation_reference" content="Prompetchara, E., Ketloy, C. &amp; Palaga, T. Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic. Asian Pacific Journal of Allergy and Immunology, doi:10.12932/ap-200220-0772 (2020)." />
<meta name="citation_reference" content="Channappanavar, R. &amp; Perlman, S. Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology. Seminars in Immunopathology 39, 529–539, doi:10.1007/s00281-017-0629-x (2017)." />
<meta name="citation_reference" content="Nahrendorf, M., Pittet, M. J. &amp; Swirski, F. K. Monocytes: Protagonists of Infarct Inflammation and Repair After Myocardial Infarction. Circulation 121, 2437–2445, doi:10.1161/circulationaha.109.916346 (2010)." />
<meta name="citation_reference" content="1 29 Camp, J. V. &amp; Jonsson, C. B. A Role for Neutrophils in Viral Respiratory Disease. Frontiers in Immunology 8, doi:10.3389/fimmu.2017.00550 (2017)." />
<meta name="citation_reference" content="Weitnauer, M., Mijošek, V. &amp; Dalpke, A. H. Control of local immunity by airway epithelial cells. Mucosal Immunology 9, 287–298, doi:10.1038/mi.2015.126 (2015)." />
<meta name="citation_reference" content="Crane, M. J., Lee, K. M., FitzGerald, E. S. &amp; Jamieson, A. M. Surviving Deadly Lung Infections: Innate Host Tolerance Mechanisms in the Pulmonary System. Frontiers in Immunology 9, doi:10.3389/fimmu.2018.01421 (2018)." />
<meta name="citation_reference" content="Teijaro John R. et al. Endothelial Cells Are Central Orchestrators of Cytokine Amplification during Influenza Virus Infection. Cell 146, 980–991, doi:10.1016/j.cell.2011.08.015 (2011)." />
<meta name="citation_reference" content="Shokri, S., Mahmoudvand, S., Taherkhani, R. &amp; Farshadpour, F. Modulation of the immune response by Middle East respiratory syndrome coronavirus. Journal of Cellular Physiology 234, 2143–2151, doi:10.1002/jcp.27155 (2018)." />
<meta name="citation_reference" content="Kong, W. p. et al. Modulation of the Immune Response to the Severe Acute Respiratory Syndrome Spike Glycoprotein by Gene-Based and Inactivated Virus Immunization. Journal of Virology 79, 13915–13923, doi:10.1128/jvi.79.22.13915-13923.2005 (2005)." />
<meta name="citation_reference" content="Cruz, J. L. G. et al. Alphacoronavirus Protein 7 Modulates Host Innate Immune Response. Journal of Virology 87, 9754–9767, doi:10.1128/jvi.01032-13 (2013)." />
<meta name="citation_reference" content="Liu, L. et al. Anti–spike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection. JCI Insight 4, doi:10.1172/jci.insight.123158 (2019)." />
<meta name="citation_reference" content="Leong, A. S. Y. et al. Multiple organ infection and the pathogenesis of SARS. Journal of Experimental Medicine 202, 415–424, doi:10.1084/jem.20050828 (2005)." />
<meta name="citation_reference" content="Yang, P. et al. Angiotensin-converting enzyme 2 (ACE2) mediates influenza H7N9 virus-induced acute lung injury. Scientific Reports 4, doi:10.1038/srep07027 (2014)." />
<meta name="citation_reference" content="Kuba, K. et al. A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus–induced lung injury. Nature Medicine 11, 875–879, doi:10.1038/nm1267 (2005)." />
<meta name="citation_reference" content="Stebbing, J. et al. COVID-19: combining antiviral and anti-inflammatory treatments. The Lancet Infectious Diseases 20, 400–402, doi:10.1016/s1473-3099(20)30132-8 (2020)." />
<meta name="citation_reference" content="Sanders, C. J. et al. Compromised respiratory function in lethal influenza infection is characterized by the depletion of type I alveolar epithelial cells beyond threshold levels. American Journal of Physiology- Lung Cellular and Molecular Physiology 304, L481–L488, doi:10.1152/ajplung.00343.2012 (2013)." />
<meta name="citation_reference" content="Allen, R. J., Rieger, T. R. &amp; Musante, C. J. Efficient Generation and Selection of Virtual Populations in Quantitative Systems Pharmacology Models. CPT: Pharmacometrics &amp; Systems Pharmacology 5, 140–146, doi:10.1002/psp4.12063 (2016)." />
<meta name="citation_reference" content="Cassidy, T. &amp; Craig, M. Determinants of combination GM-CSF immunotherapy and oncolytic virotherapy success identified through in silico treatment personalization. PLOS Computational Biology 15, doi:10.1371/journal.pcbi.1007495 (2019)." />
<meta name="citation_reference" content="Nowak, M. et al. Antigenic diversity thresholds and the development of AIDS. Science 254, 963–969, doi:10.1126/science.1683006 (1991)." />
<meta name="citation_reference" content="Wei, X. et al. Viral dynamics in human immunodeficiency virus type 1 infection. Nature 373, 117–122, doi:10.1038/373117a0 (1995)." />
<meta name="citation_reference" content="Rosenbloom, D. I. S., Hill, A. L., Laskey, S. B. &amp; Siliciano, R. F. Re-evaluating evolution in the HIV reservoir. Nature 551, E6–E9, doi:10.1038/nature24634 (2017)." />
<meta name="citation_reference" content="Reeves, D. B. et al. A majority of HIV persistence during antiretroviral therapy is due to infected cell proliferation. Nature Communications 9, doi:10.1038/s41467-018-06843-5 (2018)." />
<meta name="citation_reference" content="Koelle, K., Farrell, A. P., Brooke, C. B. &amp; Ke, R. Within-host infectious disease models accommodating cellular coinfection, with an application to influenza†. Virus Evolution 5, doi:10.1093/ve/vez018 (2019)." />
<meta name="citation_reference" content="Perelson, A. S., Neumann, A. U., Markowitz, M., Leonard, J. M. &amp; Ho, D. D. HIV-1 Dynamics in Vivo: Virion Clearance Rate, Infected Cell Life-Span, and Viral Generation Time. Science 271, 1582–1586, doi:10.1126/science.271.5255.1582 (1996)." />
<meta name="citation_reference" content="Smith, A. M. &amp; Ribeiro, R. M. Modeling the Viral Dynamics of Influenza A Virus Infection. Critical Reviews(tm) in Immunology 30, 291–298, doi:10.1615/CritRevImmunol.v30.i3.60 (2010)." />
<meta name="citation_reference" content="Smith, A. M. &amp; Perelson, A. S. Influenza A virus infection kinetics: quantitative data and models. Wiley Interdisciplinary Reviews: Systems Biology and Medicine 3, 429–445, doi:10.1002/wsbm.129 (2011)." />
<meta name="citation_reference" content="Smith, A. M., McCullers, J. A. &amp; Adler, F. R. Mathematical model of a three-stage innate immune response to a pneumococcal lung infection. Journal of Theoretical Biology 276, 106–116, doi:10.1016/j.jtbi.2011.01.052 (2011)." />
<meta name="citation_reference" content="Bonhoeffer, S., May, R. M., Shaw, G. M. &amp; Nowak, M. A. Virus dynamics and drug therapy. Proceedings of the National Academy of Sciences 94, 6971–6976, doi:10.1073/pnas.94.13.6971 (1997)." />
<meta name="citation_reference" content="Hill, A. L., Rosenbloom, D. I. S., Fu, F., Nowak, M. A. &amp; Siliciano, R. F. Predicting the outcomes of treatment to eradicate the latent reservoir for HIV-1. Proceedings of the National Academy of Sciences 111, 13475–13480, doi:10.1073/pnas.1406663111 (2014)." />
<meta name="citation_reference" content="Rosenbloom, D. I. S., Hill, A. L., Rabi, S. A., Siliciano, R. F. &amp; Nowak, M. A. Antiretroviral dynamics determines HIV evolution and predicts therapy outcome. Nature Medicine 18, 1378–1385, doi:10.1038/nm.2892 (2012)." />
<meta name="citation_reference" content="Mueller, S. N. et al. Mathematical Modeling Predicts that Increased HSV-2 Shedding in HIV-1 Infected Persons Is Due to Poor Immunologic Control in Ganglia and Genital Mucosa. Plos One 11, e0155124, doi:10.1371/journal.pone.0155124 (2016)." />
<meta name="citation_reference" content="Perelson, A. S. et al. Decay characteristics of HIV-1-infected compartments during combination therapy. Nature 387, 188–191, doi:10.1038/387188a0 (1997)." />
<meta name="citation_reference" content="Kirtane, A. R. et al. Development of an oral once-weekly drug delivery system for HIV antiretroviral therapy. Nature Communications 9, doi:10.1038/s41467-017-02294-6 (2018)." />
<meta name="citation_reference" content="Schiffer, J. T. &amp; Gottlieb, S. L. Biologic interactions between HSV-2 and HIV-1 and possible implications for HSV vaccine development. Vaccine 37, 7363–7371, doi:10.1016/j.vaccine.2017.09.044 (2019)." />
<meta name="citation_reference" content="Schiffer, J. T. et al. Mathematical modeling of herpes simplex virus-2 suppression with pritelivir predicts trial outcomes. Science Translational Medicine 8, 324ra315–324ra315, doi:10.1126/scitranslmed.aad6654 (2016)." />
<meta name="citation_reference" content="Perelson, A. S. Modelling viral and immune system dynamics. Nature Reviews Immunology 2, 28–36, doi:10.1038/nri700 (2002)." />
<meta name="citation_reference" content="Hill, A. L., Rosenbloom, D. I. S., Nowak, M. A. &amp; Siliciano, R. F. Insight into treatment of HIV infection from viral dynamics models. Immunological Reviews 285, 9–25, doi:10.1111/imr.12698 (2018)." />
<meta name="citation_reference" content="Wang, Y., Zhou, Y., Brauer, F. &amp; Heffernan, J. M. Viral dynamics model with CTL immune response incorporating antiretroviral therapy. Journal of Mathematical Biology 67, 901–934, doi:10.1007/s00285-012-0580-3 (2012)." />
<meta name="citation_reference" content="Schiffer, J. T. &amp; Corey, L. Rapid host immune response and viral dynamics in herpes simplex virus-2 infection. Nature Medicine 19, 280–288, doi:10.1038/nm.3103 (2013)." />
<meta name="citation_reference" content="Jenner, A. L., Yun, C.-O., Kim, P. S. &amp; Coster, A. C. F. Mathematical Modelling of the Interaction Between Cancer Cells and an Oncolytic Virus: Insights into the Effects of Treatment Protocols. Bulletin of Mathematical Biology 80, 1615–1629, doi:10.1007/s11538-018-0424-4 (2018)." />
<meta name="citation_reference" content="Möhler, L., Flockerzi, D., Sann, H. &amp; Reichl, U. Mathematical model of influenza A virus production in large-scale microcarrier culture. Biotechnology and Bioengineering 90, 46–58, doi:10.1002/bit.20363 (2005)." />
<meta name="citation_reference" content="Schulze-Horsel, J., Schulze, M., Agalaridis, G., Genzel, Y. &amp; Reichl, U. Infection dynamics and virusinduced apoptosis in cell culture-based influenza vaccine production—Flow cytometry and mathematical modeling. Vaccine 27, 2712–2722, doi:10.1016/j.vaccine.2009.02.027 (2009)." />
<meta name="citation_reference" content="Banks, H. T., Bortz, D. M. &amp; Holte, S. E. Incorporation of variability into the modeling of viral delays in HIV infection dynamics. Mathematical Biosciences 183, 63–91, doi:10.1016/s0025-5564(02)00218-3 (2003)." />
<meta name="citation_reference" content="Culshaw, R. V., Ruan, S. &amp; Webb, G. A mathematical model of cell-to-cell spread of HIV-1 that includes a time delay. Journal of Mathematical Biology 46, 425–444, doi:10.1007/s00285-002-0191-5 (2003)." />
<meta name="citation_reference" content="Li, M. Y. &amp; Shu, H. Impact of Intracellular Delays and Target-Cell Dynamics on In Vivo Viral Infections. SIAM Journal on Applied Mathematics 70, 2434–2448, doi:10.1137/090779322 (2010)." />
<meta name="citation_reference" content="Smith, A. M. Validated models of immune response to virus infection. Current Opinion in Systems Biology 12, 46–52, doi:10.1016/j.coisb.2018.10.005 (2018)." />
<meta name="citation_reference" content="Smith, A. M. Host-pathogen kinetics during influenza infection and coinfection: insights from predictive modeling. Immunological Reviews 285, 97–112, doi:10.1111/imr.12692 (2018)." />
<meta name="citation_reference" content="Guedj, J. et al. Modeling shows that the NS5A inhibitor daclatasvir has two modes of action and yields a shorter estimate of the hepatitis C virus half-life. Proceedings of the National Academy of Sciences 110, 3991–3996, doi:10.1073/pnas.1203110110 (2013)." />
<meta name="citation_reference" content="Heldt, F. S., Frensing, T. &amp; Reichl, U. Modeling the Intracellular Dynamics of Influenza Virus Replication To Understand the Control of Viral RNA Synthesis. Journal of Virology 86, 7806–7817, doi:10.1128/jvi.00080-12 (2012)." />
<meta name="citation_reference" content="Sidorenko, Y. &amp; Reichl, U. Structured model of influenza virus replication in MDCK cells. Biotechnology and Bioengineering 88, 1–14, doi:10.1002/bit.20096 (2004)." />
<meta name="citation_reference" content="Fachada, N., Lopes, V. V. &amp; Rosa, A. Simulating antigenic drift and shift in influenza A. 2093, doi:10.1145/1529282.1529744 (2009)." />
<meta name="citation_reference" content="Bauer, A. L., Beauchemin, C. A. A. &amp; Perelson, A. S. Agent-based modeling of host–pathogen systems: The successes and challenges. Information Sciences 179, 1379–1389, doi:10.1016/j.ins.2008.11.012 (2009)." />
<meta name="citation_reference" content="Wilke, C. O. et al. Complex Spatial Dynamics of Oncolytic Viruses In Vitro: Mathematical and Experimental Approaches. PLoS Computational Biology 8, e1002547, doi:10.1371/journal.pcbi.1002547 (2012)." />
<meta name="citation_reference" content="Sun, G.-Q. et al. Spatiotemporal Dynamics of Virus Infection Spreading in Tissues. Plos One 11, e0168576, doi:10.1371/journal.pone.0168576 (2016)." />
<meta name="citation_reference" content="Jenner, A. L., Frascoli, F., Coster, A. C. F. &amp; Kim, P. S. Enhancing oncolytic virotherapy: Observations from a Voronoi Cell-Based model. Journal of Theoretical Biology 485, 110052, doi:10.1016/j.jtbi.2019.110052 (2020)." />
<meta name="citation_reference" content="Quintela, B. M. et al. An Age-based Multiscale Mathematical Model of the Hepatitis C Virus Life-cycle During Infection and Therapy: Including Translation and Replication. 60, 508–511, doi:10.1007/978-981-10-4086-3_128 (2017)." />
<meta name="citation_reference" content="Ghaffarizadeh, A., Heiland, R., Friedman, S. H., Mumenthaler, S. M. &amp; Macklin, P. PhysiCell: An open source physics-based cell simulator for 3-D multicellular systems. PLoS Comput Biol 14, e1005991, doi:10.1371/journal.pcbi.1005991 (2018)." />
<meta name="citation_reference" content="Hillen, T. COVID-19 Physiology Group, &lt;https://sites.google.com/ualberta.ca/cov-pg/home&gt; (2020)." />
<meta name="citation_reference" content="COVID-19 Cell Atlas, &lt;https://www.covid19cellatlas.org/&gt; (2020)." />
<meta name="citation_reference" content="Hu, B. C. The human body at cellular resolution: the NIH Human Biomolecular Atlas Program. Nature 574, 187–192, doi:10.1038/s41586-019-1629-x (2019)." />
<meta name="citation_reference" content="Metzcar, J., Wang, Y., Heiland, R. &amp; Macklin, P. A Review of Cell-Based Computational Modeling in Cancer Biology. JCO Clin Cancer Inform 3, 1–13, doi:10.1200/CCI.18.00069 (2019)." />
<meta name="citation_reference" content="Ghaffarizadeh, A., Friedman, S. H. &amp; Macklin, P. BioFVM: an efficient, parallelized diffusive transport solver for 3-D biological simulations. Bioinformatics 32, 1256–1258, doi:10.1093/bioinformatics/btv730 (2016)." />
<meta name="citation_reference" content="Wang, Y., Heiland, R. &amp; Macklin, P. pc4nanobio: cancer nanotherapy simulator [nanoHUB app, Version 0.9.1], &lt;https://nanohub.org/resources/pc4nanobio&gt; (2019)." />
<meta name="citation_reference" content="Jenner, A. L. Replication Competent Oncolytic Virus expressing secretable trimeric TRAIL: hypothesis testing [nanoHUB app, Version 0.3], &lt;https://nanohub.org/resources/iu399sp19p101&gt; (2019)." />
<meta name="citation_reference" content="Wang, Y., Heiland, R. &amp; Macklin, P. Physicell: liver tissue mechanobiology [nanoHUB app, Version 1.2], &lt;https://nanohub.org/resources/pc4livermedium&gt; (2019)." />
<meta name="citation_reference" content="Ozik, J. et al. High-throughput cancer hypothesis testing with an integrated PhysiCell-EMEWS workflow. BMC Bioinformatics 19, 483, doi:10.1186/s12859-018-2510-x (2018)." />
<meta name="citation_reference" content="Ozik, J., Collier, N., Heiland, R., An, G. &amp; Macklin, P. Learning-accelerated discovery of immunetumour interactions. Mol Syst Des Eng 4, 747–760, doi:10.1039/c9me00036d (2019)." />
<meta name="citation_reference" content="Macklin, P. UCI Systems Biology Foundation Short Course - PhysiCell Short Course, &lt;https://github.com/physicell-training/UCI-sysbio-2020&gt; (2020)." />
<meta name="citation_reference" content="Hucka, M. et al. The systems biology markup language (SBML): a medium for representation and exchange of biochemical network models. Bioinformatics 19, 524–531, doi:10.1093/bioinformatics/btg015 (2003)." />
<meta name="citation_reference" content="Somogyi, E. T. et al. libRoadRunner: a high performance SBML simulation and analysis library. Bioinformatics 31, 3315–3321, doi:10.1093/bioinformatics/btv363 (2015)." />
<meta name="citation_reference" content="Stoll, G. et al. MaBoSS 2.0: an environment for stochastic Boolean modeling. Bioinformatics 33, 2226–2228, doi:10.1093/bioinformatics/btx123 (2017)." />
<meta name="citation_reference" content="Letort, G. et al. PhysiBoSS: a multi-scale agent-based modelling framework integrating physical dimension and cell signalling. Bioinformatics 35, 1188–1196, doi:10.1093/bioinformatics/bty766 (2019)." />
<meta name="citation_reference" content="Cockrell, C. &amp; An, G. Sepsis reconsidered: Identifying novel metrics for behavioral landscape characterization with a high-performance computing implementation of an agent-based model. J Theor Biol 430, 157–168, doi:10.1016/j.jtbi.2017.07.016 (2017)." />
<meta name="citation_reference" content="Cockrell, R. C. &amp; An, G. Examining the controllability of sepsis using genetic algorithms on an agent-based model of systemic inflammation. PLOS Computational Biology 14, e1005876, doi:10.1371/journal.pcbi.1005876 (2018)." />
<meta name="citation_reference" content="Ozik, J., Collier, N. T., Wozniak, J. M. &amp; Spagnuolo, C. in 2016 Winter Simulation Conference (WSC) 206-220 (2016)." />
<meta name="citation_reference" content="Ozik, J., Collier, N. T., Wozniak, J. M., Macal, C. M. &amp; An, G. Extreme-Scale Dynamic Exploration of a Distributed Agent-Based Model With the EMEWS Framework. IEEE Transactions on Computational Social Systems 5, 884–895, doi:10.1109/tcss.2018.2859189 (2018)." />
<meta name="citation_reference" content="Wozniak, J. M. et al. CANDLE/Supervisor: a workflow framework for machine learning applied to cancer research. BMC Bioinformatics 19, doi:10.1186/s12859-018-2508-4 (2018)." />
<meta name="citation_reference" content="Khanna, A. S. et al. A modeling framework to inform preexposure prophylaxis initiation and retention scale-up in the context of ‘Getting to Zero’ initiatives. Aids 33, 1911–1922, doi:10.1097/qad.0000000000002290 (2019)." />
<meta name="citation_reference" content="Tatara, E. et al. in 2019 Winter Simulation Conference (WSC) 1008-1019 (2019)." />
<meta name="citation_reference" content="Rutter, C. M., Ozik, J., DeYoreo, M. &amp; Collier, N. Microsimulation model calibration using incremental mixture approximate Bayesian computation. The Annals of Applied Statistics 13, 2189–2212, doi:10.1214/19-aoas1279 (2019)." />
<meta name="citation_reference" content="Madhavan, K. et al. nanoHUB.org: cloud-based services for nanoscale modeling, simulation, and education. Nanotechnology Reviews 2, doi:10.1515/ntrev-2012-0043 (2013)." />
<meta name="citation_reference" content="Kluyver, T. et al. in ELPUB. 87–90." />
<meta name="citation_reference" content="Friedman, S. H. et al. MultiCellDS: a community-developed standard for curating microenvironment-dependent multicellular data [Preprint]. bioRxiv 090456, doi:10.1101/090456 (2016)." />
<meta name="citation_reference" content="Heiland, R., Mishler, D., Zhang, T., Bower, E. &amp; Macklin, P. xml2jupyter: Mapping parameters between XML and Jupyter widgets. J Open Source Softw 4, doi:10.21105/joss.01408 (2019)." />
<meta name="citation_reference" content="Macklin, P., Edgerton, M. E., Thompson, A. M. &amp; Cristini, V. Patient-calibrated agent-based modelling of ductal carcinoma in situ (DCIS): from microscopic measurements to macroscopic predictions of clinical progression. J Theor Biol 301, 122–140, doi:10.1016/j.jtbi.2012.02.002 (2012)." />
<meta name="citation_reference" content="Lowe, D. Angiotensin and the Coronavirus, &lt;https://blogs.sciencemag.org/pipeline/archives/2020/03/17/angiotensin-and-the-coronavirus&gt; (2020)." />
<meta name="citation_reference" content="Beauchemin, C., Forrest, S. &amp; Koster, F. T. 23–36 (Springer Berlin Heidelberg)." />
<meta name="citation_reference" content="Sniekers, Y. H. &amp; van Donkelaar, C. C. Determining Diffusion Coefficients in Inhomogeneous Tissues Using Fluorescence Recovery after Photobleaching. Biophysical Journal 89, 1302–1307, doi:10.1529/biophysj.104.053652 (2005)." />
<meta name="citation_reference" content="Organization, W. H. WHO Director-General’s opening remarks at the media briefing on COVID-19 - 11 March 2020, &lt;https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020&gt; (2020)." />
<meta name="twitter:title" content="Rapid community-driven development of a SARS-CoV-2 tissue simulator" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The 2019 novel coronavirus, SARS-CoV-2, is an emerging pathogen of critical significance to international public health. Knowledge of the interplay between molecular-scale virus-receptor interactions, single-cell viral replication, intracellular-scale viral transport, and emergent tissue-scale viral propagation is limited. Moreover, little is known about immune system-virus-tissue interactions and how these can result in low-level (asymptomatic) infections in some cases and acute respiratory distress syndrome (ARDS) in others, particularly with respect to presentation in different age groups or pre-existing inflammatory risk factors like diabetes. A critical question for treatment and protection is why it appears that the severity of infection may correlate with the initial level of virus exposure. Given the nonlinear interactions within and among each of these processes, multiscale simulation models can shed light on the emergent dynamics that lead to divergent outcomes, identify actionable “choke points” for pharmacologic interactions, screen potential therapies, and identify potential biomarkers that differentiate response dynamics. Given the complexity of the problem and the acute need for an actionable model to guide therapy discovery and optimization, we introduce a prototype of a multiscale model of SARS-CoV-2 dynamics in lung and intestinal tissue that will be iteratively refined. The first prototype model was built and shared internationally as open source code and interactive, cloud-hosted executables in under 12 hours. In a sustained community effort, this model will integrate data and expertise across virology, immunology, mathematical biology, quantitative systems physiology, cloud and high performance computing, and other domains to accelerate our response to this critical threat to international health.

Note This is a rapid prototyping project. For the very latest, see &lt;http://covid19.physicell.org&gt;" />
<meta name="og-title" property="og:title" content="Rapid community-driven development of a SARS-CoV-2 tissue simulator" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The 2019 novel coronavirus, SARS-CoV-2, is an emerging pathogen of critical significance to international public health. Knowledge of the interplay between molecular-scale virus-receptor interactions, single-cell viral replication, intracellular-scale viral transport, and emergent tissue-scale viral propagation is limited. Moreover, little is known about immune system-virus-tissue interactions and how these can result in low-level (asymptomatic) infections in some cases and acute respiratory distress syndrome (ARDS) in others, particularly with respect to presentation in different age groups or pre-existing inflammatory risk factors like diabetes. A critical question for treatment and protection is why it appears that the severity of infection may correlate with the initial level of virus exposure. Given the nonlinear interactions within and among each of these processes, multiscale simulation models can shed light on the emergent dynamics that lead to divergent outcomes, identify actionable “choke points” for pharmacologic interactions, screen potential therapies, and identify potential biomarkers that differentiate response dynamics. Given the complexity of the problem and the acute need for an actionable model to guide therapy discovery and optimization, we introduce a prototype of a multiscale model of SARS-CoV-2 dynamics in lung and intestinal tissue that will be iteratively refined. The first prototype model was built and shared internationally as open source code and interactive, cloud-hosted executables in under 12 hours. In a sustained community effort, this model will integrate data and expertise across virology, immunology, mathematical biology, quantitative systems physiology, cloud and high performance computing, and other domains to accelerate our response to this critical threat to international health.

Note This is a rapid prototyping project. For the very latest, see &lt;http://covid19.physicell.org&gt;" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-05" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.02.019075v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.02.019075v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.02.019075v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1229969" />
    <title>Rapid community-driven development of a SARS-CoV-2 tissue simulator | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2ZsYq3NgF48ClX5dPv8rMuDPfzqqPrS8SvGAPPvBa34","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1229969","apath":"\/biorxiv\/early\/2020\/04\/05\/2020.04.02.019075.atom","pisa":"biorxiv;2020.04.02.019075v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.02.019075v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.02.019075v1","title":"Rapid community-driven development of a SARS-CoV-2 tissue simulator","identifier":"node\/1229969"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022i+XA5qCpjeW6Bm42vDntUWfjU12q3LsIFysox3zMqsZ1bFJkaAyx89MhVm0NN+fkuTBlfu15aXUcls9OJ6rerq31z\\\\\\\/OBUbWuALWsTc0y1Zi6MaPBxQcwm3VFjgdZRGrWb8qHmPsqKxzX3IU5+oZbhuMTwdXy\\\\\\\/OCAnnpyy0ScJF0Uypoce7l4cg0iozdA3dgeYDod\\\\\\\/xjCAJnTScNSupGoHNskUDA5pymzbNlOj1ZL6Pp67V3OdOWYpAo2WCEPKWBrirAMey5zOYkomSKFm26WDwdkS5H8KFy1NofAs\\\\\\\/7CDaPyKuUjeMI\\\\\\\/\\\\\\\/YuHGGpTOajHyA7MmYRHsrbDIg5wWQP8dVqYkKPmi2xi1EqXO1BlyM25iJh1oNeW7kVoJgvXSCnTg67LISixCjICmcFyyHbyqdbmp89tknsUMAqizLC7s0YY+zcb3phkHvHpq45k9P9Etp\\\\\\\/JGmzGAvXPnXB0+LcN+96lIZHEgFvYvBMx8Y8JaXiP1fkp43JewsWx6vgkIOdmmUC4mQkqQlkimGENuPLIHCqpSbcZgREcdlGQFTCuScmATFSw453QyvWursxSNjl8iKwMCp64y0daDGDydv9eLR5XiptGFeK5Tct6tsK8PcGdgP341qrgwNorbmf2WnaE+y7bAeZAk4\\\\\\\/aR8duuTCzo79XYbH1SHapIQJ4VEqNHsGzTmrguBqDZ5yc0TPrHibbWNJ9pmycaOhPB8wss1BSGDRy5ozxW8mia9Ovk2OJy+9VENWiA3M\\\\\\\/nn2i6hdlxK28ITrIYrVY2EtzB0iw\\\\\\\/NB6aQAf2lOn5gYYYW5hrzHlgN0tnjowvsmHKTUAk7q+OLiF5EQ0CZzVLLqZOBDUSNBN3iyRN0ENeSvBfjrblJAFzhD2woZTnL+gtSbsQoinIiptnMC7pBN2ozKn3DABGgEOwdF6Cdk8WNnrwvNgdnYcLhEHb2DTWUchiqI8lk5sE1jrcKqBA2yk22tdOwAZwt8fAlK3xq1m5W959Kd0Akz1auzSJ34xF8oOWUjds7oAmphMiSbUbepKnwtjjhNUpvIGbGTSB5Xq2FCa8G5x5p\\\\\\\/DeXe2TK2MHEkQhBaoB0NuPHi8sLBKy+MfAZnaCkQFhpcnTa1mJbq2L3+0K6jZBkQsPlifzYIXbvOaAi4FvLvtXH4MsRy0iOBKXpcZTlBTDMpi4G5se8HvysOUVXnz1n4BrT8jaOhD+ysFZ3pIm06I\\\\\\\/K9oW8OwaQV1GIfy0lEVMRPIbz08v8hWjpYZQGQN5X2gAfii10ATtQiUkiNj5T4+TbMpyev95gHZFOFZIz27yMojqu1WWT2BIDweWQL55Q65PVBp+6bf8YT9vURRSmbAd1DA\\\u0022,\\\u0022iv\\\u0022:\\\u0022RTNeu71JPfv9ENNL6P4IWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002205eefdd9257af6992faeb932a68618efc356e1811bba2d76e6e7d623d5b2a34b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.02.019075v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1229969 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.02.019075v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kWSHMlNoTl4JgmkQt8PVjxov5ws5OaCY_WLuT25TQmo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1229969" id="node1229969" data-pisa="biorxiv;2020.04.02.019075v1" data-pisa-master="biorxiv;2020.04.02.019075" data-apath="/biorxiv/early/2020/04/05/2020.04.02.019075.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Rapid community-driven development of a SARS-CoV-2 tissue simulator</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6594-0608" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Yafei</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7440-2905" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Randy</span> <span class="nlm-surname">Heiland</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4852-4770" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Morgan</span> <span class="nlm-surname">Craig</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3274-5707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Courtney</span> <span class="nlm-surname">Davis</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-9059-5703" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ashlee N.</span> <span class="nlm-surname">Ford Versypt</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9103-7092" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Adrianne</span> <span class="nlm-surname">Jenner</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3495-6735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Ozik</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-2376-4156" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Nicholson</span> <span class="nlm-surname">Collier</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-3224-7617" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Chase</span> <span class="nlm-surname">Cockrell</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-5406-5545" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Becker</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-4549-9004" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gary</span> <span class="nlm-surname">An</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-3634-190X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">James A.</span> <span class="nlm-surname">Glazier</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-4889-4416" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Aarthi</span> <span class="nlm-surname">Narayanan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-7092-6904" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Amber M.</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0002-9925-0151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Macklin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.02.019075 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1229969" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yafei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Intelligent Systems Engineering, Indiana University. Bloomington</span>, IN <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yafei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYafei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6594-0608" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yafei Wang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Randy Heiland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Intelligent Systems Engineering, Indiana University. Bloomington</span>, IN <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randy%2BHeiland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heiland%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARandy%2BHeiland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7440-2905" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Randy Heiland</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Morgan Craig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Mathematics, University of Montreal. Montreal</span>, QC <span class='nlm-country'>Canada</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>CHU Sainte-Justine Research Centre</span>, Montreal, QC <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morgan%2BCraig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Craig%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMorgan%2BCraig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4852-4770" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Morgan Craig</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Courtney Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Natural Science Division, Pepperdine University</span>, Malibu, CA <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Courtney%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACourtney%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3274-5707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Courtney Davis</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ashlee N. Ford Versypt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>School of Chemical Engineering, Oklahoma State University</span>, Stillwater, OK <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Oklahoma Center for Respiratory and Infectious Diseases</span>, Oklahoma State University, Stillwater, OK <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashlee%2BN.%2BFord%2BVersypt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ford%20Versypt%20AN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAshlee%2BN.%2BFord%2BVersypt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9059-5703" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ashlee N. Ford Versypt</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Adrianne Jenner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Mathematics, University of Montreal. Montreal</span>, QC <span class='nlm-country'>Canada</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>CHU Sainte-Justine Research Centre</span>, Montreal, QC <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrianne%2BJenner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jenner%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdrianne%2BJenner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9103-7092" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adrianne Jenner</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jonathan Ozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Decision and Infrastructure Sciences, Argonne National Laboratory</span>. Lemont, IL <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Consortium for Advanced Science and Engineering, University of Chicago</span>. Chicago, IL <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BOzik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozik%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BOzik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3495-6735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan Ozik</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nicholson Collier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Decision and Infrastructure Sciences, Argonne National Laboratory</span>. Lemont, IL <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Consortium for Advanced Science and Engineering, University of Chicago</span>. Chicago, IL <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholson%2BCollier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collier%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholson%2BCollier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2376-4156" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholson Collier</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Chase Cockrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>The University of Vermont Medical Center</span>, Burlington, VT <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chase%2BCockrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cockrell%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChase%2BCockrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3224-7617" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chase Cockrell</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrew Becker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>The University of Vermont Medical Center</span>, Burlington, VT <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BBecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Becker%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BBecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5406-5545" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew Becker</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Gary An </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>The University of Vermont Medical Center</span>, Burlington, VT <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BAn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=An%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BAn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4549-9004" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gary An</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">James A. Glazier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Intelligent Systems Engineering, Indiana University. Bloomington</span>, IN <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BA.%2BGlazier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glazier%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BA.%2BGlazier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3634-190X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James A. Glazier</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Aarthi Narayanan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>National Center for Biodefense and Infectious Disease, George Mason University. Manassas</span>, VA <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aarthi%2BNarayanan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Narayanan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAarthi%2BNarayanan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4889-4416" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aarthi Narayanan</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Amber M. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Pediatrics, University of Tennessee Health Science Center</span>, Memphis, TN <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amber%2BM.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmber%2BM.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7092-6904" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amber M. Smith</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Paul Macklin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Intelligent Systems Engineering, Indiana University. Bloomington</span>, IN <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMacklin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macklin%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BMacklin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9925-0151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul Macklin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:macklinp@iu.edu" class="" data-icon-position="" data-hide-link-title="0">macklinp@iu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.02.019075v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1229969" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1229969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.02.019075v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1229969" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1229969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.02.019075v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1229969" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1229969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.02.019075v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1229969" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1229969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.02.019075v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1229969" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1229969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The 2019 novel coronavirus, SARS-CoV-2, is an emerging pathogen of critical significance to international public health. Knowledge of the interplay between molecular-scale virus-receptor interactions, single-cell viral replication, intracellular-scale viral transport, and emergent tissue-scale viral propagation is limited. Moreover, little is known about immune system-virus-tissue interactions and how these can result in low-level (asymptomatic) infections in some cases and acute respiratory distress syndrome (ARDS) in others, particularly with respect to presentation in different age groups or pre-existing inflammatory risk factors like diabetes. A critical question for treatment and protection is why it appears that the severity of infection may correlate with the initial level of virus exposure. Given the nonlinear interactions within and among each of these processes, multiscale simulation models can shed light on the emergent dynamics that lead to divergent outcomes, identify actionable “choke points” for pharmacologic interactions, screen potential therapies, and identify potential biomarkers that differentiate response dynamics. Given the complexity of the problem and the acute need for an actionable model to guide therapy discovery and optimization, we introduce a prototype of a multiscale model of SARS-CoV-2 dynamics in lung and intestinal tissue that will be iteratively refined. The first prototype model was built and shared internationally as open source code and interactive, cloud-hosted executables in under 12 hours. In a sustained community effort, this model will integrate data and expertise across virology, immunology, mathematical biology, quantitative systems physiology, cloud and high performance computing, and other domains to accelerate our response to this critical threat to international health.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>Note</strong> This is a rapid prototyping project. For the very latest, see <a href="http://covid19.physicell.org">http://covid19.physicell.org</a></p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-4">The ongoing pandemic caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has illuminated the global public health threat posed by highly pathogenic coronaviruses that emerge from zoonotic sources. With the majority of the world’s population immunologically naïve and no available antivirals or vaccines, over 860,000 infections and 42,000 deaths amassed within a matter of months<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. Coronavirus disease 2019 (COVID-19) is characterized by a range of respiratory symptoms, including fever and cough<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>, that can progress to acute respiratory distress syndrome (ARDS) in some patients<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. Age and comorbidities seem to be the main risk factors for development of severe disease<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>-<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. However, the dynamics of virus replication, interaction with host immune responses, and spread within the respiratory tract are still being established. Because a vaccine may not be available for 9-18 months, there is a critical need to further understand the infection in order to quickly identify pharmacologic interventions and optimal therapeutic designs that work to lessen virus dissemination and disease severity. However, this requires an international community effort that integrates expertise across a variety of domains and a platform that can be iteratively updated as new information and data arises.</p><p id="p-5">To aid this effort, we have assembled an international, multi-disciplinary coalition to rapidly develop an open-source, multi-scale tissue simulator that can be used to investigate mechanisms of intracellular viral replication, infection of epithelial cells, host immune response, and tissue damage. The aim of this project is to concentrate community modeling efforts to create a comprehensive multiscale simulation framework that can subsequently be calibrated, validated, and used to rapidly explore and optimize therapeutic interventions for COVID-19. Once the prototype has been completed (after several design iterations), this coalition will transition to maintain and support the simulation framework and aggregate calibrated/validated parameter values.</p><p id="p-6">To address the acute need for rapid access to an actionable model, we are using a community-driven coalition and best open science practices to build and iteratively refine the model:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-7"><strong>Open source and GitHub:</strong> All simulation source code is shared as open source on GitHub, with well-defined, versioned and documented releases, and Zenodo-generated DOIs and archives.</p></li><li id="list-item-2"><p id="p-8"><strong>Interactive cloud-hosted models:</strong> Every prototype version is rapidly transformed into a cloud-hosted, interactive model to permit faster scientific communication across communities, particularly with virologists and others who have essential insights but ordinarily would not directly run the simulation models.</p></li><li id="list-item-3"><p id="p-9"><strong>Social media and virtual feedback:</strong> We enlist community participation (feedback, modeling contributions, software contributions, and data contributions) through social media, virtual seminars, web-based forms, and a dedicated Slack workspace. We are particularly encouraging feedback and data contributions by domain experts in virology, epidemiology, and mathematical biology (with a focus</p></li><li id="list-item-4"><p id="p-10"><strong>Frequent preprint updates:</strong> Each model iteration is accompanied by a cloud-hosted, interactive app (see #2) and an updated preprint on <em>bioRxiv</em>.</p></li><li id="list-item-5"><p id="p-11"><strong>Integration of feedback:</strong> All community feedback is evaluated to plan the next set of model refinements and recorded in an updated <em>bioRxiv</em> preprint.</p></li></ol><p id="p-12">Our first test of this workflow saw a first proof-of-concept software release (Steps 1-2) in 12 hours, and the first integration of community feedback and preprint dissemination was complete within a week. It is our intention to continue rapid iteration, with a new candidate model release every 7-10 days.</p><div id="sec-3" class="subsection"><h3>Goals and guiding principles</h3><p id="p-13">This project is community-driven, including the following contributions:</p><ol class="list-ord " id="list-2"><li id="list-item-6"><p id="p-14"><strong>Community priorities:</strong> The community helps define the driving research questions, definition of the project scope, and selection of critical biological components to be modeled.</p></li><li id="list-item-7"><p id="p-15"><strong>Consensus hypotheses:</strong> The community drives a shared, clearly-written consensus specification of the underlying biological hypotheses.</p></li><li id="list-item-8"><p id="p-16"><strong>Mathematical modeling:</strong> The community helps develop, review and refine the mathematical interpretation of the biological hypotheses.</p></li><li id="list-item-9"><p id="p-17"><strong>Computational implementation:</strong> The computational implementation is shared as open source, with community definition of specifications, unit tests, coding, and code review (via pull requests to the development branch).</p></li><li id="list-item-10"><p id="p-18"><strong>Community feedback:</strong> Community feedback on the model realism, hypotheses, mathematics, computational implementation, and development techniques is encouraged throughout the development process.</p></li><li id="list-item-11"><p id="p-19"><strong>Community parameter and data:</strong> Community contributions of parameter estimates and data contributions are aggregated to assist in model development and constraint.</p></li></ol></div><div id="sec-4" class="subsection"><h3>Project scope</h3><p id="p-20">While by definition the project scope can be refined by the community, the initial project scope is to:</p><ol class="list-ord " id="list-3"><li id="list-item-12"><p id="p-21">Develop the general computational framework sufficiently to address many of the community-driven research questions.</p></li><li id="list-item-13"><p id="p-22">Deliver a working simulation framework for use by others to perform calibration and validation. That is, the prototyping aims of this project are complete once the model is capable of demonstrating essential biological behaviors qualitatively.</p></li><li id="list-item-14"><p id="p-23">To provide a software framework whose underlying hypotheses, mathematics, and computational implementation have been rigorously assessed by appropriate domain experts.</p></li></ol><p id="p-24">In particular, while this project will work to constrain, estimate, and calibrate parameters to the greatest extent possible, it is <em>not</em> within scope to delay software release until full calibration and validation. Those tasks are within scope of fully funded teams with dedicated experiments.</p><p id="p-25">This project aims to deliver software that one can reasonably expect to calibrate and validate, thus freeing funded investigations from expensive early software development while providing a broad community consensus on key biological hypotheses. By rapidly prototyping this software, we aim to accelerate many funded research efforts.</p></div><div id="sec-5" class="subsection"><h3>Essential model components</h3><p id="p-26">As part of defining the project scope, we have identified the following critical model components:</p><ol class="list-ord " id="list-4"><li id="list-item-15"><p id="p-27">Virus dissemination in epithelial tissue</p></li><li id="list-item-16"><p id="p-28">Virus binding, endocytosis, replication, and exocytosis</p></li><li id="list-item-17"><p id="p-29">Infected cell responses, including changes to metabolism, secreted signals, and death</p></li><li id="list-item-18"><p id="p-30">Inflammatory response</p></li><li id="list-item-19"><p id="p-31">Ramp up the immune response (particularly in lymph nodes)</p></li><li id="list-item-20"><p id="p-32">Immune cell infiltration</p></li><li id="list-item-21"><p id="p-33">Immune cell predation of infected and other cells.</p></li><li id="list-item-22"><p id="p-34">Tissue damage by death of infected cells and immune cells</p></li><li id="list-item-23"><p id="p-35">Early fluid leakage that can be regarded as an early marker or surrogate for ARDS</p></li></ol></div><div id="sec-6" class="subsection"><h3>Guiding principles</h3><p id="p-36">The coalition works under the following guiding principles:</p><ul class="list-unord " id="list-5"><li id="list-item-24"><p id="p-37">The model framework can be built with relatively sparse data, relying upon domain expertise and observations to choose its general form and assess its qualitative behavior.</p></li><li id="list-item-25"><p id="p-38">The model will be modular. Each submodel will have well-defined inputs and outputs of submodels, allowing parallel development and replacement of submodels with improved versions without change to the rest of the model.</p></li><li id="list-item-26"><p id="p-39">The submodels should be independently executable and verifiable, allowing parallel development with confidence.</p></li><li id="list-item-27"><p id="p-40">The overall model framework will periodically release numbered versions (distributions) that bundle the best working version of each submodel as it exists at the time of release, allowing end-users (the community) to use well-defined, well-tested snapshots of the project.</p></li><li id="list-item-28"><p id="p-41">The model framework should investigate the dynamics of infection and treatment, and not merely end-points.</p></li><li id="list-item-29"><p id="p-42">The model should allow the community to as ask “what if” questions to guide experiments and interventions<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>.</p></li><li id="list-item-30"><p id="p-43">The model (and known parameter values) must be made publicly available as open source for maximum public benefit.</p></li><li id="list-item-31"><p id="p-44">The model will be made publicly available as an interactive web app to encourage community participation, to accelerate scientific dissemination, and to increase public educational benefit.</p></li><li id="list-item-32"><p id="p-45">We will use rapid prototyping to encourage a fast develop-test-refine cycle to build expertise and gain community feedback.</p></li><li id="list-item-33"><p id="p-46">We will develop model to point that it has correct qualitative behavior, so that calibration is likely to succeed. This is the “product” for use in subsequent investigations by multiple teams. See the <strong><em>scoping</em></strong> statements above.</p></li><li id="list-item-34"><p id="p-47">We will gather community consensus and pool efforts into a “standardized” model that captures key SARS-CoV-2 dynamics. We will supply this model to the community for use in parallel studies by multiple labs.</p></li><li id="list-item-35"><p id="p-48">We will encourage data and parameter sharing throughout this effort and by user community after model the model’s “completion.”</p></li><li id="list-item-36"><p id="p-49">As part of the model formulation, documentation, and dissemination, we will craft clearly delineated “conceptual model and hypotheses” to encourage development of independent models with independent methodologies and software frameworks</p></li><li id="list-item-37"><p id="p-50">After the model prototyping is complete (the goal of this paper), we will enter a maintenance and support phase to fix bugs, support scientist users, and add features identified by user community.</p></li></ul></div><div id="sec-7" class="subsection"><h3>Critical questions for the model framework</h3><p id="p-51">While the model may not be able to address all of these scientific questions, the community determined that the work should be driven by these questions, and that each iteration of the model framework should aim to be amenable to an increasing number of these questions. We aim to enable investigations that ask:</p><ul class="list-unord " id="list-6"><li id="list-item-38"><p id="p-52">What are the critical “choke points” in viral infection, replication, and propagation?</p></li><li id="list-item-39"><p id="p-53">Which interventions could most effectively leverage identified vulnerabilities in viral replication?</p></li><li id="list-item-40"><p id="p-54">What unanticipated dynamics can emerge from a single molecular-scale inhibition?</p></li><li id="list-item-41"><p id="p-55">Does the initial level of exposure to the virus affect the severity of the progression of the disease and how could this effect be ameliorated?</p></li><li id="list-item-42"><p id="p-56">What are the key points of virus-immune system interactions that drive mild versus acute (e.g., ARDS) responses?</p></li><li id="list-item-43"><p id="p-57">What are key differences at the target-cell level during innate versus adaptive immune responses?</p></li><li id="list-item-44"><p id="p-58">Are there threshold levels of infection at the cellular or tissue level that indicate a switch from asymptomatic to symptomatic or from mild to severe disease in a patient?</p></li><li id="list-item-45"><p id="p-59">Through what mechanisms do certain patient characteristics, pre-existing conditions, or background medications increase the likelihood of adverse outcomes?</p></li><li id="list-item-46"><p id="p-60">What interventions could accelerate building immunity?</p></li><li id="list-item-47"><p id="p-61">What interventions can reduce or reverse adverse immune reactions?</p></li><li id="list-item-48"><p id="p-62">At what stage is a given intervention most beneficial?</p></li><li id="list-item-49"><p id="p-63">How does viral mutagenicity affect the robustness of a therapy or a therapeutic protocol?</p></li><li id="list-item-50"><p id="p-64">How does cellular heterogeneity affect infection dynamics?</p></li><li id="list-item-51"><p id="p-65">How does the nearby tissue environment, such as the mucus layer, affect infection dynamics?</p></li><li id="list-item-52"><p id="p-66">How does the infection spread from its initial locus of infection to other tissues (in particular from upper respiratory to lungs)? How does stochasticity impact these dynamics?</p></li><li id="list-item-53"><p id="p-67">How do tissues recover after clearance of local infection? Can scarring be minimized to reduce long-term adverse effects in recovered patients?</p></li><li id="list-item-54"><p id="p-68">How do adverse effects in SARS-CoV-2 infected epithelia differ (mechanistically) from other infections and other modes of epithelial dysfunction?</p></li></ul></div><div id="sec-8" class="subsection"><h3>Essential background biology</h3><p id="p-69">This rapid prototyping effort brings together specialists from a broad variety of domains: virology and infectious diseases, mathematical biology, computer science, high performance computing, data science, and other disciplines. Therefore, it is critical that all members of the project have access to a clear description of underlying biology. In this section we outline key aspects of viral replication and host response in functional terms needed for development of agent-based, multi-scale and multi-physics models.</p></div><div id="sec-9" class="subsection"><h3>Cell infection and viral replication</h3><p id="p-70">The key cell-level process is viral infection of a single cell, followed by replication to create new virions:</p><ol class="list-ord " id="list-7"><li id="list-item-55"><p id="p-71">SARS-CoV-2 is a single-stranded enveloped RNA virus<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. A virion (single virus particle) has a lipid coating (envelope) that protects the virus when outside a cell (or host). Each virus has dozens of spike glycoproteins that bind to ACE2 (receptors) on select cell membranes<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup>.</p></li><li id="list-item-56"><p id="p-72">Virions travel in the tissue microenvironment to reach a cell membrane. The spike binds to an available ACE2 receptor on the cell’s membrane. Both passive transport (e.g., via diffusion in fluids) and active transport (e.g., by cilia-driven advection mucus) may play a role at slow and fast time scales, as well as surface contact transmission between neighboring cells.</p></li><li id="list-item-57"><p id="p-73">The cell internalizes the adhered virus via endocytosis into a vesicle.</p></li><li id="list-item-58"><p id="p-74">The endocytosed virion—now residing in a vesicle with lowered pH—is uncoated to release its mRNA contents into the cell cytoplasm.</p></li><li id="list-item-59"><p id="p-75">Copying viral RNA creates a (-) RNA template, which is used for (+) RNA production.</p></li><li id="list-item-60"><p id="p-76">RNA is used to synthesize viral RNA and proteins.</p></li><li id="list-item-61"><p id="p-77">Viral proteins are transported to the interior surface of the cell membrane.</p></li><li id="list-item-62"><p id="p-78">Viral proteins at cell membrane are assembled into virions.</p></li><li id="list-item-63"><p id="p-79">Assembled virions are exported from the cell by exocytosis.</p></li><li id="list-item-64"><p id="p-80">When a cell dies and lyses, some or all partly assembled and fully assembled virions can be released into the tissue microenvironment.</p></li></ol><p id="p-81">We note that once infected, an individual cell cannot “recover” (e.g., by actively degrading viral RNA and stopping endocytosis) to return to normal function; that cell is irreversibly committed to eventual death.</p><p id="p-82">For further detail, see review articles on RNA virus replication dynamics<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>.</p></div><div id="sec-10" class="subsection"><h3>Infected cell responses</h3><p id="p-83">While infected cells (e.g., type 1 or type 2 alveolar cells in the lung) cannot recover, they can respond to slow replication and reduce infection of nearby cells. Infected can secrete type I interferons (IFN-α,β), which diffuse and bind to receptors on nearby cells to slow viral replication, activate an inflammatory response, and induce gene transcription<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>, to slow cycling or induce apoptosis in these cells<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Secreted interferons can also modulate the function of innate and adaptive cellular immunity<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup>.</p><p id="p-84">Eventually, infected cells apoptose, lyse and release unassembled viral components may be released<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. While the mechanism of cell death in SARS-CoV-2 is currently unknown, in other RNA virus infections, cells can undergo apoptotic, necrotic, or pyroptotic death over the course of viral infection<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Disruption of cell metabolism and competition for critical substrates may contribute to cell death<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>.</p></div><div id="sec-11" class="subsection"><h3>Inflammatory and immune responses</h3><p id="p-85">Lethal SARS and MERS in humans has been correlated with elevated IFN-α,β <sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>, myeloid activity, and impaired T and B cells<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>, although the timing of Type 1 IFN is critical<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. Type I IFNs secreted by infected cells or by immune cells diffuse to surrounding cells and recruit innate immune cells, such as macrophages and neutrophils, to the area. In COVID-19 patients, decreased numbers of T cells, natural killer (NK) cells, and, to a lesser extent, B cells occur, and the extent of T cell depletion has been correlated with disease severity<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. Excessive IFN-α,β activation results in increased macrophage and neutrophil presence, which correlates with lung dysfunction<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. Delayed IFN-α,β production also promotes inflammatory macrophage recruitment that contributes to vascular leakage and impaired T-cell function<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup>. Activated macrophages also produce other proinflammatory cytokines like IL-1, IL-6, and TNF-α, among others, that enhance infiltration of immune cells and interact with endothelial cells to cause vasodilation<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. Moreover, epithelial tissue death can reduce tissue integrity, contributing to further immune infiltration, fluid leakage and edema, and acute respiratory distress<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>-<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>.</p><p id="p-86">In severe cases, a “cytokine storm” of pro-inflammatory cytokines (e.g., IL-2, IL-7, IL-10, G-CSF, IP-10, MCP-1, MIP-1A, and TNF-α) promotes extensive tissue damage<sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a></sup>. During influenza virus infection, there is some evidence that ARDS is correlated with the extent of infection in the lower respiratory tract and increased cytokine activity resulting from exposure of the endothelium<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. Other innate responses, such as neutrophils, have also been shown to be amplified in SARS-CoV-2<sup>25</sup>. These cells generally produce reactive oxygen species (ROS), which can induce the death of infected and healthy cells in the local environment<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup>.</p><p id="p-87">Coronaviruses have been shown to evade and modulate various host immune responses<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>-<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. In addition to those discussed above, some evidence suggests that an antibody to spike protein enhances disease during SARS-CoV infection by inducing macrophage switching from a wound healing phenotype to an inflammatory phenotype<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. Furthermore, influenza viruses and SARS-CoV also use the same ACE2 receptors to enter cells; these cells are known to infect macrophages and T cells<sup><a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> It is not yet known whether SARS-CoV-2 infection of these cells is productive<sup><a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a></sup>. However, the ACE2 receptor has been linked to acute lung injury for both viruses<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>.</p></div><div id="sec-12" class="subsection"><h3>Inflammation and poor clinical outcomes</h3><p id="p-88">While the underlying risk-factors for an individual developing acute respiratory distress syndrome (ARDS) in response to SARS-CoV-2 infection have not yet been elucidated, it appears clear that a dysregulated immune response is central to this aspect of the disease<sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. In particular, chemokines are released following viral infection, which leads to the invasion of neutrophils and macrophages and release of reactive oxygen species (ROS). In addition, replication in the lower airways and exposure of endothelial cells may further amplify the inflammatory response<sup><a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a></sup>. Collectively, this leads to extensive tissue damage and depletion of epithelial cells, which may be connected to lethality<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. Within the alveolar tissue, and systemically, the feedback between viral load, adaptive and innate immune response and tissue damage is clearly a complex system. By utilizing a multi-scale framework to implement these interactions, we aim to connect circulating biomarkers, putative treatments, and clinically observed disease progression to pathophysiological changes at the cell and tissue level.</p></div><div id="sec-13" class="subsection"><h3>Anticipated data to drive development and validation</h3><p id="p-89">It is important that model development takes into account the types of measurements and biological observations that will be available for later model constraint, calibration, and validation. As participation by the virology and pharmacology communities broadens, we anticipate that this list will grow. While we will endeavor to constrain and validate sub-modules of the model independently, we anticipate human clinical data to not fully determine parameters of the model. To address this concern we will apply a ‘virtual population’ approach and sensitivity analysis to explore model variability within clinically relevant bounds<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>,<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. To date, we anticipate the following data:</p></div><div id="sec-14" class="subsection"><h3>Organoid data for viral replication and targeted inhibition</h3><p id="p-90">Aarthi Narayanan’s virology lab is optimizing SARS-CoV-2 cultures in organoid model systems. The viral replication kinetics will be assessed by infection of different lung epithelial, fibroblast and endothelial cells, in addition to standard cell lines such as Vero cells. (Vero cells are likely to be the workhorse for inhibitor assessment studies). Primary cells and/or cell lines will be infected with SARS-CoV-2 at increasing multiplicities of infection and infectious viral titers in the supernatants assessed by plaque assays at multiple time points post initial infection. This will stretch from approximately 3 hours post infection up to 48 hours post infection depending on the cell type and the initial infectious dose of virus.</p><p id="p-91">In parallel, the viral genomic copy numbers will be assessed in the same supernatant samples by qRT-PCR with virus specific primers. This will provide information on how the production of infectious virions compares with the number of genomic copies available outside the cell. If the numbers are skewed in the direction of genomic copies (which may happen in the context of some kinds of inhibitors), it will shed light on the mechanisms of inhibition involving inhibition of infectivity of progeny virions.</p><p id="p-92">The viral genomic copy numbers inside the cells will also be assessed by qRT-PCR and compared to the genomic copies outside the cell. This will provide direction on the efficacy of particle packaging and the extent of production of infectious versus noninfectious virus. While it will not provide directly pertinent information about the possibility of heterogeneity of released virus populations and quasispecies, it can provide initial clues in that direction, which can then trigger more specific questions and relevant approaches. These approaches will be pursued for cell lines, primary cells and, hopefully, subsequently transitioned to organoid platforms.</p><p id="p-93">From a host response point of view, we will pursue two aspects: host cell death and inflammatory responses. For cell survival and death measurements, we will employ an assay that measures ATP activity in cells (hence a reflection of a live cell) in the context of infection and inhibitor treatments. For inflammatory responses, we will assess supernatants for inflammatory mediators by ELISA (multiplexed). The cells will be lysed to obtain RNA, which will be queried for transcription of several genes associated with inflammatory responses using gene expression arrays (multiplexed).</p><p id="p-94">Additional host response events will include mitochondrial activity and ROS production assessments in the context of infection and inhibitor treatments. The impact of anti-inflammatory strategies on mitochondrial activity and cell survival will be assessed to determine correlations between viral replication dependent and independent events.</p></div><div id="sec-15" class="subsection"><h3>Inflammation, ACE2 binding, and host response data</h3><p id="p-95">Given the impact of SARS-CoV-2, we anticipate an unprecedented amount of mechanistic data to emerge from both clinical and preclinical sources. Of particular value in testing and refining the model will be randomized controlled interventional trials in general or specific populations. As of March, 30 2020, there were 119 trials registered at clinicaltrial.gov under the search term “COVID-19+Drug”. Within this 119, there are multiple interventions at different points of the pathophysiology, including, but not limited to: hyperimmune plasma, IL-6 Antibody (e.g. Tocilizumab), protease inhibitors (e.g. Lopinavir/ritonavir), cloroquine/hydroxychloroquine, and Janus Kinases inhibitors (e.g. Baricitinib). As this platform develops, we anticipate predicting the responses to such therapies and refining the model with emerging data such that the range of clinical responses are captured with adequate fidelity. Additionally, data collected from patients or animals during infection, including the presence of various immune cell subsets in lung tissue and systemic markers of inflammation, will serve to differentiate responses to SARS-CoV-2. These data will be similarly integrated to calibrate and validate the model to ensure accurate predictions of therapeutic outcomes based on clinical characteristics.</p></div><div id="sec-16" class="subsection"><h3>Relevant prior modeling</h3><p id="p-96">Spurred initially by the emergence of HIV and relevant to the ensuing SARS-CoV-2 pandemic, the field of viral dynamics modelling has been instrumental for understanding the evolution of host-virus interactions<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>-<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>, predicting treatment responses<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>-<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>, and designing novel and more effective therapeutic approaches<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>-<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>. The classic within-host mathematical model of viral infection uses a system of ordinary differential equations (ODEs) to describe the dynamics between uninfected epithelial (“target”) cells, infected cells in the eclipse phase, infected cells producing virus, and infectious virus<sup><a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup>. This basic model has been shown to capture dynamics of both acute and chronic infection<sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>, and has been extended to also include multiple viral (potentially resistant) strains<sup><a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a></sup> and various aspects of host immune responses<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>,<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup>. While such cell population-level models ODE models generally do not account for single-cell effects, they are effective for detailing viral load, host immune response, and pathology dynamics<sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>-<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></sup>. Moreover, they can often be used to constrain and estimate parameters for more detailed models, develop novel hypotheses, and design confirmatory experiments<sup><a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>,<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a></sup>.</p><p id="p-97">Some have modeled intracellular virus replication, including very detailed models used for understanding replication and intervention points<sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a></sup>, typically using systems of ODEs<sup><a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>,<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a></sup>. These models often include virus-receptor binding, receptor trafficking, endocytosis, viral uncoating, RNA transcription, protein synthesis, viral assembly, and viral exocytosis. However, to date no such model has been integrated with detailed spatiotemporal models of viral propagation in 3-D tissues with dynamical models of immune interactions.</p><p id="p-98">Recently, agent-based models have been used to simulate viral propagation in 2-D tissues with simplified models of viral replication in individual cells, particularly in the field of influenza virus infection<sup><a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a></sup> a variety of other viral infections<sup><a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a></sup>, and oncolytic viral therapies<sup><a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>-<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a></sup>. These models have generally not included detailed intracellular models of viral replication and individual cell responses to infection. However, they demonstrate the potential for including detailed intracellular models of viral replication in 2D and 3D tissues with the milieu of immune and epithelial cell types expected in actual patients, while also offering the opportunity to test hypotheses on the impact of viral mutagenicity and host cell heterogeneity on disease progression.</p><p id="p-99">The rapid prototyping approach of this coalition will use a performance-driven agent-based modeling platform<sup><a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a></sup> to combine detailed intracellular models of viral endocytosis, replication, and exocytosis, disruption of cell processes (e.g. metabolism and compromised membranes) that culminate in cell death, inflammation signaling and immune responses, tissue damage, and other key effects outlined above in a comprehensive, open source simulation platform. We will deploy and refine interactive, web-hosted versions of the model to critical contributions by virologists, infectious disease modelers, and other domain experts. We will frequently update preprints to foster the fastest possible scientific dialog to iteratively refine this community resource.</p></div><div id="sec-17" class="subsection"><h3>Related modeling efforts and other future data sources</h3><p id="p-100">We are coordinating with related modeling efforts by a number of groups. In particular Thomas Hillen has organized a COVID-19 Physiology Reading Group<sup><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a></sup>, where a growing community of modelers are exchanging information and progress. We will regularly work with this community to seek feedback on the model hypotheses, parameter insights, and code contributions, and share our work with them.</p><p id="p-101">The COVID-19 Cell Atlas<sup><a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a></sup> organizes a variety of cell-scale datasets relevant to COVID-19; these may be of particular importance to intracellular modeling components of the project. The Human Biomolecular Atlas Program (HuBMAP)<sup><a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a></sup> is creating detailed maps of the human respiratory system at cell- and molecular-scale resolution; this will be an excellent data source for tissue geometry in later versions of the model.</p></div></div><div class="section" id="sec-18"><h2 class="">Methods</h2><div id="sec-19" class="subsection"><h3>PhysiCell: agent-based cell modeling with extracellular coupling</h3><p id="p-102"><em>PhysiCell</em> is an open source simulation agent-based modeling framework for multicellular systems in 2D and 3D dynamical tissue environments<sup><a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">82</a></sup>. (See Metzcar et al. (2019) for a general overview of agent-based modeling techniques in tissue-scale biology<sup><a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a></sup>.) In this framework, each cell (of any type) is an off-lattice agent with independent cell cycle progression, death processes, volume changes, and mechanics-driven movement. Each cell agent can have independent data and models attached to it, allowing substantial flexibility in adapting the framework to problems in cancer biology, microbiology, tissue engineering, and other fields. PhysiCell is coupled to <em>BioFVM</em> (an open source biological diffusion solver)<sup><a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a></sup> to simulate the chemical microenvironment; as part of this coupling, each individual agent can secrete or uptake diffusing substrates and track the total amount of material entering and leaving the cell.</p><p id="p-103">Relevant applications of PhysiCell-powered models have included modeling cancer nanotherapy<sup><a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a></sup>, oncolytic virus therapies<sup><a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a></sup>, tissue biomechanical feedbacks during tumor metastatic seeding <sup><a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a></sup>, and cancer immunology<sup><a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">82</a>,<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>,<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a></sup>. The platform has been built with a focus on computational efficiency and cross-platform compatibility: the same source code can be compiled and run without modification on Linux, OSX, and Windows, and simulations of up to 10 diffusing substrates on 10 mm<sup>3</sup> of tissue with 10<sup>4</sup> to 10<sup>6</sup> cells are routinely performed on desktop workstations. The platform has been combined with high-throughput computing<sup><a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">91</a></sup> and active learning techniques<sup><a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">92</a></sup> to power large-scale model exploration on high performance computing resources.</p></div><div id="sec-20" class="subsection"><h3>Integration of intracellular models in PhysiCell agents</h3><p id="p-104">Custom functions can be attached to individual cell agents to model molecular-scale, intracellular processes and couple these with cell phenotypic parameters. These internal models are often implemented as systems of ODEs. For example, cell uptake of diffusing substrates can be coupled with a metabolism model (system ODEs), and the resulting energy output can be used to set the cell’s cycle progression and necrotic death probability<sup><a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a></sup>. For small systems of ODEs, these models are coded “by hand” with standard finite difference techniques. More complex models are written in systems biology markup language (SBML)<sup><a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a></sup> for reliable scientific communication. Development versions of PhysiCell can read and integrate an individual SBML-encoded model in each cell agent using <em>libRoadrunner</em>—a highly efficient SBML integrator<sup><a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a></sup>. Similar approaches have been used to integrate Boolean signaling networks<sup><a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a></sup> in PhysiCell in the PhysiBoSS extension<sup><a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a></sup>.</p><p id="p-105">These approaches will be used to assess (1) viral replication dynamics in each cell agent, (2) cell death responses to viral load, (3) cell responses to interferons, and (4) changes in the virion endocytosis rate based on the availability of ACE2 and its receptor trafficking dynamics.</p></div><div id="sec-21" class="subsection"><h3>Cellular Immunity Agent-Based Model (CIABM)</h3><p id="p-106">As an independent model component, An, Becker, and Cockrell are developing CIABM: an agent-based model of immune system activation and expansion in lymph nodes in response to SARS-CoV-2 infections. This model will be coupled with the overall simulator to mechanistically drive immune expansion and infiltration during inflammatory responses.</p><p id="p-107">The CIABM is intended to be a generalizable model of CD8 dynamics, designed to represent different disease states resulting from different perturbations (i.e. specific infections of specific pathogens, putative vaccines and their administration strategy). This is consistent with our group’s philosophy of pathophysiological unification through modeling. We have developed multiple ABMs related to the immune response and diseases related to inflammation and immune dysfunction<sup><a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>,<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a></sup>, and will leverage this experience to integrate various aspects of these models as components of the CIABM. Many of these models are based on the IIRABM<sup><a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">98</a></sup>, which is an abstract representation and simulation of the human inflammatory signaling network response to injury; the model has been calibrated such that it reproduces the general clinical trajectories seen in sepsis. The IIRABM operates by simulating multiple cell types and their interactions, including endothelial cells, macrophages, neutrophils, TH0, TH1, and TH2 cells as well as their associated precursor cells. The simulated system dies when total damage (defined as aggregate endothelial cell damage) exceeds 80%; this threshold represents the ability of current medical technologies to keep patients alive (i.e., through organ support machines) in conditions that previously would have been lethal. The IIRABM will be used in the CIABM to represent the innate and host tissue component of the CIABM.</p><p id="p-108">The design principles of the CIABM are: 1) CD8-dynamics focused detail, 2) incorporation of the IIRABM as the innate-host tissue module, 3) incorporation of humoral and CD4 T-cell function, but at an abstracted level.The justification for aggregating humoral and CD4 functions is that they are primarily governed by interactions through the MHC2 complex, and therefore represent a distinct sensing/recognition capacity than CD8 MHC1-mediated recognition. We note that while we recognize that CD4-helper activity is known to assist the generation of CD8 memory, at the outset of this project we will attempt to see how well we can reproduce our clinically-acquired data without having to explicitly represent CD-4-helper function. If we are unable to satisfactorily map CIABM behavior to our experimental/clinical data, we will then add these components.</p><p id="p-109">As a general description, the initial components of the innate immune response represent the end-effector of the system, primarily responsible for interactions influencing tissue damage, microbial killing and abstracted tissue reconstitution. This component incorporates both pro-and anti-inflammatory components, consistent with a self-contained control structure befitting its role as a highly-evolutionarily conserved, fundamental function of multi-cellular organisms. These agent types are: tissue, viral antigen load, polymorphonuclear neutrophil cells (PMNs), macrophages, dendritic cells, CD8 T-cells (naïve CD8N, cytotoxic effector CD8E), and regulatory T-cells.</p></div><div id="sec-22" class="subsection"><h3>HPC-driven model exploration and parameterization</h3><p id="p-110">The concurrent growth and advancements in the three areas of 1) mechanistic simulation modeling, 2) advanced, AI-driven model exploration algorithms, and 3) high-performance computing (HPC) provides the opportunity for large-scale exploration of the complex design spaces in detailed dynamical simulation models. However, if we don’t take deliberate efforts to formally facilitate this intersection across our research communities, we risk producing a series of disparate individual efforts, limited in interoperability, transparency, reproducibility and scalability. The EMEWS (extreme model exploration with Swift) framework<sup><a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a></sup> was developed to directly address this issue and to provide a broadly applicable cyberinfrastructure to lower the barriers for utilization of advanced, large-scale model exploration on HPC resources. The EMEWS paradigm allows for the direct exploitation of cutting edge statistical and machine learning algorithms that make up the vibrant ecosystem of free and open source libraries that are continually added to and updated as research frontiers are expanded, all while controlling simulation workflows that can be run anywhere from desktops to campus clusters and to the largest HPC resources.</p><p id="p-111">We have utilized EMEWS for learning-accelerated exploration of the parameter spaces of agent-based models of immunosurveillance against heterogeneous tumors<sup><a id="xref-ref-91-3" class="xref-bibr" href="#ref-91">91</a>,<a id="xref-ref-92-3" class="xref-bibr" href="#ref-92">92</a></sup>. The approach allowed for iterative and efficient discovery of optimal control and regression regions within biological and clinical constraints of the multi-scale biological systems. We have applied EMEWS across multiple science domains<sup><a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a>-<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a></sup> and developed large-scale algorithms to improve parameter estimation through approximate Bayesian computation (ABC) approaches<sup><a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a></sup>. These approaches, applied to the multi-scale modeling of SARS-CoV-2 dynamics, will provide the ability to robustly characterize model behaviors and produce improved capabilities for their interpretation.</p></div><div id="sec-23" class="subsection"><h3>nanoHUB platform</h3><p id="p-112">The nanoHUB platform (nanohub.org)<sup><a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">106</a></sup> is a free, cloud-based service offering lectures, tutorials, and, of particular interest to us, interactive Web-based simulation tools. As its name implies, it is primarily focused on nanoscale science education and research. To make their simulation tools easier to use, nanoHUB provides a custom toolkit for developing graphical user interfaces (GUIs). However, since 2017, they have adopted and promoted the use of Jupyter notebooks<sup><a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a></sup>, with accompanying Python modules to provide GUI widgets and visualization. Cloud-based computing and data analysis platforms are well established now, in both academic and commercial settings. For those who provide easy-to-use Web-based GUIs and APIs, and offer affordable pricing, it is likely their rate of adoption will continue to increase, especially among researchers who may lack the expertise or resources to install complex pieces of software.</p></div><div id="sec-24" class="subsection"><h3>xml2jupyter and cloud deployment of PhysiCell models</h3><p id="p-113">Compiled PhysiCell models generate executable software that runs at the command line. Model parameters are set by editing XML (extensible markup language) configuration files, and the models save data as a combination of vector graphics outputs (scalable vector graphics: SVG) and XML and Matlab files based on the draft MultiCellDS data standard<sup><a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a></sup>.</p><p id="p-114">To facilitate rapid cloud-hosted dissemination of PhysiCell-powered models on the nanoHUB platform, we developed <em>xml2jupyter</em> to automatically generate a Jupyter-based graphical user interface (GUI) for any PhysiCell model<sup><a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a></sup>. The Jupyter notebook includes widgets to set parameters, initiate a simulation run, and visualize diffusing substrates and cell agents. In turn, we also developed a protocol to deploy the PhysiCell model and the Jupyter notebook interface on nanoHUB as a cloud-hosted, interactive model. This allows developers to rapidly convert a locally executable, command-line model to a cloud-hosted shared model with graphical interface in a matter of minutes to hours (depending upon testing speed on nanoHUB).</p><p id="p-115">In our rapid prototyping, we use rapidly-generated nanoHUB apps for scientific communication across disciplines: virologists, pharmacologists, and other domain experts can explore and visualize the model prototypes without need to download, compile, and understand the code. This facilitates faster multidisciplinary dialog, and helps to draw in broader community feedback and contributions.</p></div><div id="sec-25" class="subsection"><h3>Modular design</h3><p id="p-116">The model will be evolved with a modular architecture. The overall model and each individual model component (submodel) will have a separate GitHub software repository in the pc4covid19 GitHub organization, available at:</p><p id="p-117"><a href="https://github.org/pc4covid19">https://github.org/pc4covid19</a></p><p id="p-118">Each module’s repository will consist of a <em>master</em> branch (which will always match the latest numbered release) and a <em>development</em> branch. Contributors will fork the development branch, complete their milestones, and submit a pull request to incorporate their progress in the development branch. Whenever the submodel team is ready to make a numbered release, they will use a pull request from the development branch to the master branch and create a numbered release.</p><p id="p-119">The overall model framework and each submodel will keep a versioned design document to include:</p><ul class="list-unord " id="list-8"><li id="list-item-65"><p id="p-120">A unique name for the model component</p></li><li id="list-item-66"><p id="p-121">A clear version number and last update timestamp</p></li><li id="list-item-67"><p id="p-122">A list of contributors, including 1-2 chief scientists who serve as primary points of contact</p></li><li id="list-item-68"><p id="p-123">A “plain English” description of the primary purpose of the component</p></li><li id="list-item-69"><p id="p-124">A statement of model inputs with units of measure.</p></li><li id="list-item-70"><p id="p-125">A clear statement of the biological hypotheses and assumptions of the component</p></li><li id="list-item-71"><p id="p-126">A record of the current mathematical form of the model (generally maintained in a separate Overleaf LaTeX document), with a snapshot of the Equations in the main design document</p></li><li id="list-item-72"><p id="p-127">Any computational implementation details needed to understand the code</p></li><li id="list-item-73"><p id="p-128">A link to a GitHub repository</p></li><li id="list-item-74"><p id="p-129">A list of model parameters, units, biophysical meaning, best estimate, and data source(s) for the parameter estimate. (See the discussion in MultiCellDS<sup><a id="xref-ref-108-2" class="xref-bibr" href="#ref-108">108</a></sup>.)</p></li><li id="list-item-75"><p id="p-130">A clear list of model outputs with units</p></li><li id="list-item-76"><p id="p-131">A set of qualitative and/or quantitative <strong>unit</strong> tests to ensure proper functionality of the module. A snapshot of this design document will be included in each release of the (sub)model.</p></li></ul><p id="p-132">The overall model releases will include a clear list of the version of each submodel included in its release.</p></div><div id="sec-26" class="subsection"><h3>Management structure</h3><p id="p-133">Each submodel will have 1-2 <strong>chief scientists</strong> in charge of managing development and approving pull requests from contributors. The submodel chief scientist(s) will meet regularly with their team to assign tasks, set milestones, and asses when to make a release. The submodel chief scientist will coordinate their progress with the other submodel teams.</p><p id="p-134">A <strong><em>core team</em></strong> will consist of the <strong>overall leads</strong> (as of April 1, 2020, this is Paul Macklin, Randy Heiland, and Yafei Wang) and the <strong>chief scientists</strong>. They will meet coordinate progress of the submodels, refine project scope, exchange ideas on model hypotheses, evaluate community feedback, and plan overall strategy. They will cooperate with the overall leads to create model releases (which will always bundle the most stable version of each submodel), update the nanoHUB models, and update the <em>bioRxiv</em> preprint.</p></div><div id="sec-27" class="subsection"><h3>Iterative development protocol</h3><p id="p-135">We use rapid prototyping, using lessons learned from each step to drive iteration towards improving the model.</p></div><div id="sec-28" class="subsection"><h3>Submodel design cycle</h3><p id="p-136">Each submodel will developed in parallel, using a unified prototyping or design cycle, under coordinatoin</p><ol class="list-ord " id="list-9"><li id="list-item-77"><p id="p-137">Submodel team sets priorities for the design iteration:</p><ol class="list-alphalower " id="list-10"><li id="list-item-78"><p id="p-138">Discuss feedback and identify highest priority model refinements.</p></li><li id="list-item-79"><p id="p-139">Refine model assumptions and hypotheses.</p></li><li id="list-item-80"><p id="p-140">Assess new data to refine parameter estimates.</p></li></ol></li><li id="list-item-81"><p id="p-141">“Translate” biological hypotheses into agent model rules and other mathematical model components:</p><ol class="list-alphalower " id="list-11"><li id="list-item-82"><p id="p-142">Run the new hypotheses and rules by domain experts as their time permits.</p></li><li id="list-item-83"><p id="p-143">Define new qualitative and/or quantitative unit tests for new behaviors and functions.</p></li><li id="list-item-84"><p id="p-144">Assign implementation tasks.</p></li></ol></li><li id="list-item-85"><p id="p-145">Perform computational implementation of refined mathematical model (and submodels):</p><ol class="list-alphalower " id="list-12"><li id="list-item-86"><p id="p-146">Address any bug reports.</p></li><li id="list-item-87"><p id="p-147">Add or modify functions based on new rules in steps 1-2.</p></li><li id="list-item-88"><p id="p-148">Test new or altered functions. Satisfy all qualitative and/or quantitative unit tests.</p></li><li id="list-item-89"><p id="p-149">Qualitatively test the model for new or improved behaviors over the last iteration.</p></li></ol></li><li id="list-item-90"><p id="p-150">Software release:</p><ol class="list-alphalower " id="list-13"><li id="list-item-91"><p id="p-151">Update documentation.</p></li><li id="list-item-92"><p id="p-152">Create a new numbered release on github.</p></li><li id="list-item-93"><p id="p-153">Update list of available validation data and best parameter estimates.</p></li><li id="list-item-94"><p id="p-154">Create a Zenodo snapshot.</p></li><li id="list-item-95"><p id="p-155">Communicate with the core team on the software release.</p></li></ol></li><li id="list-item-96"><p id="p-156">Cloud-hosted submodel for multidisciplinary testing:</p><ol class="list-alphalower " id="list-14"><li id="list-item-97"><p id="p-157">Update the nanoHUB app repository with new code.</p></li><li id="list-item-98"><p id="p-158">Run xml2jupyter to update the Jupyter interface.</p></li><li id="list-item-99"><p id="p-159">Update project on nanoHUB, test/refine until successful release</p></li><li id="list-item-100"><p id="p-160">Update documentation, numbered GitHub release, zenodo snapshot of deployed model.</p></li><li id="list-item-101"><p id="p-161">Perform live demos with the core team as needed.</p></li></ol></li><li id="list-item-102"><p id="p-162">Submodel team evaluation:</p><ol class="list-alphalower " id="list-15"><li id="list-item-103"><p id="p-163">Distill feedback to assess the need for new model hypotheses, behaviors, or components.</p></li><li id="list-item-104"><p id="p-164">Assess which biological behaviors are currently exhibited by the model.</p></li><li id="list-item-105"><p id="p-165">Refine the design protocol (e.g., with refined model specification methods) as necessary.</p></li><li id="list-item-106"><p id="p-166">Assess the need for an additional design iteration.</p></li></ol></li><li id="list-item-107"><p id="p-167">Update preprint for scientific dissemination. Return to Step 1 if there is substantial feedback, or if the core team determines that further refinements are within project scope.</p></li></ol></div><div id="sec-29" class="subsection"><h3>Overall model design cycle</h3><p id="p-168">In each prototyping or design cycle:</p><ol class="list-ord " id="list-16"><li id="list-item-108"><p id="p-169">Core team sets priorities for the design iteration:</p><ol class="list-alphalower " id="list-17"><li id="list-item-109"><p id="p-170">Discuss feedback and identify highest priority model refinements.</p></li><li id="list-item-110"><p id="p-171">Collaborate to update the submodel design documents to address feedback</p></li><li id="list-item-111"><p id="p-172">Update the overall model design document as needed.</p></li><li id="list-item-112"><p id="p-173">Assess new data to refine parameter estimates.</p></li><li id="list-item-113"><p id="p-174">Refine submodel input/output formats as necessary.</p></li><li id="list-item-114"><p id="p-175">Assess next release dates for the submodels.</p></li></ol></li><li id="list-item-115"><p id="p-176">Submodel teams meet to refine their code and put out their next releases. (See above.) The chief scientists communicate releases to the overall leads.</p></li><li id="list-item-116"><p id="p-177">Integrate the latest submodel releases into a new release candidate for the overall model.</p></li><li id="list-item-117"><p id="p-178">Perform computational implementation of refined mathematical model (and submodels):</p><ol class="list-alphalower " id="list-18"><li id="list-item-118"><p id="p-179">Address any bug reports.</p></li><li id="list-item-119"><p id="p-180">Test new or altered functions. Satisfy all qualitative and/or quantitative unit tests.</p></li><li id="list-item-120"><p id="p-181">Qualitatively test the model for new or improved behaviors over the last iteration.</p></li></ol></li><li id="list-item-121"><p id="p-182">Software release:</p><ol class="list-alphalower " id="list-19"><li id="list-item-122"><p id="p-183">Update documentation.</p></li><li id="list-item-123"><p id="p-184">Create a new numbered release on GitHub.</p></li><li id="list-item-124"><p id="p-185">Update list of available validation data and best parameter estimates.</p></li><li id="list-item-125"><p id="p-186">Create a Zenodo snapshot.</p></li><li id="list-item-126"><p id="p-187">Announce on Twitter (via @PhysiCell, @MathCancer, and @SMB_imin).</p></li></ol></li><li id="list-item-127"><p id="p-188">Cloud-hosted model for multidisciplinary testing:</p><ol class="list-alphalower " id="list-20"><li id="list-item-128"><p id="p-189">Update the nanoHUB app repository with new code.</p></li><li id="list-item-129"><p id="p-190">Run xml2jupyter to update the Jupyter interface.</p></li><li id="list-item-130"><p id="p-191">Update project on nanoHUB, test/refine until successful release</p></li><li id="list-item-131"><p id="p-192">Update documentation, numbered GitHub release, zenodo snapshot of deployed model.</p></li><li id="list-item-132"><p id="p-193">Perform live demos with domain experts and community to gather feedback.</p></li></ol></li><li id="list-item-133"><p id="p-194">Seek additional community feedback via twitter and the pc4covid19 slack workspace [ref]. Integrate comments received from scientific peer review as appropriate.</p></li><li id="list-item-134"><p id="p-195">Core group evaluation:</p><ol class="list-alphalower " id="list-21"><li id="list-item-135"><p id="p-196">Distill feedback to assess the need for new model hypotheses, behaviors, or components.</p></li><li id="list-item-136"><p id="p-197">Assess which biological behaviors are currently exhibited by the model.</p></li><li id="list-item-137"><p id="p-198">Refine the design protocol (e.g., with refined model specification methods) as necessary.</p></li><li id="list-item-138"><p id="p-199">Assess the need for an additional design iteration.</p></li></ol></li><li id="list-item-139"><p id="p-200">Update preprint for scientific dissemination. Return to Step 1 if there is substantial feedback, or if the core team determines that further refinements are within project scope.</p></li></ol><p id="p-201">Once the model can qualitatively produce expected viral and immune behaviors (as determined by the core group) and receives no major domain expert or community critiques, the goal of this paper will be met: to create a SARS-CoV-2 modeling framework suitable for subsequent calibration, validation, and exploration by the community. It will be submitted to scientific peer review, disseminated to the community, and maintained.</p><p id="p-202">At the conclusion of rapid prototyping, software and community maintenance will:</p><ol class="list-ord " id="list-22"><li id="list-item-140"><p id="p-203">Address reported software bugs</p></li><li id="list-item-141"><p id="p-204">Maintain and refine documentation</p></li><li id="list-item-142"><p id="p-205">Create online training and educational materials</p></li><li id="list-item-143"><p id="p-206">Coordinate with the community to collect and refine best parameter estimates and validation results.</p></li></ol><p id="p-207">We anticipate that many teams will use this code base for independent projects that calibrate and validate the model for tailored scientific investigations. We will endeavor to create a registry of these efforts.</p></div></div><div class="section" id="sec-30"><h2 class="">Results</h2><div id="sec-31" class="subsection"><h3>Version 1 (March 25-March 31, 2020)</h3><p id="p-208">Version 1 was designed as proof of concept rapid prototype to capture essential (but highly simplified) elements of viral endocytosis, protein synthesis, viral assembly, release, and diffusion to infect other cells. The model was tailored to RNA viruses on a tissue monolayer (modeled as a layer of epithelium over a basement membrane). This version was kept deliberately simple to create an early starting framework to help coalesce community feedback and contributions. It was also designed to test the use of interactive cloud-hosted models to help accelerate feedback by virologists and other domain experts through live demos.</p><p id="p-209">The proof of concept model was created by the overall leads (Macklin, Heiland, Wang) while assembling the modeling coalition as an initial starting point and feasibility test for rapid prototyping. Feedback on this version drove the formulation of the design protocols recorded in this paper.</p></div><div id="sec-32" class="subsection"><h3>Submodels</h3><p id="p-210">The Version 1 model includes the following submodel components:</p><ul class="list-unord " id="list-23"><li id="list-item-144"><p id="p-211"><strong>T:</strong> tissue (which contains epithelial and other cells)</p></li><li id="list-item-145"><p id="p-212"><strong>V:</strong> viral endocytosis, replication, and exocytosis repsonses</p></li><li id="list-item-146"><p id="p-213"><strong>VR:</strong> cell response to viral replication, including cell death and IFN synthesis</p></li><li id="list-item-147"><p id="p-214"><strong>E:</strong> epithelial cell (incorporates V and VR).</p></li></ul><p id="p-215">The overall model components are summarized in <strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Version 1 model schematic: Left: In the overall model structure, a tissue component (T) contains multiple epithelial cells (E). Viral particles diffuse through this domain and can enter the cells (by adhering as an uptake term) or be exported by cells. Each cell includes an intracellular virus model (V) for viral replication kinetics and a viral response model (VR) which uses a pharmacodynamic model to control cell apoptotic response to viral load. Right: Each epithelial cell individually models its viral kinetics: adhered virions complete endocytosis, are uncoated, functionalize their RNA, synthesize viral components, and assemble them into virions that are subsequently exported to the external tissue compartment (T)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845456715" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Version 1 model schematic:&lt;/span&gt; &lt;em&gt;Left:&lt;/em&gt; In the overall model structure, a &lt;strong&gt;&lt;em&gt;tissue component&lt;/em&gt; (T)&lt;/strong&gt; contains multiple &lt;strong&gt;&lt;em&gt;epithelial cells&lt;/em&gt; (E)&lt;/strong&gt;. Viral particles diffuse through this domain and can enter the cells (by adhering as an uptake term) or be exported by cells. Each cell includes an intracellular &lt;strong&gt;&lt;em&gt;virus&lt;/em&gt;&lt;/strong&gt; model &lt;strong&gt;(V)&lt;/strong&gt; for viral replication kinetics and a &lt;strong&gt;&lt;em&gt;viral response&lt;/em&gt;&lt;/strong&gt; model &lt;strong&gt;(VR)&lt;/strong&gt; which uses a pharmacodynamic model to control cell apoptotic response to viral load. &lt;strong&gt;&lt;em&gt;Right:&lt;/em&gt;&lt;/strong&gt; Each epithelial cell individually models its viral kinetics: adhered virions complete endocytosis, are uncoated, functionalize their RNA, synthesize viral components, and assemble them into virions that are subsequently exported to the external tissue compartment (T).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F1.medium.gif" width="440" height="174"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F1.medium.gif" width="440" height="174"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">Version 1 model schematic:</span><p id="p-216" class="first-child"><em>Left:</em> In the overall model structure, a <strong><em>tissue component</em> (T)</strong> contains multiple <strong><em>epithelial cells</em> (E)</strong>. Viral particles diffuse through this domain and can enter the cells (by adhering as an uptake term) or be exported by cells. Each cell includes an intracellular <strong><em>virus</em></strong> model <strong>(V)</strong> for viral replication kinetics and a <strong><em>viral response</em></strong> model <strong>(VR)</strong> which uses a pharmacodynamic model to control cell apoptotic response to viral load. <strong><em>Right:</em></strong> Each epithelial cell individually models its viral kinetics: adhered virions complete endocytosis, are uncoated, functionalize their RNA, synthesize viral components, and assemble them into virions that are subsequently exported to the external tissue compartment (T).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-33" class="subsection"><h3>Biological hypotheses</h3><p id="p-217">In this proof of concept prototype, we modeled a simplified set of biological hypotheses:</p><p id="p-218">1.T.1 Virus diffuses in the microenvironment with low diffusion coefficient</p><p id="p-219">1.T.2 Virus adhesion to a cell stops its diffusion (acts as an uptake term)</p><p id="p-220">1.V.1 Adhered virus undergoes endocytosis and then becomes uncoated</p><p id="p-221">1.V.2 Uncoated virus (viral contents) lead to release of functioning RNA</p><p id="p-222">1.V.3 RNA creates protein forever, unless it degrades</p><p id="p-223">1.V.4 Protein is transformed to an assembled virus state</p><p id="p-224">1.V.5 Assembled virus is released by the cell</p><p id="p-225">1.VR.1 As a proxy for viral disruption of the cell, the probability of cell death increases with the total number of assembled virions</p><p id="p-226">1.VR.2 Apoptosed cells lyse and release some or all of their contents</p><p id="p-227">(In the above, X.C.Y denotes prototype X, modeling component C, biological hypothesis Y, allowing us to easily refer to any individual hypothesis or assumption in discussion and community feedback.) In the next version of this model, we will use the design document protocols for each of these components.</p><div id="sec-34" class="subsection"><h4>Unit tests</h4><p id="p-228">The first prototype should demonstrate the following behaviors for a single cell infected by a single virion:</p><ul class="list-unord " id="list-24"><li id="list-item-148"><p id="p-229">The virion progresses to the uncoated state.</p></li><li id="list-item-149"><p id="p-230">The uncoated virion progresses to the RNA state.</p></li><li id="list-item-150"><p id="p-231">With export and death off, RNA produces protein.</p></li><li id="list-item-151"><p id="p-232">With export and death turned off, protein produces and accumulates assembled virus (linearly).</p></li><li id="list-item-152"><p id="p-233">With export off and death on, cell undergoes apoptosis with increasing likelihood as assembled virus accumulates.</p></li><li id="list-item-153"><p id="p-234">With export on and death on, surrounding cells get infected and create virion.</p></li><li id="list-item-154"><p id="p-235">Cells nearest the initial cell are infected first.</p></li><li id="list-item-155"><p id="p-236">Apoptosis is most frequent nearest to the initial infected cell.</p></li></ul></div></div><div id="sec-35" class="subsection"><h3>Translation to mathematics, rules and model components</h3><div id="sec-36" class="subsection"><h4>Extracellular virion transport (Tissue submodel <strong><em>T</em></strong>)</h4><p id="p-237">To rapidly implement extracellular viral transport using existing model capabilities, we approximated the process as diffusion with a small diffusion coefficient as in prior nanoparticle models. Using the standard BioFVM formulation<sup><a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">87</a></sup>, if <em>ρ</em> is the concentration of virions (virions / <em>μ</em>m<sup>3</sup>), then it is modeled as:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-2.gif"/></noscript></span>

</span>
where <em>D</em> is the diffusion coefficient, <em>λ</em> is the net decay rate (which can include other removal processes), U is the uptake rate (by adhering to ACE2 and initiating endocytosis), and E is the cell’s virion export rate. (Here, delta is the Dirac delta function, <em>V</em><sub><em>i</em></sub> is the cell’s volume, and <strong>x</strong><sub><em>i</em></sub> is position of the cell’s center.) Note that in the default BioFVM implementation, uptake processes are spread across the cell’s volume.</p><p id="p-238">Note that virus propagation may require more explicit modeling of cell-cell surface contact in later versions, as well as cilia-driven advective transport and virion deposition (e.g., through airway transport).</p></div><div id="sec-37" class="subsection"><h4>Intracellular viral replication dynamics (Virus intracellular model <strong><em>V</em></strong>)</h4><p id="p-239">Within each cell, we track <em>V</em> (adhered virions in the process of endocytosis), <em>U</em> (uncoated viral RNA and proteins), R (viral RNA ready for protein synthesis; R = 1 denotes one virion’s total mRNA), P (synthesized viral proteins; P = 1 denotes sufficient viral protein to assemble a complete virion), and A (total assembled virions ready for exocytosis). Virion import (a source term for V) is handled automatically by the mass conservation terms for PhysiCell in the PDE solutions.</p><p id="p-240">We model these dynamics of internalized virions through a simplified system of ODEs:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-3.gif"/></noscript></span>

</span>
</p><p id="p-241">We model exocytosis by setting the export rate E of the assembled virions, in units of virions per time:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-4.gif"/></noscript></span>

</span>
</p></div><div id="sec-38" class="subsection"><h4>Cell response (Viral response submodel <strong><em>VR</em></strong>)</h4><p id="p-242">In this proof of concept prototype, we modeled apoptotic response to cell disruption but did not model interferon processes. As a simplification, we modeled cell disruption as correlated with assembled virions <em>A</em>, and we used a Hill pharmacodynamic model to relate the cell’s apoptosis rate to <em>A</em>:
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-5.gif"/></noscript></span>

</span>
where <em>e</em> is the effect, <em>n</em> is the Hill coefficient, and <em>A</em><sub><em>H</em></sub> is the amount of virions at which half-max effect is achieved. We then set the apoptotic death rate at
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/05/2020.04.02.019075/embed/graphic-6.gif"/></noscript></span>

</span>
where <em>e</em> is the maximum apoptosis rate (at full effect, <em>e</em> = 1). As analyzed for agent-based models with stochastic death rates<sup><a id="xref-ref-82-4" class="xref-bibr" href="#ref-82">82</a>,<a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">110</a></sup>, in any time interval [<em>t, t</em>+Δ<em>t</em>], the cell has probability <em>r</em><sub><em>death</em></sub>Δt of apoptosing, and the mean cell survival time (for fixed <em>e</em> and thus fixed <em>r</em><sub><em>death</em></sub>) is 1/<em>r</em><sub><em>death</em></sub>.</p><p id="p-243">In PhysiCell, we can set the lysing cells to release any fraction (0 ≤ <em>f</em><sub><em>release</em></sub> ≤ 1) of <em>V, A, U, R</em>, and <em>P</em> into the extracellular environment as diffusing substrates.</p></div></div><div id="sec-39" class="subsection"><h3>Other implementation notes</h3><p id="p-244">To differentiate between incoming imported and exported virions, we actually modeled two diffusing fields (for extracellular concentrations of <em>V</em> and <em><span class="underline">A</span></em>). At the end of each computational step, we manually move all of the exported assembled virions in each voxel into the concentration of diffusing virions.</p><p id="p-245">We also created diffusing fields for uncoated virions, RNA, and viral proteins, for use in future models where these may be immunogenic or linked to measureable data.</p></div><div id="sec-40" class="subsection"><h3>Software release</h3><p id="p-246">The core model associated with the v1 prototype is Version 0.1.3. The nanoHUB app associated with the v1 prototype is Version 1.0. GitHub releases and zenodo snapshots are given in the appendix in the appendix.</p></div><div id="sec-41" class="subsection"><h3>Cloud-hosted model</h3><p id="p-247">We rapidly created and deployed a cloud-hosted model with an interactive Web-based graphical user interface (GUI) running on nanoHUB (nanohub.org) using xml2jupyter Version 1.1<sup>109</sup>. The web-hosted model can be run at <a href="https://nanohub.org/tools/pc4covid19">https://nanohub.org/tools/pc4covid19</a>.</p><p id="p-248">This workflow uses a Python script that converts a PhysiCell configuration file (in XML) into a Jupyter notebook and adds additional Python modules for the GUI. The automated process of converting a standalone PhysiCell model into an interactive Jupyter notebook version (a GUI) takes just a few minutes. The resulting GitHub repository is shared with the nanoHUB system administrators who install it for testing as an online, executable model (an “app”). After we perform usability and other testing and finalize documentation,, it is published and becomes available for public use. The whole process (including the initial development of the core PhysiCell model) took less than 12 hours for this particular app. See <strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a></strong>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Version 1 on nanoHUB: A. The About tab includes background information on the model, run directions, and a legend. B. The Config basics tab allows users to set the domain size. (These should be left alone in this version.) C. The Microenvironment tab lets users set diffusion and decay coefficients, as well as initial and boundary conditions. The track_in_agents option must be enabled for the model to work correctly. D. The User Params tab sets key model parameters. E-F. Sample outputs in the Out: Plots tab. Users can scroll the &#x201C;frame&#x201D; tab to advance the simulation time, and choose whether to plot the cells, a diffusing substrate, or both. In these plots, we are looking at a sheet of epithelium (circles are individual cells). Brighter yellow cells contain more assembled virion. Black cells are apoptotic, and clear spaces show tissue damage (where cells have apoptosed and been removed.) This version of the nanoHUB model was released at DOI 10.21981/19BB-HM69." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845456715" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Version 1 on nanoHUB:&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; The &lt;em&gt;About&lt;/em&gt; tab includes background information on the model, run directions, and a legend. &lt;strong&gt;B.&lt;/strong&gt; The &lt;em&gt;Config basics&lt;/em&gt; tab allows users to set the domain size. (These should be left alone in this version.) &lt;strong&gt;C.&lt;/strong&gt; The &lt;em&gt;Microenvironment&lt;/em&gt; tab lets users set diffusion and decay coefficients, as well as initial and boundary conditions. The track_in_agents option must be enabled for the model to work correctly. &lt;strong&gt;D.&lt;/strong&gt; The &lt;em&gt;User Params&lt;/em&gt; tab sets key model parameters. &lt;strong&gt;E-F.&lt;/strong&gt; Sample outputs in the &lt;em&gt;Out: Plots&lt;/em&gt; tab. Users can scroll the &#x201C;frame&#x201D; tab to advance the simulation time, and choose whether to plot the cells, a diffusing substrate, or both. In these plots, we are looking at a sheet of epithelium (circles are individual cells). Brighter yellow cells contain more assembled virion. Black cells are apoptotic, and clear spaces show tissue damage (where cells have apoptosed and been removed.) This version of the nanoHUB model was released at DOI 10.21981/19BB-HM69.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F2.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F2.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Version 1 on nanoHUB:</span><p id="p-249" class="first-child"><strong>A.</strong> The <em>About</em> tab includes background information on the model, run directions, and a legend. <strong>B.</strong> The <em>Config basics</em> tab allows users to set the domain size. (These should be left alone in this version.) <strong>C.</strong> The <em>Microenvironment</em> tab lets users set diffusion and decay coefficients, as well as initial and boundary conditions. The track_in_agents option must be enabled for the model to work correctly. <strong>D.</strong> The <em>User Params</em> tab sets key model parameters. <strong>E-F.</strong> Sample outputs in the <em>Out: Plots</em> tab. Users can scroll the “frame” tab to advance the simulation time, and choose whether to plot the cells, a diffusing substrate, or both. In these plots, we are looking at a sheet of epithelium (circles are individual cells). Brighter yellow cells contain more assembled virion. Black cells are apoptotic, and clear spaces show tissue damage (where cells have apoptosed and been removed.) This version of the nanoHUB model was released at DOI 10.21981/19BB-HM69.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-42" class="subsection"><h3>Model behavior: what does the current version teach us?</h3><p id="p-250">Except as noted below, all simulation results use the v1 model default parameters, which are supplied in the XML configuration parameter file of the version 0.1.2 core model repository.</p><p id="p-251">In all plots, dark blue cells have 0 assembled virus, pale blue cells have 1-9 assembled virions, grey cells have 10-99 assembled virions, light yellow cells have 100-999 assembled virions, and bright yellow cells contain 1000 or more assembled virions. Black cells are apoptotic, and white spaces show regions devoid of cells (extensive tissue damage). See the legend in <strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a></strong> (A).</p><div id="sec-43" class="subsection"><h4>Behavior with default parameters</h4><p id="p-252">Running the overall model (with virus release turned on and off as appropriate for the respective unit tests) shows that the v1 prototype satisfies all the qualitative unit tests: a single cell is infected with a virion in the center of the tissue. Over time, the virion is uncoated to create functionalized RNA, which is synthesized to viral proteins and assembled to functional virus. The graphical output shows this center cell turning to a bright yellow as assembled virions accumulate. By enabling the substrate plot, we can see the diffusive field of virions first has zero concentration (no virions have been released), but as the first cell’s viral production increases, it releases virus particles that begin diffusing into the domain. See <strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3 (A)</a></strong>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Version 1 sample model results at 6, 12, 18, and 24 hours. In all plots, epithelial cells are colored from blue (no assembled virions) to bright yellow (1000 or more virions). Black cells are apoptotic, and white regions show damaged tissues where apoptotic cells have degraded to expose (unmodeled) basement membrane. Bar: 200 &#x3BC;m. A. Simulation time course for the default parameters. Note the spread of the infection from an initial infected cell at the center, with apoptotic death events focused near the center. B. Decreasing the diffusion coefficient of virions by a factor of 10 drastically reduces the rate of spread, although focusing exocytosed virions in a smaller diffusion distance increases the number of virions infecting nearby cells, leading to faster apoptosis. C. Allowing apoptosed cells to release their assembled virions at lysis had a negligible effect for these parameters, given the dominant effects of releasing virions throughout the cells&#x2019; survival times. D. Decreasing the cell&#x2019;s tolerance (half max) of assembled virions prior to apoptosis accelerates tissue damage but does not drastically accelerate the spread of the infection. E. Increasing the apoptosis rate (or decreasing the survival time) for infected cells drastically increases tissue degradation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845456715" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Version 1 sample model results at 6, 12, 18, and 24 hours.&lt;/span&gt; In all plots, epithelial cells are colored from blue (no assembled virions) to bright yellow (1000 or more virions). Black cells are apoptotic, and white regions show damaged tissues where apoptotic cells have degraded to expose (unmodeled) basement membrane. Bar: 200 &lt;em&gt;&#x3BC;&lt;/em&gt;m. &lt;strong&gt;A.&lt;/strong&gt; Simulation time course for the default parameters. Note the spread of the infection from an initial infected cell at the center, with apoptotic death events focused near the center. &lt;strong&gt;B.&lt;/strong&gt; Decreasing the diffusion coefficient of virions by a factor of 10 drastically reduces the rate of spread, although focusing exocytosed virions in a smaller diffusion distance increases the number of virions infecting nearby cells, leading to faster apoptosis. &lt;strong&gt;C.&lt;/strong&gt; Allowing apoptosed cells to release their assembled virions at lysis had a negligible effect for these parameters, given the dominant effects of releasing virions throughout the cells&#x2019; survival times. &lt;strong&gt;D.&lt;/strong&gt; Decreasing the cell&#x2019;s tolerance (half max) of assembled virions prior to apoptosis accelerates tissue damage but does not drastically accelerate the spread of the infection. &lt;strong&gt;E.&lt;/strong&gt; Increasing the apoptosis rate (or decreasing the survival time) for infected cells drastically increases tissue degradation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F3.medium.gif" width="338" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F3.medium.gif" width="338" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.02.019075/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">Version 1 sample model results at 6, 12, 18, and 24 hours.</span><p id="p-253" class="first-child">In all plots, epithelial cells are colored from blue (no assembled virions) to bright yellow (1000 or more virions). Black cells are apoptotic, and white regions show damaged tissues where apoptotic cells have degraded to expose (unmodeled) basement membrane. Bar: 200 <em>μ</em>m. <strong>A.</strong> Simulation time course for the default parameters. Note the spread of the infection from an initial infected cell at the center, with apoptotic death events focused near the center. <strong>B.</strong> Decreasing the diffusion coefficient of virions by a factor of 10 drastically reduces the rate of spread, although focusing exocytosed virions in a smaller diffusion distance increases the number of virions infecting nearby cells, leading to faster apoptosis. <strong>C.</strong> Allowing apoptosed cells to release their assembled virions at lysis had a negligible effect for these parameters, given the dominant effects of releasing virions throughout the cells’ survival times. <strong>D.</strong> Decreasing the cell’s tolerance (half max) of assembled virions prior to apoptosis accelerates tissue damage but does not drastically accelerate the spread of the infection. <strong>E.</strong> Increasing the apoptosis rate (or decreasing the survival time) for infected cells drastically increases tissue degradation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-254">Over time, neighboring cells also become infected. They, too progress towards a higher viral load (increasingly bright shades of yellow). The infection propagates outward from the initially infected cell into the remaining tissue. As each cell’s viral load (here measured as number of assembled virions) increases, the viral response model calculates the increasing effect <em>e</em>, and cells have greater probability of apoptosis. Cells nearest to the initial site of infection apoptose earliest. As these cells degrade, they are removed from the simulation, leading to the creation of a degraded, cell-free region near the center of the tissue. This degraded region spreads outwards from the initial site of infection over time.</p><p id="p-255">See <strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3 (A)</a></strong> for a simulation with default parameters. The nanoHUB distribution of this model takes approximately 60-90 seconds to execute.</p></div><div id="sec-44" class="subsection"><h4>Impact of the virion diffusion coefficient</h4><p id="p-256">We next tested the effect of the viral diffusion coefficient by reducing it from 900 μm<sup>2</sup>/min to 90 μm<sup>2</sup>/min. Because the viral particles spread less distance after their release, they reach other cells more slowly, and the overall spread of the infection is slowed. See <strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3 (B)</a></strong>.</p><p id="p-257">We left <em>D</em> = 90 μm<sup>2</sup>/min for all subsequent investigations of the v1 model.</p></div><div id="sec-45" class="subsection"><h4>Impact of the viral release at cell death</h4><p id="p-258">We next tested the effect of releasing all assembled viral particles at the time of cell death by setting <em>f</em><sub>release</sub> = 1. For this set of model parameters, the release of assembled virions had a negligent impact of the overall spread of infection: Compare the final frame of row B (no release: <em>f</em><sub>release</sub> = 0) to row C (compete release: <em>f</em><sub>release</sub> = 1) in <strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a></strong>. This is because cells release far more virions during their infected lifetimes, so the effect is dominant over the one-time release of virions at cell death. We expect this behavior would change if the cells exocytosed virions more slowly.</p></div><div id="sec-46" class="subsection"><h4>Impact of the cell tolerance to viral load</h4><p id="p-259">We next decreased the cell tolerance to viral load by decreasing the pharmacodynamic half max <em>A</em><sub><em>H</em></sub> from 500 virions to 10, while leaving <em>f</em><sub>release</sub> = 1. As expected, cell death and tissue damage occurred much more quickly under these parameters. Interestingly (and contrary to intuition), this did not significantly alter the rate at which the infection spread through the tissue. Compare the final frame of row C (higher tolerance to viral load) to row D (lower tolerance to viral load) in <strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a></strong>. This shows the importance of creating spatiotemporal models of viral replication in tissues, as the balance of competing processes can lead to unexpected dynamics at the tissue, organ, and organism levels.</p></div><div id="sec-47" class="subsection"><h4>Impact of the cell survival time under high viral loads</h4><p id="p-260">We next decreased the cell tolerance to viral load further by decreasing the mean cell survival time under high viral loads, which is equivalent to increasing the maximum apoptosis rate <em>r</em><sub>max</sub>. (Following prior analyses<sup><a id="xref-ref-82-5" class="xref-bibr" href="#ref-82">82</a>,<a id="xref-ref-110-2" class="xref-bibr" href="#ref-110">110</a></sup>, Recall that 1/<em>r</em><sub>max</sub> is the mean expected survival time as <em>A</em> → ∞.) We increased rmax from 0.001 min<sup>-1</sup> (1000 minute expected lifetime at high loads) to 0.01 min<sup>-1</sup> (100 minute expected lifetime at high viral loads). surface under the epithelial monolayer) exposed. In a later version of this model framework, we would expect this to lead to earlier onset of fluid leakage, edema, and ultimately adverse respiratory outcomes such as ARDS. Interestingly, this did not significantly increase the rate of spread of the infection. Compare the final frame of row D (higher tolerance to viral load) to row E (lower tolerance to viral load) in <strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a></strong>.</p></div></div><div id="sec-48" class="subsection"><h3>Key feedback from domain experts and the community</h3><p id="p-261">We gathered feedback from the multidisciplinary community, several of whom joined the coalition for future work. We summarize the feedback below.</p><p id="p-262"><strong>Aarthi Narayanan (virology):</strong> More detail on endocytosis, viral uncoating, and synthesis would expose more actionable points in the replication cycle. Preliminary SARS-CoV-2 experiments in her laboratory suggest that the time course (and thus general order of magnitude of rate parameters) is very similar to Venezuelen equine encephilitis virus (VEEV) dynamics measured earlier<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a></sup>. The exponential progression matches observations: the first cell is infected with one virion and so at first produces virus slowly, but neighboring cells can be infected with multiple virions and thus create virus particles more quickly.</p><p id="p-263"><strong>Simon Parkinson</strong> identified typographical errors in the original documentation, but verified that that mathematics in the C++ implementation were not affected. He emphasized the importance of implementing RNA decay (as a rate limiting step in virus replication) and the importance of integrating ACE2 receptor trafficking (as a rate limiting step in virus adhesion and endocytosis).</p><p id="p-264"><strong>Paul Macklin (multicellular systems biology, open source frameworks)</strong> noted the potential to simplify the model by removing the diffusing <em>U, R</em>, and <em>P</em> fields, and reported bugs in the initialization (where no cells are initially infected for some domain sizes, due to hard-coding of the initial seeding).</p><p id="p-265"><strong>Morgan Craig and Adrianne Jenner (mathematical biology and viral dynamics)</strong> emphasized the importance of varying virion “uptake” with ACE2 receptor availability, and hence the need to integrate receptor trafficking.</p><p id="p-266"><strong>Amber Smith (mathematical biology and infectious diseases)</strong> noted her prior work on SARS-CoV-1 will be of tremendous help in estimating parameters and building initial immunologic regulation models. Lung pathology and disease severity are closely tied to the immunologic reaction, and prior data and images from influenza will be of tremendous help with calibrating spatial considerations. She noted that she expects mouse and drug data available for SARS-CoV-2 in the coming months. She noted the importance of distinguishing between mild and severe ARDS. One quick possibility to make this match data and distinguish between possibilities is to plot the resulting viral load.</p><p id="p-267">She suggested that it would be helpful to show multi focal points of initial infection seeding (possibly of different initial seeding size) that merge together over time, which would match observations of lung histology. Future work will have a better impact if they use a true lung tissue geometry with immune cells limiting the peripheral spread. The current model seems more relevant to in vitro growth of a single plaque, which may be scrutinized.</p><p id="p-268"><strong>Richard Allen (quantitative systems pharmacology, Pfizer, Inc.)</strong> pointed out the need for clearer scoping and diagrams to clearly lay out the design of each submodel component. We will need procedures to choose future incorporations and changes of scope. He also pointed out the need to understand what happens if you bind up a lot of ACE2 with receptor; there are early insights online<sup><a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">111</a></sup>.</p><p id="p-269"><strong>Ashlee M. Ford Versypt (mathematical biology, bioengineering, inflammation and tissue damage)</strong> noted that the diffusion coefficient of 900 μm<sup>2</sup>/min = 15 μm<sup>2</sup>/s = 1.5e-11 m<sup>2</sup>/s is not particularly small; prior analyses<sup><a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">112</a></sup> considered virion diffusion in an lung epithelial monolayer for influenza with D = 3.18e-15m<sup>2</sup> estimating from experimental data. The virions for SARS-CoV-2 could be more mobile though-it’s uncertain. There are data about the diffusion coefficient for albumin in tissue being on the order of 10-50 μm<sup><a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a></sup>/s; see this reference<sup><a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">113</a></sup>. She stated that it makes sense for a viron to move more slowly than a protein with radius &lt; 5 nm unless “diffusive transport” is encompassing an active or facilitated transport mode beyond just classic diffusion. She also noted that her laboratory has looked a lot at the renin-angiotensin-system systemically and in kidneys: the kinetics of AngII, ACE, and ACE2 in the lungs would be of interest for connecting the next iteration of the ACE2 receptor model to connect to ARDS. Pfizer may also have relevant related models.</p><p id="p-270"><strong>Courtney L. Davis (mathematical biology, infectious diseases and ecology)</strong> noted that the model could study immune responses and the impact of mucosal structure in future versions. She suggested quantifying damage or disease metrics. She also noted that ultimately it would be useful to note which parameter estimates might be species-specific and which are not, to be able to switch between experimental and clinical systems. (e.g., it is worth recording if current estimates are from human, macaque, etc.)</p><p id="p-271">She also noted that it may be important to determine if apoptotic cells replaced or if there is permanent damage (in the model). If the model is run longer, it would be worthwhile to translate the visual sense of damage to a quantitative metric.</p><p id="p-272"><strong>Chase Cockrell and Gary An</strong> noted their work on modeling immune expansion in “off screen” lymph nodes, and offered to link their model to our immune infiltration functions.</p><p id="p-273"><strong>James Glazier</strong> noted the need for clearly specifying each model’s assumptions, inputs and outputs, to drive robust parallel development. He noted that it is critical to consider information flow between submodels and revise these data flows as the iterations proceed. He suggested that we state separate execution of sub models as a key design goal to support parallel development. Lastly, he noted that software should be released in conjunction with validation data and methodologies</p></div><div id="sec-49" class="subsection"><h3>Core team discussion and priorities for v2</h3><p id="p-274">The core team met by virtual conference on April 1, 2020 to discuss the first preprint, model results, and feedback. The core team set as priorities (1) to formalize design specifications for each individual model component and interfaces between components, (2) form teams responsible for each component, (3) focus v2 development on refactoring into this modular format, (3) begin development of the submodels, and (4) begin refine parameter estimates. The clearer specification and organization of submodels was the top priority. As time permits, it was also viewed as important to begin a receptor trafficking model.</p><p id="p-275">The core team agreed to keep working via the dedicated slack workspace to rapidly coalesce on the submoel teams. Each subteam will have a separate channel in the workspace.</p></div></div><div class="section" id="sec-50"><h2 class="">Discussion</h2><p id="p-276">Within three weeks of the World Health Organization’s declaration of a global pandemic of COVID-19<sup>114</sup>, community-based prototyping built upon an existing PhysiCell 3D cell-modeling framework to rapidly develop Version 1 of an intracellular and tissue-level model of SARS-CoV-2<sup>82</sup>. A growing coalition of domain experts from across STEM fields are working together to ensure accuracy and utility of this agent-based model of intracellular, extracellular, and multicellular SARS-CoV-2 infection dynamics. Version 1 development underscored the necessity of clearly explaining model components, defining scope, and communicating progress as it occurs for invaluable real-time feedback from collaborators and the broader community. This rapid prototyping already helped in growing the coalition and recruiting complementary expertise; for instance, a team modeling lymph node dynamics and immune infiltration joined during the Version 1 cycle after seeing initial progress.</p><p id="p-277">The version 1 prototype also showed the scientific benefit of rapid prototyping: even a basic coupling between extracellular virion transport, intracellular replication dynamics, and viral response (apoptosis) showed the direct relationship between the extracellular virion transport rate and the spread of infection in a tissue. More importantly, it showed that for viruses that rapidly create and exocytose new virions, release of additional assembled virions at the time of cell death does not significantly speed the spread of infection. Moreover, decreasing the cell tolerance to viral load does not drastically change the rate at which the infection spreads, but it does accelerate the rate of tissue damage and loss, which could potentially trigger edema and ARDS earlier. This suggests that working to slow apoptosis may help preserve tissue integrity and delay adverse severe respiratory responses. That such a simple model could already point to actionable hypotheses for experimental and clinical investigations points to the value of rapid model iteration and investigation, rather than waiting for a “perfect” model that incorporates all processes with mechanistic molecular-scale detail.</p><p id="p-278">As work on future versions progresses, teams will work in parallel on submodels to add, parameterize, and test new model components. It will be important to balance the need for new functionality with the requirement for constrained scope, while also balancing the importance of model validation with timely dissemination of results. Thus, this preprint will be updated with every development cycle to invite feedback and community contributions. Between cycles, the most up-to-date information about this model can be found at <a href="http://covid19.physicell.org">http://covid19.physicell.org</a>.</p><div id="sec-51" class="subsection"><h3>Getting involved</h3><p id="p-279">To get involved, we welcome biological expertise, especially related to model assumptions, hypotheses, infection dynamics, and interpretation of results. Mathematical contributions to the underlying model or model analysis as well as data contributions for crafting, parameterizing, and validating model predictions are particularly sought.</p><p id="p-280">We encourage the community to test the web-hosted hosted model at <a href="https://nanohub.org/tools/pc4covid19">https://nanohub.org/tools/pc4covid19</a>. This model will be frequently updated to reflect progress, allowing the public to take advantage of this rapid prototyping effort.</p><p id="p-281">We avidly encourage the community to test the model, offer feedback, and join our growing coalition via Google survey (<a href="https://forms.gle/SVUMYWhipSHfX8nS8">https://forms.gle/SVUMYWhipSHfX8nS8</a>), by direct messaging Paul Macklin on Twitter (<span class="underline">@MathCancer</span>), or by joining the pc4covid19 slack workspace (<span class="underline">invitation link</span>). Updates will frequently be disseminated on social media by Paul Macklin (<span class="underline">@MathCancer</span>), the PhysiCell project (<span class="underline">@PhysiCell</span>), the Society for Mathematical Biology subgroup for Immunobiology and Infection Subgroup (<span class="underline">@smb_imin</span>), and others.</p><p id="p-282">We also encourage developers to watch the pc4covid19 GitHub organization and to contribute bug reports and software patches to the corresponding (sub)model repositories. See <a href="https://github.com/pc4covid19">https://github.com/pc4covid19</a></p><p id="p-283">We are encouraged by the fast recognition of the computational and infectious disease communities that we can make rapid progress against COVID-19 if we pool our expertise and resources. Together, we can make a difference in understanding viral dynamics and suggesting treatment strategies to slow infection, improve immune response, and minimize or prevent adverse immune responses. We note that this work will not only help us address SARS-CoV-2, but will also provide a framework for readiness for future emerging pathogens.</p></div></div><div class="section" id="sec-52"><h2 class="">Appendix 1: Code availability</h2><p id="p-286">All code is being made available as open source under the standard 3-Clause BSD license. Users should cite this preprint (or the final published paper, as the case may be).</p><div class="section" id="sec-53"><h3>Core model releases</h3><div class="section" id="sec-54"><h4>Version 1 model</h4><p id="p-287">Version 0.1.0 (released March 26, 2020)</p><p id="p-288"><strong>GitHub:</strong> <a href="https://github.com/pc4covid19/COVID19/releases/tag/0.1.0">https://github.com/pc4covid19/COVID19/releases/tag/0.1.0</a></p><p id="p-289"><strong>Notes:</strong> First release.</p><p id="p-290">Version 0.1.1 (released March 26, 2020)</p><p id="p-291"><strong>GitHub:</strong> <a href="https://github.com/pc4covid19/COVID19/tree/0.1.1">https://github.com/pc4covid19/COVID19/tree/0.1.1</a></p><p id="p-292"><strong>Notes:</strong> Minor bugfixes and first inclusion of “math” directory.</p><p id="p-293">Version 0.1.2 (released March 26, 2020)</p><p id="p-294"><strong>GitHub:</strong> <a href="https://github.com/pc4covid19/COVID19/releases/tag/0.1.2">https://github.com/pc4covid19/COVID19/releases/tag/0.1.2</a></p><p id="p-295"><strong>Zenodo:</strong> <a href="https://doi.org/10.5281/zenodo.3733336">https://doi.org/10.5281/zenodo.3733336</a></p><p id="p-296"><strong>Notes:</strong> First release with Zenodo integration. Last release in 0.1.x chain (v1 model chain).</p><p id="p-297">Version 0.1.3 (released April 1, 2020)</p><p id="p-298"><strong>GitHub:</strong> <a href="https://github.com/pc4covid19/COVID19/tree/0.1.3">https://github.com/pc4covid19/COVID19/tree/0.1.3</a></p><p id="p-299"><strong>Zenodo:</strong> <a href="https://doi.org/10.5281/zenodo.3737166">https://doi.org/10.5281/zenodo.3737166</a></p><p id="p-300"><strong>Notes:</strong> First release after transferring the COVID19 tissue-level model (overall model) from Paul Macklin’s personal GitHub account to the new pc4covid19 GitHub organization.</p></div></div><div class="section" id="sec-55"><h3>nanoHUB cloud-hosted model releases</h3><p id="p-301">The latest version can always be accessed directly at <a href="https://nanohub.org/tools/pc4covid19">https://nanohub.org/tools/pc4covid19</a></p><div class="section" id="sec-56"><h4>Version 1 model</h4><p id="p-302">Version 1.0 (released March 26, 2020):</p><p id="p-303"><strong>GitHub:</strong> <a href="https://github.com/rheiland/pc4covid19/releases/tag/v1.0">https://github.com/rheiland/pc4covid19/releases/tag/v1.0</a></p><p id="p-304"><strong>Zenodo:</strong> <a href="https://zenodo.org/record/3733276#.XoOGa9NKi9t">https://zenodo.org/record/3733276#.XoOGa9NKi9t</a></p><p id="p-305"><strong>nanoHUB DOI:</strong> <a href="http://dx.doi.org/doi:10.21981/19BB-HM69">http://dx.doi.org/doi:10.21981/19BB-HM69</a></p><p id="p-306"><strong>Notes:</strong> First published version.</p></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-284">PM, RH, and YW thank the National Institutes of Health (U01-CA232137-01), National Science Foundation (1720625), and the Jayne Koskinas Ted Giovanis Foundation for Health and Policy for generous support. AS thanks the NIH for support from the NIAID (R01 AI139088). MC and AJ were supported under NSERC Discovery Grant RGPIN-2018-04546. We thank the NCN CP for fast-tracked deployment of models on nanoHUB.</p><p id="p-285">We thank the scientific community for model feedback, including Simon Parkinson, Richard Allen (Pfizer Inc.), David Dai (Pfizer Inc.), Rohit and the co-authors of this manuscript.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.1" data-doi="10.1016/s1473-3099(20)30120-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gardner</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">An interactive web-based dashboard to track COVID-19 in real time</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/s1473-3099(20)30120-1</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs1473-3099%252820%252930120-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s1473-3099(20)30120-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.2" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.3" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">270</span>–<span class="cit-lpage">273</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.4" data-doi="10.1007/s00134-020-05967-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouadma</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lescure</span>,  <span class="cit-name-given-names">F.-X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lucet</span>,  <span class="cit-name-given-names">J.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yazdanpanah</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Timsit</span>,  <span class="cit-name-given-names">J.-F.</span></span> <span class="cit-article-title">Severe SARS-CoV-2 infections: practical considerations and management strategy for intensivists</span>. <abbr class="cit-jnl-abbrev">Intensive Care Medicine</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">579</span>–<span class="cit-lpage">582</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00134-020-05967-x</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntensive%2BCare%2BMedicine%26rft.volume%253D46%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00134-020-05967-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00134-020-05967-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.5" data-doi="10.1016/s2213-2600(20)30079-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study</span>. <abbr class="cit-jnl-abbrev">The Lancet Respiratory Medicine</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/s2213-2600(20)30079-5</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BRespiratory%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs2213-2600%252820%252930079-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s2213-2600(20)30079-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.6" data-doi="10.1016/s0140-6736(20)30566-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">1054</span>–<span class="cit-lpage">1062</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(20)30566-3</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0140-6736%252820%252930566-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0140-6736(20)30566-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.7" data-doi="10.1111/all.14238"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.-j.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical characteristics of 140 patients infected with SARS-CoV-2 in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Allergy</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/all.14238</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAllergy%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fall.14238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/all.14238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.8" data-doi="10.1016/S1470-2045(20)30096-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cancer patients in SARS-CoV-2 infection: a nationwide analysis in China</span>. <abbr class="cit-jnl-abbrev">The Lancet Oncology</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">335</span>–<span class="cit-lpage">337</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/s1470-2045(20)30096-6</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BOncology%26rft.volume%253D21%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1470-2045%252820%252930096-6%26rft_id%253Dinfo%253Apmid%252F32066541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1470-2045(20)30096-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32066541&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.9" data-doi="10.1016/j.cels.2017.08.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">When Seeing Isn’t Believing: How Math Can Guide Our Interpretation of Measurements and Experiments</span>. <abbr class="cit-jnl-abbrev">Cell Systems</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">92</span>–<span class="cit-lpage">94</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cels.2017.08.005</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cels.2017.08.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cels.2017.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.10" data-doi="10.1093/gigascience/giz127"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Key challenges facing data-driven multicellular systems biology</span>. <abbr class="cit-jnl-abbrev">GigaScience</abbr> <span class="cit-vol">8</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/gigascience/giz127</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiz127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giz127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.11" data-doi="10.1126/science.abb2507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">367</span>, <span class="cit-fpage">1260</span>–<span class="cit-lpage">1263</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb2507</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMi4wMTkwNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.12" data-doi="10.14744/ejmo.2020.12220"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sahin</span>,  <span class="cit-name-given-names">A. R.</span></span> <span class="cit-article-title">2019 Novel Coronavirus (COVID-19) Outbreak: A Review of the Current Literature</span>. <abbr class="cit-jnl-abbrev">Eurasian Journal of Medicine and Oncology</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.14744/ejmo.2020.12220</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEurasian%2BJournal%2Bof%2BMedicine%2Band%2BOncology%26rft_id%253Dinfo%253Adoi%252F10.14744%252Fejmo.2020.12220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14744/ejmo.2020.12220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.13" data-doi="10.4161/rna.8.2.15663"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Berkhout</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">RNA virus replication, transcription and recombination</span>. <abbr class="cit-jnl-abbrev">RNA Biology</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">182</span>–<span class="cit-lpage">183</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4161/rna.8.2.15663</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiology%26rft.volume%253D8%26rft.spage%253D182%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.8.2.15663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.8.2.15663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.14" data-doi="10.1146/annurev.biochem.67.1.227"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stark</span>,  <span class="cit-name-given-names">G. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kerr</span>,  <span class="cit-name-given-names">I. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">B. R. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schreiber</span>,  <span class="cit-name-given-names">R. D.</span></span> <span class="cit-article-title">How Cells Respond to Interferons</span>. <abbr class="cit-jnl-abbrev">Annual Review of Biochemistry</abbr> <span class="cit-vol">67</span>, <span class="cit-fpage">227</span>–<span class="cit-lpage">264</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev.biochem.67.1.227</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DStark%26rft.auinit1%253DG.%2BR.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D227%26rft.epage%253D264%26rft.atitle%253DHOW%2BCELLS%2BRESPOND%2BTO%2BINTERFERONS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.67.1.227%26rft_id%253Dinfo%253Apmid%252F9759489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.67.1.227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9759489&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075721700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.15" data-doi="10.1038/sj.cr.7290309"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perry</span>,  <span class="cit-name-given-names">A. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">The host type I interferon response to viral and bacterial infections</span>. <abbr class="cit-jnl-abbrev">Cell Research</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">407</span>–<span class="cit-lpage">422</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/sj.cr.7290309</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DPerry%26rft.auinit1%253DA.%2BK.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D407%26rft.epage%253D422%26rft.atitle%253DThe%2Bhost%2Btype%2BI%2Binterferon%2Bresponse%2Bto%2Bviral%2Band%2Bbacterial%2Binfections.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.cr.7290309%26rft_id%253Dinfo%253Apmid%252F15987599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.cr.7290309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15987599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230444400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.16" data-doi="10.1146/annurev-virology-110615-042435"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danthi</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Viruses and the Diversity of Cell Death</span>. <abbr class="cit-jnl-abbrev">Annual Review of Virology</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">533</span>–<span class="cit-lpage">553</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-virology-110615-042435</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BVirology%26rft.volume%253D3%26rft.spage%253D533%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-110615-042435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-110615-042435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.17" data-doi="10.1038/s41419-018-0917-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yue</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-Coronavirus Open Reading Frame-3a drives multimodal necrotic cell death</span>. <abbr class="cit-jnl-abbrev">Cell Death &amp; Disease</abbr> <span class="cit-vol">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41419-018-0917-y</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BDeath%2B%2526%2BDisease%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41419-018-0917-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41419-018-0917-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.18" data-doi="10.1080/21505594.2016.1276690"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keck</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Altered mitochondrial dynamics as a consequence of Venezuelan Equine encephalitis virus infection</span>. <abbr class="cit-jnl-abbrev">Virulence</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1849</span>–<span class="cit-lpage">1866</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1080/21505594.2016.1276690</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirulence%26rft.volume%253D8%26rft.spage%253D1849%26rft_id%253Dinfo%253Adoi%252F10.1080%252F21505594.2016.1276690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/21505594.2016.1276690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.19" data-doi="10.3390/v10110606"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keck</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mitochondrial-Directed Antioxidant Reduces Microglial-Induced Inflammation in Murine In Vitro Model of TC-83 Infection</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v10110606</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv10110606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v10110606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.20" data-doi="10.1371/journal.pone.0045842"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thiel</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Lack of Innate Interferon Responses during SARS Coronavirus Infection in a Vaccination and Reinfection Ferret Model</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e45842</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0045842</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCameron%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45842%26rft.epage%253De45842%26rft.atitle%253DLack%2Bof%2Binnate%2Binterferon%2Bresponses%2Bduring%2BSARS%2Bcoronavirus%2Binfection%2Bin%2Ba%2Bvaccination%2Band%2Breinfection%2Bferret%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045842%26rft_id%253Dinfo%253Apmid%252F23029269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029269&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.21" data-doi="10.1016/j.onehlt.2016.12.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Widagdo</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okba</span>,  <span class="cit-name-given-names">N. M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stalin Raj</span>,  <span class="cit-name-given-names">V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>,  <span class="cit-name-given-names">B. L.</span></span> <span class="cit-article-title">MERS-coronavirus: From discovery to intervention</span>. <abbr class="cit-jnl-abbrev">One Health</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">11</span>–<span class="cit-lpage">16</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.onehlt.2016.12.001</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOne%2BHealth%26rft.volume%253D3%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.onehlt.2016.12.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.onehlt.2016.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.22" data-doi="10.1016/j.chom.2016.01.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Channappanavar</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dysregulated Type I Interferon and Inflammatory Monocyte-Macrophage Responses Cause Lethal Pneumonia in SARS-CoV-Infected Mice</span>. <abbr class="cit-jnl-abbrev">Cell Host &amp; Microbe</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">181</span>–<span class="cit-lpage">193</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2016.01.007</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft.volume%253D19%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.01.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.23" data-doi="10.1172/jci126363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Channappanavar</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">IFN-I response timing relative to virus replication determines MERS coronavirus infection outcomes</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Investigation</abbr> <span class="cit-vol">129</span>, <span class="cit-fpage">3625</span>–<span class="cit-lpage">3639</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1172/jci126363</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.volume%253D129%26rft.spage%253D3625%26rft_id%253Dinfo%253Adoi%252F10.1172%252Fjci126363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/jci126363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.24" data-doi="10.1016/j.sjbs.2016.02.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Al-Hazmi</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Challenges presented by MERS corona virus, and SARS corona virus to global health</span>. <abbr class="cit-jnl-abbrev">Saudi Journal of Biological Sciences</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">507</span>–<span class="cit-lpage">511</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.sjbs.2016.02.019</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSaudi%2BJournal%2Bof%2BBiological%2BSciences%26rft.volume%253D23%26rft.spage%253D507%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sjbs.2016.02.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sjbs.2016.02.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.25" data-doi="10.2139/ssrn.3541136"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dysregulation of Immune Response in Patients with COVID-19 in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">SSRN Electronic Journal</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.2139/ssrn.3541136</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSSRN%2BElectronic%2BJournal%26rft_id%253Dinfo%253Adoi%252F10.2139%252Fssrn.3541136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2139/ssrn.3541136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.26" data-doi="10.12932/ap-200220-0772"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prompetchara</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ketloy</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Palaga</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic</span>. <abbr class="cit-jnl-abbrev">Asian Pacific Journal of Allergy and Immunology</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.12932/ap-200220-0772</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsian%2BPacific%2BJournal%2Bof%2BAllergy%2Band%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.12932%252Fap-200220-0772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12932/ap-200220-0772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.27" data-doi="10.1007/s00281-017-0629-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Channappanavar</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology</span>. <abbr class="cit-jnl-abbrev">Seminars in Immunopathology</abbr> <span class="cit-vol">39</span>, <span class="cit-fpage">529</span>–<span class="cit-lpage">539</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00281-017-0629-x</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2BImmunopathology%26rft.volume%253D39%26rft.spage%253D529%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00281-017-0629-x%26rft_id%253Dinfo%253Apmid%252F28466096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00281-017-0629-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28466096&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.28" data-doi="10.1161/CIRCULATIONAHA.109.916346"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nahrendorf</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pittet</span>,  <span class="cit-name-given-names">M. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Swirski</span>,  <span class="cit-name-given-names">F. K.</span></span> <span class="cit-article-title">Monocytes: Protagonists of Infarct Inflammation and Repair After Myocardial Infarction</span>. <abbr class="cit-jnl-abbrev">Circulation</abbr> <span class="cit-vol">121</span>, <span class="cit-fpage">2437</span>–<span class="cit-lpage">2445</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1161/circulationaha.109.916346</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirculation%26rft.stitle%253DCirculation%26rft.aulast%253DNahrendorf%26rft.auinit1%253DM.%26rft.volume%253D121%26rft.issue%253D22%26rft.spage%253D2437%26rft.epage%253D2445%26rft.atitle%253DMonocytes%253A%2BProtagonists%2Bof%2BInfarct%2BInflammation%2Band%2BRepair%2BAfter%2BMyocardial%2BInfarction%26rft_id%253Dinfo%253Adoi%252F10.1161%252FCIRCULATIONAHA.109.916346%26rft_id%253Dinfo%253Apmid%252F20530020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNpcmN1bGF0aW9uYWhhIjtzOjU6InJlc2lkIjtzOjExOiIxMjEvMjIvMjQzNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDIuMDE5MDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.29" data-doi="10.3389/fimmu.2017.00550"><div class="cit-metadata"><cite>1 29 <span class="cit-auth"><span class="cit-name-surname">Camp</span>,  <span class="cit-name-given-names">J. V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jonsson</span>,  <span class="cit-name-given-names">C. B.</span></span> <span class="cit-article-title">A Role for Neutrophils in Viral Respiratory Disease</span>. <abbr class="cit-jnl-abbrev">Frontiers in Immunology</abbr> <span class="cit-vol">8</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3389/fimmu.2017.00550</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BImmunology%26rft.volume%253D8%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffimmu.2017.00550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fimmu.2017.00550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.30" data-doi="10.1038/mi.2015.126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weitnauer</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mijošek</span>,  <span class="cit-name-given-names">V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dalpke</span>,  <span class="cit-name-given-names">A. H.</span></span> <span class="cit-article-title">Control of local immunity by airway epithelial cells</span>. <abbr class="cit-jnl-abbrev">Mucosal Immunology</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">287</span>–<span class="cit-lpage">298</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/mi.2015.126</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMucosal%2BImmunology%26rft.volume%253D9%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmi.2015.126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/mi.2015.126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.31" data-doi="10.3389/fimmu.2018.01421"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crane</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">K. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">FitzGerald</span>,  <span class="cit-name-given-names">E. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jamieson</span>,  <span class="cit-name-given-names">A. M.</span></span> <span class="cit-article-title">Surviving Deadly Lung Infections: Innate Host Tolerance Mechanisms in the Pulmonary System</span>. <abbr class="cit-jnl-abbrev">Frontiers in Immunology</abbr> <span class="cit-vol">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3389/fimmu.2018.01421</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffimmu.2018.01421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fimmu.2018.01421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.32" data-doi="10.1016/j.cell.2011.08.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teijaro John</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Endothelial Cells Are Central Orchestrators of Cytokine Amplification during Influenza Virus Infection</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">146</span>, <span class="cit-fpage">980</span>–<span class="cit-lpage">991</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2011.08.015</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTeijaro%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D146%26rft.issue%253D6%26rft.spage%253D980%26rft.epage%253D991%26rft.atitle%253DEndothelial%2Bcells%2Bare%2Bcentral%2Borchestrators%2Bof%2Bcytokine%2Bamplification%2Bduring%2Binfluenza%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.08.015%26rft_id%253Dinfo%253Apmid%252F21925319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.08.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21925319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295258100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.33" data-doi="10.1002/jcp.27155"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shokri</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mahmoudvand</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taherkhani</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Farshadpour</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Modulation of the immune response by Middle East respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">Journal of Cellular Physiology</abbr> <span class="cit-vol">234</span>, <span class="cit-fpage">2143</span>–<span class="cit-lpage">2151</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jcp.27155</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCellular%2BPhysiology%26rft.volume%253D234%26rft.spage%253D2143%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcp.27155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcp.27155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.34" data-doi="10.1128/JVI.79.22.13915-13923.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">W. p.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Modulation of the Immune Response to the Severe Acute Respiratory Syndrome Spike Glycoprotein by Gene-Based and Inactivated Virus Immunization</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">79</span>, <span class="cit-fpage">13915</span>–<span class="cit-lpage">13923</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.79.22.13915-13923.2005</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKong%26rft.auinit1%253DW.-p.%26rft.volume%253D79%26rft.issue%253D22%26rft.spage%253D13915%26rft.epage%253D13923%26rft.atitle%253DModulation%2Bof%2Bthe%2BImmune%2BResponse%2Bto%2Bthe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BSpike%2BGlycoprotein%2Bby%2BGene-Based%2Band%2BInactivated%2BVirus%2BImmunization%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.22.13915-13923.2005%26rft_id%253Dinfo%253Apmid%252F16254327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OS8yMi8xMzkxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDIuMDE5MDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.35" data-doi="10.1128/JVI.01032-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cruz</span>,  <span class="cit-name-given-names">J. L. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Alphacoronavirus Protein 7 Modulates Host Innate Immune Response</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">9754</span>–<span class="cit-lpage">9767</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01032-13</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01032-13%26rft_id%253Dinfo%253Apmid%252F23824792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xNy85NzU0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMi4wMTkwNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.36" data-doi="10.1172/jci.insight.123158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Anti–spike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection</span>. <abbr class="cit-jnl-abbrev">JCI Insight</abbr> <span class="cit-vol">4</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1172/jci.insight.123158</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJCI%2BInsight%26rft.volume%253D4%26rft.spage%253D123158e%26rft_id%253Dinfo%253Adoi%252F10.1172%252Fjci.insight.123158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/jci.insight.123158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.37" data-doi="10.1084/jem.20050828"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leong</span>,  <span class="cit-name-given-names">A. S. Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiple organ infection and the pathogenesis of SARS</span>. <abbr class="cit-jnl-abbrev">Journal of Experimental Medicine</abbr> <span class="cit-vol">202</span>, <span class="cit-fpage">415</span>–<span class="cit-lpage">424</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1084/jem.20050828</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.20050828%26rft_id%253Dinfo%253Apmid%252F16043521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamVtIjtzOjU6InJlc2lkIjtzOjk6IjIwMi8zLzQxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDIuMDE5MDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.38" data-doi="10.1038/srep07027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Angiotensin-converting enzyme 2 (ACE2) mediates influenza H7N9 virus-induced acute lung injury</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr> <span class="cit-vol">4</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/srep07027</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D4%26rft.spage%253D7027%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep07027%26rft_id%253Dinfo%253Apmid%252F25391767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep07027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25391767&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.39" data-doi="10.1038/nm1267"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuba</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus–induced lung injury</span>. <abbr class="cit-jnl-abbrev">Nature Medicine</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">875</span>–<span class="cit-lpage">879</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nm1267</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DKuba%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D875%26rft.epage%253D879%26rft.atitle%253DA%2Bcrucial%2Brole%2Bof%2Bangiotensin%2Bconverting%2Benzyme%2B2%2B%2528ACE2%2529%2Bin%2BSARS%2Bcoronavirus-induced%2Blung%2Binjury.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1267%26rft_id%253Dinfo%253Apmid%252F16007097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16007097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230964700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.40" data-doi="10.1016/s1473-3099(20)30132-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stebbing</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">COVID-19: combining antiviral and anti-inflammatory treatments</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">400</span>–<span class="cit-lpage">402</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/s1473-3099(20)30132-8</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D400%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs1473-3099%252820%252930132-8%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s1473-3099(20)30132-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.41" data-doi="10.1152/ajplung.00343.2012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanders</span>,  <span class="cit-name-given-names">C. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Compromised respiratory function in lethal influenza infection is characterized by the depletion of type I alveolar epithelial cells beyond threshold levels</span>. <abbr class="cit-jnl-abbrev">American Journal of Physiology- Lung Cellular and Molecular Physiology</abbr> <span class="cit-vol">304</span>, <span class="cit-fpage">L481</span>–<span class="cit-lpage">L488</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1152/ajplung.00343.2012</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPhysiology-%2BLung%2BCellular%2Band%2BMolecular%2BPhysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fajplung.00343.2012%26rft_id%253Dinfo%253Apmid%252F23355384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/ajplung.00343.2012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23355384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317000500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.42" data-doi="10.1002/psp4.12063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rieger</span>,  <span class="cit-name-given-names">T. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Musante</span>,  <span class="cit-name-given-names">C. J.</span></span> <span class="cit-article-title">Efficient Generation and Selection of Virtual Populations in Quantitative Systems Pharmacology Models</span>. <abbr class="cit-jnl-abbrev">CPT: Pharmacometrics &amp; Systems Pharmacology</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">140</span>–<span class="cit-lpage">146</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/psp4.12063</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCPT%253A%2BPharmacometrics%2B%2526%2BSystems%2BPharmacology%26rft.volume%253D5%26rft.spage%253D140%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpsp4.12063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/psp4.12063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.43" data-doi="10.1371/journal.pcbi.1007495"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cassidy</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Craig</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Determinants of combination GM-CSF immunotherapy and oncolytic virotherapy success identified through in silico treatment personalization</span>. <abbr class="cit-jnl-abbrev">PLOS Computational Biology</abbr> <span class="cit-vol">15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1007495</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D15%26rft.spage%253De1007495%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1007495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1007495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.44" data-doi="10.1126/science.1683006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Antigenic diversity thresholds and the development of AIDS</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">254</span>, <span class="cit-fpage">963</span>–<span class="cit-lpage">969</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1683006</span> (<span class="cit-pub-date">1991</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNowak%26rft.auinit1%253DM.%26rft.volume%253D254%26rft.issue%253D5034%26rft.spage%253D963%26rft.epage%253D969%26rft.atitle%253DAntigenic%2Bdiversity%2Bthresholds%2Band%2Bthe%2Bdevelopment%2Bof%2BAIDS%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1683006%26rft_id%253Dinfo%253Apmid%252F1683006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTQvNTAzNC85NjMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNS8yMDIwLjA0LjAyLjAxOTA3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.45" data-doi="10.1038/373117a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Viral dynamics in human immunodeficiency virus type 1 infection</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">373</span>, <span class="cit-fpage">117</span>–<span class="cit-lpage">122</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/373117a0</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWei%26rft.auinit1%253DX.%26rft.volume%253D373%26rft.issue%253D6510%26rft.spage%253D117%26rft.epage%253D122%26rft.atitle%253DViral%2Bdynamics%2Bin%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F373117a0%26rft_id%253Dinfo%253Apmid%252F7529365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/373117a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7529365&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QB06300048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.46" data-doi="10.1038/nature24634"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>,  <span class="cit-name-given-names">D. I. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laskey</span>,  <span class="cit-name-given-names">S. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Siliciano</span>,  <span class="cit-name-given-names">R. F.</span></span> <span class="cit-article-title">Re-evaluating evolution in the HIV reservoir</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">551</span>, <span class="cit-fpage">E6</span>–<span class="cit-lpage">E9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature24634</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253DE6%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.47" data-doi="10.1038/s41467-018-06843-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reeves</span>,  <span class="cit-name-given-names">D. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A majority of HIV persistence during antiretroviral therapy is due to infected cell proliferation</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-018-06843-5</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D4811%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-06843-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-06843-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.48" data-doi="10.1093/ve/vez018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koelle</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farrell</span>,  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brooke</span>,  <span class="cit-name-given-names">C. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ke</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Within-host infectious disease models accommodating cellular coinfection, with an application to influenza†</span>. <abbr class="cit-jnl-abbrev">Virus Evolution</abbr> <span class="cit-vol">5</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/ve/vez018</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvez018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vez018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.49" data-doi="10.1126/science.271.5255.1582"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perelson</span>,  <span class="cit-name-given-names">A. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neumann</span>,  <span class="cit-name-given-names">A. U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markowitz</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leonard</span>,  <span class="cit-name-given-names">J. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">D. D.</span></span> <span class="cit-article-title">HIV-1 Dynamics in Vivo: Virion Clearance Rate, Infected Cell Life-Span, and Viral Generation Time</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">271</span>, <span class="cit-fpage">1582</span>–<span class="cit-lpage">1586</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.271.5255.1582</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPerelson%26rft.auinit1%253DA.%2BS.%26rft.volume%253D271%26rft.issue%253D5255%26rft.spage%253D1582%26rft.epage%253D1586%26rft.atitle%253DHIV-1%2BDynamics%2Bin%2BVivo%253A%2BVirion%2BClearance%2BRate%252C%2BInfected%2BCell%2BLife-Span%252C%2Band%2BViral%2BGeneration%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.271.5255.1582%26rft_id%253Dinfo%253Apmid%252F8599114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzEvNTI1NS8xNTgyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMi4wMTkwNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.50" data-doi="10.1615/CritRevImmunol.v30.i3.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>,  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-article-title">Modeling the Viral Dynamics of Influenza A Virus Infection</span>. <abbr class="cit-jnl-abbrev">Critical Reviews(tm) in Immunology</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">291</span>–<span class="cit-lpage">298</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1615/CritRevImmunol.v30.i3.60</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2Breviews%2Bin%2Bimmunology%26rft.stitle%253DCrit%2BRev%2BImmunol%26rft.aulast%253DSmith%26rft.auinit1%253DA.%2BM.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D291%26rft.epage%253D298%26rft.atitle%253DModeling%2Bthe%2Bviral%2Bdynamics%2Bof%2Binfluenza%2BA%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1615%252FCritRevImmunol.v30.i3.60%26rft_id%253Dinfo%253Apmid%252F20370636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1615/CritRevImmunol.v30.i3.60&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20370636&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.51" data-doi="10.1002/wsbm.129"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perelson</span>,  <span class="cit-name-given-names">A. S.</span></span> <span class="cit-article-title">Influenza A virus infection kinetics: quantitative data and models</span>. <abbr class="cit-jnl-abbrev">Wiley Interdisciplinary Reviews: Systems Biology and Medicine</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">429</span>–<span class="cit-lpage">445</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/wsbm.129</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DWiley%2BInterdiscip%2BRev%2BSyst%2BBiol%2BMed%26rft.aulast%253DSmith%26rft.auinit1%253DA.%2BM.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D429%26rft.epage%253D445%26rft.atitle%253DInfluenza%2BA%2Bvirus%2Binfection%2Bkinetics%253A%2Bquantitative%2Bdata%2Band%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwsbm.129%26rft_id%253Dinfo%253Apmid%252F21197654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wsbm.129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21197654&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52 in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.52" data-doi="10.1016/j.jtbi.2011.01.052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCullers</span>,  <span class="cit-name-given-names">J. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Adler</span>,  <span class="cit-name-given-names">F. R.</span></span> <span class="cit-article-title">Mathematical model of a three-stage innate immune response to a pneumococcal lung infection</span>. <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr> <span class="cit-vol">276</span>, <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jtbi.2011.01.052</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DSmith%26rft.auinit1%253DA.%2BM.%26rft.volume%253D276%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D116%26rft.atitle%253DMathematical%2Bmodel%2Bof%2Ba%2Bthree-stage%2Binnate%2Bimmune%2Bresponse%2Bto%2Ba%2Bpneumococcal%2Blung%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2011.01.052%26rft_id%253Dinfo%253Apmid%252F21300073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2011.01.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21300073&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53 in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.53" data-doi="10.1073/pnas.94.13.6971"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonhoeffer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">May</span>,  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>,  <span class="cit-name-given-names">G. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-article-title">Virus dynamics and drug therapy</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">94</span>, <span class="cit-fpage">6971</span>–<span class="cit-lpage">6976</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.94.13.6971</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBonhoeffer%26rft.auinit1%253DS.%26rft.volume%253D94%26rft.issue%253D13%26rft.spage%253D6971%26rft.epage%253D6976%26rft.atitle%253DVirus%2Bdynamics%2Band%2Bdrug%2Btherapy%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.13.6971%26rft_id%253Dinfo%253Apmid%252F9192676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTMvNjk3MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDIuMDE5MDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.54" data-doi="10.1073/pnas.1406663111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>,  <span class="cit-name-given-names">D. I. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">M. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Siliciano</span>,  <span class="cit-name-given-names">R. F.</span></span> <span class="cit-article-title">Predicting the outcomes of treatment to eradicate the latent reservoir for HIV-1</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">13475</span>–<span class="cit-lpage">13480</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1406663111</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1406663111%26rft_id%253Dinfo%253Apmid%252F25097264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM3LzEzNDc1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMi4wMTkwNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.55" data-doi="10.1038/nm.2892"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>,  <span class="cit-name-given-names">D. I. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabi</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siliciano</span>,  <span class="cit-name-given-names">R. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-article-title">Antiretroviral dynamics determines HIV evolution and predicts therapy outcome</span>. <abbr class="cit-jnl-abbrev">Nature Medicine</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">1378</span>–<span class="cit-lpage">1385</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nm.2892</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DRosenbloom%26rft.auinit1%253DD.%2BI.%26rft.volume%253D18%26rft.issue%253D9%26rft.spage%253D1378%26rft.epage%253D1385%26rft.atitle%253DAntiretroviral%2Bdynamics%2Bdetermines%2BHIV%2Bevolution%2Band%2Bpredicts%2Btherapy%2Boutcome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.2892%26rft_id%253Dinfo%253Apmid%252F22941277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.2892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22941277&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.56" data-doi="10.1371/journal.pone.0155124"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mueller</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mathematical Modeling Predicts that Increased HSV-2 Shedding in HIV-1 Infected Persons Is Due to Poor Immunologic Control in Ganglia and Genital Mucosa</span>. <abbr class="cit-jnl-abbrev">Plos One</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e0155124</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0155124</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlos%2BOne%26rft.volume%253D11%26rft.spage%253De0155124%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0155124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0155124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57 in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.57" data-doi="10.1038/387188a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perelson</span>,  <span class="cit-name-given-names">A. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Decay characteristics of HIV-1-infected compartments during combination therapy</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>, <span class="cit-fpage">188</span>–<span class="cit-lpage">191</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/387188a0</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPerelson%26rft.auinit1%253DA.%2BS.%26rft.volume%253D387%26rft.issue%253D6629%26rft.spage%253D188%26rft.epage%253D191%26rft.atitle%253DDecay%2Bcharacteristics%2Bof%2BHIV-1-infected%2Bcompartments%2Bduring%2Bcombination%2Btherapy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F387188a0%26rft_id%253Dinfo%253Apmid%252F9144290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/387188a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9144290&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WX94500056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58 in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.58" data-doi="10.1038/s41467-017-02294-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirtane</span>,  <span class="cit-name-given-names">A. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Development of an oral once-weekly drug delivery system for HIV antiretroviral therapy</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-017-02294-6</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-02294-6%26rft_id%253Dinfo%253Apmid%252F29317618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-02294-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29317618&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.59" data-doi="10.1016/j.vaccine.2017.09.044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schiffer</span>,  <span class="cit-name-given-names">J. T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gottlieb</span>,  <span class="cit-name-given-names">S. L.</span></span> <span class="cit-article-title">Biologic interactions between HSV-2 and HIV-1 and possible implications for HSV vaccine development</span>. <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">7363</span>–<span class="cit-lpage">7371</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vaccine.2017.09.044</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D37%26rft.spage%253D7363%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2017.09.044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2017.09.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60 in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.60" data-doi="10.1126/scitranslmed.aad6654"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schiffer</span>,  <span class="cit-name-given-names">J. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mathematical modeling of herpes simplex virus-2 suppression with pritelivir predicts trial outcomes</span>. <abbr class="cit-jnl-abbrev">Science Translational Medicine</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">324ra315</span>–<span class="cit-lpage">324ra315</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/scitranslmed.aad6654</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.volume%253D8%26rft.spage%253D324ra315%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.aad6654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/scitranslmed.aad6654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61 in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.61" data-doi="10.1038/nri700"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perelson</span>,  <span class="cit-name-given-names">A. S.</span></span> <span class="cit-article-title">Modelling viral and immune system dynamics</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Immunology</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">28</span>–<span class="cit-lpage">36</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nri700</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DPerelson%26rft.auinit1%253DA.%2BS.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D36%26rft.atitle%253DModelling%2Bviral%2Band%2Bimmune%2Bsystem%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri700%26rft_id%253Dinfo%253Apmid%252F11905835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11905835&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180398300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62 in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.62" data-doi="10.1111/imr.12698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>,  <span class="cit-name-given-names">D. I. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">M. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Siliciano</span>,  <span class="cit-name-given-names">R. F.</span></span> <span class="cit-article-title">Insight into treatment of HIV infection from viral dynamics models</span>. <abbr class="cit-jnl-abbrev">Immunological Reviews</abbr> <span class="cit-vol">285</span>, <span class="cit-fpage">9</span>–<span class="cit-lpage">25</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/imr.12698</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunological%2BReviews%26rft.volume%253D285%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fimr.12698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/imr.12698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63 in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.63" data-doi="10.1007/s00285-012-0580-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brauer</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Heffernan</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Viral dynamics model with CTL immune response incorporating antiretroviral therapy</span>. <abbr class="cit-jnl-abbrev">Journal of Mathematical Biology</abbr> <span class="cit-vol">67</span>, <span class="cit-fpage">901</span>–<span class="cit-lpage">934</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00285-012-0580-3</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMathematical%2BBiology%26rft.volume%253D67%26rft.spage%253D901%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00285-012-0580-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00285-012-0580-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64 in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.64" data-doi="10.1038/nm.3103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schiffer</span>,  <span class="cit-name-given-names">J. T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Corey</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Rapid host immune response and viral dynamics in herpes simplex virus-2 infection</span>. <abbr class="cit-jnl-abbrev">Nature Medicine</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">280</span>–<span class="cit-lpage">288</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nm.3103</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMedicine%26rft.volume%253D19%26rft.spage%253D280%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3103%26rft_id%253Dinfo%253Apmid%252F23467247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.3103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23467247&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65 in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.65" data-doi="10.1007/s11538-018-0424-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenner</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yun</span>,  <span class="cit-name-given-names">C.-O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">P. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Coster</span>,  <span class="cit-name-given-names">A. C. F.</span></span> <span class="cit-article-title">Mathematical Modelling of the Interaction Between Cancer Cells and an Oncolytic Virus: Insights into the Effects of Treatment Protocols</span>. <abbr class="cit-jnl-abbrev">Bulletin of Mathematical Biology</abbr> <span class="cit-vol">80</span>, <span class="cit-fpage">1615</span>–<span class="cit-lpage">1629</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s11538-018-0424-4</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2BMathematical%2BBiology%26rft.volume%253D80%26rft.spage%253D1615%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11538-018-0424-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11538-018-0424-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.66" data-doi="10.1002/bit.20363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Möhler</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flockerzi</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sann</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reichl</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Mathematical model of influenza A virus production in large-scale microcarrier culture</span>. <abbr class="cit-jnl-abbrev">Biotechnology and Bioengineering</abbr> <span class="cit-vol">90</span>, <span class="cit-fpage">46</span>–<span class="cit-lpage">58</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/bit.20363</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DM%25C3%25B6hler%26rft.auinit1%253DL.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D46%26rft.epage%253D58%26rft.atitle%253DMathematical%2Bmodel%2Bof%2Binfluenza%2BA%2Bvirus%2Bproduction%2Bin%2Blarge-scale%2Bmicrocarrier%2Bculture.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbit.20363%26rft_id%253Dinfo%253Apmid%252F15736163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bit.20363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15736163&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227843800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.67" data-doi="10.1016/j.vaccine.2009.02.027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schulze-Horsel</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schulze</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agalaridis</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Genzel</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reichl</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Infection dynamics and virusinduced apoptosis in cell culture-based influenza vaccine production—Flow cytometry and mathematical modeling</span>. <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2712</span>–<span class="cit-lpage">2722</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vaccine.2009.02.027</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.stitle%253DVaccine%26rft.aulast%253DSchulze-Horsel%26rft.auinit1%253DJ.%26rft.volume%253D27%26rft.issue%253D20%26rft.spage%253D2712%26rft.epage%253D2722%26rft.atitle%253DInfection%2Bdynamics%2Band%2Bvirus-induced%2Bapoptosis%2Bin%2Bcell%2Bculture-based%2Binfluenza%2Bvaccine%2Bproduction-Flow%2Bcytometry%2Band%2Bmathematical%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2009.02.027%26rft_id%253Dinfo%253Apmid%252F19428884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2009.02.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19428884&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.68" data-doi="10.1016/s0025-5564(02)00218-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banks</span>,  <span class="cit-name-given-names">H. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bortz</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holte</span>,  <span class="cit-name-given-names">S. E.</span></span> <span class="cit-article-title">Incorporation of variability into the modeling of viral delays in HIV infection dynamics</span>. <abbr class="cit-jnl-abbrev">Mathematical Biosciences</abbr> <span class="cit-vol">183</span>, <span class="cit-fpage">63</span>–<span class="cit-lpage">91</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0025-5564(02)00218-3</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DBanks%26rft.auinit1%253DH.%2BT.%26rft.volume%253D183%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D91%26rft.atitle%253DIncorporation%2Bof%2Bvariability%2Binto%2Bthe%2Bmodeling%2Bof%2Bviral%2Bdelays%2Bin%2BHIV%2Binfection%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0025-5564%252802%252900218-3%26rft_id%253Dinfo%253Apmid%252F12604136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0025-5564(02)00218-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12604136&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.69" data-doi="10.1007/s00285-002-0191-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Culshaw</span>,  <span class="cit-name-given-names">R. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruan</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">A mathematical model of cell-to-cell spread of HIV-1 that includes a time delay</span>. <abbr class="cit-jnl-abbrev">Journal of Mathematical Biology</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">425</span>–<span class="cit-lpage">444</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00285-002-0191-5</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DCulshaw%26rft.auinit1%253DR.%2BV.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D425%26rft.epage%253D444%26rft.atitle%253DA%2Bmathematical%2Bmodel%2Bof%2Bcell-to-cell%2Bspread%2Bof%2BHIV-1%2Bthat%2Bincludes%2Ba%2Btime%2Bdelay.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00285-002-0191-5%26rft_id%253Dinfo%253Apmid%252F12750834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00285-002-0191-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12750834&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183542700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70 in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.70" data-doi="10.1137/090779322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M. Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shu</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Impact of Intracellular Delays and Target-Cell Dynamics on In Vivo Viral Infections</span>. <abbr class="cit-jnl-abbrev">SIAM Journal on Applied Mathematics</abbr> <span class="cit-vol">70</span>, <span class="cit-fpage">2434</span>–<span class="cit-lpage">2448</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1137/090779322</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSIAM%2BJournal%2Bon%2BApplied%2BMathematics%26rft.volume%253D70%26rft.spage%253D2434%26rft_id%253Dinfo%253Adoi%252F10.1137%252F090779322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1137/090779322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71 in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.71" data-doi="10.1016/j.coisb.2018.10.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. M.</span></span> <span class="cit-article-title">Validated models of immune response to virus infection</span>. <abbr class="cit-jnl-abbrev">Current Opinion in Systems Biology</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">46</span>–<span class="cit-lpage">52</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.coisb.2018.10.005</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BSystems%2BBiology%26rft.volume%253D12%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coisb.2018.10.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coisb.2018.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72 in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.72" data-doi="10.1111/imr.12692"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. M.</span></span> <span class="cit-article-title">Host-pathogen kinetics during influenza infection and coinfection: insights from predictive modeling</span>. <abbr class="cit-jnl-abbrev">Immunological Reviews</abbr> <span class="cit-vol">285</span>, <span class="cit-fpage">97</span>–<span class="cit-lpage">112</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/imr.12692</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunological%2BReviews%26rft.volume%253D285%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fimr.12692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/imr.12692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73 in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.73" data-doi="10.1073/pnas.1203110110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guedj</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Modeling shows that the NS5A inhibitor daclatasvir has two modes of action and yields a shorter estimate of the hepatitis C virus half-life</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">110</span>, <span class="cit-fpage">3991</span>–<span class="cit-lpage">3996</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1203110110</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1203110110%26rft_id%253Dinfo%253Apmid%252F23431163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzEwLzM5OTEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNS8yMDIwLjA0LjAyLjAxOTA3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74 in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.74" data-doi="10.1128/JVI.00080-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heldt</span>,  <span class="cit-name-given-names">F. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frensing</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reichl</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Modeling the Intracellular Dynamics of Influenza Virus Replication To Understand the Control of Viral RNA Synthesis</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">7806</span>–<span class="cit-lpage">7817</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00080-12</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00080-12%26rft_id%253Dinfo%253Apmid%252F22593159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xNS83ODA2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMi4wMTkwNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75 in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.75" data-doi="10.1002/bit.20096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sidorenko</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reichl</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Structured model of influenza virus replication in MDCK cells</span>. <abbr class="cit-jnl-abbrev">Biotechnology and Bioengineering</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/bit.20096</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DSidorenko%26rft.auinit1%253DY.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D14%26rft.atitle%253DStructured%2Bmodel%2Bof%2Binfluenza%2Bvirus%2Breplication%2Bin%2BMDCK%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbit.20096%26rft_id%253Dinfo%253Apmid%252F15384040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bit.20096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15384040&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224136300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76 in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.76" data-doi="10.1145/1529282.1529744"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fachada</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopes</span>,  <span class="cit-name-given-names">V. V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rosa</span>,  <span class="cit-name-given-names">A.</span></span> <abbr class="cit-jnl-abbrev">Simulating antigenic drift and shift in influenza A. 2093</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1145/1529282.1529744</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSimulating%2Bantigenic%2Bdrift%2Band%2Bshift%2Bin%2Binfluenza%2BA.%2B2093%26rft_id%253Dinfo%253Adoi%252F10.1145%252F1529282.1529744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/1529282.1529744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77 in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.77" data-doi="10.1016/j.ins.2008.11.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bauer</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beauchemin</span>,  <span class="cit-name-given-names">C. A. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perelson</span>,  <span class="cit-name-given-names">A. S.</span></span> <span class="cit-article-title">Agent-based modeling of host–pathogen systems: The successes and challenges</span>. <abbr class="cit-jnl-abbrev">Information Sciences</abbr> <span class="cit-vol">179</span>, <span class="cit-fpage">1379</span>–<span class="cit-lpage">1389</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ins.2008.11.012</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInformation%2BSciences%2B%2528New%2BYork%2529%26rft.stitle%253DInformation%2BSciences%2B%2528New%2BYork%2529%26rft.aulast%253DBauer%26rft.auinit1%253DA.%2BL.%26rft.volume%253D179%26rft.issue%253D10%26rft.spage%253D1379%26rft.epage%253D1389%26rft.atitle%253DAgent-based%2Bmodeling%2Bof%2Bhost-pathogen%2Bsystems%253A%2BThe%2Bsuccesses%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ins.2008.11.012%26rft_id%253Dinfo%253Apmid%252F20161146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ins.2008.11.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20161146&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78 in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.78" data-doi="10.1371/journal.pcbi.1002547"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilke</span>,  <span class="cit-name-given-names">C. O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Complex Spatial Dynamics of Oncolytic Viruses In Vitro: Mathematical and Experimental Approaches</span>. <abbr class="cit-jnl-abbrev">PLoS Computational Biology</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e1002547</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1002547</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DWodarz%26rft.auinit1%253DD.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002547%26rft.epage%253De1002547%26rft.atitle%253DComplex%2Bspatial%2Bdynamics%2Bof%2Boncolytic%2Bviruses%2Bin%2Bvitro%253A%2Bmathematical%2Band%2Bexperimental%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002547%26rft_id%253Dinfo%253Apmid%252F22719239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22719239&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.79" data-doi="10.1371/journal.pone.0168576"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">G.-Q.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Spatiotemporal Dynamics of Virus Infection Spreading in Tissues</span>. <abbr class="cit-jnl-abbrev">Plos One</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e0168576</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0168576</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlos%2BOne%26rft.volume%253D11%26rft.spage%253De0168576%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0168576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0168576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.80" data-doi="10.1016/j.jtbi.2019.110052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenner</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frascoli</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coster</span>,  <span class="cit-name-given-names">A. C. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">P. S.</span></span> <span class="cit-article-title">Enhancing oncolytic virotherapy: Observations from a Voronoi Cell-Based model</span>. <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr> <span class="cit-vol">485</span>, <span class="cit-fpage">110052</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jtbi.2019.110052</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.volume%253D485%26rft.spage%253D110052%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2019.110052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2019.110052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81 in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.81" data-doi="10.1007/978-981-10-4086-3_128"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quintela</span>,  <span class="cit-name-given-names">B. M.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">An Age-based Multiscale Mathematical Model of the Hepatitis C Virus Life-cycle During Infection and Therapy: Including Translation and Replication</abbr>. <span class="cit-vol">60</span>, <span class="cit-fpage">508</span>–<span class="cit-lpage">511</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-981-10-4086-3_128</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAn%2BAge-based%2BMultiscale%2BMathematical%2BModel%2Bof%2Bthe%2BHepatitis%2BC%2BVirus%2BLife-cycle%2BDuring%2BInfection%2Band%2BTherapy%253A%2BIncluding%2BTranslation%2Band%2BReplication%26rft.volume%253D60%26rft.spage%253D508%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-981-10-4086-3_128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-981-10-4086-3_128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82 in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.82" data-doi="10.1371/journal.pcbi.1005991"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghaffarizadeh</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiland</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">S. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mumenthaler</span>,  <span class="cit-name-given-names">S. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">PhysiCell: An open source physics-based cell simulator for 3-D multicellular systems</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">e1005991</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1005991</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D14%26rft.spage%253De1005991%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1005991%26rft_id%253Dinfo%253Apmid%252F29474446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1005991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29474446&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83 in text" id="ref-83">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.02.019075v1.83"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hillen</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-source">COVID-19 Physiology Group</span>, &lt;<a href="https://sites.google.com/ualberta.ca/cov-pg/home">https://sites.google.com/ualberta.ca/cov-pg/home</a>&gt; (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84 in text" id="ref-84">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.02.019075v1.84"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">COVID-19 Cell Atlas</span>, &lt;<a href="https://www.covid19cellatlas.org/">https://www.covid19cellatlas.org/</a>&gt; (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85 in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.85" data-doi="10.1038/s41586-019-1629-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B. C.</span></span> <span class="cit-article-title">The human body at cellular resolution: the NIH Human Biomolecular Atlas Program</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">574</span>, <span class="cit-fpage">187</span>–<span class="cit-lpage">192</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-019-1629-x</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D574%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1629-x%26rft_id%253Dinfo%253Apmid%252F31597973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-019-1629-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31597973&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86 in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.86" data-doi="10.1200/CCI.18.00069"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Metzcar</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiland</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">A Review of Cell-Based Computational Modeling in Cancer Biology</span>. <abbr class="cit-jnl-abbrev">JCO Clin Cancer Inform</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1200/CCI.18.00069</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJCO%2BClin%2BCancer%2BInform%26rft.volume%253D3%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1200%252FCCI.18.00069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1200/CCI.18.00069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87 in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.87" data-doi="10.1093/bioinformatics/btv730"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghaffarizadeh</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">S. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">BioFVM: an efficient, parallelized diffusive transport solver for 3-D biological simulations</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1256</span>–<span class="cit-lpage">1258</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btv730</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv730%26rft_id%253Dinfo%253Apmid%252F26656933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26656933&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88 in text" id="ref-88">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.02.019075v1.88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiland</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-source">pc4nanobio: cancer nanotherapy simulator [nanoHUB app, Version 0.9.1]</span>, &lt;<a href="https://nanohub.org/resources/pc4nanobio">https://nanohub.org/resources/pc4nanobio</a>&gt; (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89 in text" id="ref-89">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.02.019075v1.89"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenner</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-source">Replication Competent Oncolytic Virus expressing secretable trimeric TRAIL: hypothesis testing [nanoHUB app, Version 0.3]</span>, &lt;<a href="https://nanohub.org/resources/iu399sp19p101">https://nanohub.org/resources/iu399sp19p101</a>&gt; (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90 in text" id="ref-90">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.02.019075v1.90"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiland</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-source">Physicell: liver tissue mechanobiology [nanoHUB app, Version 1.2]</span>, &lt;<a href="https://nanohub.org/resources/pc4livermedium">https://nanohub.org/resources/pc4livermedium</a>&gt; (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91 in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.91" data-doi="10.1186/s12859-018-2510-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ozik</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">High-throughput cancer hypothesis testing with an integrated PhysiCell-EMEWS workflow</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">483</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12859-018-2510-x</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D19%26rft.spage%253D483%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2510-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2510-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92 in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.92" data-doi="10.1039/c9me00036d"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ozik</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collier</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiland</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Learning-accelerated discovery of immunetumour interactions</span>. <abbr class="cit-jnl-abbrev">Mol Syst Des Eng</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">747</span>–<span class="cit-lpage">760</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1039/c9me00036d</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BSyst%2BDes%2BEng%26rft.volume%253D4%26rft.spage%253D747%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc9me00036d%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/c9me00036d&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93 in text" id="ref-93">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.02.019075v1.93"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-source">UCI Systems Biology Foundation Short Course - PhysiCell Short Course</span>, &lt;<a href="https://github.com/physicell-training/UCI-sysbio-2020">https://github.com/physicell-training/UCI-sysbio-2020</a>&gt; (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94 in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.94" data-doi="10.1093/bioinformatics/btg015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hucka</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The systems biology markup language (SBML): a medium for representation and exchange of biochemical network models</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">524</span>–<span class="cit-lpage">531</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btg015</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHucka%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D524%26rft.epage%253D531%26rft.atitle%253DThe%2Bsystems%2Bbiology%2Bmarkup%2Blanguage%2B%2528SBML%2529%253A%2Ba%2Bmedium%2Bfor%2Brepresentation%2Band%2Bexchange%2Bof%2Bbiochemical%2Bnetwork%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg015%26rft_id%253Dinfo%253Apmid%252F12611808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12611808&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181410700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95 in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.95" data-doi="10.1093/bioinformatics/btv363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Somogyi</span>,  <span class="cit-name-given-names">E. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">libRoadRunner: a high performance SBML simulation and analysis library</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">3315</span>–<span class="cit-lpage">3321</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btv363</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv363%26rft_id%253Dinfo%253Apmid%252F26085503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26085503&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96 in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.96" data-doi="10.1093/bioinformatics/btx123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stoll</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">MaBoSS 2.0: an environment for stochastic Boolean modeling</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">2226</span>–<span class="cit-lpage">2228</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btx123</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2226%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97 in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.97" data-doi="10.1093/bioinformatics/bty766"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letort</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">PhysiBoSS: a multi-scale agent-based modelling framework integrating physical dimension and cell signalling</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">1188</span>–<span class="cit-lpage">1196</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bty766</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98 in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.98" data-doi="10.1016/j.jtbi.2017.07.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cockrell</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Sepsis reconsidered: Identifying novel metrics for behavioral landscape characterization with a high-performance computing implementation of an agent-based model</span>. <abbr class="cit-jnl-abbrev">J Theor Biol</abbr> <span class="cit-vol">430</span>, <span class="cit-fpage">157</span>–<span class="cit-lpage">168</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jtbi.2017.07.016</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BTheor%2BBiol%26rft.volume%253D430%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2017.07.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2017.07.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99 in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.99" data-doi="10.1371/journal.pcbi.1005876"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cockrell</span>,  <span class="cit-name-given-names">R. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Examining the controllability of sepsis using genetic algorithms on an agent-based model of systemic inflammation</span>. <abbr class="cit-jnl-abbrev">PLOS Computational Biology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">e1005876</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1005876</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D14%26rft.spage%253D1005876e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1005876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1005876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100 in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ozik</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collier</span>,  <span class="cit-name-given-names">N. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wozniak</span>,  <span class="cit-name-given-names">J. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Spagnuolo</span>,  <span class="cit-name-given-names">C.</span></span> <abbr class="cit-jnl-abbrev">in 2016 Winter Simulation Conference (WSC) 206-220</abbr> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101 in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.101" data-doi="10.1109/tcss.2018.2859189"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ozik</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collier</span>,  <span class="cit-name-given-names">N. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wozniak</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macal</span>,  <span class="cit-name-given-names">C. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Extreme-Scale Dynamic Exploration of a Distributed Agent-Based Model With the EMEWS Framework</span>. <abbr class="cit-jnl-abbrev">IEEE Transactions on Computational Social Systems</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">884</span>–<span class="cit-lpage">895</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1109/tcss.2018.2859189</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTransactions%2Bon%2BComputational%2BSocial%2BSystems%26rft.volume%253D5%26rft.spage%253D884%26rft_id%253Dinfo%253Adoi%252F10.1109%252Ftcss.2018.2859189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/tcss.2018.2859189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.102" data-doi="10.1186/s12859-018-2508-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wozniak</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">CANDLE/Supervisor: a workflow framework for machine learning applied to cancer research</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">19</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12859-018-2508-4</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2508-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2508-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.019075v1.103" data-doi="10.1097/qad.0000000000002290"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khanna</span>,  <span class="cit-name-given-names">A. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A modeling framework to inform preexposure prophylaxis initiation and retention scale-up in the context of ‘Getting to Zero’ initiatives</span>. <abbr class="cit-jnl-abbrev">Aids</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">1911</span>–<span class="cit-lpage">1922</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1097/qad.0000000000002290</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAids%26rft.volume%253D33%26rft.spage%253D1911%26rft_id%253Dinfo%253Adoi%252F10.1097%252Fqad.0000000000002290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/qad.0000000000002290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104 in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tatara</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">in 2019 Winter Simulation Conference (WSC) 1008-1019</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105 in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.105" data-doi="10.1214/19-aoas1279"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rutter</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozik</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeYoreo</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Collier</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Microsimulation model calibration using incremental mixture approximate Bayesian computation</span>. <abbr class="cit-jnl-abbrev">The Annals of Applied Statistics</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">2189</span>–<span class="cit-lpage">2212</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1214/19-aoas1279</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BApplied%2BStatistics%26rft.volume%253D13%26rft.spage%253D2189%26rft_id%253Dinfo%253Adoi%252F10.1214%252F19-aoas1279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/19-aoas1279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106 in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.106" data-doi="10.1515/ntrev-2012-0043"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Madhavan</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">nanoHUB.org: cloud-based services for nanoscale modeling, simulation, and education</span>. <abbr class="cit-jnl-abbrev">Nanotechnology Reviews</abbr> <span class="cit-vol">2</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1515/ntrev-2012-0043</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNanotechnology%2BReviews%26rft_id%253Dinfo%253Adoi%252F10.1515%252Fntrev-2012-0043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1515/ntrev-2012-0043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107 in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kluyver</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">in ELPUB</abbr>. <span class="cit-fpage">87</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108 in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.108" data-doi="10.1101/090456"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">S. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">MultiCellDS: a community-developed standard for curating microenvironment-dependent multicellular data [Preprint]</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-vol">090456</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/090456</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F090456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwOTA0NTZ2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDIuMDE5MDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109 in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.109" data-doi="10.21105/joss.01408"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heiland</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mishler</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bower</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">xml2jupyter: Mapping parameters between XML and Jupyter widgets</span>. <abbr class="cit-jnl-abbrev">J Open Source Softw</abbr> <span class="cit-vol">4</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.21105/joss.01408</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BOpen%2BSource%2BSoftw%26rft.volume%253D4%26rft.spage%253D1408%26rft_id%253Dinfo%253Adoi%252F10.21105%252Fjoss.01408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21105/joss.01408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference 110 in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.110" data-doi="10.1016/j.jtbi.2012.02.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macklin</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edgerton</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">A. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cristini</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Patient-calibrated agent-based modelling of ductal carcinoma in situ (DCIS): from microscopic measurements to macroscopic predictions of clinical progression</span>. <abbr class="cit-jnl-abbrev">J Theor Biol</abbr> <span class="cit-vol">301</span>, <span class="cit-fpage">122</span>–<span class="cit-lpage">140</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jtbi.2012.02.002</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DMacklin%26rft.auinit1%253DP.%26rft.volume%253D301%26rft.spage%253D122%26rft.epage%253D140%26rft.atitle%253DPatient-calibrated%2Bagent-based%2Bmodelling%2Bof%2Bductal%2Bcarcinoma%2Bin%2Bsitu%2B%2528DCIS%2529%253A%2Bfrom%2Bmicroscopic%2Bmeasurements%2Bto%2Bmacroscopic%2Bpredictions%2Bof%2Bclinical%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2012.02.002%26rft_id%253Dinfo%253Apmid%252F22342935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2012.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22342935&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303079500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference 111 in text" id="ref-111">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.02.019075v1.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lowe</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-source">Angiotensin and the Coronavirus</span>, &lt;<a href="https://blogs.sciencemag.org/pipeline/archives/2020/03/17/angiotensin-and-the-coronavirus">https://blogs.sciencemag.org/pipeline/archives/2020/03/17/angiotensin-and-the-coronavirus</a>&gt; (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference 112 in text" id="ref-112">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.02.019075v1.112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beauchemin</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forrest</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Koster</span>,  <span class="cit-name-given-names">F. T.</span></span> <span class="cit-fpage">23</span>–<span class="cit-lpage">36</span> (<span class="cit-publ-name">Springer Berlin Heidelberg</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference 113 in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.019075v1.113" data-doi="10.1529/biophysj.104.053652"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sniekers</span>,  <span class="cit-name-given-names">Y. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">van Donkelaar</span>,  <span class="cit-name-given-names">C. C.</span></span> <span class="cit-article-title">Determining Diffusion Coefficients in Inhomogeneous Tissues Using Fluorescence Recovery after Photobleaching</span>. <abbr class="cit-jnl-abbrev">Biophysical Journal</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">1302</span>–<span class="cit-lpage">1307</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1529/biophysj.104.053652</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.104.053652%26rft_id%253Dinfo%253Apmid%252F15894637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1529/biophysj.104.053652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15894637&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.02.019075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.04.02.019075v1.114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Organization</span>,  <span class="cit-name-given-names">W. H.</span></span> <span class="cit-source">WHO Director-General’s opening remarks at the media briefing on COVID-19 - 11 March 2020</span>, &lt;<a href="https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020">https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020</a>&gt; (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.02.019075v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.03.024968v1" title="Factors required for adhesion of Salmonella enterica serovar Typhimurium to lettuce (Lactuca sativa)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/481796v3" title="Rapid Whole Cell Imaging Reveals A Calcium-APPL1-Dynein Nexus That Regulates Cohort Trafficking of Stimulated EGF Receptors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 05, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.02.019075v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.02.019075v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.02.019075v1" class="active" data-icon-position="" data-hide-link-title="0">Rapid community-driven development of a SARS-CoV-2 tissue simulator</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1229969" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sY28riD-qmAm7FSwoJ625QrvSG_CWtcha3nbq0qo-7k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1229969" id="node1229969" data-pisa="biorxiv;2020.04.02.019075v1" data-pisa-master="biorxiv;2020.04.02.019075" data-seqnum="1229969" data-apath="/biorxiv/early/2020/04/05/2020.04.02.019075.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Rapid community-driven development of a SARS-CoV-2 tissue simulator</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yafei</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Randy</span> <span class="nlm-surname">Heiland</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Morgan</span> <span class="nlm-surname">Craig</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Courtney</span> <span class="nlm-surname">Davis</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ashlee N.</span> <span class="nlm-surname">Ford Versypt</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Adrianne</span> <span class="nlm-surname">Jenner</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Ozik</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Nicholson</span> <span class="nlm-surname">Collier</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Chase</span> <span class="nlm-surname">Cockrell</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Becker</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Gary</span> <span class="nlm-surname">An</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">James A.</span> <span class="nlm-surname">Glazier</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Aarthi</span> <span class="nlm-surname">Narayanan</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Amber M.</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Macklin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.02.019075; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.02.019075 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;title=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;title=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;text=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;title=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;t=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;title=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;title=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1229969" id="node1229969--2" data-pisa="biorxiv;2020.04.02.019075v1" data-pisa-master="biorxiv;2020.04.02.019075" data-seqnum="1229969" data-apath="/biorxiv/early/2020/04/05/2020.04.02.019075.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Rapid community-driven development of a SARS-CoV-2 tissue simulator</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yafei</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Randy</span> <span class="nlm-surname">Heiland</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Morgan</span> <span class="nlm-surname">Craig</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Courtney</span> <span class="nlm-surname">Davis</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ashlee N.</span> <span class="nlm-surname">Ford Versypt</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Adrianne</span> <span class="nlm-surname">Jenner</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Ozik</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Nicholson</span> <span class="nlm-surname">Collier</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Chase</span> <span class="nlm-surname">Cockrell</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Becker</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Gary</span> <span class="nlm-surname">An</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">James A.</span> <span class="nlm-surname">Glazier</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Aarthi</span> <span class="nlm-surname">Narayanan</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Amber M.</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Macklin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.02.019075; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.02.019075 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Wang%2C+Yafei&amp;rft.au=Heiland%2C+Randy&amp;rft.au=Craig%2C+Morgan&amp;rft.au=Davis%2C+Courtney&amp;rft.au=Ford+Versypt%2C+Ashlee+N.&amp;rft.au=Jenner%2C+Adrianne&amp;rft.au=Ozik%2C+Jonathan&amp;rft.au=Collier%2C+Nicholson&amp;rft.au=Cockrell%2C+Chase&amp;rft.au=Becker%2C+Andrew&amp;rft.au=An%2C+Gary&amp;rft.au=Glazier%2C+James+A.&amp;rft.au=Narayanan%2C+Aarthi&amp;rft.au=Smith%2C+Amber+M.&amp;rft.au=Macklin%2C+Paul"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1229969/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1229969/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1229969/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1229969/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1229969/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1229969/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1229969/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1229969/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1229969/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1229969/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1229969/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1229969/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;count=horizontal&amp;via=&amp;text=Rapid%20community-driven%20development%20of%20a%20SARS-CoV-2%20tissue%20simulator&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.019075v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.02.019075v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/systems-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Systems Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
