<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Virol</span><span class="iso-abbrev" title="iso-abbrev">J. Virol</span><span class="hwp" title="hwp">jvi</span><span class="pmc" title="pmc">jvi</span><span class="publisher-id" title="publisher-id">JVI</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Virology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0022-538X</span><span class="issn-epub" tagx="issn" title="issn-epub">1098-5514</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Society for Microbiology</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">1752 N St., N.W., Washington, DC</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6854480">6854480</a></span><span class="publisher-id" title="publisher-id">01078-19</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1128/JVI.01078-19">10.1128/JVI.01078-19</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Genetic Diversity and Evolution</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Within-Host Multiplication and Speed of Colonization as Infection Traits Associated with Plant Virus Vertical Transmission</div> <div class="alt-title" title="alt-title">
Infection Traits and Plant Virus Seed Transmission</div> <div class="alt-title" title="alt-title">
Cobos et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cobos</span><span class="given-names" tagx="given-names" title="given-names">Alberto</span></span><a href="#aff1"><sup>a</sup></a><a href="#aff2"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montes</span><span class="given-names" tagx="given-names" title="given-names">Nuria</span></span><a href="#aff1"><sup>a</sup></a><a href="#aff2"><sup>b</sup></a><a href="#fn1">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-Herranz</span><span class="given-names" tagx="given-names" title="given-names">Marisa</span></span><a href="#aff1"><sup>a</sup></a><a href="#aff2"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gil-Valle</span><span class="given-names" tagx="given-names" title="given-names">Miriam</span></span><a href="#aff1"><sup>a</sup></a><a href="#aff2"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">Israel</span></span><a href="#aff1"><sup>a</sup></a><a href="#aff2"><sup>b</sup></a></span><span class="citation_author_institution" id="aff1">[a], <span class="addr-line" title="addr-line">Centro de Biotecnología y Genómica de Plantas (UPM-INIA), Universidad Politécnica de Madrid, Pozuelo de Alarcón, Madrid, Spain</span></span><span class="citation_author_institution" id="aff2">[b], <span class="addr-line" title="addr-line">E.T.S.I. Agronómica, Alimentaria y de Biosistemas, Campus de Montegancedo, Universidad Politécnica de Madrid, Pozuelo de Alarcón, Madrid, Spain</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simon</span><span class="given-names" tagx="given-names" title="given-names">Anne E.</span></span><span class="role" tagx="role" title="role">Editor</span><span class="citation_author_institution" id="">[], </span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Address correspondence to Israel Pagán, <span class="email" tagx="email" title="email">jesusisrael.pagan@upm.es</span>.</div> <div class="fn-type-present-address" title="present-address"><span class="label" tagx="label" title="label">*</span><p>Present address: Nuria Montes, Fisiología Vegetal, Departamento Ciencias Farmacéuticas y de la Salud, Facultad de Farmacia, Universidad San Pablo-CEU Universities, Boadilla Del Monte, Madrid, Spain, and Servicio de Reumatología, Hospital Universitario de la Princesa, Instituto de Investigación Sanitaria, Madrid, Spain.</p> </div> <div class="fn-type-other" title="other"> <p><b>Citation</b> Cobos A, Montes N, López-Herranz M, Gil-Valle M, Pagán I. 2019. Within-host multiplication and speed of colonization as infection traits associated with plant virus vertical transmission. J Virol 93:e01078-19. <a href="https://doi.org/10.1128/JVI.01078-19">https://doi.org/10.1128/JVI.01078-19</a>.</p> </div> </div><span class="pub-date-epreprint" title="pub-date-epreprint">epreprint: <span>2019-9-9</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-12-12</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-11-11</span></span><span class="volume" tagx="volume" title="volume">93</span><span class="issue" tagx="issue" title="issue">23</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e01078-19</span><span class="history" title="history"><span class="received" title="received">received: 2019-6-27</span><span class="accepted" title="accepted">accepted: 2019-8-30</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Cobos et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div><a class="self-uri" href="JVI.01078-19.pdf" title="self-uri">JVI.01078-19.pdf</a><div class="abstract" title="abstract"> <p>One of the major factors contributing to plant virus long-distance dispersal is the global trade of seeds. This is because more than 25% of plant viruses can infect seeds, which are the main mode of germplasm exchange/storage, and start new epidemics in areas where they were not previously present. Despite the relevance of this process for virus epidemiology and disease emergence, the infection traits associated with the efficiency of virus seed transmission are largely unknown. Using turnip mosaic and cucumber mosaic viruses and their natural host <span class="named-content" title="named-content">Arabidopsis thaliana</span> as model systems, we have identified the within-host speed of virus colonization and multiplication in the reproductive structures as the main determinants of the efficiency of seed transmission. These results contribute to shedding light on the mechanisms by which plant viruses disperse and optimize their fitness and may help in the design of more-efficient strategies to prevent seed infection.</p> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
ABSTRACT</div> <p>Although vertical transmission from parents to offspring through seeds is an important fitness component of many plant viruses, very little is known about the factors affecting this process. Viruses reach the seed by direct invasion of the embryo and/or by infection of the ovules or the pollen. Thus, it can be expected that the efficiency of seed transmission would be determined by (i) virus within-host multiplication and movement, (ii) the ability of the virus to invade gametic tissues, (iii) plant seed production upon infection, and (iv) seed survival in the presence of the virus. However, these predictions have seldom been experimentally tested. To address this question, we challenged 18 <span class="named-content" title="named-content">Arabidopsis thaliana</span> accessions with <i>Turnip mosaic virus</i> and <i>Cucumber mosaic virus</i>. Using these plant-virus interactions, we analyzed the relationship between the effect of virus infection on rosette and inflorescence weights; short-, medium-, and long-term seed survival; virulence; the number of seeds produced per plant; virus within-host speed of movement; virus accumulation in the rosette and inflorescence; and efficiency of seed transmission measured as a percentage and as the total number of infected seeds. Our results indicate that the best estimators of percent seed transmission are the within-host speed of movement and multiplication in the inflorescence. Together with these two infection traits, virulence and the number of seeds produced per infected plant were also associated with the number of infected seeds. Our results provide support for theoretical predictions and contribute to an understanding of the determinants of a process central to plant-virus interactions.</p> <p><b>IMPORTANCE</b> One of the major factors contributing to plant virus long-distance dispersal is the global trade of seeds. This is because more than 25% of plant viruses can infect seeds, which are the main mode of germplasm exchange/storage, and start new epidemics in areas where they were not previously present. Despite the relevance of this process for virus epidemiology and disease emergence, the infection traits associated with the efficiency of virus seed transmission are largely unknown. Using turnip mosaic and cucumber mosaic viruses and their natural host <span class="named-content" title="named-content">Arabidopsis thaliana</span> as model systems, we have identified the within-host speed of virus colonization and multiplication in the reproductive structures as the main determinants of the efficiency of seed transmission. These results contribute to shedding light on the mechanisms by which plant viruses disperse and optimize their fitness and may help in the design of more-efficient strategies to prevent seed infection.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
KEYWORDS</div> <div class="kwd" title="kwd"><i>Arabidopsis thaliana</i></div> <div class="kwd" title="kwd">
cucumber mosaic virus</div> <div class="kwd" title="kwd">
seed transmission</div> <div class="kwd" title="kwd">
turnip mosaic virus</div> <div class="kwd" title="kwd">
vertical transmission</div> <div class="kwd" title="kwd">
virus multiplication</div> <div class="kwd" title="kwd">
within-host movement</div> <div class="kwd" title="kwd">
virulence</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Universidad Politécnica de Madrid (UPM)</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100003759</span></div> </div><span class="award-id" tagx="award-id" title="award-id">PINV-18-O4W6R9-94-YR130U</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagan</span><span class="given-names" tagx="given-names" title="given-names">Israel</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Ministerio de Economía y Competitividad (MEC)</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100003329</span></div> </div><span class="award-id" tagx="award-id" title="award-id">RD16/0012/0011</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montes</span><span class="given-names" tagx="given-names" title="given-names">Nuria</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Ministerio de Economía y Competitividad (MEC)</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100003329</span></div> </div><span class="award-id" tagx="award-id" title="award-id">BIO2016-79165-R</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagan</span><span class="given-names" tagx="given-names" title="given-names">Israel</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Ministerio de Economía y Competitividad (MEC)</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100003329</span></div> </div><span class="award-id" tagx="award-id" title="award-id">BES-2017-080783</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cobos</span><span class="given-names" tagx="given-names" title="given-names">Alberto</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Comunidad de Madrid (Madrid Autonomous Community)</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/100012818</span></div> </div><span class="award-id" tagx="award-id" title="award-id">PEJ-2017-TL/BIO-6010</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gil-Valle</span><span class="given-names" tagx="given-names" title="given-names">Miriam</span></span></span></div> </div> <div class="counts" title="counts"><span class="count" tagx="count" title="count" /><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="cover-date" value="December 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>The ability to be transmitted is arguably the most important determinant of parasite fitness. Indeed, most theoretical models of the evolution of parasites consider infection traits, such as virulence, i.e., the effect of infection on host fitness (<a href="#B1">1</a>), or within-host multiplication, as relevant factors for parasite fitness because they affect the efficiency of between-host transmission (<a href="#B2">2</a><a href="#B3">–</a><a href="#B4">4</a>). Parasites can be transmitted from host to host, for instance, by vectors or by contact (i.e., horizontal transmission) and/or from parents to offspring (i.e., vertical transmission) (<a href="#B3">3</a>, <a href="#B5">5</a>). Most effort devoted to understanding parasite transmission has focused on identifying ecological and genetic determinants of horizontal transmission (<a href="#B6">6</a>, <a href="#B7">7</a>), and comparatively much less is known about the determinants of vertical transmission. However, a wide range of human, animal, and plant parasites that are causal agents of severe diseases are vertically transmitted or have both horizontal and vertical transmission (<a href="#B8">8</a>, <a href="#B9">9</a>). Hence, vertical transmission is a major component of parasite fitness, and exploring the factors involved in its efficiency is central to understanding host-parasite interactions (<a href="#B3">3</a>, <a href="#B10">10</a>).</p> <p>Vertical transmission is particularly frequent in plant viruses, as more than 25% of all known species are vertically transmitted through seeds (<a href="#B11">11</a>, <a href="#B12">12</a>). Seed transmission is highly relevant for plant virus epidemiology (<a href="#B11">11</a><a href="#B12">–</a><a href="#B13">13</a>). Seed infection provides the virus with the means to persist for long periods of time (years) when hosts and/or vectors are not available, as many seed-transmitted viruses can survive within the seed as long as it remains viable (<a href="#B11">11</a>, <a href="#B12">12</a>). This facilitates virus emergence and reemergence in plant populations (<a href="#B5">5</a>, <a href="#B14">14</a>). Seed transmission also allows for long-distance dissemination of the virus (<a href="#B14">14</a>, <a href="#B15">15</a>). Indeed, evidence shows that bird dispersion and human trade of infected seeds have allowed cross-continental jumps of some plant viruses (<a href="#B16">16</a>, <a href="#B17">17</a>). Finally, seed transmission represents an important source of primary inoculum for many viruses with this mode of transmission, which are disseminated afterwards via insect vectors. In this way, plant viruses initiate damaging epidemics even at very low seed transmission rates (<a href="#B11">11</a>, <a href="#B18">18</a>, <a href="#B19">19</a>). Although the central role of vertical transmission in plant virus epidemiology is widely acknowledged, very little is known regarding which host and virus traits interact to determine the efficiency of seed transmission (<a href="#B11">11</a>, <a href="#B14">14</a>).</p> <p>In general, plant viruses achieve seed transmission in two ways according to the distribution of the virus in the seed. The first is through contamination of the seed coat. In this case, during germination, the virus infects the seedling through abrasions caused by soil particles (<a href="#B11">11</a>, <a href="#B20">20</a>). This mechanism of seed transmission has been reported for a few viruses and is relatively well understood only for tobamoviruses (<a href="#B21">21</a>). The second, and most common, method of virus seed transmission is through invasion of the seed embryo (<a href="#B11">11</a>, <a href="#B22">22</a>). Embryo invasion may occur in two non-mutually exclusive ways: indirectly by infection of plant gametes prior to fertilization, either the ovules or the pollen, or directly from the mother plant to the embryonic tissue after fertilization (<a href="#B23">23</a>). Hence, for seed transmission to occur, it is crucial that the virus reaches and invades plant reproductive organs before gametogenesis and/or while the embryo is still accessible from mother cells, without affecting gamete/embryo viability. Plant defense responses that regulate virus virulence may enhance or prevent embryo invasion, for instance, by altering the virus distribution in the plant, which may modify the efficiency of seed transmission (<a href="#B24">24</a>). Thus, it has been proposed that the efficiency of seed transmission would be determined by (i) the ability of the virus to reach gametic tissues, which would be determined by the speed of within-host movement; (ii) the ability of the virus to invade gametic tissues, which would be associated with virus multiplication in reproductive organs; (iii) plant progeny production upon infection (i.e., virus virulence); and (iv) gamete and embryo survival in the presence of the virus (<a href="#B3">3</a>, <a href="#B10">10</a>, <a href="#B11">11</a>, <a href="#B20">20</a>, <a href="#B23">23</a>).</p> <p>Experimental evidence of the role of these infection traits in virus seed transmission is scarce. Plant genes involved in the efficiency of virus seed transmission have been identified only in soybean (<a href="#B24">24</a>). In this host, <i>Soybean mosaic virus</i> seed transmission is controlled by plant gene homologs of <span class="named-content" title="named-content">Arabidopsis thaliana</span><i>DCL3</i> and <i>RDR6</i>, which are involved in small RNA-mediated gene silencing (<a href="#B24">24</a>). Similarly, virus seed transmission determinants have been analyzed in only a few species. Genetic variation in <i>Barley stripe mosaic virus</i> (BSMV), <i>Cucumber mosaic virus</i> (CMV), and <i>Pea seed-borne mosaic virus</i> genes encoding the replicase and movement proteins has been associated with the efficiency of seed transmission (<a href="#B25">25</a>, <a href="#B26">26</a>). This would be compatible with a role of virus multiplication, virulence, and movement. However, whether (and how) infection traits affect the efficiency of seed transmission has been seldom analyzed, and with contradictory results. Pagán et al. (<a href="#B27">27</a>) showed that reduced CMV virulence and within-host multiplication were associated with an increased efficiency of seed transmission in <span class="named-content" title="named-content">Arabidopsis thaliana</span>. Stewart et al. (<a href="#B28">28</a>) also reported a negative correlation between BSMV virulence and the efficiency of virus seed transmission in barley, but no link to virus multiplication was detected. These works studied the effect of infection traits on the efficiency of seed transmission using univariate analyses. However, during infection, virus multiplication, movement, and reduction of plant fitness occur simultaneously, such that the efficiency of seed transmission would be determined by their combined (and not necessarily equally important) effects (<a href="#B20">20</a>). To date, such multivariate effects have not been analyzed, and the infection traits associated with virus seed transmission (and their relative importance) are still poorly understood (<a href="#B14">14</a>).</p> <p>The efficiency of seed transmission may also affect parasite evolution. The fitness of vertically transmitted parasites is highly dependent on host reproductive potential, as hosts need to reproduce for the parasite to infect new individuals. Accordingly, the "continuum hypothesis�? proposes that parasites with a higher efficiency of vertical transmission will evolve toward lower virulence and, because virulence is an unavoidable consequence of parasite growth, toward lower within-host multiplication (<a href="#B3">3</a>, <a href="#B29">29</a>, <a href="#B30">30</a>). These predictions have seldom been experimentally tested, particularly for plant viruses (<a href="#B5">5</a>, <a href="#B27">27</a>). Moreover, these works commonly estimated the efficiency of vertical transmission by determining the proportion of offspring that carry the parasite. This measure of the efficiency of seed transmission does not account for variation in the number of propagules that different host genotypes can produce, which may affect parasite fitness. Indeed, it has been proposed that the total number of infected progeny reflects more accurately the contribution of vertical transmission to parasite fitness and therefore is more directly linked to parasite evolution (<a href="#B2">2</a>, <a href="#B3">3</a>, <a href="#B5">5</a>, <a href="#B10">10</a>, <a href="#B31">31</a>). According to this theory, parasite vertical transmission-related fitness is the result of the interplay between the percentage of infected progeny, virulence, and the amount of progeny produced by the host. The first determines the proportion of infected progeny, the second determines the reduction of the host’s maximal progeny production, and the third determines progeny production upon infection (<a href="#B2">2</a>, <a href="#B3">3</a>, <a href="#B5">5</a>, <a href="#B31">31</a>). Thus, understanding how the efficiency of seed transmission affects virus evolution requires considering both the percentage of seed transmission and the total number of infected propagules.</p> <p>To address these questions, we utilized <i>Turnip mosaic virus</i> (TuMV) (<i>Potyviridae</i>), CMV (<i>Bromoviridae</i>), and <span class="named-content" title="named-content">Arabidopsis thaliana</span> (Brassicaceae) (referred to here as <i>Arabidopsis</i>). Both viruses are commonly found in natural populations of <i>Arabidopsis</i> at a prevalence of up to 80% (<a href="#B32">32</a>), indicating that <i>Arabidopsis</i>-TuMV and <i>Arabidopsis</i>-CMV interactions are significant in nature. In <i>Arabidopsis</i>, CMV has been shown to be seed transmitted (<a href="#B14">14</a>, <a href="#B27">27</a>), and the high TuMV prevalence in natural <i>Arabidopsis</i> populations early in the spring (before aphid flights) suggests that this virus may also be seed transmitted (<a href="#B32">32</a>, <a href="#B33">33</a>). In addition, TuMV and CMV infections in <i>Arabidopsis</i> differ in traits proposed to be associated with the efficiency of seed transmission. For instance, TuMV multiplies at lower levels than CMV, whereas TuMV is more virulent (<a href="#B34">34</a>; N. Montes, V. Vijayan, and I Pagán, submitted for publication). Also, both viruses differentially affect the survival of infected seeds (<a href="#B35">35</a>). Thus, our experimental system allows testing of theoretical predictions on the host and virus traits that determine seed transmission under different infection conditions. To do so, in six <i>Arabidopsis</i> accessions, we measured (i) TuMV and CMV speed of within-host movement and multiplication, as proxies of the ability of the virus to reach and invade gametic tissues, respectively; (ii) the effect of virus infection on plant growth and progeny production, as a measure of virulence; and (iii) short-, medium-, and long-term survival of infected seeds. Using multivariate mixed models, we investigated which of these infection traits are associated with the efficiency of virus seed transmission, quantified both as a percentage and as the total number of infected seeds, and their relative importance. We constructed global models that considered data for both viruses together as well as virus-specific models, such that we could differentiate traits broadly associated with seed transmission from virus species-specific determinants of this process. We validated the resulting models by measuring, in a larger set of 18 <i>Arabidopsis</i> accessions, seed transmission and the most relevant estimators of these traits, and we compared the values predicted by our models with experimental quantifications of seed transmission.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
RESULTS</div> <div class="tumvandcmvseedtransmissioninarabidopsis." title="sec"> <div class="title" tagx="title" title="title">
TuMV and CMV seed transmission in <i>Arabidopsis</i>.</div> <p>We quantified seed transmission as a percentage (<i>ST</i>) and as the total number (<i>IS</i>) of TuMV- and CMV-infected seeds in 18 <i>Arabidopsis</i> accessions (<a href="#F1">Fig. 1</a> and <a href="#T1">Table 1</a>). In TuMV-infected plants, the virus was detected in 12/18 accessions, whereas seeds harboring CMV were detected in all accessions. On average, the percentage of infected seeds was higher in TuMV-infected (4.17% ± 1.53%) than in CMV-infected (2.60% ± 0.23%) plants (Wald χ<sup>2</sup><sub>1,775</sub> = 8.59 [where subscript 1 is the degree of freedom of chi squared and 775 the degrees of freedom of the error term in the same chi-squared analysis]; <i>P</i> = 0.003). Similar results were obtained when seed transmission was quantified as the number of infected seeds (2.13 ± 0.19 versus 1.35 ± 0.07 for TuMV and CMV, respectively) (Wald χ<sup>2</sup><sub>1,775</sub> = 13.45; <i>P</i> &amp;lt; 0.001). Both measures of seed transmission depended on the <i>Arabidopsis</i> accession (Wald χ<sup>2</sup><sub>17,759</sub> ≥ 206.42; <i>P</i> &amp;lt; 0.001) and on the interaction between accession and virus species (Wald χ<sup>2</sup><sub>17,759</sub> ≥ 199.62; <i>P</i> &amp;lt; 0.001). Thus, we analyzed both <i>ST</i> and <i>IS</i> for each virus separately.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 1</span></a></div>   <p>Virus seed transmission in <i>Arabidopsis</i>. TuMV seed transmission percentage (A) and log number of infected seeds (B) and CMV seed transmission percentage (C) and log number of infected seeds (D) in 18 <i>Arabidopsis</i> accessions are shown. Data for JPN1-TuMV (black), UK1-TuMV (gray), De72-CMV (red), Fny-CMV (blue), and LS-CMV (green) are represented. Note the different scale of each panel.</p>   </div> <div class="table-wrap_UNKNOWN" id="T1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p><i>Arabidopsis</i> accessions used in this work and their geographical origin and life cycle</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div>  <thead> <tr> <th rowspan="1" colspan="1">Accession</th> <th rowspan="1" colspan="1">Origin</th> <th rowspan="1" colspan="1">Life cycle<a href="#T1F1"><sup><i>a</i></sup></a></th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">An-1</td> <td rowspan="1" colspan="1">Amberes (Belgium)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Bay-0</td> <td rowspan="1" colspan="1">Bayreuth (Germany)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Boa-0</td> <td rowspan="1" colspan="1">Boadilla (Spain)</td> <td rowspan="1" colspan="1">Long</td> </tr> <tr> <td rowspan="1" colspan="1">Cad-0</td> <td rowspan="1" colspan="1">Candelario (Spain)</td> <td rowspan="1" colspan="1">Long</td> </tr> <tr> <td rowspan="1" colspan="1">Cen-1</td> <td rowspan="1" colspan="1">Centenera (Spain)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Col-1</td> <td rowspan="1" colspan="1">Columbia (unknown)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Cum-0</td> <td rowspan="1" colspan="1">Cumbres Mayores (Spain)</td> <td rowspan="1" colspan="1">Long</td> </tr> <tr> <td rowspan="1" colspan="1">Cvi</td> <td rowspan="1" colspan="1">Cape Verde Islands</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Fei-0</td> <td rowspan="1" colspan="1">Santa María da Feira (Portugal)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Kas-0</td> <td rowspan="1" colspan="1">Kashmir (India)</td> <td rowspan="1" colspan="1">Long</td> </tr> <tr> <td rowspan="1" colspan="1">Kas-2</td> <td rowspan="1" colspan="1">Kashmir (India)</td> <td rowspan="1" colspan="1">Long</td> </tr> <tr> <td rowspan="1" colspan="1">Kyo-1</td> <td rowspan="1" colspan="1">Kyoto (Japan)</td> <td rowspan="1" colspan="1">Long</td> </tr> <tr> <td rowspan="1" colspan="1">Ler</td> <td rowspan="1" colspan="1">Landsberg (Poland)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Ll-0</td> <td rowspan="1" colspan="1">Llagostera (Spain)</td> <td rowspan="1" colspan="1">Long</td> </tr> <tr> <td rowspan="1" colspan="1">Pro-0</td> <td rowspan="1" colspan="1">Proaza (Spain)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Shak</td> <td rowspan="1" colspan="1">Shakdara (Tajikistan)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Sne</td> <td rowspan="1" colspan="1">Sierra Nevada (Spain)</td> <td rowspan="1" colspan="1">Short</td> </tr> <tr> <td rowspan="1" colspan="1">Vif-0</td> <td rowspan="1" colspan="1">Villafáfila (Spain)</td> <td rowspan="1" colspan="1">Short</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">a</span><p>Length of the accession life cycle as defined previously (<a href="#B50">50</a>).</p> </div> </div> </div> <p>TuMV isolate UK1 (UK1-TuMV) was seed transmitted in 6/18 accessions, and JPN1-TuMV was transmitted in 12/18 (<a href="#F1">Fig. 1A</a> and <a href="#F1">B</a>), with the percentage of TuMV seed transmission varying according to the <i>Arabidopsis</i> accession (Wald χ<sup>2</sup><sub>17,306</sub> = 230.64; <i>P</i> &amp;lt; 0.001) but not to the virus isolate (UK1-TuMV, 3.34% ± 2.73%; JPN1-TuMV, 4.99 ± 1.46) (Wald χ<sup>2</sup><sub>1,306</sub> = 2.51; <i>P</i> = 0.113). The interaction between these factors was significant (Wald χ<sup>2</sup><sub>17,306</sub> = 91.76; <i>P</i> ≤ 0.001). Similar trends were observed when the number of TuMV-infected seeds was analyzed (<a href="#F1">Fig. 1A</a> and <a href="#F1">B</a>).</p> <p>The three CMV isolates were seed transmitted in all <i>Arabidopsis</i> accessions (<a href="#F1">Fig. 1C</a> and <a href="#F1">D</a>). The percentages of CMV seed transmission differed between <i>Arabidopsis</i> accessions (Wald χ<sup>2</sup><sub>17,419</sub> = 81.59; <i>P</i> &amp;lt; 0.001) and between virus isolates (Wald χ<sup>2</sup><sub>2,419</sub> = 22.38; <i>P</i> &amp;lt; 0.001), with similar values for Fny-CMV (2.59% ± 0.46%) and LS-CMV (2.37% ± 0.34%) and lower values for De72-CMV (1.83% ± 0.30%). The interaction between these factors was significant (Wald χ<sup>2</sup><sub>17,419</sub> = 56.49; <i>P</i> = 0.009). Again, similar results were obtained when CMV seed transmission was quantified as the number of infected seeds (<a href="#F1">Fig. 1C</a> and <a href="#F1">D</a>).</p> <p>These results indicated that the efficiency of seed transmission depended on the host-virus genotype-per-genotype interaction. Thus, infection traits under the control of the host and/or the virus may modulate this process. We subsequently quantified, in different <i>Arabidopsis</i> accessions inoculated with TuMV and CMV, a set of infection-related traits, and we analyzed their relationship with the efficiency of seed transmission.</p> </div> <div class="tumvandcmvinfectioninarabidopsis." title="sec"> <div class="title" tagx="title" title="title">
TuMV and CMV infection in <i>Arabidopsis</i>.</div> <p>Six out of the 18 <i>Arabidopsis</i> accessions (An-1, Bay-0, Cad-0, Cum-0, Ll-0, and Fei-0) and 2 virus isolates (Fny-CMV and JPN1-TuMV) were selected to analyze the relationship between 9 traits proposed to be linked to virus seed transmission and <i>ST</i> and <i>IS</i>. Specifically, we calculated the effects of virus infection on rosette weight (<i>RW</i>) (ratio of <i>RW</i> of infected plants/<i>RW</i> of mock-inoculated plants [<i>RW<sub>i</sub></i>/<i>RW<sub>m</sub></i>]), inflorescence weight (<i>IW</i>) (<i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i>), and short-term (seed germination percentage at time zero [<i>G</i><sub>0</sub>]) (<i>G</i><sub>0</sub><i><sub>i</sub></i>/<i>G</i><sub>0</sub><i><sub>m</sub></i>), medium-term (<i>G</i><sub>24</sub><i><sub>i</sub></i>/<i>G</i><sub>24</sub><i><sub>m</sub></i>), and long-term (<i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i>) seed survival; virulence (<i>V</i>); the number of seeds produced per infected plant (<i>SN<sub>i</sub></i>); virus within-host speed of movement (<i>SM</i>); and virus accumulation in rosette (<i>VA<sub>R</sub></i>) and inflorescence (<i>VA<sub>I</sub></i>) leaves, plus <i>ST</i> and <i>IS</i> (see Materials and Methods).</p> <p>Overall, the effect of infection on <i>RW</i>, <i>IW</i>, and <i>G</i><sub>24</sub> plus <i>V</i>, <i>SN<sub>i</sub></i>, <i>SM</i>, <i>VA<sub>R</sub></i>, and <i>VA<sub>I</sub></i> differed according to the virus isolate (Wald χ<sup>2</sup><sub>1,100</sub> ≥ 3.70; <i>P</i> ≤ 0.054), the <i>Arabidopsis</i> accession (Wald χ2<sub>5,100</sub> ≥ 11.94, <i>P</i> ≤ 0.036), and the accession-per-isolate interaction (Wald χ<sup>2</sup><sub>5,100</sub> ≥ 22.83; <i>P</i> &amp;lt; 0.001), whereas the effects of virus infection on <i>G</i><sub>0</sub> and <i>G<sub>48</sub></i> did not (Wald χ<sup>2</sup> ≤ 8.97; <i>P</i> ≥ 0.110) (<a href="#T2">Table 2</a>). Note that differences in <i>SM</i> according to the <i>Arabidopsis</i> accession and the accession-per-isolate interaction could not be analyzed, as this trait was quantified as an accession-specific, rather than as an individual-specific, trait (see Materials and Methods). These results prompted us to analyze the variation in these infection traits for each virus separately. In JPN1-TuMV-infected plants, the effect of infection on <i>RW</i>, <i>IW</i>, and <i>G</i><sub>24</sub> plus <i>V</i>, <i>SN<sub>i</sub></i>, <i>VA<sub>R</sub></i>, and <i>VA<sub>I</sub></i> varied according to the <i>Arabidopsis</i> accession (Wald χ<sup>2</sup><sub>5,49</sub> ≥ 17.38; <i>P</i> ≤ 0.002), and the effects of infection on <i>G</i><sub>0</sub> and <i>G</i><sub>48</sub> did not (Wald χ<sup>2</sup><sub>5,49</sub> ≤ 8.41; <i>P</i> ≥ 0.078) (<a href="#T2">Table 2</a>). Similar results were obtained for Fny-CMV-infected plants (Wald χ<sup>2</sup><sub>5,46</sub> ≥ 19.79; <i>P</i> &amp;lt; 0.001), except that in this case, only the effect of virus infection on <i>G</i><sub>0</sub> did not vary between accessions (Wald χ<sup>2</sup><sub>5,46</sub> = 4.56; <i>P</i> = 0.472) (<a href="#T2">Table 2</a>). Note that Fny-CMV virulence was negative in some accessions, indicating plant overcompensation for the effect of infection on its fitness, as previously reported (<a href="#B33">33</a>, <a href="#B34">34</a>; Montes et al., submitted).</p> <div class="table-wrap_UNKNOWN" id="T2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 2</span> <p>Virus infection parameters measured in the six <i>Arabidopsis</i> accessions utilized to construct multivariate models for virus seed transmission</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div>  <thead> <tr> <th rowspan="2" colspan="1">Virus and parameter<a href="#T2F1"><sup><i>a</i></sup></a></th> <th colspan="7" rowspan="1">Mean value for accession ± SE <hr /> </th> </tr> <tr> <th rowspan="1" colspan="1">An-1</th> <th rowspan="1" colspan="1">Bay-0</th> <th rowspan="1" colspan="1">Fei-0</th> <th rowspan="1" colspan="1">Cad-0</th> <th rowspan="1" colspan="1">Cum-0<a href="#T2F2"><sup><i>b</i></sup></a></th> <th rowspan="1" colspan="1">Ll-0</th> <th rowspan="1" colspan="1">Avg<a href="#T2F3"><sup><i>c</i></sup></a></th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">JPN1-TuMV</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">    <i>ST</i></td> <td rowspan="1" colspan="1">22.38 ± 6.50</td> <td rowspan="1" colspan="1">0.00 ± 0.00</td> <td rowspan="1" colspan="1">20.27 ± 5.82</td> <td rowspan="1" colspan="1">1.06 ± 0.67</td> <td rowspan="1" colspan="1">0.00 ± 0.00</td> <td rowspan="1" colspan="1">6.08 ± 0.70</td> <td rowspan="1" colspan="1">8.24 ± 1.85</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>IS</i><a href="#T2F4"><sup><i>d</i></sup></a></td> <td rowspan="1" colspan="1">2.33 ± 0.33</td> <td rowspan="1" colspan="1">0.00 ± 0.00</td> <td rowspan="1" colspan="1">2.25 ± 0.53</td> <td rowspan="1" colspan="1">1.15 ± 0.53</td> <td rowspan="1" colspan="1">0.00 ± 0.00</td> <td rowspan="1" colspan="1">1.83 ± 0.31</td> <td rowspan="1" colspan="1">1.25 ± 0.18</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>RW<sub>i</sub></i>/<i>RW<sub>m</sub></i></td> <td rowspan="1" colspan="1">0.83 ± 0.23</td> <td rowspan="1" colspan="1">0.98 ± 0.13</td> <td rowspan="1" colspan="1">0.46 ± 0.02</td> <td rowspan="1" colspan="1">0.75 ± 0.03</td> <td rowspan="1" colspan="1">0.30 ± 0.02</td> <td rowspan="1" colspan="1">0.03 ± 0.00</td> <td rowspan="1" colspan="1">0.56 ± 0.07</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i></td> <td rowspan="1" colspan="1">0.55 ± 0.07</td> <td rowspan="1" colspan="1">0.67 ± 0.10</td> <td rowspan="1" colspan="1">0.68 ± 0.05</td> <td rowspan="1" colspan="1">0.76 ± 0.03</td> <td rowspan="1" colspan="1">ND</td> <td rowspan="1" colspan="1">0.16 ± 0.02</td> <td rowspan="1" colspan="1">0.55 ± 0.04</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>V</i></td> <td rowspan="1" colspan="1">0.76 ± 0.09</td> <td rowspan="1" colspan="1">0.67 ± 0.09</td> <td rowspan="1" colspan="1">0.55 ± 0.16</td> <td rowspan="1" colspan="1">0.25 ± 0.05</td> <td rowspan="1" colspan="1">1.00 ± 0.00</td> <td rowspan="1" colspan="1">0.90 ± 0.01</td> <td rowspan="1" colspan="1">0.72 ± 0.04</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>SN<sub>i</sub></i><a href="#T2F4"><sup><i>d</i></sup></a></td> <td rowspan="1" colspan="1">3.19 ± 0.27</td> <td rowspan="1" colspan="1">3.20 ± 0.36</td> <td rowspan="1" colspan="1">3.75 ± 0.19</td> <td rowspan="1" colspan="1">4.36 ± 0.03</td> <td rowspan="1" colspan="1">0.00 ± 0.00</td> <td rowspan="1" colspan="1">2.57 ± 0.12</td> <td rowspan="1" colspan="1">2.94 ± 0.21</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>SM</i></td> <td rowspan="1" colspan="1">1.09 ± 0.00</td> <td rowspan="1" colspan="1">0.87 ± 0.00</td> <td rowspan="1" colspan="1">1.40 ± 0.00</td> <td rowspan="1" colspan="1">0.60 ± 0.00</td> <td rowspan="1" colspan="1">ND</td> <td rowspan="1" colspan="1">0.92 ± 0.00</td> <td rowspan="1" colspan="1">1.00 ± 0.03</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>VA<sub>R</sub></i></td> <td rowspan="1" colspan="1">2.59 ± 0.37</td> <td rowspan="1" colspan="1">0.58 ± 0.07</td> <td rowspan="1" colspan="1">0.49 ± 0.29</td> <td rowspan="1" colspan="1">0.48 ± 0.04</td> <td rowspan="1" colspan="1">2.82 ± 0.21</td> <td rowspan="1" colspan="1">1.94 ± 0.57</td> <td rowspan="1" colspan="1">1.54 ± 0.18</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>VA<sub>I</sub></i></td> <td rowspan="1" colspan="1">0.69 ± 0.16</td> <td rowspan="1" colspan="1">0.08 ± 0.02</td> <td rowspan="1" colspan="1">0.33 ± 0.06</td> <td rowspan="1" colspan="1">0.16 ± 0.04</td> <td rowspan="1" colspan="1">ND</td> <td rowspan="1" colspan="1">0.08 ± 0.04</td> <td rowspan="1" colspan="1">0.28 ± 0.05</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>G</i><sub>0</sub><i><sub>i</sub></i>/<i>G</i><sub>0</sub><i><sub>m</sub></i></td> <td rowspan="1" colspan="1">0.99 ± 0.01</td> <td rowspan="1" colspan="1">0.98 ± 0.01</td> <td rowspan="1" colspan="1">0.99 ± 0.00</td> <td rowspan="1" colspan="1">1.01 ± 0.01</td> <td rowspan="1" colspan="1">ND</td> <td rowspan="1" colspan="1">1.00 ± 0.01</td> <td rowspan="1" colspan="1">0.99 ± 0.00</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>G</i><sub>24</sub><i><sub>i</sub></i>/<i>G</i><sub>24</sub><i><sub>m</sub></i></td> <td rowspan="1" colspan="1">0.95 ± 0.01</td> <td rowspan="1" colspan="1">1.01 ± 0.02</td> <td rowspan="1" colspan="1">0.92 ± 0.12</td> <td rowspan="1" colspan="1">1.05 ± 0.01</td> <td rowspan="1" colspan="1">ND</td> <td rowspan="1" colspan="1">0.99 ± 0.03</td> <td rowspan="1" colspan="1">0.98 ± 0.01</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i></td> <td rowspan="1" colspan="1">0.94 ± 0.08</td> <td rowspan="1" colspan="1">1.02 ± 0.12</td> <td rowspan="1" colspan="1">1.05 ± 0.18</td> <td rowspan="1" colspan="1">1.08 ± 0.15</td> <td rowspan="1" colspan="1">ND</td> <td rowspan="1" colspan="1">0.94 ± 0.29</td> <td rowspan="1" colspan="1">1.00 ± 0.07</td> </tr> <tr> <td colspan="8" rowspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Fny-CMV</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">    <i>ST</i></td> <td rowspan="1" colspan="1">1.54 ± 0.82</td> <td rowspan="1" colspan="1">1.20 ± 0.23</td> <td rowspan="1" colspan="1">1.78 ± 0.50</td> <td rowspan="1" colspan="1">2.58 ± 0.43</td> <td rowspan="1" colspan="1">6.17 ± 1.49</td> <td rowspan="1" colspan="1">6.85 ± 1.39</td> <td rowspan="1" colspan="1">3.27 ± 0.47</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>IS</i><a href="#T2F4"><sup><i>d</i></sup></a></td> <td rowspan="1" colspan="1">1.15 ± 0.41</td> <td rowspan="1" colspan="1">1.27 ± 0.20</td> <td rowspan="1" colspan="1">1.32 ± 0.34</td> <td rowspan="1" colspan="1">2.69 ± 0.47</td> <td rowspan="1" colspan="1">2.42 ± 0.10</td> <td rowspan="1" colspan="1">3.31 ± 0.13</td> <td rowspan="1" colspan="1">2.01 ± 0.15</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>RW<sub>i</sub></i>/<i>RW<sub>m</sub></i></td> <td rowspan="1" colspan="1">0.54 ± 0.07</td> <td rowspan="1" colspan="1">0.38 ± 0.02</td> <td rowspan="1" colspan="1">0.34 ± 0.02</td> <td rowspan="1" colspan="1">0.11 ± 0.02</td> <td rowspan="1" colspan="1">0.17 ± 0.03</td> <td rowspan="1" colspan="1">0.40 ± 0.04</td> <td rowspan="1" colspan="1">0.32 ± 0.02</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i></td> <td rowspan="1" colspan="1">0.57 ± 0.07</td> <td rowspan="1" colspan="1">0.34 ± 0.03</td> <td rowspan="1" colspan="1">0.30 ± 0.04</td> <td rowspan="1" colspan="1">0.27 ± 0.05</td> <td rowspan="1" colspan="1">0.17 ± 0.04</td> <td rowspan="1" colspan="1">0.93 ± 0.07</td> <td rowspan="1" colspan="1">0.42 ± 0.04</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>V</i></td> <td rowspan="1" colspan="1">0.50 ± 0.07</td> <td rowspan="1" colspan="1">0.60 ± 0.03</td> <td rowspan="1" colspan="1">0.74 ± 0.04</td> <td rowspan="1" colspan="1">0.05 ± 0.16</td> <td rowspan="1" colspan="1">−0.65 ± 0.25</td> <td rowspan="1" colspan="1">−0.57 ± 0.05</td> <td rowspan="1" colspan="1">0.13 ± 0.09</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>SN<sub>i</sub></i><a href="#T2F4"><sup><i>d</i></sup></a></td> <td rowspan="1" colspan="1">3.84 ± 0.06</td> <td rowspan="1" colspan="1">3.63 ± 0.04</td> <td rowspan="1" colspan="1">3.65 ± 0.09</td> <td rowspan="1" colspan="1">4.63 ± 0.06</td> <td rowspan="1" colspan="1">4.36 ± 0.10</td> <td rowspan="1" colspan="1">4.86 ± 0.02</td> <td rowspan="1" colspan="1">4.13 ± 0.07</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>SM</i></td> <td rowspan="1" colspan="1">0.42 ± 0.00</td> <td rowspan="1" colspan="1">0.47 ± 0.00</td> <td rowspan="1" colspan="1">0.57 ± 0.00</td> <td rowspan="1" colspan="1">0.71 ± 0.00</td> <td rowspan="1" colspan="1">0.93 ± 0.00</td> <td rowspan="1" colspan="1">0.75 ± 0.00</td> <td rowspan="1" colspan="1">0.64 ± 0.02</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>VA<sub>R</sub></i></td> <td rowspan="1" colspan="1">1.48 ± 0.14</td> <td rowspan="1" colspan="1">3.07 ± 0.63</td> <td rowspan="1" colspan="1">2.00 ± 0.16</td> <td rowspan="1" colspan="1">2.44 ± 0.29</td> <td rowspan="1" colspan="1">2.64 ± 0.30</td> <td rowspan="1" colspan="1">1.39 ± 0.07</td> <td rowspan="1" colspan="1">2.19 ± 0.16</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>VA<sub>I</sub></i></td> <td rowspan="1" colspan="1">0.25 ± 0.10</td> <td rowspan="1" colspan="1">0.21 ± 0.07</td> <td rowspan="1" colspan="1">0.23 ± 0.07</td> <td rowspan="1" colspan="1">0.32 ± 0.06</td> <td rowspan="1" colspan="1">0.28 ± 0.13</td> <td rowspan="1" colspan="1">0.41 ± 0.08</td> <td rowspan="1" colspan="1">0.28 ± 0.04</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>G</i><sub>0</sub><i><sub>i</sub></i>/<i>G</i><sub>0</sub><i><sub>m</sub></i></td> <td rowspan="1" colspan="1">1.00 ± 0.01</td> <td rowspan="1" colspan="1">1.00 ± 0.01</td> <td rowspan="1" colspan="1">1.00 ± 0.00</td> <td rowspan="1" colspan="1">1.00 ± 0.00</td> <td rowspan="1" colspan="1">1.00 ± 0.00</td> <td rowspan="1" colspan="1">1.01 ± 0.01</td> <td rowspan="1" colspan="1">1.00 ± 0.00</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>G</i><sub>24</sub><i><sub>i</sub></i>/<i>G</i><sub>24</sub><i><sub>m</sub></i></td> <td rowspan="1" colspan="1">1.00 ± 0.02</td> <td rowspan="1" colspan="1">0.97 ± 0.02</td> <td rowspan="1" colspan="1">1.00 ± 0.02</td> <td rowspan="1" colspan="1">0.74 ± 0.12</td> <td rowspan="1" colspan="1">0.79 ± 0.04</td> <td rowspan="1" colspan="1">0.81 ± 0.08</td> <td rowspan="1" colspan="1">0.89 ± 0.03</td> </tr> <tr> <td rowspan="1" colspan="1">    <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i></td> <td rowspan="1" colspan="1">0.84 ± 0.15</td> <td rowspan="1" colspan="1">0.98 ± 0.06</td> <td rowspan="1" colspan="1">0.88 ± 0.18</td> <td rowspan="1" colspan="1">0.81 ± 0.18</td> <td rowspan="1" colspan="1">0.81 ± 0.09</td> <td rowspan="1" colspan="1">0.61 ± 0.11</td> <td rowspan="1" colspan="1">0.82 ± 0.06</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">a</span><p><i>SM</i> values have a standard error of 0.00, as they were measured as an <i>Arabidopsis</i> accession-specific trait.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">b</span><p>Cum-0 plants infected by JPN1-TuMV did not produce inflorescence and seeds, and the associated parameters could not be determined (ND).</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">c</span><p>Average value for each parameter across all six <i>Arabidopsis</i> accessions.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">d</span><p>Log values are shown.</p> </div> </div> </div> <p>The seed transmission percentage and number of infected seeds were also estimated in the six <i>Arabidopsis</i> accessions. In this subset of accessions, <i>ST</i> and <i>IS</i> significantly varied according to the virus isolate (Wald χ<sup>2</sup><sub>1,100</sub> ≥ 6.25; <i>P</i> ≤ 0.012), the <i>Arabidopsis</i> accession (Wald χ<sup>2</sup><sub>5,100</sub> ≥ 39.29; <i>P</i> &amp;lt; 0.001), and the interaction between them (Wald χ<sup>2</sup><sub>5,100</sub> ≥ 56.46; <i>P</i> &amp;lt; 0.001). In agreement, virus-specific analyses showed that for both viruses, <i>ST</i> and <i>IS</i> varied significantly between <i>Arabidopsis</i> accessions (Wald χ<sup>2</sup> ≥ 30.62; <i>P</i> &amp;lt; 0.001). <i>ST</i> and <i>IS</i> in the six <i>Arabidopsis</i> accessions showed similar trends between experiments (compare data in <a href="#F1">Fig. 1</a> and <a href="#T2">Table 2</a>). Indeed, bivariate tests indicated a significant association between values in the two experiments for both <i>ST</i> and <i>IS</i> (<i>R</i><sup>2</sup> ≥ 0.49; <i>P</i> &amp;lt; 0.001).</p> <p>Thus, all the analyzed infection traits, except for the effect of infection on short-term seed survival, showed variability according to the virus isolate, the <i>Arabidopsis</i> accession, and/or both factors.</p> </div> <div class="associationbetweencmvandtumvseedtransmissionandviralinfectiontraitsinarabidopsis." title="sec"> <div class="title" tagx="title" title="title">
Association between CMV and TuMV seed transmission and viral infection traits in <i>Arabidopsis</i>.</div> <p>Since the efficiency of seed transmission depended on the plant-virus genotype-per-genotype interaction, and we showed a similar variation for most infection traits proposed to be associated with this mode of transmission, we considered these infection traits as potential estimators of seed transmission. Thus, we performed more-detailed analyses of the association between infection traits and virus seed transmission utilizing multiple-regression model selection analyses (<a href="#T3">Table 3</a>).</p> <div class="table-wrap_UNKNOWN" id="T3" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 3</span> <p>Model selection analyses for TuMV and CMV percentages and total numbers of infected seeds<a href="#T3F7"><sup><i>g</i></sup></a></p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div>  <thead> <tr> <th rowspan="1" colspan="1">Model structure<a href="#T3F1"><sup><i>a</i></sup></a></th> <th rowspan="1" colspan="1"><i>R</i><sup>2</sup><i><sub>c</sub></i><a href="#T3F2"><sup><i>b</i></sup></a></th> <th rowspan="1" colspan="1">Log likelihood</th> <th rowspan="1" colspan="1">AIC<a href="#T3F3"><sup><i>c</i></sup></a></th> <th rowspan="1" colspan="1">Δ<i><sub>i</sub></i><a href="#T3F4"><sup><i>d</i></sup></a></th> <th rowspan="1" colspan="1">ω<i><sub>i</sub></i><a href="#T3F5"><sup><i>e</i></sup></a></th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1"><i>ST</i> (%)</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">    G, −4.55 + 10.30 · <i>VA<sub>I</sub></i> (53) + 3.75 · <i>SM</i> (41) − 1.81 · <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i> (2) − 4.19 · <i>V</i> (1) + 2.10 · <i>VA<sub>R</sub></i> (1) + 4.80 · <i>G</i><sub>0</sub><i><sub>i</sub></i>/<i>G</i><sub>0</sub><i><sub>m</sub></i> (1) − 2.88 · <i>G</i><sub>24</sub><i><sub>i</sub></i>/G<sub>24</sub><i><sub>m</sub></i> (1)</td> <td rowspan="1" colspan="1">0.91*</td> <td rowspan="1" colspan="1">−226.43</td> <td rowspan="1" colspan="1">472.86</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0.48</td> </tr> <tr> <td rowspan="1" colspan="1">    T, −13.59 + 1.49 · <i>VA<sub>I</sub></i> (53) + 4.58 · <i>SM</i> (24) − 1.98 · <i>V</i> (14) − 1.49 · <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i> (6) + 12.62 · <i>G</i><sub>0</sub><i><sub>i</sub></i>/<i>G</i><sub>0</sub><i><sub>m</sub></i> (2) − 0.88 · <i>G</i><sub>24</sub><i><sub>i</sub></i>/G<sub>24</sub><i><sub>m</sub></i> (1)</td> <td rowspan="1" colspan="1">0.99*</td> <td rowspan="1" colspan="1">−101.73</td> <td rowspan="1" colspan="1">219.45</td> <td rowspan="1" colspan="1">8</td> <td rowspan="1" colspan="1">0.20</td> </tr> <tr> <td rowspan="1" colspan="1">    C, −1.13 + 1.09 · <i>VA<sub>I</sub></i> (37) + 3.19 · <i>SM</i> (37) − 0.75 · <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i> (18) + 0.60 · <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i> (8)</td> <td rowspan="1" colspan="1">0.71*</td> <td rowspan="1" colspan="1">−85.14</td> <td rowspan="1" colspan="1">182.27</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">0.33</td> </tr> <tr> <td colspan="6" rowspan="1" /> </tr> <tr> <td rowspan="1" colspan="1"><i>IS</i> (log <i>n</i>)<a href="#T3F6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">    G, 0.95 + 0.06 · <i>ST</i> (41) + 0.24 · <i>SN<sub>i</sub></i> (23) − 0.59 · <i>V</i> (20) + 0.18 · <i>VA<sub>I</sub></i> (10) − 0.42 · <i>RW<sub>i</sub></i>/<i>RW<sub>m</sub></i> (4) − 0.39 · <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i> (2)</td> <td rowspan="1" colspan="1">0.78*</td> <td rowspan="1" colspan="1">−74.49</td> <td rowspan="1" colspan="1">168.98</td> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">0.25</td> </tr> <tr> <td rowspan="1" colspan="1">    T, −2.59 + 0.05 · <i>ST</i> (45) + 0.97<i>SN<sub>i</sub></i> (20) − 1.18 · <i>V</i> (15) + 1.17 · <i>SM</i> (7) + 0.21 · <i>VA<sub>I</sub></i> (7) − 1.35 · <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i> (4) − 0.48 · <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i> (2)</td> <td rowspan="1" colspan="1">0.81*</td> <td rowspan="1" colspan="1">−32.01</td> <td rowspan="1" colspan="1">86.23</td> <td rowspan="1" colspan="1">9</td> <td rowspan="1" colspan="1">0.18</td> </tr> <tr> <td rowspan="1" colspan="1">    C, 0.45 + 0.12 · <i>ST</i> (25) − 0.36 · <i>V</i> (24) + 1.90 · 10<sup>−5</sup> · <i>SN<sub>i</sub></i> (20) + 1.90 · <i>SM</i> (13) 0.11 · <i>VA<sub>I</sub></i> (10) − 1.22 · <i>RW<sub>i</sub></i>/<i>RW<sub>m</sub></i> (4) + 1.08 · <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i> (4)</td> <td rowspan="1" colspan="1">0.78*</td> <td rowspan="1" colspan="1">−39.37</td> <td rowspan="1" colspan="1">96.74</td> <td rowspan="1" colspan="1">15</td> <td rowspan="1" colspan="1">0.13</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">a</span><p>The relative importance (percent) of each estimator variable is shown in parentheses. G, global model; T, TuMV-specific model; C, CMV-specific model.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">b</span><p>Conditional correlation coefficient. Asterisks indicate significant correlations (<i>P</i> &amp;lt; 0.01).</p> </div> <div class="fn-type-abbr" title="abbr"><span class="label" tagx="label" title="label">c</span><p>AIC, Akaike’s information criterion.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">d</span><p>Number of models closely competing with the best-ranked model (Δ<i><sub>i</sub></i> of &amp;lt;2 out of 511 for <i>ST</i> and 2,047 for <i>IS</i> models tested). Δ<i><sub>i</sub></i> is the difference between the AIC of a given model and that of the best-ranked model and quantifies how models compete (for the best-ranked model, Δ<i><sub>i</sub></i> = 0; for substantial empirical support, Δ<i><sub>i</sub></i> = 1 to 2; for considerably less support, Δ<i><sub>i</sub></i> = 2 to 7; for no support, Δ<i><sub>i</sub></i> &amp;gt; 10) (<a href="#B68">68</a>).</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">e</span><p>AIC model weight as ω<i><sub>i</sub></i> = exp(−0.5Δ<i><sub>i</sub></i>)/Σexp(−0.5Δ<i><sub>i</sub></i>). The larger the ω value, the greater the likelihood of the model relative to the competing models. The maximum ω<i><sub>i</sub></i> is 1.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">f</span><p>The number of infected seeds (<i>IS</i>) was normalized using a logarithmic transformation, and the resulting values were used for model construction.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">g</span><p>Model structures for <i>ST</i> included the effect of virus infection on rosette (<i>RW<sub>i</sub></i>/<i>RW<sub>m</sub></i>) and inflorescence (<i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i>) weights and on short-term (<i>G</i><sub>0</sub><i><sub>i</sub></i>/<i>G</i><sub>0</sub><i><sub>m</sub></i>), medium-term (<i>G</i><sub>24</sub><i><sub>i</sub></i>/<i>G</i><sub>24</sub><i><sub>m</sub></i>), and long-term (<i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i>) seed survival; virulence (<i>V</i>); virus within-host speed of movement (<i>SM</i>); and virus accumulation in rosette (<i>VA<sub>R</sub></i>) and inflorescence (<i>VA<sub>I</sub></i>) leaves. Model structures for <i>IS</i> also included <i>ST</i> and the total log number of seeds produced per plant (<i>SN</i>). Best-ranked models are shown.</p> </div> </div> </div> <p>In order to identify infection traits explaining virus seed transmission at large, we first constructed global multivariate models by merging the data from plants infected by both TuMV and CMV (see Materials and Methods). The best-ranked model contained <i>SM</i>, <i>V</i>, <i>VA<sub>R</sub></i>, <i>VA<sub>I</sub></i>, and the effect of infection on <i>IW</i>, <i>G</i><sub>0</sub>, and <i>G</i><sub>24</sub> (<i>R</i><sup>2</sup> = 0.91; <i>P</i> &amp;lt; 0.001). <i>VA<sub>I</sub></i> and <i>SM</i> had the highest relative importance (53% and 41%, respectively), explaining most of the variance in <i>ST</i> (<a href="#T3">Table 3</a>). Because additional and/or different infection traits could determine seed transmission in a virus species-specific manner, we also constructed multivariate models for data on seed transmission for each virus separately. The model best explaining JPN1-TuMV <i>ST</i> included <i>SM</i>, <i>VA<sub>I</sub></i>, <i>V</i>, and the effect of infection on <i>IW</i>, <i>G</i><sub>0</sub>, and <i>G</i><sub>24</sub> (<i>R</i><sup>2</sup> = 0.99; <i>P</i> &amp;lt; 0.001). Again, <i>VA<sub>I</sub></i> and <i>SM</i> were the most important estimators (relative importance, 53% and 24%, respectively), with virulence playing a less relevant role (relative importance, 14%) (<a href="#T3">Table 3</a>). In addition, the best-ranked model explaining Fny-CMV <i>ST</i> included <i>SM</i>, <i>VA<sub>I</sub></i>, and the effect of infection on <i>IW</i> and <i>G</i><sub>48</sub> (<i>R</i><sup>2</sup> = 0.71; <i>P</i> &amp;lt; 0.001), with <i>SM</i> and <i>VA<sub>I</sub></i> being the most important estimators (both having a relative importance of 37%) and <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i> having lower relative importance (18%). Thus, our results indicated that <i>VA<sub>I</sub></i> and <i>SM</i> were chief estimators of <i>ST</i>, with <i>V</i> and <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i> playing secondary roles in a virus-specific manner (<a href="#T3">Table 3</a>). Regardless of the utilized data set, bivariate analyses indicated a positive association between <i>ST</i> and the main infection traits identified in our multivariate models: <i>SM</i> (<i>R</i><sup>2</sup> ≥ 0.30; <i>P</i> &amp;lt; 0.001) and <i>VA<sub>I</sub></i> (<i>R</i><sup>2</sup> ≥ 0.37; <i>P</i> &amp;lt; 0.001) (<a href="#F2">Fig. 2</a>). A weaker but still significant negative relationship between <i>ST</i> and the virus-specific secondary estimators (<i>V</i> and <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i>) was also detected (<i>R</i><sup>2</sup> ≥ 0.19; <i>P</i> ≤ 0.002) (<a href="#F2">Fig. 2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 2</span></a></div>   <p>Bivariate relationships between percent virus seed transmission and infection traits. Regressions considering data for both viruses together (A and B), only JPN1-TuMV (C to E), and only Fny-CMV (F to H) are shown. Linear relationships of percent virus seed transmission and speed of virus movement in centimeters per day (purple), virus multiplication in the inflorescence in nanograms of viral RNA per microgram of total RNA (orange), virulence as 1 − (<i>SW<sub>i</sub>/SW<sub>m</sub></i>) (light blue), and long-term seed survival as <i>G</i><sub>48</sub><i><sub>i</sub></i>/<i>G</i><sub>48</sub><i><sub>m</sub></i> (green) are represented.</p>   </div> <p>The best-ranked global model explaining <i>IS</i> contained <i>ST</i>, <i>SN<sub>i</sub></i>, <i>V</i>, <i>VA<sub>I</sub></i>, and the effect of infection on <i>RW</i> and <i>G</i><sub>48</sub> (<i>R</i><sup>2</sup> = 0.78; <i>P</i> &amp;lt; 0.001). In this case, <i>ST</i>, <i>SN<sub>i</sub></i>, and <i>V</i> had the highest relative importance (41%, 23%, and 20%, respectively) (<a href="#T3">Table 3</a>). The best model explaining JPN1-TuMV <i>IS</i> included the same estimators as the ones for the best global model but replacing <i>RW<sub>i</sub></i>/<i>RW<sub>m</sub></i> by <i>IW<sub>i</sub></i>/<i>IW<sub>m</sub></i> and adding <i>SM</i> (<i>R</i><sup>2</sup> = 0.81; <i>P</i> &amp;lt; 0.001). Again, <i>ST</i>, <i>SN<sub>i</sub></i>, and <i>V</i> were the most important estimators (relative importance, 45%, 20%, and 15%, respectively). The best-ranked model explaining Fny-CMV <i>IS</i> included <i>ST</i>, <i>V</i>, <i>SN<sub>i</sub></i>, <i>SM</i>, <i>VA<sub>I</sub></i>, and the effect of infection on <i>RW</i> and <i>IW</i> (<i>R</i><sup>2</sup> = 0.78; <i>P</i> &amp;lt; 0.001), with <i>ST</i>, <i>V</i>, and <i>SN<sub>i</sub></i> being the most important estimators (relative importance, 25%, 24%, and 20%, respectively) (<a href="#T3">Table 3</a>). Bivariate analyses indicated that <i>IS</i> was positively associated with <i>ST</i> (<i>R</i><sup>2</sup> ≥ 0.34; <i>P</i> &amp;lt; 0.001) and with <i>SN<sub>i</sub></i> (<i>R</i><sup>2</sup> ≥ 0.31; <i>P</i> &amp;lt; 0.001) and negatively associated with <i>V</i> (<i>R</i><sup>2</sup> ≥ 0.38; <i>P</i> ≤ 0.001) (<a href="#F3">Fig. 3</a>). These results suggested that <i>ST</i> in general could be negatively associated with virus virulence and positively associated with the progeny production of infected plants, expanding the information provided by the <i>ST</i>-explaining models. Thus, we analyzed the association between these traits (<a href="#F4">Fig. 4</a>). <i>ST</i> was positively associated with <i>SN<sub>i</sub></i> when both viruses were considered either together or separately (<i>R</i><sup>2</sup> ≥ 0.14; <i>P</i> ≤ 0.027). In agreement with <i>ST</i>-explaining models, a weak negative association between virulence and <i>ST</i> was detected in the virus-specific models (<i>R</i><sup>2</sup> ≥ 0.19; <i>P</i> &amp;lt; 0.001) but not in the global model (<i>R</i><sup>2</sup> = 0.15; <i>P</i> = 0.138) (<a href="#F4">Fig. 4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 3</span></a></div>   <p>Bivariate relationships between the number of virus-infected seeds and infection traits. Regressions considering data for both viruses together (A to C), only JPN1-TuMV (D to F), and only Fny-CMV (G to I) are shown. Linear relationships of the log number of virus-infected seeds and percent virus seed transmission (brown), virulence as 1 − (<i>SW<sub>i</sub>/SW<sub>m</sub></i>) (light blue), virus multiplication in the inflorescence in nanograms of viral RNA per microgram of total RNA (orange), and log number of seeds per infected plant (dark blue) are represented.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 4</span></a></div>   <p>Bivariate relationships between the main predictors of the number of virus-infected seeds. Regressions considering data for both viruses together (A and B), only JPN1-TuMV (C and D), and only Fny-CMV (E and F) are shown. Linear relationships of percent infected seeds and log number of seeds per infected plant (dark blue) and of percent infected seeds and virulence as 1 − (<i>SW<sub>i</sub></i>/<i>SW<sub>m</sub></i>) (light blue) are represented.</p>   </div> </div> <div class="modelaccuracy." title="sec"> <div class="title" tagx="title" title="title">
Model accuracy.</div> <p>It could be argued that our multivariate models are based on data from six <i>Arabidopsis</i> accessions and one isolate per virus inoculated at a given host phenological stage, and therefore, the derived results are specific only for these particular plant-virus genotype-per-genotype interactions and inoculation conditions. We analyzed the applicability of our models to other <i>Arabidopsis</i> accessions and TuMV/CMV isolates by using them to estimate the efficiency of seed transmission in the set of 18 <i>Arabidopsis</i> accessions and 5 virus isolates inoculated at the same phenological stage as the 6 accessions used for model construction. To do so, we quantified the relevant infection traits identified by the models (see Data Set S1 in the supplemental material), and we incorporated these values into the best-ranked global and virus-specific models to predict percentages and numbers of virus-infected seeds. The resulting values were compared with those obtained by experimental testing of seed infection (<a href="#F5">Fig. 5</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 5</span></a></div>   <p>Association between experimental and estimated virus seed transmission of 5 TuMV and CMV isolates in 18 <i>Arabidopsis</i> accessions. Correlations between estimated values of percent TuMV seed transmission derived from the TuMV-specific model (A) and from the global model (B), percent CMV seed transmission derived from the CMV-specific model (C) and from the global model (D), the log number of TuMV-infected seeds derived from the TuMV-specific models (E) and from the global model (F), and the log number of CMV-infected seeds derived from the CMV-specific models (G) and from the global model (H) and the corresponding experimental values are shown. Data for UK1-TuMV (gray), JPN1-TuMV (black), LS-CMV (green), Fny-CMV (blue), and De72-CMV (red) are represented. Note the different scale of each panel.</p>   </div> <p>Bivariate analyses of the relationship between TuMV and CMV experimental and predicted percentages of seed transmission indicated a significant positive correlation between both variables using either the global model or the virus-specific ones (<i>R</i><sup>2</sup> ≥ 0.70; <i>P</i> &amp;lt; 0.001) (<a href="#F5">Fig. 5A</a> to <a href="#F5">D</a>). However, prediction accuracy depended on the model. The equations of the linear correlations indicated that virus-specific models accurately predicted percentages of CMV and TuMV seed transmission (<i>x</i> coefficients near 1) (<a href="#F5">Fig. 5A</a> and <a href="#F5">C</a>). Similar results were obtained when predic tions for each isolate were analyzed separately (<a href="#F6">Fig. 6</a>). On the other hand, the global model underestimated TuMV <i>ST</i> (<i>x</i> coefficient = 3.63) and overestimated CMV <i>ST</i> (<i>x</i> coefficient = 0.10), particularly at seed transmission values of &amp;gt;10% (<a href="#F5">Fig. 5B</a> and <a href="#F5">D</a>). Virus isolate-specific comparisons yielded similar biases (<a href="#F6">Fig. 6</a>). Experimental and predicted numbers of infected seeds per plant were also significantly correlated for both global and virus-specific models (<i>R</i><sup>2</sup> ≥ 0.50; <i>P</i> &amp;lt; 0.001), although <i>R</i><sup>2</sup> values were generally lower than those for percent seed transmission (<a href="#F5">Fig. 5E</a> to <a href="#F5">H</a>). Again, virus-specific models performed better (<i>x</i> coefficient near 1) than the global one, but under- and overestimates of TuMV and CMV <i>IS</i>, respectively, were smaller than in the case of <i>ST</i>-estimating models (<a href="#F5">Fig. 5</a>). Note that prediction of the absence of infected seeds was poorer in <i>IS</i> than in <i>ST</i> models. Similar results were obtained when data for each virus isolate were analyzed separately (<a href="#F7">Fig. 7</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 6</span></a></div>   <p>Association between experimental and estimated virus seed transmission of the 2 TuMV and 3 CMV isolates in 18 <i>Arabidopsis</i> accessions. (A to D) Correlations of percent TuMV seed transmission derived from the TuMV-specific model (A and B) and from the global model (C and D) to the corresponding experimental values. (E to J) Correlations of percent CMV seed transmission derived from the CMV-specific model (E to G) and from the global model (H to J) to the corresponding experimental values. Data for JPN1-TuMV (black), UK1-TuMV (gray), LS-CMV (green), Fny-CMV (blue), and De72-CMV (red) are represented.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 7</span></a></div>   <p>Association between experimental and estimated virus seed transmission of the 2 TuMV and 3 CMV isolates in 18 <i>Arabidopsis</i> accessions. (A to D) Correlations of the number of TuMV-infected seeds derived from the TuMV-specific model (A and B) and from the global model (C and D) to the corresponding experimental values. (E to J) Correlations of the number of CMV-infected seeds derived from the CMV-specific model (E to G) and from the global model (H to J) to the corresponding experimental values. Data for JPN1-TuMV (black), UK1-TuMV (gray), LS-CMV (green), Fny-CMV (blue), and De72-CMV (red) are represented.</p>   </div> <p>Hence, for the host phenological stage at inoculation used in our experiments, our multivariate models fairly estimate trends of TuMV and CMV seed transmission across <i>Arabidopsis</i> accessions, i.e., high versus low seed transmission, and virus-specific models infer seed transmission values more accurately than the global ones.</p> </div> </div> <div class="discusion" title="discusion"> <div class="title" tagx="title" title="title">
DISCUSSION</div> <p>The ability to be transmitted is a key component of parasite fitness (<a href="#B2">2</a>, <a href="#B3">3</a>, <a href="#B5">5</a>, <a href="#B9">9</a>, <a href="#B10">10</a>). Although vertical transmission from parents to offspring is not infrequent among plant parasites and plays a central role in their epidemiology, little is known about the mechanistic basis of this mode of transmission, particularly in plant viruses (<a href="#B14">14</a>, <a href="#B36">36</a>). Here, we tested the hypothesis that virus transmission through seeds is associated with (i) the ability of the virus to move from the entry points to the plant reproductive structures, (ii) the ability of the virus to invade these reproductive structures, (iii) the capacity of the infected plant to produce progeny, and (iv) the capacity of infected progeny to survive (<a href="#B20">20</a>, <a href="#B36">36</a>).</p> <p>The efficiency of TuMV and CMV seed transmission depended on the <i>Arabidopsis</i> accession-virus species/isolate interaction, indicating that traits controlled by the host and/or the parasite determine this process. Global multivariate models identified the virus speed of within-host movement and its level of multiplication in the plant inflorescence as the best estimators of percent seed transmission. These variables can be considered proxies for the ability of the virus to reach and invade gametic tissues, respectively (<a href="#B11">11</a>). Bivariate analyses indicated that the faster the virus spread across the reproductive structures, the higher its efficiency of seed transmission. Seed transmission requires reaching the plant reproductive structures during a particular window of time that in general is quite narrow: the embryo is accessible after fertilization and before the suspensor’s programmed cell death, and pollen and ovule invasion must occur prior to fertilization (<a href="#B11">11</a>, <a href="#B23">23</a>). These periods last only a few days in <i>Arabidopsis</i> (<a href="#B37">37</a>, <a href="#B38">38</a>). Hence, faster within-host movement would allow the virus to reach the reproductive organs within the required time frame in a larger number of flowers/siliques. This is compatible with experimental analyses showing that, in general, earlier virus infection leads to a higher efficiency of seed transmission (<a href="#B11">11</a>, <a href="#B14">14</a>). Reaching the reproductive organs at the right moment does not guarantee seed transmission, as the virus also needs to gain entry into the seed (<a href="#B20">20</a>). It has been proposed that a higher level virus multiplication in the plant reproductive structures favors embryo/gametophyte invasion by promoting virus crossing of the boundary between the maternal and progeny tissues (<a href="#B11">11</a>, <a href="#B39">39</a>). In line with this prediction, our results indicate that a higher level of virus multiplication in the inflorescence increases the percentage of infected seeds. This observation also agrees with previous work showing that a higher percentage of seed transmission correlates with high virus titers in flowers, ovules, and/or pollen (<a href="#B19">19</a>, <a href="#B26">26</a>, <a href="#B40">40</a>). Moreover, the only host genes identified as genetic determinants of plant virus seed transmission are involved in small RNA-mediated gene silencing, which modulates virus multiplication (<a href="#B24">24</a>). Note that in our experiments, plants were inoculated at an early developmental stage. Virus inoculation of older plants would reduce the number of flowers/siliques that the virus can reach during the appropriate window of time, especially for the most basal ones (<a href="#B14">14</a>, <a href="#B20">20</a>). In this context, the speed of within-host movement would likely become critical for seed transmission, increasing its relative importance in the multivariate models.</p> <p>Virus-specific multivariate models confirmed that the speed of within-host movement and the level of multiplication in the inflorescence were major determinants of virus seed transmission. Interestingly, these models identified other traits as minor estimators of virus vertical transmission, which differed between TuMV and CMV. For TuMV, the combination of the two above-mentioned factors and virus virulence explained 91% of the variation in percent seed transmission. TuMV is regarded as a sterilizing parasite because it frequently prevents seed production in a number of <i>Arabidopsis</i> accessions, in which no vertical transmission is attained (<a href="#B41">41</a>, <a href="#B42">42</a>). Hence, the identification of TuMV virulence as being negatively associated with seed transmission efficiency likely reflects that it determines the presence/absence of vertical transmission rather than having a role in the mechanism of this process. For CMV, the third trait (negatively) associated with the efficiency of vertical transmission was the long-term survival of infected seeds. Note that seed aging is posterior to our measures of the efficiency of vertical transmission. Thus, a lower long-term seed survival rate is likely a consequence of infection and suggests that the presence of the virus in the seed reduces its viability. We did not determine whether the seeds dying during the 48-h accelerated-aging treatment were mostly those harboring the virus. However, bivariate analyses using averaged values of the six accessions indicated that the percentage of CMV seed transmission explained 66% of the variance in the percentage of dead aged seeds. Hence, our results suggest that long-term seed survival could be a modifier of the efficiency of seed transmission in the long run. Alternatively, a lower long-term survival rate of seeds from infected plants could reflect maternal effects, but we did not find a significant difference in the weights of single seeds between infected and mock-inoculated plants, which argues against this possibility. The negative relationship between the efficiency of CMV seed transmission and long-term seed survival is in apparent contradiction with experimental analyses showing that infection of <i>Arabidopsis</i> plants by Fny-CMV renders seeds with improved tolerance to deterioration (<a href="#B35">35</a>). However, those authors used Col-0 and a 24-h accelerated-aging treatment, an accession not included in our work and conditions for which we showed that the effect of Fny-CMV infection on seed survival significantly varies between accessions.</p> <p>It is worth noting that we identified the same determinants of percent vertical transmission in two virus species with very different life histories and outcomes of infection. This suggests that our results would be applicable to the prediction of virus seed transmission in other host-virus interactions. Indeed, although some of the parameters used for model validation were estimates, both global and virus-specific ones predicted trends of seed transmission (i.e., higher versus lower transmission rates) in other <i>Arabidopsis</i> accessions and TuMV/CMV isolates with medium to high accuracy. More-accurate prediction of seed transmission values required the use of virus-specific models, which indicates that fine-tuning of the model estimative power needs to include virus-specific secondary determinants of seed transmission. Whether our models are applicable to other viruses or host species should be explored further. In any case, at least for TuMV and CMV, our results will help map the host and virus genes controlling the infection traits associated with the efficiency of seed transmission. This will allow the identification of candidate genetic determinants of this process, which currently remain elusive.</p> <p>More than one-quarter of all known plant viruses have been reported to be vertically transmitted (<a href="#B11">11</a>, <a href="#B12">12</a>), and this is likely an underestimate, as every year, more species, either long known or newly discovered, are described to be transmitted through seeds. CMV vertical transmission has been extensively reported (<a href="#B14">14</a>, <a href="#B43">43</a>), whereas to date, TuMV has been considered strictly horizontally transmitted through insect vectors (<a href="#B11">11</a>, <a href="#B44">44</a>). Our results indicate that in <i>Arabidopsis</i>, TuMV is transmitted through seeds and in some accessions with high efficiency, adding this to the list of vertically transmitted plant viruses. As mentioned above, in <i>Arabidopsis</i>, TuMV is considered a sterilizing virus (<a href="#B41">41</a>, <a href="#B42">42</a>). Host sterilization allows host resources to be diverted from reproduction to survival, which increases the infectious period and the level of parasite multiplication. These modifications maximize horizontal transmission but at the cost of no vertical one (<a href="#B45">45</a><a href="#B46">–</a><a href="#B47">47</a>). Hence, host sterilization is thought to be selectively advantageous for parasites that have strict horizontal transmission. However, our results show that TuMV maintains a mixed mode of transmission. Similarly, the fungus <span class="named-content" title="named-content">Atkinsonella hypoxylon</span> has both modes of transmission in species of the genus <i>Danthonia</i> despite inducing plant sterility (<a href="#B48">48</a>, <a href="#B49">49</a>). In this pathosystem, the coexistence of vertical and horizontal transmission has been explained on the basis of a vertical transmission-virulence trade-off. That is, in plants with most flowers sterilized, the few that are fertile produce a higher proportion of infected seeds than plants with fewer sterilized flowers (<a href="#B48">48</a>, <a href="#B49">49</a>). Thus, higher virulence favors vertical transmission. This is not the case for TuMV, as the most virulent isolate (UK1-TuMV) had a seed transmission efficiency similar to that of a less virulent one (JPN1-TuMV). More importantly, the percentage of JPN1-TuMV seed transmission was negatively correlated with virulence and positively correlated with the number of seeds produced per infected plant, indicating no vertical transmission-virulence trade-off (<a href="#F4">Fig. 4</a>). On the other hand, tolerance to TuMV infection may explain why seed transmission is maintained in the virus population: One-third of the 18 <i>Arabidopsis</i> accessions analyzed here avoided UK1-TuMV sterilization, and in all of them, the virus was seed transmitted. In these accessions, tolerance is attained by shortening the host growth period, which triggers plant reproduction before the plant experiences the full cost of infection (Montes et al., submitted). This reduces the resources available for virus multiplication, which in turn prevents maximization of horizontal transmission (<a href="#B45">45</a>). Thus, TuMV seed transmission may compensate for the loss of virus fitness due to suboptimal horizontal transmission in tolerant accessions.</p> <p>Our global and virus species-specific models explaining the total number of infected seeds, which best reflects the contribution of vertical transmission to virus fitness (<a href="#B2">2</a>, <a href="#B3">3</a>, <a href="#B5">5</a>, <a href="#B9">9</a>, <a href="#B10">10</a>), support the link between virulence/tolerance and seed transmission. Indeed, these models identified virulence as one of the most important estimators, which was negatively associated with the number of infected seeds per plant. These results are therefore compatible with theoretical elaborations on parasite evolution under mixed modes of transmission (see the introduction).</p> <p>Multivariate models also detected percent of seed transmission and the number of seeds produced by infected plants as the main estimators of the number of infected seeds, with both estimators being positively associated with this trait. The six <i>Arabidopsis</i> accessions utilized to build multivariate models were selected to represent plants with short and long life cycles (<a href="#T1">Table 1</a>). In the absence of infection, short-lived accessions generally produce more seeds than long-lived ones (<a href="#B50">50</a>, <a href="#B51">51</a>). Upon infection by JPN1-TuMV, these short-lived accessions showed a smaller reduction in seed production and a higher percentage of seed transmission than long-lived ones, which would explain the positive association between the number of infected seeds and the three main estimators of this trait. On the other hand, the efficiency of Fny-CMV seed transmission was higher in long-lived accessions. These accessions are known to display higher tolerance to CMV than short-lived ones (<a href="#B50">50</a>, <a href="#B51">51</a>). Under our conditions, this allowed long-lived accessions to produce a larger number of seeds upon CMV infection than short-lived ones (<a href="#T2">Table 2</a>). As a consequence, again, a higher level of seed production upon Fny-CMV infection was positively associated with the percentage of seed transmission and, by extension, with the number of infected seeds. Hence, the role of percent seed transmission and the number of seeds produced by infected plants as major estimators of the total number of infected seeds can be explained as a combination of plant allometry and tolerance to virus infection.</p> <p>Not surprisingly, models generally included the speed of within-host movement of the virus and its multiplication in the inflorescence as secondary determinants of the number of infected seeds, the two traits associated with percent seed transmission. This is likely the consequence of the major contribution of the percentage of seed transmission to the number of infected seeds. Indeed, equivalent models for <i>IS</i> constructed with the same estimators as those used for <i>ST</i> identified <i>VA<sub>I</sub></i>, <i>SM</i>, and <i>V</i> as major determinants (not shown). These results highlight the complexity of the interactions between different infection traits in determining the contribution of vertical transmission to virus fitness.</p> <p>In summary, by using a multivariate approach, this work provides a highly detailed analysis of the infection traits linked to the efficiency of plant virus seed transmission and identify virus within-host speed of movement and multiplication in the plant reproductive organs as major determinants of this process. We also show that a greater contribution of vertical transmission to virus fitness is associated with lower virus virulence. These results support theoretical predictions and contribute to shedding light on the mechanisms by which plant viruses achieve vertical transmission and optimize their fitness.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <div class="arabidopsisaccessionsandvirusisolates." title="sec"> <div class="title" tagx="title" title="title"><i>Arabidopsis</i> accessions and virus isolates.</div> <p>Virus isolates UK1-TuMV (GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB194802">AB194802</a>), JPN1-TuMV (GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/KM094174">KM094174</a>), Fny-CMV (GenBank accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_002034">NC_002034</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_002035">NC_002035</a> and <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001440">NC_001440</a>), LS-CMV (GenBank accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/AF416899">AF416899</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/AF416900">AF416900</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/AF127976">AF127976</a>), and De72-CMV (not sequenced) were used. JPN1-TuMV was obtained from a field-infected <span class="named-content" title="named-content">Raphanus sativus</span> (Brassicaceae) plant (<a href="#B52">52</a>), and De72-CMV was obtained from a field-infected <span class="named-content" title="named-content">Diplotaxis erucoides</span> L. plant (Brassicaceae) (<a href="#B53">53</a>); both viruses were propagated in <span class="named-content" title="named-content">Nicotiana benthamiana</span> plants. UK1-TuMV, Fny-CMV, and LS-CMV were derived from biologically active clones (<a href="#B54">54</a><a href="#B55">–</a><a href="#B56">56</a>) by <i>in vitro</i> transcription with T7 RNA polymerase (New England Biolabs, Ipswich, MA, USA), and transcripts were used to infect <span class="named-content" title="named-content">N. benthamiana</span> plants for virus multiplication.</p> <p>Eighteen <i>Arabidopsis</i> accessions were used (<a href="#T1">Table 1</a>). Ten accessions represented the Eurasian geographic distribution of the species, and the remaining eight represented its distribution in the Iberian Peninsula, a Pleistocene glacial refuge for <i>Arabidopsis</i> (<a href="#B57">57</a>). Plant seeds were stratified for 7 days at 4°C in pots with a diameter of 15 cm, in a 0.43-liter volume containing a 3:1 peat-vermiculite mix. Afterwards, pots were moved for seed germination and plant growth to a greenhouse at 22°C, under 16 h of light (intensity,120 to 150 mol s/m<sup>2</sup>). Plants were mechanically inoculated, either with <span class="named-content" title="named-content">N. benthamiana</span> TuMV- and CMV-infected tissue ground in a solution containing 0.1 M Na<sub>2</sub>HPO<sub>4</sub>, 0.5 M NaH<sub>2</sub>PO<sub>4</sub>, and 0.02% DIECA (0.01 M phosphate buffer [pH 7.0], 0.2% sodium diethyldithiocarbamate) or with inoculation buffer for mock-inoculated plants. Inoculations were done when plants were at developmental stages 1.05 to 1.06 (<a href="#B58">58</a>). After inoculation, all individuals were randomized in the greenhouse.</p> </div> <div class="experimentaldesign." title="sec"> <div class="title" tagx="title" title="title">
Experimental design.</div> <p>The 18 <i>Arabidopsis</i> accessions were inoculated with the 5 virus isolates as described above, with 10 replicates per treatment and accession. In these plants, virus multiplication, virulence, short-term seed survival, and efficiency of seed transmission were quantified as described below. Using these data, 6 out of the 18 <i>Arabidopsis</i> accessions were selected (An-1, Bay-0, Cad-0, Cum-0, Ll-0, and Fei-0) such that (i) accessions represented a range of TuMV and CMV seed transmissions (<a href="#F1">Fig. 1</a>) and (ii) accessions with different life cycles were included (<a href="#T1">Table 1</a>). We also selected one CMV isolate (Fny-CMV) and one TuMV isolate (JPN1-TuMV) for further experiments because they showed a wide range of percentages of seed transmission across accessions and were transmitted in a larger number of accessions.</p> <p>Using the six <i>Arabidopsis</i> accessions and the two virus isolates, we conducted a time course experiment of viral infection. For each <i>Arabidopsis</i> accession, 85 plants per virus were inoculated with JPN1-TuMV and Fny-CMV each, and the other 10 were mock inoculated. Five infected plants per accession were harvested at regular intervals. Because each accession has a different developmental schedule (<a href="#B50">50</a>, <a href="#B51">51</a>), intervals were established such that samples were collected from plant inoculation to silique ripening, and data from at least 15 time points were obtained. For each harvested plant, the amount of virus in the rosette and in 1-cm pieces of the inflorescence, which included inflorescence leaves, flowers, and siliques, if present, was quantified. These measures were used to calculate the speed of virus within-host movement. In parallel, 10 infected plants plus the mock-inoculated controls were allowed to complete their life cycle. In these plants, virus multiplication in the rosette and the inflorescence and rosette, inflorescence, and seed weights were obtained (see below). Seeds from these plants were used to estimate seed transmission rates and short-, medium-, and long-term seed survival. Note that the speed of virus within-host movement was measured through destructive sampling, whereas the other infection traits were quantified in the set of plants that completed their life cycle. Thus, for model building, the speed of virus within-host movement was considered an accession-specific trait (the averaged value derived from the destructive sampling was attributed to all plants of the same accession), whereas the other infection traits were considered plant-specific traits. Using this data set, we constructed global multivariate models that jointly considered all infection traits measured for both viruses as predictors of the efficiency of seed transmission as well as virus-specific models where infection traits were considered for each virus separately (see "Statistical analysis,�? below). In this way, we could differentiate infection traits broadly associated with seed transmission from virus-specific determinants of this process.</p> <p>To validate the accuracy of the constructed models, we went back to the set of 18 <i>Arabidopsis</i> accessions, retrieved the values of the parameters identified by our models as being relevant to predicting the efficiency of virus seed transmission, and interpolated these values in the constructed models. This allowed predicted values of seed transmission to be obtained, which were then compared with the values obtained experimentally. This approach allowed analysis of whether our models could be extrapolated to plant-virus interactions other than those utilized to build them and testing of their general accuracy.</p> </div> <div class="virusmultiplication." title="sec"> <div class="title" tagx="title" title="title">
Virus multiplication.</div> <p>TuMV and CMV multiplication was quantified as viral RNA accumulation via reverse transcription-quantitative PCR (qRT-PCR) for each individual plant. For plants included in the time course experiment, at each time point, virus accumulation in the rosette (<i>VA<sub>R</sub></i>) was quantified from three disks with a diameter of 4 mm collected from different systemically infected leaves, and virus accumulation in the inflorescence (<i>VA<sub>I</sub></i>) was quantified from the collected 1-cm pieces. For plants that were allowed to complete their life cycle, <i>VA<sub>R</sub></i> and <i>VA<sub>I</sub></i> were quantified at the end of the flowering period to ensure maximum viral multiplication in plant structures. Form these plant samples, total RNA extracts were obtained using TRIzol reagent (Life Technologies, Carlsbad, CA, USA), and 10 ng of total RNA was added to Brilliant III Ultra-Fast SYBR green qRT-PCR master mix (Agilent Technologies, Santa Clara, CA, USA) according to the manufacturer’s recommendations. Specific primers were used to amplify a 70-nucleotide (nt) fragment of the TuMV coat protein (CP) gene and a 106-nt fragment of the CMV CP gene (<a href="#B59">59</a>, <a href="#B60">60</a>). Each plant sample was assayed in duplicate on a LightCycler 480 II real-time PCR system (Roche, Indianapolis, IN, USA). Absolute viral RNA accumulation was quantified as nanograms of viral RNA per microgram of total RNA, utilizing internal standards. For TuMV, internal standards consisted in 10-fold dilution series of plasmid-derived RNA transcripts of the same 70-nt CP fragment from UK1-TuMV. For CMV, 10-fold dilution series were prepared using purified viral RNA. All internal standards ranged from 2 × 10<sup>−3</sup> ng to 2 × 10<sup>−7</sup> ng.</p> </div> <div class="virusspeedofwithin-hostmovement." title="sec"> <div class="title" tagx="title" title="title">
Virus speed of within-host movement.</div> <p>The speed of virus within-plant movement (<i>SM</i>) in the plant inflorescence was quantified as centimeters per day from flower meristem formation to silique ripening, thus covering the time interval during which the virus can enter the seed. Following methods described previously (<a href="#B61">61</a>, <a href="#B62">62</a>), at the time points defined in "Experimental design�? above, the presence/absence of the virus in the 1-cm inflorescence pieces was monitored. Virus was detected via qRT-PCR as described above. As a result, we obtained a matrix of inflorescence height (in centimeters) versus time postbolting (days), in which we incorporated data on virus presence/absence for each height-time pair (see Table 3 in reference <a href="#B62">62</a> for an example). The number of newly infected 1-cm segments (height) divided by the number of days that elapsed between two consecutive time points was used to calculate the speed of virus within-plant movement along the monitored period. Because virus speed of movement was analyzed for at least 15 time points, a minimum of 14 values were obtained. Virus speed of within-plant movement per accession was calculated by averaging these values between every two consecutive time points.</p> </div> <div class="effectofinfectiononplantgrowthandreproduction." title="sec"> <div class="title" tagx="title" title="title">
Effect of infection on plant growth and reproduction.</div> <p>Aboveground plant structures were harvested at complete senescence. The weights of the rosette (<i>RW</i>), inflorescence (<i>IW</i>), and seeds (<i>SW</i>) were obtained. <i>RW</i> was used as an estimate of plant resources dedicated to growth, and <i>IW</i> and <i>SW</i> were taken as estimates of plant resources dedicated to reproduction (<a href="#B63">63</a>). The effect of virus infection on these traits was quantified by calculating ratios of infected to mock-inoculated plants for each of them, dividing the value of each infected plant by the mean value for the mock-inoculated plants of the same accession (Trait<i><sub>i</sub></i>/Trait<i><sub>m</sub></i>, where <i>i</i> and <i>m</i> denote infected and mock-inoculated plants, respectively). Virulence (<i>V</i>) was estimated as 1 minus the ratio of the total seed weight of infected (<i>SW<sub>i</sub></i>) to the total seed weight of mock-inoculated (<i>SW<sub>m</sub></i>) plants. The total numbers of seeds produced per mock-inoculated (<i>SN<sub>m</sub></i>) and infected (<i>SN<sub>i</sub></i>) plant were also quantified. To do so, we obtained the weight of 200 seeds for each replicate and derived the weight of a single seed. Using this value and the total seed weight of the corresponding plant, we calculated <i>SN</i>. Virus infection did not affect the weight of 200 seeds (<i>F</i> ≤ 2.79; <i>P</i> ≥ 0.114).</p> </div> <div class="efficiencyofvirusseedtransmission." title="sec"> <div class="title" tagx="title" title="title">
Efficiency of virus seed transmission.</div> <p>The efficiency of CMV and TuMV seed transmission was estimated both as a percentage of infected seeds and as the total number of infected seeds that gave rise to infected progeny per plant in grow-out tests. For each virus, 100 seeds per replicate were washed in a 10% bleach solution to ensure that any viral infection that occurred was not simply the result of the presence of virus on the seed coat but rather was the result of embryonic infection. Next, seeds were placed into petri dishes containing Murashige-Skoog medium, stratified for 3 days at 4°C, and germinated in a growth chamber at 22°C, under 16 h of light (intensity,120 to 150 mol s/m<sup>2</sup>). According to methods described previously (<a href="#B64">64</a>), seedlings at 15 days poststratification were pooled in groups of 2 for a total of 50 groups per replicate. These groups were tested for the presence of TuMV or CMV via qRT-PCR as described above. Because we knew the proportion of samples that tested negative, we used a Poisson distribution to estimate the probability that more than one seedling would test positive in the same sample. The percentage of virus-infected seeds (<i>ST</i>) was then estimated using an expression reported previously (<a href="#B65">65</a>), <i>p</i> = 1 − (1 − <i>y</i>/<i>n</i>)<sup>1/</sup><i><sup>k</sup></i>, where <i>p</i> is the probability of virus transmission by a single seed, <i>y</i> is the number of positive samples, <i>n</i> is the total number of samples assayed (<i>n</i>  =  50), and <i>k</i> is the number of seedlings per sample (<i>k</i>  =  2). To calculate the total number of infected seeds per plant (<i>IS</i>), we multiplied <i>SN<sub>i</sub></i> by <i>ST</i>.</p> </div> <div class="seedsurvival." title="sec"> <div class="title" tagx="title" title="title">
Seed survival.</div> <p>We measured short-, medium-, and long-term seed survival as surrogates of the effect of infection on seed viability. Short-term seed survival was measured as percent germination of seeds derived from infected and mock-inoculated plants 4 months after harvest to avoid biases due to seed dormancy, according to the protocol described above. To estimate medium- and long-term seed survival, seeds were artificially aged, and their germination percentage was measured. The artificial aging process was a modification of the "basal thermotolerance assay�? described previously (<a href="#B66">66</a>). Briefly, seeds were incubated at 42°C for 24 h (medium term) or 48 h (long term) with a relative humidity of 100% and afterwards stratified for 3 days at 4°C. The germination percentage was measured every 24 h for 11 days, until a constant germination percentage was reached. Values of seed survival measures were derived from every replicate of each treatment and 100 seeds per replicate. Short-term (<i>G</i><sub>0</sub>), medium-term (<i>G</i><sub>24</sub>), and long-term (<i>G</i><sub>48</sub>) seed survival were measured as the ratio of infected to mock-inoculated seed germination percentages. We quantified these three measures of seed survival because they can yield different information on the processes determining the efficiency of seed transmission: short-term survival occurs prior to our measures of seed transmission, and it can be considered a predictor of this trait. On the other hand, seed aging is posterior to our measures of seed transmission, and modifications of medium- and long-term survival can be considered a consequence of seed infection that modify the efficiency of seed transmission in the long run. In any case, the three seed survival measures can be considered potential estimators of virus efficiency of seed transmission.</p> </div> <div class="statisticalanalysis." title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis.</div> <p>Variables of virus seed transmission, virulence and the effect of virus infection on plant growth and reproduction, and seed survival were not normally distributed, and variances were heterogeneous according to Shapiro-Wilks and Levene tests, respectively. None of these variables could be normalized except for <i>SN<sub>i</sub></i> and <i>IS</i>, which were normalized using a log transformation. Therefore, differences between virus species and plant accessions were analyzed by generalized linear models (GzLMs), applying the corresponding linked function and considering virus isolate nested to virus species and <i>Arabidopsis</i> accession as random factors. Virus species-specific analyses considered virus isolate and <i>Arabidopsis</i> accession as random factors (R package glmmTMB [<a href="#B67">67</a>]).</p> <p>The relationship between TuMV and CMV infection traits and the efficiency of seed transmission was analyzed by utilizing mixed-effect multiple-regression tests (<a href="#B68">68</a>). We considered the following infection traits as potential estimators of the percentage of infected seeds: effect of the virus on plant rosette and inflorescence weights, virulence, speed of within-host movement, within-host multiplication in the rosette and inflorescence, and short-, medium-, and long-term seed survival. The same variables plus the percentage of infected seeds and the total number of seeds produced per plant were used as estimators of the total number of infected seeds. For model construction, the log transformation of the total number of seeds and of infected seeds per plant was used, which allowed scaling of all estimators. The cross-correlation between the estimators was analyzed using variance inflation factor (VIF). VIF values were lower than 3 for all variables, indicating minimal cross-correlation. Thus, all variables were included in the models. A set of models that included a global model containing all infection traits as fixed effects and nested models that contained all possible combinations of these traits were fitted for each response variable using general linear mixed models (R package glmmTMB [<a href="#B67">67</a>]). The models combining data for TuMV and CMV included virus isolate and <i>Arabidopsis</i> accession as random effects, and TuMV- and CMV-specific models included only the latter random effect. Models for percent seed transmission were constructed using a Poisson function and a log-linked function, as this function best reflected the distribution of the data according to Akaike’s information criterion (AIC) (R package rriskDistributions [<a href="#B69">69</a>]). Models for data on the number of infected seeds were constructed using a normal distribution and an identity-linked function. Global and nested models were ranked according to AIC scores, and the model with the lowest AIC score was selected as the best-ranked one. We calculated AIC Delta (Δ<i><sub>i</sub></i>) as the difference between the AIC of a given model and that of the best-ranked model (<a href="#B68">68</a>). Finally, the Akaike relative weight (ω<i><sub>i</sub></i>) of each model was calculated according to the expression ω<i><sub>i</sub></i> = exp(−0.5Δ<i><sub>i</sub></i>)/Σexp(−0.5Δ<i><sub>i</sub></i>). The relative importance of a given estimator included in the best-ranked model was calculated by decomposing the <i>R</i><sup>2</sup> value of the model into components corresponding to each estimator using the R package relaimpo (<a href="#B70">70</a>). Bivariate tests were used to analyze the association between the most relevant infection traits and the efficiency of virus seed transmission (R package stats [<a href="#B71">71</a>]).</p> <p>We analyzed the explanatory power of the best-ranked models using data derived from the 18 <i>Arabidopsis</i> accessions and the 5 virus isolates for which the efficiency of seed transmission was initially quantified. For this set of accessions, we quantified all the variables considered in the best-ranked models except for <i>SM</i>, <i>G</i><sub>24</sub>, and <i>G</i><sub>48</sub>. We estimated <i>SM</i> using values from the six-accession experiment. To do so, we calculated bivariate correlations between <i>SM</i> and all other variables. Because <i>SM</i> was an accession-specific trait, we used average values per accession. Based on these analyses, we identified the variable with the highest association with <i>SM</i>, and we used the equation of the linear relationship to estimate <i>SM</i> in the 18 <i>Arabidopsis</i> accessions. We could not follow the same approach for <i>G</i><sub>24</sub> and <i>G</i><sub>48</sub> as they were quantified for each individual plant, and no other trait significantly correlated with these two variables. Because these variables generally had very low relative importance in the best-ranked models, we considered them a constant in our simulations. <i>ST</i> and <i>IS</i> for the 18 <i>Arabidopsis</i> accessions were simulated using the best-ranked models (R package glmmTMB [<a href="#B67">67</a>]), and their association with real values was analyzed by linear regressions.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental file 1</div>  <div class="media" title="media"><a href="JVI.01078-19-sd001.xlsx">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-supplementary-material" title="supplementary-material"> <p>Supplemental material for this article may be found at <a href="https://doi.org/10.1128/JVI.01078-19">https://doi.org/10.1128/JVI.01078-19</a>.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGMENTS</div> <p>This work was supported by funding from the Plan Nacional I+D+i, MINECO, Spain (BIO2016-79165-R), and by the Universidad Politécnica de Madrid, Spain (PINV-18-O4W6R9-94-YR130U), to I.P. A.C. was supported by a Formación de Personal Investigador contract from MINECO (BES-2017-080783), N.M. was supported by the Ministerio de Economía y Competitividad (Instituto de Salud Carlos III) (RD16/0012/0011), and M.G.-V. was supported by the Programa de Empleo Juvenil from the Comunidad de Madrid (PEJ-2017-TL/BIO-6010).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="B1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Read</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">The evolution of virulence</span>. <span class="source" tagx="source" title="source">Trends Microbiol</span><span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">73</span>–<span class="lpage" tagx="lpage" title="lpage">76</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/0966-842X(94)90537-1">10.1016/0966-842X(94)90537-1</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8156274">8156274</a></span></span></li> <li tag="ref"><a name="B2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">Coevolution of hosts and parasites</span>. <span class="source" tagx="source" title="source">Parasitology</span><span class="volume" tagx="volume" title="volume">85</span>:<span class="fpage" tagx="fpage" title="fpage">411</span>–<span class="lpage" tagx="lpage" title="lpage">426</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1017/S0031182000055360">10.1017/S0031182000055360</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6755367">6755367</a></span></span></li> <li tag="ref"><a name="B3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipsitch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siller</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">The evolution of virulence in pathogens with vertical and horizontal transmission</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">50</span>:<span class="fpage" tagx="fpage" title="fpage">1729</span>–<span class="lpage" tagx="lpage" title="lpage">1741</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1996.tb03560.x">10.1111/j.1558-5646.1996.tb03560.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28565576">28565576</a></span></span></li> <li tag="ref"><a name="B4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alizon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurford</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mideo</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Baalen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Virulence evolution and the trade�?off hypothesis: history, current state of affairs and the future</span>. <span class="source" tagx="source" title="source">J Evol Biol</span><span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">245</span>–<span class="lpage" tagx="lpage" title="lpage">259</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1420-9101.2008.01658.x">10.1111/j.1420-9101.2008.01658.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19196383">19196383</a></span></span></li> <li tag="ref"><a name="B5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamelin</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">LJS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prendeville</span><span class="given-names" tagx="given-names" title="given-names">HR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hajimorad</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeger</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The evolution of plant virus transmission pathways</span>. <span class="source" tagx="source" title="source">J Theor Biol</span><span class="volume" tagx="volume" title="volume">396</span>:<span class="fpage" tagx="fpage" title="fpage">75</span>–<span class="lpage" tagx="lpage" title="lpage">89</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2016.02.017">10.1016/j.jtbi.2016.02.017</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26908348">26908348</a></span></span></li> <li tag="ref"><a name="B6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fereres</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Insect vectors as drivers of plant virus emergence</span>. <span class="source" tagx="source" title="source">Curr Opin Virol</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">42</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2014.12.008">10.1016/j.coviro.2014.12.008</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25596461">25596461</a></span></span></li> <li tag="ref"><a name="B7" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zilio</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiévent</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koella</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Host genotype and environment affect the trade-off between horizontal and vertical transmission of the parasite <i>Edhazardia aedis</i></span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="volume" tagx="volume" title="volume">18</span>:<span class="fpage" tagx="fpage" title="fpage">59</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1186/s12862-018-1184-3">10.1186/s12862-018-1184-3</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29699504">29699504</a></span></span></li> <li tag="ref"><a name="B8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faeth</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadeler</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thieme</span><span class="given-names" tagx="given-names" title="given-names">HR</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">An apparent paradox of horizontal and vertical disease transmission</span>. <span class="source" tagx="source" title="source">J Biol Dyn</span><span class="volume" tagx="volume" title="volume">1</span>:<span class="fpage" tagx="fpage" title="fpage">45</span>–<span class="lpage" tagx="lpage" title="lpage">62</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1080/17513750601040367">10.1080/17513750601040367</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22880612">22880612</a></span></span></li> <li tag="ref"><a name="B9" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamelin</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilker</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeger</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hajimorad</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prendeville</span><span class="given-names" tagx="given-names" title="given-names">HR</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The evolution of parasitic and mutualistic plant-virus symbioses through transmission-virulence trade-offs</span>. <span class="source" tagx="source" title="source">Virus Res</span><span class="volume" tagx="volume" title="volume">241</span>:<span class="fpage" tagx="fpage" title="fpage">77</span>–<span class="lpage" tagx="lpage" title="lpage">87</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2017.04.011">10.1016/j.virusres.2017.04.011</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28434906">28434906</a></span></span></li> <li tag="ref"><a name="B10" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipsitch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebert</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">The population dynamics of vertically and horizontally transmitted parasites</span>. <span class="source" tagx="source" title="source">Proc Biol Sci</span><span class="volume" tagx="volume" title="volume">260</span>:<span class="fpage" tagx="fpage" title="fpage">321</span>–<span class="lpage" tagx="lpage" title="lpage">327</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7630898">7630898</a></span></span></li> <li tag="ref"><a name="B11" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sastry</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="source" tagx="source" title="source">Seed-borne plant virus diseases</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span>.</span></li> <li tag="ref"><a name="B12" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munkvold</span><span class="given-names" tagx="given-names" title="given-names">GP</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Seed transmission in the <i>Potyviridae</i></span>, p <span class="fpage" tagx="fpage" title="fpage">3</span>–<span class="lpage" tagx="lpage" title="lpage">15</span>. <i>In</i><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gullino</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munkvold</span><span class="given-names" tagx="given-names" title="given-names">GP</span></span></span> (ed), <span class="source" tagx="source" title="source">Global perspectives on the health of seeds and plant propagation material</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dordrecht, Netherlands</span>.</span></li> <li tag="ref"><a name="B13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mink</span><span class="given-names" tagx="given-names" title="given-names">GI</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">Pollen and seed-transmitted viruses and viroids</span>. <span class="source" tagx="source" title="source">Annu Rev Phytopathol</span><span class="volume" tagx="volume" title="volume">31</span>:<span class="fpage" tagx="fpage" title="fpage">375</span>–<span class="lpage" tagx="lpage" title="lpage">402</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.py.31.090193.002111">10.1146/annurev.py.31.090193.002111</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18643763">18643763</a></span></span></li> <li tag="ref"><a name="B14" /><span class="label" tagx="label" title="label">14.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Movement between plants: vertical transmission</span>, p <span class="fpage" tagx="fpage" title="fpage">185</span>–<span class="lpage" tagx="lpage" title="lpage">198</span>. <i>In</i><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palukaitis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span> (ed), <span class="source" tagx="source" title="source">Cucumber mosaic virus</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">APS Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC</span>.</span></li> <li tag="ref"><a name="B15" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albrechtsen</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="source" tagx="source" title="source">Testing methods for seed-transmitted viruses: principles and protocols</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">CABI Publishing</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Wallingford, United Kingdom</span>.</span></li> <li tag="ref"><a name="B16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwyer</span><span class="given-names" tagx="given-names" title="given-names">GI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">RAC</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title"><i>Wheat streak mosaic virus</i> in Australia: relationship to isolates from the Pacific Northwest of the USA and its dispersion via seed transmission</span>. <span class="source" tagx="source" title="source">Plant Dis</span><span class="volume" tagx="volume" title="volume">91</span>:<span class="fpage" tagx="fpage" title="fpage">164</span>–<span class="lpage" tagx="lpage" title="lpage">170</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1094/PDIS-91-2-0164">10.1094/PDIS-91-2-0164</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30780999">30780999</a></span></span></li> <li tag="ref"><a name="B17" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coutts</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">RAC</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Effects of introduced and indigenous viruses on native plants: exploring their disease-causing potential at the agro-ecological interface</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e91224</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0091224">10.1371/journal.pone.0091224</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24621926">24621926</a></span></span></li> <li tag="ref"><a name="B18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">RAC</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Determining ‘threshold’ levels for seed-borne virus infection in seed stocks</span>. <span class="source" tagx="source" title="source">Virus Res</span><span class="volume" tagx="volume" title="volume">71</span>:<span class="fpage" tagx="fpage" title="fpage">171</span>–<span class="lpage" tagx="lpage" title="lpage">183</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S0168-1702(00)00197-0">10.1016/S0168-1702(00)00197-0</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11137171">11137171</a></span></span></li> <li tag="ref"><a name="B19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kil</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byun</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shim</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title"><i>Tomato yellow leaf curl virus</i> (TYLCV-IL): a seed-transmissible geminivirus in tomatoes</span>. <span class="source" tagx="source" title="source">Sci Rep</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">19013</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/srep19013">10.1038/srep19013</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26743765">26743765</a></span></span></li> <li tag="ref"><a name="B20" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hull</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="source" tagx="source" title="source">Plant virology</span>, <span class="edition" tagx="edition" title="edition">5th ed</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, United Kingdom</span>.</span></li> <li tag="ref"><a name="B21" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dombrovsky</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Seed transmission of tobamoviruses: aspects of global disease distribution</span>, p <span class="fpage" tagx="fpage" title="fpage">233</span>–<span class="lpage" tagx="lpage" title="lpage">260</span>. <i>In</i><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jimenez-Lopez</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span> (ed), <span class="source" tagx="source" title="source">Advances in seed biology</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Intech Open</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, United Kingdom</span>.</span></li> <li tag="ref"><a name="B22" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampton</span><span class="given-names" tagx="given-names" title="given-names">RO</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Seed transmission of viruses: current perspectives</span>. <span class="source" tagx="source" title="source">Annu Rev Phytopathol</span><span class="volume" tagx="volume" title="volume">32</span>:<span class="fpage" tagx="fpage" title="fpage">363</span>–<span class="lpage" tagx="lpage" title="lpage">386</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.py.32.090194.002051">10.1146/annurev.py.32.090194.002051</a></span>.</span></li> <li tag="ref"><a name="B23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maule</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Seed transmission of plant viruses: a lesson in biological complexity</span>. <span class="source" tagx="source" title="source">Trends Microbiol</span><span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">153</span>–<span class="lpage" tagx="lpage" title="lpage">158</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/0966-842X(96)10016-0">10.1016/0966-842X(96)10016-0</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8728609">8728609</a></span></span></li> <li tag="ref"><a name="B24" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domier</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hobbs</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCoppin</span><span class="given-names" tagx="given-names" title="given-names">NK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowen</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinlage</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartman</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Multiple loci condition seed transmission of <i>Soybean mosaic virus</i> (SMV) and SMV-induced seed coat mottling in soybean</span>. <span class="source" tagx="source" title="source">Phytopathology</span><span class="volume" tagx="volume" title="volume">101</span>:<span class="fpage" tagx="fpage" title="fpage">750</span>–<span class="lpage" tagx="lpage" title="lpage">756</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1094/PHYTO-09-10-0239">10.1094/PHYTO-09-10-0239</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21561316">21561316</a></span></span></li> <li tag="ref"><a name="B25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Mapping of the seed transmission determinants of barley stripe mosaic virus</span>. <span class="source" tagx="source" title="source">Mol Plant Microbe Interact</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">906</span>–<span class="lpage" tagx="lpage" title="lpage">915</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1094/MPMI-8-0906">10.1094/MPMI-8-0906</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8664501">8664501</a></span></span></li> <li tag="ref"><a name="B26" /><span class="label" tagx="label" title="label">26.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macfarlane</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maule</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Viral determinants of pea early browning virus seed transmission in pea</span>. <span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">234</span>:<span class="fpage" tagx="fpage" title="fpage">112</span>–<span class="lpage" tagx="lpage" title="lpage">117</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1006/viro.1997.8637">10.1006/viro.1997.8637</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9234951">9234951</a></span></span></li> <li tag="ref"><a name="B27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montes</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Vertical transmission selects for reduced virulence in a plant virus and for increased resistance in the host</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004293</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004293">10.1371/journal.ppat.1004293</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25077948">25077948</a></span></span></li> <li tag="ref"><a name="B28" /><span class="label" tagx="label" title="label">28.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Logsdon</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelley</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">An empirical study of the evolution of virulence under both horizontal and vertical transmission</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">59</span>:<span class="fpage" tagx="fpage" title="fpage">730</span>–<span class="lpage" tagx="lpage" title="lpage">739</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.0014-3820.2005.tb01749.x">10.1111/j.0014-3820.2005.tb01749.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15926685">15926685</a></span></span></li> <li tag="ref"><a name="B29" /><span class="label" tagx="label" title="label">29.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ewald</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">Transmission modes and evolution of the parasitism-mutualism continuum</span>. <span class="source" tagx="source" title="source">Ann N Y Acad Sci</span><span class="volume" tagx="volume" title="volume">503</span>:<span class="fpage" tagx="fpage" title="fpage">295</span>–<span class="lpage" tagx="lpage" title="lpage">306</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1749-6632.1987.tb40616.x">10.1111/j.1749-6632.1987.tb40616.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3304078">3304078</a></span></span></li> <li tag="ref"><a name="B30" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herre</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Factors affecting the evolution of virulence: nematode parasites of fig wasps as a case study</span>. <span class="source" tagx="source" title="source">Parasitology</span><span class="volume" tagx="volume" title="volume">111</span>:<span class="fpage" tagx="fpage" title="fpage">S179</span>–<span class="lpage" tagx="lpage" title="lpage">S191</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1017/S0031182000075880">10.1017/S0031182000075880</a></span>.</span></li> <li tag="ref"><a name="B31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kover</span><span class="given-names" tagx="given-names" title="given-names">PX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolan</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clay</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Potential versus actual contribution of vertical transmission to pathogen fitness</span>. <span class="source" tagx="source" title="source">Proc R Soc Lond B</span><span class="volume" tagx="volume" title="volume">264</span>:<span class="fpage" tagx="fpage" title="fpage">903</span>–<span class="lpage" tagx="lpage" title="lpage">909</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.1997.0125">10.1098/rspb.1997.0125</a></span>.</span></li> <li tag="ref"><a name="B32" /><span class="label" tagx="label" title="label">32.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraile</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandez-Fueyo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montes</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso-Blanco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title"><i>Arabidopsis thaliana</i> as a model for the study of plant-virus co-evolution</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">365</span>:<span class="fpage" tagx="fpage" title="fpage">1983</span>–<span class="lpage" tagx="lpage" title="lpage">1995</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2010.0062">10.1098/rstb.2010.0062</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20478893">20478893</a></span></span></li> <li tag="ref"><a name="B33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montes</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso-Blanco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title"><i>Cucumber mosaic virus</i> infection as a potential selective pressure on <i>Arabidopsis thaliana</i> populations</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">15</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007810</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1007810">10.1371/journal.ppat.1007810</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31136630">31136630</a></span></span></li> <li tag="ref"><a name="B34" /><span class="label" tagx="label" title="label">34.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shukla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García�?Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Effective tolerance based on resource reallocation is a virus�?specific defence in <i>Arabidopsis thaliana</i></span>. <span class="source" tagx="source" title="source">Mol Plant Pathol</span><span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">1454</span>–<span class="lpage" tagx="lpage" title="lpage">1465</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/mpp.12629">10.1111/mpp.12629</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29027740">29027740</a></span></span></li> <li tag="ref"><a name="B35" /><span class="label" tagx="label" title="label">35.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bueso</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallás</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sánchez-Navarro</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Seed tolerance to deterioration in arabidopsis is affected by virus infection</span>. <span class="source" tagx="source" title="source">Plant Physiol Biochem</span><span class="volume" tagx="volume" title="volume">116</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plaphy.2017.04.020">10.1016/j.plaphy.2017.04.020</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28477474">28477474</a></span></span></li> <li tag="ref"><a name="B36" /><span class="label" tagx="label" title="label">36.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonovics</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forbes</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauffe</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kallio</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leggett</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okamura</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sait</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The evolution of transmission mode</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">372</span>:<span class="fpage" tagx="fpage" title="fpage">20160083</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2016.0083">10.1098/rstb.2016.0083</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28289251">28289251</a></span></span></li> <li tag="ref"><a name="B37" /><span class="label" tagx="label" title="label">37.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alvarez-Buylla</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benítez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corvera-Poiré</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cador</span><span class="given-names" tagx="given-names" title="given-names">�?C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Folter</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Buen</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garay-Arroyo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Ponce</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaimes-Miranda</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pérez-Ruiz</span><span class="given-names" tagx="given-names" title="given-names">RV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piñeyro-Nelson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sánchez-Corrales</span><span class="given-names" tagx="given-names" title="given-names">YE</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Flower development</span>. <span class="source" tagx="source" title="source">Arabidopsis Book</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e0127</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1199/tab.0127">10.1199/tab.0127</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22303253">22303253</a></span></span></li> <li tag="ref"><a name="B38" /><span class="label" tagx="label" title="label">38.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">MX</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Direct evidence that suspensor cells have embryogenic potential that is suppressed by the embryo proper during normal embryogenesis</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">112</span>:<span class="fpage" tagx="fpage" title="fpage">12432</span>–<span class="lpage" tagx="lpage" title="lpage">12437</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1508651112">10.1073/pnas.1508651112</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26396256">26396256</a></span></span></li> <li tag="ref"><a name="B39" /><span class="label" tagx="label" title="label">39.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maule</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">A model for seed transmission of a plant virus: genetic and structural analyses of pea embryo invasion by pea seed-borne mosaic virus</span>. <span class="source" tagx="source" title="source">Plant Cell</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">777</span>–<span class="lpage" tagx="lpage" title="lpage">787</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.2307/3869957">10.2307/3869957</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12244258">12244258</a></span></span></li> <li tag="ref"><a name="B40" /><span class="label" tagx="label" title="label">40.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amari</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burgos</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallás</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sánchez-Pina</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Vertical transmission of <i>Prunus necrotic ringspot virus</i>: hitch-hiking from gametes to seedling</span>. <span class="source" tagx="source" title="source">J Gen Virol</span><span class="volume" tagx="volume" title="volume">90</span>:<span class="fpage" tagx="fpage" title="fpage">1767</span>–<span class="lpage" tagx="lpage" title="lpage">1774</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.009647-0">10.1099/vir.0.009647-0</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19282434">19282434</a></span></span></li> <li tag="ref"><a name="B41" /><span class="label" tagx="label" title="label">41.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sánchez</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manrique</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mansilla</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunello</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-González</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jenner</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">González-Melendi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elena</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponz</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Viral strain-specific differential alterations in Arabidopsis developmental patterns</span>. <span class="source" tagx="source" title="source">Mol Plant Microbe Interact</span><span class="volume" tagx="volume" title="volume">28</span>:<span class="fpage" tagx="fpage" title="fpage">1304</span>–<span class="lpage" tagx="lpage" title="lpage">1315</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1094/MPMI-05-15-0111-R">10.1094/MPMI-05-15-0111-R</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26646245">26646245</a></span></span></li> <li tag="ref"><a name="B42" /><span class="label" tagx="label" title="label">42.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijayan</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-González</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sánchez</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponz</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Virulence evolution of a sterilizing plant virus: tuning multiplication and resource exploitation</span>. <span class="source" tagx="source" title="source">Virus Evol</span><span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">vex033</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vex033">10.1093/ve/vex033</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29250431">29250431</a></span></span></li> <li tag="ref"><a name="B43" /><span class="label" tagx="label" title="label">43.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacquemond</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Cucumber mosaic virus</span>. <span class="source" tagx="source" title="source">Adv Virus Res</span><span class="volume" tagx="volume" title="volume">84</span>:<span class="fpage" tagx="fpage" title="fpage">439</span>–<span class="lpage" tagx="lpage" title="lpage">504</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/B978-0-12-394314-9.00013-0">10.1016/B978-0-12-394314-9.00013-0</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22682176">22682176</a></span></span></li> <li tag="ref"><a name="B44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jenner</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title"><i>Turnip mosaic virus</i> and the quest for durable resistance</span>. <span class="source" tagx="source" title="source">Mol Plant Pathol</span><span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">289</span>–<span class="lpage" tagx="lpage" title="lpage">300</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1364-3703.2002.00132.x">10.1046/j.1364-3703.2002.00132.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20569337">20569337</a></span></span></li> <li tag="ref"><a name="B45" /><span class="label" tagx="label" title="label">45.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Keefe</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonovics</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Playing by different rules: the evolution of virulence in sterilizing pathogens</span>. <span class="source" tagx="source" title="source">Am Nat</span><span class="volume" tagx="volume" title="volume">159</span>:<span class="fpage" tagx="fpage" title="fpage">597</span>–<span class="lpage" tagx="lpage" title="lpage">605</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1086/339990">10.1086/339990</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18707384">18707384</a></span></span></li> <li tag="ref"><a name="B46" /><span class="label" tagx="label" title="label">46.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebert</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joachim-Carius</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Little</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decaestecker</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">The evolution of virulence when parasites cause host castration and gigantism</span>. <span class="source" tagx="source" title="source">Am Nat</span><span class="volume" tagx="volume" title="volume">164</span>:<span class="fpage" tagx="fpage" title="fpage">S19</span>–<span class="lpage" tagx="lpage" title="lpage">S32</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1086/424606">10.1086/424606</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15540139">15540139</a></span></span></li> <li tag="ref"><a name="B47" /><span class="label" tagx="label" title="label">47.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becker</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cáceres</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Parasitic castration: a perspective from a model of dynamic energy budgets</span>. <span class="source" tagx="source" title="source">Integr Comp Biol</span><span class="volume" tagx="volume" title="volume">47</span>:<span class="fpage" tagx="fpage" title="fpage">295</span>–<span class="lpage" tagx="lpage" title="lpage">309</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/icb/icm057">10.1093/icb/icm057</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21672839">21672839</a></span></span></li> <li tag="ref"><a name="B48" /><span class="label" tagx="label" title="label">48.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kover</span><span class="given-names" tagx="given-names" title="given-names">PX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clay</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Trade-off between virulence and vertical transmission and the maintenance of a virulent plant pathogen</span>. <span class="source" tagx="source" title="source">Am Nat</span><span class="volume" tagx="volume" title="volume">152</span>:<span class="fpage" tagx="fpage" title="fpage">165</span>–<span class="lpage" tagx="lpage" title="lpage">175</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1086/286159">10.1086/286159</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18811383">18811383</a></span></span></li> <li tag="ref"><a name="B49" /><span class="label" tagx="label" title="label">49.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kover</span><span class="given-names" tagx="given-names" title="given-names">PX</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Effects of parasitic castration on plant resource allocation</span>. <span class="source" tagx="source" title="source">Oecologia</span><span class="volume" tagx="volume" title="volume">123</span>:<span class="fpage" tagx="fpage" title="fpage">48</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1007/s004420050988">10.1007/s004420050988</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28308743">28308743</a></span></span></li> <li tag="ref"><a name="B50" /><span class="label" tagx="label" title="label">50.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso-Blanco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The relationship of within-host multiplication and virulence in a plant-virus system</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">2</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e786</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0000786">10.1371/journal.pone.0000786</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17726516">17726516</a></span></span></li> <li tag="ref"><a name="B51" /><span class="label" tagx="label" title="label">51.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagán</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso-Blanco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Host responses in life-history traits and tolerance to virus infection in <i>Arabidopsis thaliana</i></span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">4</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000124</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1000124">10.1371/journal.ppat.1000124</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18704166">18704166</a></span></span></li> <li tag="ref"><a name="B52" /><span class="label" tagx="label" title="label">52.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jenner</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Pathotypic variation in <i>Turnip mosaic virus</i> with special reference to European isolates</span>. <span class="source" tagx="source" title="source">Plant Pathol</span><span class="volume" tagx="volume" title="volume">45</span>:<span class="fpage" tagx="fpage" title="fpage">848</span>–<span class="lpage" tagx="lpage" title="lpage">856</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-3059.1996.tb02895.x">10.1111/j.1365-3059.1996.tb02895.x</a></span>.</span></li> <li tag="ref"><a name="B53" /><span class="label" tagx="label" title="label">53.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonnet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraile</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sacristán</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malpica</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Role of recombination in the evolution of natural populations of <i>Cucumber mosaic virus</i>, a tripartite RNA plant virus</span>. <span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">332</span>:<span class="fpage" tagx="fpage" title="fpage">359</span>–<span class="lpage" tagx="lpage" title="lpage">368</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2004.11.017">10.1016/j.virol.2004.11.017</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661167">15661167</a></span></span></li> <li tag="ref"><a name="B54" /><span class="label" tagx="label" title="label">54.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizzo</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palukaitis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">1990</span><span class="mixed-article-title" title="mixed-article-title">Construction of full-length cDNA clones of <i>Cucumber mosaic virus</i> RNAs 1, 2 and 3: generation of infectious RNA transcripts</span>. <span class="source" tagx="source" title="source">Mol Gen Genet</span><span class="volume" tagx="volume" title="volume">222</span>:<span class="fpage" tagx="fpage" title="fpage">249</span>–<span class="lpage" tagx="lpage" title="lpage">256</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1007/bf00633825">10.1007/bf00633825</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2274028">2274028</a></span></span></li> <li tag="ref"><a name="B55" /><span class="label" tagx="label" title="label">55.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanada</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palukaitis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Mapping local and systemic symptom determinants of cucumber mosaic cucumovirus in tobacco</span>. <span class="source" tagx="source" title="source">J Gen Virol</span><span class="volume" tagx="volume" title="volume">75</span>:<span class="fpage" tagx="fpage" title="fpage">3185</span>–<span class="lpage" tagx="lpage" title="lpage">3191</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-75-11-3185">10.1099/0022-1317-75-11-3185</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7964627">7964627</a></span></span></li> <li tag="ref"><a name="B56" /><span class="label" tagx="label" title="label">56.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sánchez</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martínez-Herrera</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilar</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponz</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Infectivity of turnip mosaic potyvirus cDNA clones and transcripts on the systemic host Arabidopsis thaliana and local lesion hosts</span>. <span class="source" tagx="source" title="source">Virus Res</span><span class="volume" tagx="volume" title="volume">55</span>:<span class="fpage" tagx="fpage" title="fpage">207</span>–<span class="lpage" tagx="lpage" title="lpage">219</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S0168-1702(98)00049-5">10.1016/S0168-1702(98)00049-5</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9725673">9725673</a></span></span></li> <li tag="ref"><a name="B57" /><span class="label" tagx="label" title="label">57.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharbel</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haubold</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell�?Olds</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Genetic isolation by distance in Arabidopsis thaliana: biogeography and postglacial colonization of Europe</span>. <span class="source" tagx="source" title="source">Mol Ecol</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">2109</span>–<span class="lpage" tagx="lpage" title="lpage">2118</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-294X.2000.01122.x">10.1046/j.1365-294X.2000.01122.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11123622">11123622</a></span></span></li> <li tag="ref"><a name="B58" /><span class="label" tagx="label" title="label">58.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boyes</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zayed</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ascenzi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCaskill</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffman</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Görlach</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Growth stage-based phenotypic analysis of Arabidopsis: a model for high throughput functional genomics in plants</span>. <span class="source" tagx="source" title="source">Plant Cell</span><span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">1499</span>–<span class="lpage" tagx="lpage" title="lpage">1510</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.010011">10.1105/tpc.010011</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11449047">11449047</a></span></span></li> <li tag="ref"><a name="B59" /><span class="label" tagx="label" title="label">59.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunello</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mansilla</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sánchez</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponz</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">A developmentally linked, dramatic, and transient loss of virus from roots of Arabidopsis thaliana plants infected by either of two RNA viruses</span>. <span class="source" tagx="source" title="source">Mol Plant Microbe Interact</span><span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">1589</span>–<span class="lpage" tagx="lpage" title="lpage">1595</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1094/MPMI-20-12-1589">10.1094/MPMI-20-12-1589</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17990966">17990966</a></span></span></li> <li tag="ref"><a name="B60" /><span class="label" tagx="label" title="label">60.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hily</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreno</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plaza</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilkinson</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fereres</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraile</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Arenal</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The relationship between host lifespan and pathogen reservoir potential: an analysis in the system <i>Arabidopsis thaliana-Cucumber mosaic virus</i></span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004492</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004492">10.1371/journal.ppat.1004492</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25375140">25375140</a></span></span></li> <li tag="ref"><a name="B61" /><span class="label" tagx="label" title="label">61.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grainger</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1933</span><span class="mixed-article-title" title="mixed-article-title">The movement of Tobacco mosaic virus in its host</span>. <span class="source" tagx="source" title="source">Ann Appl Biol</span><span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">236</span>–<span class="lpage" tagx="lpage" title="lpage">257</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1744-7348.1933.tb07763.x">10.1111/j.1744-7348.1933.tb07763.x</a></span>.</span></li> <li tag="ref"><a name="B62" /><span class="label" tagx="label" title="label">62.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samuel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">1934</span><span class="mixed-article-title" title="mixed-article-title">The movement of Tobacco mosaic virus within the plant</span>. <span class="source" tagx="source" title="source">Ann Appl Biol</span><span class="volume" tagx="volume" title="volume">21</span>:<span class="fpage" tagx="fpage" title="fpage">90</span>–<span class="lpage" tagx="lpage" title="lpage">111</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1744-7348.1934.tb06891.x">10.1111/j.1744-7348.1934.tb06891.x</a></span>.</span></li> <li tag="ref"><a name="B63" /><span class="label" tagx="label" title="label">63.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">AJA</span></span></span><span class="year" tagx="year" title="year">1981</span><span class="mixed-article-title" title="mixed-article-title">The measurement and meaning of reproductive effort in plants</span>. <span class="source" tagx="source" title="source">Am Nat</span><span class="volume" tagx="volume" title="volume">117</span>:<span class="fpage" tagx="fpage" title="fpage">205</span>–<span class="lpage" tagx="lpage" title="lpage">211</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1086/283700">10.1086/283700</a></span>.</span></li> <li tag="ref"><a name="B64" /><span class="label" tagx="label" title="label">64.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gildow</span><span class="given-names" tagx="given-names" title="given-names">FE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bothe-Goralczyk</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephenson</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Experimental verification of seed transmission in <i>Zucchini yellow mosaic virus</i></span>. <span class="source" tagx="source" title="source">Plant Dis</span><span class="volume" tagx="volume" title="volume">95</span>:<span class="fpage" tagx="fpage" title="fpage">751</span>–<span class="lpage" tagx="lpage" title="lpage">754</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1094/PDIS-11-10-0843">10.1094/PDIS-11-10-0843</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30731907">30731907</a></span></span></li> <li tag="ref"><a name="B65" /><span class="label" tagx="label" title="label">65.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gower</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="year" tagx="year" title="year">1960</span><span class="mixed-article-title" title="mixed-article-title">The use of a multiple-transfer method in plant virus transmission studies: some statistical points arising in the analysis results</span>. <span class="source" tagx="source" title="source">Ann Appl Biol</span><span class="volume" tagx="volume" title="volume">48</span>:<span class="fpage" tagx="fpage" title="fpage">75</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1744-7348.1960.tb03506.x">10.1111/j.1744-7348.1960.tb03506.x</a></span>.</span></li> <li tag="ref"><a name="B66" /><span class="label" tagx="label" title="label">66.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tejedor-Cano</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prieto-Dapena</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almoguera</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carranco</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiratsu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohme�?Takagi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordano</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Loss of function of the HSFA9 seed longevity program</span>. <span class="source" tagx="source" title="source">Plant Cell Environ</span><span class="volume" tagx="volume" title="volume">33</span>:<span class="fpage" tagx="fpage" title="fpage">1408</span>–<span class="lpage" tagx="lpage" title="lpage">1417</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-3040.2010.02159.x">10.1111/j.1365-3040.2010.02159.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20444218">20444218</a></span></span></li> <li tag="ref"><a name="B67" /><span class="label" tagx="label" title="label">67.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kristensen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Benthem</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magnusson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skaug</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mächler</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolker</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">glmmTMB balances speed and flexibility among packages for zero-inflated generalized linear mixed modeling</span>. <span class="source" tagx="source" title="source">R J</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">378</span>–<span class="lpage" tagx="lpage" title="lpage">400</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.32614/RJ-2017-066">10.32614/RJ-2017-066</a></span>.</span></li> <li tag="ref"><a name="B68" /><span class="label" tagx="label" title="label">68.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burnham</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="source" tagx="source" title="source">Model selection and inference: a practical information-theoretic approach</span>, <span class="edition" tagx="edition" title="edition">2nd ed</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>.</span></li> <li tag="ref"><a name="B69" /><span class="label" tagx="label" title="label">69.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belgorodski</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greiner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tolksdorf</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schueller</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">rriskDistributions: fitting distributions to given data or known quantiles R package version 2.1.2</span>. <a href="https://CRAN.R-project.org/package=rriskDistributions">https://CRAN.R-project.org/package=rriskDistributions</a>.</span></li> <li tag="ref"><a name="B70" /><span class="label" tagx="label" title="label">70.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grömping</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Relative importance for linear regression in R: the package relaimpo</span>. <span class="source" tagx="source" title="source">J Stat Softw</span><span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>.</span></li> <li tag="ref"><a name="B71" /><span class="label" tagx="label" title="label">71.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: R Core Team</span>. <span class="year" tagx="year" title="year">2018</span><span class="source" tagx="source" title="source">R: a language and environment for statistical computing</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span>.</span></li> </ul> </div> </div>  </body></html>