<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/06/2020.03.27.012906.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.27.012906v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses" />
<meta name="DC.Identifier" content="10.1101/2020.03.27.012906" />
<meta name="DC.Date" content="2020-04-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="As the COVID-19 outbreak spreads, there is a growing need for a compilation of conserved RNA genome regions in the SARS-CoV-2 virus along with their structural propensities to guide development of antivirals and diagnostics. Using sequence alignments spanning a range of betacoronaviruses, we rank genomic regions by RNA sequence conservation, identifying 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome sequences available near the beginning of the COVID-19 outbreak. We then confirm the conservation of the majority of these genome regions across 739 SARS-CoV-2 sequences reported to date from the current COVID-19 outbreak, and we present a curated list of 30 ‘SARS-related-conserved’ regions. We find that known RNA structured elements curated as Rfam families and in prior literature are enriched in these conserved genome regions, and we predict additional conserved, stable secondary structures across the viral genome. We provide 106 ‘SARS-CoV-2-conserved-structured’ regions as potential targets for antivirals that bind to structured RNA. We further provide detailed secondary structure models for the 5’ UTR, frame-shifting element, and 3’ UTR. Last, we predict regions of the SARS-CoV-2 viral genome have low propensity for RNA secondary structure and are conserved within SARS-CoV-2 strains. These 59 ‘SARS-CoV-2-conserved-unstructured’ genomic regions may be most easily targeted in primer-based diagnostic and oligonucleotide-based therapeutic strategies." />
<meta name="DC.Contributor" content="Ramya Rangan" />
<meta name="DC.Contributor" content="Ivan N. Zheludev" />
<meta name="DC.Contributor" content="Rhiju Das" />
<meta name="article:published_time" content="2020-04-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;As the COVID-19 outbreak spreads, there is a growing need for a compilation of conserved RNA genome regions in the SARS-CoV-2 virus along with their structural propensities to guide development of antivirals and diagnostics. Using sequence alignments spanning a range of betacoronaviruses, we rank genomic regions by RNA sequence conservation, identifying 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome sequences available near the beginning of the COVID-19 outbreak. We then confirm the conservation of the majority of these genome regions across 739 SARS-CoV-2 sequences reported to date from the current COVID-19 outbreak, and we present a curated list of 30 ′SARS-related-conserved′ regions. We find that known RNA structured elements curated as Rfam families and in prior literature are enriched in these conserved genome regions, and we predict additional conserved, stable secondary structures across the viral genome. We provide 106 ′SARS-CoV-2-conserved-structured′ regions as potential targets for antivirals that bind to structured RNA. We further provide detailed secondary structure models for the 5′ UTR, frame-shifting element, and 3′ UTR. Last, we predict regions of the SARS-CoV-2 viral genome have low propensity for RNA secondary structure and are conserved within SARS-CoV-2 strains. These 59 ′SARS-CoV-2-conserved-unstructured′ genomic regions may be most easily targeted in primer-based diagnostic and oligonucleotide-based therapeutic strategies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.27.012906v2" />
<meta name="citation_id" content="2020.03.27.012906v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.27.012906" />
<meta name="citation_num_pages" content="27" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.27.012906" />
<meta name="citation_author" content="Ramya Rangan" />
<meta name="citation_author_institution" content="Biophysics Program, Stanford University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0960-0825" />
<meta name="citation_author" content="Ivan N. Zheludev" />
<meta name="citation_author_institution" content="Department of Biochemistry, Stanford University School of Medicine" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9572-0574" />
<meta name="citation_author" content="Rhiju Das" />
<meta name="citation_author_institution" content="Biophysics Program, Stanford University" />
<meta name="citation_author_institution" content="Department of Biochemistry, Stanford University School of Medicine" />
<meta name="citation_author_institution" content="Department of Physics, Stanford University" />
<meta name="citation_author_email" content="rhiju@stanford.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7497-0972" />
<meta name="citation_reference" content="Fehr, A. R.; Perlman, S., Coronaviruses: an overview of their replication and pathogenesis. Methods Mol. Biol. 2015, 1282, 1–23." />
<meta name="citation_reference" content="Connelly, C. M.; Moon, M. H.; Schneekloth, J. S., Jr., The Emerging Role of RNA as a Therapeutic Target for Small Molecules. Cell Chem Biol 2016, 23 (9), 1077–1090." />
<meta name="citation_reference" content="Spurgers, K. B.; Sharkey, C. M.; Warfield, K. L.; Bavari, S., Oligonucleotide antiviral therapeutics: antisense and RNA interference for highly pathogenic RNA viruses. Antiviral Res. 2008, 78 (1), 26–36." />
<meta name="citation_reference" content="Bennett, C. F.; Krainer, A. R.; Cleveland, D. W., Antisense Oligonucleotide Therapies for Neurodegenerative Diseases. Annu. Rev. Neurosci. 2019, 42, 385–406." />
<meta name="citation_reference" content="Yang, D.; Leibowitz, J. L., The structure and functions of coronavirus genomic 3’ and 5’ ends. Virus Res. 2015, 206, 120–133." />
<meta name="citation_reference" content="van den Born, E.; Posthuma, C. C.; Gultyaev, A. P.; Snijder, E. J., Discontinuous subgenomic RNA synthesis in arteriviruses is guided by an RNA hairpin structure located in the genomic leader region. J. Virol. 2005, 79 (10), 6312–6324." />
<meta name="citation_reference" content="Plant, E. P.; Dinman, J. D., The role of programmed-1 ribosomal frameshifting in coronavirus propagation. Front. Biosci. 2008, 13, 4873–4881." />
<meta name="citation_reference" content="Stammler, S. N.; Cao, S.; Chen, S.-J.; Giedroc, D. P., A conserved RNA pseudoknot in a putative molecular switch domain of the 3’-untranslated region of coronaviruses is only marginally stable. RNA 2011, 17 (9), 1747–1759." />
<meta name="citation_reference" content="Chen, S.-C.; Olsthoorn, R. C. L., Group-specific structural features of the 5’-proximal sequences of coronavirus genomic RNAs. Virology 2010, 401 (1), 29–41." />
<meta name="citation_reference" content="Ceraolo, C.; Giorgi, F. M., Genomic variance of the 2019-nCoV coronavirus. J. Med. Virol. 2020, 92 (5), 522–528." />
<meta name="citation_reference" content="citation_title=NC_045512. Nucleotide [Internet];citation_year=2020" />
<meta name="citation_reference" content="Viruses, C. S. G. o. T. I. C. o. T. o.; Coronaviridae Study Group of the International Committee on Taxonomy of, V., The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nature Microbiology 2020, 5 (4), 536–544." />
<meta name="citation_reference" content="Madhugiri, R.; Fricke, M.; Marz, M.; Ziebuhr, J., RNA structure analysis of alphacoronavirus terminal genome regions. Virus Res. 2014, 194, 76–89." />
<meta name="citation_reference" content="Kalvari, I.; Argasinska, J.; Quinones-Olvera, N.; Nawrocki, E. P.; Rivas, E.; Eddy, S. R.; Bateman, A.; Finn, R. D.; Petrov, A. I., Rfam 13.0: shifting to a genome-centric resource for noncoding RNA families. Nucleic Acids Res. 2018, 46 (D1), D335–D342." />
<meta name="citation_reference" content="Nawrocki, E. P.; Eddy, S. R., Infernal 1.1: 100-fold faster RNA homology searches. Bioinformatics 2013, 29 (22), 2933–5." />
<meta name="citation_reference" content="Yang, D.; Liu, P.; Wudeck, E. V.; Giedroc, D. P.; Leibowitz, J. L., SHAPE analysis of the RNA secondary structure of the Mouse Hepatitis Virus 5’ untranslated region and N-terminal nsp1 coding sequences. Virology 2015, 475, 15–27." />
<meta name="citation_reference" content="Elbe, S.; Buckland-Merrett, G., Data, disease and diplomacy: GISAID’s innovative contribution to global health. Glob Chall 2017, 1 (1), 33–46." />
<meta name="citation_reference" content="Do, C. B.; Woods, D. A.; Batzoglou, S., CONTRAfold: RNA secondary structure prediction without physics-based models. Bioinformatics 2006, 22 (14), e90–8." />
<meta name="citation_reference" content="Hamada, M.; Sato, K.; Asai, K., Prediction of RNA secondary structure by maximizing pseudo-expected accuracy. BMC Bioinformatics 2010, 11, 586." />
<meta name="citation_reference" content="Herold, J.; Siddell, S. G., An ‘elaborated’ pseudoknot is required for high frequency frameshifting during translation of HCV 229E polymerase mRNA. Nucleic Acids Res 1993, 21 (25), 5838–42." />
<meta name="citation_reference" content="Gruber, A. R.; Findeiß, S.; Washietl, S.; Hofacker, I. L.; Stadler, P. F., RNAz 2.0: improved noncoding RNA detection. Pac. Symp. Biocomput. 2010, 69–79." />
<meta name="citation_reference" content="Rivas, E.; Clements, J.; Eddy, S. R., Estimating the power of sequence covariation for detecting conserved RNA structure. Bioinformatics 2020." />
<meta name="citation_reference" content="Li, L.; Kang, H.; Liu, P.; Makkinje, N.; Williamson, S. T.; Leibowitz, J. L.; Giedroc, D. P., Structural lability in stem-loop 1 drives a 5’ UTR-3’ UTR interaction in coronavirus replication. J. Mol. Biol. 2008, 377 (3), 790–803." />
<meta name="citation_reference" content="Liu, P.; Li, L.; Millership, J. J.; Kang, H.; Leibowitz, J. L.; Giedroc, D. P., A U-turn motifcontaining stem-loop in the coronavirus 5’ untranslated region plays a functional role in replication. RNA 2007, 13 (5), 763–780." />
<meta name="citation_reference" content="Kelly, J. A.; Dinman, J. D., Structural and functional conservation of the programmed-1 ribosomal frameshift signal of SARS-CoV-2. bioRxiv 2020." />
<meta name="citation_reference" content="Goebel, S. J.; Hsue, B.; Dombrowski, T. F.; Masters, P. S., Characterization of the RNA components of a putative molecular switch in the 3’ untranslated region of the murine coronavirus genome. J. Virol. 2004, 78 (2), 669–682." />
<meta name="citation_reference" content="Goebel, S. J.; Miller, T. B.; Bennett, C. J.; Bernard, K. A.; Masters, P. S., A hypervariable region within the 3’ cis-acting element of the murine coronavirus genome is nonessential for RNA synthesis but affects pathogenesis. J. Virol. 2007, 81 (3), 1274–1287." />
<meta name="citation_reference" content="Robertson, M. P.; Igel, H.; Baertsch, R.; Haussler, D.; Ares, M., Jr.; Scott, W. G., The structure of a rigorously conserved RNA element within the SARS virus genome. PLoS Biol. 2005, 3 (1), e5." />
<meta name="citation_reference" content="Jonassen, C. M.; Jonassen, T. O.; Grinde, B., A common RNA motif in the 3’ end of the genomes of astroviruses, avian infectious bronchitis virus and an equine rhinovirus. J. Gen. Virol. 1998, 79 (Pt 4), 715–718." />
<meta name="citation_reference" content="Guan, B.-J.; Su, Y.-P.; Wu, H.-Y.; Brian, D. A., Genetic evidence of a long-range RNA-RNA interaction between the genomic 5’ untranslated region and the nonstructural protein 1 coding region in murine and bovine coronaviruses. J. Virol. 2012, 86 (8), 4631–4643." />
<meta name="citation_reference" content="Kappel, K.; Zhang, K.; Su, Z.; Kladwang, W.; Li, S.; Pintilie, G.; Topkar, V. V.; Rangan, R.; Zheludev, I. N.; Watkins, A. M.; Yesselman, J. D.; Chiu, W.; Das, R., Ribosolve: Rapid determination of three-dimensional RNA-only structures. bioRxiv 2019." />
<meta name="citation_reference" content="Zhang, K.; Li, S.; Kappel, K.; Pintilie, G.; Su, Z.; Mou, T.-C.; Schmid, M. F.; Das, R.; Chiu, W., Cryo-EM structure of a 40 kDa SAM-IV riboswitch RNA at 3.7 Å resolution. Nat. Commun. 2019, 10 (1), 5511." />
<meta name="citation_reference" content="Edgar, R. C., MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004, 32 (5), 1792–1797." />
<meta name="citation_reference" content="Li, W.; Jaroszewski, L.; Godzik, A., Clustering of highly homologous sequences to reduce the size of large protein databases. Bioinformatics 2001, 17 (3), 282–283." />
<meta name="citation_reference" content="Sievers, F.; Wilm, A.; Dineen, D.; Gibson, T. J.; Karplus, K.; Li, W.; Lopez, R.; McWilliam, H.; Remmert, M.; Söding, J.; Thompson, J. D.; Higgins, D. G., Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol. Syst. Biol. 2011, 7, 539." />
<meta name="citation_reference" content="Katoh, K.; Frith, M. C., Adding unaligned sequences into an existing alignment using MAFFT and LAST. Bioinformatics 2012, 28 (23), 3144–3146." />
<meta name="citation_reference" content="Washietl, S.; Hofacker, I. L., Consensus folding of aligned sequences as a new measure for the detection of functional RNAs by comparative genomics. J Mol Biol 2004, 342 (1), 19–30." />
<meta name="citation_reference" content="Rivas, E.; Clements, J.; Eddy, S. R., A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs. Nat Methods 2017, 14 (1), 45–48." />
<meta name="citation_reference" content="Cock, P. J.; Antao, T.; Chang, J. T.; Chapman, B. A.; Cox, C. J.; Dalke, A.; Friedberg, I.; Hamelryck, T.; Kauff, F.; Wilczynski, B.; de Hoon, M. J., Biopython: freely available Python tools for computational molecular biology and bioinformatics. Bioinformatics 2009, 25 (11), 1422–3." />
<meta name="citation_reference" content="Bernhart, S. H.; Hofacker, I. L.; Stadler, P. F., Local RNA base pairing probabilities in large sequences. Bioinformatics 2006, 22 (5), 614–5." />
<meta name="citation_reference" content="Kearse, M.; Moir, R.; Wilson, A.; Stones-Havas, S.; Cheung, M.; Sturrock, S.; Buxton, S.; Cooper, A.; Markowitz, S.; Duran, C.; Thierer, T.; Ashton, B.; Meintjes, P.; Drummond, A., Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 2012, 28 (12), 1647–1649." />
<meta name="twitter:title" content="RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="As the COVID-19 outbreak spreads, there is a growing need for a compilation of conserved RNA genome regions in the SARS-CoV-2 virus along with their structural propensities to guide development of antivirals and diagnostics. Using sequence alignments spanning a range of betacoronaviruses, we rank genomic regions by RNA sequence conservation, identifying 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome sequences available near the beginning of the COVID-19 outbreak. We then confirm the conservation of the majority of these genome regions across 739 SARS-CoV-2 sequences reported to date from the current COVID-19 outbreak, and we present a curated list of 30 ‘SARS-related-conserved’ regions. We find that known RNA structured elements curated as Rfam families and in prior literature are enriched in these conserved genome regions, and we predict additional conserved, stable secondary structures across the viral genome. We provide 106 ‘SARS-CoV-2-conserved-structured’ regions as potential targets for antivirals that bind to structured RNA. We further provide detailed secondary structure models for the 5’ UTR, frame-shifting element, and 3’ UTR. Last, we predict regions of the SARS-CoV-2 viral genome have low propensity for RNA secondary structure and are conserved within SARS-CoV-2 strains. These 59 ‘SARS-CoV-2-conserved-unstructured’ genomic regions may be most easily targeted in primer-based diagnostic and oligonucleotide-based therapeutic strategies." />
<meta name="og-title" property="og:title" content="RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="As the COVID-19 outbreak spreads, there is a growing need for a compilation of conserved RNA genome regions in the SARS-CoV-2 virus along with their structural propensities to guide development of antivirals and diagnostics. Using sequence alignments spanning a range of betacoronaviruses, we rank genomic regions by RNA sequence conservation, identifying 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome sequences available near the beginning of the COVID-19 outbreak. We then confirm the conservation of the majority of these genome regions across 739 SARS-CoV-2 sequences reported to date from the current COVID-19 outbreak, and we present a curated list of 30 ‘SARS-related-conserved’ regions. We find that known RNA structured elements curated as Rfam families and in prior literature are enriched in these conserved genome regions, and we predict additional conserved, stable secondary structures across the viral genome. We provide 106 ‘SARS-CoV-2-conserved-structured’ regions as potential targets for antivirals that bind to structured RNA. We further provide detailed secondary structure models for the 5’ UTR, frame-shifting element, and 3’ UTR. Last, we predict regions of the SARS-CoV-2 viral genome have low propensity for RNA secondary structure and are conserved within SARS-CoV-2 strains. These 59 ‘SARS-CoV-2-conserved-unstructured’ genomic regions may be most easily targeted in primer-based diagnostic and oligonucleotide-based therapeutic strategies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.27.012906v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.27.012906v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.27.012906v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1231855" />
    <title>RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"not3aLUzt0gKz1NKnzcpsDXcYxfpUUzeC8jhT-ypM9o","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1231855","apath":"\/biorxiv\/early\/2020\/04\/06\/2020.03.27.012906.atom","pisa":"biorxiv;2020.03.27.012906v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.27.012906v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.27.012906v2","title":"RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses","identifier":"node\/1231855"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x2wzApoQ3Wd57msqobeIrgEGJeJVgr7rNMfhZblD8gfpYuFtlLYs0GEPBBP5rCgZJ79eTEfJce2OQkl45r4iR2JJvh1sbtytN3eb1YCMlGVVpmjRNhlyadHj79\\\\\\\/xhXwIJNAkpSDghA4VNXcKgJ4qsjTT\\\\\\\/e\\\\\\\/a6qm06dC+ZSRSLSi9h5H20+r4Jt+hVqtW3DV0o5lZGLQk7vcF6AIei+rMXeYyfySYjP\\\\\\\/tCN\\\\\\\/6LZtZ+Vy8kgFsuvbPtWYYuBrEw73MiKPw\\\\\\\/Tf9oikeVPbZhe0J9GfdkwhWgfDkqbirCnTFOqETXzwJ7gVZLen5f5sGjFan994SrVIrOIGDIBWPy1w0mhMKhtp+2YGvZURdNQkjR9PY\\\\\\\/ntBq1kwcFgMivVlhwudBGMYncyX\\\\\\\/3oQ\\\\\\\/xk51eaAKB+iP0q6pEm+CfyH3Ifxwij+5qfeib6B75BLkz2n9Cd\\\\\\\/QTRjA6g14RwuLKVJJ09rxdQgRycVIjmnqwR4j7KPupuWc7Uf\\\\\\\/mp70XVj8R7QHvqLyuyKJvastTzRHgoy9YY50Gr2i0tXOelJLM7qBMP8U3tBS2XtnZaYLiB176Oe5WKb8R+KXgkw6QlGtCdZJmHast53\\\\\\\/e5Rh\\\\\\\/FZT9zXVhZW2o2nQLxon6u0MEyijiyD4rErRnvIXFC58cUNAbeskVz7yoxazPIYdblLw5E2DskFtg9c9lkvSMpDA7oiqLGGeMI9Fod1qInus8GN3YUrV7UrTeALIWWALrVZFkl9hdnSY9y\\\\\\\/a2gky6hjUs0ORCWaBQSksQexQVl6TTnq\\\\\\\/+C9AyepL3d7Jw58ay7j9dsKVMd6Nh7a+Stvy8rsnAd1uvWUzskyf2xNwuBzTPjASxb1pYYb\\\\\\\/lSMnWNQROVXqhIGyrpofsrg49ftKXoN+izqUAh3L\\\\\\\/juzZUwg01KvBQd9+PKtrpoHneoQbshUFso2koT9pnBp43RqrdeTT1FmYDpJLHoYLyFRH0mKlvtwwal4kIu9Utsi3z4jkLteVfiwX92xNHTrvn77aBdByI1V\\\\\\\/OR\\\\\\\/llw8Nx4dwkwietZJqQlbFcExSFU2w39PD+lMedl11yce0Yf7OUjpzXelXsviZr9Aiuy\\\\\\\/IEt8umJYi2mgBizhspYnf2S\\\\\\\/dUsBF8xtA\\\\\\\/K1XDmnGJ6u\\\\\\\/l23Qk+vCSekXjYZ1J8rNQOEjfxq5AFtwEPep60\\\\\\\/os74Uzde5wXgC6jZEuPIzc5GDFjKRZcc7fvMSKCxY1ePBZxDglbgRhg1CiumLRJBqbtHYnZ011FjPaMapBjHPBOYHZ13wwN4n4ciFPoDJHV02GrrOHcqJtG00q4l2Lsi8m+kXGPHQT5qH+J4qwBgiM39Oxs8xqfJCizwa+N6lF0G3EU\\\u0022,\\\u0022iv\\\u0022:\\\u0022gq5Qab4kTgSiQZfKXv0fFA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00221384a1574affac42d840571cea01348b50b6abb880fcdbe66abc0b739fdfe211\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.27.012906v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1231855 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.27.012906v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5sGVpgdmTwBEyduvoL08SqjscKuJSJPncx5wp2TWhKw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231855" id="node1231855" data-pisa="biorxiv;2020.03.27.012906v2" data-pisa-master="biorxiv;2020.03.27.012906" data-apath="/biorxiv/early/2020/04/06/2020.03.27.012906.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0960-0825" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ramya</span> <span class="nlm-surname">Rangan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9572-0574" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ivan N.</span> <span class="nlm-surname">Zheludev</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7497-0972" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Rhiju</span> <span class="nlm-surname">Das</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.27.012906 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1231855" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ramya Rangan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Biophysics Program, Stanford University</span>, Stanford CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramya%2BRangan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rangan%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARamya%2BRangan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0960-0825" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ramya Rangan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan N. Zheludev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biochemistry, Stanford University School of Medicine</span>, Stanford CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BN.%2BZheludev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheludev%20IN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvan%2BN.%2BZheludev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9572-0574" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivan N. Zheludev</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rhiju Das </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Biophysics Program, Stanford University</span>, Stanford CA 94305</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biochemistry, Stanford University School of Medicine</span>, Stanford CA 94305</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Physics, Stanford University</span>, Stanford CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rhiju%2BDas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Das%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARhiju%2BDas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7497-0972" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rhiju Das</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rhiju@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">rhiju@stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.27.012906v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1231855" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1231855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.012906v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1231855" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1231855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.012906v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1231855" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1231855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.012906v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1231855" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1231855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.012906v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1231855" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1231855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.27.012906v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1231855" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1231855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">As the COVID-19 outbreak spreads, there is a growing need for a compilation of conserved RNA genome regions in the SARS-CoV-2 virus along with their structural propensities to guide development of antivirals and diagnostics. Using sequence alignments spanning a range of betacoronaviruses, we rank genomic regions by RNA sequence conservation, identifying 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome sequences available near the beginning of the COVID-19 outbreak. We then confirm the conservation of the majority of these genome regions across 739 SARS-CoV-2 sequences reported to date from the current COVID-19 outbreak, and we present a curated list of 30 ‘SARS-related-conserved’ regions. We find that known RNA structured elements curated as Rfam families and in prior literature are enriched in these conserved genome regions, and we predict additional conserved, stable secondary structures across the viral genome. We provide 106 ‘SARS-CoV-2-conserved-structured’ regions as potential targets for antivirals that bind to structured RNA. We further provide detailed secondary structure models for the 5’ UTR, frame-shifting element, and 3’ UTR. Last, we predict regions of the SARS-CoV-2 viral genome have low propensity for RNA secondary structure and are conserved within SARS-CoV-2 strains. These 59 ‘SARS-CoV-2-conserved-unstructured’ genomic regions may be most easily targeted in primer-based diagnostic and oligonucleotide-based therapeutic strategies.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) has caused a rapidly expanding global pandemic, with the COVID-19 outbreak responsible at this time for over 600,000 cases and 25,000 deaths. The emergence of this pandemic has revealed an urgent need for diagnostic and antiviral strategies targeting SARS-CoV-2. Like other coronaviruses, SARS-CoV-2 is a positive sense RNA virus, with a large RNA genome approaching nearly 30 kilobases in length. Its RNA genome contains protein-coding open reading frames (ORFs) for the viral replication machinery, structural proteins, and accessory proteins. The genome additionally harbors various <em>cis</em>-acting RNA elements, with structures in the 5’ and 3’ untranslated region (UTRs) guiding viral replication, RNA synthesis and viral packaging.<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> Conserved RNA elements offer compelling targets for diagnostics. In addition, such RNA elements may be useful targets for antivirals, a concept supported by the recent development of antisense oligonucleotide therapeutics and small-molecule RNA-targeting drugs for a variety of targets across infectious and chronic diseases.<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>–<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup></p><p id="p-4">Conserved structured RNA regions have already been shown to play critical functional roles in the life cycles of coronaviruses. Most coronavirus 5’ UTR’s harbor at least four stem loops, with many showing heightened sequence conservation across betacoronaviruses, and various stems demonstrating functional roles in viral replication.<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup> Furthermore, RNA secondary structure in the 5’ UTR exposes a critical sequence motif, the transcriptional regulatory sequence (TRS), that forms long-range RNA interactions necessary for facilitating the discontinuous transcription characteristic to coronaviruses.<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup> Beyond the 5’ UTR, the frame-shifting element (FSE) in the first protein-coding ORF (ORF1 ab) includes a pseudoknot structure that is necessary for the production of ORF1 a and ORF1 b from two overlapping reading frames via programmed ribosomal −1 frame-shifting.<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup> In the 3’ UTR, mutually exclusive RNA structures including the 3’ UTR pseudoknot control various stages of the RNA synthesis pathway.<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup></p><p id="p-5">Beyond these canonical structured regions, the RNA structure of the SARS-CoV-2 genome remains mostly unexplored. Unbiased discovery of other conserved regions and/or structured regions in the virus has the potential to uncover further functional <em>cis</em>-acting RNA elements. Here, we analyze RNA sequence conservation across SARS-related betacoronaviruses and currently available SARS-CoV-2 sequences, and we identify structured and unstructured regions that are conserved in each sequence set; these intervals can provide starting points for a variety of diagnostic and antiviral development strategies (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). To identify structured regions, we predict maximum expected accuracy structures around conserved regions and report the support of these single structures from predictions of each RNA’s structural ensemble. We additionally identify thermodynamically stable secondary structures across the whole genome, finding that currently known structures fall within these predictions, but also identifying various new candidate structured regions. We pinpoint unstructured genome intervals by identifying bases with low average base-pairing probabilities. Finally, we present secondary structure models for key RNA structural elements of SARS-CoV-2 annotated in the betacoronavirus family.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="We aim to provide a series of genome regions in SARS-CoV-2 that are useful for a variety of diagnostic and therapeutic strategies, including regions that are (A) conserved in SARS-related betacoronaviruses and SARS-CoV-2 sequences (Table 1), (B) regions that are structured and conserved in SARS-CoV-2 sequences (Table 2), and (C) regions that are unstructured and conserved in SARS-CoV-2 sequences (Table 3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-680506357" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;We aim to provide a series of genome regions in SARS-CoV-2 that are useful for a variety of diagnostic and therapeutic strategies, including regions that are (A) conserved in SARS-related betacoronaviruses and SARS-CoV-2 sequences (Table 1), (B) regions that are structured and conserved in SARS-CoV-2 sequences (Table 2), and (C) regions that are unstructured and conserved in SARS-CoV-2 sequences (Table 3).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F1.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F1.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-6" class="first-child">We aim to provide a series of genome regions in SARS-CoV-2 that are useful for a variety of diagnostic and therapeutic strategies, including regions that are (A) conserved in SARS-related betacoronaviruses and SARS-CoV-2 sequences (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), (B) regions that are structured and conserved in SARS-CoV-2 sequences (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), and (C) regions that are unstructured and conserved in SARS-CoV-2 sequences (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>RNA sequence conservation in SARS-related betacoronaviruses and SARS-CoV-2</h3><p id="p-7">To identify potential regions of conserved RNA secondary structure in the virus, we located stretches of the SARS-CoV-2 genome with high RNA sequence conservation across SARS-related betacoronavirus full genome sequences. By identifying regions with high RNA sequence conservation as a first step, we reasoned that we would be more likely to filter for functionally relevant structures that must be conserved through virus evolution and thereby discover targets that are potentially less likely to develop resistance against therapeutics or to escape diagnosis as the virus evolves. To ensure reasonable numbers of sequences while still focusing on conservation and structure patterns most relevant to the current pandemic, we chose to analyze not all betacoronaviruses but a subgroup of SARS-related betacoronaviruses. These include SARS, SARS-CoV-2, and SARS-related bat coronaviruses, but not MERS, MHV, or other betacoronaviruses which have been classified into distinct subgroups based on different sequence and structure features in, for example, their 5’ UTR’s.<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup></p><p>We carried out this analysis beginning with three different sequence alignments. Each captures a range of complete genome sequences across the SARS-related betacoronviruses, but differ in the total number of sequences and in the redundancy of those sequences, as follows:
</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-9">The first multiple sequence alignment (<strong>SARSr-MSA-1</strong>) was computed by aligning sequences curated by Ceraolo and Giorgi,<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> filtered by including only the reference genome sequence NC_0405512.2<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup> from the SARS-CoV-2 sequence set, removing the two MERS sequences, and leaving in all remaining betacoronavirus whole genome sequences. This alignment captures a range of SARS-related bat coronavirus and SARS sequences with only 11 sequences. These sequences correspond well to the SARS-related group defined in Gorbalenya, Baker, <em>et al<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup></em></p></li><li id="list-item-2"><p id="p-10">The second MSA (<strong>SARSr-MSA-2</strong>) was obtained from BLAST by searching for the 100 complete genome sequences closest to the SARS-CoV-2 reference genome. This alignment captures a larger set of SARS-CoV-2, SARS, and bat coronavirus sequences than SARSr-MSA-1 but includes many sequences with high pairwise similarity.</p></li><li id="list-item-3"><p id="p-11">The final MSA (<strong>SARSr-MSA-3</strong>) was obtained by locating all complete genome betacoronavirus sequences from the NCBI database, and removing mutually similar sequences with a 99% sequence conservation cutoff. With 180 sequences with at most 99% pairwise sequence similarity, this MSA captures a broader set of betacoronaviruses than SARSr-MSA-1 and SARSr-MSA-2 but is more challenging to align due to higher sequence diversity.</p></li></ol><p id="p-12">We computed conserved regions as contiguous stretches of 15 nucleotides or longer that were 100% conserved (cutoff for SARSr-MSA-1), 98% conserved (cutoff for SARSr-MSA-2), or 54% conserved (cutoff for SARSr-MSA-3). Searching for conserved regions of 15 nucleotides or more enables the design of antisense oligonucleotides that fall within these stretches. The sequence conservation cutoffs chosen ensured that at least 75 candidate conserved stretches were used for further structure analysis for each MSA. When calculating sequence conservation at the 5’ and 3’ sequence ends of the sequence, we did not include sequences that included only leading or trailing sequence deletions up to that point to avoid sequencing artefacts.</p><p id="p-13">In <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>, we depict conserved regions (100% conservation cutoff, SARSr-MSA-1) alongside the genome coordinates for the reference SARS-CoV-2 sequence. We observe intervals of conservation in the 5’ UTR and 3’ UTR genome regions, as expected based on prior work demonstrating sequence conservation surrounding structured RNA elements in these regions,<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> but we also noted stretches of RNA sequence conservation within some viral ORFs.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In black we annotate SARSr-MSA-1 conserved regions of the genome, superimposed on SARS-CoV-2 genome ORFs. We depict the top secondary structures as ranked by Matthews correlation coefficient that overlap with these conserved regions, ordered from A to E. Regions A to E are annotated on the genome in yellow and are located at genome positions: A:13743-13798, B:17511-17566, C:28990-29054, D:172-236, E:26-109. Secondary structures are colored by sequence conservation in SARSr-MSA-1 (cyan = more conserved, purple = less conserved). In magenta are depicted curated Rfam families present in coronaviruses, including the frame-shifting element (FSE), the 3&#x2019; UTR pseudoknot (PK3), and the 3&#x2019; stem-loop II-like motif (s2m). Figures prepared in Geneious41 and draw_rna (https://github.com/DasLab/draw_rna)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-680506357" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;In black we annotate SARSr-MSA-1 conserved regions of the genome, superimposed on SARS-CoV-2 genome ORFs. We depict the top secondary structures as ranked by Matthews correlation coefficient that overlap with these conserved regions, ordered from A to E. Regions A to E are annotated on the genome in yellow and are located at genome positions: A:13743-13798, B:17511-17566, C:28990-29054, D:172-236, E:26-109. Secondary structures are colored by sequence conservation in SARSr-MSA-1 (cyan = more conserved, purple = less conserved). In magenta are depicted curated Rfam families present in coronaviruses, including the frame-shifting element (FSE), the 3&#x2019; UTR pseudoknot (PK3), and the 3&#x2019; stem-loop II-like motif (s2m). Figures prepared in Geneious&lt;sup&gt;41&lt;/sup&gt; and draw_rna (https://github.com/DasLab/draw_rna).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F2.medium.gif" width="440" height="258"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F2.medium.gif" width="440" height="258"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-14" class="first-child">In black we annotate SARSr-MSA-1 conserved regions of the genome, superimposed on SARS-CoV-2 genome ORFs. We depict the top secondary structures as ranked by Matthews correlation coefficient that overlap with these conserved regions, ordered from A to E. Regions A to E are annotated on the genome in yellow and are located at genome positions: A:13743-13798, B:17511-17566, C:28990-29054, D:172-236, E:26-109. Secondary structures are colored by sequence conservation in SARSr-MSA-1 (cyan = more conserved, purple = less conserved). In magenta are depicted curated Rfam families present in coronaviruses, including the frame-shifting element (FSE), the 3’ UTR pseudoknot (PK3), and the 3’ stem-loop II-like motif (s2m). Figures prepared in Geneious<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> and draw_rna (<a href="https://github.com/DasLab/draw_rna">https://github.com/DasLab/draw_rna</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">Interestingly, in SARSr-MSA-1 and SARSr-MSA-2 we found that conserved stretches overlapped with previously curated Rfam<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup> families for <em>Coronaviridae</em> RNA secondary structures: the frameshifting stimulation element (Rfam family RF00507), the 3’ UTR pseudoknot (Rfam family RF00165), and the 3’ stem-loop II-like motif (Rfam family RF00164) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). Locations for the frameshifting stimulation element, 3’ UTR pseudoknot, and 3’ stem-loop II-like motif were confirmed using Infernal,<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup> with all regions discovered at an E&lt;10<sup>-4</sup> threshold. We also found overlap between conserved stretches and additional 5’ UTR structures that have been established for previous coronaviruses, including the original SARS virus, including stem loops 2-3 (SL2-3) and stem loop 5 (SL5).<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup> These five known RNA structures overlap with conserved regions more than expected; in 10,000 random trials, the chance that five randomly chosen intervals of these lengths all overlap with the conserved regions from SARSr-MSA-1 or SARSr-MSA-2 is less than 0.0003. The enrichment of known RNA structures in these conserved regions suggests that other conserved regions may also harbor RNA structures.</p><p id="p-16">To further tighten this list of conserved sequences to ones most relevant to the current COVID-19 outbreak, we analyzed whether sequence regions conserved across SARS and bat coronaviruses remain conserved in the SARS-Cov-2 strains, most of which emerged after our analysis above (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>). We determined the conservation of conserved genome regions from SARSr-MSA-1 across SARS-CoV-2 sequences as of deposition date 03-18-20. For this analysis, we obtained two wholegenome multiple sequence alignments, keeping only full-length genome sequences of at least 29,000 nucleotides in both cases: the first includes 103 NCBI sequences (<strong>SARS-CoV-2-MSA-1</strong>), and the second includes 739 sequences deposited to GISAID<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup> (<strong>SARS-CoV-2-MSA-2</strong>). We noted conserved regions in the betacoronavirus alignment SARSr-MSA-1 were more likely to be at least 99% conserved in both SARS-CoV-2-MSA-1 and SARS-CoV-2-MSA-2 than random intervals of the same size (binomial test p-value &lt; 1e-5). <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> lists these regions, which we term the <strong>SARS-related-conserved</strong> regions. These genome regions are conserved across the betacoronavirus sequences in SARSr-MSA-1 and have at least 99% sequence conservation across whole-genome sequences from the SARS-CoV-2 outbreak as of March 18, 2020 (SARS-CoV-2-MSA-2).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1237210/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1237210/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1237210" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <span class="caption-title">SARS-related-conserved.</span><p id="p-17" class="first-child">Conserved regions across SARSr-MSA-1 and SARS-CoV-2-MSA-2. All intervals are at least 90% conserved across the SARS and bat coronavirus sequences in SARSr-MSA-1, have length at least 15 nucleotides, and have every position at least 99% conserved in current GISAID SARS-CoV-2 sequences (SARS-CoV-2-MSA-2). Sequence intervals are relative to the reference genome NC_045512.2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Conservation percentages for SARSr-MSA-1, SARSr-MSA-2, SARS-CoV-2-MSA-1, and SARS-CoV-2-MSA-2 are included in Supplementary File 1. We expect that some diagnostic and therapeutic strategies will benefit from focusing on conserved regions across a broad range of betacoronaviruses, whereas others may benefit from focusing on regions conserved only in SARS-CoV-2; we revisit the latter category of SARS-CoV-2-unique regions below.</p></div><div id="sec-4" class="subsection"><h3>Predictions for structured regions in SARS-CoV-2</h3><p id="p-19">The intrinsic RNA structure of a conserved genome region is of interest in current medical research (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>). On one hand, stable secondary structure domains are candidates for harboring stereotyped 3D RNA folds that present targets for small-molecule drug therapeutics. On the other hand, if an RNA region is sufficiently unstructured to allow binding by hybridization probes, antisense oligonucleotides may be used to disrupt these functional structures. Such unstructured stretches may also be more likely to be accessible to diagnostic and antiviral interventions including standard RT-PCR assays.</p><p id="p-20">We used two approaches to make predictions for conserved structured regions in SARS-CoV-2. First, we predicted RNA structures centered on the most sequence-conserved regions of SARS-related betacoronavirus genomes (alignment SARSr-MSA-1). For each conserved stretch (at least 15 nucleotides long, 100% sequence conservation) along with 20 nucleotide flanking windows, we predicted maximum expected accuracy (MEA) secondary structures using Contrafold 2.0.<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup> We then sought to rank sequences based on the predicted probability that the RNA folds into the MEA structure and not other structures. For this ranking, we used the estimated Matthews correlation coefficient (MCC) from each construct’s base-pairing probability matrix.<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup> We note here that while MCC is often used in the RNA structure modeling literature to assess agreement of a prediction with a reference structure, we here use the metric to assess how tightly concentrated the ensemble of predicted secondary structures is to a single predicted secondary structure, the MEA structure. An MEA structure with a higher estimated MCC is expected to have unpaired and paired bases that better align with the construct’s predicted ensemble base-pairing probabilities, lending support to the single-structure MEA prediction. In <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a> regions A-E, we display the five conserved regions with the top maximum expected accuracy (MEA) secondary structures as ranked by the estimated MCC (all regions listed in Supplementary File 1). Regions D and E occurred within the 5’ UTR and correspond to known SARS-related virus stem loops SL5a and SL2, respectively. Interestingly, region A is close to but does not overlap with the frameshifting stimulation element; it lies 200 nucleotides downstream of the FSE and could perhaps be involved in a more elaborate structure, as has been described for human coronavirus 229E and other coronaviruses.<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup></p><p id="p-21">We also sought independent methods to identify thermodynamically stable and conserved RNA structures, without initially guiding the search to focus on extremely sequence-conserved genome regions. We made predictions for structured regions using RNAz<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>, beginning with the betacoronavirus alignment SARSr-MSA-1. RNAz predicts structured regions that are more thermodynamically stable than expected by comparison to random sequences of the same length and sequence composition (z-score), and additionally assesses regions by the support of compensatory and consistent mutations in the sequence alignment (SCI score). These two criteria are combined into a single P-score, which when tested empirically on a set of ncRNAs produced a false-positive rate of 4% at a P&gt;0.5 cutoff and 1% at a P&gt;0.9 cutoff. To predict structured regions across the full viral genome, we scanned the SARSr-MSA-1 alignment in windows of length 120 nucleotides sliding by 40 nucleotides, predicted all RNAz hits in the plus strand at a P&gt;0.5 cutoff, clustered the resulting hits to generate maximally contiguous loci of the genome with predicted structure, and filtered results to only include loci with at least one window with a P&gt;0.9 structure prediction.</p><p id="p-22">The RNAz approach led to the prediction of 44 structured genome loci comprising 117 windows with predicted structure (P&gt;0.9), with these loci covering 46% of the SARS-CoV-2 genome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). We found that five canonical RNA structures (the frameshifting element, the 3’ UTR pseudoknot, the 3’ UTR hypervariable region, 5’ UTR SL2-3, and 5’ UTR SL5) were present in these loci. Additionally, conserved SARS-CoV-2 regions overlap significantly with predicted RNAz loci, with 62 of 78 SARS-CoV-2 conserved intervals at a 97% sequence cutoff overlapping by at least 15 nucleotides with RNAz loci. This enrichment is statistically significant (p-value&lt;0.001 from comparisons to 10,000 random placements of conserved intervals). This enrichment also holds when considering overlaps with conserved regions from SARSr-MSA-1; 124 of the 229 SARSr-MSA-1 conserved intervals at a 90% conservation cutoff overlap by at least 15 nucleotides with RNAz loci (p-value 0.0038). This analysis potentially expands the set of conserved structural regions of SARS-CoV-2 beyond known Rfam families and those noted in the literature (full set of RNAz loci in Supplementary File 1). Topscoring structured windows from RNAz that overlap with conserved sequence regions in SARS-CoV-2-MSA-2 for at least 15 nucleotides are included in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>; we termed these <strong>SARS-CoV-2-conserved-structured</strong> regions. Overlapping intervals between the RNAz predictions and conserved sequence regions in SARSr-MSA-1 are included in Supplementary File 1.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structured (cyan) and unstructured (yellow) intervals on the genome ORFs for SARS-CoV-2, predicted from RNAz and a Contrafold 2.0 analysis, respectively. (A-C) highlight the three secondary structures for windows that do not overlap with known Rfam or literature-annotated structures with the highest P-value scores from RNAz (all P&gt;0.9). These windows are located at genome positions 14207-14366 (A), 17126-17245 (B), and 26176-26295 (C). Secondary structures are colored by sequence conservation (cyan = more conserved, purple = less conserved). Figures prepared in Geneious41 and draw_rna (https://github.com/DasLab/draw_rna)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-680506357" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structured (cyan) and unstructured (yellow) intervals on the genome ORFs for SARS-CoV-2, predicted from RNAz and a Contrafold 2.0 analysis, respectively. (A-C) highlight the three secondary structures for windows that do not overlap with known Rfam or literature-annotated structures with the highest P-value scores from RNAz (all P&gt;0.9). These windows are located at genome positions 14207-14366 (A), 17126-17245 (B), and 26176-26295 (C). Secondary structures are colored by sequence conservation (cyan = more conserved, purple = less conserved). Figures prepared in Geneious&lt;sup&gt;41&lt;/sup&gt; and draw_rna (https://github.com/DasLab/draw_rna).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F3.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F3.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-23" class="first-child">Structured (cyan) and unstructured (yellow) intervals on the genome ORFs for SARS-CoV-2, predicted from RNAz and a Contrafold 2.0 analysis, respectively. (A-C) highlight the three secondary structures for windows that do not overlap with known Rfam or literature-annotated structures with the highest P-value scores from RNAz (all P&gt;0.9). These windows are located at genome positions 14207-14366 (A), 17126-17245 (B), and 26176-26295 (C). Secondary structures are colored by sequence conservation (cyan = more conserved, purple = less conserved). Figures prepared in Geneious<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a></sup> and draw_rna (<a href="https://github.com/DasLab/draw_rna">https://github.com/DasLab/draw_rna</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1237216/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1237216/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <span class="caption-title">SARS-CoV-2-conserved-structured.</span><p id="p-24" class="first-child">RNAz windows as scored by the P-value (P&gt;0.9) that overlap with conserved intervals from SARS-CoV-2-MSA-2 (97% conservation cutoff) by at least 15 nucleotides. Sequence intervals are relative to the reference genome NC_045512.2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We sought to further check structured windows reported by RNAz using orthogonal approaches. First, we explored using R-scape to make structure predictions with covariation signal in the sequence alignments. However, we found that the SARSr-MSA-1 alignment had insufficient variation to detect conserved base pairs with covariation, lacking alignment power for all genomic windows.<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> Second, we validated structured window predictions with alifoldz, a program that calculates a z-score for an alignment window by comparing the window’s consensus minimum free energy structure to that of random shuffled alignments. To mirror the RNAz analysis above, we scanned through windows of length 120 nucleotides sliding by 40 nucleotides. We chose a z-score cutoff of – 2.69, which kept only 1% of windows when running alifoldz on all shuffled windows across the genome. This approach led to predicting 228 alifoldz structured windows, overlapping with 104 of the 117 RNAz structured windows (P&gt;0.9 cutoff). This overlap is statistically significant (p-value&lt;1e-05). RNAz structured windows supported by alifoldz analysis are highlighted in Supplementary File 1.</p></div><div id="sec-5" class="subsection"><h3>Conserved unstructured regions of SARS-CoV-2</h3><p id="p-26">We additionally located conserved regions of the viral genome predicted to <em>lack</em> structure, as such regions may be desired targets for some diagnostic and therapeutic approaches (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a>). We scanned the SARS-CoV-2 reference genome in windows of length 120 nucleotides sliding by 40 nucleotides, and for each window, we predicted the base-pair probability matrix with Contrafold 2.0, using these probabilities to assemble average single-nucleotide base pairing probabilities across the genome. In <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, we display the 76 stretches of the genome of length at least 15 nucleotides where every base has average base-pairing probability at most 0.4.</p><p id="p-27">It is interesting to note that some structured 120 nucleotide windows reported by RNAz include these unpaired stretches. A potential explanation for this observation is that such regions encode for well-defined, conserved RNA structures that themselves harbor long unpaired loops to recruit proteins, distal RNA elements, or other molecular machinery.</p><p id="p-28">Overall, we find that 58 of these unpaired stretches have at least 15 nucleotides of overlap with sequence regions that are at least 97% conserved in SARS-CoV-2-MSA-2 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>). These unpaired stretches termed <strong>SARS-CoV-2-conserved-unstructured</strong> regions are listed in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> (overlaps with SARSr-MSA-1 are included in Supplementary File 1.)</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1237214/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1237214/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <span class="caption-title">SARS-CoV-2-conserved-unstructured.</span><p id="p-29" class="first-child">Top unstructured regions (ranked by minimum unpaired probability over the interval, stretch of at least 15 nt) that overlap with conserved intervals from SARS-CoV-2 for at least 15 nt at a 97% sequence conservation cutoff. Sequence intervals are relative to the reference genome NC_045512.2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">As an orthogonal check for the unstructured intervals predicted using Contrafold 2.0 base-pairing probabilities, we used Vienna’s RNAplfold to compute unpaired probabilities for each genome position. In general, we found that RNAplfold predicted lower unpaired probabilities than Contrafold 2.0, with only 10 intervals of length at least 15 nucleotides having at least 0.6 probability of being unpaired, in contrast with the 76 stretches predicted by Contrafold 2.0. Nevertheless, we found that 9 of the 10 intervals predicted by Vienna’s RNAplfold overlap with unpaired intervals predicted from our Contrafold 2.0 analysis (regions listed in Supplementary File 1.)</p></div><div id="sec-6" class="subsection"><h3>Secondary structure models for canonical structured regions of SARS-CoV-2</h3><p id="p-31">Currently known RNA structures that recur across betacoronaviruses provide potential starting points for therapeutic development targeting the SARS-CoV-2 RNA genome. Here, we include secondary structures for the 5’ UTR (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>), frame-shifting element (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a>), and 3’ UTR (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4c</a>) for SARS-CoV-2, built by analyzing homology to literature-annotated structures in related betacoronaviruses. We additionally include computer-readable secondary structures in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and Supplementary File 1. A brief review of salient secondary structure features in these regions and their putative functional roles in the betacoronavirus life cycle follows.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Secondary structure diagrams for A) 5&#x2019; UTR, B) Frameshift element, C) 3&#x2019; UTR. Nucleotides are black if 100% conserved in the SARS, bat, and SARS-CoV-2 sequences in SARSr-MSA-1, and grey otherwise. Special labeled domains are in boldface. Structures are based primarily on manual identification of homology with literature coronavirus structure models. Note that numbering in (C) is relative to 3&#x2019; end of virus sequence. Figures prepared in RiboDraw (https://github.com/ribokit/RiboDraw)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-680506357" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Secondary structure diagrams for A) 5&#x2019; UTR, B) Frameshift element, C) 3&#x2019; UTR. Nucleotides are black if 100% conserved in the SARS, bat, and SARS-CoV-2 sequences in SARSr-MSA-1, and grey otherwise. Special labeled domains are in boldface. Structures are based primarily on manual identification of homology with literature coronavirus structure models. Note that numbering in (C) is relative to 3&#x2019; end of virus sequence. Figures prepared in RiboDraw (https://github.com/ribokit/RiboDraw).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F4.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F4.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-32" class="first-child">Secondary structure diagrams for A) 5’ UTR, B) Frameshift element, C) 3’ UTR. Nucleotides are black if 100% conserved in the SARS, bat, and SARS-CoV-2 sequences in SARSr-MSA-1, and grey otherwise. Special labeled domains are in boldface. Structures are based primarily on manual identification of homology with literature coronavirus structure models. Note that numbering in (C) is relative to 3’ end of virus sequence. Figures prepared in RiboDraw (<a href="https://github.com/ribokit/RiboDraw">https://github.com/ribokit/RiboDraw</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1237213/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1237213/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1237213" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4:</span> <p id="p-33" class="first-child">Sequences and secondary structure dot-brackets for key structured genome regions of SARS-CoV-2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The 5’ UTR includes five confident stem-loop structures (SL1-SL5), with structures verified by chemical mapping experiments in related coronaviruses.<sup><a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup> SL1 and SL2 are conserved across betacoronaviruses, with SL2 having the highest sequence conservation across the 5’ UTR.<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></sup> The high A-U base-pairing content in the SARS-CoV-2 SL1 sequence and the bulged nucleotides align with prior reports that SL1 is relatively thermodynamically unstable to allow for the formation of long-range interactions.<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> SL2 has been shown to be critical for subgenomic RNA synthesis, with mutations in its conserved pentaloop retaining the production of genome-sized RNA, but not subgenomic RNA segments.<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> SL3, conserved only in betacoronaviruses, presents the transcriptionregulating sequence (TRS) that base pairs with one of several complementary sequences in nascent negative-sense strands in a ‘copy-choice mechanism’ that gives rise to discontinuous transcription of subgenomic mRNAs.<sup><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a></sup> SL4 contains a short upstream ORF, here labeled uORF, which precedes the first longer ORF1ab of the genome. The uORF leads to attenuated transcription of ORF1ab that appears helpful but is not essential for viral replication.<sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a></sup> SL5 has been implicated in a potential role in viral packaging, and contains the AUG start codon for long ORF1ab which encodes the viral replicase/transcriptase polyprotein. The SARS-CoV-2 SL5 domain has common features with the domain in other group IIb betacoronaviruses, for instance including UUCGU pentaloops on SL5a and SL5b, and a GNRA tetraloop on SL5c.<sup><a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a></sup> Prior DMS-probing data for Stem 5 in SARS-CoV aligned with the proposed SL5a,b,c structures.<sup><a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">9</a></sup> Two additional stems (SL6 and SL7) are predicted from computer modeling here, but prior literature has not established whether such stems embedded in the coding region are functionally important across betacoronaviruses.</p><p id="p-35">The frameshifting element (FSE) is located in ORF1ab and is involved in regulating a (−1) ribosomal frameshift event that is necessary for producing ORF1b. The FSE consists of a conserved pseudoknot structure that regulates the rate of ribosomal frameshifting at an upstream slippery site.<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a></sup> This domain is nearly exactly conserved between SARS-CoV and SARS-CoV-2, suggesting a similar mechanism for ribosomal pausing and slippage between the two viruses.<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup></p><p id="p-36">The 3’ UTR contains various domains critical for regulating viral RNA synthesis and potentially translation. The most 5’ region of the 3’ UTR includes a switch-like domain involving mutually exclusive formation of a pseudoknot and stem-loop, both of which are essential for viral replication with putative roles in establishing the kinetics of RNA synthesis.<sup><a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup> The hyper-variable region (HVR) is not essential for viral RNA synthesis, as this can be removed while allowing for viral replication in tissue culture; however, viruses without this domain have lower pathogenicity in mice.<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> This domain contains a completely conserved octonucleotide sequence with unconfirmed functional significance. The stem-loop II-like motif (s2m) is another subregion of the HVR that is conserved in SARS-CoV-2 and other coronaviruses. A crystal structure of the SARS s2m domain has been shown to be homologous to an rRNA loop that binds translation initiation proteins, leading this domain to have a proposed role in recruiting host translational machinery.<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> The domain has been proposed to be a selfish element due to its recurrence in numerous virus families outside the <em>Coronaviridae,</em> but its function is not well understood.<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup></p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-37">Understanding the RNA structure of the SARS-CoV-2 genome can guide RNA-targeting interventions and diagnostics. Here we have presented an initial analysis of RNA sequence conservation across betacoronaviruses and current SARS-CoV-2 sequences, predictions for structured and unstructured domains of the viral RNA genome, and homology-derived secondary structure models for classic structured elements of the SARS-CoV-2 genome: the 5’ UTR, the frame-shifting element, and the 3’ UTR. By filtering for sequences that have more than one of these properties, we have curated three sets of RNA genomic regions of potential interest for further structural analysis, which we have termed the SARS-related-conserved, SARS-CoV-2-conserved-structured, and SARS-CoV-2-conserved-unstructured sets. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a> gives a more extensive presentation of how these sets overlap. Our hope is that these steps will provide useful starting points for efforts to develop antivirals and diagnostics that depend on targeting either structured or unstructured viral genomic regions.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="We depict the predicted number of structured, unstructured and conserved intervals for a choice of sequence conservation cutoffs. The SARS-related conserved intervals are all regions of at least 15 nucleotides with each position at least 90% conserved across an alignment of SARS, bat coronavirus, and SARS-CoV-2 sequences (SARSr-MSA-1). The SARS-CoV-2 intervals are regions of at least 15 nucleotides with each position at least 97% conserved across an alignment of currently available SARS-CoV-2 sequences (SARS-CoV-2-MSA-2). Structured intervals are loci predicted from RNAz with some loci containing multiple RNAz windows, and unstructured intervals are stretches of at least 15 nucleotides where all bases have base-pairing probability at most 0.4. All interval intersections are required to have at least 15 nucleotide overlaps, with the number of overlapping intervals listed for each interval type involved in the intersection. Top-scoring structured intervals conserved in SARS-CoV-2 sequences (green) are listed in Table 2. Top-scoring unstructured intervals conserved in SARS-CoV-2 sequences (blue) are listed in Table 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-680506357" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;We depict the predicted number of structured, unstructured and conserved intervals for a choice of sequence conservation cutoffs. The SARS-related conserved intervals are all regions of at least 15 nucleotides with each position at least 90% conserved across an alignment of SARS, bat coronavirus, and SARS-CoV-2 sequences (SARSr-MSA-1). The SARS-CoV-2 intervals are regions of at least 15 nucleotides with each position at least 97% conserved across an alignment of currently available SARS-CoV-2 sequences (SARS-CoV-2-MSA-2). Structured intervals are &lt;em&gt;loci&lt;/em&gt; predicted from RNAz with some &lt;em&gt;loci&lt;/em&gt; containing multiple RNAz windows, and unstructured intervals are stretches of at least 15 nucleotides where all bases have base-pairing probability at most 0.4. All interval intersections are required to have at least 15 nucleotide overlaps, with the number of overlapping intervals listed for each interval type involved in the intersection. Top-scoring structured intervals conserved in SARS-CoV-2 sequences (green) are listed in Table 2. Top-scoring unstructured intervals conserved in SARS-CoV-2 sequences (blue) are listed in Table 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F5.medium.gif" width="440" height="128"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F5.medium.gif" width="440" height="128"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.27.012906/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-38" class="first-child">We depict the predicted number of structured, unstructured and conserved intervals for a choice of sequence conservation cutoffs. The SARS-related conserved intervals are all regions of at least 15 nucleotides with each position at least 90% conserved across an alignment of SARS, bat coronavirus, and SARS-CoV-2 sequences (SARSr-MSA-1). The SARS-CoV-2 intervals are regions of at least 15 nucleotides with each position at least 97% conserved across an alignment of currently available SARS-CoV-2 sequences (SARS-CoV-2-MSA-2). Structured intervals are <em>loci</em> predicted from RNAz with some <em>loci</em> containing multiple RNAz windows, and unstructured intervals are stretches of at least 15 nucleotides where all bases have base-pairing probability at most 0.4. All interval intersections are required to have at least 15 nucleotide overlaps, with the number of overlapping intervals listed for each interval type involved in the intersection. Top-scoring structured intervals conserved in SARS-CoV-2 sequences (green) are listed in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>. Top-scoring unstructured intervals conserved in SARS-CoV-2 sequences (blue) are listed in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The abundant RNA structures involved in the replication cycle of betacoronaviruses present ample opportunities for therapeutic development, but our analysis is not complete. First, while homology to prior structures annotated in betacoronaviruses lends some confidence to the 5’ UTR, frame-shifting element, and 3’ UTR secondary structures presented here, there may still be some inaccuracies. For instance, SL6 and SL7 in the 5’ UTR are built based on computer modeling. As another example, the frame-shifting element structure presented here differs in two base pairs compared to that presented by Kelly and Dinman.<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup> Additional biochemical and genetic verification, particularly through compensatory mutagenesis, will be needed to further assess these structures.</p><p id="p-40">Beyond the secondary structures highlighted here, prior work has pinpointed a variety of RNA-RNA interactions important to the betacoronavirus life cycle. Long-range interactions between the 5’ UTR and 3’ UTR have been implicated in RNA synthesis, for instance with mutations in the 5’ UTR SL1 only supporting viral replication when co-evolving with specific mutations in the 3’ UTR.<sup><a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> Such long-range RNA-RNA interactions would be missed in our analyses above which have focused on shorter windows. In related coronaviruses, RNA structures that act as packaging signals have been identified in genomic ORFs.<sup><a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a></sup> Such packaging signals may reside in regions identified here from our RNAz computational analysis or may have been missed.</p><p id="p-41">We believe a more thorough structure/function analysis of the virus should be obtainable with recent experimental technologies that integrate multidimensional chemical mapping, electron microscopy, and computer modeling.<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>–<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> New candidates for structured RNA elements in SARS-CoV-2 could play various functional roles, perhaps regulating viral packaging, replication, RNA synthesis, or translation initiation. To further improve these structure predictions, information about protein-binding events should be integrated, and biochemical assays can be conducted with these proteins present or even in cells. Accounting for protein-binding events will be critical to completing a picture of accessible and structured RNA sites.</p><p id="p-42">The secondary structures predicted here present a reasonable starting point for 3D modeling of RNA-only structures in various regions, including the 5’ UTR stem loop 5, the frame-shifting element, the 3’ UTR pseudoknot, and the 3’ stem-loop II-like motif. Furthermore, these regions and novel predicted structures can serve as candidates for RNA-only structure determination. Such 3D structures have the potential to reveal well-defined 3D folds with conserved binding domains for small-molecule drugs, potentially presenting alternative approaches for targeting SARS-CoV-2.</p></div><div class="section" id="sec-8"><h2 class="">Methods</h2><div id="sec-9" class="subsection"><h3>Conservation analysis</h3><p>Three alignments for SARS-related viruses were prepared:
</p><ol class="list-ord " id="list-2"><li id="list-item-4"><p id="p-44">SARSr-MSA-1 was generated by re-aligning the sequences curated by Ceraolo and Giorgi<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup> with MUSCLE<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> using default alignment settings, excluding all non-reference genome copies of the SARS-CoV-2 sequence and excluding MERS sequences JX869059.2 and KT368829.1.</p></li><li id="list-item-5"><p id="p-45">SARSr-MSA-2 was generated by downloading the MSA provided by BLAST for the top 100 complete genome sequences closest to the SARS-CoV-2 reference genome.</p></li><li id="list-item-6"><p id="p-46">SARSr-MSA-3 was generated by obtaining all complete betacoronavirus genome sequences available from the NCBI database, removing mutually similar sequences using a 99% cutoff with CD-HIT-EST<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>, and computing an MSA with Clustal Omega<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup> using default settings.</p></li></ol><p>Two alignments for SARS-CoV-2 sequences were prepared:
</p><ol class="list-ord " id="list-3"><li id="list-item-7"><p id="p-48">SARS-CoV-2-MSA-1 was generated by downloading the MSA provided by NCBI for the 103 whole-genome SARS-CoV-2 sequences deposited as of 03-18-20.</p></li><li id="list-item-8"><p id="p-49">SARS-CoV-2-MSA-2 was generated from 739 GISAID<sup><a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup> sequences, including all sequences described in the Nextstrain project metadata file as of 03-18-20 (<a href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</a>). Sequences were aligned using MAFFT<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> with the --add flag to add GISAID sequences to seed alignment SARS-CoV-2-MSA-1, and the sequences in SARS-CoV-2-MSA-1 were subsequently removed to avoid duplicates.</p></li></ol><p id="p-50">All alignments are included in the associated GitHub repository: <a href="https://github.com/DasLab/SARSCoV2_Secstruct_Cons">https://github.com/DasLab/SARSCoV2_Secstruct_Cons</a>.</p></div><div id="sec-10" class="subsection"><h3>Analysis of structured elements</h3><p id="p-51">To identify regions in the SARS-CoV-2 reference genome NC_0405512.2<sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup> that were matches to Rfam<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a></sup> families, we used Infernal<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup> to build covariance models from Rfam families RR0164, RR0165, and RR0507 with cmbuild, and we ran cmscan to find hits with an E&lt;1e-04 threshold.</p><p id="p-52">MEA structures were computed using Contrafold 2.0<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup> for 20 nucleotide flanking windows around conserved intervals in SARS-CoV-2. To estimate the Matthews correlation coefficient of these single-structure predictions, we computed the pseudo-expected Matthews correlation coefficient as described in Hamada, et al.<sup><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a></sup> Base-pairing probability matrices were computed Contrafold 2.0, and these were then used to calculated the expected number of true positive, true negative, false positive, and false negative base pairs. These computations were carried out using the Arnie package (<a href="https://github.com/DasLab/arnie">https://github.com/DasLab/arnie</a>).</p><p id="p-53">RNAz<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup> structures were predicted in windows of the SARS-CoV-2 genome using the SARSr-MSA-1 alignment. We used rnazWindow.pl to compile alignment windows across SARSr-MSA-1 with at least 4 sequences in each window, using a window size of 120 nucleotides sliding by 40 nucleotides, and using default settings otherwise. RNAz hits were computed at the P&gt;0.5 threshold for the forward strand with z-scores computed without a shuffled sequence background for efficiency, using the --no-shuffle flag. The resulting RNAz structured windows were then clustered with rnazCluster.pl, filtered with rnazFilter.pl at a P&gt;0.9 threshold, and sorted with rnazSort.pl.</p><p id="p-54">We ran alifoldz<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> on the same genome windows used with RNAz above, again using SARSr-MSA-1. The alifoldz z-score computations were calculated for the forward strand only with alifoldz.pl. We additionally calculated alifoldz z-scores for alignment windows that were shuffled with shuffle-aln.pl to assess background z-scores, determining that 1% of shuffled alignment z-scores were less than −2.69.</p><p id="p-55">We computed alignment powers with R-scape<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> to assess the potential for using SARSr-MSA-1 for covariation analysis. We generated Stockholm alignment files with biopython<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> for windows of 120 nucleotides each sliding by 40 nucleotides. For each window, we ran R-scape with the –fold flag to predict new structures, obtaining estimates for the power of each base pair in the predicted structure (here, power is the expected sensitivity for detecting base pairs given the number of substitutions in the alignment at that base pair). We then averaged across base pairs in each structure to obtain the alignment power as described in Rivas, et al.,<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup> noting that all windows’ alignment powers fell below the 0.10 threshold used by Rivas, et al.<sup><a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a></sup> to distinguish low-power from high-power alignments.</p></div><div id="sec-11" class="subsection"><h3>Analysis of unstructured elements</h3><p id="p-56">To obtain probabilities that each genome position in SARS-CoV-2 was unpaired, we computed basepairing probability matrices with Contrafold 2.0<sup><a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a></sup> in windows of 120 nucleotides sliding by 40 nucleotides, and for each genome position, we summed the probabilities of pairing with all potential partners. We then averaged these nucleotide pairing probabilities across all windows that nucleotide was present in. Additionally, RNAplfold<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup> was run with window size 120 nucleotides, producing another set of unpaired probabilities for each position in the genome.</p></div><div id="sec-12" class="subsection"><h3>Code and data availability</h3><p id="p-57">Code used for the conservation, structured, and unstructured analyses above can be found at the GitHub repository: <a href="https://github.com/DasLab/SARSCoV2_Secstruct_Cons">https://github.com/DasLab/SARSCoV2_Secstruct_Cons</a>. The repository additionally includes alignment files, Rfam families and covariance models, and output from the RNAz, R-scape, alifoldz and RNAplfold analyses.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">The authors acknowledge support from the National Science Foundation Graduate Research Fellowship Program under grant no. 1650114 (R.R.), a Stanford Graduate Fellowship (I.N.Z.), and NIH grant MIRA R35 GM122579 (to R.D.). We would like to thank Dr. Rachel H. Saluti, Dr. Edward A. Pham, and Dr. Jeffrey S. Glenn for providing advice on the conserved, structured intervals desirable for antisense oligonucleotide design; Hannah Wayment-Steele for useful discussions on secondary structure modeling; Andrew M. Watkins for updates to the RiboDraw software for rendering secondary structures; and Dr. Paul Gardner for providing rapid reviews of the initial preprint of this manuscript on bioRXiv.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.1" data-doi="10.1007/978-1-4939-2438-7_1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">A. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-article-title">Coronaviruses: an overview of their replication and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">1282</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol%26rft.volume%253D1282%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2438-7_1%26rft_id%253Dinfo%253Apmid%252F25720466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2438-7_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25720466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Connelly</span>,  <span class="cit-name-given-names">C. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Moon</span>,  <span class="cit-name-given-names">M. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schneekloth</span>,  <span class="cit-name-given-names">J. S.</span>,   <span class="cit-name-suffix">Jr</span>.</span>., <span class="cit-article-title">The Emerging Role of RNA as a Therapeutic Target for Small Molecules</span>. <abbr class="cit-jnl-abbrev">Cell Chem Biol</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">23</span> (<span class="cit-issue">9</span>), <span class="cit-fpage">1077</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BChem%2BBiol%26rft.volume%253D23%26rft.spage%253D1077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.27.012906v2.3" data-doi="10.1016/j.antiviral.2007.12.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spurgers</span>,  <span class="cit-name-given-names">K. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sharkey</span>,  <span class="cit-name-given-names">C. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Warfield</span>,  <span class="cit-name-given-names">K. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bavari</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-article-title">Oligonucleotide antiviral therapeutics: antisense and RNA interference for highly pathogenic RNA viruses</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">78</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">26</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DSpurgers%26rft.auinit1%253DK.%2BB.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D26%26rft.epage%253D36%26rft.atitle%253DOligonucleotide%2Bantiviral%2Btherapeutics%253A%2Bantisense%2Band%2BRNA%2Binterference%2Bfor%2Bhighly%2Bpathogenic%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2007.12.008%26rft_id%253Dinfo%253Apmid%252F18258313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2007.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18258313&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255163300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">C. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Krainer</span>,  <span class="cit-name-given-names">A. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cleveland</span>,  <span class="cit-name-given-names">D. W.</span></span>, <span class="cit-article-title">Antisense Oligonucleotide Therapies for Neurodegenerative Diseases</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Neurosci</abbr>. <span class="cit-pub-date">2019</span>, <span class="cit-vol">42</span>, <span class="cit-fpage">385</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BNeurosci%26rft.volume%253D42%26rft.spage%253D385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.5" data-doi="10.1016/j.virusres.2015.02.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-article-title">The structure and functions of coronavirus genomic 3’ and 5’ ends</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">206</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D206%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2015.02.025%26rft_id%253Dinfo%253Apmid%252F25736566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2015.02.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25736566&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.6" data-doi="10.1128/JVI.79.10.6312-6324.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van den Born</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Posthuma</span>,  <span class="cit-name-given-names">C. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gultyaev</span>,  <span class="cit-name-given-names">A. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E. J.</span></span>, <span class="cit-article-title">Discontinuous subgenomic RNA synthesis in arteriviruses is guided by an RNA hairpin structure located in the genomic leader region</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2005</span>, <span class="cit-vol">79</span> (<span class="cit-issue">10</span>), <span class="cit-fpage">6312</span>–<span class="cit-lpage">6324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253Dvan%2Bden%2BBorn%26rft.auinit1%253DE.%26rft.volume%253D79%26rft.issue%253D10%26rft.spage%253D6312%26rft.epage%253D6324%26rft.atitle%253DDiscontinuous%2BSubgenomic%2BRNA%2BSynthesis%2Bin%2BArteriviruses%2BIs%2BGuided%2Bby%2Ban%2BRNA%2BHairpin%2BStructure%2BLocated%2Bin%2Bthe%2BGenomic%2BLeader%2BRegion%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.10.6312-6324.2005%26rft_id%253Dinfo%253Apmid%252F15858015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OS8xMC82MzEyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wMy4yNy4wMTI5MDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.7" data-doi="10.2741/3046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plant</span>,  <span class="cit-name-given-names">E. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-article-title">The role of programmed-1 ribosomal frameshifting in coronavirus propagation</span>. <abbr class="cit-jnl-abbrev">Front. Biosci</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">13</span>, <span class="cit-fpage">4873</span>–<span class="cit-lpage">4881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bbioscience%2B%253A%2B%2Ba%2Bjournal%2Band%2Bvirtual%2Blibrary%26rft.stitle%253DFront%2BBiosci%26rft.aulast%253DPlant%26rft.auinit1%253DE.%2BP.%26rft.volume%253D13%26rft.spage%253D4873%26rft.epage%253D4881%26rft.atitle%253DThe%2Brole%2Bof%2Bprogrammed-1%2Bribosomal%2Bframeshifting%2Bin%2Bcoronavirus%2Bpropagation.%26rft_id%253Dinfo%253Adoi%252F10.2741%252F3046%26rft_id%253Dinfo%253Apmid%252F18508552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2741/3046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18508552&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255885000126&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.8" data-doi="10.1261/rna.2816711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stammler</span>,  <span class="cit-name-given-names">S. N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.</span></span>-J.; <span class="cit-auth"><span class="cit-name-surname">Giedroc</span>,  <span class="cit-name-given-names">D. P.</span></span>, <span class="cit-article-title">A conserved RNA pseudoknot in a putative molecular switch domain of the 3’-untranslated region of coronaviruses is only marginally stable</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">17</span> (<span class="cit-issue">9</span>), <span class="cit-fpage">1747</span>–<span class="cit-lpage">1759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2816711%26rft_id%253Dinfo%253Apmid%252F21799029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE3LzkvMTc0NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA2LzIwMjAuMDMuMjcuMDEyOTA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.9" data-doi="10.1016/j.virol.2010.02.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.-C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Olsthoorn</span>,  <span class="cit-name-given-names">R. C. L.</span></span>, <span class="cit-article-title">Group-specific structural features of the 5’-proximal sequences of coronavirus genomic RNAs</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">401</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">29</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D401%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2010.02.007%26rft_id%253Dinfo%253Apmid%252F20202661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2010.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20202661&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ceraolo</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Giorgi</span>,  <span class="cit-name-given-names">F. M.</span></span>, <span class="cit-article-title">Genomic variance of the 2019-nCoV coronavirus</span>. <abbr class="cit-jnl-abbrev">J. Med. Virol</abbr>. <span class="cit-pub-date">2020</span>, <span class="cit-vol">92</span> (<span class="cit-issue">5</span>), <span class="cit-fpage">522</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BVirol%26rft.volume%253D92%26rft.spage%253D522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.27.012906v2.11"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed cit-collab">N. C. f. B. I</span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z.-G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">Z.-W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">J.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">Y.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">M. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">F.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">J.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-Z.</span></span>, <span class="cit-source">NC_045512. Nucleotide [Internet]</span>. <span class="cit-publ-loc">Bethesda (MD)</span>: <span class="cit-publ-name">National Library of Medicine (US)</span>, <span class="cit-ed cit-collab">N. C. f. B. I</span>., Ed. <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.12"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Viruses, C. S. G. o. T. I. C. o. T. o</span>.; <span class="cit-article-title">Coronaviridae Study Group of the International Committee on Taxonomy of, V., The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">5</span> (<span class="cit-issue">4</span>), <span class="cit-fpage">536</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft.volume%253D5%26rft.spage%253D536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.13" data-doi="10.1016/j.virusres.2014.10.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Madhugiri</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fricke</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Marz</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-article-title">RNA structure analysis of alphacoronavirus terminal genome regions</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">194</span>, <span class="cit-fpage">76</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D194%26rft.spage%253D76%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.10.001%26rft_id%253Dinfo%253Apmid%252F25307890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25307890&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.14" data-doi="10.1093/nar/gkx1038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kalvari</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Argasinska</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Quinones-Olvera</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nawrocki</span>,  <span class="cit-name-given-names">E. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rivas</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eddy</span>,  <span class="cit-name-given-names">S. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bateman</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Finn</span>,  <span class="cit-name-given-names">R. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">A. I.</span></span>, <span class="cit-article-title">Rfam 13.0: shifting to a genome-centric resource for noncoding RNA families</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2018</span>, <span class="cit-vol">46</span> (<span class="cit-issue">D1</span>), <span class="cit-fpage">D335</span>–<span class="cit-lpage">D342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD335%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1038%26rft_id%253Dinfo%253Apmid%252F29112718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29112718&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.15" data-doi="10.1093/bioinformatics/btt509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nawrocki</span>,  <span class="cit-name-given-names">E. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eddy</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-article-title">Infernal 1.1: 100-fold faster RNA homology searches</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">29</span> (<span class="cit-issue">22</span>), <span class="cit-fpage">2933</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt509%26rft_id%253Dinfo%253Apmid%252F24008419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24008419&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326643600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wudeck</span>,  <span class="cit-name-given-names">E. V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Giedroc</span>,  <span class="cit-name-given-names">D. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-article-title">SHAPE analysis of the RNA secondary structure of the Mouse Hepatitis Virus 5’ untranslated region and N-terminal nsp1 coding sequences</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">475</span>, <span class="cit-fpage">15</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D475%26rft.spage%253D15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.17" data-doi="10.1002/gch2.1018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elbe</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Buckland-Merrett</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Data, disease and diplomacy: GISAID’s innovative contribution to global health</span>. <abbr class="cit-jnl-abbrev">Glob Chall</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">1</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">33</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChall%26rft.volume%253D1%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgch2.1018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gch2.1018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.18" data-doi="10.1093/bioinformatics/btl246"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Do</span>,  <span class="cit-name-given-names">C. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Woods</span>,  <span class="cit-name-given-names">D. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Batzoglou</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-article-title">CONTRAfold: RNA secondary structure prediction without physics-based models</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">22</span> (<span class="cit-issue">14</span>), <span class="cit-fpage">e90</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DDo%26rft.auinit1%253DC.%2BB.%26rft.volume%253D22%26rft.issue%253D14%26rft.spage%253De90%26rft.epage%253De98%26rft.atitle%253DCONTRAfold%253A%2BRNA%2Bsecondary%2Bstructure%2Bprediction%2Bwithout%2Bphysics-based%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl246%26rft_id%253Dinfo%253Apmid%252F16873527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16873527&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250005000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.19" data-doi="10.1186/1471-2105-11-586"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hamada</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Asai</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-article-title">Prediction of RNA secondary structure by maximizing pseudo-expected accuracy</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">11</span>, <span class="cit-fpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHamada%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.spage%253D586%26rft.epage%253D586%26rft.atitle%253DPrediction%2Bof%2BRNA%2Bsecondary%2Bstructure%2Bby%2Bmaximizing%2Bpseudo-expected%2Baccuracy.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-586%26rft_id%253Dinfo%253Apmid%252F21118522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21118522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.20" data-doi="10.1093/nar/21.25.5838"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herold</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Siddell</span>,  <span class="cit-name-given-names">S. G.</span></span>, <span class="cit-article-title">An ‘elaborated’ pseudoknot is required for high frequency frameshifting during translation of HCV 229E polymerase mRNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">1993</span>, <span class="cit-vol">21</span> (<span class="cit-issue">25</span>), <span class="cit-fpage">5838</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHerald%26rft.auinit1%253DJ.%26rft.volume%253D21%26rft.issue%253D25%26rft.spage%253D5838%26rft.epage%253D5842%26rft.atitle%253DAn%2B%2527elaborated%2527%2Bpseudoknot%2Bis%2Brequired%2Bfor%2Bhigh%2Bfrequency%2Bframeshifting%2Bduring%2Btranslation%2Bof%2BHCV%2B229E%2Bpolymerase%2BmRNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F21.25.5838%26rft_id%253Dinfo%253Apmid%252F8290341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/21.25.5838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8290341&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MR19900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gruber</span>,  <span class="cit-name-given-names">A. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Findeiß</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Washietl</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hofacker</span>,  <span class="cit-name-given-names">I. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stadler</span>,  <span class="cit-name-given-names">P. F.</span></span>, <span class="cit-article-title">RNAz 2.0: improved noncoding RNA detection</span>. <abbr class="cit-jnl-abbrev">Pac. Symp. Biocomput</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-fpage">69</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rivas</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Clements</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eddy</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-article-title">Estimating the power of sequence covariation for detecting conserved RNA structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.23" data-doi="10.1016/j.jmb.2008.01.068"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kang</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Makkinje</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Williamson</span>,  <span class="cit-name-given-names">S. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>,  <span class="cit-name-given-names">J. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Giedroc</span>,  <span class="cit-name-given-names">D. P.</span></span>, <span class="cit-article-title">Structural lability in stem-loop 1 drives a 5’ UTR-3’ UTR interaction in coronavirus replication</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">377</span> (<span class="cit-issue">3</span>), <span class="cit-fpage">790</span>–<span class="cit-lpage">803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D377%26rft.issue%253D3%26rft.spage%253D790%26rft.epage%253D803%26rft.atitle%253DStructural%2Blability%2Bin%2Bstem-loop%2B1%2Bdrives%2Ba%2B5%2527%2BUTR-3%2527%2BUTR%2Binteraction%2Bin%2Bcoronavirus%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2008.01.068%26rft_id%253Dinfo%253Apmid%252F18289557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2008.01.068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18289557&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.24" data-doi="10.1261/rna.261807"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Millership</span>,  <span class="cit-name-given-names">J. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kang</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>,  <span class="cit-name-given-names">J. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Giedroc</span>,  <span class="cit-name-given-names">D. P.</span></span>, <span class="cit-article-title">A U-turn motifcontaining stem-loop in the coronavirus 5’ untranslated region plays a functional role in replication</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">13</span> (<span class="cit-issue">5</span>), <span class="cit-fpage">763</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.261807%26rft_id%253Dinfo%253Apmid%252F17353353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjEzLzUvNzYzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wMy4yNy4wMTI5MDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">J. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-article-title">Structural and functional conservation of the programmed-1 ribosomal frameshift signal of SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.26" data-doi="10.1128/JVI.78.2.669-682.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goebel</span>,  <span class="cit-name-given-names">S. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hsue</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dombrowski</span>,  <span class="cit-name-given-names">T. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Masters</span>,  <span class="cit-name-given-names">P. S.</span></span>, <span class="cit-article-title">Characterization of the RNA components of a putative molecular switch in the 3’ untranslated region of the murine coronavirus genome</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2004</span>, <span class="cit-vol">78</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">669</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGoebel%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D669%26rft.epage%253D682%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BRNA%2BComponents%2Bof%2Ba%2BPutative%2BMolecular%2BSwitch%2Bin%2Bthe%2B3%2527%2BUntranslated%2BRegion%2Bof%2Bthe%2BMurine%2BCoronavirus%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.2.669-682.2004%26rft_id%253Dinfo%253Apmid%252F14694098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijc4LzIvNjY5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wMy4yNy4wMTI5MDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.27" data-doi="10.1128/JVI.00803-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goebel</span>,  <span class="cit-name-given-names">S. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">T. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">C. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bernard</span>,  <span class="cit-name-given-names">K. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Masters</span>,  <span class="cit-name-given-names">P. S.</span></span>, <span class="cit-article-title">A hypervariable region within the 3’ cis-acting element of the murine coronavirus genome is nonessential for RNA synthesis but affects pathogenesis</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2007</span>, <span class="cit-vol">81</span> (<span class="cit-issue">3</span>), <span class="cit-fpage">1274</span>–<span class="cit-lpage">1287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00803-06%26rft_id%253Dinfo%253Apmid%252F17093194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzMvMTI3NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA2LzIwMjAuMDMuMjcuMDEyOTA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.28" data-doi="10.1371/journal.pbio.0030005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">M. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Igel</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Baertsch</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Haussler</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ares</span>,  <span class="cit-name-given-names">M.</span>,   <span class="cit-name-suffix">Jr</span>.</span>.; <span class="cit-auth"><span class="cit-name-surname">Scott</span>,  <span class="cit-name-given-names">W. G.</span></span>, <span class="cit-article-title">The structure of a rigorously conserved RNA element within the SARS virus genome</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-pub-date">2005</span>, <span class="cit-vol">3</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">e5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DRobertson%26rft.auinit1%253DM.%2BP.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253De5%26rft.epage%253De5%26rft.atitle%253DThe%2Bstructure%2Bof%2Ba%2Brigorously%2Bconserved%2BRNA%2Belement%2Bwithin%2Bthe%2BSARS%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030005%26rft_id%253Dinfo%253Apmid%252F15630477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15630477&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jonassen</span>,  <span class="cit-name-given-names">C. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jonassen</span>,  <span class="cit-name-given-names">T. O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grinde</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-article-title">A common RNA motif in the 3’ end of the genomes of astroviruses, avian infectious bronchitis virus and an equine rhinovirus</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr>. <span class="cit-pub-date">1998</span>, <span class="cit-vol">79</span> (Pt <span class="cit-issue">4</span>), <span class="cit-fpage">715</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DJonassen%26rft.auinit1%253DC.%2BM.%26rft.volume%253D79%26rft.issue%253D4%26rft.spage%253D715%26rft.epage%253D718%26rft.atitle%253DA%2Bcommon%2BRNA%2Bmotif%2Bin%2Bthe%2B3%2527%2Bend%2Bof%2Bthe%2Bgenomes%2Bof%2Bastroviruses%252C%2Bavian%2Binfectious%2Bbronchitis%2Bvirus%2Band%2Ban%2Bequine%2Brhinovirus%26rft_id%253Dinfo%253Apmid%252F9568965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9568965&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072889400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.30" data-doi="10.1128/JVI.06265-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">B.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">Y.-P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brian</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-article-title">Genetic evidence of a long-range RNA-RNA interaction between the genomic 5’ untranslated region and the nonstructural protein 1 coding region in murine and bovine coronaviruses</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2012</span>, <span class="cit-vol">86</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">4631</span>–<span class="cit-lpage">4643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06265-11%26rft_id%253Dinfo%253Apmid%252F22345457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzgvNDYzMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA2LzIwMjAuMDMuMjcuMDEyOTA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kappel</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kladwang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pintilie</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Topkar</span>,  <span class="cit-name-given-names">V. V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rangan</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zheludev</span>,  <span class="cit-name-given-names">I. N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Watkins</span>,  <span class="cit-name-given-names">A. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yesselman</span>,  <span class="cit-name-given-names">J. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chiu</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Das</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-article-title">Ribosolve: Rapid determination of three-dimensional RNA-only structures</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kappel</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pintilie</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mou</span>,  <span class="cit-name-given-names">T.-C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schmid</span>,  <span class="cit-name-given-names">M. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Das</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chiu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-article-title">Cryo-EM structure of a 40 kDa SAM-IV riboswitch RNA at 3.7 Å resolution</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-pub-date">2019</span>, <span class="cit-vol">10</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">5511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D10%26rft.spage%253D5511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.33" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span>, <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2004</span>, <span class="cit-vol">32</span> (<span class="cit-issue">5</span>), <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.34" data-doi="10.1093/bioinformatics/17.3.282"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jaroszewski</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Godzik</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-article-title">Clustering of highly homologous sequences to reduce the size of large protein databases</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2001</span>, <span class="cit-vol">17</span> (<span class="cit-issue">3</span>), <span class="cit-fpage">282</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D282%26rft.epage%253D283%26rft.atitle%253DClustering%2Bof%2Bhighly%2Bhomologous%2Bsequences%2Bto%2Breduce%2Bthe%2Bsize%2Bof%2B%2Blarge%2Bprotein%2Bdatabases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.3.282%26rft_id%253Dinfo%253Apmid%252F11294794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.3.282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11294794&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168053800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.35" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wilm</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dineen</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gibson</span>,  <span class="cit-name-given-names">T. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Karplus</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lopez</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McWilliam</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Remmert</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Söding</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">J. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Higgins</span>,  <span class="cit-name-given-names">D. G.</span></span>, <span class="cit-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol</abbr>. <span class="cit-pub-date">2011</span>, <span class="cit-vol">7</span>, <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol%26rft.volume%253D7%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2011.75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21988835&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.36" data-doi="10.1093/bioinformatics/bts578"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Frith</span>,  <span class="cit-name-given-names">M. C.</span></span>, <span class="cit-article-title">Adding unaligned sequences into an existing alignment using MAFFT and LAST</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">28</span> (<span class="cit-issue">23</span>), <span class="cit-fpage">3144</span>–<span class="cit-lpage">3146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts578%26rft_id%253Dinfo%253Apmid%252F23023983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts578&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23023983&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311902700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.37" data-doi="10.1016/j.jmb.2004.07.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Washietl</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hofacker</span>,  <span class="cit-name-given-names">I. L.</span></span>, <span class="cit-article-title">Consensus folding of aligned sequences as a new measure for the detection of functional RNAs by comparative genomics</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">342</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">19</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWashietl%26rft.auinit1%253DS.%26rft.volume%253D342%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D30%26rft.atitle%253DConsensus%2Bfolding%2Bof%2Baligned%2Bsequences%2Bas%2Ba%2Bnew%2Bmeasure%2Bfor%2Bthe%2Bdetection%2Bof%2Bfunctional%2BRNAs%2Bby%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.07.018%26rft_id%253Dinfo%253Apmid%252F15313604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.07.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15313604&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223578800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.38" data-doi="10.1038/nmeth.4066"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rivas</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Clements</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eddy</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-article-title">A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">14</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">45</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.39" data-doi="10.1093/bioinformatics/btp163"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cock</span>,  <span class="cit-name-given-names">P. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Antao</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">J. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chapman</span>,  <span class="cit-name-given-names">B. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">C. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dalke</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Friedberg</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hamelryck</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kauff</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wilczynski</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">de Hoon</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-article-title">Biopython: freely available Python tools for computational molecular biology and bioinformatics</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">25</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">1422</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp163%26rft_id%253Dinfo%253Apmid%252F19304878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19304878&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266109500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.40" data-doi="10.1093/bioinformatics/btk014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernhart</span>,  <span class="cit-name-given-names">S. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hofacker</span>,  <span class="cit-name-given-names">I. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stadler</span>,  <span class="cit-name-given-names">P. F.</span></span>, <span class="cit-article-title">Local RNA base pairing probabilities in large sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">22</span> (<span class="cit-issue">5</span>), <span class="cit-fpage">614</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtk014%26rft_id%253Dinfo%253Apmid%252F16368769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btk014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16368769&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235604400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.012906v2.41" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearse</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Moir</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheung</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sturrock</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Buxton</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Markowitz</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Duran</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thierer</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ashton</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Meintjes</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">28</span> (<span class="cit-issue">12</span>), <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.27.012906.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.27.012906v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.23.004135v2" title="Paternal exposure to the pesticide DDT induces intergenerational programming of breast cancer predisposition via sperm miRNA" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.06.017079v2" title="Transcriptome and regulatory maps of decidua-derived stromal cells inform gene discovery in preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 06, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.27.012906v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.27.012906v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.27.012906v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.27.012906v2" class="active" data-icon-position="" data-hide-link-title="0">RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1231855" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-e41HnBhbGCLdD6JmQEXKvypC-flRhLG0qvnc_4pSRek" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231855" id="node1231855--2" data-pisa="biorxiv;2020.03.27.012906v2" data-pisa-master="biorxiv;2020.03.27.012906" data-seqnum="1231855" data-apath="/biorxiv/early/2020/04/06/2020.03.27.012906.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ramya</span> <span class="nlm-surname">Rangan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ivan N.</span> <span class="nlm-surname">Zheludev</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rhiju</span> <span class="nlm-surname">Das</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.27.012906; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.27.012906 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;title=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;title=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;text=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;title=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;t=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;title=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;title=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231855" id="node1231855--3" data-pisa="biorxiv;2020.03.27.012906v2" data-pisa-master="biorxiv;2020.03.27.012906" data-seqnum="1231855" data-apath="/biorxiv/early/2020/04/06/2020.03.27.012906.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ramya</span> <span class="nlm-surname">Rangan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ivan N.</span> <span class="nlm-surname">Zheludev</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rhiju</span> <span class="nlm-surname">Das</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.27.012906; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.27.012906 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Rangan%2C+Ramya&amp;rft.au=Zheludev%2C+Ivan+N.&amp;rft.au=Das%2C+Rhiju"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1231855/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1231855/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1231855/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1231855/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1231855/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1231855/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1231855/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1231855/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1231855/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1231855/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1231855/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1231855/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;count=horizontal&amp;via=&amp;text=RNA%20genome%20conservation%20and%20secondary%20structure%20in%20SARS-CoV-2%20and%20SARS-related%20viruses&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.012906v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.27.012906v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genetics" class="highlight" data-icon-position="" data-hide-link-title="0">Genetics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
